<!DOCTYPE html>
<html lang="en">

  <head>
    <title>CancerFeed</title>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="robots" content="noindex, nofollow" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
    <link rel="alternate" type="application/rss+xml" title="CancerFeed" href="feed.atom" />
    <link href="index.css?v1.14.4" rel="stylesheet" />
    <style>
  :root {
    /**
     * Solarized Light
     * Ethan Schoonover (modified by aramisgithub)
     */
    --base00: #fdf6e3;
    --base01: #eee8d5;
    --base02: #93a1a1;
    --base03: #839496;
    --base04: #657b83;
    --base05: #586e75;
    --base06: #073642;
    --base07: #002b36;
    --base08: #dc322f;
    --base09: #cb4b16;
    --base0A: #b58900;
    --base0B: #859900;
    --base0C: #2aa198;
    --base0D: #268bd2;
    --base0E: #6c71c4;
    --base0F: #d33682;

    /**
    * Components
    */
    --source-name-color: var(--base0C);
    /**
   * Typograph
   */
    --font-family-default: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell,
      "Helvetica Neue", sans-serif;
    --font-size-L: 3.6rem;
    --font-size-m: 2.2rem;
    --font-size-s: 1.8rem;
    --font-size-scaler: 52.5%; /* 1rem = 10px */
    --line-height-body: 1.4;
    --line-height-heading: 1.2;

  }
</style>

  </head>

  <body>
    <!-- %after-body-begin.html% -->
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-08-10 datetime="2023-08-10T10:00:00.000Z">2023-08-10</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-08-10 datetime="2023-08-10T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37561983"
                        open
                      >
                        <summary class="article-expander__title">Associations Between Cancer Predisposition Mutations and Clonal Hematopoiesis in Patients With Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37561983/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our results suggest a close relationship between inherited variants and CH mutations within the DDR genes in patients with solid tumors. Associations identified in this study might translate into enhanced clinical surveillance for CH and associated comorbidities in patients with cancer harboring these germline mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37560406"
                        open
                      >
                        <summary class="article-expander__title">Transient abnormal myelopoiesis with extramedullary involvement in a down syndrome preemie leading to an unresponsive course despite chemotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37560406/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Transient abnormal myelopoiesis (TAM) is a transient, clonal myeloproliferative disorder unique to Down Syndrome (DS) babies. It is characterized by increased peripheral blasts and presence of GATA1 mutation. The clinical spectrum ranges from jaundice and hepatosplenomegaly to multi-organ failure and death. Here we present a case of a premature baby with DS diagnosed to have TAM with extramedullary involvement at birth who had a fatal outcome.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-08-07 datetime="2023-08-07T10:00:00.000Z">2023-08-07</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-08-07 datetime="2023-08-07T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37550517"
                        open
                      >
                        <summary class="article-expander__title">Different niches for stem cells carrying the same oncogenic driver affect pathogenesis and therapy response in myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37550517/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging facilitates the expansion of hematopoietic stem cells (HSCs) carrying clonal hematopoiesis-related somatic mutations and the development of myeloid malignancies, such as myeloproliferative neoplasms (MPNs). While cooperating mutations can cause transformation, it is unclear whether distinct bone marrow (BM) HSC-niches can influence the growth and therapy response of HSCs carrying the same oncogenic driver. Here we found different BM niches for HSCs in MPN subtypes. JAK-STAT signaling...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37549687"
                        open
                      >
                        <summary class="article-expander__title">Short-term western diet causes rapid and lasting alterations of bone marrow physiology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37549687/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Dietary factors have a significant impact on overall health and western diet (WD) disrupts arterial homeostasis, thereby promoting atherosclerosis. In our study, we investigated the effects of short-term WD on bone marrow vasculature and hematopoiesis in hypercholesterolemic mice. We found that WD rapidly remodels bone marrow arterioles, and these alterations persist even after WD cessation. The proximity between hematopoietic stem cells (HSCs) and arterioles increased with short-term WD, which...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37547364"
                        open
                      >
                        <summary class="article-expander__title">Immune dysregulation and potential targeted therapy in myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37547364/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a heterogeneous group of clonal hematological diseases and a high risk for transformation to acute myeloid leukemia (AML). The identification of key genetic alterations in MDS has enhanced our understanding of the pathogenesis and evolution. In recent years, it has been found that both innate and adaptive immune signaling are activated in the hematopoietic niche of MDS with aberrant cytokine secretion in the bone marrow microenvironment. It is also clear that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37546840"
                        open
                      >
                        <summary class="article-expander__title">Leukocytes carrying Clonal Hematopoiesis of Indeterminate Potential (CHIP) Mutations invade Human Atherosclerotic Plaques</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37546840/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Deep-DNA-sequencing reveals a high prevalence of CHIP mutations in whole blood of CAD patients. CHIP-affected leukocytes invade plaques in human coronary arteries. RNAseq data obtained from macrophages of CHIP-affected patients suggest that pro-atherosclerotic signaling differs depending on the underlying mutations. Further studies are necessary to understand whether specific pathways affected by CHIP mutations may be targeted for personalized treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37546389"
                        open
                      >
                        <summary class="article-expander__title">Cigarette smoke stimulates clonal expansion of Jak2V617F and Tet2-/- cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37546389/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Altogether, these findings demonstrate that CS results in an inflamed bone marrow environment that provides a selection pressure for existing CHIP mutations such as Jak2V617F and Tet2 loss-of-function.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37550517"
                        open
                      >
                        <summary class="article-expander__title">Different niches for stem cells carrying the same oncogenic driver affect pathogenesis and therapy response in myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37550517/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging facilitates the expansion of hematopoietic stem cells (HSCs) carrying clonal hematopoiesis-related somatic mutations and the development of myeloid malignancies, such as myeloproliferative neoplasms (MPNs). While cooperating mutations can cause transformation, it is unclear whether distinct bone marrow (BM) HSC-niches can influence the growth and therapy response of HSCs carrying the same oncogenic driver. Here we found different BM niches for HSCs in MPN subtypes. JAK-STAT signaling...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37549687"
                        open
                      >
                        <summary class="article-expander__title">Short-term western diet causes rapid and lasting alterations of bone marrow physiology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37549687/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Dietary factors have a significant impact on overall health and western diet (WD) disrupts arterial homeostasis, thereby promoting atherosclerosis. In our study, we investigated the effects of short-term WD on bone marrow vasculature and hematopoiesis in hypercholesterolemic mice. We found that WD rapidly remodels bone marrow arterioles, and these alterations persist even after WD cessation. The proximity between hematopoietic stem cells (HSCs) and arterioles increased with short-term WD, which...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37547364"
                        open
                      >
                        <summary class="article-expander__title">Immune dysregulation and potential targeted therapy in myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37547364/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a heterogeneous group of clonal hematological diseases and a high risk for transformation to acute myeloid leukemia (AML). The identification of key genetic alterations in MDS has enhanced our understanding of the pathogenesis and evolution. In recent years, it has been found that both innate and adaptive immune signaling are activated in the hematopoietic niche of MDS with aberrant cytokine secretion in the bone marrow microenvironment. It is also clear that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37546840"
                        open
                      >
                        <summary class="article-expander__title">Leukocytes carrying Clonal Hematopoiesis of Indeterminate Potential (CHIP) Mutations invade Human Atherosclerotic Plaques</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37546840/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Deep-DNA-sequencing reveals a high prevalence of CHIP mutations in whole blood of CAD patients. CHIP-affected leukocytes invade plaques in human coronary arteries. RNAseq data obtained from macrophages of CHIP-affected patients suggest that pro-atherosclerotic signaling differs depending on the underlying mutations. Further studies are necessary to understand whether specific pathways affected by CHIP mutations may be targeted for personalized treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37546389"
                        open
                      >
                        <summary class="article-expander__title">Cigarette smoke stimulates clonal expansion of Jak2V617F and Tet2-/- cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37546389/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Altogether, these findings demonstrate that CS results in an inflamed bone marrow environment that provides a selection pressure for existing CHIP mutations such as Jak2V617F and Tet2 loss-of-function.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37550517"
                        open
                      >
                        <summary class="article-expander__title">Different niches for stem cells carrying the same oncogenic driver affect pathogenesis and therapy response in myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37550517/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging facilitates the expansion of hematopoietic stem cells (HSCs) carrying clonal hematopoiesis-related somatic mutations and the development of myeloid malignancies, such as myeloproliferative neoplasms (MPNs). While cooperating mutations can cause transformation, it is unclear whether distinct bone marrow (BM) HSC-niches can influence the growth and therapy response of HSCs carrying the same oncogenic driver. Here we found different BM niches for HSCs in MPN subtypes. JAK-STAT signaling...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37549687"
                        open
                      >
                        <summary class="article-expander__title">Short-term western diet causes rapid and lasting alterations of bone marrow physiology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37549687/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Dietary factors have a significant impact on overall health and western diet (WD) disrupts arterial homeostasis, thereby promoting atherosclerosis. In our study, we investigated the effects of short-term WD on bone marrow vasculature and hematopoiesis in hypercholesterolemic mice. We found that WD rapidly remodels bone marrow arterioles, and these alterations persist even after WD cessation. The proximity between hematopoietic stem cells (HSCs) and arterioles increased with short-term WD, which...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37547364"
                        open
                      >
                        <summary class="article-expander__title">Immune dysregulation and potential targeted therapy in myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37547364/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a heterogeneous group of clonal hematological diseases and a high risk for transformation to acute myeloid leukemia (AML). The identification of key genetic alterations in MDS has enhanced our understanding of the pathogenesis and evolution. In recent years, it has been found that both innate and adaptive immune signaling are activated in the hematopoietic niche of MDS with aberrant cytokine secretion in the bone marrow microenvironment. It is also clear that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37546840"
                        open
                      >
                        <summary class="article-expander__title">Leukocytes carrying Clonal Hematopoiesis of Indeterminate Potential (CHIP) Mutations invade Human Atherosclerotic Plaques</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37546840/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Deep-DNA-sequencing reveals a high prevalence of CHIP mutations in whole blood of CAD patients. CHIP-affected leukocytes invade plaques in human coronary arteries. RNAseq data obtained from macrophages of CHIP-affected patients suggest that pro-atherosclerotic signaling differs depending on the underlying mutations. Further studies are necessary to understand whether specific pathways affected by CHIP mutations may be targeted for personalized treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37546389"
                        open
                      >
                        <summary class="article-expander__title">Cigarette smoke stimulates clonal expansion of Jak2V617F and Tet2-/- cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37546389/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Altogether, these findings demonstrate that CS results in an inflamed bone marrow environment that provides a selection pressure for existing CHIP mutations such as Jak2V617F and Tet2 loss-of-function.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37547364"
                        open
                      >
                        <summary class="article-expander__title">Immune dysregulation and potential targeted therapy in myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37547364/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a heterogeneous group of clonal hematological diseases and a high risk for transformation to acute myeloid leukemia (AML). The identification of key genetic alterations in MDS has enhanced our understanding of the pathogenesis and evolution. In recent years, it has been found that both innate and adaptive immune signaling are activated in the hematopoietic niche of MDS with aberrant cytokine secretion in the bone marrow microenvironment. It is also clear that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37546840"
                        open
                      >
                        <summary class="article-expander__title">Leukocytes carrying Clonal Hematopoiesis of Indeterminate Potential (CHIP) Mutations invade Human Atherosclerotic Plaques</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37546840/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Deep-DNA-sequencing reveals a high prevalence of CHIP mutations in whole blood of CAD patients. CHIP-affected leukocytes invade plaques in human coronary arteries. RNAseq data obtained from macrophages of CHIP-affected patients suggest that pro-atherosclerotic signaling differs depending on the underlying mutations. Further studies are necessary to understand whether specific pathways affected by CHIP mutations may be targeted for personalized treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37546389"
                        open
                      >
                        <summary class="article-expander__title">Cigarette smoke stimulates clonal expansion of Jak2V617F and Tet2-/- cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37546389/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Altogether, these findings demonstrate that CS results in an inflamed bone marrow environment that provides a selection pressure for existing CHIP mutations such as Jak2V617F and Tet2 loss-of-function.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-08-04 datetime="2023-08-04T10:00:00.000Z">2023-08-04</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-08-04 datetime="2023-08-04T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37539077"
                        open
                      >
                        <summary class="article-expander__title">Blockade of IL-6 signaling alleviates atherosclerosis in Tet2-deficient clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37539077/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) increases the risk of atherosclerotic cardiovascular disease possibly due to increased plaque inflammation. Human studies suggest that limitation of interleukin-6 (IL-6) signaling could be beneficial in people with large CH clones, particularly in TET2 CH. Here we show that IL-6 receptor antibody treatment reverses the atherosclerosis promoted by Tet2 CH, with reduction of monocytosis, lesional macrophage burden and macrophage colony-stimulating factor 1 receptor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37539077"
                        open
                      >
                        <summary class="article-expander__title">Blockade of IL-6 signaling alleviates atherosclerosis in Tet2-deficient clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37539077/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) increases the risk of atherosclerotic cardiovascular disease possibly due to increased plaque inflammation. Human studies suggest that limitation of interleukin-6 (IL-6) signaling could be beneficial in people with large CH clones, particularly in TET2 CH. Here we show that IL-6 receptor antibody treatment reverses the atherosclerosis promoted by Tet2 CH, with reduction of monocytosis, lesional macrophage burden and macrophage colony-stimulating factor 1 receptor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37539077"
                        open
                      >
                        <summary class="article-expander__title">Blockade of IL-6 signaling alleviates atherosclerosis in Tet2-deficient clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37539077/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) increases the risk of atherosclerotic cardiovascular disease possibly due to increased plaque inflammation. Human studies suggest that limitation of interleukin-6 (IL-6) signaling could be beneficial in people with large CH clones, particularly in TET2 CH. Here we show that IL-6 receptor antibody treatment reverses the atherosclerosis promoted by Tet2 CH, with reduction of monocytosis, lesional macrophage burden and macrophage colony-stimulating factor 1 receptor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37539077"
                        open
                      >
                        <summary class="article-expander__title">Blockade of IL-6 signaling alleviates atherosclerosis in Tet2-deficient clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37539077/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) increases the risk of atherosclerotic cardiovascular disease possibly due to increased plaque inflammation. Human studies suggest that limitation of interleukin-6 (IL-6) signaling could be beneficial in people with large CH clones, particularly in TET2 CH. Here we show that IL-6 receptor antibody treatment reverses the atherosclerosis promoted by Tet2 CH, with reduction of monocytosis, lesional macrophage burden and macrophage colony-stimulating factor 1 receptor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37539077"
                        open
                      >
                        <summary class="article-expander__title">Blockade of IL-6 signaling alleviates atherosclerosis in Tet2-deficient clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37539077/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) increases the risk of atherosclerotic cardiovascular disease possibly due to increased plaque inflammation. Human studies suggest that limitation of interleukin-6 (IL-6) signaling could be beneficial in people with large CH clones, particularly in TET2 CH. Here we show that IL-6 receptor antibody treatment reverses the atherosclerosis promoted by Tet2 CH, with reduction of monocytosis, lesional macrophage burden and macrophage colony-stimulating factor 1 receptor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37539077"
                        open
                      >
                        <summary class="article-expander__title">Blockade of IL-6 signaling alleviates atherosclerosis in Tet2-deficient clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37539077/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) increases the risk of atherosclerotic cardiovascular disease possibly due to increased plaque inflammation. Human studies suggest that limitation of interleukin-6 (IL-6) signaling could be beneficial in people with large CH clones, particularly in TET2 CH. Here we show that IL-6 receptor antibody treatment reverses the atherosclerosis promoted by Tet2 CH, with reduction of monocytosis, lesional macrophage burden and macrophage colony-stimulating factor 1 receptor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37539077"
                        open
                      >
                        <summary class="article-expander__title">Blockade of IL-6 signaling alleviates atherosclerosis in Tet2-deficient clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37539077/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) increases the risk of atherosclerotic cardiovascular disease possibly due to increased plaque inflammation. Human studies suggest that limitation of interleukin-6 (IL-6) signaling could be beneficial in people with large CH clones, particularly in TET2 CH. Here we show that IL-6 receptor antibody treatment reverses the atherosclerosis promoted by Tet2 CH, with reduction of monocytosis, lesional macrophage burden and macrophage colony-stimulating factor 1 receptor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37541246"
                        open
                      >
                        <summary class="article-expander__title">The winner takes it all: Competition drives clonal selection in gliomagenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37541246/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The exact mechanisms that govern clonal dynamics and selection during early tumorigenesis remain largely elusive. Ceresa et al. provide experimental and mathematical evidence that MYC-dependent competition between individual clones is one driving force of brain tumor evolution, adding a winner/loser aspect to the picture that complements existing concepts.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37541243"
                        open
                      >
                        <summary class="article-expander__title">Early clonal extinction in glioblastoma progression revealed by genetic barcoding</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37541243/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastoma progression in its early stages remains poorly understood. Here, we transfer PDGFB and genetic barcodes in mouse brain to initiate gliomagenesis and enable direct tracing of glioblastoma evolution from its earliest possible stage. Unexpectedly, we observe a high incidence of clonal extinction events and progressive divergence in clonal sizes, even after the acquisition of malignant phenotype. Computational modeling suggests these dynamics result from clonal-based cell-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37541246"
                        open
                      >
                        <summary class="article-expander__title">The winner takes it all: Competition drives clonal selection in gliomagenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37541246/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The exact mechanisms that govern clonal dynamics and selection during early tumorigenesis remain largely elusive. Ceresa et al. provide experimental and mathematical evidence that MYC-dependent competition between individual clones is one driving force of brain tumor evolution, adding a winner/loser aspect to the picture that complements existing concepts.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37541243"
                        open
                      >
                        <summary class="article-expander__title">Early clonal extinction in glioblastoma progression revealed by genetic barcoding</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37541243/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastoma progression in its early stages remains poorly understood. Here, we transfer PDGFB and genetic barcodes in mouse brain to initiate gliomagenesis and enable direct tracing of glioblastoma evolution from its earliest possible stage. Unexpectedly, we observe a high incidence of clonal extinction events and progressive divergence in clonal sizes, even after the acquisition of malignant phenotype. Computational modeling suggests these dynamics result from clonal-based cell-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37541246"
                        open
                      >
                        <summary class="article-expander__title">The winner takes it all: Competition drives clonal selection in gliomagenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37541246/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The exact mechanisms that govern clonal dynamics and selection during early tumorigenesis remain largely elusive. Ceresa et al. provide experimental and mathematical evidence that MYC-dependent competition between individual clones is one driving force of brain tumor evolution, adding a winner/loser aspect to the picture that complements existing concepts.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37541243"
                        open
                      >
                        <summary class="article-expander__title">Early clonal extinction in glioblastoma progression revealed by genetic barcoding</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37541243/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastoma progression in its early stages remains poorly understood. Here, we transfer PDGFB and genetic barcodes in mouse brain to initiate gliomagenesis and enable direct tracing of glioblastoma evolution from its earliest possible stage. Unexpectedly, we observe a high incidence of clonal extinction events and progressive divergence in clonal sizes, even after the acquisition of malignant phenotype. Computational modeling suggests these dynamics result from clonal-based cell-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37541246"
                        open
                      >
                        <summary class="article-expander__title">The winner takes it all: Competition drives clonal selection in gliomagenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37541246/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The exact mechanisms that govern clonal dynamics and selection during early tumorigenesis remain largely elusive. Ceresa et al. provide experimental and mathematical evidence that MYC-dependent competition between individual clones is one driving force of brain tumor evolution, adding a winner/loser aspect to the picture that complements existing concepts.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37541243"
                        open
                      >
                        <summary class="article-expander__title">Early clonal extinction in glioblastoma progression revealed by genetic barcoding</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37541243/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastoma progression in its early stages remains poorly understood. Here, we transfer PDGFB and genetic barcodes in mouse brain to initiate gliomagenesis and enable direct tracing of glioblastoma evolution from its earliest possible stage. Unexpectedly, we observe a high incidence of clonal extinction events and progressive divergence in clonal sizes, even after the acquisition of malignant phenotype. Computational modeling suggests these dynamics result from clonal-based cell-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37541246"
                        open
                      >
                        <summary class="article-expander__title">The winner takes it all: Competition drives clonal selection in gliomagenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37541246/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The exact mechanisms that govern clonal dynamics and selection during early tumorigenesis remain largely elusive. Ceresa et al. provide experimental and mathematical evidence that MYC-dependent competition between individual clones is one driving force of brain tumor evolution, adding a winner/loser aspect to the picture that complements existing concepts.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37541243"
                        open
                      >
                        <summary class="article-expander__title">Early clonal extinction in glioblastoma progression revealed by genetic barcoding</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37541243/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastoma progression in its early stages remains poorly understood. Here, we transfer PDGFB and genetic barcodes in mouse brain to initiate gliomagenesis and enable direct tracing of glioblastoma evolution from its earliest possible stage. Unexpectedly, we observe a high incidence of clonal extinction events and progressive divergence in clonal sizes, even after the acquisition of malignant phenotype. Computational modeling suggests these dynamics result from clonal-based cell-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37541246"
                        open
                      >
                        <summary class="article-expander__title">The winner takes it all: Competition drives clonal selection in gliomagenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37541246/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The exact mechanisms that govern clonal dynamics and selection during early tumorigenesis remain largely elusive. Ceresa et al. provide experimental and mathematical evidence that MYC-dependent competition between individual clones is one driving force of brain tumor evolution, adding a winner/loser aspect to the picture that complements existing concepts.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37541243"
                        open
                      >
                        <summary class="article-expander__title">Early clonal extinction in glioblastoma progression revealed by genetic barcoding</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37541243/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastoma progression in its early stages remains poorly understood. Here, we transfer PDGFB and genetic barcodes in mouse brain to initiate gliomagenesis and enable direct tracing of glioblastoma evolution from its earliest possible stage. Unexpectedly, we observe a high incidence of clonal extinction events and progressive divergence in clonal sizes, even after the acquisition of malignant phenotype. Computational modeling suggests these dynamics result from clonal-based cell-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-08-02 datetime="2023-08-02T10:00:00.000Z">2023-08-02</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-08-02 datetime="2023-08-02T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37532684"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Acute Ischemic Stroke Outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37532684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>OBJECTIVE: The effect of Clonal hematopoiesis of indeterminate potential (CHIP) on the manifestation and clinical outcomes of acute ischemic stroke (AIS) has not been fully elucidated.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37532684"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Acute Ischemic Stroke Outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37532684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>OBJECTIVE: The effect of Clonal hematopoiesis of indeterminate potential (CHIP) on the manifestation and clinical outcomes of acute ischemic stroke (AIS) has not been fully elucidated.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37532684"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Acute Ischemic Stroke Outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37532684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>OBJECTIVE: The effect of Clonal hematopoiesis of indeterminate potential (CHIP) on the manifestation and clinical outcomes of acute ischemic stroke (AIS) has not been fully elucidated.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37532684"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Acute Ischemic Stroke Outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37532684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>OBJECTIVE: The effect of Clonal hematopoiesis of indeterminate potential (CHIP) on the manifestation and clinical outcomes of acute ischemic stroke (AIS) has not been fully elucidated.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37532684"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Acute Ischemic Stroke Outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37532684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>OBJECTIVE: The effect of Clonal hematopoiesis of indeterminate potential (CHIP) on the manifestation and clinical outcomes of acute ischemic stroke (AIS) has not been fully elucidated.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37532684"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Acute Ischemic Stroke Outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37532684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>OBJECTIVE: The effect of Clonal hematopoiesis of indeterminate potential (CHIP) on the manifestation and clinical outcomes of acute ischemic stroke (AIS) has not been fully elucidated.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37532684"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Acute Ischemic Stroke Outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37532684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>OBJECTIVE: The effect of Clonal hematopoiesis of indeterminate potential (CHIP) on the manifestation and clinical outcomes of acute ischemic stroke (AIS) has not been fully elucidated.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37532684"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Acute Ischemic Stroke Outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37532684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>OBJECTIVE: The effect of Clonal hematopoiesis of indeterminate potential (CHIP) on the manifestation and clinical outcomes of acute ischemic stroke (AIS) has not been fully elucidated.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-08-01 datetime="2023-08-01T10:00:00.000Z">2023-08-01</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-08-01 datetime="2023-08-01T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37526082"
                        open
                      >
                        <summary class="article-expander__title">Foxm1 haploinsufficiency drives clonal hematopoiesis and promotes a stress-related transition to hematologic malignancy in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37526082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis plays a critical role in the initiation and development of hematologic malignancies. In patients with del(5q) myelodysplastic syndrome (MDS), the transcription factor FOXM1 is frequently downregulated in CD34+ cells. In this study, we demonstrated that Foxm1 haploinsufficiency disturbed normal hematopoiesis and conferred a competitive repopulation advantage for a short period. However, it impaired the long-term self-renewal capacity of hematopoietic stem cells,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37526082"
                        open
                      >
                        <summary class="article-expander__title">Foxm1 haploinsufficiency drives clonal hematopoiesis and promotes a stress-related transition to hematologic malignancy in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37526082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis plays a critical role in the initiation and development of hematologic malignancies. In patients with del(5q) myelodysplastic syndrome (MDS), the transcription factor FOXM1 is frequently downregulated in CD34+ cells. In this study, we demonstrated that Foxm1 haploinsufficiency disturbed normal hematopoiesis and conferred a competitive repopulation advantage for a short period. However, it impaired the long-term self-renewal capacity of hematopoietic stem cells,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37526082"
                        open
                      >
                        <summary class="article-expander__title">Foxm1 haploinsufficiency drives clonal hematopoiesis and promotes a stress-related transition to hematologic malignancy in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37526082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis plays a critical role in the initiation and development of hematologic malignancies. In patients with del(5q) myelodysplastic syndrome (MDS), the transcription factor FOXM1 is frequently downregulated in CD34+ cells. In this study, we demonstrated that Foxm1 haploinsufficiency disturbed normal hematopoiesis and conferred a competitive repopulation advantage for a short period. However, it impaired the long-term self-renewal capacity of hematopoietic stem cells,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37526082"
                        open
                      >
                        <summary class="article-expander__title">Foxm1 haploinsufficiency drives clonal hematopoiesis and promotes a stress-related transition to hematologic malignancy in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37526082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis plays a critical role in the initiation and development of hematologic malignancies. In patients with del(5q) myelodysplastic syndrome (MDS), the transcription factor FOXM1 is frequently downregulated in CD34+ cells. In this study, we demonstrated that Foxm1 haploinsufficiency disturbed normal hematopoiesis and conferred a competitive repopulation advantage for a short period. However, it impaired the long-term self-renewal capacity of hematopoietic stem cells,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37526082"
                        open
                      >
                        <summary class="article-expander__title">Foxm1 haploinsufficiency drives clonal hematopoiesis and promotes a stress-related transition to hematologic malignancy in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37526082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis plays a critical role in the initiation and development of hematologic malignancies. In patients with del(5q) myelodysplastic syndrome (MDS), the transcription factor FOXM1 is frequently downregulated in CD34+ cells. In this study, we demonstrated that Foxm1 haploinsufficiency disturbed normal hematopoiesis and conferred a competitive repopulation advantage for a short period. However, it impaired the long-term self-renewal capacity of hematopoietic stem cells,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37526082"
                        open
                      >
                        <summary class="article-expander__title">Foxm1 haploinsufficiency drives clonal hematopoiesis and promotes a stress-related transition to hematologic malignancy in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37526082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis plays a critical role in the initiation and development of hematologic malignancies. In patients with del(5q) myelodysplastic syndrome (MDS), the transcription factor FOXM1 is frequently downregulated in CD34+ cells. In this study, we demonstrated that Foxm1 haploinsufficiency disturbed normal hematopoiesis and conferred a competitive repopulation advantage for a short period. However, it impaired the long-term self-renewal capacity of hematopoietic stem cells,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37526082"
                        open
                      >
                        <summary class="article-expander__title">Foxm1 haploinsufficiency drives clonal hematopoiesis and promotes a stress-related transition to hematologic malignancy in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37526082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis plays a critical role in the initiation and development of hematologic malignancies. In patients with del(5q) myelodysplastic syndrome (MDS), the transcription factor FOXM1 is frequently downregulated in CD34+ cells. In this study, we demonstrated that Foxm1 haploinsufficiency disturbed normal hematopoiesis and conferred a competitive repopulation advantage for a short period. However, it impaired the long-term self-renewal capacity of hematopoietic stem cells,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37526082"
                        open
                      >
                        <summary class="article-expander__title">Foxm1 haploinsufficiency drives clonal hematopoiesis and promotes a stress-related transition to hematologic malignancy in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37526082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis plays a critical role in the initiation and development of hematologic malignancies. In patients with del(5q) myelodysplastic syndrome (MDS), the transcription factor FOXM1 is frequently downregulated in CD34+ cells. In this study, we demonstrated that Foxm1 haploinsufficiency disturbed normal hematopoiesis and conferred a competitive repopulation advantage for a short period. However, it impaired the long-term self-renewal capacity of hematopoietic stem cells,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37526082"
                        open
                      >
                        <summary class="article-expander__title">Foxm1 haploinsufficiency drives clonal hematopoiesis and promotes a stress-related transition to hematologic malignancy in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37526082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis plays a critical role in the initiation and development of hematologic malignancies. In patients with del(5q) myelodysplastic syndrome (MDS), the transcription factor FOXM1 is frequently downregulated in CD34+ cells. In this study, we demonstrated that Foxm1 haploinsufficiency disturbed normal hematopoiesis and conferred a competitive repopulation advantage for a short period. However, it impaired the long-term self-renewal capacity of hematopoietic stem cells,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37526082"
                        open
                      >
                        <summary class="article-expander__title">Foxm1 haploinsufficiency drives clonal hematopoiesis and promotes a stress-related transition to hematologic malignancy in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37526082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis plays a critical role in the initiation and development of hematologic malignancies. In patients with del(5q) myelodysplastic syndrome (MDS), the transcription factor FOXM1 is frequently downregulated in CD34+ cells. In this study, we demonstrated that Foxm1 haploinsufficiency disturbed normal hematopoiesis and conferred a competitive repopulation advantage for a short period. However, it impaired the long-term self-renewal capacity of hematopoietic stem cells,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37526082"
                        open
                      >
                        <summary class="article-expander__title">Foxm1 haploinsufficiency drives clonal hematopoiesis and promotes a stress-related transition to hematologic malignancy in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37526082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis plays a critical role in the initiation and development of hematologic malignancies. In patients with del(5q) myelodysplastic syndrome (MDS), the transcription factor FOXM1 is frequently downregulated in CD34+ cells. In this study, we demonstrated that Foxm1 haploinsufficiency disturbed normal hematopoiesis and conferred a competitive repopulation advantage for a short period. However, it impaired the long-term self-renewal capacity of hematopoietic stem cells,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37526082"
                        open
                      >
                        <summary class="article-expander__title">Foxm1 haploinsufficiency drives clonal hematopoiesis and promotes a stress-related transition to hematologic malignancy in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37526082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis plays a critical role in the initiation and development of hematologic malignancies. In patients with del(5q) myelodysplastic syndrome (MDS), the transcription factor FOXM1 is frequently downregulated in CD34+ cells. In this study, we demonstrated that Foxm1 haploinsufficiency disturbed normal hematopoiesis and conferred a competitive repopulation advantage for a short period. However, it impaired the long-term self-renewal capacity of hematopoietic stem cells,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37526082"
                        open
                      >
                        <summary class="article-expander__title">Foxm1 haploinsufficiency drives clonal hematopoiesis and promotes a stress-related transition to hematologic malignancy in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37526082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis plays a critical role in the initiation and development of hematologic malignancies. In patients with del(5q) myelodysplastic syndrome (MDS), the transcription factor FOXM1 is frequently downregulated in CD34+ cells. In this study, we demonstrated that Foxm1 haploinsufficiency disturbed normal hematopoiesis and conferred a competitive repopulation advantage for a short period. However, it impaired the long-term self-renewal capacity of hematopoietic stem cells,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37526082"
                        open
                      >
                        <summary class="article-expander__title">Foxm1 haploinsufficiency drives clonal hematopoiesis and promotes a stress-related transition to hematologic malignancy in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37526082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis plays a critical role in the initiation and development of hematologic malignancies. In patients with del(5q) myelodysplastic syndrome (MDS), the transcription factor FOXM1 is frequently downregulated in CD34+ cells. In this study, we demonstrated that Foxm1 haploinsufficiency disturbed normal hematopoiesis and conferred a competitive repopulation advantage for a short period. However, it impaired the long-term self-renewal capacity of hematopoietic stem cells,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37526082"
                        open
                      >
                        <summary class="article-expander__title">Foxm1 haploinsufficiency drives clonal hematopoiesis and promotes a stress-related transition to hematologic malignancy in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37526082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis plays a critical role in the initiation and development of hematologic malignancies. In patients with del(5q) myelodysplastic syndrome (MDS), the transcription factor FOXM1 is frequently downregulated in CD34+ cells. In this study, we demonstrated that Foxm1 haploinsufficiency disturbed normal hematopoiesis and conferred a competitive repopulation advantage for a short period. However, it impaired the long-term self-renewal capacity of hematopoietic stem cells,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37526082"
                        open
                      >
                        <summary class="article-expander__title">Foxm1 haploinsufficiency drives clonal hematopoiesis and promotes a stress-related transition to hematologic malignancy in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37526082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis plays a critical role in the initiation and development of hematologic malignancies. In patients with del(5q) myelodysplastic syndrome (MDS), the transcription factor FOXM1 is frequently downregulated in CD34+ cells. In this study, we demonstrated that Foxm1 haploinsufficiency disturbed normal hematopoiesis and conferred a competitive repopulation advantage for a short period. However, it impaired the long-term self-renewal capacity of hematopoietic stem cells,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37526082"
                        open
                      >
                        <summary class="article-expander__title">Foxm1 haploinsufficiency drives clonal hematopoiesis and promotes a stress-related transition to hematologic malignancy in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37526082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis plays a critical role in the initiation and development of hematologic malignancies. In patients with del(5q) myelodysplastic syndrome (MDS), the transcription factor FOXM1 is frequently downregulated in CD34+ cells. In this study, we demonstrated that Foxm1 haploinsufficiency disturbed normal hematopoiesis and conferred a competitive repopulation advantage for a short period. However, it impaired the long-term self-renewal capacity of hematopoietic stem cells,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37526082"
                        open
                      >
                        <summary class="article-expander__title">Foxm1 haploinsufficiency drives clonal hematopoiesis and promotes a stress-related transition to hematologic malignancy in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37526082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis plays a critical role in the initiation and development of hematologic malignancies. In patients with del(5q) myelodysplastic syndrome (MDS), the transcription factor FOXM1 is frequently downregulated in CD34+ cells. In this study, we demonstrated that Foxm1 haploinsufficiency disturbed normal hematopoiesis and conferred a competitive repopulation advantage for a short period. However, it impaired the long-term self-renewal capacity of hematopoietic stem cells,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37526082"
                        open
                      >
                        <summary class="article-expander__title">Foxm1 haploinsufficiency drives clonal hematopoiesis and promotes a stress-related transition to hematologic malignancy in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37526082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis plays a critical role in the initiation and development of hematologic malignancies. In patients with del(5q) myelodysplastic syndrome (MDS), the transcription factor FOXM1 is frequently downregulated in CD34+ cells. In this study, we demonstrated that Foxm1 haploinsufficiency disturbed normal hematopoiesis and conferred a competitive repopulation advantage for a short period. However, it impaired the long-term self-renewal capacity of hematopoietic stem cells,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-07-31 datetime="2023-07-31T10:00:00.000Z">2023-07-31</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-07-31 datetime="2023-07-31T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37520699"
                        open
                      >
                        <summary class="article-expander__title">Progressive disruption of hematopoietic architecture from clonal hematopoiesis to MDS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37520699/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) describes the age-related acquisition of somatic mutations in hematopoietic stem/progenitor cells (HSPC) leading to clonal blood cell expansion. Although CHIP mutations drive myeloid malignancies like myelodysplastic syndromes (MDS) it is unknown if clonal expansion is attributable to changes in cell type kinetics, or involves reorganization of the hematopoietic hierarchy. Using computational modeling we analyzed differentiation and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37520699"
                        open
                      >
                        <summary class="article-expander__title">Progressive disruption of hematopoietic architecture from clonal hematopoiesis to MDS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37520699/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) describes the age-related acquisition of somatic mutations in hematopoietic stem/progenitor cells (HSPC) leading to clonal blood cell expansion. Although CHIP mutations drive myeloid malignancies like myelodysplastic syndromes (MDS) it is unknown if clonal expansion is attributable to changes in cell type kinetics, or involves reorganization of the hematopoietic hierarchy. Using computational modeling we analyzed differentiation and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37520699"
                        open
                      >
                        <summary class="article-expander__title">Progressive disruption of hematopoietic architecture from clonal hematopoiesis to MDS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37520699/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) describes the age-related acquisition of somatic mutations in hematopoietic stem/progenitor cells (HSPC) leading to clonal blood cell expansion. Although CHIP mutations drive myeloid malignancies like myelodysplastic syndromes (MDS) it is unknown if clonal expansion is attributable to changes in cell type kinetics, or involves reorganization of the hematopoietic hierarchy. Using computational modeling we analyzed differentiation and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37520699"
                        open
                      >
                        <summary class="article-expander__title">Progressive disruption of hematopoietic architecture from clonal hematopoiesis to MDS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37520699/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) describes the age-related acquisition of somatic mutations in hematopoietic stem/progenitor cells (HSPC) leading to clonal blood cell expansion. Although CHIP mutations drive myeloid malignancies like myelodysplastic syndromes (MDS) it is unknown if clonal expansion is attributable to changes in cell type kinetics, or involves reorganization of the hematopoietic hierarchy. Using computational modeling we analyzed differentiation and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37520699"
                        open
                      >
                        <summary class="article-expander__title">Progressive disruption of hematopoietic architecture from clonal hematopoiesis to MDS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37520699/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) describes the age-related acquisition of somatic mutations in hematopoietic stem/progenitor cells (HSPC) leading to clonal blood cell expansion. Although CHIP mutations drive myeloid malignancies like myelodysplastic syndromes (MDS) it is unknown if clonal expansion is attributable to changes in cell type kinetics, or involves reorganization of the hematopoietic hierarchy. Using computational modeling we analyzed differentiation and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37520699"
                        open
                      >
                        <summary class="article-expander__title">Progressive disruption of hematopoietic architecture from clonal hematopoiesis to MDS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37520699/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) describes the age-related acquisition of somatic mutations in hematopoietic stem/progenitor cells (HSPC) leading to clonal blood cell expansion. Although CHIP mutations drive myeloid malignancies like myelodysplastic syndromes (MDS) it is unknown if clonal expansion is attributable to changes in cell type kinetics, or involves reorganization of the hematopoietic hierarchy. Using computational modeling we analyzed differentiation and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37520699"
                        open
                      >
                        <summary class="article-expander__title">Progressive disruption of hematopoietic architecture from clonal hematopoiesis to MDS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37520699/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) describes the age-related acquisition of somatic mutations in hematopoietic stem/progenitor cells (HSPC) leading to clonal blood cell expansion. Although CHIP mutations drive myeloid malignancies like myelodysplastic syndromes (MDS) it is unknown if clonal expansion is attributable to changes in cell type kinetics, or involves reorganization of the hematopoietic hierarchy. Using computational modeling we analyzed differentiation and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37520699"
                        open
                      >
                        <summary class="article-expander__title">Progressive disruption of hematopoietic architecture from clonal hematopoiesis to MDS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37520699/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) describes the age-related acquisition of somatic mutations in hematopoietic stem/progenitor cells (HSPC) leading to clonal blood cell expansion. Although CHIP mutations drive myeloid malignancies like myelodysplastic syndromes (MDS) it is unknown if clonal expansion is attributable to changes in cell type kinetics, or involves reorganization of the hematopoietic hierarchy. Using computational modeling we analyzed differentiation and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37520699"
                        open
                      >
                        <summary class="article-expander__title">Progressive disruption of hematopoietic architecture from clonal hematopoiesis to MDS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37520699/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) describes the age-related acquisition of somatic mutations in hematopoietic stem/progenitor cells (HSPC) leading to clonal blood cell expansion. Although CHIP mutations drive myeloid malignancies like myelodysplastic syndromes (MDS) it is unknown if clonal expansion is attributable to changes in cell type kinetics, or involves reorganization of the hematopoietic hierarchy. Using computational modeling we analyzed differentiation and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37520699"
                        open
                      >
                        <summary class="article-expander__title">Progressive disruption of hematopoietic architecture from clonal hematopoiesis to MDS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37520699/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) describes the age-related acquisition of somatic mutations in hematopoietic stem/progenitor cells (HSPC) leading to clonal blood cell expansion. Although CHIP mutations drive myeloid malignancies like myelodysplastic syndromes (MDS) it is unknown if clonal expansion is attributable to changes in cell type kinetics, or involves reorganization of the hematopoietic hierarchy. Using computational modeling we analyzed differentiation and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37520699"
                        open
                      >
                        <summary class="article-expander__title">Progressive disruption of hematopoietic architecture from clonal hematopoiesis to MDS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37520699/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) describes the age-related acquisition of somatic mutations in hematopoietic stem/progenitor cells (HSPC) leading to clonal blood cell expansion. Although CHIP mutations drive myeloid malignancies like myelodysplastic syndromes (MDS) it is unknown if clonal expansion is attributable to changes in cell type kinetics, or involves reorganization of the hematopoietic hierarchy. Using computational modeling we analyzed differentiation and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-07-30 datetime="2023-07-30T10:00:00.000Z">2023-07-30</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-07-30 datetime="2023-07-30T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37517612"
                        open
                      >
                        <summary class="article-expander__title">A pilot study of donor-engrafted clonal hematopoiesis evolution and clinical outcomes in allogeneic hematopoietic cell transplant recipients using a national registry</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37517612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CH in older HLA-matched sibling donors is relatively common and successfully engrafts and persists in recipients. In a homogenous population (myeloid malignancies, older donors and recipients, reduced-intensity conditioning, non-cyclophosphamide-containing GVHD prophylaxis), we did not detect a difference in cGVHD risk or other secondary outcomes by donor CH status. Subgroup analyses suggest potential of differential effects by clinical characteristics and CH mutations. Larger...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37517612"
                        open
                      >
                        <summary class="article-expander__title">A pilot study of donor-engrafted clonal hematopoiesis evolution and clinical outcomes in allogeneic hematopoietic cell transplant recipients using a national registry</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37517612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CH in older HLA-matched sibling donors is relatively common and successfully engrafts and persists in recipients. In a homogenous population (myeloid malignancies, older donors and recipients, reduced-intensity conditioning, non-cyclophosphamide-containing GVHD prophylaxis), we did not detect a difference in cGVHD risk or other secondary outcomes by donor CH status. Subgroup analyses suggest potential of differential effects by clinical characteristics and CH mutations. Larger...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37517612"
                        open
                      >
                        <summary class="article-expander__title">A pilot study of donor-engrafted clonal hematopoiesis evolution and clinical outcomes in allogeneic hematopoietic cell transplant recipients using a national registry</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37517612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CH in older HLA-matched sibling donors is relatively common and successfully engrafts and persists in recipients. In a homogenous population (myeloid malignancies, older donors and recipients, reduced-intensity conditioning, non-cyclophosphamide-containing GVHD prophylaxis), we did not detect a difference in cGVHD risk or other secondary outcomes by donor CH status. Subgroup analyses suggest potential of differential effects by clinical characteristics and CH mutations. Larger...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37517612"
                        open
                      >
                        <summary class="article-expander__title">A pilot study of donor-engrafted clonal hematopoiesis evolution and clinical outcomes in allogeneic hematopoietic cell transplant recipients using a national registry</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37517612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CH in older HLA-matched sibling donors is relatively common and successfully engrafts and persists in recipients. In a homogenous population (myeloid malignancies, older donors and recipients, reduced-intensity conditioning, non-cyclophosphamide-containing GVHD prophylaxis), we did not detect a difference in cGVHD risk or other secondary outcomes by donor CH status. Subgroup analyses suggest potential of differential effects by clinical characteristics and CH mutations. Larger...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37517612"
                        open
                      >
                        <summary class="article-expander__title">A pilot study of donor-engrafted clonal hematopoiesis evolution and clinical outcomes in allogeneic hematopoietic cell transplant recipients using a national registry</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37517612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CH in older HLA-matched sibling donors is relatively common and successfully engrafts and persists in recipients. In a homogenous population (myeloid malignancies, older donors and recipients, reduced-intensity conditioning, non-cyclophosphamide-containing GVHD prophylaxis), we did not detect a difference in cGVHD risk or other secondary outcomes by donor CH status. Subgroup analyses suggest potential of differential effects by clinical characteristics and CH mutations. Larger...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37517612"
                        open
                      >
                        <summary class="article-expander__title">A pilot study of donor-engrafted clonal hematopoiesis evolution and clinical outcomes in allogeneic hematopoietic cell transplant recipients using a national registry</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37517612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CH in older HLA-matched sibling donors is relatively common and successfully engrafts and persists in recipients. In a homogenous population (myeloid malignancies, older donors and recipients, reduced-intensity conditioning, non-cyclophosphamide-containing GVHD prophylaxis), we did not detect a difference in cGVHD risk or other secondary outcomes by donor CH status. Subgroup analyses suggest potential of differential effects by clinical characteristics and CH mutations. Larger...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37517612"
                        open
                      >
                        <summary class="article-expander__title">A pilot study of donor-engrafted clonal hematopoiesis evolution and clinical outcomes in allogeneic hematopoietic cell transplant recipients using a national registry</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37517612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CH in older HLA-matched sibling donors is relatively common and successfully engrafts and persists in recipients. In a homogenous population (myeloid malignancies, older donors and recipients, reduced-intensity conditioning, non-cyclophosphamide-containing GVHD prophylaxis), we did not detect a difference in cGVHD risk or other secondary outcomes by donor CH status. Subgroup analyses suggest potential of differential effects by clinical characteristics and CH mutations. Larger...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37517612"
                        open
                      >
                        <summary class="article-expander__title">A pilot study of donor-engrafted clonal hematopoiesis evolution and clinical outcomes in allogeneic hematopoietic cell transplant recipients using a national registry</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37517612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CH in older HLA-matched sibling donors is relatively common and successfully engrafts and persists in recipients. In a homogenous population (myeloid malignancies, older donors and recipients, reduced-intensity conditioning, non-cyclophosphamide-containing GVHD prophylaxis), we did not detect a difference in cGVHD risk or other secondary outcomes by donor CH status. Subgroup analyses suggest potential of differential effects by clinical characteristics and CH mutations. Larger...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37517612"
                        open
                      >
                        <summary class="article-expander__title">A pilot study of donor-engrafted clonal hematopoiesis evolution and clinical outcomes in allogeneic hematopoietic cell transplant recipients using a national registry</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37517612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CH in older HLA-matched sibling donors is relatively common and successfully engrafts and persists in recipients. In a homogenous population (myeloid malignancies, older donors and recipients, reduced-intensity conditioning, non-cyclophosphamide-containing GVHD prophylaxis), we did not detect a difference in cGVHD risk or other secondary outcomes by donor CH status. Subgroup analyses suggest potential of differential effects by clinical characteristics and CH mutations. Larger...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37517612"
                        open
                      >
                        <summary class="article-expander__title">A pilot study of donor-engrafted clonal hematopoiesis evolution and clinical outcomes in allogeneic hematopoietic cell transplant recipients using a national registry</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37517612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CH in older HLA-matched sibling donors is relatively common and successfully engrafts and persists in recipients. In a homogenous population (myeloid malignancies, older donors and recipients, reduced-intensity conditioning, non-cyclophosphamide-containing GVHD prophylaxis), we did not detect a difference in cGVHD risk or other secondary outcomes by donor CH status. Subgroup analyses suggest potential of differential effects by clinical characteristics and CH mutations. Larger...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37517612"
                        open
                      >
                        <summary class="article-expander__title">A pilot study of donor-engrafted clonal hematopoiesis evolution and clinical outcomes in allogeneic hematopoietic cell transplant recipients using a national registry</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37517612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CH in older HLA-matched sibling donors is relatively common and successfully engrafts and persists in recipients. In a homogenous population (myeloid malignancies, older donors and recipients, reduced-intensity conditioning, non-cyclophosphamide-containing GVHD prophylaxis), we did not detect a difference in cGVHD risk or other secondary outcomes by donor CH status. Subgroup analyses suggest potential of differential effects by clinical characteristics and CH mutations. Larger...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-07-29 datetime="2023-07-29T10:00:00.000Z">2023-07-29</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-07-29 datetime="2023-07-29T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37510235"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Analysis in the Omics Era: Technologies and Applications in Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37510235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer molecular profiling obtained with conventional bulk sequencing describes average alterations obtained from the entire cellular population analyzed. In the era of precision medicine, this approach is unable to track tumor heterogeneity and cannot be exploited to unravel the biological processes behind clonal evolution. In the last few years, functional single-cell omics has improved our understanding of cancer heterogeneity. This approach requires isolation and identification of single...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37508780"
                        open
                      >
                        <summary class="article-expander__title">Extreme Gradient Boosting Tuned with Metaheuristic Algorithms for Predicting Myeloid NGS Onco-Somatic Variant Pathogenicity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37508780/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The advent of next-generation sequencing (NGS) technologies has revolutionized the field of bioinformatics and genomics, particularly in the area of onco-somatic genetics. NGS has provided a wealth of information about the genetic changes that underlie cancer and has considerably improved our ability to diagnose and treat cancer. However, the large amount of data generated by NGS makes it difficult to interpret the variants. To address this, machine learning algorithms such as Extreme Gradient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37508513"
                        open
                      >
                        <summary class="article-expander__title">Mathematical Modeling of Clonal Interference by Density-Dependent Selection in Heterogeneous Cancer Cell Lines</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37508513/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Many cancer cell lines are aneuploid and heterogeneous, with multiple karyotypes co-existing within the same cell line. Karyotype heterogeneity has been shown to manifest phenotypically, thus affecting how cells respond to drugs or to minor differences in culture media. Knowing how to interpret karyotype heterogeneity phenotypically would give insights into cellular phenotypes before they unfold temporally. Here, we re-analyzed single cell RNA (scRNA) and scDNA sequencing data from eight stomach...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37510235"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Analysis in the Omics Era: Technologies and Applications in Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37510235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer molecular profiling obtained with conventional bulk sequencing describes average alterations obtained from the entire cellular population analyzed. In the era of precision medicine, this approach is unable to track tumor heterogeneity and cannot be exploited to unravel the biological processes behind clonal evolution. In the last few years, functional single-cell omics has improved our understanding of cancer heterogeneity. This approach requires isolation and identification of single...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37508780"
                        open
                      >
                        <summary class="article-expander__title">Extreme Gradient Boosting Tuned with Metaheuristic Algorithms for Predicting Myeloid NGS Onco-Somatic Variant Pathogenicity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37508780/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The advent of next-generation sequencing (NGS) technologies has revolutionized the field of bioinformatics and genomics, particularly in the area of onco-somatic genetics. NGS has provided a wealth of information about the genetic changes that underlie cancer and has considerably improved our ability to diagnose and treat cancer. However, the large amount of data generated by NGS makes it difficult to interpret the variants. To address this, machine learning algorithms such as Extreme Gradient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37508513"
                        open
                      >
                        <summary class="article-expander__title">Mathematical Modeling of Clonal Interference by Density-Dependent Selection in Heterogeneous Cancer Cell Lines</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37508513/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Many cancer cell lines are aneuploid and heterogeneous, with multiple karyotypes co-existing within the same cell line. Karyotype heterogeneity has been shown to manifest phenotypically, thus affecting how cells respond to drugs or to minor differences in culture media. Knowing how to interpret karyotype heterogeneity phenotypically would give insights into cellular phenotypes before they unfold temporally. Here, we re-analyzed single cell RNA (scRNA) and scDNA sequencing data from eight stomach...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37510235"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Analysis in the Omics Era: Technologies and Applications in Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37510235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer molecular profiling obtained with conventional bulk sequencing describes average alterations obtained from the entire cellular population analyzed. In the era of precision medicine, this approach is unable to track tumor heterogeneity and cannot be exploited to unravel the biological processes behind clonal evolution. In the last few years, functional single-cell omics has improved our understanding of cancer heterogeneity. This approach requires isolation and identification of single...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37508780"
                        open
                      >
                        <summary class="article-expander__title">Extreme Gradient Boosting Tuned with Metaheuristic Algorithms for Predicting Myeloid NGS Onco-Somatic Variant Pathogenicity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37508780/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The advent of next-generation sequencing (NGS) technologies has revolutionized the field of bioinformatics and genomics, particularly in the area of onco-somatic genetics. NGS has provided a wealth of information about the genetic changes that underlie cancer and has considerably improved our ability to diagnose and treat cancer. However, the large amount of data generated by NGS makes it difficult to interpret the variants. To address this, machine learning algorithms such as Extreme Gradient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37508513"
                        open
                      >
                        <summary class="article-expander__title">Mathematical Modeling of Clonal Interference by Density-Dependent Selection in Heterogeneous Cancer Cell Lines</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37508513/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Many cancer cell lines are aneuploid and heterogeneous, with multiple karyotypes co-existing within the same cell line. Karyotype heterogeneity has been shown to manifest phenotypically, thus affecting how cells respond to drugs or to minor differences in culture media. Knowing how to interpret karyotype heterogeneity phenotypically would give insights into cellular phenotypes before they unfold temporally. Here, we re-analyzed single cell RNA (scRNA) and scDNA sequencing data from eight stomach...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37510235"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Analysis in the Omics Era: Technologies and Applications in Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37510235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer molecular profiling obtained with conventional bulk sequencing describes average alterations obtained from the entire cellular population analyzed. In the era of precision medicine, this approach is unable to track tumor heterogeneity and cannot be exploited to unravel the biological processes behind clonal evolution. In the last few years, functional single-cell omics has improved our understanding of cancer heterogeneity. This approach requires isolation and identification of single...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37508780"
                        open
                      >
                        <summary class="article-expander__title">Extreme Gradient Boosting Tuned with Metaheuristic Algorithms for Predicting Myeloid NGS Onco-Somatic Variant Pathogenicity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37508780/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The advent of next-generation sequencing (NGS) technologies has revolutionized the field of bioinformatics and genomics, particularly in the area of onco-somatic genetics. NGS has provided a wealth of information about the genetic changes that underlie cancer and has considerably improved our ability to diagnose and treat cancer. However, the large amount of data generated by NGS makes it difficult to interpret the variants. To address this, machine learning algorithms such as Extreme Gradient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37508513"
                        open
                      >
                        <summary class="article-expander__title">Mathematical Modeling of Clonal Interference by Density-Dependent Selection in Heterogeneous Cancer Cell Lines</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37508513/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Many cancer cell lines are aneuploid and heterogeneous, with multiple karyotypes co-existing within the same cell line. Karyotype heterogeneity has been shown to manifest phenotypically, thus affecting how cells respond to drugs or to minor differences in culture media. Knowing how to interpret karyotype heterogeneity phenotypically would give insights into cellular phenotypes before they unfold temporally. Here, we re-analyzed single cell RNA (scRNA) and scDNA sequencing data from eight stomach...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37510235"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Analysis in the Omics Era: Technologies and Applications in Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37510235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer molecular profiling obtained with conventional bulk sequencing describes average alterations obtained from the entire cellular population analyzed. In the era of precision medicine, this approach is unable to track tumor heterogeneity and cannot be exploited to unravel the biological processes behind clonal evolution. In the last few years, functional single-cell omics has improved our understanding of cancer heterogeneity. This approach requires isolation and identification of single...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37508780"
                        open
                      >
                        <summary class="article-expander__title">Extreme Gradient Boosting Tuned with Metaheuristic Algorithms for Predicting Myeloid NGS Onco-Somatic Variant Pathogenicity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37508780/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The advent of next-generation sequencing (NGS) technologies has revolutionized the field of bioinformatics and genomics, particularly in the area of onco-somatic genetics. NGS has provided a wealth of information about the genetic changes that underlie cancer and has considerably improved our ability to diagnose and treat cancer. However, the large amount of data generated by NGS makes it difficult to interpret the variants. To address this, machine learning algorithms such as Extreme Gradient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37508513"
                        open
                      >
                        <summary class="article-expander__title">Mathematical Modeling of Clonal Interference by Density-Dependent Selection in Heterogeneous Cancer Cell Lines</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37508513/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Many cancer cell lines are aneuploid and heterogeneous, with multiple karyotypes co-existing within the same cell line. Karyotype heterogeneity has been shown to manifest phenotypically, thus affecting how cells respond to drugs or to minor differences in culture media. Knowing how to interpret karyotype heterogeneity phenotypically would give insights into cellular phenotypes before they unfold temporally. Here, we re-analyzed single cell RNA (scRNA) and scDNA sequencing data from eight stomach...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37510235"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Analysis in the Omics Era: Technologies and Applications in Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37510235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer molecular profiling obtained with conventional bulk sequencing describes average alterations obtained from the entire cellular population analyzed. In the era of precision medicine, this approach is unable to track tumor heterogeneity and cannot be exploited to unravel the biological processes behind clonal evolution. In the last few years, functional single-cell omics has improved our understanding of cancer heterogeneity. This approach requires isolation and identification of single...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37508780"
                        open
                      >
                        <summary class="article-expander__title">Extreme Gradient Boosting Tuned with Metaheuristic Algorithms for Predicting Myeloid NGS Onco-Somatic Variant Pathogenicity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37508780/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The advent of next-generation sequencing (NGS) technologies has revolutionized the field of bioinformatics and genomics, particularly in the area of onco-somatic genetics. NGS has provided a wealth of information about the genetic changes that underlie cancer and has considerably improved our ability to diagnose and treat cancer. However, the large amount of data generated by NGS makes it difficult to interpret the variants. To address this, machine learning algorithms such as Extreme Gradient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37508513"
                        open
                      >
                        <summary class="article-expander__title">Mathematical Modeling of Clonal Interference by Density-Dependent Selection in Heterogeneous Cancer Cell Lines</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37508513/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Many cancer cell lines are aneuploid and heterogeneous, with multiple karyotypes co-existing within the same cell line. Karyotype heterogeneity has been shown to manifest phenotypically, thus affecting how cells respond to drugs or to minor differences in culture media. Knowing how to interpret karyotype heterogeneity phenotypically would give insights into cellular phenotypes before they unfold temporally. Here, we re-analyzed single cell RNA (scRNA) and scDNA sequencing data from eight stomach...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37510235"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Analysis in the Omics Era: Technologies and Applications in Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37510235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer molecular profiling obtained with conventional bulk sequencing describes average alterations obtained from the entire cellular population analyzed. In the era of precision medicine, this approach is unable to track tumor heterogeneity and cannot be exploited to unravel the biological processes behind clonal evolution. In the last few years, functional single-cell omics has improved our understanding of cancer heterogeneity. This approach requires isolation and identification of single...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37508780"
                        open
                      >
                        <summary class="article-expander__title">Extreme Gradient Boosting Tuned with Metaheuristic Algorithms for Predicting Myeloid NGS Onco-Somatic Variant Pathogenicity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37508780/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The advent of next-generation sequencing (NGS) technologies has revolutionized the field of bioinformatics and genomics, particularly in the area of onco-somatic genetics. NGS has provided a wealth of information about the genetic changes that underlie cancer and has considerably improved our ability to diagnose and treat cancer. However, the large amount of data generated by NGS makes it difficult to interpret the variants. To address this, machine learning algorithms such as Extreme Gradient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37508513"
                        open
                      >
                        <summary class="article-expander__title">Mathematical Modeling of Clonal Interference by Density-Dependent Selection in Heterogeneous Cancer Cell Lines</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37508513/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Many cancer cell lines are aneuploid and heterogeneous, with multiple karyotypes co-existing within the same cell line. Karyotype heterogeneity has been shown to manifest phenotypically, thus affecting how cells respond to drugs or to minor differences in culture media. Knowing how to interpret karyotype heterogeneity phenotypically would give insights into cellular phenotypes before they unfold temporally. Here, we re-analyzed single cell RNA (scRNA) and scDNA sequencing data from eight stomach...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37510235"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Analysis in the Omics Era: Technologies and Applications in Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37510235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer molecular profiling obtained with conventional bulk sequencing describes average alterations obtained from the entire cellular population analyzed. In the era of precision medicine, this approach is unable to track tumor heterogeneity and cannot be exploited to unravel the biological processes behind clonal evolution. In the last few years, functional single-cell omics has improved our understanding of cancer heterogeneity. This approach requires isolation and identification of single...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37508780"
                        open
                      >
                        <summary class="article-expander__title">Extreme Gradient Boosting Tuned with Metaheuristic Algorithms for Predicting Myeloid NGS Onco-Somatic Variant Pathogenicity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37508780/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The advent of next-generation sequencing (NGS) technologies has revolutionized the field of bioinformatics and genomics, particularly in the area of onco-somatic genetics. NGS has provided a wealth of information about the genetic changes that underlie cancer and has considerably improved our ability to diagnose and treat cancer. However, the large amount of data generated by NGS makes it difficult to interpret the variants. To address this, machine learning algorithms such as Extreme Gradient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37508513"
                        open
                      >
                        <summary class="article-expander__title">Mathematical Modeling of Clonal Interference by Density-Dependent Selection in Heterogeneous Cancer Cell Lines</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37508513/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Many cancer cell lines are aneuploid and heterogeneous, with multiple karyotypes co-existing within the same cell line. Karyotype heterogeneity has been shown to manifest phenotypically, thus affecting how cells respond to drugs or to minor differences in culture media. Knowing how to interpret karyotype heterogeneity phenotypically would give insights into cellular phenotypes before they unfold temporally. Here, we re-analyzed single cell RNA (scRNA) and scDNA sequencing data from eight stomach...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37510235"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Analysis in the Omics Era: Technologies and Applications in Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37510235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer molecular profiling obtained with conventional bulk sequencing describes average alterations obtained from the entire cellular population analyzed. In the era of precision medicine, this approach is unable to track tumor heterogeneity and cannot be exploited to unravel the biological processes behind clonal evolution. In the last few years, functional single-cell omics has improved our understanding of cancer heterogeneity. This approach requires isolation and identification of single...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37508780"
                        open
                      >
                        <summary class="article-expander__title">Extreme Gradient Boosting Tuned with Metaheuristic Algorithms for Predicting Myeloid NGS Onco-Somatic Variant Pathogenicity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37508780/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The advent of next-generation sequencing (NGS) technologies has revolutionized the field of bioinformatics and genomics, particularly in the area of onco-somatic genetics. NGS has provided a wealth of information about the genetic changes that underlie cancer and has considerably improved our ability to diagnose and treat cancer. However, the large amount of data generated by NGS makes it difficult to interpret the variants. To address this, machine learning algorithms such as Extreme Gradient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37508513"
                        open
                      >
                        <summary class="article-expander__title">Mathematical Modeling of Clonal Interference by Density-Dependent Selection in Heterogeneous Cancer Cell Lines</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37508513/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Many cancer cell lines are aneuploid and heterogeneous, with multiple karyotypes co-existing within the same cell line. Karyotype heterogeneity has been shown to manifest phenotypically, thus affecting how cells respond to drugs or to minor differences in culture media. Knowing how to interpret karyotype heterogeneity phenotypically would give insights into cellular phenotypes before they unfold temporally. Here, we re-analyzed single cell RNA (scRNA) and scDNA sequencing data from eight stomach...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37510235"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Analysis in the Omics Era: Technologies and Applications in Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37510235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer molecular profiling obtained with conventional bulk sequencing describes average alterations obtained from the entire cellular population analyzed. In the era of precision medicine, this approach is unable to track tumor heterogeneity and cannot be exploited to unravel the biological processes behind clonal evolution. In the last few years, functional single-cell omics has improved our understanding of cancer heterogeneity. This approach requires isolation and identification of single...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37508780"
                        open
                      >
                        <summary class="article-expander__title">Extreme Gradient Boosting Tuned with Metaheuristic Algorithms for Predicting Myeloid NGS Onco-Somatic Variant Pathogenicity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37508780/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The advent of next-generation sequencing (NGS) technologies has revolutionized the field of bioinformatics and genomics, particularly in the area of onco-somatic genetics. NGS has provided a wealth of information about the genetic changes that underlie cancer and has considerably improved our ability to diagnose and treat cancer. However, the large amount of data generated by NGS makes it difficult to interpret the variants. To address this, machine learning algorithms such as Extreme Gradient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37508513"
                        open
                      >
                        <summary class="article-expander__title">Mathematical Modeling of Clonal Interference by Density-Dependent Selection in Heterogeneous Cancer Cell Lines</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37508513/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Many cancer cell lines are aneuploid and heterogeneous, with multiple karyotypes co-existing within the same cell line. Karyotype heterogeneity has been shown to manifest phenotypically, thus affecting how cells respond to drugs or to minor differences in culture media. Knowing how to interpret karyotype heterogeneity phenotypically would give insights into cellular phenotypes before they unfold temporally. Here, we re-analyzed single cell RNA (scRNA) and scDNA sequencing data from eight stomach...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37510235"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Analysis in the Omics Era: Technologies and Applications in Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37510235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer molecular profiling obtained with conventional bulk sequencing describes average alterations obtained from the entire cellular population analyzed. In the era of precision medicine, this approach is unable to track tumor heterogeneity and cannot be exploited to unravel the biological processes behind clonal evolution. In the last few years, functional single-cell omics has improved our understanding of cancer heterogeneity. This approach requires isolation and identification of single...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37508780"
                        open
                      >
                        <summary class="article-expander__title">Extreme Gradient Boosting Tuned with Metaheuristic Algorithms for Predicting Myeloid NGS Onco-Somatic Variant Pathogenicity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37508780/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The advent of next-generation sequencing (NGS) technologies has revolutionized the field of bioinformatics and genomics, particularly in the area of onco-somatic genetics. NGS has provided a wealth of information about the genetic changes that underlie cancer and has considerably improved our ability to diagnose and treat cancer. However, the large amount of data generated by NGS makes it difficult to interpret the variants. To address this, machine learning algorithms such as Extreme Gradient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37508513"
                        open
                      >
                        <summary class="article-expander__title">Mathematical Modeling of Clonal Interference by Density-Dependent Selection in Heterogeneous Cancer Cell Lines</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37508513/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Many cancer cell lines are aneuploid and heterogeneous, with multiple karyotypes co-existing within the same cell line. Karyotype heterogeneity has been shown to manifest phenotypically, thus affecting how cells respond to drugs or to minor differences in culture media. Knowing how to interpret karyotype heterogeneity phenotypically would give insights into cellular phenotypes before they unfold temporally. Here, we re-analyzed single cell RNA (scRNA) and scDNA sequencing data from eight stomach...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37510235"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Analysis in the Omics Era: Technologies and Applications in Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37510235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer molecular profiling obtained with conventional bulk sequencing describes average alterations obtained from the entire cellular population analyzed. In the era of precision medicine, this approach is unable to track tumor heterogeneity and cannot be exploited to unravel the biological processes behind clonal evolution. In the last few years, functional single-cell omics has improved our understanding of cancer heterogeneity. This approach requires isolation and identification of single...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37508780"
                        open
                      >
                        <summary class="article-expander__title">Extreme Gradient Boosting Tuned with Metaheuristic Algorithms for Predicting Myeloid NGS Onco-Somatic Variant Pathogenicity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37508780/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The advent of next-generation sequencing (NGS) technologies has revolutionized the field of bioinformatics and genomics, particularly in the area of onco-somatic genetics. NGS has provided a wealth of information about the genetic changes that underlie cancer and has considerably improved our ability to diagnose and treat cancer. However, the large amount of data generated by NGS makes it difficult to interpret the variants. To address this, machine learning algorithms such as Extreme Gradient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37508513"
                        open
                      >
                        <summary class="article-expander__title">Mathematical Modeling of Clonal Interference by Density-Dependent Selection in Heterogeneous Cancer Cell Lines</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37508513/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Many cancer cell lines are aneuploid and heterogeneous, with multiple karyotypes co-existing within the same cell line. Karyotype heterogeneity has been shown to manifest phenotypically, thus affecting how cells respond to drugs or to minor differences in culture media. Knowing how to interpret karyotype heterogeneity phenotypically would give insights into cellular phenotypes before they unfold temporally. Here, we re-analyzed single cell RNA (scRNA) and scDNA sequencing data from eight stomach...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37510235"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Analysis in the Omics Era: Technologies and Applications in Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37510235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer molecular profiling obtained with conventional bulk sequencing describes average alterations obtained from the entire cellular population analyzed. In the era of precision medicine, this approach is unable to track tumor heterogeneity and cannot be exploited to unravel the biological processes behind clonal evolution. In the last few years, functional single-cell omics has improved our understanding of cancer heterogeneity. This approach requires isolation and identification of single...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37508780"
                        open
                      >
                        <summary class="article-expander__title">Extreme Gradient Boosting Tuned with Metaheuristic Algorithms for Predicting Myeloid NGS Onco-Somatic Variant Pathogenicity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37508780/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The advent of next-generation sequencing (NGS) technologies has revolutionized the field of bioinformatics and genomics, particularly in the area of onco-somatic genetics. NGS has provided a wealth of information about the genetic changes that underlie cancer and has considerably improved our ability to diagnose and treat cancer. However, the large amount of data generated by NGS makes it difficult to interpret the variants. To address this, machine learning algorithms such as Extreme Gradient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37508513"
                        open
                      >
                        <summary class="article-expander__title">Mathematical Modeling of Clonal Interference by Density-Dependent Selection in Heterogeneous Cancer Cell Lines</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37508513/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Many cancer cell lines are aneuploid and heterogeneous, with multiple karyotypes co-existing within the same cell line. Karyotype heterogeneity has been shown to manifest phenotypically, thus affecting how cells respond to drugs or to minor differences in culture media. Knowing how to interpret karyotype heterogeneity phenotypically would give insights into cellular phenotypes before they unfold temporally. Here, we re-analyzed single cell RNA (scRNA) and scDNA sequencing data from eight stomach...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-07-28 datetime="2023-07-28T10:00:00.000Z">2023-07-28</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-07-28 datetime="2023-07-28T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37506341"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of DDX41 gene variants in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37506341/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline variants in the DDX41 gene have been linked to myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) development. However, the risks associated with different variants remain unknown, as do the basis of their leukemogenic properties, impact on steady-state hematopoiesis and links to other cancers. Here, we investigate the frequency and significance of DDX41 variants in 454,792 United Kingdom Biobank (UKB) participants and identify 452 unique non-synonymous DNA variants in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37505860"
                        open
                      >
                        <summary class="article-expander__title">Detection of Age-Related Somatic Alterations in Canine Blood Using Next-Generation Sequencing-Based Liquid Biopsy: An Analysis of over 4800 Dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37505860/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-related somatic genomic alterations in hematopoietic cell lines have been well characterized in humans; however, this phenomenon has not been well studied in other species. Next-generation sequencing-based liquid biopsy testing for cancer detection was recently developed for dogs and has been used to study the genomic profiles of blood samples from thousands of canine patients since 2021. In this study, 4870 client-owned dogs with and without a diagnosis or suspicion of cancer underwent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37504319"
                        open
                      >
                        <summary class="article-expander__title">Management of Patients with Lower-Risk Myelodysplastic Neoplasms (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37504319/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms (MDS) are a heterogenous group of clonal hematologic disorders characterized by morphologic dysplasia, ineffective hematopoiesis, and cytopenia. In the past year, the classification of MDS has been updated in the 5th edition of the World Health Organization (WHO) Classification of Haematolymphoid Tumours and the International Consensus Classification (ICC) of Myeloid Neoplasms and Acute Leukemia with incorporation of morphologic, clinical, and genomic data. Furthermore,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37506341"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of DDX41 gene variants in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37506341/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline variants in the DDX41 gene have been linked to myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) development. However, the risks associated with different variants remain unknown, as do the basis of their leukemogenic properties, impact on steady-state hematopoiesis and links to other cancers. Here, we investigate the frequency and significance of DDX41 variants in 454,792 United Kingdom Biobank (UKB) participants and identify 452 unique non-synonymous DNA variants in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37505860"
                        open
                      >
                        <summary class="article-expander__title">Detection of Age-Related Somatic Alterations in Canine Blood Using Next-Generation Sequencing-Based Liquid Biopsy: An Analysis of over 4800 Dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37505860/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-related somatic genomic alterations in hematopoietic cell lines have been well characterized in humans; however, this phenomenon has not been well studied in other species. Next-generation sequencing-based liquid biopsy testing for cancer detection was recently developed for dogs and has been used to study the genomic profiles of blood samples from thousands of canine patients since 2021. In this study, 4870 client-owned dogs with and without a diagnosis or suspicion of cancer underwent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37504319"
                        open
                      >
                        <summary class="article-expander__title">Management of Patients with Lower-Risk Myelodysplastic Neoplasms (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37504319/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms (MDS) are a heterogenous group of clonal hematologic disorders characterized by morphologic dysplasia, ineffective hematopoiesis, and cytopenia. In the past year, the classification of MDS has been updated in the 5th edition of the World Health Organization (WHO) Classification of Haematolymphoid Tumours and the International Consensus Classification (ICC) of Myeloid Neoplasms and Acute Leukemia with incorporation of morphologic, clinical, and genomic data. Furthermore,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37503259"
                        open
                      >
                        <summary class="article-expander__title">Unlocking the Role of Endothelial MPL Receptor and JAK2V617F Mutation: Insights into Cardiovascular Dysfunction in MPNs and CHIP</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37503259/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with JAK2V617F-positive myeloproliferative neoplasms (MPNs) and clonal hematopoiesis of indeterminate potential (CHIP) are at a significantly higher risk of cardiovascular diseases (CVDs). Endothelial cells (ECs) carrying the JAK2V617F mutation can be detected in many MPN patients. Here, we investigated the impact of endothelial JAK2V617F mutation on CVD development using both transgenic murine models and human induced pluripotent stem cell lines. Our findings revealed that JAK2V617F...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37502912"
                        open
                      >
                        <summary class="article-expander__title">Oncostatin M is a Master Regulator of an Inflammatory Network in Dnmt3a -Mutant Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37502912/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated clonal hematopoiesis (CH) occurs due to somatic mutations accrued in hematopoietic stem cells (HSCs) that confer a selective advantage in the context of aging. The mechanisms by which CH-mutant HSCs gain this advantage with aging are not comprehensively understood. Using unbiased transcriptomic approaches, we identify Oncostatin M (OSM) signaling as a candidate contributor to aging-driven Dnmt3a -mutant CH. We find that Dnmt3a -mutant HSCs from young mice do not functionally...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37506341"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of DDX41 gene variants in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37506341/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline variants in the DDX41 gene have been linked to myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) development. However, the risks associated with different variants remain unknown, as do the basis of their leukemogenic properties, impact on steady-state hematopoiesis and links to other cancers. Here, we investigate the frequency and significance of DDX41 variants in 454,792 United Kingdom Biobank (UKB) participants and identify 452 unique non-synonymous DNA variants in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37505860"
                        open
                      >
                        <summary class="article-expander__title">Detection of Age-Related Somatic Alterations in Canine Blood Using Next-Generation Sequencing-Based Liquid Biopsy: An Analysis of over 4800 Dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37505860/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-related somatic genomic alterations in hematopoietic cell lines have been well characterized in humans; however, this phenomenon has not been well studied in other species. Next-generation sequencing-based liquid biopsy testing for cancer detection was recently developed for dogs and has been used to study the genomic profiles of blood samples from thousands of canine patients since 2021. In this study, 4870 client-owned dogs with and without a diagnosis or suspicion of cancer underwent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37504319"
                        open
                      >
                        <summary class="article-expander__title">Management of Patients with Lower-Risk Myelodysplastic Neoplasms (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37504319/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms (MDS) are a heterogenous group of clonal hematologic disorders characterized by morphologic dysplasia, ineffective hematopoiesis, and cytopenia. In the past year, the classification of MDS has been updated in the 5th edition of the World Health Organization (WHO) Classification of Haematolymphoid Tumours and the International Consensus Classification (ICC) of Myeloid Neoplasms and Acute Leukemia with incorporation of morphologic, clinical, and genomic data. Furthermore,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37503259"
                        open
                      >
                        <summary class="article-expander__title">Unlocking the Role of Endothelial MPL Receptor and JAK2V617F Mutation: Insights into Cardiovascular Dysfunction in MPNs and CHIP</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37503259/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with JAK2V617F-positive myeloproliferative neoplasms (MPNs) and clonal hematopoiesis of indeterminate potential (CHIP) are at a significantly higher risk of cardiovascular diseases (CVDs). Endothelial cells (ECs) carrying the JAK2V617F mutation can be detected in many MPN patients. Here, we investigated the impact of endothelial JAK2V617F mutation on CVD development using both transgenic murine models and human induced pluripotent stem cell lines. Our findings revealed that JAK2V617F...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37502912"
                        open
                      >
                        <summary class="article-expander__title">Oncostatin M is a Master Regulator of an Inflammatory Network in Dnmt3a -Mutant Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37502912/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated clonal hematopoiesis (CH) occurs due to somatic mutations accrued in hematopoietic stem cells (HSCs) that confer a selective advantage in the context of aging. The mechanisms by which CH-mutant HSCs gain this advantage with aging are not comprehensively understood. Using unbiased transcriptomic approaches, we identify Oncostatin M (OSM) signaling as a candidate contributor to aging-driven Dnmt3a -mutant CH. We find that Dnmt3a -mutant HSCs from young mice do not functionally...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37506341"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of DDX41 gene variants in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37506341/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline variants in the DDX41 gene have been linked to myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) development. However, the risks associated with different variants remain unknown, as do the basis of their leukemogenic properties, impact on steady-state hematopoiesis and links to other cancers. Here, we investigate the frequency and significance of DDX41 variants in 454,792 United Kingdom Biobank (UKB) participants and identify 452 unique non-synonymous DNA variants in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37505860"
                        open
                      >
                        <summary class="article-expander__title">Detection of Age-Related Somatic Alterations in Canine Blood Using Next-Generation Sequencing-Based Liquid Biopsy: An Analysis of over 4800 Dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37505860/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-related somatic genomic alterations in hematopoietic cell lines have been well characterized in humans; however, this phenomenon has not been well studied in other species. Next-generation sequencing-based liquid biopsy testing for cancer detection was recently developed for dogs and has been used to study the genomic profiles of blood samples from thousands of canine patients since 2021. In this study, 4870 client-owned dogs with and without a diagnosis or suspicion of cancer underwent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37504319"
                        open
                      >
                        <summary class="article-expander__title">Management of Patients with Lower-Risk Myelodysplastic Neoplasms (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37504319/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms (MDS) are a heterogenous group of clonal hematologic disorders characterized by morphologic dysplasia, ineffective hematopoiesis, and cytopenia. In the past year, the classification of MDS has been updated in the 5th edition of the World Health Organization (WHO) Classification of Haematolymphoid Tumours and the International Consensus Classification (ICC) of Myeloid Neoplasms and Acute Leukemia with incorporation of morphologic, clinical, and genomic data. Furthermore,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37503259"
                        open
                      >
                        <summary class="article-expander__title">Unlocking the Role of Endothelial MPL Receptor and JAK2V617F Mutation: Insights into Cardiovascular Dysfunction in MPNs and CHIP</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37503259/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with JAK2V617F-positive myeloproliferative neoplasms (MPNs) and clonal hematopoiesis of indeterminate potential (CHIP) are at a significantly higher risk of cardiovascular diseases (CVDs). Endothelial cells (ECs) carrying the JAK2V617F mutation can be detected in many MPN patients. Here, we investigated the impact of endothelial JAK2V617F mutation on CVD development using both transgenic murine models and human induced pluripotent stem cell lines. Our findings revealed that JAK2V617F...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37502912"
                        open
                      >
                        <summary class="article-expander__title">Oncostatin M is a Master Regulator of an Inflammatory Network in Dnmt3a -Mutant Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37502912/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated clonal hematopoiesis (CH) occurs due to somatic mutations accrued in hematopoietic stem cells (HSCs) that confer a selective advantage in the context of aging. The mechanisms by which CH-mutant HSCs gain this advantage with aging are not comprehensively understood. Using unbiased transcriptomic approaches, we identify Oncostatin M (OSM) signaling as a candidate contributor to aging-driven Dnmt3a -mutant CH. We find that Dnmt3a -mutant HSCs from young mice do not functionally...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37501402"
                        open
                      >
                        <summary class="article-expander__title">Subclones with variants of uncertain clinical significance might contribute to ineffective hemopoiesis and leukemia predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37501402/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In conclusions, we linked both pathogenic and VUS variants in AML-related genes to clonal hematopoiesis; therefore, we proposed to consider those variants as prognostic markers in leukemia and myelodysplasia. However, further studies in larger prospective cohorts are required to validate our results.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37506341"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of DDX41 gene variants in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37506341/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline variants in the DDX41 gene have been linked to myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) development. However, the risks associated with different variants remain unknown, as do the basis of their leukemogenic properties, impact on steady-state hematopoiesis and links to other cancers. Here, we investigate the frequency and significance of DDX41 variants in 454,792 United Kingdom Biobank (UKB) participants and identify 452 unique non-synonymous DNA variants in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37505860"
                        open
                      >
                        <summary class="article-expander__title">Detection of Age-Related Somatic Alterations in Canine Blood Using Next-Generation Sequencing-Based Liquid Biopsy: An Analysis of over 4800 Dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37505860/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-related somatic genomic alterations in hematopoietic cell lines have been well characterized in humans; however, this phenomenon has not been well studied in other species. Next-generation sequencing-based liquid biopsy testing for cancer detection was recently developed for dogs and has been used to study the genomic profiles of blood samples from thousands of canine patients since 2021. In this study, 4870 client-owned dogs with and without a diagnosis or suspicion of cancer underwent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37504319"
                        open
                      >
                        <summary class="article-expander__title">Management of Patients with Lower-Risk Myelodysplastic Neoplasms (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37504319/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms (MDS) are a heterogenous group of clonal hematologic disorders characterized by morphologic dysplasia, ineffective hematopoiesis, and cytopenia. In the past year, the classification of MDS has been updated in the 5th edition of the World Health Organization (WHO) Classification of Haematolymphoid Tumours and the International Consensus Classification (ICC) of Myeloid Neoplasms and Acute Leukemia with incorporation of morphologic, clinical, and genomic data. Furthermore,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37503259"
                        open
                      >
                        <summary class="article-expander__title">Unlocking the Role of Endothelial MPL Receptor and JAK2V617F Mutation: Insights into Cardiovascular Dysfunction in MPNs and CHIP</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37503259/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with JAK2V617F-positive myeloproliferative neoplasms (MPNs) and clonal hematopoiesis of indeterminate potential (CHIP) are at a significantly higher risk of cardiovascular diseases (CVDs). Endothelial cells (ECs) carrying the JAK2V617F mutation can be detected in many MPN patients. Here, we investigated the impact of endothelial JAK2V617F mutation on CVD development using both transgenic murine models and human induced pluripotent stem cell lines. Our findings revealed that JAK2V617F...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37502912"
                        open
                      >
                        <summary class="article-expander__title">Oncostatin M is a Master Regulator of an Inflammatory Network in Dnmt3a -Mutant Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37502912/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated clonal hematopoiesis (CH) occurs due to somatic mutations accrued in hematopoietic stem cells (HSCs) that confer a selective advantage in the context of aging. The mechanisms by which CH-mutant HSCs gain this advantage with aging are not comprehensively understood. Using unbiased transcriptomic approaches, we identify Oncostatin M (OSM) signaling as a candidate contributor to aging-driven Dnmt3a -mutant CH. We find that Dnmt3a -mutant HSCs from young mice do not functionally...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37501402"
                        open
                      >
                        <summary class="article-expander__title">Subclones with variants of uncertain clinical significance might contribute to ineffective hemopoiesis and leukemia predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37501402/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In conclusions, we linked both pathogenic and VUS variants in AML-related genes to clonal hematopoiesis; therefore, we proposed to consider those variants as prognostic markers in leukemia and myelodysplasia. However, further studies in larger prospective cohorts are required to validate our results.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37506341"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of DDX41 gene variants in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37506341/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline variants in the DDX41 gene have been linked to myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) development. However, the risks associated with different variants remain unknown, as do the basis of their leukemogenic properties, impact on steady-state hematopoiesis and links to other cancers. Here, we investigate the frequency and significance of DDX41 variants in 454,792 United Kingdom Biobank (UKB) participants and identify 452 unique non-synonymous DNA variants in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37505860"
                        open
                      >
                        <summary class="article-expander__title">Detection of Age-Related Somatic Alterations in Canine Blood Using Next-Generation Sequencing-Based Liquid Biopsy: An Analysis of over 4800 Dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37505860/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-related somatic genomic alterations in hematopoietic cell lines have been well characterized in humans; however, this phenomenon has not been well studied in other species. Next-generation sequencing-based liquid biopsy testing for cancer detection was recently developed for dogs and has been used to study the genomic profiles of blood samples from thousands of canine patients since 2021. In this study, 4870 client-owned dogs with and without a diagnosis or suspicion of cancer underwent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37504319"
                        open
                      >
                        <summary class="article-expander__title">Management of Patients with Lower-Risk Myelodysplastic Neoplasms (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37504319/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms (MDS) are a heterogenous group of clonal hematologic disorders characterized by morphologic dysplasia, ineffective hematopoiesis, and cytopenia. In the past year, the classification of MDS has been updated in the 5th edition of the World Health Organization (WHO) Classification of Haematolymphoid Tumours and the International Consensus Classification (ICC) of Myeloid Neoplasms and Acute Leukemia with incorporation of morphologic, clinical, and genomic data. Furthermore,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37503259"
                        open
                      >
                        <summary class="article-expander__title">Unlocking the Role of Endothelial MPL Receptor and JAK2V617F Mutation: Insights into Cardiovascular Dysfunction in MPNs and CHIP</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37503259/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with JAK2V617F-positive myeloproliferative neoplasms (MPNs) and clonal hematopoiesis of indeterminate potential (CHIP) are at a significantly higher risk of cardiovascular diseases (CVDs). Endothelial cells (ECs) carrying the JAK2V617F mutation can be detected in many MPN patients. Here, we investigated the impact of endothelial JAK2V617F mutation on CVD development using both transgenic murine models and human induced pluripotent stem cell lines. Our findings revealed that JAK2V617F...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37502912"
                        open
                      >
                        <summary class="article-expander__title">Oncostatin M is a Master Regulator of an Inflammatory Network in Dnmt3a -Mutant Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37502912/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated clonal hematopoiesis (CH) occurs due to somatic mutations accrued in hematopoietic stem cells (HSCs) that confer a selective advantage in the context of aging. The mechanisms by which CH-mutant HSCs gain this advantage with aging are not comprehensively understood. Using unbiased transcriptomic approaches, we identify Oncostatin M (OSM) signaling as a candidate contributor to aging-driven Dnmt3a -mutant CH. We find that Dnmt3a -mutant HSCs from young mice do not functionally...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37501402"
                        open
                      >
                        <summary class="article-expander__title">Subclones with variants of uncertain clinical significance might contribute to ineffective hemopoiesis and leukemia predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37501402/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In conclusions, we linked both pathogenic and VUS variants in AML-related genes to clonal hematopoiesis; therefore, we proposed to consider those variants as prognostic markers in leukemia and myelodysplasia. However, further studies in larger prospective cohorts are required to validate our results.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37506341"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of DDX41 gene variants in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37506341/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline variants in the DDX41 gene have been linked to myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) development. However, the risks associated with different variants remain unknown, as do the basis of their leukemogenic properties, impact on steady-state hematopoiesis and links to other cancers. Here, we investigate the frequency and significance of DDX41 variants in 454,792 United Kingdom Biobank (UKB) participants and identify 452 unique non-synonymous DNA variants in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37505860"
                        open
                      >
                        <summary class="article-expander__title">Detection of Age-Related Somatic Alterations in Canine Blood Using Next-Generation Sequencing-Based Liquid Biopsy: An Analysis of over 4800 Dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37505860/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-related somatic genomic alterations in hematopoietic cell lines have been well characterized in humans; however, this phenomenon has not been well studied in other species. Next-generation sequencing-based liquid biopsy testing for cancer detection was recently developed for dogs and has been used to study the genomic profiles of blood samples from thousands of canine patients since 2021. In this study, 4870 client-owned dogs with and without a diagnosis or suspicion of cancer underwent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37504319"
                        open
                      >
                        <summary class="article-expander__title">Management of Patients with Lower-Risk Myelodysplastic Neoplasms (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37504319/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms (MDS) are a heterogenous group of clonal hematologic disorders characterized by morphologic dysplasia, ineffective hematopoiesis, and cytopenia. In the past year, the classification of MDS has been updated in the 5th edition of the World Health Organization (WHO) Classification of Haematolymphoid Tumours and the International Consensus Classification (ICC) of Myeloid Neoplasms and Acute Leukemia with incorporation of morphologic, clinical, and genomic data. Furthermore,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37503259"
                        open
                      >
                        <summary class="article-expander__title">Unlocking the Role of Endothelial MPL Receptor and JAK2V617F Mutation: Insights into Cardiovascular Dysfunction in MPNs and CHIP</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37503259/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with JAK2V617F-positive myeloproliferative neoplasms (MPNs) and clonal hematopoiesis of indeterminate potential (CHIP) are at a significantly higher risk of cardiovascular diseases (CVDs). Endothelial cells (ECs) carrying the JAK2V617F mutation can be detected in many MPN patients. Here, we investigated the impact of endothelial JAK2V617F mutation on CVD development using both transgenic murine models and human induced pluripotent stem cell lines. Our findings revealed that JAK2V617F...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37502912"
                        open
                      >
                        <summary class="article-expander__title">Oncostatin M is a Master Regulator of an Inflammatory Network in Dnmt3a -Mutant Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37502912/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated clonal hematopoiesis (CH) occurs due to somatic mutations accrued in hematopoietic stem cells (HSCs) that confer a selective advantage in the context of aging. The mechanisms by which CH-mutant HSCs gain this advantage with aging are not comprehensively understood. Using unbiased transcriptomic approaches, we identify Oncostatin M (OSM) signaling as a candidate contributor to aging-driven Dnmt3a -mutant CH. We find that Dnmt3a -mutant HSCs from young mice do not functionally...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37501402"
                        open
                      >
                        <summary class="article-expander__title">Subclones with variants of uncertain clinical significance might contribute to ineffective hemopoiesis and leukemia predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37501402/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In conclusions, we linked both pathogenic and VUS variants in AML-related genes to clonal hematopoiesis; therefore, we proposed to consider those variants as prognostic markers in leukemia and myelodysplasia. However, further studies in larger prospective cohorts are required to validate our results.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37506341"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of DDX41 gene variants in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37506341/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline variants in the DDX41 gene have been linked to myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) development. However, the risks associated with different variants remain unknown, as do the basis of their leukemogenic properties, impact on steady-state hematopoiesis and links to other cancers. Here, we investigate the frequency and significance of DDX41 variants in 454,792 United Kingdom Biobank (UKB) participants and identify 452 unique non-synonymous DNA variants in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37505860"
                        open
                      >
                        <summary class="article-expander__title">Detection of Age-Related Somatic Alterations in Canine Blood Using Next-Generation Sequencing-Based Liquid Biopsy: An Analysis of over 4800 Dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37505860/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-related somatic genomic alterations in hematopoietic cell lines have been well characterized in humans; however, this phenomenon has not been well studied in other species. Next-generation sequencing-based liquid biopsy testing for cancer detection was recently developed for dogs and has been used to study the genomic profiles of blood samples from thousands of canine patients since 2021. In this study, 4870 client-owned dogs with and without a diagnosis or suspicion of cancer underwent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37504319"
                        open
                      >
                        <summary class="article-expander__title">Management of Patients with Lower-Risk Myelodysplastic Neoplasms (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37504319/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms (MDS) are a heterogenous group of clonal hematologic disorders characterized by morphologic dysplasia, ineffective hematopoiesis, and cytopenia. In the past year, the classification of MDS has been updated in the 5th edition of the World Health Organization (WHO) Classification of Haematolymphoid Tumours and the International Consensus Classification (ICC) of Myeloid Neoplasms and Acute Leukemia with incorporation of morphologic, clinical, and genomic data. Furthermore,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37503259"
                        open
                      >
                        <summary class="article-expander__title">Unlocking the Role of Endothelial MPL Receptor and JAK2V617F Mutation: Insights into Cardiovascular Dysfunction in MPNs and CHIP</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37503259/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with JAK2V617F-positive myeloproliferative neoplasms (MPNs) and clonal hematopoiesis of indeterminate potential (CHIP) are at a significantly higher risk of cardiovascular diseases (CVDs). Endothelial cells (ECs) carrying the JAK2V617F mutation can be detected in many MPN patients. Here, we investigated the impact of endothelial JAK2V617F mutation on CVD development using both transgenic murine models and human induced pluripotent stem cell lines. Our findings revealed that JAK2V617F...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37502912"
                        open
                      >
                        <summary class="article-expander__title">Oncostatin M is a Master Regulator of an Inflammatory Network in Dnmt3a -Mutant Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37502912/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated clonal hematopoiesis (CH) occurs due to somatic mutations accrued in hematopoietic stem cells (HSCs) that confer a selective advantage in the context of aging. The mechanisms by which CH-mutant HSCs gain this advantage with aging are not comprehensively understood. Using unbiased transcriptomic approaches, we identify Oncostatin M (OSM) signaling as a candidate contributor to aging-driven Dnmt3a -mutant CH. We find that Dnmt3a -mutant HSCs from young mice do not functionally...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37501402"
                        open
                      >
                        <summary class="article-expander__title">Subclones with variants of uncertain clinical significance might contribute to ineffective hemopoiesis and leukemia predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37501402/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In conclusions, we linked both pathogenic and VUS variants in AML-related genes to clonal hematopoiesis; therefore, we proposed to consider those variants as prognostic markers in leukemia and myelodysplasia. However, further studies in larger prospective cohorts are required to validate our results.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37506341"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of DDX41 gene variants in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37506341/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline variants in the DDX41 gene have been linked to myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) development. However, the risks associated with different variants remain unknown, as do the basis of their leukemogenic properties, impact on steady-state hematopoiesis and links to other cancers. Here, we investigate the frequency and significance of DDX41 variants in 454,792 United Kingdom Biobank (UKB) participants and identify 452 unique non-synonymous DNA variants in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37505860"
                        open
                      >
                        <summary class="article-expander__title">Detection of Age-Related Somatic Alterations in Canine Blood Using Next-Generation Sequencing-Based Liquid Biopsy: An Analysis of over 4800 Dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37505860/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-related somatic genomic alterations in hematopoietic cell lines have been well characterized in humans; however, this phenomenon has not been well studied in other species. Next-generation sequencing-based liquid biopsy testing for cancer detection was recently developed for dogs and has been used to study the genomic profiles of blood samples from thousands of canine patients since 2021. In this study, 4870 client-owned dogs with and without a diagnosis or suspicion of cancer underwent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37504319"
                        open
                      >
                        <summary class="article-expander__title">Management of Patients with Lower-Risk Myelodysplastic Neoplasms (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37504319/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms (MDS) are a heterogenous group of clonal hematologic disorders characterized by morphologic dysplasia, ineffective hematopoiesis, and cytopenia. In the past year, the classification of MDS has been updated in the 5th edition of the World Health Organization (WHO) Classification of Haematolymphoid Tumours and the International Consensus Classification (ICC) of Myeloid Neoplasms and Acute Leukemia with incorporation of morphologic, clinical, and genomic data. Furthermore,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37503259"
                        open
                      >
                        <summary class="article-expander__title">Unlocking the Role of Endothelial MPL Receptor and JAK2V617F Mutation: Insights into Cardiovascular Dysfunction in MPNs and CHIP</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37503259/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with JAK2V617F-positive myeloproliferative neoplasms (MPNs) and clonal hematopoiesis of indeterminate potential (CHIP) are at a significantly higher risk of cardiovascular diseases (CVDs). Endothelial cells (ECs) carrying the JAK2V617F mutation can be detected in many MPN patients. Here, we investigated the impact of endothelial JAK2V617F mutation on CVD development using both transgenic murine models and human induced pluripotent stem cell lines. Our findings revealed that JAK2V617F...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37502912"
                        open
                      >
                        <summary class="article-expander__title">Oncostatin M is a Master Regulator of an Inflammatory Network in Dnmt3a -Mutant Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37502912/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated clonal hematopoiesis (CH) occurs due to somatic mutations accrued in hematopoietic stem cells (HSCs) that confer a selective advantage in the context of aging. The mechanisms by which CH-mutant HSCs gain this advantage with aging are not comprehensively understood. Using unbiased transcriptomic approaches, we identify Oncostatin M (OSM) signaling as a candidate contributor to aging-driven Dnmt3a -mutant CH. We find that Dnmt3a -mutant HSCs from young mice do not functionally...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37501402"
                        open
                      >
                        <summary class="article-expander__title">Subclones with variants of uncertain clinical significance might contribute to ineffective hemopoiesis and leukemia predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37501402/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In conclusions, we linked both pathogenic and VUS variants in AML-related genes to clonal hematopoiesis; therefore, we proposed to consider those variants as prognostic markers in leukemia and myelodysplasia. However, further studies in larger prospective cohorts are required to validate our results.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37506341"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of DDX41 gene variants in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37506341/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline variants in the DDX41 gene have been linked to myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) development. However, the risks associated with different variants remain unknown, as do the basis of their leukemogenic properties, impact on steady-state hematopoiesis and links to other cancers. Here, we investigate the frequency and significance of DDX41 variants in 454,792 United Kingdom Biobank (UKB) participants and identify 452 unique non-synonymous DNA variants in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37505860"
                        open
                      >
                        <summary class="article-expander__title">Detection of Age-Related Somatic Alterations in Canine Blood Using Next-Generation Sequencing-Based Liquid Biopsy: An Analysis of over 4800 Dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37505860/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-related somatic genomic alterations in hematopoietic cell lines have been well characterized in humans; however, this phenomenon has not been well studied in other species. Next-generation sequencing-based liquid biopsy testing for cancer detection was recently developed for dogs and has been used to study the genomic profiles of blood samples from thousands of canine patients since 2021. In this study, 4870 client-owned dogs with and without a diagnosis or suspicion of cancer underwent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37504319"
                        open
                      >
                        <summary class="article-expander__title">Management of Patients with Lower-Risk Myelodysplastic Neoplasms (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37504319/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms (MDS) are a heterogenous group of clonal hematologic disorders characterized by morphologic dysplasia, ineffective hematopoiesis, and cytopenia. In the past year, the classification of MDS has been updated in the 5th edition of the World Health Organization (WHO) Classification of Haematolymphoid Tumours and the International Consensus Classification (ICC) of Myeloid Neoplasms and Acute Leukemia with incorporation of morphologic, clinical, and genomic data. Furthermore,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37503259"
                        open
                      >
                        <summary class="article-expander__title">Unlocking the Role of Endothelial MPL Receptor and JAK2V617F Mutation: Insights into Cardiovascular Dysfunction in MPNs and CHIP</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37503259/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with JAK2V617F-positive myeloproliferative neoplasms (MPNs) and clonal hematopoiesis of indeterminate potential (CHIP) are at a significantly higher risk of cardiovascular diseases (CVDs). Endothelial cells (ECs) carrying the JAK2V617F mutation can be detected in many MPN patients. Here, we investigated the impact of endothelial JAK2V617F mutation on CVD development using both transgenic murine models and human induced pluripotent stem cell lines. Our findings revealed that JAK2V617F...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37502912"
                        open
                      >
                        <summary class="article-expander__title">Oncostatin M is a Master Regulator of an Inflammatory Network in Dnmt3a -Mutant Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37502912/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated clonal hematopoiesis (CH) occurs due to somatic mutations accrued in hematopoietic stem cells (HSCs) that confer a selective advantage in the context of aging. The mechanisms by which CH-mutant HSCs gain this advantage with aging are not comprehensively understood. Using unbiased transcriptomic approaches, we identify Oncostatin M (OSM) signaling as a candidate contributor to aging-driven Dnmt3a -mutant CH. We find that Dnmt3a -mutant HSCs from young mice do not functionally...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37501402"
                        open
                      >
                        <summary class="article-expander__title">Subclones with variants of uncertain clinical significance might contribute to ineffective hemopoiesis and leukemia predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37501402/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In conclusions, we linked both pathogenic and VUS variants in AML-related genes to clonal hematopoiesis; therefore, we proposed to consider those variants as prognostic markers in leukemia and myelodysplasia. However, further studies in larger prospective cohorts are required to validate our results.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37506341"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of DDX41 gene variants in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37506341/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline variants in the DDX41 gene have been linked to myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) development. However, the risks associated with different variants remain unknown, as do the basis of their leukemogenic properties, impact on steady-state hematopoiesis and links to other cancers. Here, we investigate the frequency and significance of DDX41 variants in 454,792 United Kingdom Biobank (UKB) participants and identify 452 unique non-synonymous DNA variants in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37505860"
                        open
                      >
                        <summary class="article-expander__title">Detection of Age-Related Somatic Alterations in Canine Blood Using Next-Generation Sequencing-Based Liquid Biopsy: An Analysis of over 4800 Dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37505860/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-related somatic genomic alterations in hematopoietic cell lines have been well characterized in humans; however, this phenomenon has not been well studied in other species. Next-generation sequencing-based liquid biopsy testing for cancer detection was recently developed for dogs and has been used to study the genomic profiles of blood samples from thousands of canine patients since 2021. In this study, 4870 client-owned dogs with and without a diagnosis or suspicion of cancer underwent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37504319"
                        open
                      >
                        <summary class="article-expander__title">Management of Patients with Lower-Risk Myelodysplastic Neoplasms (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37504319/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms (MDS) are a heterogenous group of clonal hematologic disorders characterized by morphologic dysplasia, ineffective hematopoiesis, and cytopenia. In the past year, the classification of MDS has been updated in the 5th edition of the World Health Organization (WHO) Classification of Haematolymphoid Tumours and the International Consensus Classification (ICC) of Myeloid Neoplasms and Acute Leukemia with incorporation of morphologic, clinical, and genomic data. Furthermore,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37503259"
                        open
                      >
                        <summary class="article-expander__title">Unlocking the Role of Endothelial MPL Receptor and JAK2V617F Mutation: Insights into Cardiovascular Dysfunction in MPNs and CHIP</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37503259/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with JAK2V617F-positive myeloproliferative neoplasms (MPNs) and clonal hematopoiesis of indeterminate potential (CHIP) are at a significantly higher risk of cardiovascular diseases (CVDs). Endothelial cells (ECs) carrying the JAK2V617F mutation can be detected in many MPN patients. Here, we investigated the impact of endothelial JAK2V617F mutation on CVD development using both transgenic murine models and human induced pluripotent stem cell lines. Our findings revealed that JAK2V617F...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37502912"
                        open
                      >
                        <summary class="article-expander__title">Oncostatin M is a Master Regulator of an Inflammatory Network in Dnmt3a -Mutant Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37502912/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated clonal hematopoiesis (CH) occurs due to somatic mutations accrued in hematopoietic stem cells (HSCs) that confer a selective advantage in the context of aging. The mechanisms by which CH-mutant HSCs gain this advantage with aging are not comprehensively understood. Using unbiased transcriptomic approaches, we identify Oncostatin M (OSM) signaling as a candidate contributor to aging-driven Dnmt3a -mutant CH. We find that Dnmt3a -mutant HSCs from young mice do not functionally...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37501402"
                        open
                      >
                        <summary class="article-expander__title">Subclones with variants of uncertain clinical significance might contribute to ineffective hemopoiesis and leukemia predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37501402/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In conclusions, we linked both pathogenic and VUS variants in AML-related genes to clonal hematopoiesis; therefore, we proposed to consider those variants as prognostic markers in leukemia and myelodysplasia. However, further studies in larger prospective cohorts are required to validate our results.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37506341"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of DDX41 gene variants in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37506341/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline variants in the DDX41 gene have been linked to myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) development. However, the risks associated with different variants remain unknown, as do the basis of their leukemogenic properties, impact on steady-state hematopoiesis and links to other cancers. Here, we investigate the frequency and significance of DDX41 variants in 454,792 United Kingdom Biobank (UKB) participants and identify 452 unique non-synonymous DNA variants in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37505860"
                        open
                      >
                        <summary class="article-expander__title">Detection of Age-Related Somatic Alterations in Canine Blood Using Next-Generation Sequencing-Based Liquid Biopsy: An Analysis of over 4800 Dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37505860/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-related somatic genomic alterations in hematopoietic cell lines have been well characterized in humans; however, this phenomenon has not been well studied in other species. Next-generation sequencing-based liquid biopsy testing for cancer detection was recently developed for dogs and has been used to study the genomic profiles of blood samples from thousands of canine patients since 2021. In this study, 4870 client-owned dogs with and without a diagnosis or suspicion of cancer underwent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37504319"
                        open
                      >
                        <summary class="article-expander__title">Management of Patients with Lower-Risk Myelodysplastic Neoplasms (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37504319/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms (MDS) are a heterogenous group of clonal hematologic disorders characterized by morphologic dysplasia, ineffective hematopoiesis, and cytopenia. In the past year, the classification of MDS has been updated in the 5th edition of the World Health Organization (WHO) Classification of Haematolymphoid Tumours and the International Consensus Classification (ICC) of Myeloid Neoplasms and Acute Leukemia with incorporation of morphologic, clinical, and genomic data. Furthermore,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37503259"
                        open
                      >
                        <summary class="article-expander__title">Unlocking the Role of Endothelial MPL Receptor and JAK2V617F Mutation: Insights into Cardiovascular Dysfunction in MPNs and CHIP</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37503259/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with JAK2V617F-positive myeloproliferative neoplasms (MPNs) and clonal hematopoiesis of indeterminate potential (CHIP) are at a significantly higher risk of cardiovascular diseases (CVDs). Endothelial cells (ECs) carrying the JAK2V617F mutation can be detected in many MPN patients. Here, we investigated the impact of endothelial JAK2V617F mutation on CVD development using both transgenic murine models and human induced pluripotent stem cell lines. Our findings revealed that JAK2V617F...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37502912"
                        open
                      >
                        <summary class="article-expander__title">Oncostatin M is a Master Regulator of an Inflammatory Network in Dnmt3a -Mutant Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37502912/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated clonal hematopoiesis (CH) occurs due to somatic mutations accrued in hematopoietic stem cells (HSCs) that confer a selective advantage in the context of aging. The mechanisms by which CH-mutant HSCs gain this advantage with aging are not comprehensively understood. Using unbiased transcriptomic approaches, we identify Oncostatin M (OSM) signaling as a candidate contributor to aging-driven Dnmt3a -mutant CH. We find that Dnmt3a -mutant HSCs from young mice do not functionally...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37501402"
                        open
                      >
                        <summary class="article-expander__title">Subclones with variants of uncertain clinical significance might contribute to ineffective hemopoiesis and leukemia predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37501402/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In conclusions, we linked both pathogenic and VUS variants in AML-related genes to clonal hematopoiesis; therefore, we proposed to consider those variants as prognostic markers in leukemia and myelodysplasia. However, further studies in larger prospective cohorts are required to validate our results.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37506341"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of DDX41 gene variants in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37506341/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline variants in the DDX41 gene have been linked to myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) development. However, the risks associated with different variants remain unknown, as do the basis of their leukemogenic properties, impact on steady-state hematopoiesis and links to other cancers. Here, we investigate the frequency and significance of DDX41 variants in 454,792 United Kingdom Biobank (UKB) participants and identify 452 unique non-synonymous DNA variants in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37505860"
                        open
                      >
                        <summary class="article-expander__title">Detection of Age-Related Somatic Alterations in Canine Blood Using Next-Generation Sequencing-Based Liquid Biopsy: An Analysis of over 4800 Dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37505860/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-related somatic genomic alterations in hematopoietic cell lines have been well characterized in humans; however, this phenomenon has not been well studied in other species. Next-generation sequencing-based liquid biopsy testing for cancer detection was recently developed for dogs and has been used to study the genomic profiles of blood samples from thousands of canine patients since 2021. In this study, 4870 client-owned dogs with and without a diagnosis or suspicion of cancer underwent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37504319"
                        open
                      >
                        <summary class="article-expander__title">Management of Patients with Lower-Risk Myelodysplastic Neoplasms (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37504319/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms (MDS) are a heterogenous group of clonal hematologic disorders characterized by morphologic dysplasia, ineffective hematopoiesis, and cytopenia. In the past year, the classification of MDS has been updated in the 5th edition of the World Health Organization (WHO) Classification of Haematolymphoid Tumours and the International Consensus Classification (ICC) of Myeloid Neoplasms and Acute Leukemia with incorporation of morphologic, clinical, and genomic data. Furthermore,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37503259"
                        open
                      >
                        <summary class="article-expander__title">Unlocking the Role of Endothelial MPL Receptor and JAK2V617F Mutation: Insights into Cardiovascular Dysfunction in MPNs and CHIP</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37503259/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with JAK2V617F-positive myeloproliferative neoplasms (MPNs) and clonal hematopoiesis of indeterminate potential (CHIP) are at a significantly higher risk of cardiovascular diseases (CVDs). Endothelial cells (ECs) carrying the JAK2V617F mutation can be detected in many MPN patients. Here, we investigated the impact of endothelial JAK2V617F mutation on CVD development using both transgenic murine models and human induced pluripotent stem cell lines. Our findings revealed that JAK2V617F...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37502912"
                        open
                      >
                        <summary class="article-expander__title">Oncostatin M is a Master Regulator of an Inflammatory Network in Dnmt3a -Mutant Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37502912/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated clonal hematopoiesis (CH) occurs due to somatic mutations accrued in hematopoietic stem cells (HSCs) that confer a selective advantage in the context of aging. The mechanisms by which CH-mutant HSCs gain this advantage with aging are not comprehensively understood. Using unbiased transcriptomic approaches, we identify Oncostatin M (OSM) signaling as a candidate contributor to aging-driven Dnmt3a -mutant CH. We find that Dnmt3a -mutant HSCs from young mice do not functionally...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37501402"
                        open
                      >
                        <summary class="article-expander__title">Subclones with variants of uncertain clinical significance might contribute to ineffective hemopoiesis and leukemia predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37501402/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In conclusions, we linked both pathogenic and VUS variants in AML-related genes to clonal hematopoiesis; therefore, we proposed to consider those variants as prognostic markers in leukemia and myelodysplasia. However, further studies in larger prospective cohorts are required to validate our results.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37506341"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of DDX41 gene variants in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37506341/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline variants in the DDX41 gene have been linked to myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) development. However, the risks associated with different variants remain unknown, as do the basis of their leukemogenic properties, impact on steady-state hematopoiesis and links to other cancers. Here, we investigate the frequency and significance of DDX41 variants in 454,792 United Kingdom Biobank (UKB) participants and identify 452 unique non-synonymous DNA variants in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37505860"
                        open
                      >
                        <summary class="article-expander__title">Detection of Age-Related Somatic Alterations in Canine Blood Using Next-Generation Sequencing-Based Liquid Biopsy: An Analysis of over 4800 Dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37505860/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-related somatic genomic alterations in hematopoietic cell lines have been well characterized in humans; however, this phenomenon has not been well studied in other species. Next-generation sequencing-based liquid biopsy testing for cancer detection was recently developed for dogs and has been used to study the genomic profiles of blood samples from thousands of canine patients since 2021. In this study, 4870 client-owned dogs with and without a diagnosis or suspicion of cancer underwent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37504319"
                        open
                      >
                        <summary class="article-expander__title">Management of Patients with Lower-Risk Myelodysplastic Neoplasms (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37504319/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms (MDS) are a heterogenous group of clonal hematologic disorders characterized by morphologic dysplasia, ineffective hematopoiesis, and cytopenia. In the past year, the classification of MDS has been updated in the 5th edition of the World Health Organization (WHO) Classification of Haematolymphoid Tumours and the International Consensus Classification (ICC) of Myeloid Neoplasms and Acute Leukemia with incorporation of morphologic, clinical, and genomic data. Furthermore,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37503259"
                        open
                      >
                        <summary class="article-expander__title">Unlocking the Role of Endothelial MPL Receptor and JAK2V617F Mutation: Insights into Cardiovascular Dysfunction in MPNs and CHIP</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37503259/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with JAK2V617F-positive myeloproliferative neoplasms (MPNs) and clonal hematopoiesis of indeterminate potential (CHIP) are at a significantly higher risk of cardiovascular diseases (CVDs). Endothelial cells (ECs) carrying the JAK2V617F mutation can be detected in many MPN patients. Here, we investigated the impact of endothelial JAK2V617F mutation on CVD development using both transgenic murine models and human induced pluripotent stem cell lines. Our findings revealed that JAK2V617F...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37502912"
                        open
                      >
                        <summary class="article-expander__title">Oncostatin M is a Master Regulator of an Inflammatory Network in Dnmt3a -Mutant Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37502912/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated clonal hematopoiesis (CH) occurs due to somatic mutations accrued in hematopoietic stem cells (HSCs) that confer a selective advantage in the context of aging. The mechanisms by which CH-mutant HSCs gain this advantage with aging are not comprehensively understood. Using unbiased transcriptomic approaches, we identify Oncostatin M (OSM) signaling as a candidate contributor to aging-driven Dnmt3a -mutant CH. We find that Dnmt3a -mutant HSCs from young mice do not functionally...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37501402"
                        open
                      >
                        <summary class="article-expander__title">Subclones with variants of uncertain clinical significance might contribute to ineffective hemopoiesis and leukemia predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37501402/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In conclusions, we linked both pathogenic and VUS variants in AML-related genes to clonal hematopoiesis; therefore, we proposed to consider those variants as prognostic markers in leukemia and myelodysplasia. However, further studies in larger prospective cohorts are required to validate our results.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37506341"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of DDX41 gene variants in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37506341/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline variants in the DDX41 gene have been linked to myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) development. However, the risks associated with different variants remain unknown, as do the basis of their leukemogenic properties, impact on steady-state hematopoiesis and links to other cancers. Here, we investigate the frequency and significance of DDX41 variants in 454,792 United Kingdom Biobank (UKB) participants and identify 452 unique non-synonymous DNA variants in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37503111"
                        open
                      >
                        <summary class="article-expander__title">Coordinated inheritance of extrachromosomal DNA species in human cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37503111/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The chromosomal theory of inheritance has dominated human genetics, including cancer genetics. Genes on the same chromosome segregate together while genes on different chromosomes assort independently, providing a fundamental tenet of Mendelian inheritance. Extrachromosomal DNA (ecDNA) is a frequent event in cancer that drives oncogene amplification, dysregulated gene expression and intratumoral heterogeneity, including through random segregation during cell division. Distinct ecDNA sequences,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37506341"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of DDX41 gene variants in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37506341/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline variants in the DDX41 gene have been linked to myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) development. However, the risks associated with different variants remain unknown, as do the basis of their leukemogenic properties, impact on steady-state hematopoiesis and links to other cancers. Here, we investigate the frequency and significance of DDX41 variants in 454,792 United Kingdom Biobank (UKB) participants and identify 452 unique non-synonymous DNA variants in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37503111"
                        open
                      >
                        <summary class="article-expander__title">Coordinated inheritance of extrachromosomal DNA species in human cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37503111/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The chromosomal theory of inheritance has dominated human genetics, including cancer genetics. Genes on the same chromosome segregate together while genes on different chromosomes assort independently, providing a fundamental tenet of Mendelian inheritance. Extrachromosomal DNA (ecDNA) is a frequent event in cancer that drives oncogene amplification, dysregulated gene expression and intratumoral heterogeneity, including through random segregation during cell division. Distinct ecDNA sequences,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37506341"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of DDX41 gene variants in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37506341/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline variants in the DDX41 gene have been linked to myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) development. However, the risks associated with different variants remain unknown, as do the basis of their leukemogenic properties, impact on steady-state hematopoiesis and links to other cancers. Here, we investigate the frequency and significance of DDX41 variants in 454,792 United Kingdom Biobank (UKB) participants and identify 452 unique non-synonymous DNA variants in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37503111"
                        open
                      >
                        <summary class="article-expander__title">Coordinated inheritance of extrachromosomal DNA species in human cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37503111/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The chromosomal theory of inheritance has dominated human genetics, including cancer genetics. Genes on the same chromosome segregate together while genes on different chromosomes assort independently, providing a fundamental tenet of Mendelian inheritance. Extrachromosomal DNA (ecDNA) is a frequent event in cancer that drives oncogene amplification, dysregulated gene expression and intratumoral heterogeneity, including through random segregation during cell division. Distinct ecDNA sequences,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37506341"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of DDX41 gene variants in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37506341/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline variants in the DDX41 gene have been linked to myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) development. However, the risks associated with different variants remain unknown, as do the basis of their leukemogenic properties, impact on steady-state hematopoiesis and links to other cancers. Here, we investigate the frequency and significance of DDX41 variants in 454,792 United Kingdom Biobank (UKB) participants and identify 452 unique non-synonymous DNA variants in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37503111"
                        open
                      >
                        <summary class="article-expander__title">Coordinated inheritance of extrachromosomal DNA species in human cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37503111/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The chromosomal theory of inheritance has dominated human genetics, including cancer genetics. Genes on the same chromosome segregate together while genes on different chromosomes assort independently, providing a fundamental tenet of Mendelian inheritance. Extrachromosomal DNA (ecDNA) is a frequent event in cancer that drives oncogene amplification, dysregulated gene expression and intratumoral heterogeneity, including through random segregation during cell division. Distinct ecDNA sequences,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37506341"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of DDX41 gene variants in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37506341/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline variants in the DDX41 gene have been linked to myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) development. However, the risks associated with different variants remain unknown, as do the basis of their leukemogenic properties, impact on steady-state hematopoiesis and links to other cancers. Here, we investigate the frequency and significance of DDX41 variants in 454,792 United Kingdom Biobank (UKB) participants and identify 452 unique non-synonymous DNA variants in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37503111"
                        open
                      >
                        <summary class="article-expander__title">Coordinated inheritance of extrachromosomal DNA species in human cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37503111/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The chromosomal theory of inheritance has dominated human genetics, including cancer genetics. Genes on the same chromosome segregate together while genes on different chromosomes assort independently, providing a fundamental tenet of Mendelian inheritance. Extrachromosomal DNA (ecDNA) is a frequent event in cancer that drives oncogene amplification, dysregulated gene expression and intratumoral heterogeneity, including through random segregation during cell division. Distinct ecDNA sequences,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37506341"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of DDX41 gene variants in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37506341/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline variants in the DDX41 gene have been linked to myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) development. However, the risks associated with different variants remain unknown, as do the basis of their leukemogenic properties, impact on steady-state hematopoiesis and links to other cancers. Here, we investigate the frequency and significance of DDX41 variants in 454,792 United Kingdom Biobank (UKB) participants and identify 452 unique non-synonymous DNA variants in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37503111"
                        open
                      >
                        <summary class="article-expander__title">Coordinated inheritance of extrachromosomal DNA species in human cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37503111/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The chromosomal theory of inheritance has dominated human genetics, including cancer genetics. Genes on the same chromosome segregate together while genes on different chromosomes assort independently, providing a fundamental tenet of Mendelian inheritance. Extrachromosomal DNA (ecDNA) is a frequent event in cancer that drives oncogene amplification, dysregulated gene expression and intratumoral heterogeneity, including through random segregation during cell division. Distinct ecDNA sequences,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37506341"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of DDX41 gene variants in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37506341/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline variants in the DDX41 gene have been linked to myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) development. However, the risks associated with different variants remain unknown, as do the basis of their leukemogenic properties, impact on steady-state hematopoiesis and links to other cancers. Here, we investigate the frequency and significance of DDX41 variants in 454,792 United Kingdom Biobank (UKB) participants and identify 452 unique non-synonymous DNA variants in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37503111"
                        open
                      >
                        <summary class="article-expander__title">Coordinated inheritance of extrachromosomal DNA species in human cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37503111/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The chromosomal theory of inheritance has dominated human genetics, including cancer genetics. Genes on the same chromosome segregate together while genes on different chromosomes assort independently, providing a fundamental tenet of Mendelian inheritance. Extrachromosomal DNA (ecDNA) is a frequent event in cancer that drives oncogene amplification, dysregulated gene expression and intratumoral heterogeneity, including through random segregation during cell division. Distinct ecDNA sequences,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37506341"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of DDX41 gene variants in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37506341/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline variants in the DDX41 gene have been linked to myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) development. However, the risks associated with different variants remain unknown, as do the basis of their leukemogenic properties, impact on steady-state hematopoiesis and links to other cancers. Here, we investigate the frequency and significance of DDX41 variants in 454,792 United Kingdom Biobank (UKB) participants and identify 452 unique non-synonymous DNA variants in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37503111"
                        open
                      >
                        <summary class="article-expander__title">Coordinated inheritance of extrachromosomal DNA species in human cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37503111/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The chromosomal theory of inheritance has dominated human genetics, including cancer genetics. Genes on the same chromosome segregate together while genes on different chromosomes assort independently, providing a fundamental tenet of Mendelian inheritance. Extrachromosomal DNA (ecDNA) is a frequent event in cancer that drives oncogene amplification, dysregulated gene expression and intratumoral heterogeneity, including through random segregation during cell division. Distinct ecDNA sequences,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37506341"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of DDX41 gene variants in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37506341/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline variants in the DDX41 gene have been linked to myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) development. However, the risks associated with different variants remain unknown, as do the basis of their leukemogenic properties, impact on steady-state hematopoiesis and links to other cancers. Here, we investigate the frequency and significance of DDX41 variants in 454,792 United Kingdom Biobank (UKB) participants and identify 452 unique non-synonymous DNA variants in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37503111"
                        open
                      >
                        <summary class="article-expander__title">Coordinated inheritance of extrachromosomal DNA species in human cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37503111/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The chromosomal theory of inheritance has dominated human genetics, including cancer genetics. Genes on the same chromosome segregate together while genes on different chromosomes assort independently, providing a fundamental tenet of Mendelian inheritance. Extrachromosomal DNA (ecDNA) is a frequent event in cancer that drives oncogene amplification, dysregulated gene expression and intratumoral heterogeneity, including through random segregation during cell division. Distinct ecDNA sequences,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37506341"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of DDX41 gene variants in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37506341/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline variants in the DDX41 gene have been linked to myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) development. However, the risks associated with different variants remain unknown, as do the basis of their leukemogenic properties, impact on steady-state hematopoiesis and links to other cancers. Here, we investigate the frequency and significance of DDX41 variants in 454,792 United Kingdom Biobank (UKB) participants and identify 452 unique non-synonymous DNA variants in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37503111"
                        open
                      >
                        <summary class="article-expander__title">Coordinated inheritance of extrachromosomal DNA species in human cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37503111/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The chromosomal theory of inheritance has dominated human genetics, including cancer genetics. Genes on the same chromosome segregate together while genes on different chromosomes assort independently, providing a fundamental tenet of Mendelian inheritance. Extrachromosomal DNA (ecDNA) is a frequent event in cancer that drives oncogene amplification, dysregulated gene expression and intratumoral heterogeneity, including through random segregation during cell division. Distinct ecDNA sequences,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37506341"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of DDX41 gene variants in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37506341/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline variants in the DDX41 gene have been linked to myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) development. However, the risks associated with different variants remain unknown, as do the basis of their leukemogenic properties, impact on steady-state hematopoiesis and links to other cancers. Here, we investigate the frequency and significance of DDX41 variants in 454,792 United Kingdom Biobank (UKB) participants and identify 452 unique non-synonymous DNA variants in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37503111"
                        open
                      >
                        <summary class="article-expander__title">Coordinated inheritance of extrachromosomal DNA species in human cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37503111/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The chromosomal theory of inheritance has dominated human genetics, including cancer genetics. Genes on the same chromosome segregate together while genes on different chromosomes assort independently, providing a fundamental tenet of Mendelian inheritance. Extrachromosomal DNA (ecDNA) is a frequent event in cancer that drives oncogene amplification, dysregulated gene expression and intratumoral heterogeneity, including through random segregation during cell division. Distinct ecDNA sequences,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37506341"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of DDX41 gene variants in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37506341/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline variants in the DDX41 gene have been linked to myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) development. However, the risks associated with different variants remain unknown, as do the basis of their leukemogenic properties, impact on steady-state hematopoiesis and links to other cancers. Here, we investigate the frequency and significance of DDX41 variants in 454,792 United Kingdom Biobank (UKB) participants and identify 452 unique non-synonymous DNA variants in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37503111"
                        open
                      >
                        <summary class="article-expander__title">Coordinated inheritance of extrachromosomal DNA species in human cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37503111/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The chromosomal theory of inheritance has dominated human genetics, including cancer genetics. Genes on the same chromosome segregate together while genes on different chromosomes assort independently, providing a fundamental tenet of Mendelian inheritance. Extrachromosomal DNA (ecDNA) is a frequent event in cancer that drives oncogene amplification, dysregulated gene expression and intratumoral heterogeneity, including through random segregation during cell division. Distinct ecDNA sequences,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37506341"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of DDX41 gene variants in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37506341/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline variants in the DDX41 gene have been linked to myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) development. However, the risks associated with different variants remain unknown, as do the basis of their leukemogenic properties, impact on steady-state hematopoiesis and links to other cancers. Here, we investigate the frequency and significance of DDX41 variants in 454,792 United Kingdom Biobank (UKB) participants and identify 452 unique non-synonymous DNA variants in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37503111"
                        open
                      >
                        <summary class="article-expander__title">Coordinated inheritance of extrachromosomal DNA species in human cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37503111/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The chromosomal theory of inheritance has dominated human genetics, including cancer genetics. Genes on the same chromosome segregate together while genes on different chromosomes assort independently, providing a fundamental tenet of Mendelian inheritance. Extrachromosomal DNA (ecDNA) is a frequent event in cancer that drives oncogene amplification, dysregulated gene expression and intratumoral heterogeneity, including through random segregation during cell division. Distinct ecDNA sequences,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37506341"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of DDX41 gene variants in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37506341/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline variants in the DDX41 gene have been linked to myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) development. However, the risks associated with different variants remain unknown, as do the basis of their leukemogenic properties, impact on steady-state hematopoiesis and links to other cancers. Here, we investigate the frequency and significance of DDX41 variants in 454,792 United Kingdom Biobank (UKB) participants and identify 452 unique non-synonymous DNA variants in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37503111"
                        open
                      >
                        <summary class="article-expander__title">Coordinated inheritance of extrachromosomal DNA species in human cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37503111/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The chromosomal theory of inheritance has dominated human genetics, including cancer genetics. Genes on the same chromosome segregate together while genes on different chromosomes assort independently, providing a fundamental tenet of Mendelian inheritance. Extrachromosomal DNA (ecDNA) is a frequent event in cancer that drives oncogene amplification, dysregulated gene expression and intratumoral heterogeneity, including through random segregation during cell division. Distinct ecDNA sequences,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-07-27 datetime="2023-07-27T10:00:00.000Z">2023-07-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-07-27 datetime="2023-07-27T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37500795"
                        open
                      >
                        <summary class="article-expander__title">Follicular helper T-cell lymphomas: disease spectrum, relationship with clonal hematopoiesis, and mimics-a report of the 2022 EA4HP/SH lymphoma workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37500795/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular helper T-cell lymphomas (TFH lymphomas) were discussed in session V of the lymphoma workshop of the European Association for Haematopathology (EA4HP)/Society for Hematopathology (SH) 2022 meeting in Florence, Italy. The session focused on the morphologic spectrum of TFH lymphoma, including its three subtypes: angioimmunoblastic-type (AITL), follicular-type, and not otherwise specified (NOS). The submitted cases encompassed classic examples of TFH lymphoma and unusual cases such as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37500795"
                        open
                      >
                        <summary class="article-expander__title">Follicular helper T-cell lymphomas: disease spectrum, relationship with clonal hematopoiesis, and mimics-a report of the 2022 EA4HP/SH lymphoma workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37500795/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular helper T-cell lymphomas (TFH lymphomas) were discussed in session V of the lymphoma workshop of the European Association for Haematopathology (EA4HP)/Society for Hematopathology (SH) 2022 meeting in Florence, Italy. The session focused on the morphologic spectrum of TFH lymphoma, including its three subtypes: angioimmunoblastic-type (AITL), follicular-type, and not otherwise specified (NOS). The submitted cases encompassed classic examples of TFH lymphoma and unusual cases such as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37498674"
                        open
                      >
                        <summary class="article-expander__title">Genetic modification of inflammation and clonal hematopoiesis-associated cardiovascular risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37498674/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is associated with an increased risk of cardiovascular diseases (CVD), putatively via inflammasome activation. We pursued an inflammatory gene modifier scan for CHIP-associated CVD risk among 424,651 UK Biobank participants. CHIP was identified using whole exome sequencing data of blood DNA and modeled both as a composite and for common drivers (DNMT3A, TET2, ASXL1, and JAK2) separately. We developed predicted gene expression scores for 26...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37500795"
                        open
                      >
                        <summary class="article-expander__title">Follicular helper T-cell lymphomas: disease spectrum, relationship with clonal hematopoiesis, and mimics-a report of the 2022 EA4HP/SH lymphoma workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37500795/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular helper T-cell lymphomas (TFH lymphomas) were discussed in session V of the lymphoma workshop of the European Association for Haematopathology (EA4HP)/Society for Hematopathology (SH) 2022 meeting in Florence, Italy. The session focused on the morphologic spectrum of TFH lymphoma, including its three subtypes: angioimmunoblastic-type (AITL), follicular-type, and not otherwise specified (NOS). The submitted cases encompassed classic examples of TFH lymphoma and unusual cases such as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37498674"
                        open
                      >
                        <summary class="article-expander__title">Genetic modification of inflammation and clonal hematopoiesis-associated cardiovascular risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37498674/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is associated with an increased risk of cardiovascular diseases (CVD), putatively via inflammasome activation. We pursued an inflammatory gene modifier scan for CHIP-associated CVD risk among 424,651 UK Biobank participants. CHIP was identified using whole exome sequencing data of blood DNA and modeled both as a composite and for common drivers (DNMT3A, TET2, ASXL1, and JAK2) separately. We developed predicted gene expression scores for 26...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37500795"
                        open
                      >
                        <summary class="article-expander__title">Follicular helper T-cell lymphomas: disease spectrum, relationship with clonal hematopoiesis, and mimics-a report of the 2022 EA4HP/SH lymphoma workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37500795/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular helper T-cell lymphomas (TFH lymphomas) were discussed in session V of the lymphoma workshop of the European Association for Haematopathology (EA4HP)/Society for Hematopathology (SH) 2022 meeting in Florence, Italy. The session focused on the morphologic spectrum of TFH lymphoma, including its three subtypes: angioimmunoblastic-type (AITL), follicular-type, and not otherwise specified (NOS). The submitted cases encompassed classic examples of TFH lymphoma and unusual cases such as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37498674"
                        open
                      >
                        <summary class="article-expander__title">Genetic modification of inflammation and clonal hematopoiesis-associated cardiovascular risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37498674/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is associated with an increased risk of cardiovascular diseases (CVD), putatively via inflammasome activation. We pursued an inflammatory gene modifier scan for CHIP-associated CVD risk among 424,651 UK Biobank participants. CHIP was identified using whole exome sequencing data of blood DNA and modeled both as a composite and for common drivers (DNMT3A, TET2, ASXL1, and JAK2) separately. We developed predicted gene expression scores for 26...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37500795"
                        open
                      >
                        <summary class="article-expander__title">Follicular helper T-cell lymphomas: disease spectrum, relationship with clonal hematopoiesis, and mimics-a report of the 2022 EA4HP/SH lymphoma workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37500795/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular helper T-cell lymphomas (TFH lymphomas) were discussed in session V of the lymphoma workshop of the European Association for Haematopathology (EA4HP)/Society for Hematopathology (SH) 2022 meeting in Florence, Italy. The session focused on the morphologic spectrum of TFH lymphoma, including its three subtypes: angioimmunoblastic-type (AITL), follicular-type, and not otherwise specified (NOS). The submitted cases encompassed classic examples of TFH lymphoma and unusual cases such as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37498674"
                        open
                      >
                        <summary class="article-expander__title">Genetic modification of inflammation and clonal hematopoiesis-associated cardiovascular risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37498674/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is associated with an increased risk of cardiovascular diseases (CVD), putatively via inflammasome activation. We pursued an inflammatory gene modifier scan for CHIP-associated CVD risk among 424,651 UK Biobank participants. CHIP was identified using whole exome sequencing data of blood DNA and modeled both as a composite and for common drivers (DNMT3A, TET2, ASXL1, and JAK2) separately. We developed predicted gene expression scores for 26...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37500795"
                        open
                      >
                        <summary class="article-expander__title">Follicular helper T-cell lymphomas: disease spectrum, relationship with clonal hematopoiesis, and mimics-a report of the 2022 EA4HP/SH lymphoma workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37500795/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular helper T-cell lymphomas (TFH lymphomas) were discussed in session V of the lymphoma workshop of the European Association for Haematopathology (EA4HP)/Society for Hematopathology (SH) 2022 meeting in Florence, Italy. The session focused on the morphologic spectrum of TFH lymphoma, including its three subtypes: angioimmunoblastic-type (AITL), follicular-type, and not otherwise specified (NOS). The submitted cases encompassed classic examples of TFH lymphoma and unusual cases such as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37498674"
                        open
                      >
                        <summary class="article-expander__title">Genetic modification of inflammation and clonal hematopoiesis-associated cardiovascular risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37498674/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is associated with an increased risk of cardiovascular diseases (CVD), putatively via inflammasome activation. We pursued an inflammatory gene modifier scan for CHIP-associated CVD risk among 424,651 UK Biobank participants. CHIP was identified using whole exome sequencing data of blood DNA and modeled both as a composite and for common drivers (DNMT3A, TET2, ASXL1, and JAK2) separately. We developed predicted gene expression scores for 26...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37500795"
                        open
                      >
                        <summary class="article-expander__title">Follicular helper T-cell lymphomas: disease spectrum, relationship with clonal hematopoiesis, and mimics-a report of the 2022 EA4HP/SH lymphoma workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37500795/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular helper T-cell lymphomas (TFH lymphomas) were discussed in session V of the lymphoma workshop of the European Association for Haematopathology (EA4HP)/Society for Hematopathology (SH) 2022 meeting in Florence, Italy. The session focused on the morphologic spectrum of TFH lymphoma, including its three subtypes: angioimmunoblastic-type (AITL), follicular-type, and not otherwise specified (NOS). The submitted cases encompassed classic examples of TFH lymphoma and unusual cases such as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37498674"
                        open
                      >
                        <summary class="article-expander__title">Genetic modification of inflammation and clonal hematopoiesis-associated cardiovascular risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37498674/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is associated with an increased risk of cardiovascular diseases (CVD), putatively via inflammasome activation. We pursued an inflammatory gene modifier scan for CHIP-associated CVD risk among 424,651 UK Biobank participants. CHIP was identified using whole exome sequencing data of blood DNA and modeled both as a composite and for common drivers (DNMT3A, TET2, ASXL1, and JAK2) separately. We developed predicted gene expression scores for 26...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37500795"
                        open
                      >
                        <summary class="article-expander__title">Follicular helper T-cell lymphomas: disease spectrum, relationship with clonal hematopoiesis, and mimics-a report of the 2022 EA4HP/SH lymphoma workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37500795/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular helper T-cell lymphomas (TFH lymphomas) were discussed in session V of the lymphoma workshop of the European Association for Haematopathology (EA4HP)/Society for Hematopathology (SH) 2022 meeting in Florence, Italy. The session focused on the morphologic spectrum of TFH lymphoma, including its three subtypes: angioimmunoblastic-type (AITL), follicular-type, and not otherwise specified (NOS). The submitted cases encompassed classic examples of TFH lymphoma and unusual cases such as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37498674"
                        open
                      >
                        <summary class="article-expander__title">Genetic modification of inflammation and clonal hematopoiesis-associated cardiovascular risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37498674/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is associated with an increased risk of cardiovascular diseases (CVD), putatively via inflammasome activation. We pursued an inflammatory gene modifier scan for CHIP-associated CVD risk among 424,651 UK Biobank participants. CHIP was identified using whole exome sequencing data of blood DNA and modeled both as a composite and for common drivers (DNMT3A, TET2, ASXL1, and JAK2) separately. We developed predicted gene expression scores for 26...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37500795"
                        open
                      >
                        <summary class="article-expander__title">Follicular helper T-cell lymphomas: disease spectrum, relationship with clonal hematopoiesis, and mimics-a report of the 2022 EA4HP/SH lymphoma workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37500795/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular helper T-cell lymphomas (TFH lymphomas) were discussed in session V of the lymphoma workshop of the European Association for Haematopathology (EA4HP)/Society for Hematopathology (SH) 2022 meeting in Florence, Italy. The session focused on the morphologic spectrum of TFH lymphoma, including its three subtypes: angioimmunoblastic-type (AITL), follicular-type, and not otherwise specified (NOS). The submitted cases encompassed classic examples of TFH lymphoma and unusual cases such as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37498674"
                        open
                      >
                        <summary class="article-expander__title">Genetic modification of inflammation and clonal hematopoiesis-associated cardiovascular risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37498674/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is associated with an increased risk of cardiovascular diseases (CVD), putatively via inflammasome activation. We pursued an inflammatory gene modifier scan for CHIP-associated CVD risk among 424,651 UK Biobank participants. CHIP was identified using whole exome sequencing data of blood DNA and modeled both as a composite and for common drivers (DNMT3A, TET2, ASXL1, and JAK2) separately. We developed predicted gene expression scores for 26...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37500795"
                        open
                      >
                        <summary class="article-expander__title">Follicular helper T-cell lymphomas: disease spectrum, relationship with clonal hematopoiesis, and mimics-a report of the 2022 EA4HP/SH lymphoma workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37500795/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular helper T-cell lymphomas (TFH lymphomas) were discussed in session V of the lymphoma workshop of the European Association for Haematopathology (EA4HP)/Society for Hematopathology (SH) 2022 meeting in Florence, Italy. The session focused on the morphologic spectrum of TFH lymphoma, including its three subtypes: angioimmunoblastic-type (AITL), follicular-type, and not otherwise specified (NOS). The submitted cases encompassed classic examples of TFH lymphoma and unusual cases such as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37498674"
                        open
                      >
                        <summary class="article-expander__title">Genetic modification of inflammation and clonal hematopoiesis-associated cardiovascular risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37498674/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is associated with an increased risk of cardiovascular diseases (CVD), putatively via inflammasome activation. We pursued an inflammatory gene modifier scan for CHIP-associated CVD risk among 424,651 UK Biobank participants. CHIP was identified using whole exome sequencing data of blood DNA and modeled both as a composite and for common drivers (DNMT3A, TET2, ASXL1, and JAK2) separately. We developed predicted gene expression scores for 26...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37500795"
                        open
                      >
                        <summary class="article-expander__title">Follicular helper T-cell lymphomas: disease spectrum, relationship with clonal hematopoiesis, and mimics-a report of the 2022 EA4HP/SH lymphoma workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37500795/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular helper T-cell lymphomas (TFH lymphomas) were discussed in session V of the lymphoma workshop of the European Association for Haematopathology (EA4HP)/Society for Hematopathology (SH) 2022 meeting in Florence, Italy. The session focused on the morphologic spectrum of TFH lymphoma, including its three subtypes: angioimmunoblastic-type (AITL), follicular-type, and not otherwise specified (NOS). The submitted cases encompassed classic examples of TFH lymphoma and unusual cases such as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37498674"
                        open
                      >
                        <summary class="article-expander__title">Genetic modification of inflammation and clonal hematopoiesis-associated cardiovascular risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37498674/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is associated with an increased risk of cardiovascular diseases (CVD), putatively via inflammasome activation. We pursued an inflammatory gene modifier scan for CHIP-associated CVD risk among 424,651 UK Biobank participants. CHIP was identified using whole exome sequencing data of blood DNA and modeled both as a composite and for common drivers (DNMT3A, TET2, ASXL1, and JAK2) separately. We developed predicted gene expression scores for 26...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37498674"
                        open
                      >
                        <summary class="article-expander__title">Genetic modification of inflammation and clonal hematopoiesis-associated cardiovascular risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37498674/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is associated with an increased risk of cardiovascular diseases (CVD), putatively via inflammasome activation. We pursued an inflammatory gene modifier scan for CHIP-associated CVD risk among 424,651 UK Biobank participants. CHIP was identified using whole exome sequencing data of blood DNA and modeled both as a composite and for common drivers (DNMT3A, TET2, ASXL1, and JAK2) separately. We developed predicted gene expression scores for 26...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-07-26 datetime="2023-07-26T10:00:00.000Z">2023-07-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-07-26 datetime="2023-07-26T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37491951"
                        open
                      >
                        <summary class="article-expander__title">Hematopoiesis and immune reconstitution after CD19 directed chimeric antigen receptor T-cells (CAR-T): A comprehensive review on incidence, risk factors and current management</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37491951/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Impaired function of hematopoiesis after treatment with chimeric antigen T-cells (CAR-T) is a frequent finding and can interest a wide range of patients, regardless of age and underlying disease. Trilinear cytopenias, as well as hypogammaglobulinemia, B-cell aplasia, and T-cell impairment, can severely affect the infectious risk of CAR-T recipients, as well as their quality of life. In this review, we provide an overview of defects in hematopoiesis after CAR-T, starting with a summary of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37491951"
                        open
                      >
                        <summary class="article-expander__title">Hematopoiesis and immune reconstitution after CD19 directed chimeric antigen receptor T-cells (CAR-T): A comprehensive review on incidence, risk factors and current management</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37491951/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Impaired function of hematopoiesis after treatment with chimeric antigen T-cells (CAR-T) is a frequent finding and can interest a wide range of patients, regardless of age and underlying disease. Trilinear cytopenias, as well as hypogammaglobulinemia, B-cell aplasia, and T-cell impairment, can severely affect the infectious risk of CAR-T recipients, as well as their quality of life. In this review, we provide an overview of defects in hematopoiesis after CAR-T, starting with a summary of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37491951"
                        open
                      >
                        <summary class="article-expander__title">Hematopoiesis and immune reconstitution after CD19 directed chimeric antigen receptor T-cells (CAR-T): A comprehensive review on incidence, risk factors and current management</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37491951/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Impaired function of hematopoiesis after treatment with chimeric antigen T-cells (CAR-T) is a frequent finding and can interest a wide range of patients, regardless of age and underlying disease. Trilinear cytopenias, as well as hypogammaglobulinemia, B-cell aplasia, and T-cell impairment, can severely affect the infectious risk of CAR-T recipients, as well as their quality of life. In this review, we provide an overview of defects in hematopoiesis after CAR-T, starting with a summary of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37491951"
                        open
                      >
                        <summary class="article-expander__title">Hematopoiesis and immune reconstitution after CD19 directed chimeric antigen receptor T-cells (CAR-T): A comprehensive review on incidence, risk factors and current management</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37491951/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Impaired function of hematopoiesis after treatment with chimeric antigen T-cells (CAR-T) is a frequent finding and can interest a wide range of patients, regardless of age and underlying disease. Trilinear cytopenias, as well as hypogammaglobulinemia, B-cell aplasia, and T-cell impairment, can severely affect the infectious risk of CAR-T recipients, as well as their quality of life. In this review, we provide an overview of defects in hematopoiesis after CAR-T, starting with a summary of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37491951"
                        open
                      >
                        <summary class="article-expander__title">Hematopoiesis and immune reconstitution after CD19 directed chimeric antigen receptor T-cells (CAR-T): A comprehensive review on incidence, risk factors and current management</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37491951/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Impaired function of hematopoiesis after treatment with chimeric antigen T-cells (CAR-T) is a frequent finding and can interest a wide range of patients, regardless of age and underlying disease. Trilinear cytopenias, as well as hypogammaglobulinemia, B-cell aplasia, and T-cell impairment, can severely affect the infectious risk of CAR-T recipients, as well as their quality of life. In this review, we provide an overview of defects in hematopoiesis after CAR-T, starting with a summary of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37491951"
                        open
                      >
                        <summary class="article-expander__title">Hematopoiesis and immune reconstitution after CD19 directed chimeric antigen receptor T-cells (CAR-T): A comprehensive review on incidence, risk factors and current management</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37491951/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Impaired function of hematopoiesis after treatment with chimeric antigen T-cells (CAR-T) is a frequent finding and can interest a wide range of patients, regardless of age and underlying disease. Trilinear cytopenias, as well as hypogammaglobulinemia, B-cell aplasia, and T-cell impairment, can severely affect the infectious risk of CAR-T recipients, as well as their quality of life. In this review, we provide an overview of defects in hematopoiesis after CAR-T, starting with a summary of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37491951"
                        open
                      >
                        <summary class="article-expander__title">Hematopoiesis and immune reconstitution after CD19 directed chimeric antigen receptor T-cells (CAR-T): A comprehensive review on incidence, risk factors and current management</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37491951/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Impaired function of hematopoiesis after treatment with chimeric antigen T-cells (CAR-T) is a frequent finding and can interest a wide range of patients, regardless of age and underlying disease. Trilinear cytopenias, as well as hypogammaglobulinemia, B-cell aplasia, and T-cell impairment, can severely affect the infectious risk of CAR-T recipients, as well as their quality of life. In this review, we provide an overview of defects in hematopoiesis after CAR-T, starting with a summary of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37491951"
                        open
                      >
                        <summary class="article-expander__title">Hematopoiesis and immune reconstitution after CD19 directed chimeric antigen receptor T-cells (CAR-T): A comprehensive review on incidence, risk factors and current management</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37491951/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Impaired function of hematopoiesis after treatment with chimeric antigen T-cells (CAR-T) is a frequent finding and can interest a wide range of patients, regardless of age and underlying disease. Trilinear cytopenias, as well as hypogammaglobulinemia, B-cell aplasia, and T-cell impairment, can severely affect the infectious risk of CAR-T recipients, as well as their quality of life. In this review, we provide an overview of defects in hematopoiesis after CAR-T, starting with a summary of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37491951"
                        open
                      >
                        <summary class="article-expander__title">Hematopoiesis and immune reconstitution after CD19 directed chimeric antigen receptor T-cells (CAR-T): A comprehensive review on incidence, risk factors and current management</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37491951/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Impaired function of hematopoiesis after treatment with chimeric antigen T-cells (CAR-T) is a frequent finding and can interest a wide range of patients, regardless of age and underlying disease. Trilinear cytopenias, as well as hypogammaglobulinemia, B-cell aplasia, and T-cell impairment, can severely affect the infectious risk of CAR-T recipients, as well as their quality of life. In this review, we provide an overview of defects in hematopoiesis after CAR-T, starting with a summary of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37491951"
                        open
                      >
                        <summary class="article-expander__title">Hematopoiesis and immune reconstitution after CD19 directed chimeric antigen receptor T-cells (CAR-T): A comprehensive review on incidence, risk factors and current management</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37491951/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Impaired function of hematopoiesis after treatment with chimeric antigen T-cells (CAR-T) is a frequent finding and can interest a wide range of patients, regardless of age and underlying disease. Trilinear cytopenias, as well as hypogammaglobulinemia, B-cell aplasia, and T-cell impairment, can severely affect the infectious risk of CAR-T recipients, as well as their quality of life. In this review, we provide an overview of defects in hematopoiesis after CAR-T, starting with a summary of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37491951"
                        open
                      >
                        <summary class="article-expander__title">Hematopoiesis and immune reconstitution after CD19 directed chimeric antigen receptor T-cells (CAR-T): A comprehensive review on incidence, risk factors and current management</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37491951/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Impaired function of hematopoiesis after treatment with chimeric antigen T-cells (CAR-T) is a frequent finding and can interest a wide range of patients, regardless of age and underlying disease. Trilinear cytopenias, as well as hypogammaglobulinemia, B-cell aplasia, and T-cell impairment, can severely affect the infectious risk of CAR-T recipients, as well as their quality of life. In this review, we provide an overview of defects in hematopoiesis after CAR-T, starting with a summary of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37491951"
                        open
                      >
                        <summary class="article-expander__title">Hematopoiesis and immune reconstitution after CD19 directed chimeric antigen receptor T-cells (CAR-T): A comprehensive review on incidence, risk factors and current management</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37491951/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230726205537&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Impaired function of hematopoiesis after treatment with chimeric antigen T-cells (CAR-T) is a frequent finding and can interest a wide range of patients, regardless of age and underlying disease. Trilinear cytopenias, as well as hypogammaglobulinemia, B-cell aplasia, and T-cell impairment, can severely affect the infectious risk of CAR-T recipients, as well as their quality of life. In this review, we provide an overview of defects in hematopoiesis after CAR-T, starting with a summary of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-07-25 datetime="2023-07-25T10:00:00.000Z">2023-07-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-07-25 datetime="2023-07-25T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37489738"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential From a Heart Failure Specialist&#x27;s Point of View</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37489738/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a common bone marrow abnormality induced by age-related DNA mutations, which give rise to proinflammatory immune cells. These immune cells exacerbate atherosclerotic cardiovascular disease and may induce or accelerate heart failure. The mechanisms involved are complex but point toward a central role for proinflammatory macrophages and an inflammasome-dependent immune response (IL-1 [interleukin-1] and IL-6 [interleukin-6]) in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37489738"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential From a Heart Failure Specialist&#x27;s Point of View</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37489738/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a common bone marrow abnormality induced by age-related DNA mutations, which give rise to proinflammatory immune cells. These immune cells exacerbate atherosclerotic cardiovascular disease and may induce or accelerate heart failure. The mechanisms involved are complex but point toward a central role for proinflammatory macrophages and an inflammasome-dependent immune response (IL-1 [interleukin-1] and IL-6 [interleukin-6]) in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37489738"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential From a Heart Failure Specialist&#x27;s Point of View</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37489738/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a common bone marrow abnormality induced by age-related DNA mutations, which give rise to proinflammatory immune cells. These immune cells exacerbate atherosclerotic cardiovascular disease and may induce or accelerate heart failure. The mechanisms involved are complex but point toward a central role for proinflammatory macrophages and an inflammasome-dependent immune response (IL-1 [interleukin-1] and IL-6 [interleukin-6]) in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37489738"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential From a Heart Failure Specialist&#x27;s Point of View</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37489738/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a common bone marrow abnormality induced by age-related DNA mutations, which give rise to proinflammatory immune cells. These immune cells exacerbate atherosclerotic cardiovascular disease and may induce or accelerate heart failure. The mechanisms involved are complex but point toward a central role for proinflammatory macrophages and an inflammasome-dependent immune response (IL-1 [interleukin-1] and IL-6 [interleukin-6]) in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37489738"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential From a Heart Failure Specialist&#x27;s Point of View</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37489738/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a common bone marrow abnormality induced by age-related DNA mutations, which give rise to proinflammatory immune cells. These immune cells exacerbate atherosclerotic cardiovascular disease and may induce or accelerate heart failure. The mechanisms involved are complex but point toward a central role for proinflammatory macrophages and an inflammasome-dependent immune response (IL-1 [interleukin-1] and IL-6 [interleukin-6]) in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37489738"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential From a Heart Failure Specialist&#x27;s Point of View</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37489738/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a common bone marrow abnormality induced by age-related DNA mutations, which give rise to proinflammatory immune cells. These immune cells exacerbate atherosclerotic cardiovascular disease and may induce or accelerate heart failure. The mechanisms involved are complex but point toward a central role for proinflammatory macrophages and an inflammasome-dependent immune response (IL-1 [interleukin-1] and IL-6 [interleukin-6]) in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37489738"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential From a Heart Failure Specialist&#x27;s Point of View</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37489738/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a common bone marrow abnormality induced by age-related DNA mutations, which give rise to proinflammatory immune cells. These immune cells exacerbate atherosclerotic cardiovascular disease and may induce or accelerate heart failure. The mechanisms involved are complex but point toward a central role for proinflammatory macrophages and an inflammasome-dependent immune response (IL-1 [interleukin-1] and IL-6 [interleukin-6]) in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37489738"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential From a Heart Failure Specialist&#x27;s Point of View</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37489738/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a common bone marrow abnormality induced by age-related DNA mutations, which give rise to proinflammatory immune cells. These immune cells exacerbate atherosclerotic cardiovascular disease and may induce or accelerate heart failure. The mechanisms involved are complex but point toward a central role for proinflammatory macrophages and an inflammasome-dependent immune response (IL-1 [interleukin-1] and IL-6 [interleukin-6]) in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37489738"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential From a Heart Failure Specialist&#x27;s Point of View</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37489738/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a common bone marrow abnormality induced by age-related DNA mutations, which give rise to proinflammatory immune cells. These immune cells exacerbate atherosclerotic cardiovascular disease and may induce or accelerate heart failure. The mechanisms involved are complex but point toward a central role for proinflammatory macrophages and an inflammasome-dependent immune response (IL-1 [interleukin-1] and IL-6 [interleukin-6]) in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37489738"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential From a Heart Failure Specialist&#x27;s Point of View</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37489738/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a common bone marrow abnormality induced by age-related DNA mutations, which give rise to proinflammatory immune cells. These immune cells exacerbate atherosclerotic cardiovascular disease and may induce or accelerate heart failure. The mechanisms involved are complex but point toward a central role for proinflammatory macrophages and an inflammasome-dependent immune response (IL-1 [interleukin-1] and IL-6 [interleukin-6]) in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37489738"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential From a Heart Failure Specialist&#x27;s Point of View</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37489738/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a common bone marrow abnormality induced by age-related DNA mutations, which give rise to proinflammatory immune cells. These immune cells exacerbate atherosclerotic cardiovascular disease and may induce or accelerate heart failure. The mechanisms involved are complex but point toward a central role for proinflammatory macrophages and an inflammasome-dependent immune response (IL-1 [interleukin-1] and IL-6 [interleukin-6]) in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37489738"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential From a Heart Failure Specialist&#x27;s Point of View</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37489738/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230726205537&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a common bone marrow abnormality induced by age-related DNA mutations, which give rise to proinflammatory immune cells. These immune cells exacerbate atherosclerotic cardiovascular disease and may induce or accelerate heart failure. The mechanisms involved are complex but point toward a central role for proinflammatory macrophages and an inflammasome-dependent immune response (IL-1 [interleukin-1] and IL-6 [interleukin-6]) in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37489738"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential From a Heart Failure Specialist&#x27;s Point of View</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37489738/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a common bone marrow abnormality induced by age-related DNA mutations, which give rise to proinflammatory immune cells. These immune cells exacerbate atherosclerotic cardiovascular disease and may induce or accelerate heart failure. The mechanisms involved are complex but point toward a central role for proinflammatory macrophages and an inflammasome-dependent immune response (IL-1 [interleukin-1] and IL-6 [interleukin-6]) in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-07-24 datetime="2023-07-24T10:00:00.000Z">2023-07-24</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-07-24 datetime="2023-07-24T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37483562"
                        open
                      >
                        <summary class="article-expander__title">Prediction of risk for myeloid malignancy in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37483562/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The CHRS provides simple prognostic framework for CHIP/CCUS, distinguishing a high risk minority from the majority of CHIP/CCUS which has minimal risk for progression to MN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37483562"
                        open
                      >
                        <summary class="article-expander__title">Prediction of risk for myeloid malignancy in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37483562/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The CHRS provides simple prognostic framework for CHIP/CCUS, distinguishing a high risk minority from the majority of CHIP/CCUS which has minimal risk for progression to MN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37483285"
                        open
                      >
                        <summary class="article-expander__title">Investigation of the impact of clonal hematopoiesis on severity and pathophysiology of COVID-19 in rhesus macaques</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37483285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clinical manifestations of COVID-19 vary widely, ranging from asymptomatic to severe respiratory failure with profound inflammation. Although risk factors for severe illness have been identified, definitive determinants remain elusive. Clonal hematopoiesis (CH), the expansion of hematopoietic stem and progenitor cells bearing acquired somatic mutations, is associated with advanced age and hyperinflammation. Given the similar age range and hyperinflammatory phenotype between frequent CH and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37483562"
                        open
                      >
                        <summary class="article-expander__title">Prediction of risk for myeloid malignancy in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37483562/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The CHRS provides simple prognostic framework for CHIP/CCUS, distinguishing a high risk minority from the majority of CHIP/CCUS which has minimal risk for progression to MN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37483285"
                        open
                      >
                        <summary class="article-expander__title">Investigation of the impact of clonal hematopoiesis on severity and pathophysiology of COVID-19 in rhesus macaques</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37483285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clinical manifestations of COVID-19 vary widely, ranging from asymptomatic to severe respiratory failure with profound inflammation. Although risk factors for severe illness have been identified, definitive determinants remain elusive. Clonal hematopoiesis (CH), the expansion of hematopoietic stem and progenitor cells bearing acquired somatic mutations, is associated with advanced age and hyperinflammation. Given the similar age range and hyperinflammatory phenotype between frequent CH and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37483562"
                        open
                      >
                        <summary class="article-expander__title">Prediction of risk for myeloid malignancy in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37483562/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The CHRS provides simple prognostic framework for CHIP/CCUS, distinguishing a high risk minority from the majority of CHIP/CCUS which has minimal risk for progression to MN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37483285"
                        open
                      >
                        <summary class="article-expander__title">Investigation of the impact of clonal hematopoiesis on severity and pathophysiology of COVID-19 in rhesus macaques</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37483285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clinical manifestations of COVID-19 vary widely, ranging from asymptomatic to severe respiratory failure with profound inflammation. Although risk factors for severe illness have been identified, definitive determinants remain elusive. Clonal hematopoiesis (CH), the expansion of hematopoietic stem and progenitor cells bearing acquired somatic mutations, is associated with advanced age and hyperinflammation. Given the similar age range and hyperinflammatory phenotype between frequent CH and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37483562"
                        open
                      >
                        <summary class="article-expander__title">Prediction of risk for myeloid malignancy in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37483562/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The CHRS provides simple prognostic framework for CHIP/CCUS, distinguishing a high risk minority from the majority of CHIP/CCUS which has minimal risk for progression to MN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37483285"
                        open
                      >
                        <summary class="article-expander__title">Investigation of the impact of clonal hematopoiesis on severity and pathophysiology of COVID-19 in rhesus macaques</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37483285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clinical manifestations of COVID-19 vary widely, ranging from asymptomatic to severe respiratory failure with profound inflammation. Although risk factors for severe illness have been identified, definitive determinants remain elusive. Clonal hematopoiesis (CH), the expansion of hematopoietic stem and progenitor cells bearing acquired somatic mutations, is associated with advanced age and hyperinflammation. Given the similar age range and hyperinflammatory phenotype between frequent CH and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37483562"
                        open
                      >
                        <summary class="article-expander__title">Prediction of risk for myeloid malignancy in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37483562/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The CHRS provides simple prognostic framework for CHIP/CCUS, distinguishing a high risk minority from the majority of CHIP/CCUS which has minimal risk for progression to MN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37483285"
                        open
                      >
                        <summary class="article-expander__title">Investigation of the impact of clonal hematopoiesis on severity and pathophysiology of COVID-19 in rhesus macaques</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37483285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clinical manifestations of COVID-19 vary widely, ranging from asymptomatic to severe respiratory failure with profound inflammation. Although risk factors for severe illness have been identified, definitive determinants remain elusive. Clonal hematopoiesis (CH), the expansion of hematopoietic stem and progenitor cells bearing acquired somatic mutations, is associated with advanced age and hyperinflammation. Given the similar age range and hyperinflammatory phenotype between frequent CH and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37483562"
                        open
                      >
                        <summary class="article-expander__title">Prediction of risk for myeloid malignancy in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37483562/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The CHRS provides simple prognostic framework for CHIP/CCUS, distinguishing a high risk minority from the majority of CHIP/CCUS which has minimal risk for progression to MN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37483285"
                        open
                      >
                        <summary class="article-expander__title">Investigation of the impact of clonal hematopoiesis on severity and pathophysiology of COVID-19 in rhesus macaques</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37483285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clinical manifestations of COVID-19 vary widely, ranging from asymptomatic to severe respiratory failure with profound inflammation. Although risk factors for severe illness have been identified, definitive determinants remain elusive. Clonal hematopoiesis (CH), the expansion of hematopoietic stem and progenitor cells bearing acquired somatic mutations, is associated with advanced age and hyperinflammation. Given the similar age range and hyperinflammatory phenotype between frequent CH and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37483562"
                        open
                      >
                        <summary class="article-expander__title">Prediction of risk for myeloid malignancy in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37483562/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The CHRS provides simple prognostic framework for CHIP/CCUS, distinguishing a high risk minority from the majority of CHIP/CCUS which has minimal risk for progression to MN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37483285"
                        open
                      >
                        <summary class="article-expander__title">Investigation of the impact of clonal hematopoiesis on severity and pathophysiology of COVID-19 in rhesus macaques</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37483285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clinical manifestations of COVID-19 vary widely, ranging from asymptomatic to severe respiratory failure with profound inflammation. Although risk factors for severe illness have been identified, definitive determinants remain elusive. Clonal hematopoiesis (CH), the expansion of hematopoietic stem and progenitor cells bearing acquired somatic mutations, is associated with advanced age and hyperinflammation. Given the similar age range and hyperinflammatory phenotype between frequent CH and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37483562"
                        open
                      >
                        <summary class="article-expander__title">Prediction of risk for myeloid malignancy in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37483562/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230726205537&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The CHRS provides simple prognostic framework for CHIP/CCUS, distinguishing a high risk minority from the majority of CHIP/CCUS which has minimal risk for progression to MN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37483285"
                        open
                      >
                        <summary class="article-expander__title">Investigation of the impact of clonal hematopoiesis on severity and pathophysiology of COVID-19 in rhesus macaques</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37483285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230726205537&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clinical manifestations of COVID-19 vary widely, ranging from asymptomatic to severe respiratory failure with profound inflammation. Although risk factors for severe illness have been identified, definitive determinants remain elusive. Clonal hematopoiesis (CH), the expansion of hematopoietic stem and progenitor cells bearing acquired somatic mutations, is associated with advanced age and hyperinflammation. Given the similar age range and hyperinflammatory phenotype between frequent CH and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37483562"
                        open
                      >
                        <summary class="article-expander__title">Prediction of risk for myeloid malignancy in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37483562/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The CHRS provides simple prognostic framework for CHIP/CCUS, distinguishing a high risk minority from the majority of CHIP/CCUS which has minimal risk for progression to MN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37483285"
                        open
                      >
                        <summary class="article-expander__title">Investigation of the impact of clonal hematopoiesis on severity and pathophysiology of COVID-19 in rhesus macaques</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37483285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clinical manifestations of COVID-19 vary widely, ranging from asymptomatic to severe respiratory failure with profound inflammation. Although risk factors for severe illness have been identified, definitive determinants remain elusive. Clonal hematopoiesis (CH), the expansion of hematopoietic stem and progenitor cells bearing acquired somatic mutations, is associated with advanced age and hyperinflammation. Given the similar age range and hyperinflammatory phenotype between frequent CH and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37483562"
                        open
                      >
                        <summary class="article-expander__title">Prediction of risk for myeloid malignancy in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37483562/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The CHRS provides simple prognostic framework for CHIP/CCUS, distinguishing a high risk minority from the majority of CHIP/CCUS which has minimal risk for progression to MN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37483285"
                        open
                      >
                        <summary class="article-expander__title">Investigation of the impact of clonal hematopoiesis on severity and pathophysiology of COVID-19 in rhesus macaques</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37483285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clinical manifestations of COVID-19 vary widely, ranging from asymptomatic to severe respiratory failure with profound inflammation. Although risk factors for severe illness have been identified, definitive determinants remain elusive. Clonal hematopoiesis (CH), the expansion of hematopoietic stem and progenitor cells bearing acquired somatic mutations, is associated with advanced age and hyperinflammation. Given the similar age range and hyperinflammatory phenotype between frequent CH and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-07-22 datetime="2023-07-22T10:00:00.000Z">2023-07-22</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-07-22 datetime="2023-07-22T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37481320"
                        open
                      >
                        <summary class="article-expander__title">Clinical Flow Cytometry Analysis in the Setting of Chronic Myeloid Neoplasms and Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37481320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of flow cytometry analysis in the evaluation of chronic myeloid neoplasms, such as myelodysplastic neoplasms and chronic myeloproliferative neoplasms, continues to be emphasized and explored. Recently flow cytometry analysis has been also proven to be able to distinguish persistent clonal hematopoiesis from measurable residual disease in patients with acute myeloid leukemia (AML), a finding with potential critical treatment impact in the management of patients with AML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37481320"
                        open
                      >
                        <summary class="article-expander__title">Clinical Flow Cytometry Analysis in the Setting of Chronic Myeloid Neoplasms and Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37481320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of flow cytometry analysis in the evaluation of chronic myeloid neoplasms, such as myelodysplastic neoplasms and chronic myeloproliferative neoplasms, continues to be emphasized and explored. Recently flow cytometry analysis has been also proven to be able to distinguish persistent clonal hematopoiesis from measurable residual disease in patients with acute myeloid leukemia (AML), a finding with potential critical treatment impact in the management of patients with AML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37481320"
                        open
                      >
                        <summary class="article-expander__title">Clinical Flow Cytometry Analysis in the Setting of Chronic Myeloid Neoplasms and Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37481320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of flow cytometry analysis in the evaluation of chronic myeloid neoplasms, such as myelodysplastic neoplasms and chronic myeloproliferative neoplasms, continues to be emphasized and explored. Recently flow cytometry analysis has been also proven to be able to distinguish persistent clonal hematopoiesis from measurable residual disease in patients with acute myeloid leukemia (AML), a finding with potential critical treatment impact in the management of patients with AML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37481320"
                        open
                      >
                        <summary class="article-expander__title">Clinical Flow Cytometry Analysis in the Setting of Chronic Myeloid Neoplasms and Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37481320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of flow cytometry analysis in the evaluation of chronic myeloid neoplasms, such as myelodysplastic neoplasms and chronic myeloproliferative neoplasms, continues to be emphasized and explored. Recently flow cytometry analysis has been also proven to be able to distinguish persistent clonal hematopoiesis from measurable residual disease in patients with acute myeloid leukemia (AML), a finding with potential critical treatment impact in the management of patients with AML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37481320"
                        open
                      >
                        <summary class="article-expander__title">Clinical Flow Cytometry Analysis in the Setting of Chronic Myeloid Neoplasms and Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37481320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of flow cytometry analysis in the evaluation of chronic myeloid neoplasms, such as myelodysplastic neoplasms and chronic myeloproliferative neoplasms, continues to be emphasized and explored. Recently flow cytometry analysis has been also proven to be able to distinguish persistent clonal hematopoiesis from measurable residual disease in patients with acute myeloid leukemia (AML), a finding with potential critical treatment impact in the management of patients with AML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37481320"
                        open
                      >
                        <summary class="article-expander__title">Clinical Flow Cytometry Analysis in the Setting of Chronic Myeloid Neoplasms and Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37481320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230726205537&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of flow cytometry analysis in the evaluation of chronic myeloid neoplasms, such as myelodysplastic neoplasms and chronic myeloproliferative neoplasms, continues to be emphasized and explored. Recently flow cytometry analysis has been also proven to be able to distinguish persistent clonal hematopoiesis from measurable residual disease in patients with acute myeloid leukemia (AML), a finding with potential critical treatment impact in the management of patients with AML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37481320"
                        open
                      >
                        <summary class="article-expander__title">Clinical Flow Cytometry Analysis in the Setting of Chronic Myeloid Neoplasms and Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37481320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of flow cytometry analysis in the evaluation of chronic myeloid neoplasms, such as myelodysplastic neoplasms and chronic myeloproliferative neoplasms, continues to be emphasized and explored. Recently flow cytometry analysis has been also proven to be able to distinguish persistent clonal hematopoiesis from measurable residual disease in patients with acute myeloid leukemia (AML), a finding with potential critical treatment impact in the management of patients with AML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37481320"
                        open
                      >
                        <summary class="article-expander__title">Clinical Flow Cytometry Analysis in the Setting of Chronic Myeloid Neoplasms and Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37481320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of flow cytometry analysis in the evaluation of chronic myeloid neoplasms, such as myelodysplastic neoplasms and chronic myeloproliferative neoplasms, continues to be emphasized and explored. Recently flow cytometry analysis has been also proven to be able to distinguish persistent clonal hematopoiesis from measurable residual disease in patients with acute myeloid leukemia (AML), a finding with potential critical treatment impact in the management of patients with AML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37481320"
                        open
                      >
                        <summary class="article-expander__title">Clinical Flow Cytometry Analysis in the Setting of Chronic Myeloid Neoplasms and Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37481320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230723210314&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of flow cytometry analysis in the evaluation of chronic myeloid neoplasms, such as myelodysplastic neoplasms and chronic myeloproliferative neoplasms, continues to be emphasized and explored. Recently flow cytometry analysis has been also proven to be able to distinguish persistent clonal hematopoiesis from measurable residual disease in patients with acute myeloid leukemia (AML), a finding with potential critical treatment impact in the management of patients with AML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37481320"
                        open
                      >
                        <summary class="article-expander__title">Clinical Flow Cytometry Analysis in the Setting of Chronic Myeloid Neoplasms and Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37481320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of flow cytometry analysis in the evaluation of chronic myeloid neoplasms, such as myelodysplastic neoplasms and chronic myeloproliferative neoplasms, continues to be emphasized and explored. Recently flow cytometry analysis has been also proven to be able to distinguish persistent clonal hematopoiesis from measurable residual disease in patients with acute myeloid leukemia (AML), a finding with potential critical treatment impact in the management of patients with AML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37481320"
                        open
                      >
                        <summary class="article-expander__title">Clinical Flow Cytometry Analysis in the Setting of Chronic Myeloid Neoplasms and Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37481320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of flow cytometry analysis in the evaluation of chronic myeloid neoplasms, such as myelodysplastic neoplasms and chronic myeloproliferative neoplasms, continues to be emphasized and explored. Recently flow cytometry analysis has been also proven to be able to distinguish persistent clonal hematopoiesis from measurable residual disease in patients with acute myeloid leukemia (AML), a finding with potential critical treatment impact in the management of patients with AML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37481320"
                        open
                      >
                        <summary class="article-expander__title">Clinical Flow Cytometry Analysis in the Setting of Chronic Myeloid Neoplasms and Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37481320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of flow cytometry analysis in the evaluation of chronic myeloid neoplasms, such as myelodysplastic neoplasms and chronic myeloproliferative neoplasms, continues to be emphasized and explored. Recently flow cytometry analysis has been also proven to be able to distinguish persistent clonal hematopoiesis from measurable residual disease in patients with acute myeloid leukemia (AML), a finding with potential critical treatment impact in the management of patients with AML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37481320"
                        open
                      >
                        <summary class="article-expander__title">Clinical Flow Cytometry Analysis in the Setting of Chronic Myeloid Neoplasms and Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37481320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of flow cytometry analysis in the evaluation of chronic myeloid neoplasms, such as myelodysplastic neoplasms and chronic myeloproliferative neoplasms, continues to be emphasized and explored. Recently flow cytometry analysis has been also proven to be able to distinguish persistent clonal hematopoiesis from measurable residual disease in patients with acute myeloid leukemia (AML), a finding with potential critical treatment impact in the management of patients with AML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37481320"
                        open
                      >
                        <summary class="article-expander__title">Clinical Flow Cytometry Analysis in the Setting of Chronic Myeloid Neoplasms and Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37481320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of flow cytometry analysis in the evaluation of chronic myeloid neoplasms, such as myelodysplastic neoplasms and chronic myeloproliferative neoplasms, continues to be emphasized and explored. Recently flow cytometry analysis has been also proven to be able to distinguish persistent clonal hematopoiesis from measurable residual disease in patients with acute myeloid leukemia (AML), a finding with potential critical treatment impact in the management of patients with AML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37481320"
                        open
                      >
                        <summary class="article-expander__title">Clinical Flow Cytometry Analysis in the Setting of Chronic Myeloid Neoplasms and Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37481320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of flow cytometry analysis in the evaluation of chronic myeloid neoplasms, such as myelodysplastic neoplasms and chronic myeloproliferative neoplasms, continues to be emphasized and explored. Recently flow cytometry analysis has been also proven to be able to distinguish persistent clonal hematopoiesis from measurable residual disease in patients with acute myeloid leukemia (AML), a finding with potential critical treatment impact in the management of patients with AML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37481320"
                        open
                      >
                        <summary class="article-expander__title">Clinical Flow Cytometry Analysis in the Setting of Chronic Myeloid Neoplasms and Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37481320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of flow cytometry analysis in the evaluation of chronic myeloid neoplasms, such as myelodysplastic neoplasms and chronic myeloproliferative neoplasms, continues to be emphasized and explored. Recently flow cytometry analysis has been also proven to be able to distinguish persistent clonal hematopoiesis from measurable residual disease in patients with acute myeloid leukemia (AML), a finding with potential critical treatment impact in the management of patients with AML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37481320"
                        open
                      >
                        <summary class="article-expander__title">Clinical Flow Cytometry Analysis in the Setting of Chronic Myeloid Neoplasms and Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37481320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of flow cytometry analysis in the evaluation of chronic myeloid neoplasms, such as myelodysplastic neoplasms and chronic myeloproliferative neoplasms, continues to be emphasized and explored. Recently flow cytometry analysis has been also proven to be able to distinguish persistent clonal hematopoiesis from measurable residual disease in patients with acute myeloid leukemia (AML), a finding with potential critical treatment impact in the management of patients with AML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37481320"
                        open
                      >
                        <summary class="article-expander__title">Clinical Flow Cytometry Analysis in the Setting of Chronic Myeloid Neoplasms and Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37481320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of flow cytometry analysis in the evaluation of chronic myeloid neoplasms, such as myelodysplastic neoplasms and chronic myeloproliferative neoplasms, continues to be emphasized and explored. Recently flow cytometry analysis has been also proven to be able to distinguish persistent clonal hematopoiesis from measurable residual disease in patients with acute myeloid leukemia (AML), a finding with potential critical treatment impact in the management of patients with AML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37481320"
                        open
                      >
                        <summary class="article-expander__title">Clinical Flow Cytometry Analysis in the Setting of Chronic Myeloid Neoplasms and Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37481320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of flow cytometry analysis in the evaluation of chronic myeloid neoplasms, such as myelodysplastic neoplasms and chronic myeloproliferative neoplasms, continues to be emphasized and explored. Recently flow cytometry analysis has been also proven to be able to distinguish persistent clonal hematopoiesis from measurable residual disease in patients with acute myeloid leukemia (AML), a finding with potential critical treatment impact in the management of patients with AML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37481320"
                        open
                      >
                        <summary class="article-expander__title">Clinical Flow Cytometry Analysis in the Setting of Chronic Myeloid Neoplasms and Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37481320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of flow cytometry analysis in the evaluation of chronic myeloid neoplasms, such as myelodysplastic neoplasms and chronic myeloproliferative neoplasms, continues to be emphasized and explored. Recently flow cytometry analysis has been also proven to be able to distinguish persistent clonal hematopoiesis from measurable residual disease in patients with acute myeloid leukemia (AML), a finding with potential critical treatment impact in the management of patients with AML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37481320"
                        open
                      >
                        <summary class="article-expander__title">Clinical Flow Cytometry Analysis in the Setting of Chronic Myeloid Neoplasms and Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37481320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of flow cytometry analysis in the evaluation of chronic myeloid neoplasms, such as myelodysplastic neoplasms and chronic myeloproliferative neoplasms, continues to be emphasized and explored. Recently flow cytometry analysis has been also proven to be able to distinguish persistent clonal hematopoiesis from measurable residual disease in patients with acute myeloid leukemia (AML), a finding with potential critical treatment impact in the management of patients with AML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37481320"
                        open
                      >
                        <summary class="article-expander__title">Clinical Flow Cytometry Analysis in the Setting of Chronic Myeloid Neoplasms and Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37481320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of flow cytometry analysis in the evaluation of chronic myeloid neoplasms, such as myelodysplastic neoplasms and chronic myeloproliferative neoplasms, continues to be emphasized and explored. Recently flow cytometry analysis has been also proven to be able to distinguish persistent clonal hematopoiesis from measurable residual disease in patients with acute myeloid leukemia (AML), a finding with potential critical treatment impact in the management of patients with AML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37481320"
                        open
                      >
                        <summary class="article-expander__title">Clinical Flow Cytometry Analysis in the Setting of Chronic Myeloid Neoplasms and Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37481320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of flow cytometry analysis in the evaluation of chronic myeloid neoplasms, such as myelodysplastic neoplasms and chronic myeloproliferative neoplasms, continues to be emphasized and explored. Recently flow cytometry analysis has been also proven to be able to distinguish persistent clonal hematopoiesis from measurable residual disease in patients with acute myeloid leukemia (AML), a finding with potential critical treatment impact in the management of patients with AML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37481320"
                        open
                      >
                        <summary class="article-expander__title">Clinical Flow Cytometry Analysis in the Setting of Chronic Myeloid Neoplasms and Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37481320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of flow cytometry analysis in the evaluation of chronic myeloid neoplasms, such as myelodysplastic neoplasms and chronic myeloproliferative neoplasms, continues to be emphasized and explored. Recently flow cytometry analysis has been also proven to be able to distinguish persistent clonal hematopoiesis from measurable residual disease in patients with acute myeloid leukemia (AML), a finding with potential critical treatment impact in the management of patients with AML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37481320"
                        open
                      >
                        <summary class="article-expander__title">Clinical Flow Cytometry Analysis in the Setting of Chronic Myeloid Neoplasms and Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37481320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230726205538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of flow cytometry analysis in the evaluation of chronic myeloid neoplasms, such as myelodysplastic neoplasms and chronic myeloproliferative neoplasms, continues to be emphasized and explored. Recently flow cytometry analysis has been also proven to be able to distinguish persistent clonal hematopoiesis from measurable residual disease in patients with acute myeloid leukemia (AML), a finding with potential critical treatment impact in the management of patients with AML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37481320"
                        open
                      >
                        <summary class="article-expander__title">Clinical Flow Cytometry Analysis in the Setting of Chronic Myeloid Neoplasms and Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37481320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of flow cytometry analysis in the evaluation of chronic myeloid neoplasms, such as myelodysplastic neoplasms and chronic myeloproliferative neoplasms, continues to be emphasized and explored. Recently flow cytometry analysis has been also proven to be able to distinguish persistent clonal hematopoiesis from measurable residual disease in patients with acute myeloid leukemia (AML), a finding with potential critical treatment impact in the management of patients with AML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37481320"
                        open
                      >
                        <summary class="article-expander__title">Clinical Flow Cytometry Analysis in the Setting of Chronic Myeloid Neoplasms and Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37481320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of flow cytometry analysis in the evaluation of chronic myeloid neoplasms, such as myelodysplastic neoplasms and chronic myeloproliferative neoplasms, continues to be emphasized and explored. Recently flow cytometry analysis has been also proven to be able to distinguish persistent clonal hematopoiesis from measurable residual disease in patients with acute myeloid leukemia (AML), a finding with potential critical treatment impact in the management of patients with AML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-07-21 datetime="2023-07-21T10:00:00.000Z">2023-07-21</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-07-21 datetime="2023-07-21T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37478398"
                        open
                      >
                        <summary class="article-expander__title">Single cell genomics in acquired bone marrow failure syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37478398/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mechanistic studies of immune bone marrow failure are difficult due to the scarcity of residual cells, the involvement of multiple cell types, and the inherent complexities of hematopoiesis and immunity. Single cell genomic technologies and bioinformatic analyses allow extensive, multi-dimensional analysis of very limited number of cells. We review the emerging applications of single cell techniques, and early results related to disease pathogenesis: effector and target cell populations and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37476029"
                        open
                      >
                        <summary class="article-expander__title">Malignancy and viral infections in Sub-Saharan Africa: A review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37476029/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The burden of malignancy related to viral infection is increasing in Sub-Saharan Africa (SSA). In 2018, approximately 2 million new cancer cases worldwide were attributable to infection. Prevention or treatment of these infections could reduce cancer cases by 23% in less developed regions and about 7% in developed regions. Contemporaneous increases in longevity and changes in lifestyle have contributed to the cancer burden in SSA. African hospitals are reporting more cases of cancer related to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37478398"
                        open
                      >
                        <summary class="article-expander__title">Single cell genomics in acquired bone marrow failure syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37478398/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mechanistic studies of immune bone marrow failure are difficult due to the scarcity of residual cells, the involvement of multiple cell types, and the inherent complexities of hematopoiesis and immunity. Single cell genomic technologies and bioinformatic analyses allow extensive, multi-dimensional analysis of very limited number of cells. We review the emerging applications of single cell techniques, and early results related to disease pathogenesis: effector and target cell populations and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37476029"
                        open
                      >
                        <summary class="article-expander__title">Malignancy and viral infections in Sub-Saharan Africa: A review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37476029/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The burden of malignancy related to viral infection is increasing in Sub-Saharan Africa (SSA). In 2018, approximately 2 million new cancer cases worldwide were attributable to infection. Prevention or treatment of these infections could reduce cancer cases by 23% in less developed regions and about 7% in developed regions. Contemporaneous increases in longevity and changes in lifestyle have contributed to the cancer burden in SSA. African hospitals are reporting more cases of cancer related to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37478398"
                        open
                      >
                        <summary class="article-expander__title">Single cell genomics in acquired bone marrow failure syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37478398/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mechanistic studies of immune bone marrow failure are difficult due to the scarcity of residual cells, the involvement of multiple cell types, and the inherent complexities of hematopoiesis and immunity. Single cell genomic technologies and bioinformatic analyses allow extensive, multi-dimensional analysis of very limited number of cells. We review the emerging applications of single cell techniques, and early results related to disease pathogenesis: effector and target cell populations and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37476029"
                        open
                      >
                        <summary class="article-expander__title">Malignancy and viral infections in Sub-Saharan Africa: A review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37476029/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The burden of malignancy related to viral infection is increasing in Sub-Saharan Africa (SSA). In 2018, approximately 2 million new cancer cases worldwide were attributable to infection. Prevention or treatment of these infections could reduce cancer cases by 23% in less developed regions and about 7% in developed regions. Contemporaneous increases in longevity and changes in lifestyle have contributed to the cancer burden in SSA. African hospitals are reporting more cases of cancer related to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37478398"
                        open
                      >
                        <summary class="article-expander__title">Single cell genomics in acquired bone marrow failure syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37478398/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mechanistic studies of immune bone marrow failure are difficult due to the scarcity of residual cells, the involvement of multiple cell types, and the inherent complexities of hematopoiesis and immunity. Single cell genomic technologies and bioinformatic analyses allow extensive, multi-dimensional analysis of very limited number of cells. We review the emerging applications of single cell techniques, and early results related to disease pathogenesis: effector and target cell populations and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37476029"
                        open
                      >
                        <summary class="article-expander__title">Malignancy and viral infections in Sub-Saharan Africa: A review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37476029/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The burden of malignancy related to viral infection is increasing in Sub-Saharan Africa (SSA). In 2018, approximately 2 million new cancer cases worldwide were attributable to infection. Prevention or treatment of these infections could reduce cancer cases by 23% in less developed regions and about 7% in developed regions. Contemporaneous increases in longevity and changes in lifestyle have contributed to the cancer burden in SSA. African hospitals are reporting more cases of cancer related to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37478398"
                        open
                      >
                        <summary class="article-expander__title">Single cell genomics in acquired bone marrow failure syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37478398/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230726205537&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mechanistic studies of immune bone marrow failure are difficult due to the scarcity of residual cells, the involvement of multiple cell types, and the inherent complexities of hematopoiesis and immunity. Single cell genomic technologies and bioinformatic analyses allow extensive, multi-dimensional analysis of very limited number of cells. We review the emerging applications of single cell techniques, and early results related to disease pathogenesis: effector and target cell populations and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37476029"
                        open
                      >
                        <summary class="article-expander__title">Malignancy and viral infections in Sub-Saharan Africa: A review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37476029/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230726205537&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The burden of malignancy related to viral infection is increasing in Sub-Saharan Africa (SSA). In 2018, approximately 2 million new cancer cases worldwide were attributable to infection. Prevention or treatment of these infections could reduce cancer cases by 23% in less developed regions and about 7% in developed regions. Contemporaneous increases in longevity and changes in lifestyle have contributed to the cancer burden in SSA. African hospitals are reporting more cases of cancer related to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37478398"
                        open
                      >
                        <summary class="article-expander__title">Single cell genomics in acquired bone marrow failure syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37478398/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mechanistic studies of immune bone marrow failure are difficult due to the scarcity of residual cells, the involvement of multiple cell types, and the inherent complexities of hematopoiesis and immunity. Single cell genomic technologies and bioinformatic analyses allow extensive, multi-dimensional analysis of very limited number of cells. We review the emerging applications of single cell techniques, and early results related to disease pathogenesis: effector and target cell populations and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37476029"
                        open
                      >
                        <summary class="article-expander__title">Malignancy and viral infections in Sub-Saharan Africa: A review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37476029/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The burden of malignancy related to viral infection is increasing in Sub-Saharan Africa (SSA). In 2018, approximately 2 million new cancer cases worldwide were attributable to infection. Prevention or treatment of these infections could reduce cancer cases by 23% in less developed regions and about 7% in developed regions. Contemporaneous increases in longevity and changes in lifestyle have contributed to the cancer burden in SSA. African hospitals are reporting more cases of cancer related to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37478398"
                        open
                      >
                        <summary class="article-expander__title">Single cell genomics in acquired bone marrow failure syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37478398/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mechanistic studies of immune bone marrow failure are difficult due to the scarcity of residual cells, the involvement of multiple cell types, and the inherent complexities of hematopoiesis and immunity. Single cell genomic technologies and bioinformatic analyses allow extensive, multi-dimensional analysis of very limited number of cells. We review the emerging applications of single cell techniques, and early results related to disease pathogenesis: effector and target cell populations and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37476029"
                        open
                      >
                        <summary class="article-expander__title">Malignancy and viral infections in Sub-Saharan Africa: A review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37476029/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The burden of malignancy related to viral infection is increasing in Sub-Saharan Africa (SSA). In 2018, approximately 2 million new cancer cases worldwide were attributable to infection. Prevention or treatment of these infections could reduce cancer cases by 23% in less developed regions and about 7% in developed regions. Contemporaneous increases in longevity and changes in lifestyle have contributed to the cancer burden in SSA. African hospitals are reporting more cases of cancer related to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37478398"
                        open
                      >
                        <summary class="article-expander__title">Single cell genomics in acquired bone marrow failure syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37478398/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230723210314&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mechanistic studies of immune bone marrow failure are difficult due to the scarcity of residual cells, the involvement of multiple cell types, and the inherent complexities of hematopoiesis and immunity. Single cell genomic technologies and bioinformatic analyses allow extensive, multi-dimensional analysis of very limited number of cells. We review the emerging applications of single cell techniques, and early results related to disease pathogenesis: effector and target cell populations and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37476029"
                        open
                      >
                        <summary class="article-expander__title">Malignancy and viral infections in Sub-Saharan Africa: A review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37476029/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230723210314&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The burden of malignancy related to viral infection is increasing in Sub-Saharan Africa (SSA). In 2018, approximately 2 million new cancer cases worldwide were attributable to infection. Prevention or treatment of these infections could reduce cancer cases by 23% in less developed regions and about 7% in developed regions. Contemporaneous increases in longevity and changes in lifestyle have contributed to the cancer burden in SSA. African hospitals are reporting more cases of cancer related to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37478398"
                        open
                      >
                        <summary class="article-expander__title">Single cell genomics in acquired bone marrow failure syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37478398/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mechanistic studies of immune bone marrow failure are difficult due to the scarcity of residual cells, the involvement of multiple cell types, and the inherent complexities of hematopoiesis and immunity. Single cell genomic technologies and bioinformatic analyses allow extensive, multi-dimensional analysis of very limited number of cells. We review the emerging applications of single cell techniques, and early results related to disease pathogenesis: effector and target cell populations and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37476029"
                        open
                      >
                        <summary class="article-expander__title">Malignancy and viral infections in Sub-Saharan Africa: A review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37476029/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The burden of malignancy related to viral infection is increasing in Sub-Saharan Africa (SSA). In 2018, approximately 2 million new cancer cases worldwide were attributable to infection. Prevention or treatment of these infections could reduce cancer cases by 23% in less developed regions and about 7% in developed regions. Contemporaneous increases in longevity and changes in lifestyle have contributed to the cancer burden in SSA. African hospitals are reporting more cases of cancer related to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37478398"
                        open
                      >
                        <summary class="article-expander__title">Single cell genomics in acquired bone marrow failure syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37478398/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mechanistic studies of immune bone marrow failure are difficult due to the scarcity of residual cells, the involvement of multiple cell types, and the inherent complexities of hematopoiesis and immunity. Single cell genomic technologies and bioinformatic analyses allow extensive, multi-dimensional analysis of very limited number of cells. We review the emerging applications of single cell techniques, and early results related to disease pathogenesis: effector and target cell populations and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37476029"
                        open
                      >
                        <summary class="article-expander__title">Malignancy and viral infections in Sub-Saharan Africa: A review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37476029/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The burden of malignancy related to viral infection is increasing in Sub-Saharan Africa (SSA). In 2018, approximately 2 million new cancer cases worldwide were attributable to infection. Prevention or treatment of these infections could reduce cancer cases by 23% in less developed regions and about 7% in developed regions. Contemporaneous increases in longevity and changes in lifestyle have contributed to the cancer burden in SSA. African hospitals are reporting more cases of cancer related to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-07-20 datetime="2023-07-20T10:00:00.000Z">2023-07-20</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-07-20 datetime="2023-07-20T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37470778"
                        open
                      >
                        <summary class="article-expander__title">A SINGLE-CELL TAXONOMY PREDICTS INFLAMMATORY NICHE REMODELING TO DRIVE TISSUE FAILURE AND OUTCOME IN HUMAN AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37470778/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer initiation is orchestrated by interplay between tumor-initiating cells and their stromal/immune environment. Here, by adapted single-cell RNA sequencing, we decipher the predicted signaling between tissue-resident hematopoietic stem/progenitor cells (HSPCs) and their neoplastic counterparts with their native niches in the human bone marrow. LEPR+ stromal cells are identified as central regulators of hematopoiesis through predicted interactions with all cells in the marrow. Inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37470778"
                        open
                      >
                        <summary class="article-expander__title">A SINGLE-CELL TAXONOMY PREDICTS INFLAMMATORY NICHE REMODELING TO DRIVE TISSUE FAILURE AND OUTCOME IN HUMAN AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37470778/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230726205537&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer initiation is orchestrated by interplay between tumor-initiating cells and their stromal/immune environment. Here, by adapted single-cell RNA sequencing, we decipher the predicted signaling between tissue-resident hematopoietic stem/progenitor cells (HSPCs) and their neoplastic counterparts with their native niches in the human bone marrow. LEPR+ stromal cells are identified as central regulators of hematopoiesis through predicted interactions with all cells in the marrow. Inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37470778"
                        open
                      >
                        <summary class="article-expander__title">A SINGLE-CELL TAXONOMY PREDICTS INFLAMMATORY NICHE REMODELING TO DRIVE TISSUE FAILURE AND OUTCOME IN HUMAN AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37470778/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer initiation is orchestrated by interplay between tumor-initiating cells and their stromal/immune environment. Here, by adapted single-cell RNA sequencing, we decipher the predicted signaling between tissue-resident hematopoietic stem/progenitor cells (HSPCs) and their neoplastic counterparts with their native niches in the human bone marrow. LEPR+ stromal cells are identified as central regulators of hematopoiesis through predicted interactions with all cells in the marrow. Inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37470778"
                        open
                      >
                        <summary class="article-expander__title">A SINGLE-CELL TAXONOMY PREDICTS INFLAMMATORY NICHE REMODELING TO DRIVE TISSUE FAILURE AND OUTCOME IN HUMAN AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37470778/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer initiation is orchestrated by interplay between tumor-initiating cells and their stromal/immune environment. Here, by adapted single-cell RNA sequencing, we decipher the predicted signaling between tissue-resident hematopoietic stem/progenitor cells (HSPCs) and their neoplastic counterparts with their native niches in the human bone marrow. LEPR+ stromal cells are identified as central regulators of hematopoiesis through predicted interactions with all cells in the marrow. Inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37470778"
                        open
                      >
                        <summary class="article-expander__title">A SINGLE-CELL TAXONOMY PREDICTS INFLAMMATORY NICHE REMODELING TO DRIVE TISSUE FAILURE AND OUTCOME IN HUMAN AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37470778/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230723210314&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer initiation is orchestrated by interplay between tumor-initiating cells and their stromal/immune environment. Here, by adapted single-cell RNA sequencing, we decipher the predicted signaling between tissue-resident hematopoietic stem/progenitor cells (HSPCs) and their neoplastic counterparts with their native niches in the human bone marrow. LEPR+ stromal cells are identified as central regulators of hematopoiesis through predicted interactions with all cells in the marrow. Inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37470778"
                        open
                      >
                        <summary class="article-expander__title">A SINGLE-CELL TAXONOMY PREDICTS INFLAMMATORY NICHE REMODELING TO DRIVE TISSUE FAILURE AND OUTCOME IN HUMAN AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37470778/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer initiation is orchestrated by interplay between tumor-initiating cells and their stromal/immune environment. Here, by adapted single-cell RNA sequencing, we decipher the predicted signaling between tissue-resident hematopoietic stem/progenitor cells (HSPCs) and their neoplastic counterparts with their native niches in the human bone marrow. LEPR+ stromal cells are identified as central regulators of hematopoiesis through predicted interactions with all cells in the marrow. Inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37470778"
                        open
                      >
                        <summary class="article-expander__title">A SINGLE-CELL TAXONOMY PREDICTS INFLAMMATORY NICHE REMODELING TO DRIVE TISSUE FAILURE AND OUTCOME IN HUMAN AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37470778/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer initiation is orchestrated by interplay between tumor-initiating cells and their stromal/immune environment. Here, by adapted single-cell RNA sequencing, we decipher the predicted signaling between tissue-resident hematopoietic stem/progenitor cells (HSPCs) and their neoplastic counterparts with their native niches in the human bone marrow. LEPR+ stromal cells are identified as central regulators of hematopoiesis through predicted interactions with all cells in the marrow. Inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37470778"
                        open
                      >
                        <summary class="article-expander__title">A SINGLE-CELL TAXONOMY PREDICTS INFLAMMATORY NICHE REMODELING TO DRIVE TISSUE FAILURE AND OUTCOME IN HUMAN AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37470778/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230720210259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer initiation is orchestrated by interplay between tumor-initiating cells and their stromal/immune environment. Here, by adapted single-cell RNA sequencing, we decipher the predicted signaling between tissue-resident hematopoietic stem/progenitor cells (HSPCs) and their neoplastic counterparts with their native niches in the human bone marrow. LEPR+ stromal cells are identified as central regulators of hematopoiesis through predicted interactions with all cells in the marrow. Inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-07-19 datetime="2023-07-19T10:00:00.000Z">2023-07-19</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-07-19 datetime="2023-07-19T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37466697"
                        open
                      >
                        <summary class="article-expander__title">Associations between ambient air pollutants and clonal hematopoiesis of indeterminate potential (CHIP)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37466697/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related somatic mutation associated with incident hematologic cancer. Environmental stressors which, like air pollution, generate oxidative stress at the cellular level, may induce somatic mutations and some mutations may provide a selection advantage for persistence and expansion of specific clones. Materials and Methods We used data from the Multi-Ethnic Study of Atherosclerosis (MESA) N&#x3D;4,379 and the Women&#x27;s Health...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37465589"
                        open
                      >
                        <summary class="article-expander__title">Gilteritinib Affects the Selection of Dominant Clones in Clonal Hematopoiesis: Sequential Genetic Analysis of an FLT3-ITD Positive AML Patient with Long-Term Gilteritinib Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37465589/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We performed sequential molecular analyses of a 75-year-old woman with de novo FLT3-ITD positive acute myeloid leukemia (AML) who had received gilteritinib therapy for 43 months. At the time of diagnosis, her karyotype was normal; however, FLT3-ITD, NPM1, DNMT3A, and IDH2 mutations were detected. She received induction therapy with daunorubicin and cytarabine and achieved hematological complete remission (HCR). After attaining HCR, she underwent consolidation therapy with azacytidine or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37466697"
                        open
                      >
                        <summary class="article-expander__title">Associations between ambient air pollutants and clonal hematopoiesis of indeterminate potential (CHIP)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37466697/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230726205537&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related somatic mutation associated with incident hematologic cancer. Environmental stressors which, like air pollution, generate oxidative stress at the cellular level, may induce somatic mutations and some mutations may provide a selection advantage for persistence and expansion of specific clones. Materials and Methods We used data from the Multi-Ethnic Study of Atherosclerosis (MESA) N&#x3D;4,379 and the Women&#x27;s Health...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37465589"
                        open
                      >
                        <summary class="article-expander__title">Gilteritinib Affects the Selection of Dominant Clones in Clonal Hematopoiesis: Sequential Genetic Analysis of an FLT3-ITD Positive AML Patient with Long-Term Gilteritinib Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37465589/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230726205537&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We performed sequential molecular analyses of a 75-year-old woman with de novo FLT3-ITD positive acute myeloid leukemia (AML) who had received gilteritinib therapy for 43 months. At the time of diagnosis, her karyotype was normal; however, FLT3-ITD, NPM1, DNMT3A, and IDH2 mutations were detected. She received induction therapy with daunorubicin and cytarabine and achieved hematological complete remission (HCR). After attaining HCR, she underwent consolidation therapy with azacytidine or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37466697"
                        open
                      >
                        <summary class="article-expander__title">Associations between ambient air pollutants and clonal hematopoiesis of indeterminate potential (CHIP)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37466697/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related somatic mutation associated with incident hematologic cancer. Environmental stressors which, like air pollution, generate oxidative stress at the cellular level, may induce somatic mutations and some mutations may provide a selection advantage for persistence and expansion of specific clones. Materials and Methods We used data from the Multi-Ethnic Study of Atherosclerosis (MESA) N&#x3D;4,379 and the Women&#x27;s Health...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37465589"
                        open
                      >
                        <summary class="article-expander__title">Gilteritinib Affects the Selection of Dominant Clones in Clonal Hematopoiesis: Sequential Genetic Analysis of an FLT3-ITD Positive AML Patient with Long-Term Gilteritinib Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37465589/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We performed sequential molecular analyses of a 75-year-old woman with de novo FLT3-ITD positive acute myeloid leukemia (AML) who had received gilteritinib therapy for 43 months. At the time of diagnosis, her karyotype was normal; however, FLT3-ITD, NPM1, DNMT3A, and IDH2 mutations were detected. She received induction therapy with daunorubicin and cytarabine and achieved hematological complete remission (HCR). After attaining HCR, she underwent consolidation therapy with azacytidine or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37466697"
                        open
                      >
                        <summary class="article-expander__title">Associations between ambient air pollutants and clonal hematopoiesis of indeterminate potential (CHIP)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37466697/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related somatic mutation associated with incident hematologic cancer. Environmental stressors which, like air pollution, generate oxidative stress at the cellular level, may induce somatic mutations and some mutations may provide a selection advantage for persistence and expansion of specific clones. Materials and Methods We used data from the Multi-Ethnic Study of Atherosclerosis (MESA) N&#x3D;4,379 and the Women&#x27;s Health...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37465589"
                        open
                      >
                        <summary class="article-expander__title">Gilteritinib Affects the Selection of Dominant Clones in Clonal Hematopoiesis: Sequential Genetic Analysis of an FLT3-ITD Positive AML Patient with Long-Term Gilteritinib Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37465589/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We performed sequential molecular analyses of a 75-year-old woman with de novo FLT3-ITD positive acute myeloid leukemia (AML) who had received gilteritinib therapy for 43 months. At the time of diagnosis, her karyotype was normal; however, FLT3-ITD, NPM1, DNMT3A, and IDH2 mutations were detected. She received induction therapy with daunorubicin and cytarabine and achieved hematological complete remission (HCR). After attaining HCR, she underwent consolidation therapy with azacytidine or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37466697"
                        open
                      >
                        <summary class="article-expander__title">Associations between ambient air pollutants and clonal hematopoiesis of indeterminate potential (CHIP)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37466697/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230723210314&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related somatic mutation associated with incident hematologic cancer. Environmental stressors which, like air pollution, generate oxidative stress at the cellular level, may induce somatic mutations and some mutations may provide a selection advantage for persistence and expansion of specific clones. Materials and Methods We used data from the Multi-Ethnic Study of Atherosclerosis (MESA) N&#x3D;4,379 and the Women&#x27;s Health...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37465589"
                        open
                      >
                        <summary class="article-expander__title">Gilteritinib Affects the Selection of Dominant Clones in Clonal Hematopoiesis: Sequential Genetic Analysis of an FLT3-ITD Positive AML Patient with Long-Term Gilteritinib Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37465589/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230723210314&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We performed sequential molecular analyses of a 75-year-old woman with de novo FLT3-ITD positive acute myeloid leukemia (AML) who had received gilteritinib therapy for 43 months. At the time of diagnosis, her karyotype was normal; however, FLT3-ITD, NPM1, DNMT3A, and IDH2 mutations were detected. She received induction therapy with daunorubicin and cytarabine and achieved hematological complete remission (HCR). After attaining HCR, she underwent consolidation therapy with azacytidine or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37466697"
                        open
                      >
                        <summary class="article-expander__title">Associations between ambient air pollutants and clonal hematopoiesis of indeterminate potential (CHIP)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37466697/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related somatic mutation associated with incident hematologic cancer. Environmental stressors which, like air pollution, generate oxidative stress at the cellular level, may induce somatic mutations and some mutations may provide a selection advantage for persistence and expansion of specific clones. Materials and Methods We used data from the Multi-Ethnic Study of Atherosclerosis (MESA) N&#x3D;4,379 and the Women&#x27;s Health...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37465589"
                        open
                      >
                        <summary class="article-expander__title">Gilteritinib Affects the Selection of Dominant Clones in Clonal Hematopoiesis: Sequential Genetic Analysis of an FLT3-ITD Positive AML Patient with Long-Term Gilteritinib Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37465589/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We performed sequential molecular analyses of a 75-year-old woman with de novo FLT3-ITD positive acute myeloid leukemia (AML) who had received gilteritinib therapy for 43 months. At the time of diagnosis, her karyotype was normal; however, FLT3-ITD, NPM1, DNMT3A, and IDH2 mutations were detected. She received induction therapy with daunorubicin and cytarabine and achieved hematological complete remission (HCR). After attaining HCR, she underwent consolidation therapy with azacytidine or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37466697"
                        open
                      >
                        <summary class="article-expander__title">Associations between ambient air pollutants and clonal hematopoiesis of indeterminate potential (CHIP)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37466697/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related somatic mutation associated with incident hematologic cancer. Environmental stressors which, like air pollution, generate oxidative stress at the cellular level, may induce somatic mutations and some mutations may provide a selection advantage for persistence and expansion of specific clones. Materials and Methods We used data from the Multi-Ethnic Study of Atherosclerosis (MESA) N&#x3D;4,379 and the Women&#x27;s Health...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37465589"
                        open
                      >
                        <summary class="article-expander__title">Gilteritinib Affects the Selection of Dominant Clones in Clonal Hematopoiesis: Sequential Genetic Analysis of an FLT3-ITD Positive AML Patient with Long-Term Gilteritinib Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37465589/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We performed sequential molecular analyses of a 75-year-old woman with de novo FLT3-ITD positive acute myeloid leukemia (AML) who had received gilteritinib therapy for 43 months. At the time of diagnosis, her karyotype was normal; however, FLT3-ITD, NPM1, DNMT3A, and IDH2 mutations were detected. She received induction therapy with daunorubicin and cytarabine and achieved hematological complete remission (HCR). After attaining HCR, she underwent consolidation therapy with azacytidine or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37466697"
                        open
                      >
                        <summary class="article-expander__title">Associations between ambient air pollutants and clonal hematopoiesis of indeterminate potential (CHIP)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37466697/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230720210259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related somatic mutation associated with incident hematologic cancer. Environmental stressors which, like air pollution, generate oxidative stress at the cellular level, may induce somatic mutations and some mutations may provide a selection advantage for persistence and expansion of specific clones. Materials and Methods We used data from the Multi-Ethnic Study of Atherosclerosis (MESA) N&#x3D;4,379 and the Women&#x27;s Health...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37465589"
                        open
                      >
                        <summary class="article-expander__title">Gilteritinib Affects the Selection of Dominant Clones in Clonal Hematopoiesis: Sequential Genetic Analysis of an FLT3-ITD Positive AML Patient with Long-Term Gilteritinib Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37465589/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230720210259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We performed sequential molecular analyses of a 75-year-old woman with de novo FLT3-ITD positive acute myeloid leukemia (AML) who had received gilteritinib therapy for 43 months. At the time of diagnosis, her karyotype was normal; however, FLT3-ITD, NPM1, DNMT3A, and IDH2 mutations were detected. She received induction therapy with daunorubicin and cytarabine and achieved hematological complete remission (HCR). After attaining HCR, she underwent consolidation therapy with azacytidine or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37466697"
                        open
                      >
                        <summary class="article-expander__title">Associations between ambient air pollutants and clonal hematopoiesis of indeterminate potential (CHIP)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37466697/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related somatic mutation associated with incident hematologic cancer. Environmental stressors which, like air pollution, generate oxidative stress at the cellular level, may induce somatic mutations and some mutations may provide a selection advantage for persistence and expansion of specific clones. Materials and Methods We used data from the Multi-Ethnic Study of Atherosclerosis (MESA) N&#x3D;4,379 and the Women&#x27;s Health...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37465589"
                        open
                      >
                        <summary class="article-expander__title">Gilteritinib Affects the Selection of Dominant Clones in Clonal Hematopoiesis: Sequential Genetic Analysis of an FLT3-ITD Positive AML Patient with Long-Term Gilteritinib Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37465589/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We performed sequential molecular analyses of a 75-year-old woman with de novo FLT3-ITD positive acute myeloid leukemia (AML) who had received gilteritinib therapy for 43 months. At the time of diagnosis, her karyotype was normal; however, FLT3-ITD, NPM1, DNMT3A, and IDH2 mutations were detected. She received induction therapy with daunorubicin and cytarabine and achieved hematological complete remission (HCR). After attaining HCR, she underwent consolidation therapy with azacytidine or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-07-17 datetime="2023-07-17T10:00:00.000Z">2023-07-17</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-07-17 datetime="2023-07-17T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37457265"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis as a novel risk factor for type 2 diabetes mellitus in patients with hypercholesterolemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37457265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The presence of CHIP was a significant risk factor for new-onset type 2 diabetes, especially in subjects with high LDL cholesterol. These results show the synergism between CHIP and high LDL cholesterol as a high-risk factor for diabetes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37457265"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis as a novel risk factor for type 2 diabetes mellitus in patients with hypercholesterolemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37457265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230726205537&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The presence of CHIP was a significant risk factor for new-onset type 2 diabetes, especially in subjects with high LDL cholesterol. These results show the synergism between CHIP and high LDL cholesterol as a high-risk factor for diabetes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37457265"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis as a novel risk factor for type 2 diabetes mellitus in patients with hypercholesterolemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37457265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The presence of CHIP was a significant risk factor for new-onset type 2 diabetes, especially in subjects with high LDL cholesterol. These results show the synergism between CHIP and high LDL cholesterol as a high-risk factor for diabetes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37457265"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis as a novel risk factor for type 2 diabetes mellitus in patients with hypercholesterolemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37457265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The presence of CHIP was a significant risk factor for new-onset type 2 diabetes, especially in subjects with high LDL cholesterol. These results show the synergism between CHIP and high LDL cholesterol as a high-risk factor for diabetes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37457265"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis as a novel risk factor for type 2 diabetes mellitus in patients with hypercholesterolemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37457265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230723210314&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The presence of CHIP was a significant risk factor for new-onset type 2 diabetes, especially in subjects with high LDL cholesterol. These results show the synergism between CHIP and high LDL cholesterol as a high-risk factor for diabetes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37457265"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis as a novel risk factor for type 2 diabetes mellitus in patients with hypercholesterolemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37457265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The presence of CHIP was a significant risk factor for new-onset type 2 diabetes, especially in subjects with high LDL cholesterol. These results show the synergism between CHIP and high LDL cholesterol as a high-risk factor for diabetes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37457265"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis as a novel risk factor for type 2 diabetes mellitus in patients with hypercholesterolemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37457265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The presence of CHIP was a significant risk factor for new-onset type 2 diabetes, especially in subjects with high LDL cholesterol. These results show the synergism between CHIP and high LDL cholesterol as a high-risk factor for diabetes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37457265"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis as a novel risk factor for type 2 diabetes mellitus in patients with hypercholesterolemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37457265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230720210259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The presence of CHIP was a significant risk factor for new-onset type 2 diabetes, especially in subjects with high LDL cholesterol. These results show the synergism between CHIP and high LDL cholesterol as a high-risk factor for diabetes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37457265"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis as a novel risk factor for type 2 diabetes mellitus in patients with hypercholesterolemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37457265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The presence of CHIP was a significant risk factor for new-onset type 2 diabetes, especially in subjects with high LDL cholesterol. These results show the synergism between CHIP and high LDL cholesterol as a high-risk factor for diabetes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37457265"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis as a novel risk factor for type 2 diabetes mellitus in patients with hypercholesterolemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37457265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The presence of CHIP was a significant risk factor for new-onset type 2 diabetes, especially in subjects with high LDL cholesterol. These results show the synergism between CHIP and high LDL cholesterol as a high-risk factor for diabetes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37457265"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis as a novel risk factor for type 2 diabetes mellitus in patients with hypercholesterolemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37457265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The presence of CHIP was a significant risk factor for new-onset type 2 diabetes, especially in subjects with high LDL cholesterol. These results show the synergism between CHIP and high LDL cholesterol as a high-risk factor for diabetes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37460928"
                        open
                      >
                        <summary class="article-expander__title">Genomic analyses of germline and somatic variation in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37460928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: From 71 patients with HGCS, we performed targeted germline and tumor sequencing and provided a comprehensive analysis of these 557 genes. We identified germline and somatic genetic alterations including somatic copy number alterations and analyzed their associations with relapse-free and overall survival. This single-site long-term follow-up study provides additional information on genetic alterations related to occurrence and outcome of HGSC. Our findings suggest that targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37460928"
                        open
                      >
                        <summary class="article-expander__title">Genomic analyses of germline and somatic variation in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37460928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: From 71 patients with HGCS, we performed targeted germline and tumor sequencing and provided a comprehensive analysis of these 557 genes. We identified germline and somatic genetic alterations including somatic copy number alterations and analyzed their associations with relapse-free and overall survival. This single-site long-term follow-up study provides additional information on genetic alterations related to occurrence and outcome of HGSC. Our findings suggest that targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37460928"
                        open
                      >
                        <summary class="article-expander__title">Genomic analyses of germline and somatic variation in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37460928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: From 71 patients with HGCS, we performed targeted germline and tumor sequencing and provided a comprehensive analysis of these 557 genes. We identified germline and somatic genetic alterations including somatic copy number alterations and analyzed their associations with relapse-free and overall survival. This single-site long-term follow-up study provides additional information on genetic alterations related to occurrence and outcome of HGSC. Our findings suggest that targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37460928"
                        open
                      >
                        <summary class="article-expander__title">Genomic analyses of germline and somatic variation in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37460928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: From 71 patients with HGCS, we performed targeted germline and tumor sequencing and provided a comprehensive analysis of these 557 genes. We identified germline and somatic genetic alterations including somatic copy number alterations and analyzed their associations with relapse-free and overall survival. This single-site long-term follow-up study provides additional information on genetic alterations related to occurrence and outcome of HGSC. Our findings suggest that targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37460928"
                        open
                      >
                        <summary class="article-expander__title">Genomic analyses of germline and somatic variation in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37460928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: From 71 patients with HGCS, we performed targeted germline and tumor sequencing and provided a comprehensive analysis of these 557 genes. We identified germline and somatic genetic alterations including somatic copy number alterations and analyzed their associations with relapse-free and overall survival. This single-site long-term follow-up study provides additional information on genetic alterations related to occurrence and outcome of HGSC. Our findings suggest that targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37460928"
                        open
                      >
                        <summary class="article-expander__title">Genomic analyses of germline and somatic variation in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37460928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: From 71 patients with HGCS, we performed targeted germline and tumor sequencing and provided a comprehensive analysis of these 557 genes. We identified germline and somatic genetic alterations including somatic copy number alterations and analyzed their associations with relapse-free and overall survival. This single-site long-term follow-up study provides additional information on genetic alterations related to occurrence and outcome of HGSC. Our findings suggest that targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37460928"
                        open
                      >
                        <summary class="article-expander__title">Genomic analyses of germline and somatic variation in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37460928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: From 71 patients with HGCS, we performed targeted germline and tumor sequencing and provided a comprehensive analysis of these 557 genes. We identified germline and somatic genetic alterations including somatic copy number alterations and analyzed their associations with relapse-free and overall survival. This single-site long-term follow-up study provides additional information on genetic alterations related to occurrence and outcome of HGSC. Our findings suggest that targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37460928"
                        open
                      >
                        <summary class="article-expander__title">Genomic analyses of germline and somatic variation in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37460928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: From 71 patients with HGCS, we performed targeted germline and tumor sequencing and provided a comprehensive analysis of these 557 genes. We identified germline and somatic genetic alterations including somatic copy number alterations and analyzed their associations with relapse-free and overall survival. This single-site long-term follow-up study provides additional information on genetic alterations related to occurrence and outcome of HGSC. Our findings suggest that targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37460928"
                        open
                      >
                        <summary class="article-expander__title">Genomic analyses of germline and somatic variation in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37460928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: From 71 patients with HGCS, we performed targeted germline and tumor sequencing and provided a comprehensive analysis of these 557 genes. We identified germline and somatic genetic alterations including somatic copy number alterations and analyzed their associations with relapse-free and overall survival. This single-site long-term follow-up study provides additional information on genetic alterations related to occurrence and outcome of HGSC. Our findings suggest that targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37460928"
                        open
                      >
                        <summary class="article-expander__title">Genomic analyses of germline and somatic variation in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37460928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: From 71 patients with HGCS, we performed targeted germline and tumor sequencing and provided a comprehensive analysis of these 557 genes. We identified germline and somatic genetic alterations including somatic copy number alterations and analyzed their associations with relapse-free and overall survival. This single-site long-term follow-up study provides additional information on genetic alterations related to occurrence and outcome of HGSC. Our findings suggest that targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37460928"
                        open
                      >
                        <summary class="article-expander__title">Genomic analyses of germline and somatic variation in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37460928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: From 71 patients with HGCS, we performed targeted germline and tumor sequencing and provided a comprehensive analysis of these 557 genes. We identified germline and somatic genetic alterations including somatic copy number alterations and analyzed their associations with relapse-free and overall survival. This single-site long-term follow-up study provides additional information on genetic alterations related to occurrence and outcome of HGSC. Our findings suggest that targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37460928"
                        open
                      >
                        <summary class="article-expander__title">Genomic analyses of germline and somatic variation in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37460928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: From 71 patients with HGCS, we performed targeted germline and tumor sequencing and provided a comprehensive analysis of these 557 genes. We identified germline and somatic genetic alterations including somatic copy number alterations and analyzed their associations with relapse-free and overall survival. This single-site long-term follow-up study provides additional information on genetic alterations related to occurrence and outcome of HGSC. Our findings suggest that targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37460928"
                        open
                      >
                        <summary class="article-expander__title">Genomic analyses of germline and somatic variation in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37460928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: From 71 patients with HGCS, we performed targeted germline and tumor sequencing and provided a comprehensive analysis of these 557 genes. We identified germline and somatic genetic alterations including somatic copy number alterations and analyzed their associations with relapse-free and overall survival. This single-site long-term follow-up study provides additional information on genetic alterations related to occurrence and outcome of HGSC. Our findings suggest that targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37460928"
                        open
                      >
                        <summary class="article-expander__title">Genomic analyses of germline and somatic variation in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37460928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: From 71 patients with HGCS, we performed targeted germline and tumor sequencing and provided a comprehensive analysis of these 557 genes. We identified germline and somatic genetic alterations including somatic copy number alterations and analyzed their associations with relapse-free and overall survival. This single-site long-term follow-up study provides additional information on genetic alterations related to occurrence and outcome of HGSC. Our findings suggest that targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37460928"
                        open
                      >
                        <summary class="article-expander__title">Genomic analyses of germline and somatic variation in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37460928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: From 71 patients with HGCS, we performed targeted germline and tumor sequencing and provided a comprehensive analysis of these 557 genes. We identified germline and somatic genetic alterations including somatic copy number alterations and analyzed their associations with relapse-free and overall survival. This single-site long-term follow-up study provides additional information on genetic alterations related to occurrence and outcome of HGSC. Our findings suggest that targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37460928"
                        open
                      >
                        <summary class="article-expander__title">Genomic analyses of germline and somatic variation in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37460928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230726205538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: From 71 patients with HGCS, we performed targeted germline and tumor sequencing and provided a comprehensive analysis of these 557 genes. We identified germline and somatic genetic alterations including somatic copy number alterations and analyzed their associations with relapse-free and overall survival. This single-site long-term follow-up study provides additional information on genetic alterations related to occurrence and outcome of HGSC. Our findings suggest that targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37460928"
                        open
                      >
                        <summary class="article-expander__title">Genomic analyses of germline and somatic variation in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37460928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: From 71 patients with HGCS, we performed targeted germline and tumor sequencing and provided a comprehensive analysis of these 557 genes. We identified germline and somatic genetic alterations including somatic copy number alterations and analyzed their associations with relapse-free and overall survival. This single-site long-term follow-up study provides additional information on genetic alterations related to occurrence and outcome of HGSC. Our findings suggest that targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37460928"
                        open
                      >
                        <summary class="article-expander__title">Genomic analyses of germline and somatic variation in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37460928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: From 71 patients with HGCS, we performed targeted germline and tumor sequencing and provided a comprehensive analysis of these 557 genes. We identified germline and somatic genetic alterations including somatic copy number alterations and analyzed their associations with relapse-free and overall survival. This single-site long-term follow-up study provides additional information on genetic alterations related to occurrence and outcome of HGSC. Our findings suggest that targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37460928"
                        open
                      >
                        <summary class="article-expander__title">Genomic analyses of germline and somatic variation in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37460928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230723210310&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: From 71 patients with HGCS, we performed targeted germline and tumor sequencing and provided a comprehensive analysis of these 557 genes. We identified germline and somatic genetic alterations including somatic copy number alterations and analyzed their associations with relapse-free and overall survival. This single-site long-term follow-up study provides additional information on genetic alterations related to occurrence and outcome of HGSC. Our findings suggest that targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37460928"
                        open
                      >
                        <summary class="article-expander__title">Genomic analyses of germline and somatic variation in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37460928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: From 71 patients with HGCS, we performed targeted germline and tumor sequencing and provided a comprehensive analysis of these 557 genes. We identified germline and somatic genetic alterations including somatic copy number alterations and analyzed their associations with relapse-free and overall survival. This single-site long-term follow-up study provides additional information on genetic alterations related to occurrence and outcome of HGSC. Our findings suggest that targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37460928"
                        open
                      >
                        <summary class="article-expander__title">Genomic analyses of germline and somatic variation in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37460928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: From 71 patients with HGCS, we performed targeted germline and tumor sequencing and provided a comprehensive analysis of these 557 genes. We identified germline and somatic genetic alterations including somatic copy number alterations and analyzed their associations with relapse-free and overall survival. This single-site long-term follow-up study provides additional information on genetic alterations related to occurrence and outcome of HGSC. Our findings suggest that targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37460928"
                        open
                      >
                        <summary class="article-expander__title">Genomic analyses of germline and somatic variation in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37460928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230720210258&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: From 71 patients with HGCS, we performed targeted germline and tumor sequencing and provided a comprehensive analysis of these 557 genes. We identified germline and somatic genetic alterations including somatic copy number alterations and analyzed their associations with relapse-free and overall survival. This single-site long-term follow-up study provides additional information on genetic alterations related to occurrence and outcome of HGSC. Our findings suggest that targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37460928"
                        open
                      >
                        <summary class="article-expander__title">Genomic analyses of germline and somatic variation in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37460928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: From 71 patients with HGCS, we performed targeted germline and tumor sequencing and provided a comprehensive analysis of these 557 genes. We identified germline and somatic genetic alterations including somatic copy number alterations and analyzed their associations with relapse-free and overall survival. This single-site long-term follow-up study provides additional information on genetic alterations related to occurrence and outcome of HGSC. Our findings suggest that targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37460928"
                        open
                      >
                        <summary class="article-expander__title">Genomic analyses of germline and somatic variation in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37460928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: From 71 patients with HGCS, we performed targeted germline and tumor sequencing and provided a comprehensive analysis of these 557 genes. We identified germline and somatic genetic alterations including somatic copy number alterations and analyzed their associations with relapse-free and overall survival. This single-site long-term follow-up study provides additional information on genetic alterations related to occurrence and outcome of HGSC. Our findings suggest that targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-07-14 datetime="2023-07-14T10:00:00.000Z">2023-07-14</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-07-14 datetime="2023-07-14T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37444494"
                        open
                      >
                        <summary class="article-expander__title">Presence of Myeloid Mutations in Patients with Chronic Myeloid Leukemia Increases Risk of Cardiovascular Event on Tyrosine Kinase Inhibitor Treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37444494/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>For chronic myeloid leukemia (CML) patients with a known risk of cardiovascular events (CVE), imatinib is often recommended for first-line tyrosine kinase inhibitor (TKI) treatment rather than a second-generation TKI (2G-TKI) such as nilotinib or dasatinib. To date, very few studies have evaluated the genetic predisposition associated with CVE development on TKI treatment. In this retrospective study of 102 CML patients, 26 CVEs were reported during an average follow-up of over 10 years....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37444416"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Characteristics of Circulating Tumor DNA in Pt1a Clear Cell Renal Cell Carcinoma: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37444416/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) is a promising biomarker for clear cell renal cell carcinoma (ccRCC); however, its characteristics in small renal masses of ccRCC remain unclear. In this pilot study, we explored the characteristics of ctDNA in pT1a ccRCC. Plasma samples were collected preoperatively from 53 patients with pT1a ccRCC. The ctDNA of pT1a ccRCC was profiled using next-generation sequencing and compared with that of higher-stage ccRCC. The association of ctDNA in pT1a ccRCC with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37444494"
                        open
                      >
                        <summary class="article-expander__title">Presence of Myeloid Mutations in Patients with Chronic Myeloid Leukemia Increases Risk of Cardiovascular Event on Tyrosine Kinase Inhibitor Treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37444494/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230726205537&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>For chronic myeloid leukemia (CML) patients with a known risk of cardiovascular events (CVE), imatinib is often recommended for first-line tyrosine kinase inhibitor (TKI) treatment rather than a second-generation TKI (2G-TKI) such as nilotinib or dasatinib. To date, very few studies have evaluated the genetic predisposition associated with CVE development on TKI treatment. In this retrospective study of 102 CML patients, 26 CVEs were reported during an average follow-up of over 10 years....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37444416"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Characteristics of Circulating Tumor DNA in Pt1a Clear Cell Renal Cell Carcinoma: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37444416/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230726205537&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) is a promising biomarker for clear cell renal cell carcinoma (ccRCC); however, its characteristics in small renal masses of ccRCC remain unclear. In this pilot study, we explored the characteristics of ctDNA in pT1a ccRCC. Plasma samples were collected preoperatively from 53 patients with pT1a ccRCC. The ctDNA of pT1a ccRCC was profiled using next-generation sequencing and compared with that of higher-stage ccRCC. The association of ctDNA in pT1a ccRCC with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37444494"
                        open
                      >
                        <summary class="article-expander__title">Presence of Myeloid Mutations in Patients with Chronic Myeloid Leukemia Increases Risk of Cardiovascular Event on Tyrosine Kinase Inhibitor Treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37444494/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>For chronic myeloid leukemia (CML) patients with a known risk of cardiovascular events (CVE), imatinib is often recommended for first-line tyrosine kinase inhibitor (TKI) treatment rather than a second-generation TKI (2G-TKI) such as nilotinib or dasatinib. To date, very few studies have evaluated the genetic predisposition associated with CVE development on TKI treatment. In this retrospective study of 102 CML patients, 26 CVEs were reported during an average follow-up of over 10 years....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37444416"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Characteristics of Circulating Tumor DNA in Pt1a Clear Cell Renal Cell Carcinoma: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37444416/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) is a promising biomarker for clear cell renal cell carcinoma (ccRCC); however, its characteristics in small renal masses of ccRCC remain unclear. In this pilot study, we explored the characteristics of ctDNA in pT1a ccRCC. Plasma samples were collected preoperatively from 53 patients with pT1a ccRCC. The ctDNA of pT1a ccRCC was profiled using next-generation sequencing and compared with that of higher-stage ccRCC. The association of ctDNA in pT1a ccRCC with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37444494"
                        open
                      >
                        <summary class="article-expander__title">Presence of Myeloid Mutations in Patients with Chronic Myeloid Leukemia Increases Risk of Cardiovascular Event on Tyrosine Kinase Inhibitor Treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37444494/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>For chronic myeloid leukemia (CML) patients with a known risk of cardiovascular events (CVE), imatinib is often recommended for first-line tyrosine kinase inhibitor (TKI) treatment rather than a second-generation TKI (2G-TKI) such as nilotinib or dasatinib. To date, very few studies have evaluated the genetic predisposition associated with CVE development on TKI treatment. In this retrospective study of 102 CML patients, 26 CVEs were reported during an average follow-up of over 10 years....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37444416"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Characteristics of Circulating Tumor DNA in Pt1a Clear Cell Renal Cell Carcinoma: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37444416/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) is a promising biomarker for clear cell renal cell carcinoma (ccRCC); however, its characteristics in small renal masses of ccRCC remain unclear. In this pilot study, we explored the characteristics of ctDNA in pT1a ccRCC. Plasma samples were collected preoperatively from 53 patients with pT1a ccRCC. The ctDNA of pT1a ccRCC was profiled using next-generation sequencing and compared with that of higher-stage ccRCC. The association of ctDNA in pT1a ccRCC with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37444494"
                        open
                      >
                        <summary class="article-expander__title">Presence of Myeloid Mutations in Patients with Chronic Myeloid Leukemia Increases Risk of Cardiovascular Event on Tyrosine Kinase Inhibitor Treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37444494/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230723210314&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>For chronic myeloid leukemia (CML) patients with a known risk of cardiovascular events (CVE), imatinib is often recommended for first-line tyrosine kinase inhibitor (TKI) treatment rather than a second-generation TKI (2G-TKI) such as nilotinib or dasatinib. To date, very few studies have evaluated the genetic predisposition associated with CVE development on TKI treatment. In this retrospective study of 102 CML patients, 26 CVEs were reported during an average follow-up of over 10 years....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37444416"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Characteristics of Circulating Tumor DNA in Pt1a Clear Cell Renal Cell Carcinoma: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37444416/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230723210314&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) is a promising biomarker for clear cell renal cell carcinoma (ccRCC); however, its characteristics in small renal masses of ccRCC remain unclear. In this pilot study, we explored the characteristics of ctDNA in pT1a ccRCC. Plasma samples were collected preoperatively from 53 patients with pT1a ccRCC. The ctDNA of pT1a ccRCC was profiled using next-generation sequencing and compared with that of higher-stage ccRCC. The association of ctDNA in pT1a ccRCC with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37444494"
                        open
                      >
                        <summary class="article-expander__title">Presence of Myeloid Mutations in Patients with Chronic Myeloid Leukemia Increases Risk of Cardiovascular Event on Tyrosine Kinase Inhibitor Treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37444494/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>For chronic myeloid leukemia (CML) patients with a known risk of cardiovascular events (CVE), imatinib is often recommended for first-line tyrosine kinase inhibitor (TKI) treatment rather than a second-generation TKI (2G-TKI) such as nilotinib or dasatinib. To date, very few studies have evaluated the genetic predisposition associated with CVE development on TKI treatment. In this retrospective study of 102 CML patients, 26 CVEs were reported during an average follow-up of over 10 years....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37444416"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Characteristics of Circulating Tumor DNA in Pt1a Clear Cell Renal Cell Carcinoma: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37444416/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) is a promising biomarker for clear cell renal cell carcinoma (ccRCC); however, its characteristics in small renal masses of ccRCC remain unclear. In this pilot study, we explored the characteristics of ctDNA in pT1a ccRCC. Plasma samples were collected preoperatively from 53 patients with pT1a ccRCC. The ctDNA of pT1a ccRCC was profiled using next-generation sequencing and compared with that of higher-stage ccRCC. The association of ctDNA in pT1a ccRCC with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37444494"
                        open
                      >
                        <summary class="article-expander__title">Presence of Myeloid Mutations in Patients with Chronic Myeloid Leukemia Increases Risk of Cardiovascular Event on Tyrosine Kinase Inhibitor Treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37444494/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>For chronic myeloid leukemia (CML) patients with a known risk of cardiovascular events (CVE), imatinib is often recommended for first-line tyrosine kinase inhibitor (TKI) treatment rather than a second-generation TKI (2G-TKI) such as nilotinib or dasatinib. To date, very few studies have evaluated the genetic predisposition associated with CVE development on TKI treatment. In this retrospective study of 102 CML patients, 26 CVEs were reported during an average follow-up of over 10 years....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37444416"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Characteristics of Circulating Tumor DNA in Pt1a Clear Cell Renal Cell Carcinoma: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37444416/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) is a promising biomarker for clear cell renal cell carcinoma (ccRCC); however, its characteristics in small renal masses of ccRCC remain unclear. In this pilot study, we explored the characteristics of ctDNA in pT1a ccRCC. Plasma samples were collected preoperatively from 53 patients with pT1a ccRCC. The ctDNA of pT1a ccRCC was profiled using next-generation sequencing and compared with that of higher-stage ccRCC. The association of ctDNA in pT1a ccRCC with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37444494"
                        open
                      >
                        <summary class="article-expander__title">Presence of Myeloid Mutations in Patients with Chronic Myeloid Leukemia Increases Risk of Cardiovascular Event on Tyrosine Kinase Inhibitor Treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37444494/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230720210259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>For chronic myeloid leukemia (CML) patients with a known risk of cardiovascular events (CVE), imatinib is often recommended for first-line tyrosine kinase inhibitor (TKI) treatment rather than a second-generation TKI (2G-TKI) such as nilotinib or dasatinib. To date, very few studies have evaluated the genetic predisposition associated with CVE development on TKI treatment. In this retrospective study of 102 CML patients, 26 CVEs were reported during an average follow-up of over 10 years....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37444416"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Characteristics of Circulating Tumor DNA in Pt1a Clear Cell Renal Cell Carcinoma: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37444416/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230720210259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) is a promising biomarker for clear cell renal cell carcinoma (ccRCC); however, its characteristics in small renal masses of ccRCC remain unclear. In this pilot study, we explored the characteristics of ctDNA in pT1a ccRCC. Plasma samples were collected preoperatively from 53 patients with pT1a ccRCC. The ctDNA of pT1a ccRCC was profiled using next-generation sequencing and compared with that of higher-stage ccRCC. The association of ctDNA in pT1a ccRCC with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37444494"
                        open
                      >
                        <summary class="article-expander__title">Presence of Myeloid Mutations in Patients with Chronic Myeloid Leukemia Increases Risk of Cardiovascular Event on Tyrosine Kinase Inhibitor Treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37444494/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>For chronic myeloid leukemia (CML) patients with a known risk of cardiovascular events (CVE), imatinib is often recommended for first-line tyrosine kinase inhibitor (TKI) treatment rather than a second-generation TKI (2G-TKI) such as nilotinib or dasatinib. To date, very few studies have evaluated the genetic predisposition associated with CVE development on TKI treatment. In this retrospective study of 102 CML patients, 26 CVEs were reported during an average follow-up of over 10 years....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37444416"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Characteristics of Circulating Tumor DNA in Pt1a Clear Cell Renal Cell Carcinoma: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37444416/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) is a promising biomarker for clear cell renal cell carcinoma (ccRCC); however, its characteristics in small renal masses of ccRCC remain unclear. In this pilot study, we explored the characteristics of ctDNA in pT1a ccRCC. Plasma samples were collected preoperatively from 53 patients with pT1a ccRCC. The ctDNA of pT1a ccRCC was profiled using next-generation sequencing and compared with that of higher-stage ccRCC. The association of ctDNA in pT1a ccRCC with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37444494"
                        open
                      >
                        <summary class="article-expander__title">Presence of Myeloid Mutations in Patients with Chronic Myeloid Leukemia Increases Risk of Cardiovascular Event on Tyrosine Kinase Inhibitor Treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37444494/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>For chronic myeloid leukemia (CML) patients with a known risk of cardiovascular events (CVE), imatinib is often recommended for first-line tyrosine kinase inhibitor (TKI) treatment rather than a second-generation TKI (2G-TKI) such as nilotinib or dasatinib. To date, very few studies have evaluated the genetic predisposition associated with CVE development on TKI treatment. In this retrospective study of 102 CML patients, 26 CVEs were reported during an average follow-up of over 10 years....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37444416"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Characteristics of Circulating Tumor DNA in Pt1a Clear Cell Renal Cell Carcinoma: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37444416/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) is a promising biomarker for clear cell renal cell carcinoma (ccRCC); however, its characteristics in small renal masses of ccRCC remain unclear. In this pilot study, we explored the characteristics of ctDNA in pT1a ccRCC. Plasma samples were collected preoperatively from 53 patients with pT1a ccRCC. The ctDNA of pT1a ccRCC was profiled using next-generation sequencing and compared with that of higher-stage ccRCC. The association of ctDNA in pT1a ccRCC with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37444494"
                        open
                      >
                        <summary class="article-expander__title">Presence of Myeloid Mutations in Patients with Chronic Myeloid Leukemia Increases Risk of Cardiovascular Event on Tyrosine Kinase Inhibitor Treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37444494/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>For chronic myeloid leukemia (CML) patients with a known risk of cardiovascular events (CVE), imatinib is often recommended for first-line tyrosine kinase inhibitor (TKI) treatment rather than a second-generation TKI (2G-TKI) such as nilotinib or dasatinib. To date, very few studies have evaluated the genetic predisposition associated with CVE development on TKI treatment. In this retrospective study of 102 CML patients, 26 CVEs were reported during an average follow-up of over 10 years....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37444416"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Characteristics of Circulating Tumor DNA in Pt1a Clear Cell Renal Cell Carcinoma: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37444416/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) is a promising biomarker for clear cell renal cell carcinoma (ccRCC); however, its characteristics in small renal masses of ccRCC remain unclear. In this pilot study, we explored the characteristics of ctDNA in pT1a ccRCC. Plasma samples were collected preoperatively from 53 patients with pT1a ccRCC. The ctDNA of pT1a ccRCC was profiled using next-generation sequencing and compared with that of higher-stage ccRCC. The association of ctDNA in pT1a ccRCC with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37444494"
                        open
                      >
                        <summary class="article-expander__title">Presence of Myeloid Mutations in Patients with Chronic Myeloid Leukemia Increases Risk of Cardiovascular Event on Tyrosine Kinase Inhibitor Treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37444494/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>For chronic myeloid leukemia (CML) patients with a known risk of cardiovascular events (CVE), imatinib is often recommended for first-line tyrosine kinase inhibitor (TKI) treatment rather than a second-generation TKI (2G-TKI) such as nilotinib or dasatinib. To date, very few studies have evaluated the genetic predisposition associated with CVE development on TKI treatment. In this retrospective study of 102 CML patients, 26 CVEs were reported during an average follow-up of over 10 years....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37444416"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Characteristics of Circulating Tumor DNA in Pt1a Clear Cell Renal Cell Carcinoma: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37444416/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) is a promising biomarker for clear cell renal cell carcinoma (ccRCC); however, its characteristics in small renal masses of ccRCC remain unclear. In this pilot study, we explored the characteristics of ctDNA in pT1a ccRCC. Plasma samples were collected preoperatively from 53 patients with pT1a ccRCC. The ctDNA of pT1a ccRCC was profiled using next-generation sequencing and compared with that of higher-stage ccRCC. The association of ctDNA in pT1a ccRCC with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37444494"
                        open
                      >
                        <summary class="article-expander__title">Presence of Myeloid Mutations in Patients with Chronic Myeloid Leukemia Increases Risk of Cardiovascular Event on Tyrosine Kinase Inhibitor Treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37444494/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>For chronic myeloid leukemia (CML) patients with a known risk of cardiovascular events (CVE), imatinib is often recommended for first-line tyrosine kinase inhibitor (TKI) treatment rather than a second-generation TKI (2G-TKI) such as nilotinib or dasatinib. To date, very few studies have evaluated the genetic predisposition associated with CVE development on TKI treatment. In this retrospective study of 102 CML patients, 26 CVEs were reported during an average follow-up of over 10 years....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37444416"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Characteristics of Circulating Tumor DNA in Pt1a Clear Cell Renal Cell Carcinoma: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37444416/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) is a promising biomarker for clear cell renal cell carcinoma (ccRCC); however, its characteristics in small renal masses of ccRCC remain unclear. In this pilot study, we explored the characteristics of ctDNA in pT1a ccRCC. Plasma samples were collected preoperatively from 53 patients with pT1a ccRCC. The ctDNA of pT1a ccRCC was profiled using next-generation sequencing and compared with that of higher-stage ccRCC. The association of ctDNA in pT1a ccRCC with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37444494"
                        open
                      >
                        <summary class="article-expander__title">Presence of Myeloid Mutations in Patients with Chronic Myeloid Leukemia Increases Risk of Cardiovascular Event on Tyrosine Kinase Inhibitor Treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37444494/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>For chronic myeloid leukemia (CML) patients with a known risk of cardiovascular events (CVE), imatinib is often recommended for first-line tyrosine kinase inhibitor (TKI) treatment rather than a second-generation TKI (2G-TKI) such as nilotinib or dasatinib. To date, very few studies have evaluated the genetic predisposition associated with CVE development on TKI treatment. In this retrospective study of 102 CML patients, 26 CVEs were reported during an average follow-up of over 10 years....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37444416"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Characteristics of Circulating Tumor DNA in Pt1a Clear Cell Renal Cell Carcinoma: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37444416/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) is a promising biomarker for clear cell renal cell carcinoma (ccRCC); however, its characteristics in small renal masses of ccRCC remain unclear. In this pilot study, we explored the characteristics of ctDNA in pT1a ccRCC. Plasma samples were collected preoperatively from 53 patients with pT1a ccRCC. The ctDNA of pT1a ccRCC was profiled using next-generation sequencing and compared with that of higher-stage ccRCC. The association of ctDNA in pT1a ccRCC with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37445976"
                        open
                      >
                        <summary class="article-expander__title">Liquid Biopsy in NSCLC: An Investigation with Multiple Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37445976/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue biopsy is essential for NSCLC diagnosis and treatment management. Over the past decades, liquid biopsy has proven to be a powerful tool in clinical oncology, isolating tumor-derived entities from the blood. Liquid biopsy permits several advantages over tissue biopsy: it is non-invasive, and it should provide a better view of tumor heterogeneity, gene alterations, and clonal evolution. Consequentially, liquid biopsy has gained attention as a cancer biomarker tool, with growing clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37445976"
                        open
                      >
                        <summary class="article-expander__title">Liquid Biopsy in NSCLC: An Investigation with Multiple Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37445976/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue biopsy is essential for NSCLC diagnosis and treatment management. Over the past decades, liquid biopsy has proven to be a powerful tool in clinical oncology, isolating tumor-derived entities from the blood. Liquid biopsy permits several advantages over tissue biopsy: it is non-invasive, and it should provide a better view of tumor heterogeneity, gene alterations, and clonal evolution. Consequentially, liquid biopsy has gained attention as a cancer biomarker tool, with growing clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37445976"
                        open
                      >
                        <summary class="article-expander__title">Liquid Biopsy in NSCLC: An Investigation with Multiple Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37445976/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue biopsy is essential for NSCLC diagnosis and treatment management. Over the past decades, liquid biopsy has proven to be a powerful tool in clinical oncology, isolating tumor-derived entities from the blood. Liquid biopsy permits several advantages over tissue biopsy: it is non-invasive, and it should provide a better view of tumor heterogeneity, gene alterations, and clonal evolution. Consequentially, liquid biopsy has gained attention as a cancer biomarker tool, with growing clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37445976"
                        open
                      >
                        <summary class="article-expander__title">Liquid Biopsy in NSCLC: An Investigation with Multiple Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37445976/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue biopsy is essential for NSCLC diagnosis and treatment management. Over the past decades, liquid biopsy has proven to be a powerful tool in clinical oncology, isolating tumor-derived entities from the blood. Liquid biopsy permits several advantages over tissue biopsy: it is non-invasive, and it should provide a better view of tumor heterogeneity, gene alterations, and clonal evolution. Consequentially, liquid biopsy has gained attention as a cancer biomarker tool, with growing clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37445976"
                        open
                      >
                        <summary class="article-expander__title">Liquid Biopsy in NSCLC: An Investigation with Multiple Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37445976/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue biopsy is essential for NSCLC diagnosis and treatment management. Over the past decades, liquid biopsy has proven to be a powerful tool in clinical oncology, isolating tumor-derived entities from the blood. Liquid biopsy permits several advantages over tissue biopsy: it is non-invasive, and it should provide a better view of tumor heterogeneity, gene alterations, and clonal evolution. Consequentially, liquid biopsy has gained attention as a cancer biomarker tool, with growing clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37445976"
                        open
                      >
                        <summary class="article-expander__title">Liquid Biopsy in NSCLC: An Investigation with Multiple Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37445976/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue biopsy is essential for NSCLC diagnosis and treatment management. Over the past decades, liquid biopsy has proven to be a powerful tool in clinical oncology, isolating tumor-derived entities from the blood. Liquid biopsy permits several advantages over tissue biopsy: it is non-invasive, and it should provide a better view of tumor heterogeneity, gene alterations, and clonal evolution. Consequentially, liquid biopsy has gained attention as a cancer biomarker tool, with growing clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37445976"
                        open
                      >
                        <summary class="article-expander__title">Liquid Biopsy in NSCLC: An Investigation with Multiple Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37445976/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue biopsy is essential for NSCLC diagnosis and treatment management. Over the past decades, liquid biopsy has proven to be a powerful tool in clinical oncology, isolating tumor-derived entities from the blood. Liquid biopsy permits several advantages over tissue biopsy: it is non-invasive, and it should provide a better view of tumor heterogeneity, gene alterations, and clonal evolution. Consequentially, liquid biopsy has gained attention as a cancer biomarker tool, with growing clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37445976"
                        open
                      >
                        <summary class="article-expander__title">Liquid Biopsy in NSCLC: An Investigation with Multiple Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37445976/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue biopsy is essential for NSCLC diagnosis and treatment management. Over the past decades, liquid biopsy has proven to be a powerful tool in clinical oncology, isolating tumor-derived entities from the blood. Liquid biopsy permits several advantages over tissue biopsy: it is non-invasive, and it should provide a better view of tumor heterogeneity, gene alterations, and clonal evolution. Consequentially, liquid biopsy has gained attention as a cancer biomarker tool, with growing clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37445976"
                        open
                      >
                        <summary class="article-expander__title">Liquid Biopsy in NSCLC: An Investigation with Multiple Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37445976/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue biopsy is essential for NSCLC diagnosis and treatment management. Over the past decades, liquid biopsy has proven to be a powerful tool in clinical oncology, isolating tumor-derived entities from the blood. Liquid biopsy permits several advantages over tissue biopsy: it is non-invasive, and it should provide a better view of tumor heterogeneity, gene alterations, and clonal evolution. Consequentially, liquid biopsy has gained attention as a cancer biomarker tool, with growing clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37445976"
                        open
                      >
                        <summary class="article-expander__title">Liquid Biopsy in NSCLC: An Investigation with Multiple Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37445976/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue biopsy is essential for NSCLC diagnosis and treatment management. Over the past decades, liquid biopsy has proven to be a powerful tool in clinical oncology, isolating tumor-derived entities from the blood. Liquid biopsy permits several advantages over tissue biopsy: it is non-invasive, and it should provide a better view of tumor heterogeneity, gene alterations, and clonal evolution. Consequentially, liquid biopsy has gained attention as a cancer biomarker tool, with growing clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37445976"
                        open
                      >
                        <summary class="article-expander__title">Liquid Biopsy in NSCLC: An Investigation with Multiple Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37445976/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue biopsy is essential for NSCLC diagnosis and treatment management. Over the past decades, liquid biopsy has proven to be a powerful tool in clinical oncology, isolating tumor-derived entities from the blood. Liquid biopsy permits several advantages over tissue biopsy: it is non-invasive, and it should provide a better view of tumor heterogeneity, gene alterations, and clonal evolution. Consequentially, liquid biopsy has gained attention as a cancer biomarker tool, with growing clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37445976"
                        open
                      >
                        <summary class="article-expander__title">Liquid Biopsy in NSCLC: An Investigation with Multiple Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37445976/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue biopsy is essential for NSCLC diagnosis and treatment management. Over the past decades, liquid biopsy has proven to be a powerful tool in clinical oncology, isolating tumor-derived entities from the blood. Liquid biopsy permits several advantages over tissue biopsy: it is non-invasive, and it should provide a better view of tumor heterogeneity, gene alterations, and clonal evolution. Consequentially, liquid biopsy has gained attention as a cancer biomarker tool, with growing clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37445976"
                        open
                      >
                        <summary class="article-expander__title">Liquid Biopsy in NSCLC: An Investigation with Multiple Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37445976/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue biopsy is essential for NSCLC diagnosis and treatment management. Over the past decades, liquid biopsy has proven to be a powerful tool in clinical oncology, isolating tumor-derived entities from the blood. Liquid biopsy permits several advantages over tissue biopsy: it is non-invasive, and it should provide a better view of tumor heterogeneity, gene alterations, and clonal evolution. Consequentially, liquid biopsy has gained attention as a cancer biomarker tool, with growing clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37445976"
                        open
                      >
                        <summary class="article-expander__title">Liquid Biopsy in NSCLC: An Investigation with Multiple Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37445976/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue biopsy is essential for NSCLC diagnosis and treatment management. Over the past decades, liquid biopsy has proven to be a powerful tool in clinical oncology, isolating tumor-derived entities from the blood. Liquid biopsy permits several advantages over tissue biopsy: it is non-invasive, and it should provide a better view of tumor heterogeneity, gene alterations, and clonal evolution. Consequentially, liquid biopsy has gained attention as a cancer biomarker tool, with growing clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37445976"
                        open
                      >
                        <summary class="article-expander__title">Liquid Biopsy in NSCLC: An Investigation with Multiple Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37445976/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue biopsy is essential for NSCLC diagnosis and treatment management. Over the past decades, liquid biopsy has proven to be a powerful tool in clinical oncology, isolating tumor-derived entities from the blood. Liquid biopsy permits several advantages over tissue biopsy: it is non-invasive, and it should provide a better view of tumor heterogeneity, gene alterations, and clonal evolution. Consequentially, liquid biopsy has gained attention as a cancer biomarker tool, with growing clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37445976"
                        open
                      >
                        <summary class="article-expander__title">Liquid Biopsy in NSCLC: An Investigation with Multiple Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37445976/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230726205538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue biopsy is essential for NSCLC diagnosis and treatment management. Over the past decades, liquid biopsy has proven to be a powerful tool in clinical oncology, isolating tumor-derived entities from the blood. Liquid biopsy permits several advantages over tissue biopsy: it is non-invasive, and it should provide a better view of tumor heterogeneity, gene alterations, and clonal evolution. Consequentially, liquid biopsy has gained attention as a cancer biomarker tool, with growing clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37445976"
                        open
                      >
                        <summary class="article-expander__title">Liquid Biopsy in NSCLC: An Investigation with Multiple Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37445976/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue biopsy is essential for NSCLC diagnosis and treatment management. Over the past decades, liquid biopsy has proven to be a powerful tool in clinical oncology, isolating tumor-derived entities from the blood. Liquid biopsy permits several advantages over tissue biopsy: it is non-invasive, and it should provide a better view of tumor heterogeneity, gene alterations, and clonal evolution. Consequentially, liquid biopsy has gained attention as a cancer biomarker tool, with growing clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37445976"
                        open
                      >
                        <summary class="article-expander__title">Liquid Biopsy in NSCLC: An Investigation with Multiple Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37445976/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue biopsy is essential for NSCLC diagnosis and treatment management. Over the past decades, liquid biopsy has proven to be a powerful tool in clinical oncology, isolating tumor-derived entities from the blood. Liquid biopsy permits several advantages over tissue biopsy: it is non-invasive, and it should provide a better view of tumor heterogeneity, gene alterations, and clonal evolution. Consequentially, liquid biopsy has gained attention as a cancer biomarker tool, with growing clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37445976"
                        open
                      >
                        <summary class="article-expander__title">Liquid Biopsy in NSCLC: An Investigation with Multiple Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37445976/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230723210310&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue biopsy is essential for NSCLC diagnosis and treatment management. Over the past decades, liquid biopsy has proven to be a powerful tool in clinical oncology, isolating tumor-derived entities from the blood. Liquid biopsy permits several advantages over tissue biopsy: it is non-invasive, and it should provide a better view of tumor heterogeneity, gene alterations, and clonal evolution. Consequentially, liquid biopsy has gained attention as a cancer biomarker tool, with growing clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37445976"
                        open
                      >
                        <summary class="article-expander__title">Liquid Biopsy in NSCLC: An Investigation with Multiple Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37445976/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue biopsy is essential for NSCLC diagnosis and treatment management. Over the past decades, liquid biopsy has proven to be a powerful tool in clinical oncology, isolating tumor-derived entities from the blood. Liquid biopsy permits several advantages over tissue biopsy: it is non-invasive, and it should provide a better view of tumor heterogeneity, gene alterations, and clonal evolution. Consequentially, liquid biopsy has gained attention as a cancer biomarker tool, with growing clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37445976"
                        open
                      >
                        <summary class="article-expander__title">Liquid Biopsy in NSCLC: An Investigation with Multiple Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37445976/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue biopsy is essential for NSCLC diagnosis and treatment management. Over the past decades, liquid biopsy has proven to be a powerful tool in clinical oncology, isolating tumor-derived entities from the blood. Liquid biopsy permits several advantages over tissue biopsy: it is non-invasive, and it should provide a better view of tumor heterogeneity, gene alterations, and clonal evolution. Consequentially, liquid biopsy has gained attention as a cancer biomarker tool, with growing clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37445976"
                        open
                      >
                        <summary class="article-expander__title">Liquid Biopsy in NSCLC: An Investigation with Multiple Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37445976/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230720210258&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue biopsy is essential for NSCLC diagnosis and treatment management. Over the past decades, liquid biopsy has proven to be a powerful tool in clinical oncology, isolating tumor-derived entities from the blood. Liquid biopsy permits several advantages over tissue biopsy: it is non-invasive, and it should provide a better view of tumor heterogeneity, gene alterations, and clonal evolution. Consequentially, liquid biopsy has gained attention as a cancer biomarker tool, with growing clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37445976"
                        open
                      >
                        <summary class="article-expander__title">Liquid Biopsy in NSCLC: An Investigation with Multiple Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37445976/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue biopsy is essential for NSCLC diagnosis and treatment management. Over the past decades, liquid biopsy has proven to be a powerful tool in clinical oncology, isolating tumor-derived entities from the blood. Liquid biopsy permits several advantages over tissue biopsy: it is non-invasive, and it should provide a better view of tumor heterogeneity, gene alterations, and clonal evolution. Consequentially, liquid biopsy has gained attention as a cancer biomarker tool, with growing clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37445976"
                        open
                      >
                        <summary class="article-expander__title">Liquid Biopsy in NSCLC: An Investigation with Multiple Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37445976/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue biopsy is essential for NSCLC diagnosis and treatment management. Over the past decades, liquid biopsy has proven to be a powerful tool in clinical oncology, isolating tumor-derived entities from the blood. Liquid biopsy permits several advantages over tissue biopsy: it is non-invasive, and it should provide a better view of tumor heterogeneity, gene alterations, and clonal evolution. Consequentially, liquid biopsy has gained attention as a cancer biomarker tool, with growing clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37445976"
                        open
                      >
                        <summary class="article-expander__title">Liquid Biopsy in NSCLC: An Investigation with Multiple Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37445976/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue biopsy is essential for NSCLC diagnosis and treatment management. Over the past decades, liquid biopsy has proven to be a powerful tool in clinical oncology, isolating tumor-derived entities from the blood. Liquid biopsy permits several advantages over tissue biopsy: it is non-invasive, and it should provide a better view of tumor heterogeneity, gene alterations, and clonal evolution. Consequentially, liquid biopsy has gained attention as a cancer biomarker tool, with growing clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37445976"
                        open
                      >
                        <summary class="article-expander__title">Liquid Biopsy in NSCLC: An Investigation with Multiple Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37445976/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue biopsy is essential for NSCLC diagnosis and treatment management. Over the past decades, liquid biopsy has proven to be a powerful tool in clinical oncology, isolating tumor-derived entities from the blood. Liquid biopsy permits several advantages over tissue biopsy: it is non-invasive, and it should provide a better view of tumor heterogeneity, gene alterations, and clonal evolution. Consequentially, liquid biopsy has gained attention as a cancer biomarker tool, with growing clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37445976"
                        open
                      >
                        <summary class="article-expander__title">Liquid Biopsy in NSCLC: An Investigation with Multiple Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37445976/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue biopsy is essential for NSCLC diagnosis and treatment management. Over the past decades, liquid biopsy has proven to be a powerful tool in clinical oncology, isolating tumor-derived entities from the blood. Liquid biopsy permits several advantages over tissue biopsy: it is non-invasive, and it should provide a better view of tumor heterogeneity, gene alterations, and clonal evolution. Consequentially, liquid biopsy has gained attention as a cancer biomarker tool, with growing clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37445976"
                        open
                      >
                        <summary class="article-expander__title">Liquid Biopsy in NSCLC: An Investigation with Multiple Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37445976/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue biopsy is essential for NSCLC diagnosis and treatment management. Over the past decades, liquid biopsy has proven to be a powerful tool in clinical oncology, isolating tumor-derived entities from the blood. Liquid biopsy permits several advantages over tissue biopsy: it is non-invasive, and it should provide a better view of tumor heterogeneity, gene alterations, and clonal evolution. Consequentially, liquid biopsy has gained attention as a cancer biomarker tool, with growing clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-07-13 datetime="2023-07-13T10:00:00.000Z">2023-07-13</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-07-13 datetime="2023-07-13T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37439808"
                        open
                      >
                        <summary class="article-expander__title">A Phase 1b trial of AVID200, a TGFβ 1/3 trap, in patients with myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37439808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: AVID200 is a well-tolerated, rational, therapeutic agent for the treatment of MF patients and should be evaluated further in thrombocytopenic MF patients in combination with agents that target aberrant MF intracellular signaling pathways.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37439808"
                        open
                      >
                        <summary class="article-expander__title">A Phase 1b trial of AVID200, a TGFβ 1/3 trap, in patients with myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37439808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230726205537&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: AVID200 is a well-tolerated, rational, therapeutic agent for the treatment of MF patients and should be evaluated further in thrombocytopenic MF patients in combination with agents that target aberrant MF intracellular signaling pathways.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37439808"
                        open
                      >
                        <summary class="article-expander__title">A Phase 1b trial of AVID200, a TGFβ 1/3 trap, in patients with myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37439808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: AVID200 is a well-tolerated, rational, therapeutic agent for the treatment of MF patients and should be evaluated further in thrombocytopenic MF patients in combination with agents that target aberrant MF intracellular signaling pathways.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37439808"
                        open
                      >
                        <summary class="article-expander__title">A Phase 1b trial of AVID200, a TGFβ 1/3 trap, in patients with myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37439808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: AVID200 is a well-tolerated, rational, therapeutic agent for the treatment of MF patients and should be evaluated further in thrombocytopenic MF patients in combination with agents that target aberrant MF intracellular signaling pathways.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37439808"
                        open
                      >
                        <summary class="article-expander__title">A Phase 1b trial of AVID200, a TGFβ 1/3 trap, in patients with myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37439808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230723210314&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: AVID200 is a well-tolerated, rational, therapeutic agent for the treatment of MF patients and should be evaluated further in thrombocytopenic MF patients in combination with agents that target aberrant MF intracellular signaling pathways.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37439808"
                        open
                      >
                        <summary class="article-expander__title">A Phase 1b trial of AVID200, a TGFβ 1/3 trap, in patients with myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37439808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: AVID200 is a well-tolerated, rational, therapeutic agent for the treatment of MF patients and should be evaluated further in thrombocytopenic MF patients in combination with agents that target aberrant MF intracellular signaling pathways.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37439808"
                        open
                      >
                        <summary class="article-expander__title">A Phase 1b trial of AVID200, a TGFβ 1/3 trap, in patients with myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37439808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: AVID200 is a well-tolerated, rational, therapeutic agent for the treatment of MF patients and should be evaluated further in thrombocytopenic MF patients in combination with agents that target aberrant MF intracellular signaling pathways.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37439808"
                        open
                      >
                        <summary class="article-expander__title">A Phase 1b trial of AVID200, a TGFβ 1/3 trap, in patients with myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37439808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230720210259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: AVID200 is a well-tolerated, rational, therapeutic agent for the treatment of MF patients and should be evaluated further in thrombocytopenic MF patients in combination with agents that target aberrant MF intracellular signaling pathways.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37439808"
                        open
                      >
                        <summary class="article-expander__title">A Phase 1b trial of AVID200, a TGFβ 1/3 trap, in patients with myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37439808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: AVID200 is a well-tolerated, rational, therapeutic agent for the treatment of MF patients and should be evaluated further in thrombocytopenic MF patients in combination with agents that target aberrant MF intracellular signaling pathways.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37439808"
                        open
                      >
                        <summary class="article-expander__title">A Phase 1b trial of AVID200, a TGFβ 1/3 trap, in patients with myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37439808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: AVID200 is a well-tolerated, rational, therapeutic agent for the treatment of MF patients and should be evaluated further in thrombocytopenic MF patients in combination with agents that target aberrant MF intracellular signaling pathways.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37439808"
                        open
                      >
                        <summary class="article-expander__title">A Phase 1b trial of AVID200, a TGFβ 1/3 trap, in patients with myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37439808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: AVID200 is a well-tolerated, rational, therapeutic agent for the treatment of MF patients and should be evaluated further in thrombocytopenic MF patients in combination with agents that target aberrant MF intracellular signaling pathways.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37439808"
                        open
                      >
                        <summary class="article-expander__title">A Phase 1b trial of AVID200, a TGFβ 1/3 trap, in patients with myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37439808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: AVID200 is a well-tolerated, rational, therapeutic agent for the treatment of MF patients and should be evaluated further in thrombocytopenic MF patients in combination with agents that target aberrant MF intracellular signaling pathways.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37439808"
                        open
                      >
                        <summary class="article-expander__title">A Phase 1b trial of AVID200, a TGFβ 1/3 trap, in patients with myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37439808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: AVID200 is a well-tolerated, rational, therapeutic agent for the treatment of MF patients and should be evaluated further in thrombocytopenic MF patients in combination with agents that target aberrant MF intracellular signaling pathways.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37439808"
                        open
                      >
                        <summary class="article-expander__title">A Phase 1b trial of AVID200, a TGFβ 1/3 trap, in patients with myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37439808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: AVID200 is a well-tolerated, rational, therapeutic agent for the treatment of MF patients and should be evaluated further in thrombocytopenic MF patients in combination with agents that target aberrant MF intracellular signaling pathways.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37439808"
                        open
                      >
                        <summary class="article-expander__title">A Phase 1b trial of AVID200, a TGFβ 1/3 trap, in patients with myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37439808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: AVID200 is a well-tolerated, rational, therapeutic agent for the treatment of MF patients and should be evaluated further in thrombocytopenic MF patients in combination with agents that target aberrant MF intracellular signaling pathways.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-07-12 datetime="2023-07-12T10:00:00.000Z">2023-07-12</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-07-12 datetime="2023-07-12T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435889"
                        open
                      >
                        <summary class="article-expander__title">Clinical relevance of clonal hematopoiesis and its interference in cell-free DNA profiling of patients with gastric cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435889/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230726205537&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This study revealed the clinical impact of CH in Korean patients and suggests the potential for its interference in cfDNA tests.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435889"
                        open
                      >
                        <summary class="article-expander__title">Clinical relevance of clonal hematopoiesis and its interference in cell-free DNA profiling of patients with gastric cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435889/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This study revealed the clinical impact of CH in Korean patients and suggests the potential for its interference in cfDNA tests.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435412"
                        open
                      >
                        <summary class="article-expander__title">Donor Cell Leukemia Following Allogeneic Hematopoietic Stem Cell Transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Approximately 25,000 allogeneic transplants are performed annually worldwide; a figure that has steadily increased over the past three decades. The study of transplant recipient survivorship has become a cogent topic and post-transplant donor cell pathology warrants further study. Donor cell leukemia (DCL) is a rare but serious complication of allogeneic stem cell transplantation (SCT) where the recipient develops a form leukemia originating from the donor cells used for transplantation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435889"
                        open
                      >
                        <summary class="article-expander__title">Clinical relevance of clonal hematopoiesis and its interference in cell-free DNA profiling of patients with gastric cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435889/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This study revealed the clinical impact of CH in Korean patients and suggests the potential for its interference in cfDNA tests.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435412"
                        open
                      >
                        <summary class="article-expander__title">Donor Cell Leukemia Following Allogeneic Hematopoietic Stem Cell Transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Approximately 25,000 allogeneic transplants are performed annually worldwide; a figure that has steadily increased over the past three decades. The study of transplant recipient survivorship has become a cogent topic and post-transplant donor cell pathology warrants further study. Donor cell leukemia (DCL) is a rare but serious complication of allogeneic stem cell transplantation (SCT) where the recipient develops a form leukemia originating from the donor cells used for transplantation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435035"
                        open
                      >
                        <summary class="article-expander__title">The Mechanism of miR-155/miR-15b Axis Contributed to Apoptosis of CD34+ Cells by Upregulation of PD-L1 in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435035/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of heterogeneous myeloid clonal diseases that are characterized by ineffective bone marrow hematopoiesis. Since studies have confirmed the significance of miRNAs in ineffective hematopoiesis in MDS, the current report elucidated the mechanism mediated by miR-155-5p. The bone marrow of MDS patients was collected to detect miR-155-5p and to analyze the correlation between miR-155-5p and clinicopathological variables. Isolated bone marrow CD34+ cells were...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435889"
                        open
                      >
                        <summary class="article-expander__title">Clinical relevance of clonal hematopoiesis and its interference in cell-free DNA profiling of patients with gastric cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435889/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230723210314&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This study revealed the clinical impact of CH in Korean patients and suggests the potential for its interference in cfDNA tests.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435412"
                        open
                      >
                        <summary class="article-expander__title">Donor Cell Leukemia Following Allogeneic Hematopoietic Stem Cell Transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230723210314&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Approximately 25,000 allogeneic transplants are performed annually worldwide; a figure that has steadily increased over the past three decades. The study of transplant recipient survivorship has become a cogent topic and post-transplant donor cell pathology warrants further study. Donor cell leukemia (DCL) is a rare but serious complication of allogeneic stem cell transplantation (SCT) where the recipient develops a form leukemia originating from the donor cells used for transplantation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435035"
                        open
                      >
                        <summary class="article-expander__title">The Mechanism of miR-155/miR-15b Axis Contributed to Apoptosis of CD34+ Cells by Upregulation of PD-L1 in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435035/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230723210314&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of heterogeneous myeloid clonal diseases that are characterized by ineffective bone marrow hematopoiesis. Since studies have confirmed the significance of miRNAs in ineffective hematopoiesis in MDS, the current report elucidated the mechanism mediated by miR-155-5p. The bone marrow of MDS patients was collected to detect miR-155-5p and to analyze the correlation between miR-155-5p and clinicopathological variables. Isolated bone marrow CD34+ cells were...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435889"
                        open
                      >
                        <summary class="article-expander__title">Clinical relevance of clonal hematopoiesis and its interference in cell-free DNA profiling of patients with gastric cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435889/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This study revealed the clinical impact of CH in Korean patients and suggests the potential for its interference in cfDNA tests.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435412"
                        open
                      >
                        <summary class="article-expander__title">Donor Cell Leukemia Following Allogeneic Hematopoietic Stem Cell Transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Approximately 25,000 allogeneic transplants are performed annually worldwide; a figure that has steadily increased over the past three decades. The study of transplant recipient survivorship has become a cogent topic and post-transplant donor cell pathology warrants further study. Donor cell leukemia (DCL) is a rare but serious complication of allogeneic stem cell transplantation (SCT) where the recipient develops a form leukemia originating from the donor cells used for transplantation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435035"
                        open
                      >
                        <summary class="article-expander__title">The Mechanism of miR-155/miR-15b Axis Contributed to Apoptosis of CD34+ Cells by Upregulation of PD-L1 in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435035/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of heterogeneous myeloid clonal diseases that are characterized by ineffective bone marrow hematopoiesis. Since studies have confirmed the significance of miRNAs in ineffective hematopoiesis in MDS, the current report elucidated the mechanism mediated by miR-155-5p. The bone marrow of MDS patients was collected to detect miR-155-5p and to analyze the correlation between miR-155-5p and clinicopathological variables. Isolated bone marrow CD34+ cells were...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435889"
                        open
                      >
                        <summary class="article-expander__title">Clinical relevance of clonal hematopoiesis and its interference in cell-free DNA profiling of patients with gastric cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435889/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This study revealed the clinical impact of CH in Korean patients and suggests the potential for its interference in cfDNA tests.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435412"
                        open
                      >
                        <summary class="article-expander__title">Donor Cell Leukemia Following Allogeneic Hematopoietic Stem Cell Transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Approximately 25,000 allogeneic transplants are performed annually worldwide; a figure that has steadily increased over the past three decades. The study of transplant recipient survivorship has become a cogent topic and post-transplant donor cell pathology warrants further study. Donor cell leukemia (DCL) is a rare but serious complication of allogeneic stem cell transplantation (SCT) where the recipient develops a form leukemia originating from the donor cells used for transplantation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435035"
                        open
                      >
                        <summary class="article-expander__title">The Mechanism of miR-155/miR-15b Axis Contributed to Apoptosis of CD34+ Cells by Upregulation of PD-L1 in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435035/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of heterogeneous myeloid clonal diseases that are characterized by ineffective bone marrow hematopoiesis. Since studies have confirmed the significance of miRNAs in ineffective hematopoiesis in MDS, the current report elucidated the mechanism mediated by miR-155-5p. The bone marrow of MDS patients was collected to detect miR-155-5p and to analyze the correlation between miR-155-5p and clinicopathological variables. Isolated bone marrow CD34+ cells were...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435889"
                        open
                      >
                        <summary class="article-expander__title">Clinical relevance of clonal hematopoiesis and its interference in cell-free DNA profiling of patients with gastric cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435889/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230720210259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This study revealed the clinical impact of CH in Korean patients and suggests the potential for its interference in cfDNA tests.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435412"
                        open
                      >
                        <summary class="article-expander__title">Donor Cell Leukemia Following Allogeneic Hematopoietic Stem Cell Transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230720210259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Approximately 25,000 allogeneic transplants are performed annually worldwide; a figure that has steadily increased over the past three decades. The study of transplant recipient survivorship has become a cogent topic and post-transplant donor cell pathology warrants further study. Donor cell leukemia (DCL) is a rare but serious complication of allogeneic stem cell transplantation (SCT) where the recipient develops a form leukemia originating from the donor cells used for transplantation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435035"
                        open
                      >
                        <summary class="article-expander__title">The Mechanism of miR-155/miR-15b Axis Contributed to Apoptosis of CD34+ Cells by Upregulation of PD-L1 in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435035/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230720210259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of heterogeneous myeloid clonal diseases that are characterized by ineffective bone marrow hematopoiesis. Since studies have confirmed the significance of miRNAs in ineffective hematopoiesis in MDS, the current report elucidated the mechanism mediated by miR-155-5p. The bone marrow of MDS patients was collected to detect miR-155-5p and to analyze the correlation between miR-155-5p and clinicopathological variables. Isolated bone marrow CD34+ cells were...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435889"
                        open
                      >
                        <summary class="article-expander__title">Clinical relevance of clonal hematopoiesis and its interference in cell-free DNA profiling of patients with gastric cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435889/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This study revealed the clinical impact of CH in Korean patients and suggests the potential for its interference in cfDNA tests.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435412"
                        open
                      >
                        <summary class="article-expander__title">Donor Cell Leukemia Following Allogeneic Hematopoietic Stem Cell Transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Approximately 25,000 allogeneic transplants are performed annually worldwide; a figure that has steadily increased over the past three decades. The study of transplant recipient survivorship has become a cogent topic and post-transplant donor cell pathology warrants further study. Donor cell leukemia (DCL) is a rare but serious complication of allogeneic stem cell transplantation (SCT) where the recipient develops a form leukemia originating from the donor cells used for transplantation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435035"
                        open
                      >
                        <summary class="article-expander__title">The Mechanism of miR-155/miR-15b Axis Contributed to Apoptosis of CD34+ Cells by Upregulation of PD-L1 in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435035/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of heterogeneous myeloid clonal diseases that are characterized by ineffective bone marrow hematopoiesis. Since studies have confirmed the significance of miRNAs in ineffective hematopoiesis in MDS, the current report elucidated the mechanism mediated by miR-155-5p. The bone marrow of MDS patients was collected to detect miR-155-5p and to analyze the correlation between miR-155-5p and clinicopathological variables. Isolated bone marrow CD34+ cells were...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435889"
                        open
                      >
                        <summary class="article-expander__title">Clinical relevance of clonal hematopoiesis and its interference in cell-free DNA profiling of patients with gastric cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435889/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This study revealed the clinical impact of CH in Korean patients and suggests the potential for its interference in cfDNA tests.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435412"
                        open
                      >
                        <summary class="article-expander__title">Donor Cell Leukemia Following Allogeneic Hematopoietic Stem Cell Transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Approximately 25,000 allogeneic transplants are performed annually worldwide; a figure that has steadily increased over the past three decades. The study of transplant recipient survivorship has become a cogent topic and post-transplant donor cell pathology warrants further study. Donor cell leukemia (DCL) is a rare but serious complication of allogeneic stem cell transplantation (SCT) where the recipient develops a form leukemia originating from the donor cells used for transplantation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435035"
                        open
                      >
                        <summary class="article-expander__title">The Mechanism of miR-155/miR-15b Axis Contributed to Apoptosis of CD34+ Cells by Upregulation of PD-L1 in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435035/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of heterogeneous myeloid clonal diseases that are characterized by ineffective bone marrow hematopoiesis. Since studies have confirmed the significance of miRNAs in ineffective hematopoiesis in MDS, the current report elucidated the mechanism mediated by miR-155-5p. The bone marrow of MDS patients was collected to detect miR-155-5p and to analyze the correlation between miR-155-5p and clinicopathological variables. Isolated bone marrow CD34+ cells were...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435889"
                        open
                      >
                        <summary class="article-expander__title">Clinical relevance of clonal hematopoiesis and its interference in cell-free DNA profiling of patients with gastric cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435889/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This study revealed the clinical impact of CH in Korean patients and suggests the potential for its interference in cfDNA tests.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435412"
                        open
                      >
                        <summary class="article-expander__title">Donor Cell Leukemia Following Allogeneic Hematopoietic Stem Cell Transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Approximately 25,000 allogeneic transplants are performed annually worldwide; a figure that has steadily increased over the past three decades. The study of transplant recipient survivorship has become a cogent topic and post-transplant donor cell pathology warrants further study. Donor cell leukemia (DCL) is a rare but serious complication of allogeneic stem cell transplantation (SCT) where the recipient develops a form leukemia originating from the donor cells used for transplantation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435035"
                        open
                      >
                        <summary class="article-expander__title">The Mechanism of miR-155/miR-15b Axis Contributed to Apoptosis of CD34+ Cells by Upregulation of PD-L1 in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435035/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of heterogeneous myeloid clonal diseases that are characterized by ineffective bone marrow hematopoiesis. Since studies have confirmed the significance of miRNAs in ineffective hematopoiesis in MDS, the current report elucidated the mechanism mediated by miR-155-5p. The bone marrow of MDS patients was collected to detect miR-155-5p and to analyze the correlation between miR-155-5p and clinicopathological variables. Isolated bone marrow CD34+ cells were...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435889"
                        open
                      >
                        <summary class="article-expander__title">Clinical relevance of clonal hematopoiesis and its interference in cell-free DNA profiling of patients with gastric cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435889/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This study revealed the clinical impact of CH in Korean patients and suggests the potential for its interference in cfDNA tests.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435412"
                        open
                      >
                        <summary class="article-expander__title">Donor Cell Leukemia Following Allogeneic Hematopoietic Stem Cell Transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Approximately 25,000 allogeneic transplants are performed annually worldwide; a figure that has steadily increased over the past three decades. The study of transplant recipient survivorship has become a cogent topic and post-transplant donor cell pathology warrants further study. Donor cell leukemia (DCL) is a rare but serious complication of allogeneic stem cell transplantation (SCT) where the recipient develops a form leukemia originating from the donor cells used for transplantation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435035"
                        open
                      >
                        <summary class="article-expander__title">The Mechanism of miR-155/miR-15b Axis Contributed to Apoptosis of CD34+ Cells by Upregulation of PD-L1 in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435035/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of heterogeneous myeloid clonal diseases that are characterized by ineffective bone marrow hematopoiesis. Since studies have confirmed the significance of miRNAs in ineffective hematopoiesis in MDS, the current report elucidated the mechanism mediated by miR-155-5p. The bone marrow of MDS patients was collected to detect miR-155-5p and to analyze the correlation between miR-155-5p and clinicopathological variables. Isolated bone marrow CD34+ cells were...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435889"
                        open
                      >
                        <summary class="article-expander__title">Clinical relevance of clonal hematopoiesis and its interference in cell-free DNA profiling of patients with gastric cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435889/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This study revealed the clinical impact of CH in Korean patients and suggests the potential for its interference in cfDNA tests.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435412"
                        open
                      >
                        <summary class="article-expander__title">Donor Cell Leukemia Following Allogeneic Hematopoietic Stem Cell Transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Approximately 25,000 allogeneic transplants are performed annually worldwide; a figure that has steadily increased over the past three decades. The study of transplant recipient survivorship has become a cogent topic and post-transplant donor cell pathology warrants further study. Donor cell leukemia (DCL) is a rare but serious complication of allogeneic stem cell transplantation (SCT) where the recipient develops a form leukemia originating from the donor cells used for transplantation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435035"
                        open
                      >
                        <summary class="article-expander__title">The Mechanism of miR-155/miR-15b Axis Contributed to Apoptosis of CD34+ Cells by Upregulation of PD-L1 in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435035/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of heterogeneous myeloid clonal diseases that are characterized by ineffective bone marrow hematopoiesis. Since studies have confirmed the significance of miRNAs in ineffective hematopoiesis in MDS, the current report elucidated the mechanism mediated by miR-155-5p. The bone marrow of MDS patients was collected to detect miR-155-5p and to analyze the correlation between miR-155-5p and clinicopathological variables. Isolated bone marrow CD34+ cells were...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435889"
                        open
                      >
                        <summary class="article-expander__title">Clinical relevance of clonal hematopoiesis and its interference in cell-free DNA profiling of patients with gastric cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435889/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This study revealed the clinical impact of CH in Korean patients and suggests the potential for its interference in cfDNA tests.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435412"
                        open
                      >
                        <summary class="article-expander__title">Donor Cell Leukemia Following Allogeneic Hematopoietic Stem Cell Transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Approximately 25,000 allogeneic transplants are performed annually worldwide; a figure that has steadily increased over the past three decades. The study of transplant recipient survivorship has become a cogent topic and post-transplant donor cell pathology warrants further study. Donor cell leukemia (DCL) is a rare but serious complication of allogeneic stem cell transplantation (SCT) where the recipient develops a form leukemia originating from the donor cells used for transplantation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435035"
                        open
                      >
                        <summary class="article-expander__title">The Mechanism of miR-155/miR-15b Axis Contributed to Apoptosis of CD34+ Cells by Upregulation of PD-L1 in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435035/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of heterogeneous myeloid clonal diseases that are characterized by ineffective bone marrow hematopoiesis. Since studies have confirmed the significance of miRNAs in ineffective hematopoiesis in MDS, the current report elucidated the mechanism mediated by miR-155-5p. The bone marrow of MDS patients was collected to detect miR-155-5p and to analyze the correlation between miR-155-5p and clinicopathological variables. Isolated bone marrow CD34+ cells were...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435889"
                        open
                      >
                        <summary class="article-expander__title">Clinical relevance of clonal hematopoiesis and its interference in cell-free DNA profiling of patients with gastric cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435889/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This study revealed the clinical impact of CH in Korean patients and suggests the potential for its interference in cfDNA tests.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435412"
                        open
                      >
                        <summary class="article-expander__title">Donor Cell Leukemia Following Allogeneic Hematopoietic Stem Cell Transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Approximately 25,000 allogeneic transplants are performed annually worldwide; a figure that has steadily increased over the past three decades. The study of transplant recipient survivorship has become a cogent topic and post-transplant donor cell pathology warrants further study. Donor cell leukemia (DCL) is a rare but serious complication of allogeneic stem cell transplantation (SCT) where the recipient develops a form leukemia originating from the donor cells used for transplantation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435035"
                        open
                      >
                        <summary class="article-expander__title">The Mechanism of miR-155/miR-15b Axis Contributed to Apoptosis of CD34+ Cells by Upregulation of PD-L1 in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435035/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of heterogeneous myeloid clonal diseases that are characterized by ineffective bone marrow hematopoiesis. Since studies have confirmed the significance of miRNAs in ineffective hematopoiesis in MDS, the current report elucidated the mechanism mediated by miR-155-5p. The bone marrow of MDS patients was collected to detect miR-155-5p and to analyze the correlation between miR-155-5p and clinicopathological variables. Isolated bone marrow CD34+ cells were...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435889"
                        open
                      >
                        <summary class="article-expander__title">Clinical relevance of clonal hematopoiesis and its interference in cell-free DNA profiling of patients with gastric cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435889/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This study revealed the clinical impact of CH in Korean patients and suggests the potential for its interference in cfDNA tests.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435412"
                        open
                      >
                        <summary class="article-expander__title">Donor Cell Leukemia Following Allogeneic Hematopoietic Stem Cell Transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Approximately 25,000 allogeneic transplants are performed annually worldwide; a figure that has steadily increased over the past three decades. The study of transplant recipient survivorship has become a cogent topic and post-transplant donor cell pathology warrants further study. Donor cell leukemia (DCL) is a rare but serious complication of allogeneic stem cell transplantation (SCT) where the recipient develops a form leukemia originating from the donor cells used for transplantation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37435035"
                        open
                      >
                        <summary class="article-expander__title">The Mechanism of miR-155/miR-15b Axis Contributed to Apoptosis of CD34+ Cells by Upregulation of PD-L1 in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37435035/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of heterogeneous myeloid clonal diseases that are characterized by ineffective bone marrow hematopoiesis. Since studies have confirmed the significance of miRNAs in ineffective hematopoiesis in MDS, the current report elucidated the mechanism mediated by miR-155-5p. The bone marrow of MDS patients was collected to detect miR-155-5p and to analyze the correlation between miR-155-5p and clinicopathological variables. Isolated bone marrow CD34+ cells were...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230726205538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230723210310&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230720210258&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37434969"
                        open
                      >
                        <summary class="article-expander__title">Case Report: Longitudinal monitoring of clonal evolution by circulating tumor DNA for resistance to anti-EGFR antibody in a case of metastatic colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37434969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In this report, tracking of ctDNA allowed us to describe clonal evolution in a case in which we observed genomic alterations in KRAS and NRAS in a patient who acquired resistance to anti-EGFR antibody drugs during treatment. It is reasonable to consider repeat molecular interrogation during progression in patients with mCRC by using ctDNA analysis, which could help to identify patients who may benefit from a rechallenge strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-07-10 datetime="2023-07-10T10:00:00.000Z">2023-07-10</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-07-10 datetime="2023-07-10T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37424322"
                        open
                      >
                        <summary class="article-expander__title">Integrated genomic and single-cell transcriptomic analyses reveal clonal evolution and immune signature in donor cell leukemia after haploidentical allogeneic hematopoietic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37424322/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230723210314&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The pathogenesis of donor cell leukemia (DCL) after allogeneic hematopoietic stem cell transplantation (allo-HSCT) is unclear and likely multifactorial. Leukemic transformation of healthy donor HSCs in recipient&#x27;s bone marrow microenvironment provides a useful in vivo model for investigating the mechanisms involved in leukemogenesis. Here, we report a rare case of late-onset DCL developing in a recipient. Whole-genome sequencing indicates that donor-derived cells harboring clonal hematopoiesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37424322"
                        open
                      >
                        <summary class="article-expander__title">Integrated genomic and single-cell transcriptomic analyses reveal clonal evolution and immune signature in donor cell leukemia after haploidentical allogeneic hematopoietic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37424322/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The pathogenesis of donor cell leukemia (DCL) after allogeneic hematopoietic stem cell transplantation (allo-HSCT) is unclear and likely multifactorial. Leukemic transformation of healthy donor HSCs in recipient&#x27;s bone marrow microenvironment provides a useful in vivo model for investigating the mechanisms involved in leukemogenesis. Here, we report a rare case of late-onset DCL developing in a recipient. Whole-genome sequencing indicates that donor-derived cells harboring clonal hematopoiesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37424322"
                        open
                      >
                        <summary class="article-expander__title">Integrated genomic and single-cell transcriptomic analyses reveal clonal evolution and immune signature in donor cell leukemia after haploidentical allogeneic hematopoietic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37424322/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The pathogenesis of donor cell leukemia (DCL) after allogeneic hematopoietic stem cell transplantation (allo-HSCT) is unclear and likely multifactorial. Leukemic transformation of healthy donor HSCs in recipient&#x27;s bone marrow microenvironment provides a useful in vivo model for investigating the mechanisms involved in leukemogenesis. Here, we report a rare case of late-onset DCL developing in a recipient. Whole-genome sequencing indicates that donor-derived cells harboring clonal hematopoiesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37424322"
                        open
                      >
                        <summary class="article-expander__title">Integrated genomic and single-cell transcriptomic analyses reveal clonal evolution and immune signature in donor cell leukemia after haploidentical allogeneic hematopoietic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37424322/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230720210259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The pathogenesis of donor cell leukemia (DCL) after allogeneic hematopoietic stem cell transplantation (allo-HSCT) is unclear and likely multifactorial. Leukemic transformation of healthy donor HSCs in recipient&#x27;s bone marrow microenvironment provides a useful in vivo model for investigating the mechanisms involved in leukemogenesis. Here, we report a rare case of late-onset DCL developing in a recipient. Whole-genome sequencing indicates that donor-derived cells harboring clonal hematopoiesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37424322"
                        open
                      >
                        <summary class="article-expander__title">Integrated genomic and single-cell transcriptomic analyses reveal clonal evolution and immune signature in donor cell leukemia after haploidentical allogeneic hematopoietic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37424322/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The pathogenesis of donor cell leukemia (DCL) after allogeneic hematopoietic stem cell transplantation (allo-HSCT) is unclear and likely multifactorial. Leukemic transformation of healthy donor HSCs in recipient&#x27;s bone marrow microenvironment provides a useful in vivo model for investigating the mechanisms involved in leukemogenesis. Here, we report a rare case of late-onset DCL developing in a recipient. Whole-genome sequencing indicates that donor-derived cells harboring clonal hematopoiesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37424322"
                        open
                      >
                        <summary class="article-expander__title">Integrated genomic and single-cell transcriptomic analyses reveal clonal evolution and immune signature in donor cell leukemia after haploidentical allogeneic hematopoietic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37424322/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The pathogenesis of donor cell leukemia (DCL) after allogeneic hematopoietic stem cell transplantation (allo-HSCT) is unclear and likely multifactorial. Leukemic transformation of healthy donor HSCs in recipient&#x27;s bone marrow microenvironment provides a useful in vivo model for investigating the mechanisms involved in leukemogenesis. Here, we report a rare case of late-onset DCL developing in a recipient. Whole-genome sequencing indicates that donor-derived cells harboring clonal hematopoiesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37424322"
                        open
                      >
                        <summary class="article-expander__title">Integrated genomic and single-cell transcriptomic analyses reveal clonal evolution and immune signature in donor cell leukemia after haploidentical allogeneic hematopoietic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37424322/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The pathogenesis of donor cell leukemia (DCL) after allogeneic hematopoietic stem cell transplantation (allo-HSCT) is unclear and likely multifactorial. Leukemic transformation of healthy donor HSCs in recipient&#x27;s bone marrow microenvironment provides a useful in vivo model for investigating the mechanisms involved in leukemogenesis. Here, we report a rare case of late-onset DCL developing in a recipient. Whole-genome sequencing indicates that donor-derived cells harboring clonal hematopoiesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37424322"
                        open
                      >
                        <summary class="article-expander__title">Integrated genomic and single-cell transcriptomic analyses reveal clonal evolution and immune signature in donor cell leukemia after haploidentical allogeneic hematopoietic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37424322/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The pathogenesis of donor cell leukemia (DCL) after allogeneic hematopoietic stem cell transplantation (allo-HSCT) is unclear and likely multifactorial. Leukemic transformation of healthy donor HSCs in recipient&#x27;s bone marrow microenvironment provides a useful in vivo model for investigating the mechanisms involved in leukemogenesis. Here, we report a rare case of late-onset DCL developing in a recipient. Whole-genome sequencing indicates that donor-derived cells harboring clonal hematopoiesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37424322"
                        open
                      >
                        <summary class="article-expander__title">Integrated genomic and single-cell transcriptomic analyses reveal clonal evolution and immune signature in donor cell leukemia after haploidentical allogeneic hematopoietic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37424322/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The pathogenesis of donor cell leukemia (DCL) after allogeneic hematopoietic stem cell transplantation (allo-HSCT) is unclear and likely multifactorial. Leukemic transformation of healthy donor HSCs in recipient&#x27;s bone marrow microenvironment provides a useful in vivo model for investigating the mechanisms involved in leukemogenesis. Here, we report a rare case of late-onset DCL developing in a recipient. Whole-genome sequencing indicates that donor-derived cells harboring clonal hematopoiesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37424322"
                        open
                      >
                        <summary class="article-expander__title">Integrated genomic and single-cell transcriptomic analyses reveal clonal evolution and immune signature in donor cell leukemia after haploidentical allogeneic hematopoietic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37424322/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The pathogenesis of donor cell leukemia (DCL) after allogeneic hematopoietic stem cell transplantation (allo-HSCT) is unclear and likely multifactorial. Leukemic transformation of healthy donor HSCs in recipient&#x27;s bone marrow microenvironment provides a useful in vivo model for investigating the mechanisms involved in leukemogenesis. Here, we report a rare case of late-onset DCL developing in a recipient. Whole-genome sequencing indicates that donor-derived cells harboring clonal hematopoiesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37424322"
                        open
                      >
                        <summary class="article-expander__title">Integrated genomic and single-cell transcriptomic analyses reveal clonal evolution and immune signature in donor cell leukemia after haploidentical allogeneic hematopoietic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37424322/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The pathogenesis of donor cell leukemia (DCL) after allogeneic hematopoietic stem cell transplantation (allo-HSCT) is unclear and likely multifactorial. Leukemic transformation of healthy donor HSCs in recipient&#x27;s bone marrow microenvironment provides a useful in vivo model for investigating the mechanisms involved in leukemogenesis. Here, we report a rare case of late-onset DCL developing in a recipient. Whole-genome sequencing indicates that donor-derived cells harboring clonal hematopoiesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37424322"
                        open
                      >
                        <summary class="article-expander__title">Integrated genomic and single-cell transcriptomic analyses reveal clonal evolution and immune signature in donor cell leukemia after haploidentical allogeneic hematopoietic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37424322/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The pathogenesis of donor cell leukemia (DCL) after allogeneic hematopoietic stem cell transplantation (allo-HSCT) is unclear and likely multifactorial. Leukemic transformation of healthy donor HSCs in recipient&#x27;s bone marrow microenvironment provides a useful in vivo model for investigating the mechanisms involved in leukemogenesis. Here, we report a rare case of late-onset DCL developing in a recipient. Whole-genome sequencing indicates that donor-derived cells harboring clonal hematopoiesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37424322"
                        open
                      >
                        <summary class="article-expander__title">Integrated genomic and single-cell transcriptomic analyses reveal clonal evolution and immune signature in donor cell leukemia after haploidentical allogeneic hematopoietic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37424322/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The pathogenesis of donor cell leukemia (DCL) after allogeneic hematopoietic stem cell transplantation (allo-HSCT) is unclear and likely multifactorial. Leukemic transformation of healthy donor HSCs in recipient&#x27;s bone marrow microenvironment provides a useful in vivo model for investigating the mechanisms involved in leukemogenesis. Here, we report a rare case of late-onset DCL developing in a recipient. Whole-genome sequencing indicates that donor-derived cells harboring clonal hematopoiesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37424322"
                        open
                      >
                        <summary class="article-expander__title">Integrated genomic and single-cell transcriptomic analyses reveal clonal evolution and immune signature in donor cell leukemia after haploidentical allogeneic hematopoietic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37424322/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The pathogenesis of donor cell leukemia (DCL) after allogeneic hematopoietic stem cell transplantation (allo-HSCT) is unclear and likely multifactorial. Leukemic transformation of healthy donor HSCs in recipient&#x27;s bone marrow microenvironment provides a useful in vivo model for investigating the mechanisms involved in leukemogenesis. Here, we report a rare case of late-onset DCL developing in a recipient. Whole-genome sequencing indicates that donor-derived cells harboring clonal hematopoiesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-07-09 datetime="2023-07-09T10:00:00.000Z">2023-07-09</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-07-09 datetime="2023-07-09T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37423830"
                        open
                      >
                        <summary class="article-expander__title">Pathophysiology of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37423830/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230723210314&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During aging, the onset of mutations at low frequency in hematopoietic cells or clonal hematopoiesis of indeterminate significance favors the evolution towards hemopathies such as myelodysplastic syndromes or acute leukemias, but also cardiovascular diseases and other pathologies. Acute or chronic inflammation related to age influences the clonal evolution and the immune response. Conversely, mutated hematopoietic cells create an inflammatory bone marrow environment facilitating their expansion....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37423830"
                        open
                      >
                        <summary class="article-expander__title">Pathophysiology of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37423830/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During aging, the onset of mutations at low frequency in hematopoietic cells or clonal hematopoiesis of indeterminate significance favors the evolution towards hemopathies such as myelodysplastic syndromes or acute leukemias, but also cardiovascular diseases and other pathologies. Acute or chronic inflammation related to age influences the clonal evolution and the immune response. Conversely, mutated hematopoietic cells create an inflammatory bone marrow environment facilitating their expansion....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37423830"
                        open
                      >
                        <summary class="article-expander__title">Pathophysiology of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37423830/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During aging, the onset of mutations at low frequency in hematopoietic cells or clonal hematopoiesis of indeterminate significance favors the evolution towards hemopathies such as myelodysplastic syndromes or acute leukemias, but also cardiovascular diseases and other pathologies. Acute or chronic inflammation related to age influences the clonal evolution and the immune response. Conversely, mutated hematopoietic cells create an inflammatory bone marrow environment facilitating their expansion....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37423830"
                        open
                      >
                        <summary class="article-expander__title">Pathophysiology of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37423830/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230720210259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During aging, the onset of mutations at low frequency in hematopoietic cells or clonal hematopoiesis of indeterminate significance favors the evolution towards hemopathies such as myelodysplastic syndromes or acute leukemias, but also cardiovascular diseases and other pathologies. Acute or chronic inflammation related to age influences the clonal evolution and the immune response. Conversely, mutated hematopoietic cells create an inflammatory bone marrow environment facilitating their expansion....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37423830"
                        open
                      >
                        <summary class="article-expander__title">Pathophysiology of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37423830/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During aging, the onset of mutations at low frequency in hematopoietic cells or clonal hematopoiesis of indeterminate significance favors the evolution towards hemopathies such as myelodysplastic syndromes or acute leukemias, but also cardiovascular diseases and other pathologies. Acute or chronic inflammation related to age influences the clonal evolution and the immune response. Conversely, mutated hematopoietic cells create an inflammatory bone marrow environment facilitating their expansion....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37423830"
                        open
                      >
                        <summary class="article-expander__title">Pathophysiology of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37423830/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During aging, the onset of mutations at low frequency in hematopoietic cells or clonal hematopoiesis of indeterminate significance favors the evolution towards hemopathies such as myelodysplastic syndromes or acute leukemias, but also cardiovascular diseases and other pathologies. Acute or chronic inflammation related to age influences the clonal evolution and the immune response. Conversely, mutated hematopoietic cells create an inflammatory bone marrow environment facilitating their expansion....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37423830"
                        open
                      >
                        <summary class="article-expander__title">Pathophysiology of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37423830/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During aging, the onset of mutations at low frequency in hematopoietic cells or clonal hematopoiesis of indeterminate significance favors the evolution towards hemopathies such as myelodysplastic syndromes or acute leukemias, but also cardiovascular diseases and other pathologies. Acute or chronic inflammation related to age influences the clonal evolution and the immune response. Conversely, mutated hematopoietic cells create an inflammatory bone marrow environment facilitating their expansion....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37423830"
                        open
                      >
                        <summary class="article-expander__title">Pathophysiology of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37423830/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During aging, the onset of mutations at low frequency in hematopoietic cells or clonal hematopoiesis of indeterminate significance favors the evolution towards hemopathies such as myelodysplastic syndromes or acute leukemias, but also cardiovascular diseases and other pathologies. Acute or chronic inflammation related to age influences the clonal evolution and the immune response. Conversely, mutated hematopoietic cells create an inflammatory bone marrow environment facilitating their expansion....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37423830"
                        open
                      >
                        <summary class="article-expander__title">Pathophysiology of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37423830/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During aging, the onset of mutations at low frequency in hematopoietic cells or clonal hematopoiesis of indeterminate significance favors the evolution towards hemopathies such as myelodysplastic syndromes or acute leukemias, but also cardiovascular diseases and other pathologies. Acute or chronic inflammation related to age influences the clonal evolution and the immune response. Conversely, mutated hematopoietic cells create an inflammatory bone marrow environment facilitating their expansion....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37423830"
                        open
                      >
                        <summary class="article-expander__title">Pathophysiology of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37423830/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During aging, the onset of mutations at low frequency in hematopoietic cells or clonal hematopoiesis of indeterminate significance favors the evolution towards hemopathies such as myelodysplastic syndromes or acute leukemias, but also cardiovascular diseases and other pathologies. Acute or chronic inflammation related to age influences the clonal evolution and the immune response. Conversely, mutated hematopoietic cells create an inflammatory bone marrow environment facilitating their expansion....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37423830"
                        open
                      >
                        <summary class="article-expander__title">Pathophysiology of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37423830/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During aging, the onset of mutations at low frequency in hematopoietic cells or clonal hematopoiesis of indeterminate significance favors the evolution towards hemopathies such as myelodysplastic syndromes or acute leukemias, but also cardiovascular diseases and other pathologies. Acute or chronic inflammation related to age influences the clonal evolution and the immune response. Conversely, mutated hematopoietic cells create an inflammatory bone marrow environment facilitating their expansion....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37423830"
                        open
                      >
                        <summary class="article-expander__title">Pathophysiology of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37423830/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During aging, the onset of mutations at low frequency in hematopoietic cells or clonal hematopoiesis of indeterminate significance favors the evolution towards hemopathies such as myelodysplastic syndromes or acute leukemias, but also cardiovascular diseases and other pathologies. Acute or chronic inflammation related to age influences the clonal evolution and the immune response. Conversely, mutated hematopoietic cells create an inflammatory bone marrow environment facilitating their expansion....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37423830"
                        open
                      >
                        <summary class="article-expander__title">Pathophysiology of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37423830/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During aging, the onset of mutations at low frequency in hematopoietic cells or clonal hematopoiesis of indeterminate significance favors the evolution towards hemopathies such as myelodysplastic syndromes or acute leukemias, but also cardiovascular diseases and other pathologies. Acute or chronic inflammation related to age influences the clonal evolution and the immune response. Conversely, mutated hematopoietic cells create an inflammatory bone marrow environment facilitating their expansion....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37423830"
                        open
                      >
                        <summary class="article-expander__title">Pathophysiology of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37423830/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During aging, the onset of mutations at low frequency in hematopoietic cells or clonal hematopoiesis of indeterminate significance favors the evolution towards hemopathies such as myelodysplastic syndromes or acute leukemias, but also cardiovascular diseases and other pathologies. Acute or chronic inflammation related to age influences the clonal evolution and the immune response. Conversely, mutated hematopoietic cells create an inflammatory bone marrow environment facilitating their expansion....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-07-08 datetime="2023-07-08T10:00:00.000Z">2023-07-08</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-07-08 datetime="2023-07-08T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422676"
                        open
                      >
                        <summary class="article-expander__title">Role of innate immunological/inflammatory pathways in myelodysplastic syndromes and AML: a narrative review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422676/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Dysregulation of the innate immune system and inflammatory-related pathways has been implicated in hematopoietic defects in the bone marrow microenvironment and associated with aging, clonal hematopoiesis, myelodysplastic syndromes (MDS), and acute myeloid leukemia (AML). As the innate immune system and its pathway regulators have been implicated in the pathogenesis of MDS/AML, novel approaches targeting these pathways have shown promising results. Variability in expression of Toll like...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422676"
                        open
                      >
                        <summary class="article-expander__title">Role of innate immunological/inflammatory pathways in myelodysplastic syndromes and AML: a narrative review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422676/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Dysregulation of the innate immune system and inflammatory-related pathways has been implicated in hematopoietic defects in the bone marrow microenvironment and associated with aging, clonal hematopoiesis, myelodysplastic syndromes (MDS), and acute myeloid leukemia (AML). As the innate immune system and its pathway regulators have been implicated in the pathogenesis of MDS/AML, novel approaches targeting these pathways have shown promising results. Variability in expression of Toll like...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422676"
                        open
                      >
                        <summary class="article-expander__title">Role of innate immunological/inflammatory pathways in myelodysplastic syndromes and AML: a narrative review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422676/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230720210259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Dysregulation of the innate immune system and inflammatory-related pathways has been implicated in hematopoietic defects in the bone marrow microenvironment and associated with aging, clonal hematopoiesis, myelodysplastic syndromes (MDS), and acute myeloid leukemia (AML). As the innate immune system and its pathway regulators have been implicated in the pathogenesis of MDS/AML, novel approaches targeting these pathways have shown promising results. Variability in expression of Toll like...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421681"
                        open
                      >
                        <summary class="article-expander__title">Germline predisposition to clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421681/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230720210259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We now recognize that with aging, hematopoietic stem and progenitor cells (HSPCs) acquire mutations that confer a fitness advantage and clonally expand in a process now termed clonal hematopoiesis (CH). Because CH predisposes to a variety of health problems, including cancers, cardiovascular diseases, and inflammatory conditions, there is intense interest in the inherited alleles associated with the development of CH. DNA variants near TERT, SMC4, KPNA4, IL12A, CD164, and ATM confer the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422676"
                        open
                      >
                        <summary class="article-expander__title">Role of innate immunological/inflammatory pathways in myelodysplastic syndromes and AML: a narrative review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422676/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Dysregulation of the innate immune system and inflammatory-related pathways has been implicated in hematopoietic defects in the bone marrow microenvironment and associated with aging, clonal hematopoiesis, myelodysplastic syndromes (MDS), and acute myeloid leukemia (AML). As the innate immune system and its pathway regulators have been implicated in the pathogenesis of MDS/AML, novel approaches targeting these pathways have shown promising results. Variability in expression of Toll like...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421681"
                        open
                      >
                        <summary class="article-expander__title">Germline predisposition to clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421681/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We now recognize that with aging, hematopoietic stem and progenitor cells (HSPCs) acquire mutations that confer a fitness advantage and clonally expand in a process now termed clonal hematopoiesis (CH). Because CH predisposes to a variety of health problems, including cancers, cardiovascular diseases, and inflammatory conditions, there is intense interest in the inherited alleles associated with the development of CH. DNA variants near TERT, SMC4, KPNA4, IL12A, CD164, and ATM confer the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422676"
                        open
                      >
                        <summary class="article-expander__title">Role of innate immunological/inflammatory pathways in myelodysplastic syndromes and AML: a narrative review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422676/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Dysregulation of the innate immune system and inflammatory-related pathways has been implicated in hematopoietic defects in the bone marrow microenvironment and associated with aging, clonal hematopoiesis, myelodysplastic syndromes (MDS), and acute myeloid leukemia (AML). As the innate immune system and its pathway regulators have been implicated in the pathogenesis of MDS/AML, novel approaches targeting these pathways have shown promising results. Variability in expression of Toll like...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421681"
                        open
                      >
                        <summary class="article-expander__title">Germline predisposition to clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421681/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We now recognize that with aging, hematopoietic stem and progenitor cells (HSPCs) acquire mutations that confer a fitness advantage and clonally expand in a process now termed clonal hematopoiesis (CH). Because CH predisposes to a variety of health problems, including cancers, cardiovascular diseases, and inflammatory conditions, there is intense interest in the inherited alleles associated with the development of CH. DNA variants near TERT, SMC4, KPNA4, IL12A, CD164, and ATM confer the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422676"
                        open
                      >
                        <summary class="article-expander__title">Role of innate immunological/inflammatory pathways in myelodysplastic syndromes and AML: a narrative review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422676/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Dysregulation of the innate immune system and inflammatory-related pathways has been implicated in hematopoietic defects in the bone marrow microenvironment and associated with aging, clonal hematopoiesis, myelodysplastic syndromes (MDS), and acute myeloid leukemia (AML). As the innate immune system and its pathway regulators have been implicated in the pathogenesis of MDS/AML, novel approaches targeting these pathways have shown promising results. Variability in expression of Toll like...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421681"
                        open
                      >
                        <summary class="article-expander__title">Germline predisposition to clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421681/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We now recognize that with aging, hematopoietic stem and progenitor cells (HSPCs) acquire mutations that confer a fitness advantage and clonally expand in a process now termed clonal hematopoiesis (CH). Because CH predisposes to a variety of health problems, including cancers, cardiovascular diseases, and inflammatory conditions, there is intense interest in the inherited alleles associated with the development of CH. DNA variants near TERT, SMC4, KPNA4, IL12A, CD164, and ATM confer the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422676"
                        open
                      >
                        <summary class="article-expander__title">Role of innate immunological/inflammatory pathways in myelodysplastic syndromes and AML: a narrative review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422676/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Dysregulation of the innate immune system and inflammatory-related pathways has been implicated in hematopoietic defects in the bone marrow microenvironment and associated with aging, clonal hematopoiesis, myelodysplastic syndromes (MDS), and acute myeloid leukemia (AML). As the innate immune system and its pathway regulators have been implicated in the pathogenesis of MDS/AML, novel approaches targeting these pathways have shown promising results. Variability in expression of Toll like...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421681"
                        open
                      >
                        <summary class="article-expander__title">Germline predisposition to clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421681/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We now recognize that with aging, hematopoietic stem and progenitor cells (HSPCs) acquire mutations that confer a fitness advantage and clonally expand in a process now termed clonal hematopoiesis (CH). Because CH predisposes to a variety of health problems, including cancers, cardiovascular diseases, and inflammatory conditions, there is intense interest in the inherited alleles associated with the development of CH. DNA variants near TERT, SMC4, KPNA4, IL12A, CD164, and ATM confer the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422676"
                        open
                      >
                        <summary class="article-expander__title">Role of innate immunological/inflammatory pathways in myelodysplastic syndromes and AML: a narrative review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422676/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Dysregulation of the innate immune system and inflammatory-related pathways has been implicated in hematopoietic defects in the bone marrow microenvironment and associated with aging, clonal hematopoiesis, myelodysplastic syndromes (MDS), and acute myeloid leukemia (AML). As the innate immune system and its pathway regulators have been implicated in the pathogenesis of MDS/AML, novel approaches targeting these pathways have shown promising results. Variability in expression of Toll like...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421681"
                        open
                      >
                        <summary class="article-expander__title">Germline predisposition to clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421681/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We now recognize that with aging, hematopoietic stem and progenitor cells (HSPCs) acquire mutations that confer a fitness advantage and clonally expand in a process now termed clonal hematopoiesis (CH). Because CH predisposes to a variety of health problems, including cancers, cardiovascular diseases, and inflammatory conditions, there is intense interest in the inherited alleles associated with the development of CH. DNA variants near TERT, SMC4, KPNA4, IL12A, CD164, and ATM confer the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422676"
                        open
                      >
                        <summary class="article-expander__title">Role of innate immunological/inflammatory pathways in myelodysplastic syndromes and AML: a narrative review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422676/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Dysregulation of the innate immune system and inflammatory-related pathways has been implicated in hematopoietic defects in the bone marrow microenvironment and associated with aging, clonal hematopoiesis, myelodysplastic syndromes (MDS), and acute myeloid leukemia (AML). As the innate immune system and its pathway regulators have been implicated in the pathogenesis of MDS/AML, novel approaches targeting these pathways have shown promising results. Variability in expression of Toll like...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421681"
                        open
                      >
                        <summary class="article-expander__title">Germline predisposition to clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421681/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We now recognize that with aging, hematopoietic stem and progenitor cells (HSPCs) acquire mutations that confer a fitness advantage and clonally expand in a process now termed clonal hematopoiesis (CH). Because CH predisposes to a variety of health problems, including cancers, cardiovascular diseases, and inflammatory conditions, there is intense interest in the inherited alleles associated with the development of CH. DNA variants near TERT, SMC4, KPNA4, IL12A, CD164, and ATM confer the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422676"
                        open
                      >
                        <summary class="article-expander__title">Role of innate immunological/inflammatory pathways in myelodysplastic syndromes and AML: a narrative review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422676/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Dysregulation of the innate immune system and inflammatory-related pathways has been implicated in hematopoietic defects in the bone marrow microenvironment and associated with aging, clonal hematopoiesis, myelodysplastic syndromes (MDS), and acute myeloid leukemia (AML). As the innate immune system and its pathway regulators have been implicated in the pathogenesis of MDS/AML, novel approaches targeting these pathways have shown promising results. Variability in expression of Toll like...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421681"
                        open
                      >
                        <summary class="article-expander__title">Germline predisposition to clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421681/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We now recognize that with aging, hematopoietic stem and progenitor cells (HSPCs) acquire mutations that confer a fitness advantage and clonally expand in a process now termed clonal hematopoiesis (CH). Because CH predisposes to a variety of health problems, including cancers, cardiovascular diseases, and inflammatory conditions, there is intense interest in the inherited alleles associated with the development of CH. DNA variants near TERT, SMC4, KPNA4, IL12A, CD164, and ATM confer the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422676"
                        open
                      >
                        <summary class="article-expander__title">Role of innate immunological/inflammatory pathways in myelodysplastic syndromes and AML: a narrative review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422676/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Dysregulation of the innate immune system and inflammatory-related pathways has been implicated in hematopoietic defects in the bone marrow microenvironment and associated with aging, clonal hematopoiesis, myelodysplastic syndromes (MDS), and acute myeloid leukemia (AML). As the innate immune system and its pathway regulators have been implicated in the pathogenesis of MDS/AML, novel approaches targeting these pathways have shown promising results. Variability in expression of Toll like...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421681"
                        open
                      >
                        <summary class="article-expander__title">Germline predisposition to clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421681/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We now recognize that with aging, hematopoietic stem and progenitor cells (HSPCs) acquire mutations that confer a fitness advantage and clonally expand in a process now termed clonal hematopoiesis (CH). Because CH predisposes to a variety of health problems, including cancers, cardiovascular diseases, and inflammatory conditions, there is intense interest in the inherited alleles associated with the development of CH. DNA variants near TERT, SMC4, KPNA4, IL12A, CD164, and ATM confer the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422676"
                        open
                      >
                        <summary class="article-expander__title">Role of innate immunological/inflammatory pathways in myelodysplastic syndromes and AML: a narrative review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422676/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Dysregulation of the innate immune system and inflammatory-related pathways has been implicated in hematopoietic defects in the bone marrow microenvironment and associated with aging, clonal hematopoiesis, myelodysplastic syndromes (MDS), and acute myeloid leukemia (AML). As the innate immune system and its pathway regulators have been implicated in the pathogenesis of MDS/AML, novel approaches targeting these pathways have shown promising results. Variability in expression of Toll like...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421681"
                        open
                      >
                        <summary class="article-expander__title">Germline predisposition to clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421681/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We now recognize that with aging, hematopoietic stem and progenitor cells (HSPCs) acquire mutations that confer a fitness advantage and clonally expand in a process now termed clonal hematopoiesis (CH). Because CH predisposes to a variety of health problems, including cancers, cardiovascular diseases, and inflammatory conditions, there is intense interest in the inherited alleles associated with the development of CH. DNA variants near TERT, SMC4, KPNA4, IL12A, CD164, and ATM confer the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422676"
                        open
                      >
                        <summary class="article-expander__title">Role of innate immunological/inflammatory pathways in myelodysplastic syndromes and AML: a narrative review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422676/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Dysregulation of the innate immune system and inflammatory-related pathways has been implicated in hematopoietic defects in the bone marrow microenvironment and associated with aging, clonal hematopoiesis, myelodysplastic syndromes (MDS), and acute myeloid leukemia (AML). As the innate immune system and its pathway regulators have been implicated in the pathogenesis of MDS/AML, novel approaches targeting these pathways have shown promising results. Variability in expression of Toll like...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421681"
                        open
                      >
                        <summary class="article-expander__title">Germline predisposition to clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421681/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We now recognize that with aging, hematopoietic stem and progenitor cells (HSPCs) acquire mutations that confer a fitness advantage and clonally expand in a process now termed clonal hematopoiesis (CH). Because CH predisposes to a variety of health problems, including cancers, cardiovascular diseases, and inflammatory conditions, there is intense interest in the inherited alleles associated with the development of CH. DNA variants near TERT, SMC4, KPNA4, IL12A, CD164, and ATM confer the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422676"
                        open
                      >
                        <summary class="article-expander__title">Role of innate immunological/inflammatory pathways in myelodysplastic syndromes and AML: a narrative review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422676/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Dysregulation of the innate immune system and inflammatory-related pathways has been implicated in hematopoietic defects in the bone marrow microenvironment and associated with aging, clonal hematopoiesis, myelodysplastic syndromes (MDS), and acute myeloid leukemia (AML). As the innate immune system and its pathway regulators have been implicated in the pathogenesis of MDS/AML, novel approaches targeting these pathways have shown promising results. Variability in expression of Toll like...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421681"
                        open
                      >
                        <summary class="article-expander__title">Germline predisposition to clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421681/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We now recognize that with aging, hematopoietic stem and progenitor cells (HSPCs) acquire mutations that confer a fitness advantage and clonally expand in a process now termed clonal hematopoiesis (CH). Because CH predisposes to a variety of health problems, including cancers, cardiovascular diseases, and inflammatory conditions, there is intense interest in the inherited alleles associated with the development of CH. DNA variants near TERT, SMC4, KPNA4, IL12A, CD164, and ATM confer the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421681"
                        open
                      >
                        <summary class="article-expander__title">Germline predisposition to clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421681/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We now recognize that with aging, hematopoietic stem and progenitor cells (HSPCs) acquire mutations that confer a fitness advantage and clonally expand in a process now termed clonal hematopoiesis (CH). Because CH predisposes to a variety of health problems, including cancers, cardiovascular diseases, and inflammatory conditions, there is intense interest in the inherited alleles associated with the development of CH. DNA variants near TERT, SMC4, KPNA4, IL12A, CD164, and ATM confer the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230726205538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230726205538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230723210310&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230723210310&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230720210258&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230720210258&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37422528"
                        open
                      >
                        <summary class="article-expander__title">Genomic and evolutionary characteristics of metastatic gastric cancer by routes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37422528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic characteristics of metastatic gastric cancer is distinctive by routes and associated with patients&#x27; prognosis along with genomic evolution pattenrs, indicating that both primary and metastatic gastric cancers require genomic evaluation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37421230"
                        open
                      >
                        <summary class="article-expander__title">Gene fusions during the early evolution of mesothelioma correlate with impaired DNA repair and Hippo pathways</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37421230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Malignant pleural mesothelioma (MPM), a rare cancer a long latency period (up to 40 years) between asbestos exposure and disease presentation. The mechanisms coupling asbestos to recurrent somatic alterations are poorly defined. Gene fusions arising through genomic instability may create novel drivers during early MPM evolution. We explored the gene fusions that occurred early in the evolutionary history of the tumor. We conducted multiregional whole exome sequencing (WES) of 106 samples from 20...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-07-07 datetime="2023-07-07T10:00:00.000Z">2023-07-07</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-07-07 datetime="2023-07-07T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230809205914&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230808205751&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230807205649&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230806205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230805205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230726205538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230723210310&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230720210258&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37419716"
                        open
                      >
                        <summary class="article-expander__title">Clinical applications of circulating tumor DNA in indolent B-cell lymphomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37419716/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Indolent B-cell lymphomas are generally incurable with standard therapy and most patients have a prolonged disease course that includes multiple treatments and periods of time in which they do not require therapy. Currently available tools to monitor disease burden and define response to treatment rely heavily on imaging scans that lack tumor specificity are unable to detect disease at the molecular level. Circulating tumor DNA (ctDNA) is a versatile and promising biomarker being developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-07-06 datetime="2023-07-06T10:00:00.000Z">2023-07-06</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-07-06 datetime="2023-07-06T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37410230"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing Phylogenetic Relationship in Bladder Cancer: A Methodological Overview</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37410230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bladder cancer (BC) expresses itself as a highly heterogeneous disease both at the histological and molecular level, often occurring as synchronous or metachronous multifocal disease with high risk of recurrence and potential to metastasize. Multiple sequencing studies focusing on both non-muscle-invasive bladder cancer (NMIBC) and muscle-invasive bladder cancer (MIBC) gave insights into the extent of both inter- and intrapatient heterogeneity, but many questions on clonal evolution in BC remain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37410230"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing Phylogenetic Relationship in Bladder Cancer: A Methodological Overview</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37410230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bladder cancer (BC) expresses itself as a highly heterogeneous disease both at the histological and molecular level, often occurring as synchronous or metachronous multifocal disease with high risk of recurrence and potential to metastasize. Multiple sequencing studies focusing on both non-muscle-invasive bladder cancer (NMIBC) and muscle-invasive bladder cancer (MIBC) gave insights into the extent of both inter- and intrapatient heterogeneity, but many questions on clonal evolution in BC remain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37410230"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing Phylogenetic Relationship in Bladder Cancer: A Methodological Overview</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37410230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bladder cancer (BC) expresses itself as a highly heterogeneous disease both at the histological and molecular level, often occurring as synchronous or metachronous multifocal disease with high risk of recurrence and potential to metastasize. Multiple sequencing studies focusing on both non-muscle-invasive bladder cancer (NMIBC) and muscle-invasive bladder cancer (MIBC) gave insights into the extent of both inter- and intrapatient heterogeneity, but many questions on clonal evolution in BC remain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37410230"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing Phylogenetic Relationship in Bladder Cancer: A Methodological Overview</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37410230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bladder cancer (BC) expresses itself as a highly heterogeneous disease both at the histological and molecular level, often occurring as synchronous or metachronous multifocal disease with high risk of recurrence and potential to metastasize. Multiple sequencing studies focusing on both non-muscle-invasive bladder cancer (NMIBC) and muscle-invasive bladder cancer (MIBC) gave insights into the extent of both inter- and intrapatient heterogeneity, but many questions on clonal evolution in BC remain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37410230"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing Phylogenetic Relationship in Bladder Cancer: A Methodological Overview</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37410230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bladder cancer (BC) expresses itself as a highly heterogeneous disease both at the histological and molecular level, often occurring as synchronous or metachronous multifocal disease with high risk of recurrence and potential to metastasize. Multiple sequencing studies focusing on both non-muscle-invasive bladder cancer (NMIBC) and muscle-invasive bladder cancer (MIBC) gave insights into the extent of both inter- and intrapatient heterogeneity, but many questions on clonal evolution in BC remain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37410230"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing Phylogenetic Relationship in Bladder Cancer: A Methodological Overview</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37410230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bladder cancer (BC) expresses itself as a highly heterogeneous disease both at the histological and molecular level, often occurring as synchronous or metachronous multifocal disease with high risk of recurrence and potential to metastasize. Multiple sequencing studies focusing on both non-muscle-invasive bladder cancer (NMIBC) and muscle-invasive bladder cancer (MIBC) gave insights into the extent of both inter- and intrapatient heterogeneity, but many questions on clonal evolution in BC remain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37410230"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing Phylogenetic Relationship in Bladder Cancer: A Methodological Overview</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37410230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bladder cancer (BC) expresses itself as a highly heterogeneous disease both at the histological and molecular level, often occurring as synchronous or metachronous multifocal disease with high risk of recurrence and potential to metastasize. Multiple sequencing studies focusing on both non-muscle-invasive bladder cancer (NMIBC) and muscle-invasive bladder cancer (MIBC) gave insights into the extent of both inter- and intrapatient heterogeneity, but many questions on clonal evolution in BC remain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37410230"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing Phylogenetic Relationship in Bladder Cancer: A Methodological Overview</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37410230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bladder cancer (BC) expresses itself as a highly heterogeneous disease both at the histological and molecular level, often occurring as synchronous or metachronous multifocal disease with high risk of recurrence and potential to metastasize. Multiple sequencing studies focusing on both non-muscle-invasive bladder cancer (NMIBC) and muscle-invasive bladder cancer (MIBC) gave insights into the extent of both inter- and intrapatient heterogeneity, but many questions on clonal evolution in BC remain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37410230"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing Phylogenetic Relationship in Bladder Cancer: A Methodological Overview</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37410230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bladder cancer (BC) expresses itself as a highly heterogeneous disease both at the histological and molecular level, often occurring as synchronous or metachronous multifocal disease with high risk of recurrence and potential to metastasize. Multiple sequencing studies focusing on both non-muscle-invasive bladder cancer (NMIBC) and muscle-invasive bladder cancer (MIBC) gave insights into the extent of both inter- and intrapatient heterogeneity, but many questions on clonal evolution in BC remain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37410230"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing Phylogenetic Relationship in Bladder Cancer: A Methodological Overview</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37410230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230726205538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bladder cancer (BC) expresses itself as a highly heterogeneous disease both at the histological and molecular level, often occurring as synchronous or metachronous multifocal disease with high risk of recurrence and potential to metastasize. Multiple sequencing studies focusing on both non-muscle-invasive bladder cancer (NMIBC) and muscle-invasive bladder cancer (MIBC) gave insights into the extent of both inter- and intrapatient heterogeneity, but many questions on clonal evolution in BC remain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37410230"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing Phylogenetic Relationship in Bladder Cancer: A Methodological Overview</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37410230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bladder cancer (BC) expresses itself as a highly heterogeneous disease both at the histological and molecular level, often occurring as synchronous or metachronous multifocal disease with high risk of recurrence and potential to metastasize. Multiple sequencing studies focusing on both non-muscle-invasive bladder cancer (NMIBC) and muscle-invasive bladder cancer (MIBC) gave insights into the extent of both inter- and intrapatient heterogeneity, but many questions on clonal evolution in BC remain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37410230"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing Phylogenetic Relationship in Bladder Cancer: A Methodological Overview</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37410230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bladder cancer (BC) expresses itself as a highly heterogeneous disease both at the histological and molecular level, often occurring as synchronous or metachronous multifocal disease with high risk of recurrence and potential to metastasize. Multiple sequencing studies focusing on both non-muscle-invasive bladder cancer (NMIBC) and muscle-invasive bladder cancer (MIBC) gave insights into the extent of both inter- and intrapatient heterogeneity, but many questions on clonal evolution in BC remain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37410230"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing Phylogenetic Relationship in Bladder Cancer: A Methodological Overview</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37410230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230723210310&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bladder cancer (BC) expresses itself as a highly heterogeneous disease both at the histological and molecular level, often occurring as synchronous or metachronous multifocal disease with high risk of recurrence and potential to metastasize. Multiple sequencing studies focusing on both non-muscle-invasive bladder cancer (NMIBC) and muscle-invasive bladder cancer (MIBC) gave insights into the extent of both inter- and intrapatient heterogeneity, but many questions on clonal evolution in BC remain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37410230"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing Phylogenetic Relationship in Bladder Cancer: A Methodological Overview</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37410230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bladder cancer (BC) expresses itself as a highly heterogeneous disease both at the histological and molecular level, often occurring as synchronous or metachronous multifocal disease with high risk of recurrence and potential to metastasize. Multiple sequencing studies focusing on both non-muscle-invasive bladder cancer (NMIBC) and muscle-invasive bladder cancer (MIBC) gave insights into the extent of both inter- and intrapatient heterogeneity, but many questions on clonal evolution in BC remain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37410230"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing Phylogenetic Relationship in Bladder Cancer: A Methodological Overview</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37410230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bladder cancer (BC) expresses itself as a highly heterogeneous disease both at the histological and molecular level, often occurring as synchronous or metachronous multifocal disease with high risk of recurrence and potential to metastasize. Multiple sequencing studies focusing on both non-muscle-invasive bladder cancer (NMIBC) and muscle-invasive bladder cancer (MIBC) gave insights into the extent of both inter- and intrapatient heterogeneity, but many questions on clonal evolution in BC remain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37410230"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing Phylogenetic Relationship in Bladder Cancer: A Methodological Overview</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37410230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230720210258&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bladder cancer (BC) expresses itself as a highly heterogeneous disease both at the histological and molecular level, often occurring as synchronous or metachronous multifocal disease with high risk of recurrence and potential to metastasize. Multiple sequencing studies focusing on both non-muscle-invasive bladder cancer (NMIBC) and muscle-invasive bladder cancer (MIBC) gave insights into the extent of both inter- and intrapatient heterogeneity, but many questions on clonal evolution in BC remain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37410230"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing Phylogenetic Relationship in Bladder Cancer: A Methodological Overview</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37410230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bladder cancer (BC) expresses itself as a highly heterogeneous disease both at the histological and molecular level, often occurring as synchronous or metachronous multifocal disease with high risk of recurrence and potential to metastasize. Multiple sequencing studies focusing on both non-muscle-invasive bladder cancer (NMIBC) and muscle-invasive bladder cancer (MIBC) gave insights into the extent of both inter- and intrapatient heterogeneity, but many questions on clonal evolution in BC remain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37410230"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing Phylogenetic Relationship in Bladder Cancer: A Methodological Overview</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37410230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bladder cancer (BC) expresses itself as a highly heterogeneous disease both at the histological and molecular level, often occurring as synchronous or metachronous multifocal disease with high risk of recurrence and potential to metastasize. Multiple sequencing studies focusing on both non-muscle-invasive bladder cancer (NMIBC) and muscle-invasive bladder cancer (MIBC) gave insights into the extent of both inter- and intrapatient heterogeneity, but many questions on clonal evolution in BC remain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37410230"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing Phylogenetic Relationship in Bladder Cancer: A Methodological Overview</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37410230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bladder cancer (BC) expresses itself as a highly heterogeneous disease both at the histological and molecular level, often occurring as synchronous or metachronous multifocal disease with high risk of recurrence and potential to metastasize. Multiple sequencing studies focusing on both non-muscle-invasive bladder cancer (NMIBC) and muscle-invasive bladder cancer (MIBC) gave insights into the extent of both inter- and intrapatient heterogeneity, but many questions on clonal evolution in BC remain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37410230"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing Phylogenetic Relationship in Bladder Cancer: A Methodological Overview</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37410230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bladder cancer (BC) expresses itself as a highly heterogeneous disease both at the histological and molecular level, often occurring as synchronous or metachronous multifocal disease with high risk of recurrence and potential to metastasize. Multiple sequencing studies focusing on both non-muscle-invasive bladder cancer (NMIBC) and muscle-invasive bladder cancer (MIBC) gave insights into the extent of both inter- and intrapatient heterogeneity, but many questions on clonal evolution in BC remain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37410230"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing Phylogenetic Relationship in Bladder Cancer: A Methodological Overview</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37410230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bladder cancer (BC) expresses itself as a highly heterogeneous disease both at the histological and molecular level, often occurring as synchronous or metachronous multifocal disease with high risk of recurrence and potential to metastasize. Multiple sequencing studies focusing on both non-muscle-invasive bladder cancer (NMIBC) and muscle-invasive bladder cancer (MIBC) gave insights into the extent of both inter- and intrapatient heterogeneity, but many questions on clonal evolution in BC remain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37410230"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing Phylogenetic Relationship in Bladder Cancer: A Methodological Overview</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37410230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bladder cancer (BC) expresses itself as a highly heterogeneous disease both at the histological and molecular level, often occurring as synchronous or metachronous multifocal disease with high risk of recurrence and potential to metastasize. Multiple sequencing studies focusing on both non-muscle-invasive bladder cancer (NMIBC) and muscle-invasive bladder cancer (MIBC) gave insights into the extent of both inter- and intrapatient heterogeneity, but many questions on clonal evolution in BC remain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37410230"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing Phylogenetic Relationship in Bladder Cancer: A Methodological Overview</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37410230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bladder cancer (BC) expresses itself as a highly heterogeneous disease both at the histological and molecular level, often occurring as synchronous or metachronous multifocal disease with high risk of recurrence and potential to metastasize. Multiple sequencing studies focusing on both non-muscle-invasive bladder cancer (NMIBC) and muscle-invasive bladder cancer (MIBC) gave insights into the extent of both inter- and intrapatient heterogeneity, but many questions on clonal evolution in BC remain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37410230"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing Phylogenetic Relationship in Bladder Cancer: A Methodological Overview</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37410230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bladder cancer (BC) expresses itself as a highly heterogeneous disease both at the histological and molecular level, often occurring as synchronous or metachronous multifocal disease with high risk of recurrence and potential to metastasize. Multiple sequencing studies focusing on both non-muscle-invasive bladder cancer (NMIBC) and muscle-invasive bladder cancer (MIBC) gave insights into the extent of both inter- and intrapatient heterogeneity, but many questions on clonal evolution in BC remain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37410230"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing Phylogenetic Relationship in Bladder Cancer: A Methodological Overview</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37410230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bladder cancer (BC) expresses itself as a highly heterogeneous disease both at the histological and molecular level, often occurring as synchronous or metachronous multifocal disease with high risk of recurrence and potential to metastasize. Multiple sequencing studies focusing on both non-muscle-invasive bladder cancer (NMIBC) and muscle-invasive bladder cancer (MIBC) gave insights into the extent of both inter- and intrapatient heterogeneity, but many questions on clonal evolution in BC remain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37410230"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing Phylogenetic Relationship in Bladder Cancer: A Methodological Overview</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37410230/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bladder cancer (BC) expresses itself as a highly heterogeneous disease both at the histological and molecular level, often occurring as synchronous or metachronous multifocal disease with high risk of recurrence and potential to metastasize. Multiple sequencing studies focusing on both non-muscle-invasive bladder cancer (NMIBC) and muscle-invasive bladder cancer (MIBC) gave insights into the extent of both inter- and intrapatient heterogeneity, but many questions on clonal evolution in BC remain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-07-05 datetime="2023-07-05T10:00:00.000Z">2023-07-05</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-07-05 datetime="2023-07-05T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37407470"
                        open
                      >
                        <summary class="article-expander__title">Clinical significance of clonal hematopoiesis and disease boundaries in bone marrow failure diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37407470/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230720210259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) is a non-neoplastic bone marrow failure syndrome caused by the destruction of hematopoietic stem and progenitor cells by the immune system. However, in some cases of AA, a small number of specific clones with gene mutations are observed without clinical manifestations. Cases with mutated PIG-A, BCOR/BCORL1, or HLA class I allele clones respond better to immunosuppressive therapies (ISTs). Cases with MDS-related clones, such as DNMT3A or ASXL1 mutations, are at a higher risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37407470"
                        open
                      >
                        <summary class="article-expander__title">Clinical significance of clonal hematopoiesis and disease boundaries in bone marrow failure diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37407470/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) is a non-neoplastic bone marrow failure syndrome caused by the destruction of hematopoietic stem and progenitor cells by the immune system. However, in some cases of AA, a small number of specific clones with gene mutations are observed without clinical manifestations. Cases with mutated PIG-A, BCOR/BCORL1, or HLA class I allele clones respond better to immunosuppressive therapies (ISTs). Cases with MDS-related clones, such as DNMT3A or ASXL1 mutations, are at a higher risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37406166"
                        open
                      >
                        <summary class="article-expander__title">Somatic mutational landscape of hereditary hematopoietic malignancies caused by germ line RUNX1, GATA2, and DDX41 variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37406166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with germline variants associated with hereditary hematopoietic malignancies (HHMs) have a highly variable risk for leukemogenesis. Gaps in our understanding of pre-malignant states in HHMs have hampered efforts to design effective clinical surveillance programs, provide personalized pre-emptive treatments and inform appropriate counselling for patients. We used the largest known comparative international cohort of germline RUNX1, GATA2, or DDX41 variant carriers without and with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37407470"
                        open
                      >
                        <summary class="article-expander__title">Clinical significance of clonal hematopoiesis and disease boundaries in bone marrow failure diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37407470/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) is a non-neoplastic bone marrow failure syndrome caused by the destruction of hematopoietic stem and progenitor cells by the immune system. However, in some cases of AA, a small number of specific clones with gene mutations are observed without clinical manifestations. Cases with mutated PIG-A, BCOR/BCORL1, or HLA class I allele clones respond better to immunosuppressive therapies (ISTs). Cases with MDS-related clones, such as DNMT3A or ASXL1 mutations, are at a higher risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37406166"
                        open
                      >
                        <summary class="article-expander__title">Somatic mutational landscape of hereditary hematopoietic malignancies caused by germ line RUNX1, GATA2, and DDX41 variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37406166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with germline variants associated with hereditary hematopoietic malignancies (HHMs) have a highly variable risk for leukemogenesis. Gaps in our understanding of pre-malignant states in HHMs have hampered efforts to design effective clinical surveillance programs, provide personalized pre-emptive treatments and inform appropriate counselling for patients. We used the largest known comparative international cohort of germline RUNX1, GATA2, or DDX41 variant carriers without and with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37407470"
                        open
                      >
                        <summary class="article-expander__title">Clinical significance of clonal hematopoiesis and disease boundaries in bone marrow failure diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37407470/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) is a non-neoplastic bone marrow failure syndrome caused by the destruction of hematopoietic stem and progenitor cells by the immune system. However, in some cases of AA, a small number of specific clones with gene mutations are observed without clinical manifestations. Cases with mutated PIG-A, BCOR/BCORL1, or HLA class I allele clones respond better to immunosuppressive therapies (ISTs). Cases with MDS-related clones, such as DNMT3A or ASXL1 mutations, are at a higher risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37406166"
                        open
                      >
                        <summary class="article-expander__title">Somatic mutational landscape of hereditary hematopoietic malignancies caused by germ line RUNX1, GATA2, and DDX41 variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37406166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with germline variants associated with hereditary hematopoietic malignancies (HHMs) have a highly variable risk for leukemogenesis. Gaps in our understanding of pre-malignant states in HHMs have hampered efforts to design effective clinical surveillance programs, provide personalized pre-emptive treatments and inform appropriate counselling for patients. We used the largest known comparative international cohort of germline RUNX1, GATA2, or DDX41 variant carriers without and with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37407470"
                        open
                      >
                        <summary class="article-expander__title">Clinical significance of clonal hematopoiesis and disease boundaries in bone marrow failure diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37407470/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) is a non-neoplastic bone marrow failure syndrome caused by the destruction of hematopoietic stem and progenitor cells by the immune system. However, in some cases of AA, a small number of specific clones with gene mutations are observed without clinical manifestations. Cases with mutated PIG-A, BCOR/BCORL1, or HLA class I allele clones respond better to immunosuppressive therapies (ISTs). Cases with MDS-related clones, such as DNMT3A or ASXL1 mutations, are at a higher risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37406166"
                        open
                      >
                        <summary class="article-expander__title">Somatic mutational landscape of hereditary hematopoietic malignancies caused by germ line RUNX1, GATA2, and DDX41 variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37406166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with germline variants associated with hereditary hematopoietic malignancies (HHMs) have a highly variable risk for leukemogenesis. Gaps in our understanding of pre-malignant states in HHMs have hampered efforts to design effective clinical surveillance programs, provide personalized pre-emptive treatments and inform appropriate counselling for patients. We used the largest known comparative international cohort of germline RUNX1, GATA2, or DDX41 variant carriers without and with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37407470"
                        open
                      >
                        <summary class="article-expander__title">Clinical significance of clonal hematopoiesis and disease boundaries in bone marrow failure diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37407470/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) is a non-neoplastic bone marrow failure syndrome caused by the destruction of hematopoietic stem and progenitor cells by the immune system. However, in some cases of AA, a small number of specific clones with gene mutations are observed without clinical manifestations. Cases with mutated PIG-A, BCOR/BCORL1, or HLA class I allele clones respond better to immunosuppressive therapies (ISTs). Cases with MDS-related clones, such as DNMT3A or ASXL1 mutations, are at a higher risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37406166"
                        open
                      >
                        <summary class="article-expander__title">Somatic mutational landscape of hereditary hematopoietic malignancies caused by germ line RUNX1, GATA2, and DDX41 variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37406166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with germline variants associated with hereditary hematopoietic malignancies (HHMs) have a highly variable risk for leukemogenesis. Gaps in our understanding of pre-malignant states in HHMs have hampered efforts to design effective clinical surveillance programs, provide personalized pre-emptive treatments and inform appropriate counselling for patients. We used the largest known comparative international cohort of germline RUNX1, GATA2, or DDX41 variant carriers without and with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37407470"
                        open
                      >
                        <summary class="article-expander__title">Clinical significance of clonal hematopoiesis and disease boundaries in bone marrow failure diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37407470/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) is a non-neoplastic bone marrow failure syndrome caused by the destruction of hematopoietic stem and progenitor cells by the immune system. However, in some cases of AA, a small number of specific clones with gene mutations are observed without clinical manifestations. Cases with mutated PIG-A, BCOR/BCORL1, or HLA class I allele clones respond better to immunosuppressive therapies (ISTs). Cases with MDS-related clones, such as DNMT3A or ASXL1 mutations, are at a higher risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37406166"
                        open
                      >
                        <summary class="article-expander__title">Somatic mutational landscape of hereditary hematopoietic malignancies caused by germ line RUNX1, GATA2, and DDX41 variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37406166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with germline variants associated with hereditary hematopoietic malignancies (HHMs) have a highly variable risk for leukemogenesis. Gaps in our understanding of pre-malignant states in HHMs have hampered efforts to design effective clinical surveillance programs, provide personalized pre-emptive treatments and inform appropriate counselling for patients. We used the largest known comparative international cohort of germline RUNX1, GATA2, or DDX41 variant carriers without and with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37407470"
                        open
                      >
                        <summary class="article-expander__title">Clinical significance of clonal hematopoiesis and disease boundaries in bone marrow failure diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37407470/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) is a non-neoplastic bone marrow failure syndrome caused by the destruction of hematopoietic stem and progenitor cells by the immune system. However, in some cases of AA, a small number of specific clones with gene mutations are observed without clinical manifestations. Cases with mutated PIG-A, BCOR/BCORL1, or HLA class I allele clones respond better to immunosuppressive therapies (ISTs). Cases with MDS-related clones, such as DNMT3A or ASXL1 mutations, are at a higher risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37406166"
                        open
                      >
                        <summary class="article-expander__title">Somatic mutational landscape of hereditary hematopoietic malignancies caused by germ line RUNX1, GATA2, and DDX41 variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37406166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with germline variants associated with hereditary hematopoietic malignancies (HHMs) have a highly variable risk for leukemogenesis. Gaps in our understanding of pre-malignant states in HHMs have hampered efforts to design effective clinical surveillance programs, provide personalized pre-emptive treatments and inform appropriate counselling for patients. We used the largest known comparative international cohort of germline RUNX1, GATA2, or DDX41 variant carriers without and with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37407470"
                        open
                      >
                        <summary class="article-expander__title">Clinical significance of clonal hematopoiesis and disease boundaries in bone marrow failure diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37407470/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) is a non-neoplastic bone marrow failure syndrome caused by the destruction of hematopoietic stem and progenitor cells by the immune system. However, in some cases of AA, a small number of specific clones with gene mutations are observed without clinical manifestations. Cases with mutated PIG-A, BCOR/BCORL1, or HLA class I allele clones respond better to immunosuppressive therapies (ISTs). Cases with MDS-related clones, such as DNMT3A or ASXL1 mutations, are at a higher risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37406166"
                        open
                      >
                        <summary class="article-expander__title">Somatic mutational landscape of hereditary hematopoietic malignancies caused by germ line RUNX1, GATA2, and DDX41 variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37406166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with germline variants associated with hereditary hematopoietic malignancies (HHMs) have a highly variable risk for leukemogenesis. Gaps in our understanding of pre-malignant states in HHMs have hampered efforts to design effective clinical surveillance programs, provide personalized pre-emptive treatments and inform appropriate counselling for patients. We used the largest known comparative international cohort of germline RUNX1, GATA2, or DDX41 variant carriers without and with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37407470"
                        open
                      >
                        <summary class="article-expander__title">Clinical significance of clonal hematopoiesis and disease boundaries in bone marrow failure diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37407470/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) is a non-neoplastic bone marrow failure syndrome caused by the destruction of hematopoietic stem and progenitor cells by the immune system. However, in some cases of AA, a small number of specific clones with gene mutations are observed without clinical manifestations. Cases with mutated PIG-A, BCOR/BCORL1, or HLA class I allele clones respond better to immunosuppressive therapies (ISTs). Cases with MDS-related clones, such as DNMT3A or ASXL1 mutations, are at a higher risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37406166"
                        open
                      >
                        <summary class="article-expander__title">Somatic mutational landscape of hereditary hematopoietic malignancies caused by germ line RUNX1, GATA2, and DDX41 variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37406166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with germline variants associated with hereditary hematopoietic malignancies (HHMs) have a highly variable risk for leukemogenesis. Gaps in our understanding of pre-malignant states in HHMs have hampered efforts to design effective clinical surveillance programs, provide personalized pre-emptive treatments and inform appropriate counselling for patients. We used the largest known comparative international cohort of germline RUNX1, GATA2, or DDX41 variant carriers without and with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37407470"
                        open
                      >
                        <summary class="article-expander__title">Clinical significance of clonal hematopoiesis and disease boundaries in bone marrow failure diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37407470/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) is a non-neoplastic bone marrow failure syndrome caused by the destruction of hematopoietic stem and progenitor cells by the immune system. However, in some cases of AA, a small number of specific clones with gene mutations are observed without clinical manifestations. Cases with mutated PIG-A, BCOR/BCORL1, or HLA class I allele clones respond better to immunosuppressive therapies (ISTs). Cases with MDS-related clones, such as DNMT3A or ASXL1 mutations, are at a higher risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37406166"
                        open
                      >
                        <summary class="article-expander__title">Somatic mutational landscape of hereditary hematopoietic malignancies caused by germ line RUNX1, GATA2, and DDX41 variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37406166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with germline variants associated with hereditary hematopoietic malignancies (HHMs) have a highly variable risk for leukemogenesis. Gaps in our understanding of pre-malignant states in HHMs have hampered efforts to design effective clinical surveillance programs, provide personalized pre-emptive treatments and inform appropriate counselling for patients. We used the largest known comparative international cohort of germline RUNX1, GATA2, or DDX41 variant carriers without and with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37407470"
                        open
                      >
                        <summary class="article-expander__title">Clinical significance of clonal hematopoiesis and disease boundaries in bone marrow failure diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37407470/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) is a non-neoplastic bone marrow failure syndrome caused by the destruction of hematopoietic stem and progenitor cells by the immune system. However, in some cases of AA, a small number of specific clones with gene mutations are observed without clinical manifestations. Cases with mutated PIG-A, BCOR/BCORL1, or HLA class I allele clones respond better to immunosuppressive therapies (ISTs). Cases with MDS-related clones, such as DNMT3A or ASXL1 mutations, are at a higher risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37406166"
                        open
                      >
                        <summary class="article-expander__title">Somatic mutational landscape of hereditary hematopoietic malignancies caused by germ line RUNX1, GATA2, and DDX41 variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37406166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with germline variants associated with hereditary hematopoietic malignancies (HHMs) have a highly variable risk for leukemogenesis. Gaps in our understanding of pre-malignant states in HHMs have hampered efforts to design effective clinical surveillance programs, provide personalized pre-emptive treatments and inform appropriate counselling for patients. We used the largest known comparative international cohort of germline RUNX1, GATA2, or DDX41 variant carriers without and with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37407470"
                        open
                      >
                        <summary class="article-expander__title">Clinical significance of clonal hematopoiesis and disease boundaries in bone marrow failure diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37407470/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) is a non-neoplastic bone marrow failure syndrome caused by the destruction of hematopoietic stem and progenitor cells by the immune system. However, in some cases of AA, a small number of specific clones with gene mutations are observed without clinical manifestations. Cases with mutated PIG-A, BCOR/BCORL1, or HLA class I allele clones respond better to immunosuppressive therapies (ISTs). Cases with MDS-related clones, such as DNMT3A or ASXL1 mutations, are at a higher risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37406166"
                        open
                      >
                        <summary class="article-expander__title">Somatic mutational landscape of hereditary hematopoietic malignancies caused by germ line RUNX1, GATA2, and DDX41 variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37406166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with germline variants associated with hereditary hematopoietic malignancies (HHMs) have a highly variable risk for leukemogenesis. Gaps in our understanding of pre-malignant states in HHMs have hampered efforts to design effective clinical surveillance programs, provide personalized pre-emptive treatments and inform appropriate counselling for patients. We used the largest known comparative international cohort of germline RUNX1, GATA2, or DDX41 variant carriers without and with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37407470"
                        open
                      >
                        <summary class="article-expander__title">Clinical significance of clonal hematopoiesis and disease boundaries in bone marrow failure diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37407470/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) is a non-neoplastic bone marrow failure syndrome caused by the destruction of hematopoietic stem and progenitor cells by the immune system. However, in some cases of AA, a small number of specific clones with gene mutations are observed without clinical manifestations. Cases with mutated PIG-A, BCOR/BCORL1, or HLA class I allele clones respond better to immunosuppressive therapies (ISTs). Cases with MDS-related clones, such as DNMT3A or ASXL1 mutations, are at a higher risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37406166"
                        open
                      >
                        <summary class="article-expander__title">Somatic mutational landscape of hereditary hematopoietic malignancies caused by germ line RUNX1, GATA2, and DDX41 variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37406166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with germline variants associated with hereditary hematopoietic malignancies (HHMs) have a highly variable risk for leukemogenesis. Gaps in our understanding of pre-malignant states in HHMs have hampered efforts to design effective clinical surveillance programs, provide personalized pre-emptive treatments and inform appropriate counselling for patients. We used the largest known comparative international cohort of germline RUNX1, GATA2, or DDX41 variant carriers without and with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37407470"
                        open
                      >
                        <summary class="article-expander__title">Clinical significance of clonal hematopoiesis and disease boundaries in bone marrow failure diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37407470/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) is a non-neoplastic bone marrow failure syndrome caused by the destruction of hematopoietic stem and progenitor cells by the immune system. However, in some cases of AA, a small number of specific clones with gene mutations are observed without clinical manifestations. Cases with mutated PIG-A, BCOR/BCORL1, or HLA class I allele clones respond better to immunosuppressive therapies (ISTs). Cases with MDS-related clones, such as DNMT3A or ASXL1 mutations, are at a higher risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37406166"
                        open
                      >
                        <summary class="article-expander__title">Somatic mutational landscape of hereditary hematopoietic malignancies caused by germ line RUNX1, GATA2, and DDX41 variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37406166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with germline variants associated with hereditary hematopoietic malignancies (HHMs) have a highly variable risk for leukemogenesis. Gaps in our understanding of pre-malignant states in HHMs have hampered efforts to design effective clinical surveillance programs, provide personalized pre-emptive treatments and inform appropriate counselling for patients. We used the largest known comparative international cohort of germline RUNX1, GATA2, or DDX41 variant carriers without and with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37406166"
                        open
                      >
                        <summary class="article-expander__title">Somatic mutational landscape of hereditary hematopoietic malignancies caused by germ line RUNX1, GATA2, and DDX41 variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37406166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230705211413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with germline variants associated with hereditary hematopoietic malignancies (HHMs) have a highly variable risk for leukemogenesis. Gaps in our understanding of pre-malignant states in HHMs have hampered efforts to design effective clinical surveillance programs, provide personalized pre-emptive treatments and inform appropriate counselling for patients. We used the largest known comparative international cohort of germline RUNX1, GATA2, or DDX41 variant carriers without and with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-07-03 datetime="2023-07-03T10:00:00.000Z">2023-07-03</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-07-03 datetime="2023-07-03T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37396092"
                        open
                      >
                        <summary class="article-expander__title">Multiple Myeloma (Part 1) - Update on Epidemiology, Diagnostic Criteria, Systemic Treatment and Prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37396092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy characterized by unregulated and clonal proliferation of plasma cells in the bone marrow; these cells produce and secrete an anomalous monoclonal immunoglobulin, or a fragment of this, called M protein. The clinical manifestations of MM result from the proliferation of these plasmocytes, the excessive production of monoclonal immunoglobulin and the suppression of normal humoral immunity, leading to hypercalcemia, bone destruction, renal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37396092"
                        open
                      >
                        <summary class="article-expander__title">Multiple Myeloma (Part 1) - Update on Epidemiology, Diagnostic Criteria, Systemic Treatment and Prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37396092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy characterized by unregulated and clonal proliferation of plasma cells in the bone marrow; these cells produce and secrete an anomalous monoclonal immunoglobulin, or a fragment of this, called M protein. The clinical manifestations of MM result from the proliferation of these plasmocytes, the excessive production of monoclonal immunoglobulin and the suppression of normal humoral immunity, leading to hypercalcemia, bone destruction, renal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37396092"
                        open
                      >
                        <summary class="article-expander__title">Multiple Myeloma (Part 1) - Update on Epidemiology, Diagnostic Criteria, Systemic Treatment and Prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37396092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy characterized by unregulated and clonal proliferation of plasma cells in the bone marrow; these cells produce and secrete an anomalous monoclonal immunoglobulin, or a fragment of this, called M protein. The clinical manifestations of MM result from the proliferation of these plasmocytes, the excessive production of monoclonal immunoglobulin and the suppression of normal humoral immunity, leading to hypercalcemia, bone destruction, renal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37395777"
                        open
                      >
                        <summary class="article-expander__title">Low-Density Lipoprotein Cholesterol Levels in Adults With Coronary Artery Disease in the US, January 2015 to March 2020</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37395777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>No abstract</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37396092"
                        open
                      >
                        <summary class="article-expander__title">Multiple Myeloma (Part 1) - Update on Epidemiology, Diagnostic Criteria, Systemic Treatment and Prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37396092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy characterized by unregulated and clonal proliferation of plasma cells in the bone marrow; these cells produce and secrete an anomalous monoclonal immunoglobulin, or a fragment of this, called M protein. The clinical manifestations of MM result from the proliferation of these plasmocytes, the excessive production of monoclonal immunoglobulin and the suppression of normal humoral immunity, leading to hypercalcemia, bone destruction, renal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37395777"
                        open
                      >
                        <summary class="article-expander__title">Low-Density Lipoprotein Cholesterol Levels in Adults With Coronary Artery Disease in the US, January 2015 to March 2020</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37395777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>No abstract</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37396092"
                        open
                      >
                        <summary class="article-expander__title">Multiple Myeloma (Part 1) - Update on Epidemiology, Diagnostic Criteria, Systemic Treatment and Prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37396092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy characterized by unregulated and clonal proliferation of plasma cells in the bone marrow; these cells produce and secrete an anomalous monoclonal immunoglobulin, or a fragment of this, called M protein. The clinical manifestations of MM result from the proliferation of these plasmocytes, the excessive production of monoclonal immunoglobulin and the suppression of normal humoral immunity, leading to hypercalcemia, bone destruction, renal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37395777"
                        open
                      >
                        <summary class="article-expander__title">Low-Density Lipoprotein Cholesterol Levels in Adults With Coronary Artery Disease in the US, January 2015 to March 2020</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37395777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>No abstract</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37396092"
                        open
                      >
                        <summary class="article-expander__title">Multiple Myeloma (Part 1) - Update on Epidemiology, Diagnostic Criteria, Systemic Treatment and Prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37396092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy characterized by unregulated and clonal proliferation of plasma cells in the bone marrow; these cells produce and secrete an anomalous monoclonal immunoglobulin, or a fragment of this, called M protein. The clinical manifestations of MM result from the proliferation of these plasmocytes, the excessive production of monoclonal immunoglobulin and the suppression of normal humoral immunity, leading to hypercalcemia, bone destruction, renal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37395777"
                        open
                      >
                        <summary class="article-expander__title">Low-Density Lipoprotein Cholesterol Levels in Adults With Coronary Artery Disease in the US, January 2015 to March 2020</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37395777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>No abstract</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37396092"
                        open
                      >
                        <summary class="article-expander__title">Multiple Myeloma (Part 1) - Update on Epidemiology, Diagnostic Criteria, Systemic Treatment and Prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37396092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy characterized by unregulated and clonal proliferation of plasma cells in the bone marrow; these cells produce and secrete an anomalous monoclonal immunoglobulin, or a fragment of this, called M protein. The clinical manifestations of MM result from the proliferation of these plasmocytes, the excessive production of monoclonal immunoglobulin and the suppression of normal humoral immunity, leading to hypercalcemia, bone destruction, renal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37395777"
                        open
                      >
                        <summary class="article-expander__title">Low-Density Lipoprotein Cholesterol Levels in Adults With Coronary Artery Disease in the US, January 2015 to March 2020</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37395777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>No abstract</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37396092"
                        open
                      >
                        <summary class="article-expander__title">Multiple Myeloma (Part 1) - Update on Epidemiology, Diagnostic Criteria, Systemic Treatment and Prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37396092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy characterized by unregulated and clonal proliferation of plasma cells in the bone marrow; these cells produce and secrete an anomalous monoclonal immunoglobulin, or a fragment of this, called M protein. The clinical manifestations of MM result from the proliferation of these plasmocytes, the excessive production of monoclonal immunoglobulin and the suppression of normal humoral immunity, leading to hypercalcemia, bone destruction, renal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37395777"
                        open
                      >
                        <summary class="article-expander__title">Low-Density Lipoprotein Cholesterol Levels in Adults With Coronary Artery Disease in the US, January 2015 to March 2020</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37395777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>No abstract</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37396092"
                        open
                      >
                        <summary class="article-expander__title">Multiple Myeloma (Part 1) - Update on Epidemiology, Diagnostic Criteria, Systemic Treatment and Prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37396092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy characterized by unregulated and clonal proliferation of plasma cells in the bone marrow; these cells produce and secrete an anomalous monoclonal immunoglobulin, or a fragment of this, called M protein. The clinical manifestations of MM result from the proliferation of these plasmocytes, the excessive production of monoclonal immunoglobulin and the suppression of normal humoral immunity, leading to hypercalcemia, bone destruction, renal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37395777"
                        open
                      >
                        <summary class="article-expander__title">Low-Density Lipoprotein Cholesterol Levels in Adults With Coronary Artery Disease in the US, January 2015 to March 2020</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37395777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>No abstract</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37396092"
                        open
                      >
                        <summary class="article-expander__title">Multiple Myeloma (Part 1) - Update on Epidemiology, Diagnostic Criteria, Systemic Treatment and Prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37396092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy characterized by unregulated and clonal proliferation of plasma cells in the bone marrow; these cells produce and secrete an anomalous monoclonal immunoglobulin, or a fragment of this, called M protein. The clinical manifestations of MM result from the proliferation of these plasmocytes, the excessive production of monoclonal immunoglobulin and the suppression of normal humoral immunity, leading to hypercalcemia, bone destruction, renal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37395777"
                        open
                      >
                        <summary class="article-expander__title">Low-Density Lipoprotein Cholesterol Levels in Adults With Coronary Artery Disease in the US, January 2015 to March 2020</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37395777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>No abstract</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37396092"
                        open
                      >
                        <summary class="article-expander__title">Multiple Myeloma (Part 1) - Update on Epidemiology, Diagnostic Criteria, Systemic Treatment and Prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37396092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy characterized by unregulated and clonal proliferation of plasma cells in the bone marrow; these cells produce and secrete an anomalous monoclonal immunoglobulin, or a fragment of this, called M protein. The clinical manifestations of MM result from the proliferation of these plasmocytes, the excessive production of monoclonal immunoglobulin and the suppression of normal humoral immunity, leading to hypercalcemia, bone destruction, renal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37395777"
                        open
                      >
                        <summary class="article-expander__title">Low-Density Lipoprotein Cholesterol Levels in Adults With Coronary Artery Disease in the US, January 2015 to March 2020</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37395777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>No abstract</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37396092"
                        open
                      >
                        <summary class="article-expander__title">Multiple Myeloma (Part 1) - Update on Epidemiology, Diagnostic Criteria, Systemic Treatment and Prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37396092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy characterized by unregulated and clonal proliferation of plasma cells in the bone marrow; these cells produce and secrete an anomalous monoclonal immunoglobulin, or a fragment of this, called M protein. The clinical manifestations of MM result from the proliferation of these plasmocytes, the excessive production of monoclonal immunoglobulin and the suppression of normal humoral immunity, leading to hypercalcemia, bone destruction, renal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37395777"
                        open
                      >
                        <summary class="article-expander__title">Low-Density Lipoprotein Cholesterol Levels in Adults With Coronary Artery Disease in the US, January 2015 to March 2020</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37395777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>No abstract</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37396092"
                        open
                      >
                        <summary class="article-expander__title">Multiple Myeloma (Part 1) - Update on Epidemiology, Diagnostic Criteria, Systemic Treatment and Prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37396092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy characterized by unregulated and clonal proliferation of plasma cells in the bone marrow; these cells produce and secrete an anomalous monoclonal immunoglobulin, or a fragment of this, called M protein. The clinical manifestations of MM result from the proliferation of these plasmocytes, the excessive production of monoclonal immunoglobulin and the suppression of normal humoral immunity, leading to hypercalcemia, bone destruction, renal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37395777"
                        open
                      >
                        <summary class="article-expander__title">Low-Density Lipoprotein Cholesterol Levels in Adults With Coronary Artery Disease in the US, January 2015 to March 2020</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37395777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>No abstract</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230705211413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37396092"
                        open
                      >
                        <summary class="article-expander__title">Multiple Myeloma (Part 1) - Update on Epidemiology, Diagnostic Criteria, Systemic Treatment and Prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37396092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230705211413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy characterized by unregulated and clonal proliferation of plasma cells in the bone marrow; these cells produce and secrete an anomalous monoclonal immunoglobulin, or a fragment of this, called M protein. The clinical manifestations of MM result from the proliferation of these plasmocytes, the excessive production of monoclonal immunoglobulin and the suppression of normal humoral immunity, leading to hypercalcemia, bone destruction, renal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37395777"
                        open
                      >
                        <summary class="article-expander__title">Low-Density Lipoprotein Cholesterol Levels in Adults With Coronary Artery Disease in the US, January 2015 to March 2020</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37395777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230705211413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>No abstract</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37396092"
                        open
                      >
                        <summary class="article-expander__title">Multiple Myeloma (Part 1) - Update on Epidemiology, Diagnostic Criteria, Systemic Treatment and Prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37396092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy characterized by unregulated and clonal proliferation of plasma cells in the bone marrow; these cells produce and secrete an anomalous monoclonal immunoglobulin, or a fragment of this, called M protein. The clinical manifestations of MM result from the proliferation of these plasmocytes, the excessive production of monoclonal immunoglobulin and the suppression of normal humoral immunity, leading to hypercalcemia, bone destruction, renal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37396092"
                        open
                      >
                        <summary class="article-expander__title">Multiple Myeloma (Part 1) - Update on Epidemiology, Diagnostic Criteria, Systemic Treatment and Prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37396092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy characterized by unregulated and clonal proliferation of plasma cells in the bone marrow; these cells produce and secrete an anomalous monoclonal immunoglobulin, or a fragment of this, called M protein. The clinical manifestations of MM result from the proliferation of these plasmocytes, the excessive production of monoclonal immunoglobulin and the suppression of normal humoral immunity, leading to hypercalcemia, bone destruction, renal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230804205603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230803205941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230802205831&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230801205540&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230731210325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230730205823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230729205947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230726205538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230726205538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230723210310&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230723210310&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230720210258&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230720210258&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230705211412&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230705211412&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399708"
                        open
                      >
                        <summary class="article-expander__title">Molecular screening of head neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsy has become a significant tool in personalized medicine, enabling real-time monitoring of cancer evolution and patient follow-up. This minimally invasive procedure analyzes circulating tumor cells (CTCs) and circulating tumor-derived materials, such as ctDNA, miRNAs, and EVs. CTC analysis significantly impacts prognosis, detection of minimal residual disease (MRD), treatment selection, and monitoring of cancer patients. Liquid biopsy is an attractive option for mouth cancer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37399248"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics and copy number variants by single-cell analysis in leukemic evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37399248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transformation from chronic (CP) to blast phase (BP) in myeloproliferative neoplasm (MPN) remains poorly characterized, and no specific mutation pattern has been highlighted. BP-MPN represents an unmet need, due to its refractoriness to treatment and dismal outcome. Taking advantage of the granularity provided by single-cell sequencing (SCS), we analyzed paired samples of CP and BP in 10 patients to map clonal trajectories and interrogate target copy number variants (CNVs). Already at diagnosis,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-07-02 datetime="2023-07-02T10:00:00.000Z">2023-07-02</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-07-02 datetime="2023-07-02T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37394583"
                        open
                      >
                        <summary class="article-expander__title">Metastatic pattern of ovarian cancer delineated by tracing the evolution of mitochondrial DNA mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37394583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer (OC) is the most lethal gynecologic tumor and is characterized by a high rate of metastasis. Challenges in accurately delineating the metastatic pattern have greatly restricted the improvement of treatment in OC patients. An increasing number of studies have leveraged mitochondrial DNA (mtDNA) mutations as efficient lineage-tracing markers of tumor clonality. We applied multiregional sampling and high-depth mtDNA sequencing to determine the metastatic patterns in advanced-stage OC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37394583"
                        open
                      >
                        <summary class="article-expander__title">Metastatic pattern of ovarian cancer delineated by tracing the evolution of mitochondrial DNA mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37394583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer (OC) is the most lethal gynecologic tumor and is characterized by a high rate of metastasis. Challenges in accurately delineating the metastatic pattern have greatly restricted the improvement of treatment in OC patients. An increasing number of studies have leveraged mitochondrial DNA (mtDNA) mutations as efficient lineage-tracing markers of tumor clonality. We applied multiregional sampling and high-depth mtDNA sequencing to determine the metastatic patterns in advanced-stage OC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37394583"
                        open
                      >
                        <summary class="article-expander__title">Metastatic pattern of ovarian cancer delineated by tracing the evolution of mitochondrial DNA mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37394583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230726205538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer (OC) is the most lethal gynecologic tumor and is characterized by a high rate of metastasis. Challenges in accurately delineating the metastatic pattern have greatly restricted the improvement of treatment in OC patients. An increasing number of studies have leveraged mitochondrial DNA (mtDNA) mutations as efficient lineage-tracing markers of tumor clonality. We applied multiregional sampling and high-depth mtDNA sequencing to determine the metastatic patterns in advanced-stage OC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37394583"
                        open
                      >
                        <summary class="article-expander__title">Metastatic pattern of ovarian cancer delineated by tracing the evolution of mitochondrial DNA mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37394583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer (OC) is the most lethal gynecologic tumor and is characterized by a high rate of metastasis. Challenges in accurately delineating the metastatic pattern have greatly restricted the improvement of treatment in OC patients. An increasing number of studies have leveraged mitochondrial DNA (mtDNA) mutations as efficient lineage-tracing markers of tumor clonality. We applied multiregional sampling and high-depth mtDNA sequencing to determine the metastatic patterns in advanced-stage OC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37394583"
                        open
                      >
                        <summary class="article-expander__title">Metastatic pattern of ovarian cancer delineated by tracing the evolution of mitochondrial DNA mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37394583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer (OC) is the most lethal gynecologic tumor and is characterized by a high rate of metastasis. Challenges in accurately delineating the metastatic pattern have greatly restricted the improvement of treatment in OC patients. An increasing number of studies have leveraged mitochondrial DNA (mtDNA) mutations as efficient lineage-tracing markers of tumor clonality. We applied multiregional sampling and high-depth mtDNA sequencing to determine the metastatic patterns in advanced-stage OC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37394583"
                        open
                      >
                        <summary class="article-expander__title">Metastatic pattern of ovarian cancer delineated by tracing the evolution of mitochondrial DNA mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37394583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230723210310&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer (OC) is the most lethal gynecologic tumor and is characterized by a high rate of metastasis. Challenges in accurately delineating the metastatic pattern have greatly restricted the improvement of treatment in OC patients. An increasing number of studies have leveraged mitochondrial DNA (mtDNA) mutations as efficient lineage-tracing markers of tumor clonality. We applied multiregional sampling and high-depth mtDNA sequencing to determine the metastatic patterns in advanced-stage OC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37394583"
                        open
                      >
                        <summary class="article-expander__title">Metastatic pattern of ovarian cancer delineated by tracing the evolution of mitochondrial DNA mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37394583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer (OC) is the most lethal gynecologic tumor and is characterized by a high rate of metastasis. Challenges in accurately delineating the metastatic pattern have greatly restricted the improvement of treatment in OC patients. An increasing number of studies have leveraged mitochondrial DNA (mtDNA) mutations as efficient lineage-tracing markers of tumor clonality. We applied multiregional sampling and high-depth mtDNA sequencing to determine the metastatic patterns in advanced-stage OC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37394583"
                        open
                      >
                        <summary class="article-expander__title">Metastatic pattern of ovarian cancer delineated by tracing the evolution of mitochondrial DNA mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37394583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer (OC) is the most lethal gynecologic tumor and is characterized by a high rate of metastasis. Challenges in accurately delineating the metastatic pattern have greatly restricted the improvement of treatment in OC patients. An increasing number of studies have leveraged mitochondrial DNA (mtDNA) mutations as efficient lineage-tracing markers of tumor clonality. We applied multiregional sampling and high-depth mtDNA sequencing to determine the metastatic patterns in advanced-stage OC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37394583"
                        open
                      >
                        <summary class="article-expander__title">Metastatic pattern of ovarian cancer delineated by tracing the evolution of mitochondrial DNA mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37394583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230720210258&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer (OC) is the most lethal gynecologic tumor and is characterized by a high rate of metastasis. Challenges in accurately delineating the metastatic pattern have greatly restricted the improvement of treatment in OC patients. An increasing number of studies have leveraged mitochondrial DNA (mtDNA) mutations as efficient lineage-tracing markers of tumor clonality. We applied multiregional sampling and high-depth mtDNA sequencing to determine the metastatic patterns in advanced-stage OC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37394583"
                        open
                      >
                        <summary class="article-expander__title">Metastatic pattern of ovarian cancer delineated by tracing the evolution of mitochondrial DNA mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37394583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer (OC) is the most lethal gynecologic tumor and is characterized by a high rate of metastasis. Challenges in accurately delineating the metastatic pattern have greatly restricted the improvement of treatment in OC patients. An increasing number of studies have leveraged mitochondrial DNA (mtDNA) mutations as efficient lineage-tracing markers of tumor clonality. We applied multiregional sampling and high-depth mtDNA sequencing to determine the metastatic patterns in advanced-stage OC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37394583"
                        open
                      >
                        <summary class="article-expander__title">Metastatic pattern of ovarian cancer delineated by tracing the evolution of mitochondrial DNA mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37394583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer (OC) is the most lethal gynecologic tumor and is characterized by a high rate of metastasis. Challenges in accurately delineating the metastatic pattern have greatly restricted the improvement of treatment in OC patients. An increasing number of studies have leveraged mitochondrial DNA (mtDNA) mutations as efficient lineage-tracing markers of tumor clonality. We applied multiregional sampling and high-depth mtDNA sequencing to determine the metastatic patterns in advanced-stage OC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37394583"
                        open
                      >
                        <summary class="article-expander__title">Metastatic pattern of ovarian cancer delineated by tracing the evolution of mitochondrial DNA mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37394583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer (OC) is the most lethal gynecologic tumor and is characterized by a high rate of metastasis. Challenges in accurately delineating the metastatic pattern have greatly restricted the improvement of treatment in OC patients. An increasing number of studies have leveraged mitochondrial DNA (mtDNA) mutations as efficient lineage-tracing markers of tumor clonality. We applied multiregional sampling and high-depth mtDNA sequencing to determine the metastatic patterns in advanced-stage OC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37394583"
                        open
                      >
                        <summary class="article-expander__title">Metastatic pattern of ovarian cancer delineated by tracing the evolution of mitochondrial DNA mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37394583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer (OC) is the most lethal gynecologic tumor and is characterized by a high rate of metastasis. Challenges in accurately delineating the metastatic pattern have greatly restricted the improvement of treatment in OC patients. An increasing number of studies have leveraged mitochondrial DNA (mtDNA) mutations as efficient lineage-tracing markers of tumor clonality. We applied multiregional sampling and high-depth mtDNA sequencing to determine the metastatic patterns in advanced-stage OC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37394583"
                        open
                      >
                        <summary class="article-expander__title">Metastatic pattern of ovarian cancer delineated by tracing the evolution of mitochondrial DNA mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37394583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer (OC) is the most lethal gynecologic tumor and is characterized by a high rate of metastasis. Challenges in accurately delineating the metastatic pattern have greatly restricted the improvement of treatment in OC patients. An increasing number of studies have leveraged mitochondrial DNA (mtDNA) mutations as efficient lineage-tracing markers of tumor clonality. We applied multiregional sampling and high-depth mtDNA sequencing to determine the metastatic patterns in advanced-stage OC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37394583"
                        open
                      >
                        <summary class="article-expander__title">Metastatic pattern of ovarian cancer delineated by tracing the evolution of mitochondrial DNA mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37394583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer (OC) is the most lethal gynecologic tumor and is characterized by a high rate of metastasis. Challenges in accurately delineating the metastatic pattern have greatly restricted the improvement of treatment in OC patients. An increasing number of studies have leveraged mitochondrial DNA (mtDNA) mutations as efficient lineage-tracing markers of tumor clonality. We applied multiregional sampling and high-depth mtDNA sequencing to determine the metastatic patterns in advanced-stage OC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37394583"
                        open
                      >
                        <summary class="article-expander__title">Metastatic pattern of ovarian cancer delineated by tracing the evolution of mitochondrial DNA mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37394583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer (OC) is the most lethal gynecologic tumor and is characterized by a high rate of metastasis. Challenges in accurately delineating the metastatic pattern have greatly restricted the improvement of treatment in OC patients. An increasing number of studies have leveraged mitochondrial DNA (mtDNA) mutations as efficient lineage-tracing markers of tumor clonality. We applied multiregional sampling and high-depth mtDNA sequencing to determine the metastatic patterns in advanced-stage OC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37394583"
                        open
                      >
                        <summary class="article-expander__title">Metastatic pattern of ovarian cancer delineated by tracing the evolution of mitochondrial DNA mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37394583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer (OC) is the most lethal gynecologic tumor and is characterized by a high rate of metastasis. Challenges in accurately delineating the metastatic pattern have greatly restricted the improvement of treatment in OC patients. An increasing number of studies have leveraged mitochondrial DNA (mtDNA) mutations as efficient lineage-tracing markers of tumor clonality. We applied multiregional sampling and high-depth mtDNA sequencing to determine the metastatic patterns in advanced-stage OC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37394583"
                        open
                      >
                        <summary class="article-expander__title">Metastatic pattern of ovarian cancer delineated by tracing the evolution of mitochondrial DNA mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37394583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer (OC) is the most lethal gynecologic tumor and is characterized by a high rate of metastasis. Challenges in accurately delineating the metastatic pattern have greatly restricted the improvement of treatment in OC patients. An increasing number of studies have leveraged mitochondrial DNA (mtDNA) mutations as efficient lineage-tracing markers of tumor clonality. We applied multiregional sampling and high-depth mtDNA sequencing to determine the metastatic patterns in advanced-stage OC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37394583"
                        open
                      >
                        <summary class="article-expander__title">Metastatic pattern of ovarian cancer delineated by tracing the evolution of mitochondrial DNA mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37394583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer (OC) is the most lethal gynecologic tumor and is characterized by a high rate of metastasis. Challenges in accurately delineating the metastatic pattern have greatly restricted the improvement of treatment in OC patients. An increasing number of studies have leveraged mitochondrial DNA (mtDNA) mutations as efficient lineage-tracing markers of tumor clonality. We applied multiregional sampling and high-depth mtDNA sequencing to determine the metastatic patterns in advanced-stage OC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37394583"
                        open
                      >
                        <summary class="article-expander__title">Metastatic pattern of ovarian cancer delineated by tracing the evolution of mitochondrial DNA mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37394583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer (OC) is the most lethal gynecologic tumor and is characterized by a high rate of metastasis. Challenges in accurately delineating the metastatic pattern have greatly restricted the improvement of treatment in OC patients. An increasing number of studies have leveraged mitochondrial DNA (mtDNA) mutations as efficient lineage-tracing markers of tumor clonality. We applied multiregional sampling and high-depth mtDNA sequencing to determine the metastatic patterns in advanced-stage OC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37394583"
                        open
                      >
                        <summary class="article-expander__title">Metastatic pattern of ovarian cancer delineated by tracing the evolution of mitochondrial DNA mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37394583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer (OC) is the most lethal gynecologic tumor and is characterized by a high rate of metastasis. Challenges in accurately delineating the metastatic pattern have greatly restricted the improvement of treatment in OC patients. An increasing number of studies have leveraged mitochondrial DNA (mtDNA) mutations as efficient lineage-tracing markers of tumor clonality. We applied multiregional sampling and high-depth mtDNA sequencing to determine the metastatic patterns in advanced-stage OC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37394583"
                        open
                      >
                        <summary class="article-expander__title">Metastatic pattern of ovarian cancer delineated by tracing the evolution of mitochondrial DNA mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37394583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer (OC) is the most lethal gynecologic tumor and is characterized by a high rate of metastasis. Challenges in accurately delineating the metastatic pattern have greatly restricted the improvement of treatment in OC patients. An increasing number of studies have leveraged mitochondrial DNA (mtDNA) mutations as efficient lineage-tracing markers of tumor clonality. We applied multiregional sampling and high-depth mtDNA sequencing to determine the metastatic patterns in advanced-stage OC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37394583"
                        open
                      >
                        <summary class="article-expander__title">Metastatic pattern of ovarian cancer delineated by tracing the evolution of mitochondrial DNA mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37394583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer (OC) is the most lethal gynecologic tumor and is characterized by a high rate of metastasis. Challenges in accurately delineating the metastatic pattern have greatly restricted the improvement of treatment in OC patients. An increasing number of studies have leveraged mitochondrial DNA (mtDNA) mutations as efficient lineage-tracing markers of tumor clonality. We applied multiregional sampling and high-depth mtDNA sequencing to determine the metastatic patterns in advanced-stage OC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37394583"
                        open
                      >
                        <summary class="article-expander__title">Metastatic pattern of ovarian cancer delineated by tracing the evolution of mitochondrial DNA mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37394583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230705211412&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer (OC) is the most lethal gynecologic tumor and is characterized by a high rate of metastasis. Challenges in accurately delineating the metastatic pattern have greatly restricted the improvement of treatment in OC patients. An increasing number of studies have leveraged mitochondrial DNA (mtDNA) mutations as efficient lineage-tracing markers of tumor clonality. We applied multiregional sampling and high-depth mtDNA sequencing to determine the metastatic patterns in advanced-stage OC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37394583"
                        open
                      >
                        <summary class="article-expander__title">Metastatic pattern of ovarian cancer delineated by tracing the evolution of mitochondrial DNA mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37394583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer (OC) is the most lethal gynecologic tumor and is characterized by a high rate of metastasis. Challenges in accurately delineating the metastatic pattern have greatly restricted the improvement of treatment in OC patients. An increasing number of studies have leveraged mitochondrial DNA (mtDNA) mutations as efficient lineage-tracing markers of tumor clonality. We applied multiregional sampling and high-depth mtDNA sequencing to determine the metastatic patterns in advanced-stage OC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37394583"
                        open
                      >
                        <summary class="article-expander__title">Metastatic pattern of ovarian cancer delineated by tracing the evolution of mitochondrial DNA mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37394583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer (OC) is the most lethal gynecologic tumor and is characterized by a high rate of metastasis. Challenges in accurately delineating the metastatic pattern have greatly restricted the improvement of treatment in OC patients. An increasing number of studies have leveraged mitochondrial DNA (mtDNA) mutations as efficient lineage-tracing markers of tumor clonality. We applied multiregional sampling and high-depth mtDNA sequencing to determine the metastatic patterns in advanced-stage OC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-30 datetime="2023-06-30T10:00:00.000Z">2023-06-30</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-30 datetime="2023-06-30T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391583"
                        open
                      >
                        <summary class="article-expander__title">Study on distribution of sidewall earth pressure on open caissons considering soil arching effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Based on the soil arching effect theory, the magnitude and distribution of sidewall earth pressure on open caissons when the embedded depth is large was analyzed by using theory of non-limit state earth pressure theory and horizontal differential element method. The theoretical formula was deduced. The theoretical calculation results are compared with the field test results and centrifugal model test results respectively. The results show that when the embedded depth of the open caisson is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391453"
                        open
                      >
                        <summary class="article-expander__title">Effectiveness of creating digital twins with different digital dentition models and cone-beam computed tomography</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391453/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Distortion of dentition may occur in cone-beam computed tomography (CBCT) scans due to artifacts, and further imaging is frequently required to produce digital twins. The use of a plaster model is common; however, it has certain drawbacks. This study aimed to assess the feasibility of different digital dentition models over that of plaster casts. Plaster models, alginate impressions, intraoral scan (IOS) images, and CBCT images of 20 patients were obtained. The desktop model scanner was used to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391583"
                        open
                      >
                        <summary class="article-expander__title">Study on distribution of sidewall earth pressure on open caissons considering soil arching effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Based on the soil arching effect theory, the magnitude and distribution of sidewall earth pressure on open caissons when the embedded depth is large was analyzed by using theory of non-limit state earth pressure theory and horizontal differential element method. The theoretical formula was deduced. The theoretical calculation results are compared with the field test results and centrifugal model test results respectively. The results show that when the embedded depth of the open caisson is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391453"
                        open
                      >
                        <summary class="article-expander__title">Effectiveness of creating digital twins with different digital dentition models and cone-beam computed tomography</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391453/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Distortion of dentition may occur in cone-beam computed tomography (CBCT) scans due to artifacts, and further imaging is frequently required to produce digital twins. The use of a plaster model is common; however, it has certain drawbacks. This study aimed to assess the feasibility of different digital dentition models over that of plaster casts. Plaster models, alginate impressions, intraoral scan (IOS) images, and CBCT images of 20 patients were obtained. The desktop model scanner was used to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391357"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related myelodysplastic syndromes in the genomics era</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391357/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myelodysplastic syndromes (t-MDS) represent a heterogeneous group of malignancies that arise as a late complication of prior exposure to chemotherapy and/or radiotherapy administered for a primary condition. T-MDS account for approximately 20% of all MDS and are characterized by resistance to current treatment strategies and poor prognosis. Our understanding of t-MDS pathogenesis has considerably improved over the last 5 years with the availability of deep sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391583"
                        open
                      >
                        <summary class="article-expander__title">Study on distribution of sidewall earth pressure on open caissons considering soil arching effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Based on the soil arching effect theory, the magnitude and distribution of sidewall earth pressure on open caissons when the embedded depth is large was analyzed by using theory of non-limit state earth pressure theory and horizontal differential element method. The theoretical formula was deduced. The theoretical calculation results are compared with the field test results and centrifugal model test results respectively. The results show that when the embedded depth of the open caisson is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391453"
                        open
                      >
                        <summary class="article-expander__title">Effectiveness of creating digital twins with different digital dentition models and cone-beam computed tomography</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391453/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Distortion of dentition may occur in cone-beam computed tomography (CBCT) scans due to artifacts, and further imaging is frequently required to produce digital twins. The use of a plaster model is common; however, it has certain drawbacks. This study aimed to assess the feasibility of different digital dentition models over that of plaster casts. Plaster models, alginate impressions, intraoral scan (IOS) images, and CBCT images of 20 patients were obtained. The desktop model scanner was used to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391357"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related myelodysplastic syndromes in the genomics era</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391357/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myelodysplastic syndromes (t-MDS) represent a heterogeneous group of malignancies that arise as a late complication of prior exposure to chemotherapy and/or radiotherapy administered for a primary condition. T-MDS account for approximately 20% of all MDS and are characterized by resistance to current treatment strategies and poor prognosis. Our understanding of t-MDS pathogenesis has considerably improved over the last 5 years with the availability of deep sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37389519"
                        open
                      >
                        <summary class="article-expander__title">Impact of Geroscience on Therapeutic Strategies for Older Adults With Cardiovascular Disease: JACC Scientific Statement</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37389519/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Geroscience posits that cardiovascular disease (CVD) and other chronic diseases result from progressive erosion of the effectiveness of homeostatic mechanisms that oppose age-related accumulation of molecular damage. This hypothetical common root to chronic diseases explains why patients with CVD are often affected by multimorbidity and frailty and why older age negatively affects CVD prognosis and treatment response. Gerotherapeutics enhance resilience mechanisms that counter age-related...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391583"
                        open
                      >
                        <summary class="article-expander__title">Study on distribution of sidewall earth pressure on open caissons considering soil arching effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Based on the soil arching effect theory, the magnitude and distribution of sidewall earth pressure on open caissons when the embedded depth is large was analyzed by using theory of non-limit state earth pressure theory and horizontal differential element method. The theoretical formula was deduced. The theoretical calculation results are compared with the field test results and centrifugal model test results respectively. The results show that when the embedded depth of the open caisson is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391453"
                        open
                      >
                        <summary class="article-expander__title">Effectiveness of creating digital twins with different digital dentition models and cone-beam computed tomography</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391453/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Distortion of dentition may occur in cone-beam computed tomography (CBCT) scans due to artifacts, and further imaging is frequently required to produce digital twins. The use of a plaster model is common; however, it has certain drawbacks. This study aimed to assess the feasibility of different digital dentition models over that of plaster casts. Plaster models, alginate impressions, intraoral scan (IOS) images, and CBCT images of 20 patients were obtained. The desktop model scanner was used to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391357"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related myelodysplastic syndromes in the genomics era</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391357/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myelodysplastic syndromes (t-MDS) represent a heterogeneous group of malignancies that arise as a late complication of prior exposure to chemotherapy and/or radiotherapy administered for a primary condition. T-MDS account for approximately 20% of all MDS and are characterized by resistance to current treatment strategies and poor prognosis. Our understanding of t-MDS pathogenesis has considerably improved over the last 5 years with the availability of deep sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37389519"
                        open
                      >
                        <summary class="article-expander__title">Impact of Geroscience on Therapeutic Strategies for Older Adults With Cardiovascular Disease: JACC Scientific Statement</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37389519/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Geroscience posits that cardiovascular disease (CVD) and other chronic diseases result from progressive erosion of the effectiveness of homeostatic mechanisms that oppose age-related accumulation of molecular damage. This hypothetical common root to chronic diseases explains why patients with CVD are often affected by multimorbidity and frailty and why older age negatively affects CVD prognosis and treatment response. Gerotherapeutics enhance resilience mechanisms that counter age-related...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391583"
                        open
                      >
                        <summary class="article-expander__title">Study on distribution of sidewall earth pressure on open caissons considering soil arching effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Based on the soil arching effect theory, the magnitude and distribution of sidewall earth pressure on open caissons when the embedded depth is large was analyzed by using theory of non-limit state earth pressure theory and horizontal differential element method. The theoretical formula was deduced. The theoretical calculation results are compared with the field test results and centrifugal model test results respectively. The results show that when the embedded depth of the open caisson is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391453"
                        open
                      >
                        <summary class="article-expander__title">Effectiveness of creating digital twins with different digital dentition models and cone-beam computed tomography</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391453/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Distortion of dentition may occur in cone-beam computed tomography (CBCT) scans due to artifacts, and further imaging is frequently required to produce digital twins. The use of a plaster model is common; however, it has certain drawbacks. This study aimed to assess the feasibility of different digital dentition models over that of plaster casts. Plaster models, alginate impressions, intraoral scan (IOS) images, and CBCT images of 20 patients were obtained. The desktop model scanner was used to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391357"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related myelodysplastic syndromes in the genomics era</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391357/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myelodysplastic syndromes (t-MDS) represent a heterogeneous group of malignancies that arise as a late complication of prior exposure to chemotherapy and/or radiotherapy administered for a primary condition. T-MDS account for approximately 20% of all MDS and are characterized by resistance to current treatment strategies and poor prognosis. Our understanding of t-MDS pathogenesis has considerably improved over the last 5 years with the availability of deep sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37389519"
                        open
                      >
                        <summary class="article-expander__title">Impact of Geroscience on Therapeutic Strategies for Older Adults With Cardiovascular Disease: JACC Scientific Statement</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37389519/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Geroscience posits that cardiovascular disease (CVD) and other chronic diseases result from progressive erosion of the effectiveness of homeostatic mechanisms that oppose age-related accumulation of molecular damage. This hypothetical common root to chronic diseases explains why patients with CVD are often affected by multimorbidity and frailty and why older age negatively affects CVD prognosis and treatment response. Gerotherapeutics enhance resilience mechanisms that counter age-related...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391583"
                        open
                      >
                        <summary class="article-expander__title">Study on distribution of sidewall earth pressure on open caissons considering soil arching effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Based on the soil arching effect theory, the magnitude and distribution of sidewall earth pressure on open caissons when the embedded depth is large was analyzed by using theory of non-limit state earth pressure theory and horizontal differential element method. The theoretical formula was deduced. The theoretical calculation results are compared with the field test results and centrifugal model test results respectively. The results show that when the embedded depth of the open caisson is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391453"
                        open
                      >
                        <summary class="article-expander__title">Effectiveness of creating digital twins with different digital dentition models and cone-beam computed tomography</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391453/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Distortion of dentition may occur in cone-beam computed tomography (CBCT) scans due to artifacts, and further imaging is frequently required to produce digital twins. The use of a plaster model is common; however, it has certain drawbacks. This study aimed to assess the feasibility of different digital dentition models over that of plaster casts. Plaster models, alginate impressions, intraoral scan (IOS) images, and CBCT images of 20 patients were obtained. The desktop model scanner was used to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391357"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related myelodysplastic syndromes in the genomics era</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391357/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myelodysplastic syndromes (t-MDS) represent a heterogeneous group of malignancies that arise as a late complication of prior exposure to chemotherapy and/or radiotherapy administered for a primary condition. T-MDS account for approximately 20% of all MDS and are characterized by resistance to current treatment strategies and poor prognosis. Our understanding of t-MDS pathogenesis has considerably improved over the last 5 years with the availability of deep sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37389519"
                        open
                      >
                        <summary class="article-expander__title">Impact of Geroscience on Therapeutic Strategies for Older Adults With Cardiovascular Disease: JACC Scientific Statement</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37389519/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Geroscience posits that cardiovascular disease (CVD) and other chronic diseases result from progressive erosion of the effectiveness of homeostatic mechanisms that oppose age-related accumulation of molecular damage. This hypothetical common root to chronic diseases explains why patients with CVD are often affected by multimorbidity and frailty and why older age negatively affects CVD prognosis and treatment response. Gerotherapeutics enhance resilience mechanisms that counter age-related...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391583"
                        open
                      >
                        <summary class="article-expander__title">Study on distribution of sidewall earth pressure on open caissons considering soil arching effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Based on the soil arching effect theory, the magnitude and distribution of sidewall earth pressure on open caissons when the embedded depth is large was analyzed by using theory of non-limit state earth pressure theory and horizontal differential element method. The theoretical formula was deduced. The theoretical calculation results are compared with the field test results and centrifugal model test results respectively. The results show that when the embedded depth of the open caisson is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391453"
                        open
                      >
                        <summary class="article-expander__title">Effectiveness of creating digital twins with different digital dentition models and cone-beam computed tomography</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391453/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Distortion of dentition may occur in cone-beam computed tomography (CBCT) scans due to artifacts, and further imaging is frequently required to produce digital twins. The use of a plaster model is common; however, it has certain drawbacks. This study aimed to assess the feasibility of different digital dentition models over that of plaster casts. Plaster models, alginate impressions, intraoral scan (IOS) images, and CBCT images of 20 patients were obtained. The desktop model scanner was used to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391357"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related myelodysplastic syndromes in the genomics era</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391357/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myelodysplastic syndromes (t-MDS) represent a heterogeneous group of malignancies that arise as a late complication of prior exposure to chemotherapy and/or radiotherapy administered for a primary condition. T-MDS account for approximately 20% of all MDS and are characterized by resistance to current treatment strategies and poor prognosis. Our understanding of t-MDS pathogenesis has considerably improved over the last 5 years with the availability of deep sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37389519"
                        open
                      >
                        <summary class="article-expander__title">Impact of Geroscience on Therapeutic Strategies for Older Adults With Cardiovascular Disease: JACC Scientific Statement</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37389519/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Geroscience posits that cardiovascular disease (CVD) and other chronic diseases result from progressive erosion of the effectiveness of homeostatic mechanisms that oppose age-related accumulation of molecular damage. This hypothetical common root to chronic diseases explains why patients with CVD are often affected by multimorbidity and frailty and why older age negatively affects CVD prognosis and treatment response. Gerotherapeutics enhance resilience mechanisms that counter age-related...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391583"
                        open
                      >
                        <summary class="article-expander__title">Study on distribution of sidewall earth pressure on open caissons considering soil arching effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Based on the soil arching effect theory, the magnitude and distribution of sidewall earth pressure on open caissons when the embedded depth is large was analyzed by using theory of non-limit state earth pressure theory and horizontal differential element method. The theoretical formula was deduced. The theoretical calculation results are compared with the field test results and centrifugal model test results respectively. The results show that when the embedded depth of the open caisson is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391453"
                        open
                      >
                        <summary class="article-expander__title">Effectiveness of creating digital twins with different digital dentition models and cone-beam computed tomography</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391453/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Distortion of dentition may occur in cone-beam computed tomography (CBCT) scans due to artifacts, and further imaging is frequently required to produce digital twins. The use of a plaster model is common; however, it has certain drawbacks. This study aimed to assess the feasibility of different digital dentition models over that of plaster casts. Plaster models, alginate impressions, intraoral scan (IOS) images, and CBCT images of 20 patients were obtained. The desktop model scanner was used to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391357"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related myelodysplastic syndromes in the genomics era</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391357/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myelodysplastic syndromes (t-MDS) represent a heterogeneous group of malignancies that arise as a late complication of prior exposure to chemotherapy and/or radiotherapy administered for a primary condition. T-MDS account for approximately 20% of all MDS and are characterized by resistance to current treatment strategies and poor prognosis. Our understanding of t-MDS pathogenesis has considerably improved over the last 5 years with the availability of deep sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37389519"
                        open
                      >
                        <summary class="article-expander__title">Impact of Geroscience on Therapeutic Strategies for Older Adults With Cardiovascular Disease: JACC Scientific Statement</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37389519/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Geroscience posits that cardiovascular disease (CVD) and other chronic diseases result from progressive erosion of the effectiveness of homeostatic mechanisms that oppose age-related accumulation of molecular damage. This hypothetical common root to chronic diseases explains why patients with CVD are often affected by multimorbidity and frailty and why older age negatively affects CVD prognosis and treatment response. Gerotherapeutics enhance resilience mechanisms that counter age-related...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391583"
                        open
                      >
                        <summary class="article-expander__title">Study on distribution of sidewall earth pressure on open caissons considering soil arching effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230705211413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Based on the soil arching effect theory, the magnitude and distribution of sidewall earth pressure on open caissons when the embedded depth is large was analyzed by using theory of non-limit state earth pressure theory and horizontal differential element method. The theoretical formula was deduced. The theoretical calculation results are compared with the field test results and centrifugal model test results respectively. The results show that when the embedded depth of the open caisson is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391453"
                        open
                      >
                        <summary class="article-expander__title">Effectiveness of creating digital twins with different digital dentition models and cone-beam computed tomography</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391453/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230705211413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Distortion of dentition may occur in cone-beam computed tomography (CBCT) scans due to artifacts, and further imaging is frequently required to produce digital twins. The use of a plaster model is common; however, it has certain drawbacks. This study aimed to assess the feasibility of different digital dentition models over that of plaster casts. Plaster models, alginate impressions, intraoral scan (IOS) images, and CBCT images of 20 patients were obtained. The desktop model scanner was used to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391357"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related myelodysplastic syndromes in the genomics era</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391357/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230705211413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myelodysplastic syndromes (t-MDS) represent a heterogeneous group of malignancies that arise as a late complication of prior exposure to chemotherapy and/or radiotherapy administered for a primary condition. T-MDS account for approximately 20% of all MDS and are characterized by resistance to current treatment strategies and poor prognosis. Our understanding of t-MDS pathogenesis has considerably improved over the last 5 years with the availability of deep sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37389519"
                        open
                      >
                        <summary class="article-expander__title">Impact of Geroscience on Therapeutic Strategies for Older Adults With Cardiovascular Disease: JACC Scientific Statement</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37389519/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230705211413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Geroscience posits that cardiovascular disease (CVD) and other chronic diseases result from progressive erosion of the effectiveness of homeostatic mechanisms that oppose age-related accumulation of molecular damage. This hypothetical common root to chronic diseases explains why patients with CVD are often affected by multimorbidity and frailty and why older age negatively affects CVD prognosis and treatment response. Gerotherapeutics enhance resilience mechanisms that counter age-related...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391583"
                        open
                      >
                        <summary class="article-expander__title">Study on distribution of sidewall earth pressure on open caissons considering soil arching effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Based on the soil arching effect theory, the magnitude and distribution of sidewall earth pressure on open caissons when the embedded depth is large was analyzed by using theory of non-limit state earth pressure theory and horizontal differential element method. The theoretical formula was deduced. The theoretical calculation results are compared with the field test results and centrifugal model test results respectively. The results show that when the embedded depth of the open caisson is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391453"
                        open
                      >
                        <summary class="article-expander__title">Effectiveness of creating digital twins with different digital dentition models and cone-beam computed tomography</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391453/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Distortion of dentition may occur in cone-beam computed tomography (CBCT) scans due to artifacts, and further imaging is frequently required to produce digital twins. The use of a plaster model is common; however, it has certain drawbacks. This study aimed to assess the feasibility of different digital dentition models over that of plaster casts. Plaster models, alginate impressions, intraoral scan (IOS) images, and CBCT images of 20 patients were obtained. The desktop model scanner was used to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391357"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related myelodysplastic syndromes in the genomics era</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391357/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myelodysplastic syndromes (t-MDS) represent a heterogeneous group of malignancies that arise as a late complication of prior exposure to chemotherapy and/or radiotherapy administered for a primary condition. T-MDS account for approximately 20% of all MDS and are characterized by resistance to current treatment strategies and poor prognosis. Our understanding of t-MDS pathogenesis has considerably improved over the last 5 years with the availability of deep sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37389519"
                        open
                      >
                        <summary class="article-expander__title">Impact of Geroscience on Therapeutic Strategies for Older Adults With Cardiovascular Disease: JACC Scientific Statement</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37389519/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Geroscience posits that cardiovascular disease (CVD) and other chronic diseases result from progressive erosion of the effectiveness of homeostatic mechanisms that oppose age-related accumulation of molecular damage. This hypothetical common root to chronic diseases explains why patients with CVD are often affected by multimorbidity and frailty and why older age negatively affects CVD prognosis and treatment response. Gerotherapeutics enhance resilience mechanisms that counter age-related...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391583"
                        open
                      >
                        <summary class="article-expander__title">Study on distribution of sidewall earth pressure on open caissons considering soil arching effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391583/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Based on the soil arching effect theory, the magnitude and distribution of sidewall earth pressure on open caissons when the embedded depth is large was analyzed by using theory of non-limit state earth pressure theory and horizontal differential element method. The theoretical formula was deduced. The theoretical calculation results are compared with the field test results and centrifugal model test results respectively. The results show that when the embedded depth of the open caisson is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391453"
                        open
                      >
                        <summary class="article-expander__title">Effectiveness of creating digital twins with different digital dentition models and cone-beam computed tomography</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391453/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Distortion of dentition may occur in cone-beam computed tomography (CBCT) scans due to artifacts, and further imaging is frequently required to produce digital twins. The use of a plaster model is common; however, it has certain drawbacks. This study aimed to assess the feasibility of different digital dentition models over that of plaster casts. Plaster models, alginate impressions, intraoral scan (IOS) images, and CBCT images of 20 patients were obtained. The desktop model scanner was used to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391357"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related myelodysplastic syndromes in the genomics era</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391357/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myelodysplastic syndromes (t-MDS) represent a heterogeneous group of malignancies that arise as a late complication of prior exposure to chemotherapy and/or radiotherapy administered for a primary condition. T-MDS account for approximately 20% of all MDS and are characterized by resistance to current treatment strategies and poor prognosis. Our understanding of t-MDS pathogenesis has considerably improved over the last 5 years with the availability of deep sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37389519"
                        open
                      >
                        <summary class="article-expander__title">Impact of Geroscience on Therapeutic Strategies for Older Adults With Cardiovascular Disease: JACC Scientific Statement</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37389519/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Geroscience posits that cardiovascular disease (CVD) and other chronic diseases result from progressive erosion of the effectiveness of homeostatic mechanisms that oppose age-related accumulation of molecular damage. This hypothetical common root to chronic diseases explains why patients with CVD are often affected by multimorbidity and frailty and why older age negatively affects CVD prognosis and treatment response. Gerotherapeutics enhance resilience mechanisms that counter age-related...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391357"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related myelodysplastic syndromes in the genomics era</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391357/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myelodysplastic syndromes (t-MDS) represent a heterogeneous group of malignancies that arise as a late complication of prior exposure to chemotherapy and/or radiotherapy administered for a primary condition. T-MDS account for approximately 20% of all MDS and are characterized by resistance to current treatment strategies and poor prognosis. Our understanding of t-MDS pathogenesis has considerably improved over the last 5 years with the availability of deep sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37389519"
                        open
                      >
                        <summary class="article-expander__title">Impact of Geroscience on Therapeutic Strategies for Older Adults With Cardiovascular Disease: JACC Scientific Statement</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37389519/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Geroscience posits that cardiovascular disease (CVD) and other chronic diseases result from progressive erosion of the effectiveness of homeostatic mechanisms that oppose age-related accumulation of molecular damage. This hypothetical common root to chronic diseases explains why patients with CVD are often affected by multimorbidity and frailty and why older age negatively affects CVD prognosis and treatment response. Gerotherapeutics enhance resilience mechanisms that counter age-related...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37391357"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related myelodysplastic syndromes in the genomics era</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37391357/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myelodysplastic syndromes (t-MDS) represent a heterogeneous group of malignancies that arise as a late complication of prior exposure to chemotherapy and/or radiotherapy administered for a primary condition. T-MDS account for approximately 20% of all MDS and are characterized by resistance to current treatment strategies and poor prognosis. Our understanding of t-MDS pathogenesis has considerably improved over the last 5 years with the availability of deep sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37389519"
                        open
                      >
                        <summary class="article-expander__title">Impact of Geroscience on Therapeutic Strategies for Older Adults With Cardiovascular Disease: JACC Scientific Statement</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37389519/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Geroscience posits that cardiovascular disease (CVD) and other chronic diseases result from progressive erosion of the effectiveness of homeostatic mechanisms that oppose age-related accumulation of molecular damage. This hypothetical common root to chronic diseases explains why patients with CVD are often affected by multimorbidity and frailty and why older age negatively affects CVD prognosis and treatment response. Gerotherapeutics enhance resilience mechanisms that counter age-related...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37389519"
                        open
                      >
                        <summary class="article-expander__title">Impact of Geroscience on Therapeutic Strategies for Older Adults With Cardiovascular Disease: JACC Scientific Statement</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37389519/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Geroscience posits that cardiovascular disease (CVD) and other chronic diseases result from progressive erosion of the effectiveness of homeostatic mechanisms that oppose age-related accumulation of molecular damage. This hypothetical common root to chronic diseases explains why patients with CVD are often affected by multimorbidity and frailty and why older age negatively affects CVD prognosis and treatment response. Gerotherapeutics enhance resilience mechanisms that counter age-related...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37387177"
                        open
                      >
                        <summary class="article-expander__title">A weighted distance-based approach for deriving consensus tumor evolutionary trees</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37387177/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations by a tumor can be modeled by a type of evolutionary tree. However, it is impossible to observe this tree directly. Instead, numerous algorithms have been developed to infer such a tree from different types of sequencing data. But such methods can produce conflicting trees for the same patient, making it desirable to have approaches that can combine several such tumor trees into a consensus or summary tree. We introduce The Weighted m-Tumor Tree...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37387177"
                        open
                      >
                        <summary class="article-expander__title">A weighted distance-based approach for deriving consensus tumor evolutionary trees</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37387177/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations by a tumor can be modeled by a type of evolutionary tree. However, it is impossible to observe this tree directly. Instead, numerous algorithms have been developed to infer such a tree from different types of sequencing data. But such methods can produce conflicting trees for the same patient, making it desirable to have approaches that can combine several such tumor trees into a consensus or summary tree. We introduce The Weighted m-Tumor Tree...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37387177"
                        open
                      >
                        <summary class="article-expander__title">A weighted distance-based approach for deriving consensus tumor evolutionary trees</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37387177/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230726205538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations by a tumor can be modeled by a type of evolutionary tree. However, it is impossible to observe this tree directly. Instead, numerous algorithms have been developed to infer such a tree from different types of sequencing data. But such methods can produce conflicting trees for the same patient, making it desirable to have approaches that can combine several such tumor trees into a consensus or summary tree. We introduce The Weighted m-Tumor Tree...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37387177"
                        open
                      >
                        <summary class="article-expander__title">A weighted distance-based approach for deriving consensus tumor evolutionary trees</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37387177/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations by a tumor can be modeled by a type of evolutionary tree. However, it is impossible to observe this tree directly. Instead, numerous algorithms have been developed to infer such a tree from different types of sequencing data. But such methods can produce conflicting trees for the same patient, making it desirable to have approaches that can combine several such tumor trees into a consensus or summary tree. We introduce The Weighted m-Tumor Tree...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37387177"
                        open
                      >
                        <summary class="article-expander__title">A weighted distance-based approach for deriving consensus tumor evolutionary trees</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37387177/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations by a tumor can be modeled by a type of evolutionary tree. However, it is impossible to observe this tree directly. Instead, numerous algorithms have been developed to infer such a tree from different types of sequencing data. But such methods can produce conflicting trees for the same patient, making it desirable to have approaches that can combine several such tumor trees into a consensus or summary tree. We introduce The Weighted m-Tumor Tree...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37387177"
                        open
                      >
                        <summary class="article-expander__title">A weighted distance-based approach for deriving consensus tumor evolutionary trees</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37387177/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230723210310&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations by a tumor can be modeled by a type of evolutionary tree. However, it is impossible to observe this tree directly. Instead, numerous algorithms have been developed to infer such a tree from different types of sequencing data. But such methods can produce conflicting trees for the same patient, making it desirable to have approaches that can combine several such tumor trees into a consensus or summary tree. We introduce The Weighted m-Tumor Tree...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37387177"
                        open
                      >
                        <summary class="article-expander__title">A weighted distance-based approach for deriving consensus tumor evolutionary trees</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37387177/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations by a tumor can be modeled by a type of evolutionary tree. However, it is impossible to observe this tree directly. Instead, numerous algorithms have been developed to infer such a tree from different types of sequencing data. But such methods can produce conflicting trees for the same patient, making it desirable to have approaches that can combine several such tumor trees into a consensus or summary tree. We introduce The Weighted m-Tumor Tree...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37387177"
                        open
                      >
                        <summary class="article-expander__title">A weighted distance-based approach for deriving consensus tumor evolutionary trees</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37387177/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations by a tumor can be modeled by a type of evolutionary tree. However, it is impossible to observe this tree directly. Instead, numerous algorithms have been developed to infer such a tree from different types of sequencing data. But such methods can produce conflicting trees for the same patient, making it desirable to have approaches that can combine several such tumor trees into a consensus or summary tree. We introduce The Weighted m-Tumor Tree...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37387177"
                        open
                      >
                        <summary class="article-expander__title">A weighted distance-based approach for deriving consensus tumor evolutionary trees</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37387177/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230720210258&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations by a tumor can be modeled by a type of evolutionary tree. However, it is impossible to observe this tree directly. Instead, numerous algorithms have been developed to infer such a tree from different types of sequencing data. But such methods can produce conflicting trees for the same patient, making it desirable to have approaches that can combine several such tumor trees into a consensus or summary tree. We introduce The Weighted m-Tumor Tree...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37387177"
                        open
                      >
                        <summary class="article-expander__title">A weighted distance-based approach for deriving consensus tumor evolutionary trees</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37387177/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations by a tumor can be modeled by a type of evolutionary tree. However, it is impossible to observe this tree directly. Instead, numerous algorithms have been developed to infer such a tree from different types of sequencing data. But such methods can produce conflicting trees for the same patient, making it desirable to have approaches that can combine several such tumor trees into a consensus or summary tree. We introduce The Weighted m-Tumor Tree...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37387177"
                        open
                      >
                        <summary class="article-expander__title">A weighted distance-based approach for deriving consensus tumor evolutionary trees</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37387177/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations by a tumor can be modeled by a type of evolutionary tree. However, it is impossible to observe this tree directly. Instead, numerous algorithms have been developed to infer such a tree from different types of sequencing data. But such methods can produce conflicting trees for the same patient, making it desirable to have approaches that can combine several such tumor trees into a consensus or summary tree. We introduce The Weighted m-Tumor Tree...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37387177"
                        open
                      >
                        <summary class="article-expander__title">A weighted distance-based approach for deriving consensus tumor evolutionary trees</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37387177/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations by a tumor can be modeled by a type of evolutionary tree. However, it is impossible to observe this tree directly. Instead, numerous algorithms have been developed to infer such a tree from different types of sequencing data. But such methods can produce conflicting trees for the same patient, making it desirable to have approaches that can combine several such tumor trees into a consensus or summary tree. We introduce The Weighted m-Tumor Tree...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37387177"
                        open
                      >
                        <summary class="article-expander__title">A weighted distance-based approach for deriving consensus tumor evolutionary trees</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37387177/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations by a tumor can be modeled by a type of evolutionary tree. However, it is impossible to observe this tree directly. Instead, numerous algorithms have been developed to infer such a tree from different types of sequencing data. But such methods can produce conflicting trees for the same patient, making it desirable to have approaches that can combine several such tumor trees into a consensus or summary tree. We introduce The Weighted m-Tumor Tree...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37387177"
                        open
                      >
                        <summary class="article-expander__title">A weighted distance-based approach for deriving consensus tumor evolutionary trees</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37387177/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations by a tumor can be modeled by a type of evolutionary tree. However, it is impossible to observe this tree directly. Instead, numerous algorithms have been developed to infer such a tree from different types of sequencing data. But such methods can produce conflicting trees for the same patient, making it desirable to have approaches that can combine several such tumor trees into a consensus or summary tree. We introduce The Weighted m-Tumor Tree...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37387177"
                        open
                      >
                        <summary class="article-expander__title">A weighted distance-based approach for deriving consensus tumor evolutionary trees</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37387177/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations by a tumor can be modeled by a type of evolutionary tree. However, it is impossible to observe this tree directly. Instead, numerous algorithms have been developed to infer such a tree from different types of sequencing data. But such methods can produce conflicting trees for the same patient, making it desirable to have approaches that can combine several such tumor trees into a consensus or summary tree. We introduce The Weighted m-Tumor Tree...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37387177"
                        open
                      >
                        <summary class="article-expander__title">A weighted distance-based approach for deriving consensus tumor evolutionary trees</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37387177/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations by a tumor can be modeled by a type of evolutionary tree. However, it is impossible to observe this tree directly. Instead, numerous algorithms have been developed to infer such a tree from different types of sequencing data. But such methods can produce conflicting trees for the same patient, making it desirable to have approaches that can combine several such tumor trees into a consensus or summary tree. We introduce The Weighted m-Tumor Tree...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37387177"
                        open
                      >
                        <summary class="article-expander__title">A weighted distance-based approach for deriving consensus tumor evolutionary trees</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37387177/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations by a tumor can be modeled by a type of evolutionary tree. However, it is impossible to observe this tree directly. Instead, numerous algorithms have been developed to infer such a tree from different types of sequencing data. But such methods can produce conflicting trees for the same patient, making it desirable to have approaches that can combine several such tumor trees into a consensus or summary tree. We introduce The Weighted m-Tumor Tree...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37387177"
                        open
                      >
                        <summary class="article-expander__title">A weighted distance-based approach for deriving consensus tumor evolutionary trees</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37387177/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations by a tumor can be modeled by a type of evolutionary tree. However, it is impossible to observe this tree directly. Instead, numerous algorithms have been developed to infer such a tree from different types of sequencing data. But such methods can produce conflicting trees for the same patient, making it desirable to have approaches that can combine several such tumor trees into a consensus or summary tree. We introduce The Weighted m-Tumor Tree...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37387177"
                        open
                      >
                        <summary class="article-expander__title">A weighted distance-based approach for deriving consensus tumor evolutionary trees</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37387177/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations by a tumor can be modeled by a type of evolutionary tree. However, it is impossible to observe this tree directly. Instead, numerous algorithms have been developed to infer such a tree from different types of sequencing data. But such methods can produce conflicting trees for the same patient, making it desirable to have approaches that can combine several such tumor trees into a consensus or summary tree. We introduce The Weighted m-Tumor Tree...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37387177"
                        open
                      >
                        <summary class="article-expander__title">A weighted distance-based approach for deriving consensus tumor evolutionary trees</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37387177/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations by a tumor can be modeled by a type of evolutionary tree. However, it is impossible to observe this tree directly. Instead, numerous algorithms have been developed to infer such a tree from different types of sequencing data. But such methods can produce conflicting trees for the same patient, making it desirable to have approaches that can combine several such tumor trees into a consensus or summary tree. We introduce The Weighted m-Tumor Tree...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37387177"
                        open
                      >
                        <summary class="article-expander__title">A weighted distance-based approach for deriving consensus tumor evolutionary trees</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37387177/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations by a tumor can be modeled by a type of evolutionary tree. However, it is impossible to observe this tree directly. Instead, numerous algorithms have been developed to infer such a tree from different types of sequencing data. But such methods can produce conflicting trees for the same patient, making it desirable to have approaches that can combine several such tumor trees into a consensus or summary tree. We introduce The Weighted m-Tumor Tree...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37387177"
                        open
                      >
                        <summary class="article-expander__title">A weighted distance-based approach for deriving consensus tumor evolutionary trees</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37387177/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations by a tumor can be modeled by a type of evolutionary tree. However, it is impossible to observe this tree directly. Instead, numerous algorithms have been developed to infer such a tree from different types of sequencing data. But such methods can produce conflicting trees for the same patient, making it desirable to have approaches that can combine several such tumor trees into a consensus or summary tree. We introduce The Weighted m-Tumor Tree...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37387177"
                        open
                      >
                        <summary class="article-expander__title">A weighted distance-based approach for deriving consensus tumor evolutionary trees</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37387177/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations by a tumor can be modeled by a type of evolutionary tree. However, it is impossible to observe this tree directly. Instead, numerous algorithms have been developed to infer such a tree from different types of sequencing data. But such methods can produce conflicting trees for the same patient, making it desirable to have approaches that can combine several such tumor trees into a consensus or summary tree. We introduce The Weighted m-Tumor Tree...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37387177"
                        open
                      >
                        <summary class="article-expander__title">A weighted distance-based approach for deriving consensus tumor evolutionary trees</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37387177/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230705211412&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations by a tumor can be modeled by a type of evolutionary tree. However, it is impossible to observe this tree directly. Instead, numerous algorithms have been developed to infer such a tree from different types of sequencing data. But such methods can produce conflicting trees for the same patient, making it desirable to have approaches that can combine several such tumor trees into a consensus or summary tree. We introduce The Weighted m-Tumor Tree...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37387177"
                        open
                      >
                        <summary class="article-expander__title">A weighted distance-based approach for deriving consensus tumor evolutionary trees</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37387177/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations by a tumor can be modeled by a type of evolutionary tree. However, it is impossible to observe this tree directly. Instead, numerous algorithms have been developed to infer such a tree from different types of sequencing data. But such methods can produce conflicting trees for the same patient, making it desirable to have approaches that can combine several such tumor trees into a consensus or summary tree. We introduce The Weighted m-Tumor Tree...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37387177"
                        open
                      >
                        <summary class="article-expander__title">A weighted distance-based approach for deriving consensus tumor evolutionary trees</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37387177/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations by a tumor can be modeled by a type of evolutionary tree. However, it is impossible to observe this tree directly. Instead, numerous algorithms have been developed to infer such a tree from different types of sequencing data. But such methods can produce conflicting trees for the same patient, making it desirable to have approaches that can combine several such tumor trees into a consensus or summary tree. We introduce The Weighted m-Tumor Tree...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-29 datetime="2023-06-29T10:00:00.000Z">2023-06-29</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-29 datetime="2023-06-29T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37381752"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis with DNMT3A and PPM1D mutations impairs regeneration in autologous stem cell transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37381752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an age-related condition driven by stem- and progenitor cells harboring recurrent mutations linked to myeloid neoplasms. Currently, potential effects on hematopoiesis, stem cell function and regenerative potential under stress conditions are unknown. We performed targeted DNA sequencing of 457 hematopoietic stem cell grafts collected for autologous stem cell transplantation (ASCT) in myeloma patients and correlated our findings with high-dimensional longitudinal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37381752"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis with DNMT3A and PPM1D mutations impairs regeneration in autologous stem cell transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37381752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an age-related condition driven by stem- and progenitor cells harboring recurrent mutations linked to myeloid neoplasms. Currently, potential effects on hematopoiesis, stem cell function and regenerative potential under stress conditions are unknown. We performed targeted DNA sequencing of 457 hematopoietic stem cell grafts collected for autologous stem cell transplantation (ASCT) in myeloma patients and correlated our findings with high-dimensional longitudinal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37381752"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis with DNMT3A and PPM1D mutations impairs regeneration in autologous stem cell transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37381752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an age-related condition driven by stem- and progenitor cells harboring recurrent mutations linked to myeloid neoplasms. Currently, potential effects on hematopoiesis, stem cell function and regenerative potential under stress conditions are unknown. We performed targeted DNA sequencing of 457 hematopoietic stem cell grafts collected for autologous stem cell transplantation (ASCT) in myeloma patients and correlated our findings with high-dimensional longitudinal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37381752"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis with DNMT3A and PPM1D mutations impairs regeneration in autologous stem cell transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37381752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an age-related condition driven by stem- and progenitor cells harboring recurrent mutations linked to myeloid neoplasms. Currently, potential effects on hematopoiesis, stem cell function and regenerative potential under stress conditions are unknown. We performed targeted DNA sequencing of 457 hematopoietic stem cell grafts collected for autologous stem cell transplantation (ASCT) in myeloma patients and correlated our findings with high-dimensional longitudinal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37381752"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis with DNMT3A and PPM1D mutations impairs regeneration in autologous stem cell transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37381752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an age-related condition driven by stem- and progenitor cells harboring recurrent mutations linked to myeloid neoplasms. Currently, potential effects on hematopoiesis, stem cell function and regenerative potential under stress conditions are unknown. We performed targeted DNA sequencing of 457 hematopoietic stem cell grafts collected for autologous stem cell transplantation (ASCT) in myeloma patients and correlated our findings with high-dimensional longitudinal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37381752"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis with DNMT3A and PPM1D mutations impairs regeneration in autologous stem cell transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37381752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an age-related condition driven by stem- and progenitor cells harboring recurrent mutations linked to myeloid neoplasms. Currently, potential effects on hematopoiesis, stem cell function and regenerative potential under stress conditions are unknown. We performed targeted DNA sequencing of 457 hematopoietic stem cell grafts collected for autologous stem cell transplantation (ASCT) in myeloma patients and correlated our findings with high-dimensional longitudinal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37381752"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis with DNMT3A and PPM1D mutations impairs regeneration in autologous stem cell transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37381752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230705211413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an age-related condition driven by stem- and progenitor cells harboring recurrent mutations linked to myeloid neoplasms. Currently, potential effects on hematopoiesis, stem cell function and regenerative potential under stress conditions are unknown. We performed targeted DNA sequencing of 457 hematopoietic stem cell grafts collected for autologous stem cell transplantation (ASCT) in myeloma patients and correlated our findings with high-dimensional longitudinal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37381752"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis with DNMT3A and PPM1D mutations impairs regeneration in autologous stem cell transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37381752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an age-related condition driven by stem- and progenitor cells harboring recurrent mutations linked to myeloid neoplasms. Currently, potential effects on hematopoiesis, stem cell function and regenerative potential under stress conditions are unknown. We performed targeted DNA sequencing of 457 hematopoietic stem cell grafts collected for autologous stem cell transplantation (ASCT) in myeloma patients and correlated our findings with high-dimensional longitudinal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37381752"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis with DNMT3A and PPM1D mutations impairs regeneration in autologous stem cell transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37381752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an age-related condition driven by stem- and progenitor cells harboring recurrent mutations linked to myeloid neoplasms. Currently, potential effects on hematopoiesis, stem cell function and regenerative potential under stress conditions are unknown. We performed targeted DNA sequencing of 457 hematopoietic stem cell grafts collected for autologous stem cell transplantation (ASCT) in myeloma patients and correlated our findings with high-dimensional longitudinal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37381752"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis with DNMT3A and PPM1D mutations impairs regeneration in autologous stem cell transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37381752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an age-related condition driven by stem- and progenitor cells harboring recurrent mutations linked to myeloid neoplasms. Currently, potential effects on hematopoiesis, stem cell function and regenerative potential under stress conditions are unknown. We performed targeted DNA sequencing of 457 hematopoietic stem cell grafts collected for autologous stem cell transplantation (ASCT) in myeloma patients and correlated our findings with high-dimensional longitudinal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37381752"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis with DNMT3A and PPM1D mutations impairs regeneration in autologous stem cell transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37381752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an age-related condition driven by stem- and progenitor cells harboring recurrent mutations linked to myeloid neoplasms. Currently, potential effects on hematopoiesis, stem cell function and regenerative potential under stress conditions are unknown. We performed targeted DNA sequencing of 457 hematopoietic stem cell grafts collected for autologous stem cell transplantation (ASCT) in myeloma patients and correlated our findings with high-dimensional longitudinal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37381752"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis with DNMT3A and PPM1D mutations impairs regeneration in autologous stem cell transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37381752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an age-related condition driven by stem- and progenitor cells harboring recurrent mutations linked to myeloid neoplasms. Currently, potential effects on hematopoiesis, stem cell function and regenerative potential under stress conditions are unknown. We performed targeted DNA sequencing of 457 hematopoietic stem cell grafts collected for autologous stem cell transplantation (ASCT) in myeloma patients and correlated our findings with high-dimensional longitudinal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37381752"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis with DNMT3A and PPM1D mutations impairs regeneration in autologous stem cell transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37381752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an age-related condition driven by stem- and progenitor cells harboring recurrent mutations linked to myeloid neoplasms. Currently, potential effects on hematopoiesis, stem cell function and regenerative potential under stress conditions are unknown. We performed targeted DNA sequencing of 457 hematopoietic stem cell grafts collected for autologous stem cell transplantation (ASCT) in myeloma patients and correlated our findings with high-dimensional longitudinal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-28 datetime="2023-06-28T10:00:00.000Z">2023-06-28</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-28 datetime="2023-06-28T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37374147"
                        open
                      >
                        <summary class="article-expander__title">Obesity and Clonal Hematopoiesis of Indeterminate Potential: Allies in Cardiovascular Diseases and Malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37374147/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clonal hematopoiesis of indeterminate potential (CHIP) is a term used to describe individuals who have detectable somatic mutations in genes commonly found in individuals with hematologic cancers but without any apparent evidence of such conditions. The mortality rate in individuals with CHIP is remarkably higher than the influence ascribed to hematologic malignancies, and it is plausible that cardiovascular diseases (CVD) could elucidate the apparent disparity. Studies have shown that the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37374147"
                        open
                      >
                        <summary class="article-expander__title">Obesity and Clonal Hematopoiesis of Indeterminate Potential: Allies in Cardiovascular Diseases and Malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37374147/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clonal hematopoiesis of indeterminate potential (CHIP) is a term used to describe individuals who have detectable somatic mutations in genes commonly found in individuals with hematologic cancers but without any apparent evidence of such conditions. The mortality rate in individuals with CHIP is remarkably higher than the influence ascribed to hematologic malignancies, and it is plausible that cardiovascular diseases (CVD) could elucidate the apparent disparity. Studies have shown that the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37374147"
                        open
                      >
                        <summary class="article-expander__title">Obesity and Clonal Hematopoiesis of Indeterminate Potential: Allies in Cardiovascular Diseases and Malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37374147/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clonal hematopoiesis of indeterminate potential (CHIP) is a term used to describe individuals who have detectable somatic mutations in genes commonly found in individuals with hematologic cancers but without any apparent evidence of such conditions. The mortality rate in individuals with CHIP is remarkably higher than the influence ascribed to hematologic malignancies, and it is plausible that cardiovascular diseases (CVD) could elucidate the apparent disparity. Studies have shown that the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37374147"
                        open
                      >
                        <summary class="article-expander__title">Obesity and Clonal Hematopoiesis of Indeterminate Potential: Allies in Cardiovascular Diseases and Malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37374147/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clonal hematopoiesis of indeterminate potential (CHIP) is a term used to describe individuals who have detectable somatic mutations in genes commonly found in individuals with hematologic cancers but without any apparent evidence of such conditions. The mortality rate in individuals with CHIP is remarkably higher than the influence ascribed to hematologic malignancies, and it is plausible that cardiovascular diseases (CVD) could elucidate the apparent disparity. Studies have shown that the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37374147"
                        open
                      >
                        <summary class="article-expander__title">Obesity and Clonal Hematopoiesis of Indeterminate Potential: Allies in Cardiovascular Diseases and Malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37374147/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clonal hematopoiesis of indeterminate potential (CHIP) is a term used to describe individuals who have detectable somatic mutations in genes commonly found in individuals with hematologic cancers but without any apparent evidence of such conditions. The mortality rate in individuals with CHIP is remarkably higher than the influence ascribed to hematologic malignancies, and it is plausible that cardiovascular diseases (CVD) could elucidate the apparent disparity. Studies have shown that the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37374147"
                        open
                      >
                        <summary class="article-expander__title">Obesity and Clonal Hematopoiesis of Indeterminate Potential: Allies in Cardiovascular Diseases and Malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37374147/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clonal hematopoiesis of indeterminate potential (CHIP) is a term used to describe individuals who have detectable somatic mutations in genes commonly found in individuals with hematologic cancers but without any apparent evidence of such conditions. The mortality rate in individuals with CHIP is remarkably higher than the influence ascribed to hematologic malignancies, and it is plausible that cardiovascular diseases (CVD) could elucidate the apparent disparity. Studies have shown that the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37374147"
                        open
                      >
                        <summary class="article-expander__title">Obesity and Clonal Hematopoiesis of Indeterminate Potential: Allies in Cardiovascular Diseases and Malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37374147/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230705211413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clonal hematopoiesis of indeterminate potential (CHIP) is a term used to describe individuals who have detectable somatic mutations in genes commonly found in individuals with hematologic cancers but without any apparent evidence of such conditions. The mortality rate in individuals with CHIP is remarkably higher than the influence ascribed to hematologic malignancies, and it is plausible that cardiovascular diseases (CVD) could elucidate the apparent disparity. Studies have shown that the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37374147"
                        open
                      >
                        <summary class="article-expander__title">Obesity and Clonal Hematopoiesis of Indeterminate Potential: Allies in Cardiovascular Diseases and Malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37374147/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clonal hematopoiesis of indeterminate potential (CHIP) is a term used to describe individuals who have detectable somatic mutations in genes commonly found in individuals with hematologic cancers but without any apparent evidence of such conditions. The mortality rate in individuals with CHIP is remarkably higher than the influence ascribed to hematologic malignancies, and it is plausible that cardiovascular diseases (CVD) could elucidate the apparent disparity. Studies have shown that the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37374147"
                        open
                      >
                        <summary class="article-expander__title">Obesity and Clonal Hematopoiesis of Indeterminate Potential: Allies in Cardiovascular Diseases and Malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37374147/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clonal hematopoiesis of indeterminate potential (CHIP) is a term used to describe individuals who have detectable somatic mutations in genes commonly found in individuals with hematologic cancers but without any apparent evidence of such conditions. The mortality rate in individuals with CHIP is remarkably higher than the influence ascribed to hematologic malignancies, and it is plausible that cardiovascular diseases (CVD) could elucidate the apparent disparity. Studies have shown that the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37374147"
                        open
                      >
                        <summary class="article-expander__title">Obesity and Clonal Hematopoiesis of Indeterminate Potential: Allies in Cardiovascular Diseases and Malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37374147/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clonal hematopoiesis of indeterminate potential (CHIP) is a term used to describe individuals who have detectable somatic mutations in genes commonly found in individuals with hematologic cancers but without any apparent evidence of such conditions. The mortality rate in individuals with CHIP is remarkably higher than the influence ascribed to hematologic malignancies, and it is plausible that cardiovascular diseases (CVD) could elucidate the apparent disparity. Studies have shown that the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37374147"
                        open
                      >
                        <summary class="article-expander__title">Obesity and Clonal Hematopoiesis of Indeterminate Potential: Allies in Cardiovascular Diseases and Malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37374147/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clonal hematopoiesis of indeterminate potential (CHIP) is a term used to describe individuals who have detectable somatic mutations in genes commonly found in individuals with hematologic cancers but without any apparent evidence of such conditions. The mortality rate in individuals with CHIP is remarkably higher than the influence ascribed to hematologic malignancies, and it is plausible that cardiovascular diseases (CVD) could elucidate the apparent disparity. Studies have shown that the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37374147"
                        open
                      >
                        <summary class="article-expander__title">Obesity and Clonal Hematopoiesis of Indeterminate Potential: Allies in Cardiovascular Diseases and Malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37374147/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clonal hematopoiesis of indeterminate potential (CHIP) is a term used to describe individuals who have detectable somatic mutations in genes commonly found in individuals with hematologic cancers but without any apparent evidence of such conditions. The mortality rate in individuals with CHIP is remarkably higher than the influence ascribed to hematologic malignancies, and it is plausible that cardiovascular diseases (CVD) could elucidate the apparent disparity. Studies have shown that the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37374147"
                        open
                      >
                        <summary class="article-expander__title">Obesity and Clonal Hematopoiesis of Indeterminate Potential: Allies in Cardiovascular Diseases and Malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37374147/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clonal hematopoiesis of indeterminate potential (CHIP) is a term used to describe individuals who have detectable somatic mutations in genes commonly found in individuals with hematologic cancers but without any apparent evidence of such conditions. The mortality rate in individuals with CHIP is remarkably higher than the influence ascribed to hematologic malignancies, and it is plausible that cardiovascular diseases (CVD) could elucidate the apparent disparity. Studies have shown that the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37374147"
                        open
                      >
                        <summary class="article-expander__title">Obesity and Clonal Hematopoiesis of Indeterminate Potential: Allies in Cardiovascular Diseases and Malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37374147/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clonal hematopoiesis of indeterminate potential (CHIP) is a term used to describe individuals who have detectable somatic mutations in genes commonly found in individuals with hematologic cancers but without any apparent evidence of such conditions. The mortality rate in individuals with CHIP is remarkably higher than the influence ascribed to hematologic malignancies, and it is plausible that cardiovascular diseases (CVD) could elucidate the apparent disparity. Studies have shown that the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230728205723&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230726205538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230723210310&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230720210258&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230705211412&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37377900"
                        open
                      >
                        <summary class="article-expander__title">Dissecting the Origin of Heterogeneity in Uterine and Ovarian Carcinosarcomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37377900/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gynecologic carcinosarcomas (CS) are biphasic neoplasms composed of carcinomatous (C) and sarcomatous (S) malignant components. Because of their rarity and histologic complexity, genetic and functional studies on CS are scarce and the mechanisms of initiation and development remain largely unknown. Whole-genome analysis of the C and S components reveals shared genomic alterations, thus emphasizing the clonal evolution of CS. Reconstructions of the evolutionary history of each tumor further...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-27 datetime="2023-06-27T10:00:00.000Z">2023-06-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-27 datetime="2023-06-27T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37370129"
                        open
                      >
                        <summary class="article-expander__title">Panhematopoietic RNA barcoding enables kinetic measurements of nucleate and anucleate lineages and the activation of myeloid clones following acute platelet depletion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37370129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our approach enables kinetic studies of multi-lineage output in the peripheral blood and transcriptional heterogeneity of individual hematopoietic stem cells. Our results give a unique insight into hematopoietic stem cell reactivation upon platelet depletion and of clonal dynamics in both steady state and under stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37370129"
                        open
                      >
                        <summary class="article-expander__title">Panhematopoietic RNA barcoding enables kinetic measurements of nucleate and anucleate lineages and the activation of myeloid clones following acute platelet depletion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37370129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our approach enables kinetic studies of multi-lineage output in the peripheral blood and transcriptional heterogeneity of individual hematopoietic stem cells. Our results give a unique insight into hematopoietic stem cell reactivation upon platelet depletion and of clonal dynamics in both steady state and under stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37370129"
                        open
                      >
                        <summary class="article-expander__title">Panhematopoietic RNA barcoding enables kinetic measurements of nucleate and anucleate lineages and the activation of myeloid clones following acute platelet depletion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37370129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our approach enables kinetic studies of multi-lineage output in the peripheral blood and transcriptional heterogeneity of individual hematopoietic stem cells. Our results give a unique insight into hematopoietic stem cell reactivation upon platelet depletion and of clonal dynamics in both steady state and under stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37370129"
                        open
                      >
                        <summary class="article-expander__title">Panhematopoietic RNA barcoding enables kinetic measurements of nucleate and anucleate lineages and the activation of myeloid clones following acute platelet depletion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37370129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our approach enables kinetic studies of multi-lineage output in the peripheral blood and transcriptional heterogeneity of individual hematopoietic stem cells. Our results give a unique insight into hematopoietic stem cell reactivation upon platelet depletion and of clonal dynamics in both steady state and under stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37370129"
                        open
                      >
                        <summary class="article-expander__title">Panhematopoietic RNA barcoding enables kinetic measurements of nucleate and anucleate lineages and the activation of myeloid clones following acute platelet depletion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37370129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230705211413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our approach enables kinetic studies of multi-lineage output in the peripheral blood and transcriptional heterogeneity of individual hematopoietic stem cells. Our results give a unique insight into hematopoietic stem cell reactivation upon platelet depletion and of clonal dynamics in both steady state and under stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37370129"
                        open
                      >
                        <summary class="article-expander__title">Panhematopoietic RNA barcoding enables kinetic measurements of nucleate and anucleate lineages and the activation of myeloid clones following acute platelet depletion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37370129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our approach enables kinetic studies of multi-lineage output in the peripheral blood and transcriptional heterogeneity of individual hematopoietic stem cells. Our results give a unique insight into hematopoietic stem cell reactivation upon platelet depletion and of clonal dynamics in both steady state and under stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37370129"
                        open
                      >
                        <summary class="article-expander__title">Panhematopoietic RNA barcoding enables kinetic measurements of nucleate and anucleate lineages and the activation of myeloid clones following acute platelet depletion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37370129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our approach enables kinetic studies of multi-lineage output in the peripheral blood and transcriptional heterogeneity of individual hematopoietic stem cells. Our results give a unique insight into hematopoietic stem cell reactivation upon platelet depletion and of clonal dynamics in both steady state and under stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37370129"
                        open
                      >
                        <summary class="article-expander__title">Panhematopoietic RNA barcoding enables kinetic measurements of nucleate and anucleate lineages and the activation of myeloid clones following acute platelet depletion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37370129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our approach enables kinetic studies of multi-lineage output in the peripheral blood and transcriptional heterogeneity of individual hematopoietic stem cells. Our results give a unique insight into hematopoietic stem cell reactivation upon platelet depletion and of clonal dynamics in both steady state and under stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37370129"
                        open
                      >
                        <summary class="article-expander__title">Panhematopoietic RNA barcoding enables kinetic measurements of nucleate and anucleate lineages and the activation of myeloid clones following acute platelet depletion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37370129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our approach enables kinetic studies of multi-lineage output in the peripheral blood and transcriptional heterogeneity of individual hematopoietic stem cells. Our results give a unique insight into hematopoietic stem cell reactivation upon platelet depletion and of clonal dynamics in both steady state and under stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37370129"
                        open
                      >
                        <summary class="article-expander__title">Panhematopoietic RNA barcoding enables kinetic measurements of nucleate and anucleate lineages and the activation of myeloid clones following acute platelet depletion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37370129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our approach enables kinetic studies of multi-lineage output in the peripheral blood and transcriptional heterogeneity of individual hematopoietic stem cells. Our results give a unique insight into hematopoietic stem cell reactivation upon platelet depletion and of clonal dynamics in both steady state and under stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37370129"
                        open
                      >
                        <summary class="article-expander__title">Panhematopoietic RNA barcoding enables kinetic measurements of nucleate and anucleate lineages and the activation of myeloid clones following acute platelet depletion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37370129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our approach enables kinetic studies of multi-lineage output in the peripheral blood and transcriptional heterogeneity of individual hematopoietic stem cells. Our results give a unique insight into hematopoietic stem cell reactivation upon platelet depletion and of clonal dynamics in both steady state and under stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37370129"
                        open
                      >
                        <summary class="article-expander__title">Panhematopoietic RNA barcoding enables kinetic measurements of nucleate and anucleate lineages and the activation of myeloid clones following acute platelet depletion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37370129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our approach enables kinetic studies of multi-lineage output in the peripheral blood and transcriptional heterogeneity of individual hematopoietic stem cells. Our results give a unique insight into hematopoietic stem cell reactivation upon platelet depletion and of clonal dynamics in both steady state and under stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230726205538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230723210310&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230720210258&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230705211412&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37367082"
                        open
                      >
                        <summary class="article-expander__title">Additional Genetic Alterations and Clonal Evolution of MPNs with Double Mutations on the MPL Gene: Two Case Reports</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37367082/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230627211357&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Essential thrombocythemia (ET) and primary myelofibrosis (PMF) are two of the main BCR-ABL1-negative chronic myeloproliferative neoplasms (MPNs) characterized by abnormal megakaryocytic proliferation. Janus kinase 2 (JAK2) mutations are detected in 50-60% of ET and PMF, while myeloproliferative leukemia (MPL) virus oncogene mutations are present in 3-5% of cases. While Sanger sequencing is a valuable diagnostic tool to discriminate the most common MPN mutations, next-generation sequencing (NGS)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-26 datetime="2023-06-26T10:00:00.000Z">2023-06-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-26 datetime="2023-06-26T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37361587"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms - blurring the lines between cancer and chronic inflammatory disorder</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37361587/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative Neoplasm (MPN) is a group of chronic blood cancers that arise from a hematopoietic stem cell (HSC) clone with somatic mutations causing constitutive activation of myeloid cytokine receptor signaling. In addition to elevated blood cell counts, MPN typically presents with increased inflammatory signaling and inflammation symptoms. Therefore, while being a clonally derived neoplasm, MPN has much in common with chronic non-cancerous inflammatory conditions, such as rheumatoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37361587"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms - blurring the lines between cancer and chronic inflammatory disorder</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37361587/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative Neoplasm (MPN) is a group of chronic blood cancers that arise from a hematopoietic stem cell (HSC) clone with somatic mutations causing constitutive activation of myeloid cytokine receptor signaling. In addition to elevated blood cell counts, MPN typically presents with increased inflammatory signaling and inflammation symptoms. Therefore, while being a clonally derived neoplasm, MPN has much in common with chronic non-cancerous inflammatory conditions, such as rheumatoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37361587"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms - blurring the lines between cancer and chronic inflammatory disorder</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37361587/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative Neoplasm (MPN) is a group of chronic blood cancers that arise from a hematopoietic stem cell (HSC) clone with somatic mutations causing constitutive activation of myeloid cytokine receptor signaling. In addition to elevated blood cell counts, MPN typically presents with increased inflammatory signaling and inflammation symptoms. Therefore, while being a clonally derived neoplasm, MPN has much in common with chronic non-cancerous inflammatory conditions, such as rheumatoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37361587"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms - blurring the lines between cancer and chronic inflammatory disorder</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37361587/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative Neoplasm (MPN) is a group of chronic blood cancers that arise from a hematopoietic stem cell (HSC) clone with somatic mutations causing constitutive activation of myeloid cytokine receptor signaling. In addition to elevated blood cell counts, MPN typically presents with increased inflammatory signaling and inflammation symptoms. Therefore, while being a clonally derived neoplasm, MPN has much in common with chronic non-cancerous inflammatory conditions, such as rheumatoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37361587"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms - blurring the lines between cancer and chronic inflammatory disorder</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37361587/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230705211413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative Neoplasm (MPN) is a group of chronic blood cancers that arise from a hematopoietic stem cell (HSC) clone with somatic mutations causing constitutive activation of myeloid cytokine receptor signaling. In addition to elevated blood cell counts, MPN typically presents with increased inflammatory signaling and inflammation symptoms. Therefore, while being a clonally derived neoplasm, MPN has much in common with chronic non-cancerous inflammatory conditions, such as rheumatoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37361587"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms - blurring the lines between cancer and chronic inflammatory disorder</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37361587/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative Neoplasm (MPN) is a group of chronic blood cancers that arise from a hematopoietic stem cell (HSC) clone with somatic mutations causing constitutive activation of myeloid cytokine receptor signaling. In addition to elevated blood cell counts, MPN typically presents with increased inflammatory signaling and inflammation symptoms. Therefore, while being a clonally derived neoplasm, MPN has much in common with chronic non-cancerous inflammatory conditions, such as rheumatoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37361587"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms - blurring the lines between cancer and chronic inflammatory disorder</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37361587/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative Neoplasm (MPN) is a group of chronic blood cancers that arise from a hematopoietic stem cell (HSC) clone with somatic mutations causing constitutive activation of myeloid cytokine receptor signaling. In addition to elevated blood cell counts, MPN typically presents with increased inflammatory signaling and inflammation symptoms. Therefore, while being a clonally derived neoplasm, MPN has much in common with chronic non-cancerous inflammatory conditions, such as rheumatoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37361587"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms - blurring the lines between cancer and chronic inflammatory disorder</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37361587/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative Neoplasm (MPN) is a group of chronic blood cancers that arise from a hematopoietic stem cell (HSC) clone with somatic mutations causing constitutive activation of myeloid cytokine receptor signaling. In addition to elevated blood cell counts, MPN typically presents with increased inflammatory signaling and inflammation symptoms. Therefore, while being a clonally derived neoplasm, MPN has much in common with chronic non-cancerous inflammatory conditions, such as rheumatoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37361587"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms - blurring the lines between cancer and chronic inflammatory disorder</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37361587/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative Neoplasm (MPN) is a group of chronic blood cancers that arise from a hematopoietic stem cell (HSC) clone with somatic mutations causing constitutive activation of myeloid cytokine receptor signaling. In addition to elevated blood cell counts, MPN typically presents with increased inflammatory signaling and inflammation symptoms. Therefore, while being a clonally derived neoplasm, MPN has much in common with chronic non-cancerous inflammatory conditions, such as rheumatoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37361587"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms - blurring the lines between cancer and chronic inflammatory disorder</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37361587/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative Neoplasm (MPN) is a group of chronic blood cancers that arise from a hematopoietic stem cell (HSC) clone with somatic mutations causing constitutive activation of myeloid cytokine receptor signaling. In addition to elevated blood cell counts, MPN typically presents with increased inflammatory signaling and inflammation symptoms. Therefore, while being a clonally derived neoplasm, MPN has much in common with chronic non-cancerous inflammatory conditions, such as rheumatoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37361587"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms - blurring the lines between cancer and chronic inflammatory disorder</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37361587/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative Neoplasm (MPN) is a group of chronic blood cancers that arise from a hematopoietic stem cell (HSC) clone with somatic mutations causing constitutive activation of myeloid cytokine receptor signaling. In addition to elevated blood cell counts, MPN typically presents with increased inflammatory signaling and inflammation symptoms. Therefore, while being a clonally derived neoplasm, MPN has much in common with chronic non-cancerous inflammatory conditions, such as rheumatoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37361587"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms - blurring the lines between cancer and chronic inflammatory disorder</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37361587/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative Neoplasm (MPN) is a group of chronic blood cancers that arise from a hematopoietic stem cell (HSC) clone with somatic mutations causing constitutive activation of myeloid cytokine receptor signaling. In addition to elevated blood cell counts, MPN typically presents with increased inflammatory signaling and inflammation symptoms. Therefore, while being a clonally derived neoplasm, MPN has much in common with chronic non-cancerous inflammatory conditions, such as rheumatoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37361587"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms - blurring the lines between cancer and chronic inflammatory disorder</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37361587/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230627211357&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative Neoplasm (MPN) is a group of chronic blood cancers that arise from a hematopoietic stem cell (HSC) clone with somatic mutations causing constitutive activation of myeloid cytokine receptor signaling. In addition to elevated blood cell counts, MPN typically presents with increased inflammatory signaling and inflammation symptoms. Therefore, while being a clonally derived neoplasm, MPN has much in common with chronic non-cancerous inflammatory conditions, such as rheumatoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37361587"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms - blurring the lines between cancer and chronic inflammatory disorder</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37361587/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230626211237&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative Neoplasm (MPN) is a group of chronic blood cancers that arise from a hematopoietic stem cell (HSC) clone with somatic mutations causing constitutive activation of myeloid cytokine receptor signaling. In addition to elevated blood cell counts, MPN typically presents with increased inflammatory signaling and inflammation symptoms. Therefore, while being a clonally derived neoplasm, MPN has much in common with chronic non-cancerous inflammatory conditions, such as rheumatoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230727205636&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;23
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230726205538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230725210130&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230724210621&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230723210310&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364233"
                        open
                      >
                        <summary class="article-expander__title">Genomics of ERBB2-Positive Breast Cancer in Young Women Before and After Exposure to Chemotherapy Plus Trastuzumab</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364233/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230723210310&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: ERBB2-positive BCs in young women displayed substantial genomic evolution after treatment with chemo + H. Approximately half of patients with paired samples demonstrated acquired and/or clonally enriched genomic changes in cancer genes. ERBB2 CN changes were uncommon. We identified several genes warranting exploration as potential mechanisms of resistance to therapy in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364233"
                        open
                      >
                        <summary class="article-expander__title">Genomics of ERBB2-Positive Breast Cancer in Young Women Before and After Exposure to Chemotherapy Plus Trastuzumab</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364233/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230722210714&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: ERBB2-positive BCs in young women displayed substantial genomic evolution after treatment with chemo + H. Approximately half of patients with paired samples demonstrated acquired and/or clonally enriched genomic changes in cancer genes. ERBB2 CN changes were uncommon. We identified several genes warranting exploration as potential mechanisms of resistance to therapy in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364233"
                        open
                      >
                        <summary class="article-expander__title">Genomics of ERBB2-Positive Breast Cancer in Young Women Before and After Exposure to Chemotherapy Plus Trastuzumab</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364233/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230721210218&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: ERBB2-positive BCs in young women displayed substantial genomic evolution after treatment with chemo + H. Approximately half of patients with paired samples demonstrated acquired and/or clonally enriched genomic changes in cancer genes. ERBB2 CN changes were uncommon. We identified several genes warranting exploration as potential mechanisms of resistance to therapy in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230720210258&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364233"
                        open
                      >
                        <summary class="article-expander__title">Genomics of ERBB2-Positive Breast Cancer in Young Women Before and After Exposure to Chemotherapy Plus Trastuzumab</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364233/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230720210258&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: ERBB2-positive BCs in young women displayed substantial genomic evolution after treatment with chemo + H. Approximately half of patients with paired samples demonstrated acquired and/or clonally enriched genomic changes in cancer genes. ERBB2 CN changes were uncommon. We identified several genes warranting exploration as potential mechanisms of resistance to therapy in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364233"
                        open
                      >
                        <summary class="article-expander__title">Genomics of ERBB2-Positive Breast Cancer in Young Women Before and After Exposure to Chemotherapy Plus Trastuzumab</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364233/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230719210227&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: ERBB2-positive BCs in young women displayed substantial genomic evolution after treatment with chemo + H. Approximately half of patients with paired samples demonstrated acquired and/or clonally enriched genomic changes in cancer genes. ERBB2 CN changes were uncommon. We identified several genes warranting exploration as potential mechanisms of resistance to therapy in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364233"
                        open
                      >
                        <summary class="article-expander__title">Genomics of ERBB2-Positive Breast Cancer in Young Women Before and After Exposure to Chemotherapy Plus Trastuzumab</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364233/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230718220955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: ERBB2-positive BCs in young women displayed substantial genomic evolution after treatment with chemo + H. Approximately half of patients with paired samples demonstrated acquired and/or clonally enriched genomic changes in cancer genes. ERBB2 CN changes were uncommon. We identified several genes warranting exploration as potential mechanisms of resistance to therapy in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364233"
                        open
                      >
                        <summary class="article-expander__title">Genomics of ERBB2-Positive Breast Cancer in Young Women Before and After Exposure to Chemotherapy Plus Trastuzumab</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364233/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: ERBB2-positive BCs in young women displayed substantial genomic evolution after treatment with chemo + H. Approximately half of patients with paired samples demonstrated acquired and/or clonally enriched genomic changes in cancer genes. ERBB2 CN changes were uncommon. We identified several genes warranting exploration as potential mechanisms of resistance to therapy in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364000"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of metastatic castration-resistant prostate cancer samples resistant to androgen-receptor pathway inhibitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230717212712&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Low AR activity, activation of stemness programs, and Hedgehog pathway were associated with primary ARPIs&#x27; resistance, whereas most acquired resistance was associated with subclonal evolution, AR-related events, and neuroendocrine differentiation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364233"
                        open
                      >
                        <summary class="article-expander__title">Genomics of ERBB2-Positive Breast Cancer in Young Women Before and After Exposure to Chemotherapy Plus Trastuzumab</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364233/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: ERBB2-positive BCs in young women displayed substantial genomic evolution after treatment with chemo + H. Approximately half of patients with paired samples demonstrated acquired and/or clonally enriched genomic changes in cancer genes. ERBB2 CN changes were uncommon. We identified several genes warranting exploration as potential mechanisms of resistance to therapy in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364000"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of metastatic castration-resistant prostate cancer samples resistant to androgen-receptor pathway inhibitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230716211510&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Low AR activity, activation of stemness programs, and Hedgehog pathway were associated with primary ARPIs&#x27; resistance, whereas most acquired resistance was associated with subclonal evolution, AR-related events, and neuroendocrine differentiation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364233"
                        open
                      >
                        <summary class="article-expander__title">Genomics of ERBB2-Positive Breast Cancer in Young Women Before and After Exposure to Chemotherapy Plus Trastuzumab</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364233/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: ERBB2-positive BCs in young women displayed substantial genomic evolution after treatment with chemo + H. Approximately half of patients with paired samples demonstrated acquired and/or clonally enriched genomic changes in cancer genes. ERBB2 CN changes were uncommon. We identified several genes warranting exploration as potential mechanisms of resistance to therapy in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364000"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of metastatic castration-resistant prostate cancer samples resistant to androgen-receptor pathway inhibitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230715212045&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Low AR activity, activation of stemness programs, and Hedgehog pathway were associated with primary ARPIs&#x27; resistance, whereas most acquired resistance was associated with subclonal evolution, AR-related events, and neuroendocrine differentiation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364233"
                        open
                      >
                        <summary class="article-expander__title">Genomics of ERBB2-Positive Breast Cancer in Young Women Before and After Exposure to Chemotherapy Plus Trastuzumab</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364233/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: ERBB2-positive BCs in young women displayed substantial genomic evolution after treatment with chemo + H. Approximately half of patients with paired samples demonstrated acquired and/or clonally enriched genomic changes in cancer genes. ERBB2 CN changes were uncommon. We identified several genes warranting exploration as potential mechanisms of resistance to therapy in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364000"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of metastatic castration-resistant prostate cancer samples resistant to androgen-receptor pathway inhibitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230714211200&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Low AR activity, activation of stemness programs, and Hedgehog pathway were associated with primary ARPIs&#x27; resistance, whereas most acquired resistance was associated with subclonal evolution, AR-related events, and neuroendocrine differentiation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364233"
                        open
                      >
                        <summary class="article-expander__title">Genomics of ERBB2-Positive Breast Cancer in Young Women Before and After Exposure to Chemotherapy Plus Trastuzumab</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364233/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: ERBB2-positive BCs in young women displayed substantial genomic evolution after treatment with chemo + H. Approximately half of patients with paired samples demonstrated acquired and/or clonally enriched genomic changes in cancer genes. ERBB2 CN changes were uncommon. We identified several genes warranting exploration as potential mechanisms of resistance to therapy in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364000"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of metastatic castration-resistant prostate cancer samples resistant to androgen-receptor pathway inhibitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Low AR activity, activation of stemness programs, and Hedgehog pathway were associated with primary ARPIs&#x27; resistance, whereas most acquired resistance was associated with subclonal evolution, AR-related events, and neuroendocrine differentiation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364233"
                        open
                      >
                        <summary class="article-expander__title">Genomics of ERBB2-Positive Breast Cancer in Young Women Before and After Exposure to Chemotherapy Plus Trastuzumab</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364233/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: ERBB2-positive BCs in young women displayed substantial genomic evolution after treatment with chemo + H. Approximately half of patients with paired samples demonstrated acquired and/or clonally enriched genomic changes in cancer genes. ERBB2 CN changes were uncommon. We identified several genes warranting exploration as potential mechanisms of resistance to therapy in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364000"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of metastatic castration-resistant prostate cancer samples resistant to androgen-receptor pathway inhibitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Low AR activity, activation of stemness programs, and Hedgehog pathway were associated with primary ARPIs&#x27; resistance, whereas most acquired resistance was associated with subclonal evolution, AR-related events, and neuroendocrine differentiation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364233"
                        open
                      >
                        <summary class="article-expander__title">Genomics of ERBB2-Positive Breast Cancer in Young Women Before and After Exposure to Chemotherapy Plus Trastuzumab</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364233/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: ERBB2-positive BCs in young women displayed substantial genomic evolution after treatment with chemo + H. Approximately half of patients with paired samples demonstrated acquired and/or clonally enriched genomic changes in cancer genes. ERBB2 CN changes were uncommon. We identified several genes warranting exploration as potential mechanisms of resistance to therapy in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364000"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of metastatic castration-resistant prostate cancer samples resistant to androgen-receptor pathway inhibitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Low AR activity, activation of stemness programs, and Hedgehog pathway were associated with primary ARPIs&#x27; resistance, whereas most acquired resistance was associated with subclonal evolution, AR-related events, and neuroendocrine differentiation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364233"
                        open
                      >
                        <summary class="article-expander__title">Genomics of ERBB2-Positive Breast Cancer in Young Women Before and After Exposure to Chemotherapy Plus Trastuzumab</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364233/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: ERBB2-positive BCs in young women displayed substantial genomic evolution after treatment with chemo + H. Approximately half of patients with paired samples demonstrated acquired and/or clonally enriched genomic changes in cancer genes. ERBB2 CN changes were uncommon. We identified several genes warranting exploration as potential mechanisms of resistance to therapy in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364000"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of metastatic castration-resistant prostate cancer samples resistant to androgen-receptor pathway inhibitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Low AR activity, activation of stemness programs, and Hedgehog pathway were associated with primary ARPIs&#x27; resistance, whereas most acquired resistance was associated with subclonal evolution, AR-related events, and neuroendocrine differentiation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364233"
                        open
                      >
                        <summary class="article-expander__title">Genomics of ERBB2-Positive Breast Cancer in Young Women Before and After Exposure to Chemotherapy Plus Trastuzumab</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364233/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: ERBB2-positive BCs in young women displayed substantial genomic evolution after treatment with chemo + H. Approximately half of patients with paired samples demonstrated acquired and/or clonally enriched genomic changes in cancer genes. ERBB2 CN changes were uncommon. We identified several genes warranting exploration as potential mechanisms of resistance to therapy in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364000"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of metastatic castration-resistant prostate cancer samples resistant to androgen-receptor pathway inhibitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Low AR activity, activation of stemness programs, and Hedgehog pathway were associated with primary ARPIs&#x27; resistance, whereas most acquired resistance was associated with subclonal evolution, AR-related events, and neuroendocrine differentiation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364233"
                        open
                      >
                        <summary class="article-expander__title">Genomics of ERBB2-Positive Breast Cancer in Young Women Before and After Exposure to Chemotherapy Plus Trastuzumab</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364233/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: ERBB2-positive BCs in young women displayed substantial genomic evolution after treatment with chemo + H. Approximately half of patients with paired samples demonstrated acquired and/or clonally enriched genomic changes in cancer genes. ERBB2 CN changes were uncommon. We identified several genes warranting exploration as potential mechanisms of resistance to therapy in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364000"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of metastatic castration-resistant prostate cancer samples resistant to androgen-receptor pathway inhibitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Low AR activity, activation of stemness programs, and Hedgehog pathway were associated with primary ARPIs&#x27; resistance, whereas most acquired resistance was associated with subclonal evolution, AR-related events, and neuroendocrine differentiation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364233"
                        open
                      >
                        <summary class="article-expander__title">Genomics of ERBB2-Positive Breast Cancer in Young Women Before and After Exposure to Chemotherapy Plus Trastuzumab</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364233/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: ERBB2-positive BCs in young women displayed substantial genomic evolution after treatment with chemo + H. Approximately half of patients with paired samples demonstrated acquired and/or clonally enriched genomic changes in cancer genes. ERBB2 CN changes were uncommon. We identified several genes warranting exploration as potential mechanisms of resistance to therapy in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364000"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of metastatic castration-resistant prostate cancer samples resistant to androgen-receptor pathway inhibitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Low AR activity, activation of stemness programs, and Hedgehog pathway were associated with primary ARPIs&#x27; resistance, whereas most acquired resistance was associated with subclonal evolution, AR-related events, and neuroendocrine differentiation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364233"
                        open
                      >
                        <summary class="article-expander__title">Genomics of ERBB2-Positive Breast Cancer in Young Women Before and After Exposure to Chemotherapy Plus Trastuzumab</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364233/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: ERBB2-positive BCs in young women displayed substantial genomic evolution after treatment with chemo + H. Approximately half of patients with paired samples demonstrated acquired and/or clonally enriched genomic changes in cancer genes. ERBB2 CN changes were uncommon. We identified several genes warranting exploration as potential mechanisms of resistance to therapy in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364000"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of metastatic castration-resistant prostate cancer samples resistant to androgen-receptor pathway inhibitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Low AR activity, activation of stemness programs, and Hedgehog pathway were associated with primary ARPIs&#x27; resistance, whereas most acquired resistance was associated with subclonal evolution, AR-related events, and neuroendocrine differentiation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230705211412&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364233"
                        open
                      >
                        <summary class="article-expander__title">Genomics of ERBB2-Positive Breast Cancer in Young Women Before and After Exposure to Chemotherapy Plus Trastuzumab</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364233/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230705211412&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: ERBB2-positive BCs in young women displayed substantial genomic evolution after treatment with chemo + H. Approximately half of patients with paired samples demonstrated acquired and/or clonally enriched genomic changes in cancer genes. ERBB2 CN changes were uncommon. We identified several genes warranting exploration as potential mechanisms of resistance to therapy in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364000"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of metastatic castration-resistant prostate cancer samples resistant to androgen-receptor pathway inhibitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230705211412&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Low AR activity, activation of stemness programs, and Hedgehog pathway were associated with primary ARPIs&#x27; resistance, whereas most acquired resistance was associated with subclonal evolution, AR-related events, and neuroendocrine differentiation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364233"
                        open
                      >
                        <summary class="article-expander__title">Genomics of ERBB2-Positive Breast Cancer in Young Women Before and After Exposure to Chemotherapy Plus Trastuzumab</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364233/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: ERBB2-positive BCs in young women displayed substantial genomic evolution after treatment with chemo + H. Approximately half of patients with paired samples demonstrated acquired and/or clonally enriched genomic changes in cancer genes. ERBB2 CN changes were uncommon. We identified several genes warranting exploration as potential mechanisms of resistance to therapy in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364000"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of metastatic castration-resistant prostate cancer samples resistant to androgen-receptor pathway inhibitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Low AR activity, activation of stemness programs, and Hedgehog pathway were associated with primary ARPIs&#x27; resistance, whereas most acquired resistance was associated with subclonal evolution, AR-related events, and neuroendocrine differentiation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364233"
                        open
                      >
                        <summary class="article-expander__title">Genomics of ERBB2-Positive Breast Cancer in Young Women Before and After Exposure to Chemotherapy Plus Trastuzumab</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364233/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: ERBB2-positive BCs in young women displayed substantial genomic evolution after treatment with chemo + H. Approximately half of patients with paired samples demonstrated acquired and/or clonally enriched genomic changes in cancer genes. ERBB2 CN changes were uncommon. We identified several genes warranting exploration as potential mechanisms of resistance to therapy in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364000"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of metastatic castration-resistant prostate cancer samples resistant to androgen-receptor pathway inhibitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Low AR activity, activation of stemness programs, and Hedgehog pathway were associated with primary ARPIs&#x27; resistance, whereas most acquired resistance was associated with subclonal evolution, AR-related events, and neuroendocrine differentiation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364233"
                        open
                      >
                        <summary class="article-expander__title">Genomics of ERBB2-Positive Breast Cancer in Young Women Before and After Exposure to Chemotherapy Plus Trastuzumab</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364233/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: ERBB2-positive BCs in young women displayed substantial genomic evolution after treatment with chemo + H. Approximately half of patients with paired samples demonstrated acquired and/or clonally enriched genomic changes in cancer genes. ERBB2 CN changes were uncommon. We identified several genes warranting exploration as potential mechanisms of resistance to therapy in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364000"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of metastatic castration-resistant prostate cancer samples resistant to androgen-receptor pathway inhibitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Low AR activity, activation of stemness programs, and Hedgehog pathway were associated with primary ARPIs&#x27; resistance, whereas most acquired resistance was associated with subclonal evolution, AR-related events, and neuroendocrine differentiation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364233"
                        open
                      >
                        <summary class="article-expander__title">Genomics of ERBB2-Positive Breast Cancer in Young Women Before and After Exposure to Chemotherapy Plus Trastuzumab</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364233/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: ERBB2-positive BCs in young women displayed substantial genomic evolution after treatment with chemo + H. Approximately half of patients with paired samples demonstrated acquired and/or clonally enriched genomic changes in cancer genes. ERBB2 CN changes were uncommon. We identified several genes warranting exploration as potential mechanisms of resistance to therapy in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364000"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of metastatic castration-resistant prostate cancer samples resistant to androgen-receptor pathway inhibitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Low AR activity, activation of stemness programs, and Hedgehog pathway were associated with primary ARPIs&#x27; resistance, whereas most acquired resistance was associated with subclonal evolution, AR-related events, and neuroendocrine differentiation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364233"
                        open
                      >
                        <summary class="article-expander__title">Genomics of ERBB2-Positive Breast Cancer in Young Women Before and After Exposure to Chemotherapy Plus Trastuzumab</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364233/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: ERBB2-positive BCs in young women displayed substantial genomic evolution after treatment with chemo + H. Approximately half of patients with paired samples demonstrated acquired and/or clonally enriched genomic changes in cancer genes. ERBB2 CN changes were uncommon. We identified several genes warranting exploration as potential mechanisms of resistance to therapy in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364000"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of metastatic castration-resistant prostate cancer samples resistant to androgen-receptor pathway inhibitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Low AR activity, activation of stemness programs, and Hedgehog pathway were associated with primary ARPIs&#x27; resistance, whereas most acquired resistance was associated with subclonal evolution, AR-related events, and neuroendocrine differentiation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364233"
                        open
                      >
                        <summary class="article-expander__title">Genomics of ERBB2-Positive Breast Cancer in Young Women Before and After Exposure to Chemotherapy Plus Trastuzumab</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364233/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: ERBB2-positive BCs in young women displayed substantial genomic evolution after treatment with chemo + H. Approximately half of patients with paired samples demonstrated acquired and/or clonally enriched genomic changes in cancer genes. ERBB2 CN changes were uncommon. We identified several genes warranting exploration as potential mechanisms of resistance to therapy in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364000"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of metastatic castration-resistant prostate cancer samples resistant to androgen-receptor pathway inhibitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Low AR activity, activation of stemness programs, and Hedgehog pathway were associated with primary ARPIs&#x27; resistance, whereas most acquired resistance was associated with subclonal evolution, AR-related events, and neuroendocrine differentiation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364233"
                        open
                      >
                        <summary class="article-expander__title">Genomics of ERBB2-Positive Breast Cancer in Young Women Before and After Exposure to Chemotherapy Plus Trastuzumab</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364233/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: ERBB2-positive BCs in young women displayed substantial genomic evolution after treatment with chemo + H. Approximately half of patients with paired samples demonstrated acquired and/or clonally enriched genomic changes in cancer genes. ERBB2 CN changes were uncommon. We identified several genes warranting exploration as potential mechanisms of resistance to therapy in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364000"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of metastatic castration-resistant prostate cancer samples resistant to androgen-receptor pathway inhibitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Low AR activity, activation of stemness programs, and Hedgehog pathway were associated with primary ARPIs&#x27; resistance, whereas most acquired resistance was associated with subclonal evolution, AR-related events, and neuroendocrine differentiation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37365326"
                        open
                      >
                        <summary class="article-expander__title">Interferon signaling promotes tolerance to chromosomal instability during metastatic evolution in renal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37365326/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230627211357&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Molecular routes to metastatic dissemination are critical determinants of aggressive cancers. Through in vivo CRISPR-Cas9 genome editing, we generated somatic mosaic genetically engineered models that faithfully recapitulate metastatic renal tumors. Disruption of 9p21 locus is an evolutionary driver to systemic disease through the rapid acquisition of complex karyotypes in cancer cells. Cross-species analysis revealed that recurrent patterns of copy number variations, including 21q loss and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364233"
                        open
                      >
                        <summary class="article-expander__title">Genomics of ERBB2-Positive Breast Cancer in Young Women Before and After Exposure to Chemotherapy Plus Trastuzumab</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364233/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230627211357&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: ERBB2-positive BCs in young women displayed substantial genomic evolution after treatment with chemo + H. Approximately half of patients with paired samples demonstrated acquired and/or clonally enriched genomic changes in cancer genes. ERBB2 CN changes were uncommon. We identified several genes warranting exploration as potential mechanisms of resistance to therapy in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364000"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of metastatic castration-resistant prostate cancer samples resistant to androgen-receptor pathway inhibitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230627211357&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Low AR activity, activation of stemness programs, and Hedgehog pathway were associated with primary ARPIs&#x27; resistance, whereas most acquired resistance was associated with subclonal evolution, AR-related events, and neuroendocrine differentiation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364233"
                        open
                      >
                        <summary class="article-expander__title">Genomics of ERBB2-Positive Breast Cancer in Young Women Before and After Exposure to Chemotherapy Plus Trastuzumab</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364233/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230626211237&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: ERBB2-positive BCs in young women displayed substantial genomic evolution after treatment with chemo + H. Approximately half of patients with paired samples demonstrated acquired and/or clonally enriched genomic changes in cancer genes. ERBB2 CN changes were uncommon. We identified several genes warranting exploration as potential mechanisms of resistance to therapy in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37364000"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of metastatic castration-resistant prostate cancer samples resistant to androgen-receptor pathway inhibitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37364000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230626211237&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Low AR activity, activation of stemness programs, and Hedgehog pathway were associated with primary ARPIs&#x27; resistance, whereas most acquired resistance was associated with subclonal evolution, AR-related events, and neuroendocrine differentiation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-25 datetime="2023-06-25T10:00:00.000Z">2023-06-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-25 datetime="2023-06-25T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356959"
                        open
                      >
                        <summary class="article-expander__title">Research Progress on Gene Mutation and CHIP in Pathogenesis of MDS --Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356959/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the development of molecular biology techniques, the people&#x27;s understanding of myelodysplastic syndromes (MDS) has greatly improved, a heterogeneous hematopoietic pre-malignant disorder of the stem cells. Gene mutations include RNA splicing, DNA methylation, chromosome modification, transcription factors, signal transduction kinases, RAS pathways, cohesion complexes, DNA repair, etc. Gene mutation is the determinant of diagnostic typing and therapeutic efficacy of MDS. The new concepts of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356959"
                        open
                      >
                        <summary class="article-expander__title">Research Progress on Gene Mutation and CHIP in Pathogenesis of MDS --Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356959/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the development of molecular biology techniques, the people&#x27;s understanding of myelodysplastic syndromes (MDS) has greatly improved, a heterogeneous hematopoietic pre-malignant disorder of the stem cells. Gene mutations include RNA splicing, DNA methylation, chromosome modification, transcription factors, signal transduction kinases, RAS pathways, cohesion complexes, DNA repair, etc. Gene mutation is the determinant of diagnostic typing and therapeutic efficacy of MDS. The new concepts of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356959"
                        open
                      >
                        <summary class="article-expander__title">Research Progress on Gene Mutation and CHIP in Pathogenesis of MDS --Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356959/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the development of molecular biology techniques, the people&#x27;s understanding of myelodysplastic syndromes (MDS) has greatly improved, a heterogeneous hematopoietic pre-malignant disorder of the stem cells. Gene mutations include RNA splicing, DNA methylation, chromosome modification, transcription factors, signal transduction kinases, RAS pathways, cohesion complexes, DNA repair, etc. Gene mutation is the determinant of diagnostic typing and therapeutic efficacy of MDS. The new concepts of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356959"
                        open
                      >
                        <summary class="article-expander__title">Research Progress on Gene Mutation and CHIP in Pathogenesis of MDS --Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356959/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230705211413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the development of molecular biology techniques, the people&#x27;s understanding of myelodysplastic syndromes (MDS) has greatly improved, a heterogeneous hematopoietic pre-malignant disorder of the stem cells. Gene mutations include RNA splicing, DNA methylation, chromosome modification, transcription factors, signal transduction kinases, RAS pathways, cohesion complexes, DNA repair, etc. Gene mutation is the determinant of diagnostic typing and therapeutic efficacy of MDS. The new concepts of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230705211413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356959"
                        open
                      >
                        <summary class="article-expander__title">Research Progress on Gene Mutation and CHIP in Pathogenesis of MDS --Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356959/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the development of molecular biology techniques, the people&#x27;s understanding of myelodysplastic syndromes (MDS) has greatly improved, a heterogeneous hematopoietic pre-malignant disorder of the stem cells. Gene mutations include RNA splicing, DNA methylation, chromosome modification, transcription factors, signal transduction kinases, RAS pathways, cohesion complexes, DNA repair, etc. Gene mutation is the determinant of diagnostic typing and therapeutic efficacy of MDS. The new concepts of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356959"
                        open
                      >
                        <summary class="article-expander__title">Research Progress on Gene Mutation and CHIP in Pathogenesis of MDS --Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356959/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the development of molecular biology techniques, the people&#x27;s understanding of myelodysplastic syndromes (MDS) has greatly improved, a heterogeneous hematopoietic pre-malignant disorder of the stem cells. Gene mutations include RNA splicing, DNA methylation, chromosome modification, transcription factors, signal transduction kinases, RAS pathways, cohesion complexes, DNA repair, etc. Gene mutation is the determinant of diagnostic typing and therapeutic efficacy of MDS. The new concepts of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356959"
                        open
                      >
                        <summary class="article-expander__title">Research Progress on Gene Mutation and CHIP in Pathogenesis of MDS --Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356959/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the development of molecular biology techniques, the people&#x27;s understanding of myelodysplastic syndromes (MDS) has greatly improved, a heterogeneous hematopoietic pre-malignant disorder of the stem cells. Gene mutations include RNA splicing, DNA methylation, chromosome modification, transcription factors, signal transduction kinases, RAS pathways, cohesion complexes, DNA repair, etc. Gene mutation is the determinant of diagnostic typing and therapeutic efficacy of MDS. The new concepts of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356959"
                        open
                      >
                        <summary class="article-expander__title">Research Progress on Gene Mutation and CHIP in Pathogenesis of MDS --Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356959/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the development of molecular biology techniques, the people&#x27;s understanding of myelodysplastic syndromes (MDS) has greatly improved, a heterogeneous hematopoietic pre-malignant disorder of the stem cells. Gene mutations include RNA splicing, DNA methylation, chromosome modification, transcription factors, signal transduction kinases, RAS pathways, cohesion complexes, DNA repair, etc. Gene mutation is the determinant of diagnostic typing and therapeutic efficacy of MDS. The new concepts of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356959"
                        open
                      >
                        <summary class="article-expander__title">Research Progress on Gene Mutation and CHIP in Pathogenesis of MDS --Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356959/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the development of molecular biology techniques, the people&#x27;s understanding of myelodysplastic syndromes (MDS) has greatly improved, a heterogeneous hematopoietic pre-malignant disorder of the stem cells. Gene mutations include RNA splicing, DNA methylation, chromosome modification, transcription factors, signal transduction kinases, RAS pathways, cohesion complexes, DNA repair, etc. Gene mutation is the determinant of diagnostic typing and therapeutic efficacy of MDS. The new concepts of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356959"
                        open
                      >
                        <summary class="article-expander__title">Research Progress on Gene Mutation and CHIP in Pathogenesis of MDS --Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356959/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the development of molecular biology techniques, the people&#x27;s understanding of myelodysplastic syndromes (MDS) has greatly improved, a heterogeneous hematopoietic pre-malignant disorder of the stem cells. Gene mutations include RNA splicing, DNA methylation, chromosome modification, transcription factors, signal transduction kinases, RAS pathways, cohesion complexes, DNA repair, etc. Gene mutation is the determinant of diagnostic typing and therapeutic efficacy of MDS. The new concepts of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356959"
                        open
                      >
                        <summary class="article-expander__title">Research Progress on Gene Mutation and CHIP in Pathogenesis of MDS --Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356959/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the development of molecular biology techniques, the people&#x27;s understanding of myelodysplastic syndromes (MDS) has greatly improved, a heterogeneous hematopoietic pre-malignant disorder of the stem cells. Gene mutations include RNA splicing, DNA methylation, chromosome modification, transcription factors, signal transduction kinases, RAS pathways, cohesion complexes, DNA repair, etc. Gene mutation is the determinant of diagnostic typing and therapeutic efficacy of MDS. The new concepts of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356959"
                        open
                      >
                        <summary class="article-expander__title">Research Progress on Gene Mutation and CHIP in Pathogenesis of MDS --Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356959/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230627211357&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the development of molecular biology techniques, the people&#x27;s understanding of myelodysplastic syndromes (MDS) has greatly improved, a heterogeneous hematopoietic pre-malignant disorder of the stem cells. Gene mutations include RNA splicing, DNA methylation, chromosome modification, transcription factors, signal transduction kinases, RAS pathways, cohesion complexes, DNA repair, etc. Gene mutation is the determinant of diagnostic typing and therapeutic efficacy of MDS. The new concepts of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230627211357&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356959"
                        open
                      >
                        <summary class="article-expander__title">Research Progress on Gene Mutation and CHIP in Pathogenesis of MDS --Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356959/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230626211237&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the development of molecular biology techniques, the people&#x27;s understanding of myelodysplastic syndromes (MDS) has greatly improved, a heterogeneous hematopoietic pre-malignant disorder of the stem cells. Gene mutations include RNA splicing, DNA methylation, chromosome modification, transcription factors, signal transduction kinases, RAS pathways, cohesion complexes, DNA repair, etc. Gene mutation is the determinant of diagnostic typing and therapeutic efficacy of MDS. The new concepts of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230626211237&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230713211352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230712211426&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230705211412&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230627211357&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230626211237&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-24 datetime="2023-06-24T10:00:00.000Z">2023-06-24</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-24 datetime="2023-06-24T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37355534"
                        open
                      >
                        <summary class="article-expander__title">In Vitro and In Vivo Models for Metastatic Intestinal Tumors Using Genotype-Defined Organoids</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37355534/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230711211114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It has been established that the accumulation of driver gene mutations causes malignant progression of colorectal cancer (CRC) through positive selection and clonal expansion, similar to Darwin&#x27;s evolution. Following this multistep tumorigenesis concept, we previously showed the specific mutation patterns for each process of malignant progression, including submucosal invasion, epithelial mesenchymal transition (EMT), intravasation, and metastasis, using genetically engineered mouse and organoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37355534"
                        open
                      >
                        <summary class="article-expander__title">In Vitro and In Vivo Models for Metastatic Intestinal Tumors Using Genotype-Defined Organoids</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37355534/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230710210549&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It has been established that the accumulation of driver gene mutations causes malignant progression of colorectal cancer (CRC) through positive selection and clonal expansion, similar to Darwin&#x27;s evolution. Following this multistep tumorigenesis concept, we previously showed the specific mutation patterns for each process of malignant progression, including submucosal invasion, epithelial mesenchymal transition (EMT), intravasation, and metastasis, using genetically engineered mouse and organoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37355534"
                        open
                      >
                        <summary class="article-expander__title">In Vitro and In Vivo Models for Metastatic Intestinal Tumors Using Genotype-Defined Organoids</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37355534/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It has been established that the accumulation of driver gene mutations causes malignant progression of colorectal cancer (CRC) through positive selection and clonal expansion, similar to Darwin&#x27;s evolution. Following this multistep tumorigenesis concept, we previously showed the specific mutation patterns for each process of malignant progression, including submucosal invasion, epithelial mesenchymal transition (EMT), intravasation, and metastasis, using genetically engineered mouse and organoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37355534"
                        open
                      >
                        <summary class="article-expander__title">In Vitro and In Vivo Models for Metastatic Intestinal Tumors Using Genotype-Defined Organoids</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37355534/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It has been established that the accumulation of driver gene mutations causes malignant progression of colorectal cancer (CRC) through positive selection and clonal expansion, similar to Darwin&#x27;s evolution. Following this multistep tumorigenesis concept, we previously showed the specific mutation patterns for each process of malignant progression, including submucosal invasion, epithelial mesenchymal transition (EMT), intravasation, and metastasis, using genetically engineered mouse and organoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37355534"
                        open
                      >
                        <summary class="article-expander__title">In Vitro and In Vivo Models for Metastatic Intestinal Tumors Using Genotype-Defined Organoids</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37355534/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It has been established that the accumulation of driver gene mutations causes malignant progression of colorectal cancer (CRC) through positive selection and clonal expansion, similar to Darwin&#x27;s evolution. Following this multistep tumorigenesis concept, we previously showed the specific mutation patterns for each process of malignant progression, including submucosal invasion, epithelial mesenchymal transition (EMT), intravasation, and metastasis, using genetically engineered mouse and organoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37355534"
                        open
                      >
                        <summary class="article-expander__title">In Vitro and In Vivo Models for Metastatic Intestinal Tumors Using Genotype-Defined Organoids</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37355534/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It has been established that the accumulation of driver gene mutations causes malignant progression of colorectal cancer (CRC) through positive selection and clonal expansion, similar to Darwin&#x27;s evolution. Following this multistep tumorigenesis concept, we previously showed the specific mutation patterns for each process of malignant progression, including submucosal invasion, epithelial mesenchymal transition (EMT), intravasation, and metastasis, using genetically engineered mouse and organoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37355534"
                        open
                      >
                        <summary class="article-expander__title">In Vitro and In Vivo Models for Metastatic Intestinal Tumors Using Genotype-Defined Organoids</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37355534/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230705211412&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It has been established that the accumulation of driver gene mutations causes malignant progression of colorectal cancer (CRC) through positive selection and clonal expansion, similar to Darwin&#x27;s evolution. Following this multistep tumorigenesis concept, we previously showed the specific mutation patterns for each process of malignant progression, including submucosal invasion, epithelial mesenchymal transition (EMT), intravasation, and metastasis, using genetically engineered mouse and organoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37355534"
                        open
                      >
                        <summary class="article-expander__title">In Vitro and In Vivo Models for Metastatic Intestinal Tumors Using Genotype-Defined Organoids</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37355534/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It has been established that the accumulation of driver gene mutations causes malignant progression of colorectal cancer (CRC) through positive selection and clonal expansion, similar to Darwin&#x27;s evolution. Following this multistep tumorigenesis concept, we previously showed the specific mutation patterns for each process of malignant progression, including submucosal invasion, epithelial mesenchymal transition (EMT), intravasation, and metastasis, using genetically engineered mouse and organoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37355534"
                        open
                      >
                        <summary class="article-expander__title">In Vitro and In Vivo Models for Metastatic Intestinal Tumors Using Genotype-Defined Organoids</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37355534/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It has been established that the accumulation of driver gene mutations causes malignant progression of colorectal cancer (CRC) through positive selection and clonal expansion, similar to Darwin&#x27;s evolution. Following this multistep tumorigenesis concept, we previously showed the specific mutation patterns for each process of malignant progression, including submucosal invasion, epithelial mesenchymal transition (EMT), intravasation, and metastasis, using genetically engineered mouse and organoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37355534"
                        open
                      >
                        <summary class="article-expander__title">In Vitro and In Vivo Models for Metastatic Intestinal Tumors Using Genotype-Defined Organoids</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37355534/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It has been established that the accumulation of driver gene mutations causes malignant progression of colorectal cancer (CRC) through positive selection and clonal expansion, similar to Darwin&#x27;s evolution. Following this multistep tumorigenesis concept, we previously showed the specific mutation patterns for each process of malignant progression, including submucosal invasion, epithelial mesenchymal transition (EMT), intravasation, and metastasis, using genetically engineered mouse and organoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37355534"
                        open
                      >
                        <summary class="article-expander__title">In Vitro and In Vivo Models for Metastatic Intestinal Tumors Using Genotype-Defined Organoids</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37355534/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It has been established that the accumulation of driver gene mutations causes malignant progression of colorectal cancer (CRC) through positive selection and clonal expansion, similar to Darwin&#x27;s evolution. Following this multistep tumorigenesis concept, we previously showed the specific mutation patterns for each process of malignant progression, including submucosal invasion, epithelial mesenchymal transition (EMT), intravasation, and metastasis, using genetically engineered mouse and organoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37355534"
                        open
                      >
                        <summary class="article-expander__title">In Vitro and In Vivo Models for Metastatic Intestinal Tumors Using Genotype-Defined Organoids</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37355534/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It has been established that the accumulation of driver gene mutations causes malignant progression of colorectal cancer (CRC) through positive selection and clonal expansion, similar to Darwin&#x27;s evolution. Following this multistep tumorigenesis concept, we previously showed the specific mutation patterns for each process of malignant progression, including submucosal invasion, epithelial mesenchymal transition (EMT), intravasation, and metastasis, using genetically engineered mouse and organoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37355534"
                        open
                      >
                        <summary class="article-expander__title">In Vitro and In Vivo Models for Metastatic Intestinal Tumors Using Genotype-Defined Organoids</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37355534/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It has been established that the accumulation of driver gene mutations causes malignant progression of colorectal cancer (CRC) through positive selection and clonal expansion, similar to Darwin&#x27;s evolution. Following this multistep tumorigenesis concept, we previously showed the specific mutation patterns for each process of malignant progression, including submucosal invasion, epithelial mesenchymal transition (EMT), intravasation, and metastasis, using genetically engineered mouse and organoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37355534"
                        open
                      >
                        <summary class="article-expander__title">In Vitro and In Vivo Models for Metastatic Intestinal Tumors Using Genotype-Defined Organoids</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37355534/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It has been established that the accumulation of driver gene mutations causes malignant progression of colorectal cancer (CRC) through positive selection and clonal expansion, similar to Darwin&#x27;s evolution. Following this multistep tumorigenesis concept, we previously showed the specific mutation patterns for each process of malignant progression, including submucosal invasion, epithelial mesenchymal transition (EMT), intravasation, and metastasis, using genetically engineered mouse and organoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37355534"
                        open
                      >
                        <summary class="article-expander__title">In Vitro and In Vivo Models for Metastatic Intestinal Tumors Using Genotype-Defined Organoids</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37355534/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230627211357&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It has been established that the accumulation of driver gene mutations causes malignant progression of colorectal cancer (CRC) through positive selection and clonal expansion, similar to Darwin&#x27;s evolution. Following this multistep tumorigenesis concept, we previously showed the specific mutation patterns for each process of malignant progression, including submucosal invasion, epithelial mesenchymal transition (EMT), intravasation, and metastasis, using genetically engineered mouse and organoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37355534"
                        open
                      >
                        <summary class="article-expander__title">In Vitro and In Vivo Models for Metastatic Intestinal Tumors Using Genotype-Defined Organoids</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37355534/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230626211237&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It has been established that the accumulation of driver gene mutations causes malignant progression of colorectal cancer (CRC) through positive selection and clonal expansion, similar to Darwin&#x27;s evolution. Following this multistep tumorigenesis concept, we previously showed the specific mutation patterns for each process of malignant progression, including submucosal invasion, epithelial mesenchymal transition (EMT), intravasation, and metastasis, using genetically engineered mouse and organoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37355534"
                        open
                      >
                        <summary class="article-expander__title">In Vitro and In Vivo Models for Metastatic Intestinal Tumors Using Genotype-Defined Organoids</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37355534/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It has been established that the accumulation of driver gene mutations causes malignant progression of colorectal cancer (CRC) through positive selection and clonal expansion, similar to Darwin&#x27;s evolution. Following this multistep tumorigenesis concept, we previously showed the specific mutation patterns for each process of malignant progression, including submucosal invasion, epithelial mesenchymal transition (EMT), intravasation, and metastasis, using genetically engineered mouse and organoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-22 datetime="2023-06-22T10:00:00.000Z">2023-06-22</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-22 datetime="2023-06-22T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37349125"
                        open
                      >
                        <summary class="article-expander__title">Genomic approaches to cancer and minimal residual disease detection using circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37349125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230705211413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsies using cell-free circulating tumor DNA (ctDNA) are being used frequently in both research and clinical settings. ctDNA can be used to identify actionable mutations to personalize systemic therapy, detect post-treatment minimal residual disease (MRD), and predict responses to immunotherapy. ctDNA can also be isolated from a range of different biofluids, with the possibility of detecting locoregional MRD with increased sensitivity if sampling more proximally than blood plasma....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37349125"
                        open
                      >
                        <summary class="article-expander__title">Genomic approaches to cancer and minimal residual disease detection using circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37349125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsies using cell-free circulating tumor DNA (ctDNA) are being used frequently in both research and clinical settings. ctDNA can be used to identify actionable mutations to personalize systemic therapy, detect post-treatment minimal residual disease (MRD), and predict responses to immunotherapy. ctDNA can also be isolated from a range of different biofluids, with the possibility of detecting locoregional MRD with increased sensitivity if sampling more proximally than blood plasma....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37345823"
                        open
                      >
                        <summary class="article-expander__title">Birth Weight Is Associated With Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Outcomes in Adulthood</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37345823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background High and low birth weight are independently associated with increased cardiovascular disease risk in adulthood. Clonal hematopoiesis of indeterminate potential (CHIP), the age-related clonal expansion of hematopoietic cells with preleukemic somatic mutations, predicts incident cardiovascular disease independent of traditional cardiovascular risk factors. Whether birth weight predicts development of CHIP later in life is unknown. Methods and Results A total of 221 047 adults enrolled...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37349125"
                        open
                      >
                        <summary class="article-expander__title">Genomic approaches to cancer and minimal residual disease detection using circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37349125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsies using cell-free circulating tumor DNA (ctDNA) are being used frequently in both research and clinical settings. ctDNA can be used to identify actionable mutations to personalize systemic therapy, detect post-treatment minimal residual disease (MRD), and predict responses to immunotherapy. ctDNA can also be isolated from a range of different biofluids, with the possibility of detecting locoregional MRD with increased sensitivity if sampling more proximally than blood plasma....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37345823"
                        open
                      >
                        <summary class="article-expander__title">Birth Weight Is Associated With Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Outcomes in Adulthood</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37345823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background High and low birth weight are independently associated with increased cardiovascular disease risk in adulthood. Clonal hematopoiesis of indeterminate potential (CHIP), the age-related clonal expansion of hematopoietic cells with preleukemic somatic mutations, predicts incident cardiovascular disease independent of traditional cardiovascular risk factors. Whether birth weight predicts development of CHIP later in life is unknown. Methods and Results A total of 221 047 adults enrolled...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37349125"
                        open
                      >
                        <summary class="article-expander__title">Genomic approaches to cancer and minimal residual disease detection using circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37349125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsies using cell-free circulating tumor DNA (ctDNA) are being used frequently in both research and clinical settings. ctDNA can be used to identify actionable mutations to personalize systemic therapy, detect post-treatment minimal residual disease (MRD), and predict responses to immunotherapy. ctDNA can also be isolated from a range of different biofluids, with the possibility of detecting locoregional MRD with increased sensitivity if sampling more proximally than blood plasma....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37345823"
                        open
                      >
                        <summary class="article-expander__title">Birth Weight Is Associated With Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Outcomes in Adulthood</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37345823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background High and low birth weight are independently associated with increased cardiovascular disease risk in adulthood. Clonal hematopoiesis of indeterminate potential (CHIP), the age-related clonal expansion of hematopoietic cells with preleukemic somatic mutations, predicts incident cardiovascular disease independent of traditional cardiovascular risk factors. Whether birth weight predicts development of CHIP later in life is unknown. Methods and Results A total of 221 047 adults enrolled...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37349125"
                        open
                      >
                        <summary class="article-expander__title">Genomic approaches to cancer and minimal residual disease detection using circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37349125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsies using cell-free circulating tumor DNA (ctDNA) are being used frequently in both research and clinical settings. ctDNA can be used to identify actionable mutations to personalize systemic therapy, detect post-treatment minimal residual disease (MRD), and predict responses to immunotherapy. ctDNA can also be isolated from a range of different biofluids, with the possibility of detecting locoregional MRD with increased sensitivity if sampling more proximally than blood plasma....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37345823"
                        open
                      >
                        <summary class="article-expander__title">Birth Weight Is Associated With Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Outcomes in Adulthood</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37345823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background High and low birth weight are independently associated with increased cardiovascular disease risk in adulthood. Clonal hematopoiesis of indeterminate potential (CHIP), the age-related clonal expansion of hematopoietic cells with preleukemic somatic mutations, predicts incident cardiovascular disease independent of traditional cardiovascular risk factors. Whether birth weight predicts development of CHIP later in life is unknown. Methods and Results A total of 221 047 adults enrolled...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37349125"
                        open
                      >
                        <summary class="article-expander__title">Genomic approaches to cancer and minimal residual disease detection using circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37349125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsies using cell-free circulating tumor DNA (ctDNA) are being used frequently in both research and clinical settings. ctDNA can be used to identify actionable mutations to personalize systemic therapy, detect post-treatment minimal residual disease (MRD), and predict responses to immunotherapy. ctDNA can also be isolated from a range of different biofluids, with the possibility of detecting locoregional MRD with increased sensitivity if sampling more proximally than blood plasma....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37345823"
                        open
                      >
                        <summary class="article-expander__title">Birth Weight Is Associated With Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Outcomes in Adulthood</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37345823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background High and low birth weight are independently associated with increased cardiovascular disease risk in adulthood. Clonal hematopoiesis of indeterminate potential (CHIP), the age-related clonal expansion of hematopoietic cells with preleukemic somatic mutations, predicts incident cardiovascular disease independent of traditional cardiovascular risk factors. Whether birth weight predicts development of CHIP later in life is unknown. Methods and Results A total of 221 047 adults enrolled...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37349125"
                        open
                      >
                        <summary class="article-expander__title">Genomic approaches to cancer and minimal residual disease detection using circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37349125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsies using cell-free circulating tumor DNA (ctDNA) are being used frequently in both research and clinical settings. ctDNA can be used to identify actionable mutations to personalize systemic therapy, detect post-treatment minimal residual disease (MRD), and predict responses to immunotherapy. ctDNA can also be isolated from a range of different biofluids, with the possibility of detecting locoregional MRD with increased sensitivity if sampling more proximally than blood plasma....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37345823"
                        open
                      >
                        <summary class="article-expander__title">Birth Weight Is Associated With Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Outcomes in Adulthood</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37345823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background High and low birth weight are independently associated with increased cardiovascular disease risk in adulthood. Clonal hematopoiesis of indeterminate potential (CHIP), the age-related clonal expansion of hematopoietic cells with preleukemic somatic mutations, predicts incident cardiovascular disease independent of traditional cardiovascular risk factors. Whether birth weight predicts development of CHIP later in life is unknown. Methods and Results A total of 221 047 adults enrolled...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37349125"
                        open
                      >
                        <summary class="article-expander__title">Genomic approaches to cancer and minimal residual disease detection using circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37349125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsies using cell-free circulating tumor DNA (ctDNA) are being used frequently in both research and clinical settings. ctDNA can be used to identify actionable mutations to personalize systemic therapy, detect post-treatment minimal residual disease (MRD), and predict responses to immunotherapy. ctDNA can also be isolated from a range of different biofluids, with the possibility of detecting locoregional MRD with increased sensitivity if sampling more proximally than blood plasma....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37345823"
                        open
                      >
                        <summary class="article-expander__title">Birth Weight Is Associated With Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Outcomes in Adulthood</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37345823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background High and low birth weight are independently associated with increased cardiovascular disease risk in adulthood. Clonal hematopoiesis of indeterminate potential (CHIP), the age-related clonal expansion of hematopoietic cells with preleukemic somatic mutations, predicts incident cardiovascular disease independent of traditional cardiovascular risk factors. Whether birth weight predicts development of CHIP later in life is unknown. Methods and Results A total of 221 047 adults enrolled...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37349125"
                        open
                      >
                        <summary class="article-expander__title">Genomic approaches to cancer and minimal residual disease detection using circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37349125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230627211357&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsies using cell-free circulating tumor DNA (ctDNA) are being used frequently in both research and clinical settings. ctDNA can be used to identify actionable mutations to personalize systemic therapy, detect post-treatment minimal residual disease (MRD), and predict responses to immunotherapy. ctDNA can also be isolated from a range of different biofluids, with the possibility of detecting locoregional MRD with increased sensitivity if sampling more proximally than blood plasma....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37345823"
                        open
                      >
                        <summary class="article-expander__title">Birth Weight Is Associated With Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Outcomes in Adulthood</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37345823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230627211357&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background High and low birth weight are independently associated with increased cardiovascular disease risk in adulthood. Clonal hematopoiesis of indeterminate potential (CHIP), the age-related clonal expansion of hematopoietic cells with preleukemic somatic mutations, predicts incident cardiovascular disease independent of traditional cardiovascular risk factors. Whether birth weight predicts development of CHIP later in life is unknown. Methods and Results A total of 221 047 adults enrolled...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37349125"
                        open
                      >
                        <summary class="article-expander__title">Genomic approaches to cancer and minimal residual disease detection using circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37349125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230626211237&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsies using cell-free circulating tumor DNA (ctDNA) are being used frequently in both research and clinical settings. ctDNA can be used to identify actionable mutations to personalize systemic therapy, detect post-treatment minimal residual disease (MRD), and predict responses to immunotherapy. ctDNA can also be isolated from a range of different biofluids, with the possibility of detecting locoregional MRD with increased sensitivity if sampling more proximally than blood plasma....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37345823"
                        open
                      >
                        <summary class="article-expander__title">Birth Weight Is Associated With Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Outcomes in Adulthood</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37345823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230626211237&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background High and low birth weight are independently associated with increased cardiovascular disease risk in adulthood. Clonal hematopoiesis of indeterminate potential (CHIP), the age-related clonal expansion of hematopoietic cells with preleukemic somatic mutations, predicts incident cardiovascular disease independent of traditional cardiovascular risk factors. Whether birth weight predicts development of CHIP later in life is unknown. Methods and Results A total of 221 047 adults enrolled...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37349125"
                        open
                      >
                        <summary class="article-expander__title">Genomic approaches to cancer and minimal residual disease detection using circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37349125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsies using cell-free circulating tumor DNA (ctDNA) are being used frequently in both research and clinical settings. ctDNA can be used to identify actionable mutations to personalize systemic therapy, detect post-treatment minimal residual disease (MRD), and predict responses to immunotherapy. ctDNA can also be isolated from a range of different biofluids, with the possibility of detecting locoregional MRD with increased sensitivity if sampling more proximally than blood plasma....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37345823"
                        open
                      >
                        <summary class="article-expander__title">Birth Weight Is Associated With Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Outcomes in Adulthood</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37345823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background High and low birth weight are independently associated with increased cardiovascular disease risk in adulthood. Clonal hematopoiesis of indeterminate potential (CHIP), the age-related clonal expansion of hematopoietic cells with preleukemic somatic mutations, predicts incident cardiovascular disease independent of traditional cardiovascular risk factors. Whether birth weight predicts development of CHIP later in life is unknown. Methods and Results A total of 221 047 adults enrolled...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37349125"
                        open
                      >
                        <summary class="article-expander__title">Genomic approaches to cancer and minimal residual disease detection using circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37349125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsies using cell-free circulating tumor DNA (ctDNA) are being used frequently in both research and clinical settings. ctDNA can be used to identify actionable mutations to personalize systemic therapy, detect post-treatment minimal residual disease (MRD), and predict responses to immunotherapy. ctDNA can also be isolated from a range of different biofluids, with the possibility of detecting locoregional MRD with increased sensitivity if sampling more proximally than blood plasma....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37345823"
                        open
                      >
                        <summary class="article-expander__title">Birth Weight Is Associated With Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Outcomes in Adulthood</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37345823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background High and low birth weight are independently associated with increased cardiovascular disease risk in adulthood. Clonal hematopoiesis of indeterminate potential (CHIP), the age-related clonal expansion of hematopoietic cells with preleukemic somatic mutations, predicts incident cardiovascular disease independent of traditional cardiovascular risk factors. Whether birth weight predicts development of CHIP later in life is unknown. Methods and Results A total of 221 047 adults enrolled...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37349125"
                        open
                      >
                        <summary class="article-expander__title">Genomic approaches to cancer and minimal residual disease detection using circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37349125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsies using cell-free circulating tumor DNA (ctDNA) are being used frequently in both research and clinical settings. ctDNA can be used to identify actionable mutations to personalize systemic therapy, detect post-treatment minimal residual disease (MRD), and predict responses to immunotherapy. ctDNA can also be isolated from a range of different biofluids, with the possibility of detecting locoregional MRD with increased sensitivity if sampling more proximally than blood plasma....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37345823"
                        open
                      >
                        <summary class="article-expander__title">Birth Weight Is Associated With Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Outcomes in Adulthood</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37345823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background High and low birth weight are independently associated with increased cardiovascular disease risk in adulthood. Clonal hematopoiesis of indeterminate potential (CHIP), the age-related clonal expansion of hematopoietic cells with preleukemic somatic mutations, predicts incident cardiovascular disease independent of traditional cardiovascular risk factors. Whether birth weight predicts development of CHIP later in life is unknown. Methods and Results A total of 221 047 adults enrolled...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37345823"
                        open
                      >
                        <summary class="article-expander__title">Birth Weight Is Associated With Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Outcomes in Adulthood</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37345823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background High and low birth weight are independently associated with increased cardiovascular disease risk in adulthood. Clonal hematopoiesis of indeterminate potential (CHIP), the age-related clonal expansion of hematopoietic cells with preleukemic somatic mutations, predicts incident cardiovascular disease independent of traditional cardiovascular risk factors. Whether birth weight predicts development of CHIP later in life is unknown. Methods and Results A total of 221 047 adults enrolled...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-21 datetime="2023-06-21T10:00:00.000Z">2023-06-21</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-21 datetime="2023-06-21T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344430"
                        open
                      >
                        <summary class="article-expander__title">Bilateral Adrenal Infarction that Developed in Latent Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344430/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bilateral adrenal infarction is an extremely rare disease, and it has been reported that some coagulation abnormalities, including essential thrombocythemia (ET), exist in the background. We herein report a 76-year-old patient in whom the platelet count had been in the normal range at the onset of adrenal infarction but subsequently increased to 102×10⁴/μL at 7 months later, leading to the diagnosis of JAK2V617F-positive ET. As the presence of the JAK2V617F mutation increases the risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344430"
                        open
                      >
                        <summary class="article-expander__title">Bilateral Adrenal Infarction that Developed in Latent Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344430/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bilateral adrenal infarction is an extremely rare disease, and it has been reported that some coagulation abnormalities, including essential thrombocythemia (ET), exist in the background. We herein report a 76-year-old patient in whom the platelet count had been in the normal range at the onset of adrenal infarction but subsequently increased to 102×10⁴/μL at 7 months later, leading to the diagnosis of JAK2V617F-positive ET. As the presence of the JAK2V617F mutation increases the risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344430"
                        open
                      >
                        <summary class="article-expander__title">Bilateral Adrenal Infarction that Developed in Latent Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344430/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bilateral adrenal infarction is an extremely rare disease, and it has been reported that some coagulation abnormalities, including essential thrombocythemia (ET), exist in the background. We herein report a 76-year-old patient in whom the platelet count had been in the normal range at the onset of adrenal infarction but subsequently increased to 102×10⁴/μL at 7 months later, leading to the diagnosis of JAK2V617F-positive ET. As the presence of the JAK2V617F mutation increases the risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344430"
                        open
                      >
                        <summary class="article-expander__title">Bilateral Adrenal Infarction that Developed in Latent Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344430/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bilateral adrenal infarction is an extremely rare disease, and it has been reported that some coagulation abnormalities, including essential thrombocythemia (ET), exist in the background. We herein report a 76-year-old patient in whom the platelet count had been in the normal range at the onset of adrenal infarction but subsequently increased to 102×10⁴/μL at 7 months later, leading to the diagnosis of JAK2V617F-positive ET. As the presence of the JAK2V617F mutation increases the risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344430"
                        open
                      >
                        <summary class="article-expander__title">Bilateral Adrenal Infarction that Developed in Latent Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344430/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bilateral adrenal infarction is an extremely rare disease, and it has been reported that some coagulation abnormalities, including essential thrombocythemia (ET), exist in the background. We herein report a 76-year-old patient in whom the platelet count had been in the normal range at the onset of adrenal infarction but subsequently increased to 102×10⁴/μL at 7 months later, leading to the diagnosis of JAK2V617F-positive ET. As the presence of the JAK2V617F mutation increases the risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344430"
                        open
                      >
                        <summary class="article-expander__title">Bilateral Adrenal Infarction that Developed in Latent Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344430/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bilateral adrenal infarction is an extremely rare disease, and it has been reported that some coagulation abnormalities, including essential thrombocythemia (ET), exist in the background. We herein report a 76-year-old patient in whom the platelet count had been in the normal range at the onset of adrenal infarction but subsequently increased to 102×10⁴/μL at 7 months later, leading to the diagnosis of JAK2V617F-positive ET. As the presence of the JAK2V617F mutation increases the risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344430"
                        open
                      >
                        <summary class="article-expander__title">Bilateral Adrenal Infarction that Developed in Latent Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344430/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bilateral adrenal infarction is an extremely rare disease, and it has been reported that some coagulation abnormalities, including essential thrombocythemia (ET), exist in the background. We herein report a 76-year-old patient in whom the platelet count had been in the normal range at the onset of adrenal infarction but subsequently increased to 102×10⁴/μL at 7 months later, leading to the diagnosis of JAK2V617F-positive ET. As the presence of the JAK2V617F mutation increases the risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344430"
                        open
                      >
                        <summary class="article-expander__title">Bilateral Adrenal Infarction that Developed in Latent Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344430/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230627211357&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bilateral adrenal infarction is an extremely rare disease, and it has been reported that some coagulation abnormalities, including essential thrombocythemia (ET), exist in the background. We herein report a 76-year-old patient in whom the platelet count had been in the normal range at the onset of adrenal infarction but subsequently increased to 102×10⁴/μL at 7 months later, leading to the diagnosis of JAK2V617F-positive ET. As the presence of the JAK2V617F mutation increases the risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230627211357&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344430"
                        open
                      >
                        <summary class="article-expander__title">Bilateral Adrenal Infarction that Developed in Latent Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344430/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230626211237&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bilateral adrenal infarction is an extremely rare disease, and it has been reported that some coagulation abnormalities, including essential thrombocythemia (ET), exist in the background. We herein report a 76-year-old patient in whom the platelet count had been in the normal range at the onset of adrenal infarction but subsequently increased to 102×10⁴/μL at 7 months later, leading to the diagnosis of JAK2V617F-positive ET. As the presence of the JAK2V617F mutation increases the risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230626211237&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344430"
                        open
                      >
                        <summary class="article-expander__title">Bilateral Adrenal Infarction that Developed in Latent Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344430/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bilateral adrenal infarction is an extremely rare disease, and it has been reported that some coagulation abnormalities, including essential thrombocythemia (ET), exist in the background. We herein report a 76-year-old patient in whom the platelet count had been in the normal range at the onset of adrenal infarction but subsequently increased to 102×10⁴/μL at 7 months later, leading to the diagnosis of JAK2V617F-positive ET. As the presence of the JAK2V617F mutation increases the risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344430"
                        open
                      >
                        <summary class="article-expander__title">Bilateral Adrenal Infarction that Developed in Latent Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344430/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bilateral adrenal infarction is an extremely rare disease, and it has been reported that some coagulation abnormalities, including essential thrombocythemia (ET), exist in the background. We herein report a 76-year-old patient in whom the platelet count had been in the normal range at the onset of adrenal infarction but subsequently increased to 102×10⁴/μL at 7 months later, leading to the diagnosis of JAK2V617F-positive ET. As the presence of the JAK2V617F mutation increases the risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344430"
                        open
                      >
                        <summary class="article-expander__title">Bilateral Adrenal Infarction that Developed in Latent Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344430/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bilateral adrenal infarction is an extremely rare disease, and it has been reported that some coagulation abnormalities, including essential thrombocythemia (ET), exist in the background. We herein report a 76-year-old patient in whom the platelet count had been in the normal range at the onset of adrenal infarction but subsequently increased to 102×10⁴/μL at 7 months later, leading to the diagnosis of JAK2V617F-positive ET. As the presence of the JAK2V617F mutation increases the risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344430"
                        open
                      >
                        <summary class="article-expander__title">Bilateral Adrenal Infarction that Developed in Latent Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344430/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bilateral adrenal infarction is an extremely rare disease, and it has been reported that some coagulation abnormalities, including essential thrombocythemia (ET), exist in the background. We herein report a 76-year-old patient in whom the platelet count had been in the normal range at the onset of adrenal infarction but subsequently increased to 102×10⁴/μL at 7 months later, leading to the diagnosis of JAK2V617F-positive ET. As the presence of the JAK2V617F mutation increases the risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344104"
                        open
                      >
                        <summary class="article-expander__title">Modeling and predicting cancer clonal evolution with reinforcement learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230709211149&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer results from an evolutionary process that typically yields multiple clones with varying sets of mutations within the same tumor. Accurately modeling this process is key to understanding and predicting cancer evolution. Here, we introduce CloMu (Clone To Mutation), a flexible and low-parameter tree-generative model of cancer evolution. CloMu uses a two-layer neural network trained via reinforcement learning to determine the probability of new mutations based on the existing mutations on a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344104"
                        open
                      >
                        <summary class="article-expander__title">Modeling and predicting cancer clonal evolution with reinforcement learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer results from an evolutionary process that typically yields multiple clones with varying sets of mutations within the same tumor. Accurately modeling this process is key to understanding and predicting cancer evolution. Here, we introduce CloMu (Clone To Mutation), a flexible and low-parameter tree-generative model of cancer evolution. CloMu uses a two-layer neural network trained via reinforcement learning to determine the probability of new mutations based on the existing mutations on a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230708211710&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344104"
                        open
                      >
                        <summary class="article-expander__title">Modeling and predicting cancer clonal evolution with reinforcement learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer results from an evolutionary process that typically yields multiple clones with varying sets of mutations within the same tumor. Accurately modeling this process is key to understanding and predicting cancer evolution. Here, we introduce CloMu (Clone To Mutation), a flexible and low-parameter tree-generative model of cancer evolution. CloMu uses a two-layer neural network trained via reinforcement learning to determine the probability of new mutations based on the existing mutations on a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37342197"
                        open
                      >
                        <summary class="article-expander__title">Phenotyping of lymphoproliferative tumours generated in xenografts of non-small cell lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37342197/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Patient-derived xenograft (PDX) models involve the engraftment of tumour tissue in immunocompromised mice and represent an important pre-clinical oncology research method. A limitation of non-small cell lung cancer (NSCLC) PDX model derivation in NOD-scid IL2Rgamma^(null) (NSG) mice is that a subset of initial engraftments are of lymphocytic, rather than tumour origin.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344104"
                        open
                      >
                        <summary class="article-expander__title">Modeling and predicting cancer clonal evolution with reinforcement learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer results from an evolutionary process that typically yields multiple clones with varying sets of mutations within the same tumor. Accurately modeling this process is key to understanding and predicting cancer evolution. Here, we introduce CloMu (Clone To Mutation), a flexible and low-parameter tree-generative model of cancer evolution. CloMu uses a two-layer neural network trained via reinforcement learning to determine the probability of new mutations based on the existing mutations on a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37342197"
                        open
                      >
                        <summary class="article-expander__title">Phenotyping of lymphoproliferative tumours generated in xenografts of non-small cell lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37342197/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Patient-derived xenograft (PDX) models involve the engraftment of tumour tissue in immunocompromised mice and represent an important pre-clinical oncology research method. A limitation of non-small cell lung cancer (NSCLC) PDX model derivation in NOD-scid IL2Rgamma^(null) (NSG) mice is that a subset of initial engraftments are of lymphocytic, rather than tumour origin.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344104"
                        open
                      >
                        <summary class="article-expander__title">Modeling and predicting cancer clonal evolution with reinforcement learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230705211412&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer results from an evolutionary process that typically yields multiple clones with varying sets of mutations within the same tumor. Accurately modeling this process is key to understanding and predicting cancer evolution. Here, we introduce CloMu (Clone To Mutation), a flexible and low-parameter tree-generative model of cancer evolution. CloMu uses a two-layer neural network trained via reinforcement learning to determine the probability of new mutations based on the existing mutations on a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230705211412&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37342197"
                        open
                      >
                        <summary class="article-expander__title">Phenotyping of lymphoproliferative tumours generated in xenografts of non-small cell lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37342197/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230705211412&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Patient-derived xenograft (PDX) models involve the engraftment of tumour tissue in immunocompromised mice and represent an important pre-clinical oncology research method. A limitation of non-small cell lung cancer (NSCLC) PDX model derivation in NOD-scid IL2Rgamma^(null) (NSG) mice is that a subset of initial engraftments are of lymphocytic, rather than tumour origin.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344104"
                        open
                      >
                        <summary class="article-expander__title">Modeling and predicting cancer clonal evolution with reinforcement learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer results from an evolutionary process that typically yields multiple clones with varying sets of mutations within the same tumor. Accurately modeling this process is key to understanding and predicting cancer evolution. Here, we introduce CloMu (Clone To Mutation), a flexible and low-parameter tree-generative model of cancer evolution. CloMu uses a two-layer neural network trained via reinforcement learning to determine the probability of new mutations based on the existing mutations on a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37342197"
                        open
                      >
                        <summary class="article-expander__title">Phenotyping of lymphoproliferative tumours generated in xenografts of non-small cell lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37342197/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Patient-derived xenograft (PDX) models involve the engraftment of tumour tissue in immunocompromised mice and represent an important pre-clinical oncology research method. A limitation of non-small cell lung cancer (NSCLC) PDX model derivation in NOD-scid IL2Rgamma^(null) (NSG) mice is that a subset of initial engraftments are of lymphocytic, rather than tumour origin.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344104"
                        open
                      >
                        <summary class="article-expander__title">Modeling and predicting cancer clonal evolution with reinforcement learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer results from an evolutionary process that typically yields multiple clones with varying sets of mutations within the same tumor. Accurately modeling this process is key to understanding and predicting cancer evolution. Here, we introduce CloMu (Clone To Mutation), a flexible and low-parameter tree-generative model of cancer evolution. CloMu uses a two-layer neural network trained via reinforcement learning to determine the probability of new mutations based on the existing mutations on a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37342197"
                        open
                      >
                        <summary class="article-expander__title">Phenotyping of lymphoproliferative tumours generated in xenografts of non-small cell lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37342197/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Patient-derived xenograft (PDX) models involve the engraftment of tumour tissue in immunocompromised mice and represent an important pre-clinical oncology research method. A limitation of non-small cell lung cancer (NSCLC) PDX model derivation in NOD-scid IL2Rgamma^(null) (NSG) mice is that a subset of initial engraftments are of lymphocytic, rather than tumour origin.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344104"
                        open
                      >
                        <summary class="article-expander__title">Modeling and predicting cancer clonal evolution with reinforcement learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer results from an evolutionary process that typically yields multiple clones with varying sets of mutations within the same tumor. Accurately modeling this process is key to understanding and predicting cancer evolution. Here, we introduce CloMu (Clone To Mutation), a flexible and low-parameter tree-generative model of cancer evolution. CloMu uses a two-layer neural network trained via reinforcement learning to determine the probability of new mutations based on the existing mutations on a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37342197"
                        open
                      >
                        <summary class="article-expander__title">Phenotyping of lymphoproliferative tumours generated in xenografts of non-small cell lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37342197/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Patient-derived xenograft (PDX) models involve the engraftment of tumour tissue in immunocompromised mice and represent an important pre-clinical oncology research method. A limitation of non-small cell lung cancer (NSCLC) PDX model derivation in NOD-scid IL2Rgamma^(null) (NSG) mice is that a subset of initial engraftments are of lymphocytic, rather than tumour origin.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344104"
                        open
                      >
                        <summary class="article-expander__title">Modeling and predicting cancer clonal evolution with reinforcement learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer results from an evolutionary process that typically yields multiple clones with varying sets of mutations within the same tumor. Accurately modeling this process is key to understanding and predicting cancer evolution. Here, we introduce CloMu (Clone To Mutation), a flexible and low-parameter tree-generative model of cancer evolution. CloMu uses a two-layer neural network trained via reinforcement learning to determine the probability of new mutations based on the existing mutations on a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37342197"
                        open
                      >
                        <summary class="article-expander__title">Phenotyping of lymphoproliferative tumours generated in xenografts of non-small cell lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37342197/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Patient-derived xenograft (PDX) models involve the engraftment of tumour tissue in immunocompromised mice and represent an important pre-clinical oncology research method. A limitation of non-small cell lung cancer (NSCLC) PDX model derivation in NOD-scid IL2Rgamma^(null) (NSG) mice is that a subset of initial engraftments are of lymphocytic, rather than tumour origin.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344104"
                        open
                      >
                        <summary class="article-expander__title">Modeling and predicting cancer clonal evolution with reinforcement learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer results from an evolutionary process that typically yields multiple clones with varying sets of mutations within the same tumor. Accurately modeling this process is key to understanding and predicting cancer evolution. Here, we introduce CloMu (Clone To Mutation), a flexible and low-parameter tree-generative model of cancer evolution. CloMu uses a two-layer neural network trained via reinforcement learning to determine the probability of new mutations based on the existing mutations on a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37342197"
                        open
                      >
                        <summary class="article-expander__title">Phenotyping of lymphoproliferative tumours generated in xenografts of non-small cell lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37342197/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Patient-derived xenograft (PDX) models involve the engraftment of tumour tissue in immunocompromised mice and represent an important pre-clinical oncology research method. A limitation of non-small cell lung cancer (NSCLC) PDX model derivation in NOD-scid IL2Rgamma^(null) (NSG) mice is that a subset of initial engraftments are of lymphocytic, rather than tumour origin.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344104"
                        open
                      >
                        <summary class="article-expander__title">Modeling and predicting cancer clonal evolution with reinforcement learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer results from an evolutionary process that typically yields multiple clones with varying sets of mutations within the same tumor. Accurately modeling this process is key to understanding and predicting cancer evolution. Here, we introduce CloMu (Clone To Mutation), a flexible and low-parameter tree-generative model of cancer evolution. CloMu uses a two-layer neural network trained via reinforcement learning to determine the probability of new mutations based on the existing mutations on a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37342197"
                        open
                      >
                        <summary class="article-expander__title">Phenotyping of lymphoproliferative tumours generated in xenografts of non-small cell lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37342197/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Patient-derived xenograft (PDX) models involve the engraftment of tumour tissue in immunocompromised mice and represent an important pre-clinical oncology research method. A limitation of non-small cell lung cancer (NSCLC) PDX model derivation in NOD-scid IL2Rgamma^(null) (NSG) mice is that a subset of initial engraftments are of lymphocytic, rather than tumour origin.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344104"
                        open
                      >
                        <summary class="article-expander__title">Modeling and predicting cancer clonal evolution with reinforcement learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer results from an evolutionary process that typically yields multiple clones with varying sets of mutations within the same tumor. Accurately modeling this process is key to understanding and predicting cancer evolution. Here, we introduce CloMu (Clone To Mutation), a flexible and low-parameter tree-generative model of cancer evolution. CloMu uses a two-layer neural network trained via reinforcement learning to determine the probability of new mutations based on the existing mutations on a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37342197"
                        open
                      >
                        <summary class="article-expander__title">Phenotyping of lymphoproliferative tumours generated in xenografts of non-small cell lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37342197/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Patient-derived xenograft (PDX) models involve the engraftment of tumour tissue in immunocompromised mice and represent an important pre-clinical oncology research method. A limitation of non-small cell lung cancer (NSCLC) PDX model derivation in NOD-scid IL2Rgamma^(null) (NSG) mice is that a subset of initial engraftments are of lymphocytic, rather than tumour origin.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344104"
                        open
                      >
                        <summary class="article-expander__title">Modeling and predicting cancer clonal evolution with reinforcement learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230627211357&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer results from an evolutionary process that typically yields multiple clones with varying sets of mutations within the same tumor. Accurately modeling this process is key to understanding and predicting cancer evolution. Here, we introduce CloMu (Clone To Mutation), a flexible and low-parameter tree-generative model of cancer evolution. CloMu uses a two-layer neural network trained via reinforcement learning to determine the probability of new mutations based on the existing mutations on a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230627211357&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37342197"
                        open
                      >
                        <summary class="article-expander__title">Phenotyping of lymphoproliferative tumours generated in xenografts of non-small cell lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37342197/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230627211357&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Patient-derived xenograft (PDX) models involve the engraftment of tumour tissue in immunocompromised mice and represent an important pre-clinical oncology research method. A limitation of non-small cell lung cancer (NSCLC) PDX model derivation in NOD-scid IL2Rgamma^(null) (NSG) mice is that a subset of initial engraftments are of lymphocytic, rather than tumour origin.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344104"
                        open
                      >
                        <summary class="article-expander__title">Modeling and predicting cancer clonal evolution with reinforcement learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230626211237&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer results from an evolutionary process that typically yields multiple clones with varying sets of mutations within the same tumor. Accurately modeling this process is key to understanding and predicting cancer evolution. Here, we introduce CloMu (Clone To Mutation), a flexible and low-parameter tree-generative model of cancer evolution. CloMu uses a two-layer neural network trained via reinforcement learning to determine the probability of new mutations based on the existing mutations on a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230626211237&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37342197"
                        open
                      >
                        <summary class="article-expander__title">Phenotyping of lymphoproliferative tumours generated in xenografts of non-small cell lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37342197/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230626211237&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Patient-derived xenograft (PDX) models involve the engraftment of tumour tissue in immunocompromised mice and represent an important pre-clinical oncology research method. A limitation of non-small cell lung cancer (NSCLC) PDX model derivation in NOD-scid IL2Rgamma^(null) (NSG) mice is that a subset of initial engraftments are of lymphocytic, rather than tumour origin.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344104"
                        open
                      >
                        <summary class="article-expander__title">Modeling and predicting cancer clonal evolution with reinforcement learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer results from an evolutionary process that typically yields multiple clones with varying sets of mutations within the same tumor. Accurately modeling this process is key to understanding and predicting cancer evolution. Here, we introduce CloMu (Clone To Mutation), a flexible and low-parameter tree-generative model of cancer evolution. CloMu uses a two-layer neural network trained via reinforcement learning to determine the probability of new mutations based on the existing mutations on a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37342197"
                        open
                      >
                        <summary class="article-expander__title">Phenotyping of lymphoproliferative tumours generated in xenografts of non-small cell lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37342197/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Patient-derived xenograft (PDX) models involve the engraftment of tumour tissue in immunocompromised mice and represent an important pre-clinical oncology research method. A limitation of non-small cell lung cancer (NSCLC) PDX model derivation in NOD-scid IL2Rgamma^(null) (NSG) mice is that a subset of initial engraftments are of lymphocytic, rather than tumour origin.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344104"
                        open
                      >
                        <summary class="article-expander__title">Modeling and predicting cancer clonal evolution with reinforcement learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer results from an evolutionary process that typically yields multiple clones with varying sets of mutations within the same tumor. Accurately modeling this process is key to understanding and predicting cancer evolution. Here, we introduce CloMu (Clone To Mutation), a flexible and low-parameter tree-generative model of cancer evolution. CloMu uses a two-layer neural network trained via reinforcement learning to determine the probability of new mutations based on the existing mutations on a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37342197"
                        open
                      >
                        <summary class="article-expander__title">Phenotyping of lymphoproliferative tumours generated in xenografts of non-small cell lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37342197/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Patient-derived xenograft (PDX) models involve the engraftment of tumour tissue in immunocompromised mice and represent an important pre-clinical oncology research method. A limitation of non-small cell lung cancer (NSCLC) PDX model derivation in NOD-scid IL2Rgamma^(null) (NSG) mice is that a subset of initial engraftments are of lymphocytic, rather than tumour origin.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344104"
                        open
                      >
                        <summary class="article-expander__title">Modeling and predicting cancer clonal evolution with reinforcement learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer results from an evolutionary process that typically yields multiple clones with varying sets of mutations within the same tumor. Accurately modeling this process is key to understanding and predicting cancer evolution. Here, we introduce CloMu (Clone To Mutation), a flexible and low-parameter tree-generative model of cancer evolution. CloMu uses a two-layer neural network trained via reinforcement learning to determine the probability of new mutations based on the existing mutations on a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37342197"
                        open
                      >
                        <summary class="article-expander__title">Phenotyping of lymphoproliferative tumours generated in xenografts of non-small cell lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37342197/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Patient-derived xenograft (PDX) models involve the engraftment of tumour tissue in immunocompromised mice and represent an important pre-clinical oncology research method. A limitation of non-small cell lung cancer (NSCLC) PDX model derivation in NOD-scid IL2Rgamma^(null) (NSG) mice is that a subset of initial engraftments are of lymphocytic, rather than tumour origin.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344104"
                        open
                      >
                        <summary class="article-expander__title">Modeling and predicting cancer clonal evolution with reinforcement learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer results from an evolutionary process that typically yields multiple clones with varying sets of mutations within the same tumor. Accurately modeling this process is key to understanding and predicting cancer evolution. Here, we introduce CloMu (Clone To Mutation), a flexible and low-parameter tree-generative model of cancer evolution. CloMu uses a two-layer neural network trained via reinforcement learning to determine the probability of new mutations based on the existing mutations on a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37342197"
                        open
                      >
                        <summary class="article-expander__title">Phenotyping of lymphoproliferative tumours generated in xenografts of non-small cell lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37342197/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Patient-derived xenograft (PDX) models involve the engraftment of tumour tissue in immunocompromised mice and represent an important pre-clinical oncology research method. A limitation of non-small cell lung cancer (NSCLC) PDX model derivation in NOD-scid IL2Rgamma^(null) (NSG) mice is that a subset of initial engraftments are of lymphocytic, rather than tumour origin.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37342197"
                        open
                      >
                        <summary class="article-expander__title">Phenotyping of lymphoproliferative tumours generated in xenografts of non-small cell lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37342197/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Patient-derived xenograft (PDX) models involve the engraftment of tumour tissue in immunocompromised mice and represent an important pre-clinical oncology research method. A limitation of non-small cell lung cancer (NSCLC) PDX model derivation in NOD-scid IL2Rgamma^(null) (NSG) mice is that a subset of initial engraftments are of lymphocytic, rather than tumour origin.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-20 datetime="2023-06-20T10:00:00.000Z">2023-06-20</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-20 datetime="2023-06-20T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37339686"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic remodelling by neoadjuvant chemoradiotherapy (nCRT) and indicative role of acquired INDEL percentage for nCRT efficacy in esophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37339686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230707211257&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: nCRT remodels the genome and transcriptome of ESCC. Acquired INDEL% is a potential biomarker to indicate the effectiveness of nCRT and radiation sensitivity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37339686"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic remodelling by neoadjuvant chemoradiotherapy (nCRT) and indicative role of acquired INDEL percentage for nCRT efficacy in esophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37339686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230706211438&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: nCRT remodels the genome and transcriptome of ESCC. Acquired INDEL% is a potential biomarker to indicate the effectiveness of nCRT and radiation sensitivity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37339686"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic remodelling by neoadjuvant chemoradiotherapy (nCRT) and indicative role of acquired INDEL percentage for nCRT efficacy in esophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37339686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230705211412&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: nCRT remodels the genome and transcriptome of ESCC. Acquired INDEL% is a potential biomarker to indicate the effectiveness of nCRT and radiation sensitivity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37339686"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic remodelling by neoadjuvant chemoradiotherapy (nCRT) and indicative role of acquired INDEL percentage for nCRT efficacy in esophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37339686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230704211358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: nCRT remodels the genome and transcriptome of ESCC. Acquired INDEL% is a potential biomarker to indicate the effectiveness of nCRT and radiation sensitivity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37339686"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic remodelling by neoadjuvant chemoradiotherapy (nCRT) and indicative role of acquired INDEL percentage for nCRT efficacy in esophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37339686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: nCRT remodels the genome and transcriptome of ESCC. Acquired INDEL% is a potential biomarker to indicate the effectiveness of nCRT and radiation sensitivity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37339686"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic remodelling by neoadjuvant chemoradiotherapy (nCRT) and indicative role of acquired INDEL percentage for nCRT efficacy in esophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37339686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: nCRT remodels the genome and transcriptome of ESCC. Acquired INDEL% is a potential biomarker to indicate the effectiveness of nCRT and radiation sensitivity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37339686"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic remodelling by neoadjuvant chemoradiotherapy (nCRT) and indicative role of acquired INDEL percentage for nCRT efficacy in esophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37339686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: nCRT remodels the genome and transcriptome of ESCC. Acquired INDEL% is a potential biomarker to indicate the effectiveness of nCRT and radiation sensitivity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37339686"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic remodelling by neoadjuvant chemoradiotherapy (nCRT) and indicative role of acquired INDEL percentage for nCRT efficacy in esophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37339686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: nCRT remodels the genome and transcriptome of ESCC. Acquired INDEL% is a potential biomarker to indicate the effectiveness of nCRT and radiation sensitivity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37339686"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic remodelling by neoadjuvant chemoradiotherapy (nCRT) and indicative role of acquired INDEL percentage for nCRT efficacy in esophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37339686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: nCRT remodels the genome and transcriptome of ESCC. Acquired INDEL% is a potential biomarker to indicate the effectiveness of nCRT and radiation sensitivity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37339686"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic remodelling by neoadjuvant chemoradiotherapy (nCRT) and indicative role of acquired INDEL percentage for nCRT efficacy in esophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37339686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: nCRT remodels the genome and transcriptome of ESCC. Acquired INDEL% is a potential biomarker to indicate the effectiveness of nCRT and radiation sensitivity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37339686"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic remodelling by neoadjuvant chemoradiotherapy (nCRT) and indicative role of acquired INDEL percentage for nCRT efficacy in esophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37339686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230627211357&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: nCRT remodels the genome and transcriptome of ESCC. Acquired INDEL% is a potential biomarker to indicate the effectiveness of nCRT and radiation sensitivity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37339686"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic remodelling by neoadjuvant chemoradiotherapy (nCRT) and indicative role of acquired INDEL percentage for nCRT efficacy in esophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37339686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230626211237&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: nCRT remodels the genome and transcriptome of ESCC. Acquired INDEL% is a potential biomarker to indicate the effectiveness of nCRT and radiation sensitivity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37339686"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic remodelling by neoadjuvant chemoradiotherapy (nCRT) and indicative role of acquired INDEL percentage for nCRT efficacy in esophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37339686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: nCRT remodels the genome and transcriptome of ESCC. Acquired INDEL% is a potential biomarker to indicate the effectiveness of nCRT and radiation sensitivity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37339686"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic remodelling by neoadjuvant chemoradiotherapy (nCRT) and indicative role of acquired INDEL percentage for nCRT efficacy in esophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37339686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: nCRT remodels the genome and transcriptome of ESCC. Acquired INDEL% is a potential biomarker to indicate the effectiveness of nCRT and radiation sensitivity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37339686"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic remodelling by neoadjuvant chemoradiotherapy (nCRT) and indicative role of acquired INDEL percentage for nCRT efficacy in esophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37339686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: nCRT remodels the genome and transcriptome of ESCC. Acquired INDEL% is a potential biomarker to indicate the effectiveness of nCRT and radiation sensitivity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37339686"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic remodelling by neoadjuvant chemoradiotherapy (nCRT) and indicative role of acquired INDEL percentage for nCRT efficacy in esophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37339686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: nCRT remodels the genome and transcriptome of ESCC. Acquired INDEL% is a potential biomarker to indicate the effectiveness of nCRT and radiation sensitivity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37339686"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic remodelling by neoadjuvant chemoradiotherapy (nCRT) and indicative role of acquired INDEL percentage for nCRT efficacy in esophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37339686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: nCRT remodels the genome and transcriptome of ESCC. Acquired INDEL% is a potential biomarker to indicate the effectiveness of nCRT and radiation sensitivity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-19 datetime="2023-06-19T10:00:00.000Z">2023-06-19</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-19 datetime="2023-06-19T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333361"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Incident Heart Failure with Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333361/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP, particularly mutations in TET2 , represents a potential new risk factor for incident HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333361"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Incident Heart Failure with Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333361/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP, particularly mutations in TET2 , represents a potential new risk factor for incident HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333361"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Incident Heart Failure with Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333361/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP, particularly mutations in TET2 , represents a potential new risk factor for incident HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333361"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Incident Heart Failure with Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333361/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP, particularly mutations in TET2 , represents a potential new risk factor for incident HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333361"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Incident Heart Failure with Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333361/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP, particularly mutations in TET2 , represents a potential new risk factor for incident HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333361"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Incident Heart Failure with Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333361/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230627211357&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP, particularly mutations in TET2 , represents a potential new risk factor for incident HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333361"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Incident Heart Failure with Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333361/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230626211237&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP, particularly mutations in TET2 , represents a potential new risk factor for incident HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333361"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Incident Heart Failure with Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333361/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP, particularly mutations in TET2 , represents a potential new risk factor for incident HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333361"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Incident Heart Failure with Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333361/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP, particularly mutations in TET2 , represents a potential new risk factor for incident HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333361"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Incident Heart Failure with Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333361/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP, particularly mutations in TET2 , represents a potential new risk factor for incident HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333361"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Incident Heart Failure with Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333361/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP, particularly mutations in TET2 , represents a potential new risk factor for incident HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333361"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Incident Heart Failure with Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333361/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP, particularly mutations in TET2 , represents a potential new risk factor for incident HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333361"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Incident Heart Failure with Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333361/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP, particularly mutations in TET2 , represents a potential new risk factor for incident HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333361"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Incident Heart Failure with Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333361/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP, particularly mutations in TET2 , represents a potential new risk factor for incident HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37336285"
                        open
                      >
                        <summary class="article-expander__title">Cellular barcoding: From developmental tracing to anti-tumor drug discovery</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37336285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230703211307&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution has gained immense attention in explaining cancer cell status, history, and fate during cancer progression. Current single-cell or spatial transcriptome technologies have broadened our understanding of various mechanisms underlying cancer initiation, relapse, and drug resistance. However, technical challenges still hinder a better understanding of the dynamics of distinctive phenotypic states and abnormal trajectories from normal physiological transition to malignant stages....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37336285"
                        open
                      >
                        <summary class="article-expander__title">Cellular barcoding: From developmental tracing to anti-tumor drug discovery</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37336285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution has gained immense attention in explaining cancer cell status, history, and fate during cancer progression. Current single-cell or spatial transcriptome technologies have broadened our understanding of various mechanisms underlying cancer initiation, relapse, and drug resistance. However, technical challenges still hinder a better understanding of the dynamics of distinctive phenotypic states and abnormal trajectories from normal physiological transition to malignant stages....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333132"
                        open
                      >
                        <summary class="article-expander__title">Melanoma clonal subline analysis uncovers heterogeneity-driven immunotherapy resistance mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) can promote cancer progression and treatment failure, but the complexity of the regulatory programs and contextual factors involved complicates its study. To understand the specific contribution of ITH to immune checkpoint blockade (ICB) response, we generated single cell-derived clonal sublines from an ICB-sensitive and genetically and phenotypically heterogeneous mouse melanoma model, M4. Genomic and single cell transcriptomic analyses uncovered the diversity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37336285"
                        open
                      >
                        <summary class="article-expander__title">Cellular barcoding: From developmental tracing to anti-tumor drug discovery</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37336285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution has gained immense attention in explaining cancer cell status, history, and fate during cancer progression. Current single-cell or spatial transcriptome technologies have broadened our understanding of various mechanisms underlying cancer initiation, relapse, and drug resistance. However, technical challenges still hinder a better understanding of the dynamics of distinctive phenotypic states and abnormal trajectories from normal physiological transition to malignant stages....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333132"
                        open
                      >
                        <summary class="article-expander__title">Melanoma clonal subline analysis uncovers heterogeneity-driven immunotherapy resistance mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) can promote cancer progression and treatment failure, but the complexity of the regulatory programs and contextual factors involved complicates its study. To understand the specific contribution of ITH to immune checkpoint blockade (ICB) response, we generated single cell-derived clonal sublines from an ICB-sensitive and genetically and phenotypically heterogeneous mouse melanoma model, M4. Genomic and single cell transcriptomic analyses uncovered the diversity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37336285"
                        open
                      >
                        <summary class="article-expander__title">Cellular barcoding: From developmental tracing to anti-tumor drug discovery</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37336285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution has gained immense attention in explaining cancer cell status, history, and fate during cancer progression. Current single-cell or spatial transcriptome technologies have broadened our understanding of various mechanisms underlying cancer initiation, relapse, and drug resistance. However, technical challenges still hinder a better understanding of the dynamics of distinctive phenotypic states and abnormal trajectories from normal physiological transition to malignant stages....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333132"
                        open
                      >
                        <summary class="article-expander__title">Melanoma clonal subline analysis uncovers heterogeneity-driven immunotherapy resistance mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) can promote cancer progression and treatment failure, but the complexity of the regulatory programs and contextual factors involved complicates its study. To understand the specific contribution of ITH to immune checkpoint blockade (ICB) response, we generated single cell-derived clonal sublines from an ICB-sensitive and genetically and phenotypically heterogeneous mouse melanoma model, M4. Genomic and single cell transcriptomic analyses uncovered the diversity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37336285"
                        open
                      >
                        <summary class="article-expander__title">Cellular barcoding: From developmental tracing to anti-tumor drug discovery</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37336285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution has gained immense attention in explaining cancer cell status, history, and fate during cancer progression. Current single-cell or spatial transcriptome technologies have broadened our understanding of various mechanisms underlying cancer initiation, relapse, and drug resistance. However, technical challenges still hinder a better understanding of the dynamics of distinctive phenotypic states and abnormal trajectories from normal physiological transition to malignant stages....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333132"
                        open
                      >
                        <summary class="article-expander__title">Melanoma clonal subline analysis uncovers heterogeneity-driven immunotherapy resistance mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) can promote cancer progression and treatment failure, but the complexity of the regulatory programs and contextual factors involved complicates its study. To understand the specific contribution of ITH to immune checkpoint blockade (ICB) response, we generated single cell-derived clonal sublines from an ICB-sensitive and genetically and phenotypically heterogeneous mouse melanoma model, M4. Genomic and single cell transcriptomic analyses uncovered the diversity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37336285"
                        open
                      >
                        <summary class="article-expander__title">Cellular barcoding: From developmental tracing to anti-tumor drug discovery</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37336285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution has gained immense attention in explaining cancer cell status, history, and fate during cancer progression. Current single-cell or spatial transcriptome technologies have broadened our understanding of various mechanisms underlying cancer initiation, relapse, and drug resistance. However, technical challenges still hinder a better understanding of the dynamics of distinctive phenotypic states and abnormal trajectories from normal physiological transition to malignant stages....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333132"
                        open
                      >
                        <summary class="article-expander__title">Melanoma clonal subline analysis uncovers heterogeneity-driven immunotherapy resistance mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) can promote cancer progression and treatment failure, but the complexity of the regulatory programs and contextual factors involved complicates its study. To understand the specific contribution of ITH to immune checkpoint blockade (ICB) response, we generated single cell-derived clonal sublines from an ICB-sensitive and genetically and phenotypically heterogeneous mouse melanoma model, M4. Genomic and single cell transcriptomic analyses uncovered the diversity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37336285"
                        open
                      >
                        <summary class="article-expander__title">Cellular barcoding: From developmental tracing to anti-tumor drug discovery</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37336285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230627211357&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution has gained immense attention in explaining cancer cell status, history, and fate during cancer progression. Current single-cell or spatial transcriptome technologies have broadened our understanding of various mechanisms underlying cancer initiation, relapse, and drug resistance. However, technical challenges still hinder a better understanding of the dynamics of distinctive phenotypic states and abnormal trajectories from normal physiological transition to malignant stages....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333132"
                        open
                      >
                        <summary class="article-expander__title">Melanoma clonal subline analysis uncovers heterogeneity-driven immunotherapy resistance mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230627211357&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) can promote cancer progression and treatment failure, but the complexity of the regulatory programs and contextual factors involved complicates its study. To understand the specific contribution of ITH to immune checkpoint blockade (ICB) response, we generated single cell-derived clonal sublines from an ICB-sensitive and genetically and phenotypically heterogeneous mouse melanoma model, M4. Genomic and single cell transcriptomic analyses uncovered the diversity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37336285"
                        open
                      >
                        <summary class="article-expander__title">Cellular barcoding: From developmental tracing to anti-tumor drug discovery</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37336285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230626211237&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution has gained immense attention in explaining cancer cell status, history, and fate during cancer progression. Current single-cell or spatial transcriptome technologies have broadened our understanding of various mechanisms underlying cancer initiation, relapse, and drug resistance. However, technical challenges still hinder a better understanding of the dynamics of distinctive phenotypic states and abnormal trajectories from normal physiological transition to malignant stages....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333132"
                        open
                      >
                        <summary class="article-expander__title">Melanoma clonal subline analysis uncovers heterogeneity-driven immunotherapy resistance mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230626211237&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) can promote cancer progression and treatment failure, but the complexity of the regulatory programs and contextual factors involved complicates its study. To understand the specific contribution of ITH to immune checkpoint blockade (ICB) response, we generated single cell-derived clonal sublines from an ICB-sensitive and genetically and phenotypically heterogeneous mouse melanoma model, M4. Genomic and single cell transcriptomic analyses uncovered the diversity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37336285"
                        open
                      >
                        <summary class="article-expander__title">Cellular barcoding: From developmental tracing to anti-tumor drug discovery</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37336285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution has gained immense attention in explaining cancer cell status, history, and fate during cancer progression. Current single-cell or spatial transcriptome technologies have broadened our understanding of various mechanisms underlying cancer initiation, relapse, and drug resistance. However, technical challenges still hinder a better understanding of the dynamics of distinctive phenotypic states and abnormal trajectories from normal physiological transition to malignant stages....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333132"
                        open
                      >
                        <summary class="article-expander__title">Melanoma clonal subline analysis uncovers heterogeneity-driven immunotherapy resistance mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) can promote cancer progression and treatment failure, but the complexity of the regulatory programs and contextual factors involved complicates its study. To understand the specific contribution of ITH to immune checkpoint blockade (ICB) response, we generated single cell-derived clonal sublines from an ICB-sensitive and genetically and phenotypically heterogeneous mouse melanoma model, M4. Genomic and single cell transcriptomic analyses uncovered the diversity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37336285"
                        open
                      >
                        <summary class="article-expander__title">Cellular barcoding: From developmental tracing to anti-tumor drug discovery</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37336285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution has gained immense attention in explaining cancer cell status, history, and fate during cancer progression. Current single-cell or spatial transcriptome technologies have broadened our understanding of various mechanisms underlying cancer initiation, relapse, and drug resistance. However, technical challenges still hinder a better understanding of the dynamics of distinctive phenotypic states and abnormal trajectories from normal physiological transition to malignant stages....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333132"
                        open
                      >
                        <summary class="article-expander__title">Melanoma clonal subline analysis uncovers heterogeneity-driven immunotherapy resistance mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) can promote cancer progression and treatment failure, but the complexity of the regulatory programs and contextual factors involved complicates its study. To understand the specific contribution of ITH to immune checkpoint blockade (ICB) response, we generated single cell-derived clonal sublines from an ICB-sensitive and genetically and phenotypically heterogeneous mouse melanoma model, M4. Genomic and single cell transcriptomic analyses uncovered the diversity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37336285"
                        open
                      >
                        <summary class="article-expander__title">Cellular barcoding: From developmental tracing to anti-tumor drug discovery</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37336285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution has gained immense attention in explaining cancer cell status, history, and fate during cancer progression. Current single-cell or spatial transcriptome technologies have broadened our understanding of various mechanisms underlying cancer initiation, relapse, and drug resistance. However, technical challenges still hinder a better understanding of the dynamics of distinctive phenotypic states and abnormal trajectories from normal physiological transition to malignant stages....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333132"
                        open
                      >
                        <summary class="article-expander__title">Melanoma clonal subline analysis uncovers heterogeneity-driven immunotherapy resistance mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) can promote cancer progression and treatment failure, but the complexity of the regulatory programs and contextual factors involved complicates its study. To understand the specific contribution of ITH to immune checkpoint blockade (ICB) response, we generated single cell-derived clonal sublines from an ICB-sensitive and genetically and phenotypically heterogeneous mouse melanoma model, M4. Genomic and single cell transcriptomic analyses uncovered the diversity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37336285"
                        open
                      >
                        <summary class="article-expander__title">Cellular barcoding: From developmental tracing to anti-tumor drug discovery</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37336285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution has gained immense attention in explaining cancer cell status, history, and fate during cancer progression. Current single-cell or spatial transcriptome technologies have broadened our understanding of various mechanisms underlying cancer initiation, relapse, and drug resistance. However, technical challenges still hinder a better understanding of the dynamics of distinctive phenotypic states and abnormal trajectories from normal physiological transition to malignant stages....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333132"
                        open
                      >
                        <summary class="article-expander__title">Melanoma clonal subline analysis uncovers heterogeneity-driven immunotherapy resistance mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) can promote cancer progression and treatment failure, but the complexity of the regulatory programs and contextual factors involved complicates its study. To understand the specific contribution of ITH to immune checkpoint blockade (ICB) response, we generated single cell-derived clonal sublines from an ICB-sensitive and genetically and phenotypically heterogeneous mouse melanoma model, M4. Genomic and single cell transcriptomic analyses uncovered the diversity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37336285"
                        open
                      >
                        <summary class="article-expander__title">Cellular barcoding: From developmental tracing to anti-tumor drug discovery</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37336285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution has gained immense attention in explaining cancer cell status, history, and fate during cancer progression. Current single-cell or spatial transcriptome technologies have broadened our understanding of various mechanisms underlying cancer initiation, relapse, and drug resistance. However, technical challenges still hinder a better understanding of the dynamics of distinctive phenotypic states and abnormal trajectories from normal physiological transition to malignant stages....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333132"
                        open
                      >
                        <summary class="article-expander__title">Melanoma clonal subline analysis uncovers heterogeneity-driven immunotherapy resistance mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) can promote cancer progression and treatment failure, but the complexity of the regulatory programs and contextual factors involved complicates its study. To understand the specific contribution of ITH to immune checkpoint blockade (ICB) response, we generated single cell-derived clonal sublines from an ICB-sensitive and genetically and phenotypically heterogeneous mouse melanoma model, M4. Genomic and single cell transcriptomic analyses uncovered the diversity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37336285"
                        open
                      >
                        <summary class="article-expander__title">Cellular barcoding: From developmental tracing to anti-tumor drug discovery</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37336285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution has gained immense attention in explaining cancer cell status, history, and fate during cancer progression. Current single-cell or spatial transcriptome technologies have broadened our understanding of various mechanisms underlying cancer initiation, relapse, and drug resistance. However, technical challenges still hinder a better understanding of the dynamics of distinctive phenotypic states and abnormal trajectories from normal physiological transition to malignant stages....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333132"
                        open
                      >
                        <summary class="article-expander__title">Melanoma clonal subline analysis uncovers heterogeneity-driven immunotherapy resistance mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) can promote cancer progression and treatment failure, but the complexity of the regulatory programs and contextual factors involved complicates its study. To understand the specific contribution of ITH to immune checkpoint blockade (ICB) response, we generated single cell-derived clonal sublines from an ICB-sensitive and genetically and phenotypically heterogeneous mouse melanoma model, M4. Genomic and single cell transcriptomic analyses uncovered the diversity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333132"
                        open
                      >
                        <summary class="article-expander__title">Melanoma clonal subline analysis uncovers heterogeneity-driven immunotherapy resistance mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) can promote cancer progression and treatment failure, but the complexity of the regulatory programs and contextual factors involved complicates its study. To understand the specific contribution of ITH to immune checkpoint blockade (ICB) response, we generated single cell-derived clonal sublines from an ICB-sensitive and genetically and phenotypically heterogeneous mouse melanoma model, M4. Genomic and single cell transcriptomic analyses uncovered the diversity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-18 datetime="2023-06-18T10:00:00.000Z">2023-06-18</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-18 datetime="2023-06-18T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331781"
                        open
                      >
                        <summary class="article-expander__title">Emerging Roles of Circulating Tumor DNA for Increased Precision and Personalization in Radiation Oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent breakthroughs in circulating tumor DNA (ctDNA) technologies present a compelling opportunity to combine this emerging liquid biopsy approach with the field of radiogenomics, the study of how tumor genomics correlate with radiotherapy response and radiotoxicity. Canonically, ctDNA levels reflect metastatic tumor burden, although newer ultrasensitive technologies can be used after curative-intent radiotherapy of localized disease to assess ctDNA for minimal residual disease (MRD) detection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331781"
                        open
                      >
                        <summary class="article-expander__title">Emerging Roles of Circulating Tumor DNA for Increased Precision and Personalization in Radiation Oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent breakthroughs in circulating tumor DNA (ctDNA) technologies present a compelling opportunity to combine this emerging liquid biopsy approach with the field of radiogenomics, the study of how tumor genomics correlate with radiotherapy response and radiotoxicity. Canonically, ctDNA levels reflect metastatic tumor burden, although newer ultrasensitive technologies can be used after curative-intent radiotherapy of localized disease to assess ctDNA for minimal residual disease (MRD) detection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331781"
                        open
                      >
                        <summary class="article-expander__title">Emerging Roles of Circulating Tumor DNA for Increased Precision and Personalization in Radiation Oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent breakthroughs in circulating tumor DNA (ctDNA) technologies present a compelling opportunity to combine this emerging liquid biopsy approach with the field of radiogenomics, the study of how tumor genomics correlate with radiotherapy response and radiotoxicity. Canonically, ctDNA levels reflect metastatic tumor burden, although newer ultrasensitive technologies can be used after curative-intent radiotherapy of localized disease to assess ctDNA for minimal residual disease (MRD) detection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331781"
                        open
                      >
                        <summary class="article-expander__title">Emerging Roles of Circulating Tumor DNA for Increased Precision and Personalization in Radiation Oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent breakthroughs in circulating tumor DNA (ctDNA) technologies present a compelling opportunity to combine this emerging liquid biopsy approach with the field of radiogenomics, the study of how tumor genomics correlate with radiotherapy response and radiotoxicity. Canonically, ctDNA levels reflect metastatic tumor burden, although newer ultrasensitive technologies can be used after curative-intent radiotherapy of localized disease to assess ctDNA for minimal residual disease (MRD) detection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331781"
                        open
                      >
                        <summary class="article-expander__title">Emerging Roles of Circulating Tumor DNA for Increased Precision and Personalization in Radiation Oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent breakthroughs in circulating tumor DNA (ctDNA) technologies present a compelling opportunity to combine this emerging liquid biopsy approach with the field of radiogenomics, the study of how tumor genomics correlate with radiotherapy response and radiotoxicity. Canonically, ctDNA levels reflect metastatic tumor burden, although newer ultrasensitive technologies can be used after curative-intent radiotherapy of localized disease to assess ctDNA for minimal residual disease (MRD) detection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331781"
                        open
                      >
                        <summary class="article-expander__title">Emerging Roles of Circulating Tumor DNA for Increased Precision and Personalization in Radiation Oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230627211357&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent breakthroughs in circulating tumor DNA (ctDNA) technologies present a compelling opportunity to combine this emerging liquid biopsy approach with the field of radiogenomics, the study of how tumor genomics correlate with radiotherapy response and radiotoxicity. Canonically, ctDNA levels reflect metastatic tumor burden, although newer ultrasensitive technologies can be used after curative-intent radiotherapy of localized disease to assess ctDNA for minimal residual disease (MRD) detection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331781"
                        open
                      >
                        <summary class="article-expander__title">Emerging Roles of Circulating Tumor DNA for Increased Precision and Personalization in Radiation Oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230626211237&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent breakthroughs in circulating tumor DNA (ctDNA) technologies present a compelling opportunity to combine this emerging liquid biopsy approach with the field of radiogenomics, the study of how tumor genomics correlate with radiotherapy response and radiotoxicity. Canonically, ctDNA levels reflect metastatic tumor burden, although newer ultrasensitive technologies can be used after curative-intent radiotherapy of localized disease to assess ctDNA for minimal residual disease (MRD) detection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331781"
                        open
                      >
                        <summary class="article-expander__title">Emerging Roles of Circulating Tumor DNA for Increased Precision and Personalization in Radiation Oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent breakthroughs in circulating tumor DNA (ctDNA) technologies present a compelling opportunity to combine this emerging liquid biopsy approach with the field of radiogenomics, the study of how tumor genomics correlate with radiotherapy response and radiotoxicity. Canonically, ctDNA levels reflect metastatic tumor burden, although newer ultrasensitive technologies can be used after curative-intent radiotherapy of localized disease to assess ctDNA for minimal residual disease (MRD) detection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331781"
                        open
                      >
                        <summary class="article-expander__title">Emerging Roles of Circulating Tumor DNA for Increased Precision and Personalization in Radiation Oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent breakthroughs in circulating tumor DNA (ctDNA) technologies present a compelling opportunity to combine this emerging liquid biopsy approach with the field of radiogenomics, the study of how tumor genomics correlate with radiotherapy response and radiotoxicity. Canonically, ctDNA levels reflect metastatic tumor burden, although newer ultrasensitive technologies can be used after curative-intent radiotherapy of localized disease to assess ctDNA for minimal residual disease (MRD) detection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331781"
                        open
                      >
                        <summary class="article-expander__title">Emerging Roles of Circulating Tumor DNA for Increased Precision and Personalization in Radiation Oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent breakthroughs in circulating tumor DNA (ctDNA) technologies present a compelling opportunity to combine this emerging liquid biopsy approach with the field of radiogenomics, the study of how tumor genomics correlate with radiotherapy response and radiotoxicity. Canonically, ctDNA levels reflect metastatic tumor burden, although newer ultrasensitive technologies can be used after curative-intent radiotherapy of localized disease to assess ctDNA for minimal residual disease (MRD) detection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331781"
                        open
                      >
                        <summary class="article-expander__title">Emerging Roles of Circulating Tumor DNA for Increased Precision and Personalization in Radiation Oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent breakthroughs in circulating tumor DNA (ctDNA) technologies present a compelling opportunity to combine this emerging liquid biopsy approach with the field of radiogenomics, the study of how tumor genomics correlate with radiotherapy response and radiotoxicity. Canonically, ctDNA levels reflect metastatic tumor burden, although newer ultrasensitive technologies can be used after curative-intent radiotherapy of localized disease to assess ctDNA for minimal residual disease (MRD) detection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331781"
                        open
                      >
                        <summary class="article-expander__title">Emerging Roles of Circulating Tumor DNA for Increased Precision and Personalization in Radiation Oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent breakthroughs in circulating tumor DNA (ctDNA) technologies present a compelling opportunity to combine this emerging liquid biopsy approach with the field of radiogenomics, the study of how tumor genomics correlate with radiotherapy response and radiotoxicity. Canonically, ctDNA levels reflect metastatic tumor burden, although newer ultrasensitive technologies can be used after curative-intent radiotherapy of localized disease to assess ctDNA for minimal residual disease (MRD) detection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331781"
                        open
                      >
                        <summary class="article-expander__title">Emerging Roles of Circulating Tumor DNA for Increased Precision and Personalization in Radiation Oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent breakthroughs in circulating tumor DNA (ctDNA) technologies present a compelling opportunity to combine this emerging liquid biopsy approach with the field of radiogenomics, the study of how tumor genomics correlate with radiotherapy response and radiotoxicity. Canonically, ctDNA levels reflect metastatic tumor burden, although newer ultrasensitive technologies can be used after curative-intent radiotherapy of localized disease to assess ctDNA for minimal residual disease (MRD) detection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331781"
                        open
                      >
                        <summary class="article-expander__title">Emerging Roles of Circulating Tumor DNA for Increased Precision and Personalization in Radiation Oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent breakthroughs in circulating tumor DNA (ctDNA) technologies present a compelling opportunity to combine this emerging liquid biopsy approach with the field of radiogenomics, the study of how tumor genomics correlate with radiotherapy response and radiotoxicity. Canonically, ctDNA levels reflect metastatic tumor burden, although newer ultrasensitive technologies can be used after curative-intent radiotherapy of localized disease to assess ctDNA for minimal residual disease (MRD) detection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331179"
                        open
                      >
                        <summary class="article-expander__title">Clinical trial designs for evaluating and exploiting cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331179/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolution of drug-resistant cell subpopulations causes cancer treatment failure. Current preclinical evidence shows that it is possible to model herding of clonal evolution and collateral sensitivity where an initial treatment could favourably influence the response to a subsequent one. Novel therapy strategies exploiting this understanding are being considered, and clinical trial designs for steering cancer evolution are needed. Furthermore, preclinical evidence suggests that different...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331179"
                        open
                      >
                        <summary class="article-expander__title">Clinical trial designs for evaluating and exploiting cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331179/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolution of drug-resistant cell subpopulations causes cancer treatment failure. Current preclinical evidence shows that it is possible to model herding of clonal evolution and collateral sensitivity where an initial treatment could favourably influence the response to a subsequent one. Novel therapy strategies exploiting this understanding are being considered, and clinical trial designs for steering cancer evolution are needed. Furthermore, preclinical evidence suggests that different...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331179"
                        open
                      >
                        <summary class="article-expander__title">Clinical trial designs for evaluating and exploiting cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331179/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolution of drug-resistant cell subpopulations causes cancer treatment failure. Current preclinical evidence shows that it is possible to model herding of clonal evolution and collateral sensitivity where an initial treatment could favourably influence the response to a subsequent one. Novel therapy strategies exploiting this understanding are being considered, and clinical trial designs for steering cancer evolution are needed. Furthermore, preclinical evidence suggests that different...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331179"
                        open
                      >
                        <summary class="article-expander__title">Clinical trial designs for evaluating and exploiting cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331179/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolution of drug-resistant cell subpopulations causes cancer treatment failure. Current preclinical evidence shows that it is possible to model herding of clonal evolution and collateral sensitivity where an initial treatment could favourably influence the response to a subsequent one. Novel therapy strategies exploiting this understanding are being considered, and clinical trial designs for steering cancer evolution are needed. Furthermore, preclinical evidence suggests that different...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331179"
                        open
                      >
                        <summary class="article-expander__title">Clinical trial designs for evaluating and exploiting cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331179/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolution of drug-resistant cell subpopulations causes cancer treatment failure. Current preclinical evidence shows that it is possible to model herding of clonal evolution and collateral sensitivity where an initial treatment could favourably influence the response to a subsequent one. Novel therapy strategies exploiting this understanding are being considered, and clinical trial designs for steering cancer evolution are needed. Furthermore, preclinical evidence suggests that different...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331179"
                        open
                      >
                        <summary class="article-expander__title">Clinical trial designs for evaluating and exploiting cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331179/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230627211357&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolution of drug-resistant cell subpopulations causes cancer treatment failure. Current preclinical evidence shows that it is possible to model herding of clonal evolution and collateral sensitivity where an initial treatment could favourably influence the response to a subsequent one. Novel therapy strategies exploiting this understanding are being considered, and clinical trial designs for steering cancer evolution are needed. Furthermore, preclinical evidence suggests that different...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331179"
                        open
                      >
                        <summary class="article-expander__title">Clinical trial designs for evaluating and exploiting cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331179/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230626211237&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolution of drug-resistant cell subpopulations causes cancer treatment failure. Current preclinical evidence shows that it is possible to model herding of clonal evolution and collateral sensitivity where an initial treatment could favourably influence the response to a subsequent one. Novel therapy strategies exploiting this understanding are being considered, and clinical trial designs for steering cancer evolution are needed. Furthermore, preclinical evidence suggests that different...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331179"
                        open
                      >
                        <summary class="article-expander__title">Clinical trial designs for evaluating and exploiting cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331179/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolution of drug-resistant cell subpopulations causes cancer treatment failure. Current preclinical evidence shows that it is possible to model herding of clonal evolution and collateral sensitivity where an initial treatment could favourably influence the response to a subsequent one. Novel therapy strategies exploiting this understanding are being considered, and clinical trial designs for steering cancer evolution are needed. Furthermore, preclinical evidence suggests that different...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331179"
                        open
                      >
                        <summary class="article-expander__title">Clinical trial designs for evaluating and exploiting cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331179/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolution of drug-resistant cell subpopulations causes cancer treatment failure. Current preclinical evidence shows that it is possible to model herding of clonal evolution and collateral sensitivity where an initial treatment could favourably influence the response to a subsequent one. Novel therapy strategies exploiting this understanding are being considered, and clinical trial designs for steering cancer evolution are needed. Furthermore, preclinical evidence suggests that different...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331179"
                        open
                      >
                        <summary class="article-expander__title">Clinical trial designs for evaluating and exploiting cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331179/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolution of drug-resistant cell subpopulations causes cancer treatment failure. Current preclinical evidence shows that it is possible to model herding of clonal evolution and collateral sensitivity where an initial treatment could favourably influence the response to a subsequent one. Novel therapy strategies exploiting this understanding are being considered, and clinical trial designs for steering cancer evolution are needed. Furthermore, preclinical evidence suggests that different...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331179"
                        open
                      >
                        <summary class="article-expander__title">Clinical trial designs for evaluating and exploiting cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331179/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolution of drug-resistant cell subpopulations causes cancer treatment failure. Current preclinical evidence shows that it is possible to model herding of clonal evolution and collateral sensitivity where an initial treatment could favourably influence the response to a subsequent one. Novel therapy strategies exploiting this understanding are being considered, and clinical trial designs for steering cancer evolution are needed. Furthermore, preclinical evidence suggests that different...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331179"
                        open
                      >
                        <summary class="article-expander__title">Clinical trial designs for evaluating and exploiting cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331179/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolution of drug-resistant cell subpopulations causes cancer treatment failure. Current preclinical evidence shows that it is possible to model herding of clonal evolution and collateral sensitivity where an initial treatment could favourably influence the response to a subsequent one. Novel therapy strategies exploiting this understanding are being considered, and clinical trial designs for steering cancer evolution are needed. Furthermore, preclinical evidence suggests that different...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331179"
                        open
                      >
                        <summary class="article-expander__title">Clinical trial designs for evaluating and exploiting cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331179/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolution of drug-resistant cell subpopulations causes cancer treatment failure. Current preclinical evidence shows that it is possible to model herding of clonal evolution and collateral sensitivity where an initial treatment could favourably influence the response to a subsequent one. Novel therapy strategies exploiting this understanding are being considered, and clinical trial designs for steering cancer evolution are needed. Furthermore, preclinical evidence suggests that different...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331179"
                        open
                      >
                        <summary class="article-expander__title">Clinical trial designs for evaluating and exploiting cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331179/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolution of drug-resistant cell subpopulations causes cancer treatment failure. Current preclinical evidence shows that it is possible to model herding of clonal evolution and collateral sensitivity where an initial treatment could favourably influence the response to a subsequent one. Novel therapy strategies exploiting this understanding are being considered, and clinical trial designs for steering cancer evolution are needed. Furthermore, preclinical evidence suggests that different...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-16 datetime="2023-06-16T10:00:00.000Z">2023-06-16</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-16 datetime="2023-06-16T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327996"
                        open
                      >
                        <summary class="article-expander__title">Pure red cell aplasia: The second hundred years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327996/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pure red cell aplasia (PRCA) is a rare hematologic syndrome, characterized by an isolated normocytic anemia with severe reticulocytopenia, and defined by absence or near absence of erythroid precursors in the bone marrow. First described in 1922, PRCA may be a primary autoimmune or clonal myeloid or lymphoid disorder, but may also be secondary to other disorders of immune dysregulation/autoimmunity, to infections, to neoplasms, or to drugs. Insights from the study of PRCA have helped illuminate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327996"
                        open
                      >
                        <summary class="article-expander__title">Pure red cell aplasia: The second hundred years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327996/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pure red cell aplasia (PRCA) is a rare hematologic syndrome, characterized by an isolated normocytic anemia with severe reticulocytopenia, and defined by absence or near absence of erythroid precursors in the bone marrow. First described in 1922, PRCA may be a primary autoimmune or clonal myeloid or lymphoid disorder, but may also be secondary to other disorders of immune dysregulation/autoimmunity, to infections, to neoplasms, or to drugs. Insights from the study of PRCA have helped illuminate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327996"
                        open
                      >
                        <summary class="article-expander__title">Pure red cell aplasia: the second hundred years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327996/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pure red cell aplasia (PRCA) is a rare hematologic syndrome, characterized by an isolated normocytic anemia with severe reticulocytopenia, and defined by absence or near absence of erythroid precursors in the bone marrow. First described in 1922, PRCA may be a primary autoimmune or clonal myeloid or lymphoid disorder, but may also be secondary to other disorders of immune dysregulation/autoimmunity, to infections, to neoplasms, or to drugs. Insights from the study of PRCA have helped illuminate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327320"
                        open
                      >
                        <summary class="article-expander__title">Identifying mechanisms of resistance by circulating tumour DNA in EVOLVE, a Phase II Trial of Cediranib Plus Olaparib for Ovarian Cancer at time of PARP Inhibitor Progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Purpose To evaluate the use of blood cell-free DNA (cfDNA) to identify emerging mechanisms of resistance to PARP inhibitors (PARPi) in high grade serous ovarian cancer (HGSOC). Patients and Methods We used targeted sequencing (TS) to analyse 78 longitudinal cfDNA samples collected from 30 patients with HGSOC enrolled in a phase II clinical trial evaluating cediranib (VEGF inhibitor) plus olaparib (PARPi) after progression on PARPi alone. cfDNA was collected at baseline, before treatment cycle 2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327996"
                        open
                      >
                        <summary class="article-expander__title">Pure red cell aplasia: the second hundred years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327996/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pure red cell aplasia (PRCA) is a rare hematologic syndrome, characterized by an isolated normocytic anemia with severe reticulocytopenia, and defined by absence or near absence of erythroid precursors in the bone marrow. First described in 1922, PRCA may be a primary autoimmune or clonal myeloid or lymphoid disorder, but may also be secondary to other disorders of immune dysregulation/autoimmunity, to infections, to neoplasms, or to drugs. Insights from the study of PRCA have helped illuminate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327320"
                        open
                      >
                        <summary class="article-expander__title">Identifying mechanisms of resistance by circulating tumour DNA in EVOLVE, a Phase II Trial of Cediranib Plus Olaparib for Ovarian Cancer at time of PARP Inhibitor Progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Purpose To evaluate the use of blood cell-free DNA (cfDNA) to identify emerging mechanisms of resistance to PARP inhibitors (PARPi) in high grade serous ovarian cancer (HGSOC). Patients and Methods We used targeted sequencing (TS) to analyse 78 longitudinal cfDNA samples collected from 30 patients with HGSOC enrolled in a phase II clinical trial evaluating cediranib (VEGF inhibitor) plus olaparib (PARPi) after progression on PARPi alone. cfDNA was collected at baseline, before treatment cycle 2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327996"
                        open
                      >
                        <summary class="article-expander__title">Pure red cell aplasia: the second hundred years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327996/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pure red cell aplasia (PRCA) is a rare hematologic syndrome, characterized by an isolated normocytic anemia with severe reticulocytopenia, and defined by absence or near absence of erythroid precursors in the bone marrow. First described in 1922, PRCA may be a primary autoimmune or clonal myeloid or lymphoid disorder, but may also be secondary to other disorders of immune dysregulation/autoimmunity, to infections, to neoplasms, or to drugs. Insights from the study of PRCA have helped illuminate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327320"
                        open
                      >
                        <summary class="article-expander__title">Identifying mechanisms of resistance by circulating tumour DNA in EVOLVE, a Phase II Trial of Cediranib Plus Olaparib for Ovarian Cancer at time of PARP Inhibitor Progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Purpose To evaluate the use of blood cell-free DNA (cfDNA) to identify emerging mechanisms of resistance to PARP inhibitors (PARPi) in high grade serous ovarian cancer (HGSOC). Patients and Methods We used targeted sequencing (TS) to analyse 78 longitudinal cfDNA samples collected from 30 patients with HGSOC enrolled in a phase II clinical trial evaluating cediranib (VEGF inhibitor) plus olaparib (PARPi) after progression on PARPi alone. cfDNA was collected at baseline, before treatment cycle 2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327996"
                        open
                      >
                        <summary class="article-expander__title">Pure red cell aplasia: the second hundred years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327996/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230627211357&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pure red cell aplasia (PRCA) is a rare hematologic syndrome, characterized by an isolated normocytic anemia with severe reticulocytopenia, and defined by absence or near absence of erythroid precursors in the bone marrow. First described in 1922, PRCA may be a primary autoimmune or clonal myeloid or lymphoid disorder, but may also be secondary to other disorders of immune dysregulation/autoimmunity, to infections, to neoplasms, or to drugs. Insights from the study of PRCA have helped illuminate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327320"
                        open
                      >
                        <summary class="article-expander__title">Identifying mechanisms of resistance by circulating tumour DNA in EVOLVE, a Phase II Trial of Cediranib Plus Olaparib for Ovarian Cancer at time of PARP Inhibitor Progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230627211357&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Purpose To evaluate the use of blood cell-free DNA (cfDNA) to identify emerging mechanisms of resistance to PARP inhibitors (PARPi) in high grade serous ovarian cancer (HGSOC). Patients and Methods We used targeted sequencing (TS) to analyse 78 longitudinal cfDNA samples collected from 30 patients with HGSOC enrolled in a phase II clinical trial evaluating cediranib (VEGF inhibitor) plus olaparib (PARPi) after progression on PARPi alone. cfDNA was collected at baseline, before treatment cycle 2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327996"
                        open
                      >
                        <summary class="article-expander__title">Pure red cell aplasia: the second hundred years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327996/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230626211237&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pure red cell aplasia (PRCA) is a rare hematologic syndrome, characterized by an isolated normocytic anemia with severe reticulocytopenia, and defined by absence or near absence of erythroid precursors in the bone marrow. First described in 1922, PRCA may be a primary autoimmune or clonal myeloid or lymphoid disorder, but may also be secondary to other disorders of immune dysregulation/autoimmunity, to infections, to neoplasms, or to drugs. Insights from the study of PRCA have helped illuminate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327320"
                        open
                      >
                        <summary class="article-expander__title">Identifying mechanisms of resistance by circulating tumour DNA in EVOLVE, a Phase II Trial of Cediranib Plus Olaparib for Ovarian Cancer at time of PARP Inhibitor Progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230626211237&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Purpose To evaluate the use of blood cell-free DNA (cfDNA) to identify emerging mechanisms of resistance to PARP inhibitors (PARPi) in high grade serous ovarian cancer (HGSOC). Patients and Methods We used targeted sequencing (TS) to analyse 78 longitudinal cfDNA samples collected from 30 patients with HGSOC enrolled in a phase II clinical trial evaluating cediranib (VEGF inhibitor) plus olaparib (PARPi) after progression on PARPi alone. cfDNA was collected at baseline, before treatment cycle 2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327996"
                        open
                      >
                        <summary class="article-expander__title">Pure red cell aplasia: the second hundred years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327996/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pure red cell aplasia (PRCA) is a rare hematologic syndrome, characterized by an isolated normocytic anemia with severe reticulocytopenia, and defined by absence or near absence of erythroid precursors in the bone marrow. First described in 1922, PRCA may be a primary autoimmune or clonal myeloid or lymphoid disorder, but may also be secondary to other disorders of immune dysregulation/autoimmunity, to infections, to neoplasms, or to drugs. Insights from the study of PRCA have helped illuminate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327320"
                        open
                      >
                        <summary class="article-expander__title">Identifying mechanisms of resistance by circulating tumour DNA in EVOLVE, a Phase II Trial of Cediranib Plus Olaparib for Ovarian Cancer at time of PARP Inhibitor Progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Purpose To evaluate the use of blood cell-free DNA (cfDNA) to identify emerging mechanisms of resistance to PARP inhibitors (PARPi) in high grade serous ovarian cancer (HGSOC). Patients and Methods We used targeted sequencing (TS) to analyse 78 longitudinal cfDNA samples collected from 30 patients with HGSOC enrolled in a phase II clinical trial evaluating cediranib (VEGF inhibitor) plus olaparib (PARPi) after progression on PARPi alone. cfDNA was collected at baseline, before treatment cycle 2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327996"
                        open
                      >
                        <summary class="article-expander__title">Pure red cell aplasia: the second hundred years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327996/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pure red cell aplasia (PRCA) is a rare hematologic syndrome, characterized by an isolated normocytic anemia with severe reticulocytopenia, and defined by absence or near absence of erythroid precursors in the bone marrow. First described in 1922, PRCA may be a primary autoimmune or clonal myeloid or lymphoid disorder, but may also be secondary to other disorders of immune dysregulation/autoimmunity, to infections, to neoplasms, or to drugs. Insights from the study of PRCA have helped illuminate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327320"
                        open
                      >
                        <summary class="article-expander__title">Identifying mechanisms of resistance by circulating tumour DNA in EVOLVE, a Phase II Trial of Cediranib Plus Olaparib for Ovarian Cancer at time of PARP Inhibitor Progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Purpose To evaluate the use of blood cell-free DNA (cfDNA) to identify emerging mechanisms of resistance to PARP inhibitors (PARPi) in high grade serous ovarian cancer (HGSOC). Patients and Methods We used targeted sequencing (TS) to analyse 78 longitudinal cfDNA samples collected from 30 patients with HGSOC enrolled in a phase II clinical trial evaluating cediranib (VEGF inhibitor) plus olaparib (PARPi) after progression on PARPi alone. cfDNA was collected at baseline, before treatment cycle 2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327996"
                        open
                      >
                        <summary class="article-expander__title">Pure red cell aplasia: the second hundred years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327996/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pure red cell aplasia (PRCA) is a rare hematologic syndrome, characterized by an isolated normocytic anemia with severe reticulocytopenia, and defined by absence or near absence of erythroid precursors in the bone marrow. First described in 1922, PRCA may be a primary autoimmune or clonal myeloid or lymphoid disorder, but may also be secondary to other disorders of immune dysregulation/autoimmunity, to infections, to neoplasms, or to drugs. Insights from the study of PRCA have helped illuminate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327320"
                        open
                      >
                        <summary class="article-expander__title">Identifying mechanisms of resistance by circulating tumour DNA in EVOLVE, a Phase II Trial of Cediranib Plus Olaparib for Ovarian Cancer at time of PARP Inhibitor Progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Purpose To evaluate the use of blood cell-free DNA (cfDNA) to identify emerging mechanisms of resistance to PARP inhibitors (PARPi) in high grade serous ovarian cancer (HGSOC). Patients and Methods We used targeted sequencing (TS) to analyse 78 longitudinal cfDNA samples collected from 30 patients with HGSOC enrolled in a phase II clinical trial evaluating cediranib (VEGF inhibitor) plus olaparib (PARPi) after progression on PARPi alone. cfDNA was collected at baseline, before treatment cycle 2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327996"
                        open
                      >
                        <summary class="article-expander__title">Pure red cell aplasia: the second hundred years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327996/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pure red cell aplasia (PRCA) is a rare hematologic syndrome, characterized by an isolated normocytic anemia with severe reticulocytopenia, and defined by absence or near absence of erythroid precursors in the bone marrow. First described in 1922, PRCA may be a primary autoimmune or clonal myeloid or lymphoid disorder, but may also be secondary to other disorders of immune dysregulation/autoimmunity, to infections, to neoplasms, or to drugs. Insights from the study of PRCA have helped illuminate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327320"
                        open
                      >
                        <summary class="article-expander__title">Identifying mechanisms of resistance by circulating tumour DNA in EVOLVE, a Phase II Trial of Cediranib Plus Olaparib for Ovarian Cancer at time of PARP Inhibitor Progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Purpose To evaluate the use of blood cell-free DNA (cfDNA) to identify emerging mechanisms of resistance to PARP inhibitors (PARPi) in high grade serous ovarian cancer (HGSOC). Patients and Methods We used targeted sequencing (TS) to analyse 78 longitudinal cfDNA samples collected from 30 patients with HGSOC enrolled in a phase II clinical trial evaluating cediranib (VEGF inhibitor) plus olaparib (PARPi) after progression on PARPi alone. cfDNA was collected at baseline, before treatment cycle 2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327996"
                        open
                      >
                        <summary class="article-expander__title">Pure red cell aplasia: the second hundred years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327996/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pure red cell aplasia (PRCA) is a rare hematologic syndrome, characterized by an isolated normocytic anemia with severe reticulocytopenia, and defined by absence or near absence of erythroid precursors in the bone marrow. First described in 1922, PRCA may be a primary autoimmune or clonal myeloid or lymphoid disorder, but may also be secondary to other disorders of immune dysregulation/autoimmunity, to infections, to neoplasms, or to drugs. Insights from the study of PRCA have helped illuminate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327320"
                        open
                      >
                        <summary class="article-expander__title">Identifying mechanisms of resistance by circulating tumour DNA in EVOLVE, a Phase II Trial of Cediranib Plus Olaparib for Ovarian Cancer at time of PARP Inhibitor Progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Purpose To evaluate the use of blood cell-free DNA (cfDNA) to identify emerging mechanisms of resistance to PARP inhibitors (PARPi) in high grade serous ovarian cancer (HGSOC). Patients and Methods We used targeted sequencing (TS) to analyse 78 longitudinal cfDNA samples collected from 30 patients with HGSOC enrolled in a phase II clinical trial evaluating cediranib (VEGF inhibitor) plus olaparib (PARPi) after progression on PARPi alone. cfDNA was collected at baseline, before treatment cycle 2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327996"
                        open
                      >
                        <summary class="article-expander__title">Pure red cell aplasia: the second hundred years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327996/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pure red cell aplasia (PRCA) is a rare hematologic syndrome, characterized by an isolated normocytic anemia with severe reticulocytopenia, and defined by absence or near absence of erythroid precursors in the bone marrow. First described in 1922, PRCA may be a primary autoimmune or clonal myeloid or lymphoid disorder, but may also be secondary to other disorders of immune dysregulation/autoimmunity, to infections, to neoplasms, or to drugs. Insights from the study of PRCA have helped illuminate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327320"
                        open
                      >
                        <summary class="article-expander__title">Identifying mechanisms of resistance by circulating tumour DNA in EVOLVE, a Phase II Trial of Cediranib Plus Olaparib for Ovarian Cancer at time of PARP Inhibitor Progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Purpose To evaluate the use of blood cell-free DNA (cfDNA) to identify emerging mechanisms of resistance to PARP inhibitors (PARPi) in high grade serous ovarian cancer (HGSOC). Patients and Methods We used targeted sequencing (TS) to analyse 78 longitudinal cfDNA samples collected from 30 patients with HGSOC enrolled in a phase II clinical trial evaluating cediranib (VEGF inhibitor) plus olaparib (PARPi) after progression on PARPi alone. cfDNA was collected at baseline, before treatment cycle 2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327996"
                        open
                      >
                        <summary class="article-expander__title">Pure red cell aplasia: the second hundred years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327996/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pure red cell aplasia (PRCA) is a rare hematologic syndrome, characterized by an isolated normocytic anemia with severe reticulocytopenia, and defined by absence or near absence of erythroid precursors in the bone marrow. First described in 1922, PRCA may be a primary autoimmune or clonal myeloid or lymphoid disorder, but may also be secondary to other disorders of immune dysregulation/autoimmunity, to infections, to neoplasms, or to drugs. Insights from the study of PRCA have helped illuminate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327320"
                        open
                      >
                        <summary class="article-expander__title">Identifying mechanisms of resistance by circulating tumour DNA in EVOLVE, a Phase II Trial of Cediranib Plus Olaparib for Ovarian Cancer at time of PARP Inhibitor Progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Purpose To evaluate the use of blood cell-free DNA (cfDNA) to identify emerging mechanisms of resistance to PARP inhibitors (PARPi) in high grade serous ovarian cancer (HGSOC). Patients and Methods We used targeted sequencing (TS) to analyse 78 longitudinal cfDNA samples collected from 30 patients with HGSOC enrolled in a phase II clinical trial evaluating cediranib (VEGF inhibitor) plus olaparib (PARPi) after progression on PARPi alone. cfDNA was collected at baseline, before treatment cycle 2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327996"
                        open
                      >
                        <summary class="article-expander__title">Pure red cell aplasia: the second hundred years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327996/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pure red cell aplasia (PRCA) is a rare hematologic syndrome, characterized by an isolated normocytic anemia with severe reticulocytopenia, and defined by absence or near absence of erythroid precursors in the bone marrow. First described in 1922, PRCA may be a primary autoimmune or clonal myeloid or lymphoid disorder, but may also be secondary to other disorders of immune dysregulation/autoimmunity, to infections, to neoplasms, or to drugs. Insights from the study of PRCA have helped illuminate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327320"
                        open
                      >
                        <summary class="article-expander__title">Identifying mechanisms of resistance by circulating tumour DNA in EVOLVE, a Phase II Trial of Cediranib Plus Olaparib for Ovarian Cancer at time of PARP Inhibitor Progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Purpose To evaluate the use of blood cell-free DNA (cfDNA) to identify emerging mechanisms of resistance to PARP inhibitors (PARPi) in high grade serous ovarian cancer (HGSOC). Patients and Methods We used targeted sequencing (TS) to analyse 78 longitudinal cfDNA samples collected from 30 patients with HGSOC enrolled in a phase II clinical trial evaluating cediranib (VEGF inhibitor) plus olaparib (PARPi) after progression on PARPi alone. cfDNA was collected at baseline, before treatment cycle 2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327996"
                        open
                      >
                        <summary class="article-expander__title">Pure red cell aplasia: the second hundred years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327996/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pure red cell aplasia (PRCA) is a rare hematologic syndrome, characterized by an isolated normocytic anemia with severe reticulocytopenia, and defined by absence or near absence of erythroid precursors in the bone marrow. First described in 1922, PRCA may be a primary autoimmune or clonal myeloid or lymphoid disorder, but may also be secondary to other disorders of immune dysregulation/autoimmunity, to infections, to neoplasms, or to drugs. Insights from the study of PRCA have helped illuminate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327320"
                        open
                      >
                        <summary class="article-expander__title">Identifying mechanisms of resistance by circulating tumour DNA in EVOLVE, a Phase II Trial of Cediranib Plus Olaparib for Ovarian Cancer at time of PARP Inhibitor Progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Purpose To evaluate the use of blood cell-free DNA (cfDNA) to identify emerging mechanisms of resistance to PARP inhibitors (PARPi) in high grade serous ovarian cancer (HGSOC). Patients and Methods We used targeted sequencing (TS) to analyse 78 longitudinal cfDNA samples collected from 30 patients with HGSOC enrolled in a phase II clinical trial evaluating cediranib (VEGF inhibitor) plus olaparib (PARPi) after progression on PARPi alone. cfDNA was collected at baseline, before treatment cycle 2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327320"
                        open
                      >
                        <summary class="article-expander__title">Identifying mechanisms of resistance by circulating tumour DNA in EVOLVE, a Phase II Trial of Cediranib Plus Olaparib for Ovarian Cancer at time of PARP Inhibitor Progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Purpose To evaluate the use of blood cell-free DNA (cfDNA) to identify emerging mechanisms of resistance to PARP inhibitors (PARPi) in high grade serous ovarian cancer (HGSOC). Patients and Methods We used targeted sequencing (TS) to analyse 78 longitudinal cfDNA samples collected from 30 patients with HGSOC enrolled in a phase II clinical trial evaluating cediranib (VEGF inhibitor) plus olaparib (PARPi) after progression on PARPi alone. cfDNA was collected at baseline, before treatment cycle 2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328818"
                        open
                      >
                        <summary class="article-expander__title">Personalized follow-up of circulating DNA in resected stage III/IV melanoma: PERCIMEL multicentric prospective study protocol</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230702211325&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: With more than 15,000 new cases /year in France and 2,000 deaths, cutaneous melanoma represents approximately 4% of incidental cancers and 1.2% of cancer related deaths. In locally advanced (stage III) or resectable metastatic (stage IV) melanomas, medical adjuvant treatment is proposed and recent advances had shown the benefit of anti-PD1/PDL1 and anti-CTLA4 immunotherapy as well as anti-BRAF and anti-MEK targeted therapy in BRAF V600 mutated tumors. However, the recurence rate at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328818"
                        open
                      >
                        <summary class="article-expander__title">Personalized follow-up of circulating DNA in resected stage III/IV melanoma: PERCIMEL multicentric prospective study protocol</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230701211521&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: With more than 15,000 new cases /year in France and 2,000 deaths, cutaneous melanoma represents approximately 4% of incidental cancers and 1.2% of cancer related deaths. In locally advanced (stage III) or resectable metastatic (stage IV) melanomas, medical adjuvant treatment is proposed and recent advances had shown the benefit of anti-PD1/PDL1 and anti-CTLA4 immunotherapy as well as anti-BRAF and anti-MEK targeted therapy in BRAF V600 mutated tumors. However, the recurence rate at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328818"
                        open
                      >
                        <summary class="article-expander__title">Personalized follow-up of circulating DNA in resected stage III/IV melanoma: PERCIMEL multicentric prospective study protocol</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230630211759&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: With more than 15,000 new cases /year in France and 2,000 deaths, cutaneous melanoma represents approximately 4% of incidental cancers and 1.2% of cancer related deaths. In locally advanced (stage III) or resectable metastatic (stage IV) melanomas, medical adjuvant treatment is proposed and recent advances had shown the benefit of anti-PD1/PDL1 and anti-CTLA4 immunotherapy as well as anti-BRAF and anti-MEK targeted therapy in BRAF V600 mutated tumors. However, the recurence rate at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328818"
                        open
                      >
                        <summary class="article-expander__title">Personalized follow-up of circulating DNA in resected stage III/IV melanoma: PERCIMEL multicentric prospective study protocol</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: With more than 15,000 new cases /year in France and 2,000 deaths, cutaneous melanoma represents approximately 4% of incidental cancers and 1.2% of cancer related deaths. In locally advanced (stage III) or resectable metastatic (stage IV) melanomas, medical adjuvant treatment is proposed and recent advances had shown the benefit of anti-PD1/PDL1 and anti-CTLA4 immunotherapy as well as anti-BRAF and anti-MEK targeted therapy in BRAF V600 mutated tumors. However, the recurence rate at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328818"
                        open
                      >
                        <summary class="article-expander__title">Personalized follow-up of circulating DNA in resected stage III/IV melanoma: PERCIMEL multicentric prospective study protocol</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: With more than 15,000 new cases /year in France and 2,000 deaths, cutaneous melanoma represents approximately 4% of incidental cancers and 1.2% of cancer related deaths. In locally advanced (stage III) or resectable metastatic (stage IV) melanomas, medical adjuvant treatment is proposed and recent advances had shown the benefit of anti-PD1/PDL1 and anti-CTLA4 immunotherapy as well as anti-BRAF and anti-MEK targeted therapy in BRAF V600 mutated tumors. However, the recurence rate at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328818"
                        open
                      >
                        <summary class="article-expander__title">Personalized follow-up of circulating DNA in resected stage III/IV melanoma: PERCIMEL multicentric prospective study protocol</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230627211357&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: With more than 15,000 new cases /year in France and 2,000 deaths, cutaneous melanoma represents approximately 4% of incidental cancers and 1.2% of cancer related deaths. In locally advanced (stage III) or resectable metastatic (stage IV) melanomas, medical adjuvant treatment is proposed and recent advances had shown the benefit of anti-PD1/PDL1 and anti-CTLA4 immunotherapy as well as anti-BRAF and anti-MEK targeted therapy in BRAF V600 mutated tumors. However, the recurence rate at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328455"
                        open
                      >
                        <summary class="article-expander__title">Cancer genomes tolerate deleterious coding mutations through somatic copy number amplifications of wild-type regions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230627211357&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers evolve under the accumulation of thousands of somatic mutations and chromosomal aberrations. While most coding mutations are deleterious, almost all protein-coding genes lack detectable signals of negative selection. This raises the question of how tumors tolerate such large amounts of deleterious mutations. Using 8,690 tumor samples from The Cancer Genome Atlas, we demonstrate that copy number amplifications frequently cover haploinsufficient genes in mutation-prone regions. This could...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328818"
                        open
                      >
                        <summary class="article-expander__title">Personalized follow-up of circulating DNA in resected stage III/IV melanoma: PERCIMEL multicentric prospective study protocol</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230626211237&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: With more than 15,000 new cases /year in France and 2,000 deaths, cutaneous melanoma represents approximately 4% of incidental cancers and 1.2% of cancer related deaths. In locally advanced (stage III) or resectable metastatic (stage IV) melanomas, medical adjuvant treatment is proposed and recent advances had shown the benefit of anti-PD1/PDL1 and anti-CTLA4 immunotherapy as well as anti-BRAF and anti-MEK targeted therapy in BRAF V600 mutated tumors. However, the recurence rate at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328455"
                        open
                      >
                        <summary class="article-expander__title">Cancer genomes tolerate deleterious coding mutations through somatic copy number amplifications of wild-type regions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230626211237&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers evolve under the accumulation of thousands of somatic mutations and chromosomal aberrations. While most coding mutations are deleterious, almost all protein-coding genes lack detectable signals of negative selection. This raises the question of how tumors tolerate such large amounts of deleterious mutations. Using 8,690 tumor samples from The Cancer Genome Atlas, we demonstrate that copy number amplifications frequently cover haploinsufficient genes in mutation-prone regions. This could...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323185"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal heterogeneity of digestive neuroendocrine neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323185/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230626211237&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroendocrine neoplasms (NENs) are initially monoclonal neoplasms that progressively become polyclonal, with very different genotypic and phenotypic characteristics leading to biological differences, including the Ki-67 proliferation index, morphology, or sensitivity to treatments. Whereas inter-patient heterogeneity has been well described, intra-tumor heterogeneity has been little studied. However, NENs present a high degree of heterogeneity, both spatially within the same location or between...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323170"
                        open
                      >
                        <summary class="article-expander__title">Genomic alterations dissection revealed MUC4 mutation as a potential driver in lung adenocarcinoma local recurrence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230626211237&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic alteration landscape was changing during LUAD recurrence to construct a more suitable environment for the survival of tumor cells. Several potential driver mutations and targets during LUAD recurrence were identified, such as MUC4, and more investigation was needed to verify the specific functions and roles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328818"
                        open
                      >
                        <summary class="article-expander__title">Personalized follow-up of circulating DNA in resected stage III/IV melanoma: PERCIMEL multicentric prospective study protocol</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: With more than 15,000 new cases /year in France and 2,000 deaths, cutaneous melanoma represents approximately 4% of incidental cancers and 1.2% of cancer related deaths. In locally advanced (stage III) or resectable metastatic (stage IV) melanomas, medical adjuvant treatment is proposed and recent advances had shown the benefit of anti-PD1/PDL1 and anti-CTLA4 immunotherapy as well as anti-BRAF and anti-MEK targeted therapy in BRAF V600 mutated tumors. However, the recurence rate at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328455"
                        open
                      >
                        <summary class="article-expander__title">Cancer genomes tolerate deleterious coding mutations through somatic copy number amplifications of wild-type regions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers evolve under the accumulation of thousands of somatic mutations and chromosomal aberrations. While most coding mutations are deleterious, almost all protein-coding genes lack detectable signals of negative selection. This raises the question of how tumors tolerate such large amounts of deleterious mutations. Using 8,690 tumor samples from The Cancer Genome Atlas, we demonstrate that copy number amplifications frequently cover haploinsufficient genes in mutation-prone regions. This could...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323185"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal heterogeneity of digestive neuroendocrine neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323185/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroendocrine neoplasms (NENs) are initially monoclonal neoplasms that progressively become polyclonal, with very different genotypic and phenotypic characteristics leading to biological differences, including the Ki-67 proliferation index, morphology, or sensitivity to treatments. Whereas inter-patient heterogeneity has been well described, intra-tumor heterogeneity has been little studied. However, NENs present a high degree of heterogeneity, both spatially within the same location or between...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323170"
                        open
                      >
                        <summary class="article-expander__title">Genomic alterations dissection revealed MUC4 mutation as a potential driver in lung adenocarcinoma local recurrence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic alteration landscape was changing during LUAD recurrence to construct a more suitable environment for the survival of tumor cells. Several potential driver mutations and targets during LUAD recurrence were identified, such as MUC4, and more investigation was needed to verify the specific functions and roles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328818"
                        open
                      >
                        <summary class="article-expander__title">Personalized follow-up of circulating DNA in resected stage III/IV melanoma: PERCIMEL multicentric prospective study protocol</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: With more than 15,000 new cases /year in France and 2,000 deaths, cutaneous melanoma represents approximately 4% of incidental cancers and 1.2% of cancer related deaths. In locally advanced (stage III) or resectable metastatic (stage IV) melanomas, medical adjuvant treatment is proposed and recent advances had shown the benefit of anti-PD1/PDL1 and anti-CTLA4 immunotherapy as well as anti-BRAF and anti-MEK targeted therapy in BRAF V600 mutated tumors. However, the recurence rate at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328455"
                        open
                      >
                        <summary class="article-expander__title">Cancer genomes tolerate deleterious coding mutations through somatic copy number amplifications of wild-type regions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers evolve under the accumulation of thousands of somatic mutations and chromosomal aberrations. While most coding mutations are deleterious, almost all protein-coding genes lack detectable signals of negative selection. This raises the question of how tumors tolerate such large amounts of deleterious mutations. Using 8,690 tumor samples from The Cancer Genome Atlas, we demonstrate that copy number amplifications frequently cover haploinsufficient genes in mutation-prone regions. This could...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323185"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal heterogeneity of digestive neuroendocrine neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323185/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroendocrine neoplasms (NENs) are initially monoclonal neoplasms that progressively become polyclonal, with very different genotypic and phenotypic characteristics leading to biological differences, including the Ki-67 proliferation index, morphology, or sensitivity to treatments. Whereas inter-patient heterogeneity has been well described, intra-tumor heterogeneity has been little studied. However, NENs present a high degree of heterogeneity, both spatially within the same location or between...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323170"
                        open
                      >
                        <summary class="article-expander__title">Genomic alterations dissection revealed MUC4 mutation as a potential driver in lung adenocarcinoma local recurrence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic alteration landscape was changing during LUAD recurrence to construct a more suitable environment for the survival of tumor cells. Several potential driver mutations and targets during LUAD recurrence were identified, such as MUC4, and more investigation was needed to verify the specific functions and roles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328818"
                        open
                      >
                        <summary class="article-expander__title">Personalized follow-up of circulating DNA in resected stage III/IV melanoma: PERCIMEL multicentric prospective study protocol</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: With more than 15,000 new cases /year in France and 2,000 deaths, cutaneous melanoma represents approximately 4% of incidental cancers and 1.2% of cancer related deaths. In locally advanced (stage III) or resectable metastatic (stage IV) melanomas, medical adjuvant treatment is proposed and recent advances had shown the benefit of anti-PD1/PDL1 and anti-CTLA4 immunotherapy as well as anti-BRAF and anti-MEK targeted therapy in BRAF V600 mutated tumors. However, the recurence rate at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328455"
                        open
                      >
                        <summary class="article-expander__title">Cancer genomes tolerate deleterious coding mutations through somatic copy number amplifications of wild-type regions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers evolve under the accumulation of thousands of somatic mutations and chromosomal aberrations. While most coding mutations are deleterious, almost all protein-coding genes lack detectable signals of negative selection. This raises the question of how tumors tolerate such large amounts of deleterious mutations. Using 8,690 tumor samples from The Cancer Genome Atlas, we demonstrate that copy number amplifications frequently cover haploinsufficient genes in mutation-prone regions. This could...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323185"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal heterogeneity of digestive neuroendocrine neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323185/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroendocrine neoplasms (NENs) are initially monoclonal neoplasms that progressively become polyclonal, with very different genotypic and phenotypic characteristics leading to biological differences, including the Ki-67 proliferation index, morphology, or sensitivity to treatments. Whereas inter-patient heterogeneity has been well described, intra-tumor heterogeneity has been little studied. However, NENs present a high degree of heterogeneity, both spatially within the same location or between...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323170"
                        open
                      >
                        <summary class="article-expander__title">Genomic alterations dissection revealed MUC4 mutation as a potential driver in lung adenocarcinoma local recurrence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic alteration landscape was changing during LUAD recurrence to construct a more suitable environment for the survival of tumor cells. Several potential driver mutations and targets during LUAD recurrence were identified, such as MUC4, and more investigation was needed to verify the specific functions and roles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328818"
                        open
                      >
                        <summary class="article-expander__title">Personalized follow-up of circulating DNA in resected stage III/IV melanoma: PERCIMEL multicentric prospective study protocol</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: With more than 15,000 new cases /year in France and 2,000 deaths, cutaneous melanoma represents approximately 4% of incidental cancers and 1.2% of cancer related deaths. In locally advanced (stage III) or resectable metastatic (stage IV) melanomas, medical adjuvant treatment is proposed and recent advances had shown the benefit of anti-PD1/PDL1 and anti-CTLA4 immunotherapy as well as anti-BRAF and anti-MEK targeted therapy in BRAF V600 mutated tumors. However, the recurence rate at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328455"
                        open
                      >
                        <summary class="article-expander__title">Cancer genomes tolerate deleterious coding mutations through somatic copy number amplifications of wild-type regions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers evolve under the accumulation of thousands of somatic mutations and chromosomal aberrations. While most coding mutations are deleterious, almost all protein-coding genes lack detectable signals of negative selection. This raises the question of how tumors tolerate such large amounts of deleterious mutations. Using 8,690 tumor samples from The Cancer Genome Atlas, we demonstrate that copy number amplifications frequently cover haploinsufficient genes in mutation-prone regions. This could...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323185"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal heterogeneity of digestive neuroendocrine neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323185/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroendocrine neoplasms (NENs) are initially monoclonal neoplasms that progressively become polyclonal, with very different genotypic and phenotypic characteristics leading to biological differences, including the Ki-67 proliferation index, morphology, or sensitivity to treatments. Whereas inter-patient heterogeneity has been well described, intra-tumor heterogeneity has been little studied. However, NENs present a high degree of heterogeneity, both spatially within the same location or between...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323170"
                        open
                      >
                        <summary class="article-expander__title">Genomic alterations dissection revealed MUC4 mutation as a potential driver in lung adenocarcinoma local recurrence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic alteration landscape was changing during LUAD recurrence to construct a more suitable environment for the survival of tumor cells. Several potential driver mutations and targets during LUAD recurrence were identified, such as MUC4, and more investigation was needed to verify the specific functions and roles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328818"
                        open
                      >
                        <summary class="article-expander__title">Personalized follow-up of circulating DNA in resected stage III/IV melanoma: PERCIMEL multicentric prospective study protocol</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: With more than 15,000 new cases /year in France and 2,000 deaths, cutaneous melanoma represents approximately 4% of incidental cancers and 1.2% of cancer related deaths. In locally advanced (stage III) or resectable metastatic (stage IV) melanomas, medical adjuvant treatment is proposed and recent advances had shown the benefit of anti-PD1/PDL1 and anti-CTLA4 immunotherapy as well as anti-BRAF and anti-MEK targeted therapy in BRAF V600 mutated tumors. However, the recurence rate at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328455"
                        open
                      >
                        <summary class="article-expander__title">Cancer genomes tolerate deleterious coding mutations through somatic copy number amplifications of wild-type regions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers evolve under the accumulation of thousands of somatic mutations and chromosomal aberrations. While most coding mutations are deleterious, almost all protein-coding genes lack detectable signals of negative selection. This raises the question of how tumors tolerate such large amounts of deleterious mutations. Using 8,690 tumor samples from The Cancer Genome Atlas, we demonstrate that copy number amplifications frequently cover haploinsufficient genes in mutation-prone regions. This could...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323185"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal heterogeneity of digestive neuroendocrine neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323185/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroendocrine neoplasms (NENs) are initially monoclonal neoplasms that progressively become polyclonal, with very different genotypic and phenotypic characteristics leading to biological differences, including the Ki-67 proliferation index, morphology, or sensitivity to treatments. Whereas inter-patient heterogeneity has been well described, intra-tumor heterogeneity has been little studied. However, NENs present a high degree of heterogeneity, both spatially within the same location or between...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323170"
                        open
                      >
                        <summary class="article-expander__title">Genomic alterations dissection revealed MUC4 mutation as a potential driver in lung adenocarcinoma local recurrence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic alteration landscape was changing during LUAD recurrence to construct a more suitable environment for the survival of tumor cells. Several potential driver mutations and targets during LUAD recurrence were identified, such as MUC4, and more investigation was needed to verify the specific functions and roles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328818"
                        open
                      >
                        <summary class="article-expander__title">Personalized follow-up of circulating DNA in resected stage III/IV melanoma: PERCIMEL multicentric prospective study protocol</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: With more than 15,000 new cases /year in France and 2,000 deaths, cutaneous melanoma represents approximately 4% of incidental cancers and 1.2% of cancer related deaths. In locally advanced (stage III) or resectable metastatic (stage IV) melanomas, medical adjuvant treatment is proposed and recent advances had shown the benefit of anti-PD1/PDL1 and anti-CTLA4 immunotherapy as well as anti-BRAF and anti-MEK targeted therapy in BRAF V600 mutated tumors. However, the recurence rate at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328455"
                        open
                      >
                        <summary class="article-expander__title">Cancer genomes tolerate deleterious coding mutations through somatic copy number amplifications of wild-type regions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers evolve under the accumulation of thousands of somatic mutations and chromosomal aberrations. While most coding mutations are deleterious, almost all protein-coding genes lack detectable signals of negative selection. This raises the question of how tumors tolerate such large amounts of deleterious mutations. Using 8,690 tumor samples from The Cancer Genome Atlas, we demonstrate that copy number amplifications frequently cover haploinsufficient genes in mutation-prone regions. This could...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323185"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal heterogeneity of digestive neuroendocrine neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323185/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroendocrine neoplasms (NENs) are initially monoclonal neoplasms that progressively become polyclonal, with very different genotypic and phenotypic characteristics leading to biological differences, including the Ki-67 proliferation index, morphology, or sensitivity to treatments. Whereas inter-patient heterogeneity has been well described, intra-tumor heterogeneity has been little studied. However, NENs present a high degree of heterogeneity, both spatially within the same location or between...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323170"
                        open
                      >
                        <summary class="article-expander__title">Genomic alterations dissection revealed MUC4 mutation as a potential driver in lung adenocarcinoma local recurrence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic alteration landscape was changing during LUAD recurrence to construct a more suitable environment for the survival of tumor cells. Several potential driver mutations and targets during LUAD recurrence were identified, such as MUC4, and more investigation was needed to verify the specific functions and roles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328818"
                        open
                      >
                        <summary class="article-expander__title">Personalized follow-up of circulating DNA in resected stage III/IV melanoma: PERCIMEL multicentric prospective study protocol</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: With more than 15,000 new cases /year in France and 2,000 deaths, cutaneous melanoma represents approximately 4% of incidental cancers and 1.2% of cancer related deaths. In locally advanced (stage III) or resectable metastatic (stage IV) melanomas, medical adjuvant treatment is proposed and recent advances had shown the benefit of anti-PD1/PDL1 and anti-CTLA4 immunotherapy as well as anti-BRAF and anti-MEK targeted therapy in BRAF V600 mutated tumors. However, the recurence rate at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328455"
                        open
                      >
                        <summary class="article-expander__title">Cancer genomes tolerate deleterious coding mutations through somatic copy number amplifications of wild-type regions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers evolve under the accumulation of thousands of somatic mutations and chromosomal aberrations. While most coding mutations are deleterious, almost all protein-coding genes lack detectable signals of negative selection. This raises the question of how tumors tolerate such large amounts of deleterious mutations. Using 8,690 tumor samples from The Cancer Genome Atlas, we demonstrate that copy number amplifications frequently cover haploinsufficient genes in mutation-prone regions. This could...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323185"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal heterogeneity of digestive neuroendocrine neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323185/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroendocrine neoplasms (NENs) are initially monoclonal neoplasms that progressively become polyclonal, with very different genotypic and phenotypic characteristics leading to biological differences, including the Ki-67 proliferation index, morphology, or sensitivity to treatments. Whereas inter-patient heterogeneity has been well described, intra-tumor heterogeneity has been little studied. However, NENs present a high degree of heterogeneity, both spatially within the same location or between...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323170"
                        open
                      >
                        <summary class="article-expander__title">Genomic alterations dissection revealed MUC4 mutation as a potential driver in lung adenocarcinoma local recurrence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic alteration landscape was changing during LUAD recurrence to construct a more suitable environment for the survival of tumor cells. Several potential driver mutations and targets during LUAD recurrence were identified, such as MUC4, and more investigation was needed to verify the specific functions and roles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328818"
                        open
                      >
                        <summary class="article-expander__title">Personalized follow-up of circulating DNA in resected stage III/IV melanoma: PERCIMEL multicentric prospective study protocol</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: With more than 15,000 new cases /year in France and 2,000 deaths, cutaneous melanoma represents approximately 4% of incidental cancers and 1.2% of cancer related deaths. In locally advanced (stage III) or resectable metastatic (stage IV) melanomas, medical adjuvant treatment is proposed and recent advances had shown the benefit of anti-PD1/PDL1 and anti-CTLA4 immunotherapy as well as anti-BRAF and anti-MEK targeted therapy in BRAF V600 mutated tumors. However, the recurence rate at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328455"
                        open
                      >
                        <summary class="article-expander__title">Cancer genomes tolerate deleterious coding mutations through somatic copy number amplifications of wild-type regions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers evolve under the accumulation of thousands of somatic mutations and chromosomal aberrations. While most coding mutations are deleterious, almost all protein-coding genes lack detectable signals of negative selection. This raises the question of how tumors tolerate such large amounts of deleterious mutations. Using 8,690 tumor samples from The Cancer Genome Atlas, we demonstrate that copy number amplifications frequently cover haploinsufficient genes in mutation-prone regions. This could...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323185"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal heterogeneity of digestive neuroendocrine neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323185/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroendocrine neoplasms (NENs) are initially monoclonal neoplasms that progressively become polyclonal, with very different genotypic and phenotypic characteristics leading to biological differences, including the Ki-67 proliferation index, morphology, or sensitivity to treatments. Whereas inter-patient heterogeneity has been well described, intra-tumor heterogeneity has been little studied. However, NENs present a high degree of heterogeneity, both spatially within the same location or between...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323170"
                        open
                      >
                        <summary class="article-expander__title">Genomic alterations dissection revealed MUC4 mutation as a potential driver in lung adenocarcinoma local recurrence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic alteration landscape was changing during LUAD recurrence to construct a more suitable environment for the survival of tumor cells. Several potential driver mutations and targets during LUAD recurrence were identified, such as MUC4, and more investigation was needed to verify the specific functions and roles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328818"
                        open
                      >
                        <summary class="article-expander__title">Personalized follow-up of circulating DNA in resected stage III/IV melanoma: PERCIMEL multicentric prospective study protocol</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: With more than 15,000 new cases /year in France and 2,000 deaths, cutaneous melanoma represents approximately 4% of incidental cancers and 1.2% of cancer related deaths. In locally advanced (stage III) or resectable metastatic (stage IV) melanomas, medical adjuvant treatment is proposed and recent advances had shown the benefit of anti-PD1/PDL1 and anti-CTLA4 immunotherapy as well as anti-BRAF and anti-MEK targeted therapy in BRAF V600 mutated tumors. However, the recurence rate at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328455"
                        open
                      >
                        <summary class="article-expander__title">Cancer genomes tolerate deleterious coding mutations through somatic copy number amplifications of wild-type regions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers evolve under the accumulation of thousands of somatic mutations and chromosomal aberrations. While most coding mutations are deleterious, almost all protein-coding genes lack detectable signals of negative selection. This raises the question of how tumors tolerate such large amounts of deleterious mutations. Using 8,690 tumor samples from The Cancer Genome Atlas, we demonstrate that copy number amplifications frequently cover haploinsufficient genes in mutation-prone regions. This could...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323185"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal heterogeneity of digestive neuroendocrine neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323185/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroendocrine neoplasms (NENs) are initially monoclonal neoplasms that progressively become polyclonal, with very different genotypic and phenotypic characteristics leading to biological differences, including the Ki-67 proliferation index, morphology, or sensitivity to treatments. Whereas inter-patient heterogeneity has been well described, intra-tumor heterogeneity has been little studied. However, NENs present a high degree of heterogeneity, both spatially within the same location or between...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323170"
                        open
                      >
                        <summary class="article-expander__title">Genomic alterations dissection revealed MUC4 mutation as a potential driver in lung adenocarcinoma local recurrence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic alteration landscape was changing during LUAD recurrence to construct a more suitable environment for the survival of tumor cells. Several potential driver mutations and targets during LUAD recurrence were identified, such as MUC4, and more investigation was needed to verify the specific functions and roles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323185"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal heterogeneity of digestive neuroendocrine neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323185/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroendocrine neoplasms (NENs) are initially monoclonal neoplasms that progressively become polyclonal, with very different genotypic and phenotypic characteristics leading to biological differences, including the Ki-67 proliferation index, morphology, or sensitivity to treatments. Whereas inter-patient heterogeneity has been well described, intra-tumor heterogeneity has been little studied. However, NENs present a high degree of heterogeneity, both spatially within the same location or between...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323170"
                        open
                      >
                        <summary class="article-expander__title">Genomic alterations dissection revealed MUC4 mutation as a potential driver in lung adenocarcinoma local recurrence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic alteration landscape was changing during LUAD recurrence to construct a more suitable environment for the survival of tumor cells. Several potential driver mutations and targets during LUAD recurrence were identified, such as MUC4, and more investigation was needed to verify the specific functions and roles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-15 datetime="2023-06-15T10:00:00.000Z">2023-06-15</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-15 datetime="2023-06-15T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608003622&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37322115"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is associated with protection from Alzheimer&#x27;s disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37322115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230629210818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a premalignant expansion of mutated hematopoietic stem cells. As CHIP-associated mutations are known to alter the development and function of myeloid cells, we hypothesized that CHIP may also be associated with the risk of Alzheimer&#x27;s disease (AD), a disease in which brain-resident myeloid cells are thought to have a major role. To perform association tests between CHIP and AD dementia, we analyzed blood DNA sequencing data from 1,362...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37322115"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is associated with protection from Alzheimer&#x27;s disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37322115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230628211019&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a premalignant expansion of mutated hematopoietic stem cells. As CHIP-associated mutations are known to alter the development and function of myeloid cells, we hypothesized that CHIP may also be associated with the risk of Alzheimer&#x27;s disease (AD), a disease in which brain-resident myeloid cells are thought to have a major role. To perform association tests between CHIP and AD dementia, we analyzed blood DNA sequencing data from 1,362...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37322115"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is associated with protection from Alzheimer&#x27;s disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37322115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230627211357&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a premalignant expansion of mutated hematopoietic stem cells. As CHIP-associated mutations are known to alter the development and function of myeloid cells, we hypothesized that CHIP may also be associated with the risk of Alzheimer&#x27;s disease (AD), a disease in which brain-resident myeloid cells are thought to have a major role. To perform association tests between CHIP and AD dementia, we analyzed blood DNA sequencing data from 1,362...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37322115"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is associated with protection from Alzheimer&#x27;s disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37322115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230626211237&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a premalignant expansion of mutated hematopoietic stem cells. As CHIP-associated mutations are known to alter the development and function of myeloid cells, we hypothesized that CHIP may also be associated with the risk of Alzheimer&#x27;s disease (AD), a disease in which brain-resident myeloid cells are thought to have a major role. To perform association tests between CHIP and AD dementia, we analyzed blood DNA sequencing data from 1,362...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37322115"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is associated with protection from Alzheimer&#x27;s disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37322115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a premalignant expansion of mutated hematopoietic stem cells. As CHIP-associated mutations are known to alter the development and function of myeloid cells, we hypothesized that CHIP may also be associated with the risk of Alzheimer&#x27;s disease (AD), a disease in which brain-resident myeloid cells are thought to have a major role. To perform association tests between CHIP and AD dementia, we analyzed blood DNA sequencing data from 1,362...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37322115"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is associated with protection from Alzheimer&#x27;s disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37322115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a premalignant expansion of mutated hematopoietic stem cells. As CHIP-associated mutations are known to alter the development and function of myeloid cells, we hypothesized that CHIP may also be associated with the risk of Alzheimer&#x27;s disease (AD), a disease in which brain-resident myeloid cells are thought to have a major role. To perform association tests between CHIP and AD dementia, we analyzed blood DNA sequencing data from 1,362...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37322115"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is associated with protection from Alzheimer&#x27;s disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37322115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a premalignant expansion of mutated hematopoietic stem cells. As CHIP-associated mutations are known to alter the development and function of myeloid cells, we hypothesized that CHIP may also be associated with the risk of Alzheimer&#x27;s disease (AD), a disease in which brain-resident myeloid cells are thought to have a major role. To perform association tests between CHIP and AD dementia, we analyzed blood DNA sequencing data from 1,362...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37322115"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is associated with protection from Alzheimer&#x27;s disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37322115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a premalignant expansion of mutated hematopoietic stem cells. As CHIP-associated mutations are known to alter the development and function of myeloid cells, we hypothesized that CHIP may also be associated with the risk of Alzheimer&#x27;s disease (AD), a disease in which brain-resident myeloid cells are thought to have a major role. To perform association tests between CHIP and AD dementia, we analyzed blood DNA sequencing data from 1,362...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37322115"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is associated with protection from Alzheimer&#x27;s disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37322115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a premalignant expansion of mutated hematopoietic stem cells. As CHIP-associated mutations are known to alter the development and function of myeloid cells, we hypothesized that CHIP may also be associated with the risk of Alzheimer&#x27;s disease (AD), a disease in which brain-resident myeloid cells are thought to have a major role. To perform association tests between CHIP and AD dementia, we analyzed blood DNA sequencing data from 1,362...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37322115"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is associated with protection from Alzheimer&#x27;s disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37322115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a premalignant expansion of mutated hematopoietic stem cells. As CHIP-associated mutations are known to alter the development and function of myeloid cells, we hypothesized that CHIP may also be associated with the risk of Alzheimer&#x27;s disease (AD), a disease in which brain-resident myeloid cells are thought to have a major role. To perform association tests between CHIP and AD dementia, we analyzed blood DNA sequencing data from 1,362...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37322115"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is associated with protection from Alzheimer&#x27;s disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37322115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a premalignant expansion of mutated hematopoietic stem cells. As CHIP-associated mutations are known to alter the development and function of myeloid cells, we hypothesized that CHIP may also be associated with the risk of Alzheimer&#x27;s disease (AD), a disease in which brain-resident myeloid cells are thought to have a major role. To perform association tests between CHIP and AD dementia, we analyzed blood DNA sequencing data from 1,362...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37322115"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is associated with protection from Alzheimer&#x27;s disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37322115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a premalignant expansion of mutated hematopoietic stem cells. As CHIP-associated mutations are known to alter the development and function of myeloid cells, we hypothesized that CHIP may also be associated with the risk of Alzheimer&#x27;s disease (AD), a disease in which brain-resident myeloid cells are thought to have a major role. To perform association tests between CHIP and AD dementia, we analyzed blood DNA sequencing data from 1,362...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37322115"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is associated with protection from Alzheimer&#x27;s disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37322115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a premalignant expansion of mutated hematopoietic stem cells. As CHIP-associated mutations are known to alter the development and function of myeloid cells, we hypothesized that CHIP may also be associated with the risk of Alzheimer&#x27;s disease (AD), a disease in which brain-resident myeloid cells are thought to have a major role. To perform association tests between CHIP and AD dementia, we analyzed blood DNA sequencing data from 1,362...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37322115"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is associated with protection from Alzheimer&#x27;s disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37322115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a premalignant expansion of mutated hematopoietic stem cells. As CHIP-associated mutations are known to alter the development and function of myeloid cells, we hypothesized that CHIP may also be associated with the risk of Alzheimer&#x27;s disease (AD), a disease in which brain-resident myeloid cells are thought to have a major role. To perform association tests between CHIP and AD dementia, we analyzed blood DNA sequencing data from 1,362...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-14 datetime="2023-06-14T10:00:00.000Z">2023-06-14</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-14 datetime="2023-06-14T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37316755"
                        open
                      >
                        <summary class="article-expander__title">Circulating cancer giant cells with unique characteristics frequently found in patients with myelodysplastic syndromes (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37316755/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are incurable diseases characterized by dysplastic hematopoietic cells, cytopenias in the blood and an inherent tendency for transformation to secondary acute myeloid leukemia (AML). Since most therapies fail to prevent rapid clonal evolution and disease resistance, new and non-invasive predictive markers are needed to monitor patients and adapt the therapeutic strategy. By using ISET, a very sensitive approach to isolate cells larger than mature leukocytes from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37316755"
                        open
                      >
                        <summary class="article-expander__title">Circulating cancer giant cells with unique characteristics frequently found in patients with myelodysplastic syndromes (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37316755/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are incurable diseases characterized by dysplastic hematopoietic cells, cytopenias in the blood and an inherent tendency for transformation to secondary acute myeloid leukemia (AML). Since most therapies fail to prevent rapid clonal evolution and disease resistance, new and non-invasive predictive markers are needed to monitor patients and adapt the therapeutic strategy. By using ISET, a very sensitive approach to isolate cells larger than mature leukocytes from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37316755"
                        open
                      >
                        <summary class="article-expander__title">Circulating cancer giant cells with unique characteristics frequently found in patients with myelodysplastic syndromes (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37316755/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are incurable diseases characterized by dysplastic hematopoietic cells, cytopenias in the blood and an inherent tendency for transformation to secondary acute myeloid leukemia (AML). Since most therapies fail to prevent rapid clonal evolution and disease resistance, new and non-invasive predictive markers are needed to monitor patients and adapt the therapeutic strategy. By using ISET, a very sensitive approach to isolate cells larger than mature leukocytes from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37316755"
                        open
                      >
                        <summary class="article-expander__title">Circulating cancer giant cells with unique characteristics frequently found in patients with myelodysplastic syndromes (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37316755/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are incurable diseases characterized by dysplastic hematopoietic cells, cytopenias in the blood and an inherent tendency for transformation to secondary acute myeloid leukemia (AML). Since most therapies fail to prevent rapid clonal evolution and disease resistance, new and non-invasive predictive markers are needed to monitor patients and adapt the therapeutic strategy. By using ISET, a very sensitive approach to isolate cells larger than mature leukocytes from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37316755"
                        open
                      >
                        <summary class="article-expander__title">Circulating cancer giant cells with unique characteristics frequently found in patients with myelodysplastic syndromes (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37316755/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are incurable diseases characterized by dysplastic hematopoietic cells, cytopenias in the blood and an inherent tendency for transformation to secondary acute myeloid leukemia (AML). Since most therapies fail to prevent rapid clonal evolution and disease resistance, new and non-invasive predictive markers are needed to monitor patients and adapt the therapeutic strategy. By using ISET, a very sensitive approach to isolate cells larger than mature leukocytes from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37316755"
                        open
                      >
                        <summary class="article-expander__title">Circulating cancer giant cells with unique characteristics frequently found in patients with myelodysplastic syndromes (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37316755/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are incurable diseases characterized by dysplastic hematopoietic cells, cytopenias in the blood and an inherent tendency for transformation to secondary acute myeloid leukemia (AML). Since most therapies fail to prevent rapid clonal evolution and disease resistance, new and non-invasive predictive markers are needed to monitor patients and adapt the therapeutic strategy. By using ISET, a very sensitive approach to isolate cells larger than mature leukocytes from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37316755"
                        open
                      >
                        <summary class="article-expander__title">Circulating cancer giant cells with unique characteristics frequently found in patients with myelodysplastic syndromes (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37316755/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are incurable diseases characterized by dysplastic hematopoietic cells, cytopenias in the blood and an inherent tendency for transformation to secondary acute myeloid leukemia (AML). Since most therapies fail to prevent rapid clonal evolution and disease resistance, new and non-invasive predictive markers are needed to monitor patients and adapt the therapeutic strategy. By using ISET, a very sensitive approach to isolate cells larger than mature leukocytes from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37316755"
                        open
                      >
                        <summary class="article-expander__title">Circulating cancer giant cells with unique characteristics frequently found in patients with myelodysplastic syndromes (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37316755/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are incurable diseases characterized by dysplastic hematopoietic cells, cytopenias in the blood and an inherent tendency for transformation to secondary acute myeloid leukemia (AML). Since most therapies fail to prevent rapid clonal evolution and disease resistance, new and non-invasive predictive markers are needed to monitor patients and adapt the therapeutic strategy. By using ISET, a very sensitive approach to isolate cells larger than mature leukocytes from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37316755"
                        open
                      >
                        <summary class="article-expander__title">Circulating cancer giant cells with unique characteristics frequently found in patients with myelodysplastic syndromes (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37316755/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are incurable diseases characterized by dysplastic hematopoietic cells, cytopenias in the blood and an inherent tendency for transformation to secondary acute myeloid leukemia (AML). Since most therapies fail to prevent rapid clonal evolution and disease resistance, new and non-invasive predictive markers are needed to monitor patients and adapt the therapeutic strategy. By using ISET, a very sensitive approach to isolate cells larger than mature leukocytes from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37316755"
                        open
                      >
                        <summary class="article-expander__title">Circulating cancer giant cells with unique characteristics frequently found in patients with myelodysplastic syndromes (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37316755/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are incurable diseases characterized by dysplastic hematopoietic cells, cytopenias in the blood and an inherent tendency for transformation to secondary acute myeloid leukemia (AML). Since most therapies fail to prevent rapid clonal evolution and disease resistance, new and non-invasive predictive markers are needed to monitor patients and adapt the therapeutic strategy. By using ISET, a very sensitive approach to isolate cells larger than mature leukocytes from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37316755"
                        open
                      >
                        <summary class="article-expander__title">Circulating cancer giant cells with unique characteristics frequently found in patients with myelodysplastic syndromes (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37316755/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are incurable diseases characterized by dysplastic hematopoietic cells, cytopenias in the blood and an inherent tendency for transformation to secondary acute myeloid leukemia (AML). Since most therapies fail to prevent rapid clonal evolution and disease resistance, new and non-invasive predictive markers are needed to monitor patients and adapt the therapeutic strategy. By using ISET, a very sensitive approach to isolate cells larger than mature leukocytes from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-13 datetime="2023-06-13T10:00:00.000Z">2023-06-13</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-13 datetime="2023-06-13T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20230810204637&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37311413"
                        open
                      >
                        <summary class="article-expander__title">Single cell clonotypic and transcriptional evolution of multiple myeloma precursor disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37311413/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma remains an incurable disease, and the cellular and molecular evolution from precursor conditions, including monoclonal gammopathy of undetermined significance and smoldering multiple myeloma, is incompletely understood. Here, we combine single-cell RNA and B cell receptor sequencing from fifty-two patients with myeloma precursors in comparison with myeloma and normal donors. Our comprehensive analysis reveals early genomic drivers of malignant transformation, distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37311413"
                        open
                      >
                        <summary class="article-expander__title">Single cell clonotypic and transcriptional evolution of multiple myeloma precursor disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37311413/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma remains an incurable disease, and the cellular and molecular evolution from precursor conditions, including monoclonal gammopathy of undetermined significance and smoldering multiple myeloma, is incompletely understood. Here, we combine single-cell RNA and B cell receptor sequencing from fifty-two patients with myeloma precursors in comparison with myeloma and normal donors. Our comprehensive analysis reveals early genomic drivers of malignant transformation, distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37310508"
                        open
                      >
                        <summary class="article-expander__title">Tamoxifen resistance induction results in the upregulation of ABCG2 expression and mitoxantrone resistance in MCF-7 breast cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37310508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer endocrine therapy can promote evolutionary dynamics and lead to changes in the gene expression profile of tumor cells. We aimed to assess the effect of tamoxifen (TAM)-resistance induction on ABCG2 pump mRNA, protein, and activity in ER + MCF-7 breast cancer cells. We also evaluated if the resistance to TAM leads to the cross-resistance toward mitoxantrone (MX), a well-known substrate of the ABCG2 pump. The ABCG2 mRNA and protein expression were compared in MCF-7 and its TAM-resistant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37311413"
                        open
                      >
                        <summary class="article-expander__title">Single cell clonotypic and transcriptional evolution of multiple myeloma precursor disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37311413/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma remains an incurable disease, and the cellular and molecular evolution from precursor conditions, including monoclonal gammopathy of undetermined significance and smoldering multiple myeloma, is incompletely understood. Here, we combine single-cell RNA and B cell receptor sequencing from fifty-two patients with myeloma precursors in comparison with myeloma and normal donors. Our comprehensive analysis reveals early genomic drivers of malignant transformation, distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37310508"
                        open
                      >
                        <summary class="article-expander__title">Tamoxifen resistance induction results in the upregulation of ABCG2 expression and mitoxantrone resistance in MCF-7 breast cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37310508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer endocrine therapy can promote evolutionary dynamics and lead to changes in the gene expression profile of tumor cells. We aimed to assess the effect of tamoxifen (TAM)-resistance induction on ABCG2 pump mRNA, protein, and activity in ER + MCF-7 breast cancer cells. We also evaluated if the resistance to TAM leads to the cross-resistance toward mitoxantrone (MX), a well-known substrate of the ABCG2 pump. The ABCG2 mRNA and protein expression were compared in MCF-7 and its TAM-resistant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37311413"
                        open
                      >
                        <summary class="article-expander__title">Single cell clonotypic and transcriptional evolution of multiple myeloma precursor disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37311413/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma remains an incurable disease, and the cellular and molecular evolution from precursor conditions, including monoclonal gammopathy of undetermined significance and smoldering multiple myeloma, is incompletely understood. Here, we combine single-cell RNA and B cell receptor sequencing from fifty-two patients with myeloma precursors in comparison with myeloma and normal donors. Our comprehensive analysis reveals early genomic drivers of malignant transformation, distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37310508"
                        open
                      >
                        <summary class="article-expander__title">Tamoxifen resistance induction results in the upregulation of ABCG2 expression and mitoxantrone resistance in MCF-7 breast cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37310508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer endocrine therapy can promote evolutionary dynamics and lead to changes in the gene expression profile of tumor cells. We aimed to assess the effect of tamoxifen (TAM)-resistance induction on ABCG2 pump mRNA, protein, and activity in ER + MCF-7 breast cancer cells. We also evaluated if the resistance to TAM leads to the cross-resistance toward mitoxantrone (MX), a well-known substrate of the ABCG2 pump. The ABCG2 mRNA and protein expression were compared in MCF-7 and its TAM-resistant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37311413"
                        open
                      >
                        <summary class="article-expander__title">Single cell clonotypic and transcriptional evolution of multiple myeloma precursor disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37311413/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma remains an incurable disease, and the cellular and molecular evolution from precursor conditions, including monoclonal gammopathy of undetermined significance and smoldering multiple myeloma, is incompletely understood. Here, we combine single-cell RNA and B cell receptor sequencing from fifty-two patients with myeloma precursors in comparison with myeloma and normal donors. Our comprehensive analysis reveals early genomic drivers of malignant transformation, distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37310508"
                        open
                      >
                        <summary class="article-expander__title">Tamoxifen resistance induction results in the upregulation of ABCG2 expression and mitoxantrone resistance in MCF-7 breast cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37310508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer endocrine therapy can promote evolutionary dynamics and lead to changes in the gene expression profile of tumor cells. We aimed to assess the effect of tamoxifen (TAM)-resistance induction on ABCG2 pump mRNA, protein, and activity in ER + MCF-7 breast cancer cells. We also evaluated if the resistance to TAM leads to the cross-resistance toward mitoxantrone (MX), a well-known substrate of the ABCG2 pump. The ABCG2 mRNA and protein expression were compared in MCF-7 and its TAM-resistant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37311413"
                        open
                      >
                        <summary class="article-expander__title">Single cell clonotypic and transcriptional evolution of multiple myeloma precursor disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37311413/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma remains an incurable disease, and the cellular and molecular evolution from precursor conditions, including monoclonal gammopathy of undetermined significance and smoldering multiple myeloma, is incompletely understood. Here, we combine single-cell RNA and B cell receptor sequencing from fifty-two patients with myeloma precursors in comparison with myeloma and normal donors. Our comprehensive analysis reveals early genomic drivers of malignant transformation, distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37310508"
                        open
                      >
                        <summary class="article-expander__title">Tamoxifen resistance induction results in the upregulation of ABCG2 expression and mitoxantrone resistance in MCF-7 breast cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37310508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer endocrine therapy can promote evolutionary dynamics and lead to changes in the gene expression profile of tumor cells. We aimed to assess the effect of tamoxifen (TAM)-resistance induction on ABCG2 pump mRNA, protein, and activity in ER + MCF-7 breast cancer cells. We also evaluated if the resistance to TAM leads to the cross-resistance toward mitoxantrone (MX), a well-known substrate of the ABCG2 pump. The ABCG2 mRNA and protein expression were compared in MCF-7 and its TAM-resistant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37311413"
                        open
                      >
                        <summary class="article-expander__title">Single cell clonotypic and transcriptional evolution of multiple myeloma precursor disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37311413/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma remains an incurable disease, and the cellular and molecular evolution from precursor conditions, including monoclonal gammopathy of undetermined significance and smoldering multiple myeloma, is incompletely understood. Here, we combine single-cell RNA and B cell receptor sequencing from fifty-two patients with myeloma precursors in comparison with myeloma and normal donors. Our comprehensive analysis reveals early genomic drivers of malignant transformation, distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37310508"
                        open
                      >
                        <summary class="article-expander__title">Tamoxifen resistance induction results in the upregulation of ABCG2 expression and mitoxantrone resistance in MCF-7 breast cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37310508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer endocrine therapy can promote evolutionary dynamics and lead to changes in the gene expression profile of tumor cells. We aimed to assess the effect of tamoxifen (TAM)-resistance induction on ABCG2 pump mRNA, protein, and activity in ER + MCF-7 breast cancer cells. We also evaluated if the resistance to TAM leads to the cross-resistance toward mitoxantrone (MX), a well-known substrate of the ABCG2 pump. The ABCG2 mRNA and protein expression were compared in MCF-7 and its TAM-resistant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37311413"
                        open
                      >
                        <summary class="article-expander__title">Single cell clonotypic and transcriptional evolution of multiple myeloma precursor disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37311413/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma remains an incurable disease, and the cellular and molecular evolution from precursor conditions, including monoclonal gammopathy of undetermined significance and smoldering multiple myeloma, is incompletely understood. Here, we combine single-cell RNA and B cell receptor sequencing from fifty-two patients with myeloma precursors in comparison with myeloma and normal donors. Our comprehensive analysis reveals early genomic drivers of malignant transformation, distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37310508"
                        open
                      >
                        <summary class="article-expander__title">Tamoxifen resistance induction results in the upregulation of ABCG2 expression and mitoxantrone resistance in MCF-7 breast cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37310508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer endocrine therapy can promote evolutionary dynamics and lead to changes in the gene expression profile of tumor cells. We aimed to assess the effect of tamoxifen (TAM)-resistance induction on ABCG2 pump mRNA, protein, and activity in ER + MCF-7 breast cancer cells. We also evaluated if the resistance to TAM leads to the cross-resistance toward mitoxantrone (MX), a well-known substrate of the ABCG2 pump. The ABCG2 mRNA and protein expression were compared in MCF-7 and its TAM-resistant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37311413"
                        open
                      >
                        <summary class="article-expander__title">Single cell clonotypic and transcriptional evolution of multiple myeloma precursor disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37311413/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma remains an incurable disease, and the cellular and molecular evolution from precursor conditions, including monoclonal gammopathy of undetermined significance and smoldering multiple myeloma, is incompletely understood. Here, we combine single-cell RNA and B cell receptor sequencing from fifty-two patients with myeloma precursors in comparison with myeloma and normal donors. Our comprehensive analysis reveals early genomic drivers of malignant transformation, distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37310508"
                        open
                      >
                        <summary class="article-expander__title">Tamoxifen resistance induction results in the upregulation of ABCG2 expression and mitoxantrone resistance in MCF-7 breast cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37310508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer endocrine therapy can promote evolutionary dynamics and lead to changes in the gene expression profile of tumor cells. We aimed to assess the effect of tamoxifen (TAM)-resistance induction on ABCG2 pump mRNA, protein, and activity in ER + MCF-7 breast cancer cells. We also evaluated if the resistance to TAM leads to the cross-resistance toward mitoxantrone (MX), a well-known substrate of the ABCG2 pump. The ABCG2 mRNA and protein expression were compared in MCF-7 and its TAM-resistant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37311413"
                        open
                      >
                        <summary class="article-expander__title">Single cell clonotypic and transcriptional evolution of multiple myeloma precursor disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37311413/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma remains an incurable disease, and the cellular and molecular evolution from precursor conditions, including monoclonal gammopathy of undetermined significance and smoldering multiple myeloma, is incompletely understood. Here, we combine single-cell RNA and B cell receptor sequencing from fifty-two patients with myeloma precursors in comparison with myeloma and normal donors. Our comprehensive analysis reveals early genomic drivers of malignant transformation, distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37310508"
                        open
                      >
                        <summary class="article-expander__title">Tamoxifen resistance induction results in the upregulation of ABCG2 expression and mitoxantrone resistance in MCF-7 breast cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37310508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer endocrine therapy can promote evolutionary dynamics and lead to changes in the gene expression profile of tumor cells. We aimed to assess the effect of tamoxifen (TAM)-resistance induction on ABCG2 pump mRNA, protein, and activity in ER + MCF-7 breast cancer cells. We also evaluated if the resistance to TAM leads to the cross-resistance toward mitoxantrone (MX), a well-known substrate of the ABCG2 pump. The ABCG2 mRNA and protein expression were compared in MCF-7 and its TAM-resistant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37311413"
                        open
                      >
                        <summary class="article-expander__title">Single cell clonotypic and transcriptional evolution of multiple myeloma precursor disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37311413/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma remains an incurable disease, and the cellular and molecular evolution from precursor conditions, including monoclonal gammopathy of undetermined significance and smoldering multiple myeloma, is incompletely understood. Here, we combine single-cell RNA and B cell receptor sequencing from fifty-two patients with myeloma precursors in comparison with myeloma and normal donors. Our comprehensive analysis reveals early genomic drivers of malignant transformation, distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37310508"
                        open
                      >
                        <summary class="article-expander__title">Tamoxifen resistance induction results in the upregulation of ABCG2 expression and mitoxantrone resistance in MCF-7 breast cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37310508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer endocrine therapy can promote evolutionary dynamics and lead to changes in the gene expression profile of tumor cells. We aimed to assess the effect of tamoxifen (TAM)-resistance induction on ABCG2 pump mRNA, protein, and activity in ER + MCF-7 breast cancer cells. We also evaluated if the resistance to TAM leads to the cross-resistance toward mitoxantrone (MX), a well-known substrate of the ABCG2 pump. The ABCG2 mRNA and protein expression were compared in MCF-7 and its TAM-resistant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37311413"
                        open
                      >
                        <summary class="article-expander__title">Single cell clonotypic and transcriptional evolution of multiple myeloma precursor disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37311413/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma remains an incurable disease, and the cellular and molecular evolution from precursor conditions, including monoclonal gammopathy of undetermined significance and smoldering multiple myeloma, is incompletely understood. Here, we combine single-cell RNA and B cell receptor sequencing from fifty-two patients with myeloma precursors in comparison with myeloma and normal donors. Our comprehensive analysis reveals early genomic drivers of malignant transformation, distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37310508"
                        open
                      >
                        <summary class="article-expander__title">Tamoxifen resistance induction results in the upregulation of ABCG2 expression and mitoxantrone resistance in MCF-7 breast cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37310508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer endocrine therapy can promote evolutionary dynamics and lead to changes in the gene expression profile of tumor cells. We aimed to assess the effect of tamoxifen (TAM)-resistance induction on ABCG2 pump mRNA, protein, and activity in ER + MCF-7 breast cancer cells. We also evaluated if the resistance to TAM leads to the cross-resistance toward mitoxantrone (MX), a well-known substrate of the ABCG2 pump. The ABCG2 mRNA and protein expression were compared in MCF-7 and its TAM-resistant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37310508"
                        open
                      >
                        <summary class="article-expander__title">Tamoxifen resistance induction results in the upregulation of ABCG2 expression and mitoxantrone resistance in MCF-7 breast cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37310508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer endocrine therapy can promote evolutionary dynamics and lead to changes in the gene expression profile of tumor cells. We aimed to assess the effect of tamoxifen (TAM)-resistance induction on ABCG2 pump mRNA, protein, and activity in ER + MCF-7 breast cancer cells. We also evaluated if the resistance to TAM leads to the cross-resistance toward mitoxantrone (MX), a well-known substrate of the ABCG2 pump. The ABCG2 mRNA and protein expression were compared in MCF-7 and its TAM-resistant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-06 datetime="2022-06-06T18:28:04.000Z">2022-06-06</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-06 datetime="2022-06-06T18:28:04.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="72a3a11d39c217b1a80cbb551444e6f6"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Nodal interferon exposure drives tumour-specific tolerance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1533878403463405569">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Nodal interferon exposure drives tumour-specific tolerance https://t.co/FT4NeBcFTg pic.twitter.com/NRHkVDUsOr
— NatureReviewsCancer (@NatureRevCancer) June 6, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-30 datetime="2022-05-30T04:37:42.000Z">2022-05-30</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-30 datetime="2022-05-30T04:37:42.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="bb6bc242bcbc356c4a38db850e74d812"
                        open
                      >
                        <summary class="article-expander__title">@VIBConferences: #Cancer #immunotherapy has become a crucial pillar of anticancer therapy. 19 leading researchers at #ImmunoOnco22 will give new insights into the immune response to tumors and tackle unresolved questions.⏰Registration Deadline: Thursday 2 June. Read more</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/VIBConferences/status/1531132720952266752">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>#Cancer #immunotherapy has become a crucial pillar of anticancer therapy. 19 leading researchers at #ImmunoOnco22 will give new insights into the immune response to tumors and tackle unresolved questions.
⏰Registration Deadline: Thursday 2 June. Read more https://t.co/cc0HC5cj6W pic.twitter.com/u8LFuzBbsV
— VIB Conferences (@VIBConferences) May 30, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-27 datetime="2022-05-27T14:45:03.000Z">2022-05-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-27 datetime="2022-05-27T14:45:03.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="ed6c996dc1a718afb523de6229a53131"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Nonsense-mediated RNA decay: an emerging modulator of malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1530198403686072320">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Nonsense-mediated RNA decay: an emerging modulator of malignancy https://t.co/wjuHplkpof pic.twitter.com/1SoKYhRsPO
— NatureReviewsCancer (@NatureRevCancer) May 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="d15a3a4b24853812d7fa21fd56de5815"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: Our June issue, out NOW! With articles on #therapyresistance, #senescence &amp; targeting CDK4 &amp; CDK6 in #cancer! Plus, a Comment on cancer mRNA vaccines as well as the launch of our new Tools of the Trade articles #NRCToT. So much content 👇🏻💌</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1530174481313972224">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Our June issue, out NOW! With articles on #therapyresistance, #senescence &amp; targeting CDK4 &amp; CDK6 in #cancer! Plus, a Comment on cancer mRNA vaccines as well as the launch of our new Tools of the Trade articles #NRCToT. So much content 👇🏻
💌 https://t.co/Fo6XRWV3OZ@natrescancer pic.twitter.com/rbxRPqLV6g
— NatureReviewsCancer (@NatureRevCancer) May 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-26 datetime="2022-05-26T08:53:50.000Z">2022-05-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-26 datetime="2022-05-26T08:53:50.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="fe13755a2693ea65459e5e21df097d2a"
                        open
                      >
                        <summary class="article-expander__title">@UlrikeHarjes: This Review by @MarcusDGon @cantleylab is part of the @NatureRevCancer series on #Diet &amp; #SystemicMetabolism in cancer:  do love this topic in general plus I am handling research at @NatureMedicine now.. get in touch if you have something of interest☺️</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/UlrikeHarjes/status/1529747629936193537">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>This Review by @MarcusDGon @cantleylab is part of the @NatureRevCancer series on #Diet &amp; #SystemicMetabolism in cancer: https://t.co/p3Y2cdNhco
I do love this topic in general plus I am handling research at @NatureMedicine now.. get in touch if you have something of interest☺️ https://t.co/qWH2dTv8j9
— Ulrike Harjes (@UlrikeHarjes) May 26, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-25 datetime="2022-05-25T17:35:04.000Z">2022-05-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-25 datetime="2022-05-25T17:35:04.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="9ba5279424b31b0122a4ae20c18e6d03"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Developing dietary interventions as therapy for cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1529516411059437568">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Developing dietary interventions as therapy for cancer https://t.co/ijuqctk1wJ pic.twitter.com/KHPpMKnCFa
— NatureReviewsCancer (@NatureRevCancer) May 25, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-18 datetime="2022-05-18T14:11:32.000Z">2022-05-18</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-18 datetime="2022-05-18T14:11:32.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="611d40f2e6ea2796b028bebda41a86dd"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Digging for treasures in the tumour interactome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1526928478490095616">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Digging for treasures in the tumour interactome https://t.co/NhHYAwUgLE
— NatureReviewsCancer (@NatureRevCancer) May 18, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-17 datetime="2022-05-17T13:48:21.000Z">2022-05-17</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-17 datetime="2022-05-17T13:48:21.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="9590a971b00448cd312078183965b21d"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: As part of our overall commitment to diversity, equity and inclusion at Springer Nature, we’d like to better understand what DEI means within the global research community. Take the survey here</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1526560254091313156">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>As part of our overall commitment to diversity, equity and inclusion at Springer Nature, we’d like to better understand what DEI means within the global research community. Take the survey here https://t.co/03yLiUxFTq
— NatureReviewsCancer (@NatureRevCancer) May 17, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-12 datetime="2022-05-12T15:25:46.000Z">2022-05-12</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-12 datetime="2022-05-12T15:25:46.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="aa38ff4245b93e964773f191844ddc7a"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: Newsflash! We have a new article type in the journal - Tools of the Trade #NRCToT 🧰 🛠️ where we hope to enable more #ECRs to interact with the editorial team &amp; have a bigger voice in our content. For more info, check out the Editorial 👇🏻</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1524772831904509953">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Newsflash! We have a new article type in the journal - Tools of the Trade #NRCToT 🧰 🛠️ where we hope to enable more #ECRs to interact with the editorial team &amp; have a bigger voice in our content. For more info, check out the Editorial 👇🏻 https://t.co/VbyoUJmTRJ
— NatureReviewsCancer (@NatureRevCancer) May 12, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-10 datetime="2022-05-10T16:46:03.000Z">2022-05-10</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-10 datetime="2022-05-10T16:46:03.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="65aca76b9f794aaa668d84ad32dca4a0"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Tools for the next generation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1524068259712036864">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Tools for the next generation https://t.co/lEzTPFYDf5
— NatureReviewsCancer (@NatureRevCancer) May 10, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="afa9b6784b4ccc7c05ed4efd26ddfb2d"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Hunting down rare drug-tolerant cycling cells with Watermelon</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1524022709386190856">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Hunting down rare drug-tolerant cycling cells with Watermelon https://t.co/XXT0c9SAYX
— NatureReviewsCancer (@NatureRevCancer) May 10, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-29 datetime="2022-04-29T14:40:05.000Z">2022-04-29</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-29 datetime="2022-04-29T14:40:05.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="43e2d5ccebc9030f737d656f47ac9d98"
                        open
                      >
                        <summary class="article-expander__title">@VarunVenkatara2: Very happy that our latest Perspective is now online @NatureRevCancer ! We discuss implications of malignant, multicellular networks and potential therapeutic strategies how to disconnect them. Thanks to everyone involved ! #CancerNeuroscience #TumorMicrotubes #neurotwitter</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/VarunVenkatara2/status/1520050291986604033">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Very happy that our latest Perspective is now online @NatureRevCancer ! We discuss implications of malignant, multicellular networks and potential therapeutic strategies how to disconnect them. Thanks to everyone involved ! #CancerNeuroscience #TumorMicrotubes #neurotwitter https://t.co/S55cKTGNFd
— Varun Venkataramani (@VarunVenkatara2) April 29, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="9f6b8644378881f4c2e6b6833c93fc2f"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! An inhospitable site</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1520020349068316673">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! An inhospitable site https://t.co/0dvqIKCWxh pic.twitter.com/PtY8J9Fz7r
— NatureReviewsCancer (@NatureRevCancer) April 29, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="f6d91efcd60dae769e2722585bdfa8df"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Disconnecting multicellular networks in brain tumours</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1520020340201598978">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Disconnecting multicellular networks in brain tumours https://t.co/LJrzy8bRma pic.twitter.com/CO7BnooP0r
— NatureReviewsCancer (@NatureRevCancer) April 29, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-28 datetime="2022-04-28T14:04:34.000Z">2022-04-28</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-28 datetime="2022-04-28T14:04:34.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="76e3b436bc8e43ad21366a80c120e395"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: Our May issue, out NOW! With Reviews on #CRISPR in cancer biology &amp; #therapy, ROS in tumour development &amp; progression, and cancer #proteogenomics, and Comments on esophageal cancer research in Eastern Africa and on Critical Race Theory.💌</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1519678964364623872">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Our May issue, out NOW! With Reviews on #CRISPR in cancer biology &amp; #therapy, ROS in tumour development &amp; progression, and cancer #proteogenomics, and Comments on esophageal cancer research in Eastern Africa and on Critical Race Theory.
💌 https://t.co/vQwHFJhpwv@natrescancer pic.twitter.com/6kYgjRYhdT
— NatureReviewsCancer (@NatureRevCancer) April 28, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="fdf224ea293cdf039086d16e5699a59a"
                        open
                      >
                        <summary class="article-expander__title">@tmurraystewart: Our latest publication on #polyamines is online!@HopkinsMedicine</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/tmurraystewart/status/1519413460978704384">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Our latest publication on #polyamines is online!@HopkinsMedicine https://t.co/ehRGgN1w9R
— Tracy Murray Stewart (@tmurraystewart) April 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-27 datetime="2022-04-27T18:22:03.000Z">2022-04-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-27 datetime="2022-04-27T18:22:03.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="4f43226ec4dc8e531ff6e64a9c5ab8e6"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Polyamines in cancer: integrating organismal metabolism and antitumour immunity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1519381375874265088">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Polyamines in cancer: integrating organismal metabolism and antitumour immunity https://t.co/YCJX1hRQLO pic.twitter.com/zRUCRNXyxH
— NatureReviewsCancer (@NatureRevCancer) April 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="a44ed79b9f76aad377b9272fb392f553"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW #ResearchHighlight! Lv, Liu, Mo &amp; co demonstrate that in pancreatic #PDAC cells, gasdermin E transports YBX1 to the nucleus, which promotes mucin expression, thereby providing tumour cells with a barrier against digestive enzymes.📯</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1519317944995569664">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW #ResearchHighlight! Lv, Liu, Mo &amp; co demonstrate that in pancreatic #PDAC cells, gasdermin E transports YBX1 to the nucleus, which promotes mucin expression, thereby providing tumour cells with a barrier against digestive enzymes.
📯 https://t.co/JTUhX0DHQ1@natrescancer pic.twitter.com/2PdCmsDu2p
— NatureReviewsCancer (@NatureRevCancer) April 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-26 datetime="2022-04-26T14:58:00.000Z">2022-04-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-26 datetime="2022-04-26T14:58:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="06ff7641d8f5fcc0dff91f835ac3bbc1"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: In the April issue of @NatureRevCancer | Comment, Reimagining cancer research with art, by @CancerSciArtist, Stefani Shoreibah &amp; @tdanino  @natrescancer @Columbia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1518967636561956865">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>In the April issue of @NatureRevCancer | Comment, Reimagining cancer research with art, by @CancerSciArtist, Stefani Shoreibah &amp; @tdanino
 https://t.co/GeTVCMJQsp @natrescancer
 @Columbia pic.twitter.com/ZXW6A0Fe0c
— NatureReviewsCancer (@NatureRevCancer) April 26, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-25 datetime="2022-04-25T12:12:32.000Z">2022-04-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-25 datetime="2022-04-25T12:12:32.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="96b2ddff4eb269cf343213c6a1b1ce95"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Tracking dormant disseminated tumour cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1518563607440621569">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Tracking dormant disseminated tumour cells https://t.co/kSXyWAvCxF
— NatureReviewsCancer (@NatureRevCancer) April 25, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>

    <footer>
      <a class="footer-link" href="https://github.com/MacwanEric/cancer.rss.feed/actions/runs/5827634461">
        <time id="build-timestamp" datetime="2023-08-11T00:46:40.536Z">2023-08-11T00:46:40.536Z</time>
      </a>
      <a class="footer-link" href="https://github.com/osmoscraft/osmosfeed">osmosfeed 1.15.1</a>
    </footer>
    <script src="index.js?v1.14.4"></script>
    <!-- %before-body-end.html% -->
  </body>

</html>