<!DOCTYPE html>
<html lang="en">

  <head>
    <title>CancerFeed</title>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="robots" content="noindex, nofollow" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
    <link rel="alternate" type="application/rss+xml" title="CancerFeed" href="feed.atom" />
    <link href="index.css?v1.14.4" rel="stylesheet" />
    <style>
  :root {
    /**
     * Solarized Light
     * Ethan Schoonover (modified by aramisgithub)
     */
    --base00: #fdf6e3;
    --base01: #eee8d5;
    --base02: #93a1a1;
    --base03: #839496;
    --base04: #657b83;
    --base05: #586e75;
    --base06: #073642;
    --base07: #002b36;
    --base08: #dc322f;
    --base09: #cb4b16;
    --base0A: #b58900;
    --base0B: #859900;
    --base0C: #2aa198;
    --base0D: #268bd2;
    --base0E: #6c71c4;
    --base0F: #d33682;

    /**
    * Components
    */
    --source-name-color: var(--base0C);
    /**
   * Typograph
   */
    --font-family-default: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell,
      "Helvetica Neue", sans-serif;
    --font-size-L: 3.6rem;
    --font-size-m: 2.2rem;
    --font-size-s: 1.8rem;
    --font-size-scaler: 52.5%; /* 1rem = 10px */
    --line-height-body: 1.4;
    --line-height-heading: 1.2;

  }
</style>

  </head>

  <body>
    <!-- %after-body-begin.html% -->
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-25 datetime="2023-06-25T10:00:00.000Z">2023-06-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-25 datetime="2023-06-25T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37356012"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in aplastic anemia: failed tumor surveillance or maladaptive recovery?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37356012/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution to secondary paroxysmal nocturnal hemoglobinuria (PNH) or myeloid neoplasia (MN) represents one of the long-term complications of patients with aplastic anemia (AA). The recent evidence in the field of immunology and the application of next-generation sequencing have shed light on the molecular underpinnings of these clonal complications, revealing clinical and molecular risk factors as well as potential immunological players. Particularly, whether MN evolution represents a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-24 datetime="2023-06-24T10:00:00.000Z">2023-06-24</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-24 datetime="2023-06-24T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37355534"
                        open
                      >
                        <summary class="article-expander__title">In Vitro and In Vivo Models for Metastatic Intestinal Tumors Using Genotype-Defined Organoids</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37355534/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It has been established that the accumulation of driver gene mutations causes malignant progression of colorectal cancer (CRC) through positive selection and clonal expansion, similar to Darwin&#x27;s evolution. Following this multistep tumorigenesis concept, we previously showed the specific mutation patterns for each process of malignant progression, including submucosal invasion, epithelial mesenchymal transition (EMT), intravasation, and metastasis, using genetically engineered mouse and organoid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-22 datetime="2023-06-22T10:00:00.000Z">2023-06-22</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-22 datetime="2023-06-22T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37349125"
                        open
                      >
                        <summary class="article-expander__title">Genomic approaches to cancer and minimal residual disease detection using circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37349125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsies using cell-free circulating tumor DNA (ctDNA) are being used frequently in both research and clinical settings. ctDNA can be used to identify actionable mutations to personalize systemic therapy, detect post-treatment minimal residual disease (MRD), and predict responses to immunotherapy. ctDNA can also be isolated from a range of different biofluids, with the possibility of detecting locoregional MRD with increased sensitivity if sampling more proximally than blood plasma....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37345823"
                        open
                      >
                        <summary class="article-expander__title">Birth Weight Is Associated With Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Outcomes in Adulthood</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37345823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background High and low birth weight are independently associated with increased cardiovascular disease risk in adulthood. Clonal hematopoiesis of indeterminate potential (CHIP), the age-related clonal expansion of hematopoietic cells with preleukemic somatic mutations, predicts incident cardiovascular disease independent of traditional cardiovascular risk factors. Whether birth weight predicts development of CHIP later in life is unknown. Methods and Results A total of 221 047 adults enrolled...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37349125"
                        open
                      >
                        <summary class="article-expander__title">Genomic approaches to cancer and minimal residual disease detection using circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37349125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsies using cell-free circulating tumor DNA (ctDNA) are being used frequently in both research and clinical settings. ctDNA can be used to identify actionable mutations to personalize systemic therapy, detect post-treatment minimal residual disease (MRD), and predict responses to immunotherapy. ctDNA can also be isolated from a range of different biofluids, with the possibility of detecting locoregional MRD with increased sensitivity if sampling more proximally than blood plasma....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37345823"
                        open
                      >
                        <summary class="article-expander__title">Birth Weight Is Associated With Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Outcomes in Adulthood</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37345823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background High and low birth weight are independently associated with increased cardiovascular disease risk in adulthood. Clonal hematopoiesis of indeterminate potential (CHIP), the age-related clonal expansion of hematopoietic cells with preleukemic somatic mutations, predicts incident cardiovascular disease independent of traditional cardiovascular risk factors. Whether birth weight predicts development of CHIP later in life is unknown. Methods and Results A total of 221 047 adults enrolled...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37349125"
                        open
                      >
                        <summary class="article-expander__title">Genomic approaches to cancer and minimal residual disease detection using circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37349125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid biopsies using cell-free circulating tumor DNA (ctDNA) are being used frequently in both research and clinical settings. ctDNA can be used to identify actionable mutations to personalize systemic therapy, detect post-treatment minimal residual disease (MRD), and predict responses to immunotherapy. ctDNA can also be isolated from a range of different biofluids, with the possibility of detecting locoregional MRD with increased sensitivity if sampling more proximally than blood plasma....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37345823"
                        open
                      >
                        <summary class="article-expander__title">Birth Weight Is Associated With Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Outcomes in Adulthood</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37345823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background High and low birth weight are independently associated with increased cardiovascular disease risk in adulthood. Clonal hematopoiesis of indeterminate potential (CHIP), the age-related clonal expansion of hematopoietic cells with preleukemic somatic mutations, predicts incident cardiovascular disease independent of traditional cardiovascular risk factors. Whether birth weight predicts development of CHIP later in life is unknown. Methods and Results A total of 221 047 adults enrolled...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37345823"
                        open
                      >
                        <summary class="article-expander__title">Birth Weight Is Associated With Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Outcomes in Adulthood</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37345823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background High and low birth weight are independently associated with increased cardiovascular disease risk in adulthood. Clonal hematopoiesis of indeterminate potential (CHIP), the age-related clonal expansion of hematopoietic cells with preleukemic somatic mutations, predicts incident cardiovascular disease independent of traditional cardiovascular risk factors. Whether birth weight predicts development of CHIP later in life is unknown. Methods and Results A total of 221 047 adults enrolled...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-21 datetime="2023-06-21T10:00:00.000Z">2023-06-21</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-21 datetime="2023-06-21T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344430"
                        open
                      >
                        <summary class="article-expander__title">Bilateral Adrenal Infarction that Developed in Latent Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344430/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bilateral adrenal infarction is an extremely rare disease, and it has been reported that some coagulation abnormalities, including essential thrombocythemia (ET), exist in the background. We herein report a 76-year-old patient in whom the platelet count had been in the normal range at the onset of adrenal infarction but subsequently increased to 102×10⁴/μL at 7 months later, leading to the diagnosis of JAK2V617F-positive ET. As the presence of the JAK2V617F mutation increases the risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344430"
                        open
                      >
                        <summary class="article-expander__title">Bilateral Adrenal Infarction that Developed in Latent Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344430/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bilateral adrenal infarction is an extremely rare disease, and it has been reported that some coagulation abnormalities, including essential thrombocythemia (ET), exist in the background. We herein report a 76-year-old patient in whom the platelet count had been in the normal range at the onset of adrenal infarction but subsequently increased to 102×10⁴/μL at 7 months later, leading to the diagnosis of JAK2V617F-positive ET. As the presence of the JAK2V617F mutation increases the risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344430"
                        open
                      >
                        <summary class="article-expander__title">Bilateral Adrenal Infarction that Developed in Latent Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344430/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bilateral adrenal infarction is an extremely rare disease, and it has been reported that some coagulation abnormalities, including essential thrombocythemia (ET), exist in the background. We herein report a 76-year-old patient in whom the platelet count had been in the normal range at the onset of adrenal infarction but subsequently increased to 102×10⁴/μL at 7 months later, leading to the diagnosis of JAK2V617F-positive ET. As the presence of the JAK2V617F mutation increases the risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344430"
                        open
                      >
                        <summary class="article-expander__title">Bilateral Adrenal Infarction that Developed in Latent Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344430/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bilateral adrenal infarction is an extremely rare disease, and it has been reported that some coagulation abnormalities, including essential thrombocythemia (ET), exist in the background. We herein report a 76-year-old patient in whom the platelet count had been in the normal range at the onset of adrenal infarction but subsequently increased to 102×10⁴/μL at 7 months later, leading to the diagnosis of JAK2V617F-positive ET. As the presence of the JAK2V617F mutation increases the risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344104"
                        open
                      >
                        <summary class="article-expander__title">Modeling and predicting cancer clonal evolution with reinforcement learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer results from an evolutionary process that typically yields multiple clones with varying sets of mutations within the same tumor. Accurately modeling this process is key to understanding and predicting cancer evolution. Here, we introduce CloMu (Clone To Mutation), a flexible and low-parameter tree-generative model of cancer evolution. CloMu uses a two-layer neural network trained via reinforcement learning to determine the probability of new mutations based on the existing mutations on a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37342197"
                        open
                      >
                        <summary class="article-expander__title">Phenotyping of lymphoproliferative tumours generated in xenografts of non-small cell lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37342197/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Patient-derived xenograft (PDX) models involve the engraftment of tumour tissue in immunocompromised mice and represent an important pre-clinical oncology research method. A limitation of non-small cell lung cancer (NSCLC) PDX model derivation in NOD-scid IL2Rgamma^(null) (NSG) mice is that a subset of initial engraftments are of lymphocytic, rather than tumour origin.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344104"
                        open
                      >
                        <summary class="article-expander__title">Modeling and predicting cancer clonal evolution with reinforcement learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer results from an evolutionary process that typically yields multiple clones with varying sets of mutations within the same tumor. Accurately modeling this process is key to understanding and predicting cancer evolution. Here, we introduce CloMu (Clone To Mutation), a flexible and low-parameter tree-generative model of cancer evolution. CloMu uses a two-layer neural network trained via reinforcement learning to determine the probability of new mutations based on the existing mutations on a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37342197"
                        open
                      >
                        <summary class="article-expander__title">Phenotyping of lymphoproliferative tumours generated in xenografts of non-small cell lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37342197/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Patient-derived xenograft (PDX) models involve the engraftment of tumour tissue in immunocompromised mice and represent an important pre-clinical oncology research method. A limitation of non-small cell lung cancer (NSCLC) PDX model derivation in NOD-scid IL2Rgamma^(null) (NSG) mice is that a subset of initial engraftments are of lymphocytic, rather than tumour origin.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344104"
                        open
                      >
                        <summary class="article-expander__title">Modeling and predicting cancer clonal evolution with reinforcement learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer results from an evolutionary process that typically yields multiple clones with varying sets of mutations within the same tumor. Accurately modeling this process is key to understanding and predicting cancer evolution. Here, we introduce CloMu (Clone To Mutation), a flexible and low-parameter tree-generative model of cancer evolution. CloMu uses a two-layer neural network trained via reinforcement learning to determine the probability of new mutations based on the existing mutations on a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37343201"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis: Updates and Implications at the Solid Tumor-Immune Interface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37343201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent larger-scale studies of patients with cancer and longitudinal population cohorts have revealed how age-related expansions of mutant hematopoietic cells (clonal hematopoiesis [CH]) have differential associations with incident and prevalent cancers and their outcomes. Increasing recognition and deeper understanding of genetic subtypes of CH are yielding insights into the tumor-immune interface that may help to explain the heterogeneous impact of CH on tumorigenesis and treatment. Herein, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37342197"
                        open
                      >
                        <summary class="article-expander__title">Phenotyping of lymphoproliferative tumours generated in xenografts of non-small cell lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37342197/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Patient-derived xenograft (PDX) models involve the engraftment of tumour tissue in immunocompromised mice and represent an important pre-clinical oncology research method. A limitation of non-small cell lung cancer (NSCLC) PDX model derivation in NOD-scid IL2Rgamma^(null) (NSG) mice is that a subset of initial engraftments are of lymphocytic, rather than tumour origin.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37344104"
                        open
                      >
                        <summary class="article-expander__title">Modeling and predicting cancer clonal evolution with reinforcement learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37344104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer results from an evolutionary process that typically yields multiple clones with varying sets of mutations within the same tumor. Accurately modeling this process is key to understanding and predicting cancer evolution. Here, we introduce CloMu (Clone To Mutation), a flexible and low-parameter tree-generative model of cancer evolution. CloMu uses a two-layer neural network trained via reinforcement learning to determine the probability of new mutations based on the existing mutations on a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37342197"
                        open
                      >
                        <summary class="article-expander__title">Phenotyping of lymphoproliferative tumours generated in xenografts of non-small cell lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37342197/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Patient-derived xenograft (PDX) models involve the engraftment of tumour tissue in immunocompromised mice and represent an important pre-clinical oncology research method. A limitation of non-small cell lung cancer (NSCLC) PDX model derivation in NOD-scid IL2Rgamma^(null) (NSG) mice is that a subset of initial engraftments are of lymphocytic, rather than tumour origin.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37342197"
                        open
                      >
                        <summary class="article-expander__title">Phenotyping of lymphoproliferative tumours generated in xenografts of non-small cell lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37342197/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Patient-derived xenograft (PDX) models involve the engraftment of tumour tissue in immunocompromised mice and represent an important pre-clinical oncology research method. A limitation of non-small cell lung cancer (NSCLC) PDX model derivation in NOD-scid IL2Rgamma^(null) (NSG) mice is that a subset of initial engraftments are of lymphocytic, rather than tumour origin.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-20 datetime="2023-06-20T10:00:00.000Z">2023-06-20</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-20 datetime="2023-06-20T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37339686"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic remodelling by neoadjuvant chemoradiotherapy (nCRT) and indicative role of acquired INDEL percentage for nCRT efficacy in esophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37339686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: nCRT remodels the genome and transcriptome of ESCC. Acquired INDEL% is a potential biomarker to indicate the effectiveness of nCRT and radiation sensitivity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37339686"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic remodelling by neoadjuvant chemoradiotherapy (nCRT) and indicative role of acquired INDEL percentage for nCRT efficacy in esophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37339686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: nCRT remodels the genome and transcriptome of ESCC. Acquired INDEL% is a potential biomarker to indicate the effectiveness of nCRT and radiation sensitivity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37339686"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic remodelling by neoadjuvant chemoradiotherapy (nCRT) and indicative role of acquired INDEL percentage for nCRT efficacy in esophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37339686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: nCRT remodels the genome and transcriptome of ESCC. Acquired INDEL% is a potential biomarker to indicate the effectiveness of nCRT and radiation sensitivity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37339686"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic remodelling by neoadjuvant chemoradiotherapy (nCRT) and indicative role of acquired INDEL percentage for nCRT efficacy in esophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37339686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: nCRT remodels the genome and transcriptome of ESCC. Acquired INDEL% is a potential biomarker to indicate the effectiveness of nCRT and radiation sensitivity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37339686"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic remodelling by neoadjuvant chemoradiotherapy (nCRT) and indicative role of acquired INDEL percentage for nCRT efficacy in esophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37339686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: nCRT remodels the genome and transcriptome of ESCC. Acquired INDEL% is a potential biomarker to indicate the effectiveness of nCRT and radiation sensitivity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-19 datetime="2023-06-19T10:00:00.000Z">2023-06-19</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-19 datetime="2023-06-19T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333361"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Incident Heart Failure with Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333361/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP, particularly mutations in TET2 , represents a potential new risk factor for incident HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333361"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Incident Heart Failure with Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333361/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP, particularly mutations in TET2 , represents a potential new risk factor for incident HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333361"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Incident Heart Failure with Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333361/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP, particularly mutations in TET2 , represents a potential new risk factor for incident HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333361"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Incident Heart Failure with Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333361/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP, particularly mutations in TET2 , represents a potential new risk factor for incident HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333361"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Incident Heart Failure with Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333361/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP, particularly mutations in TET2 , represents a potential new risk factor for incident HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333361"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Incident Heart Failure with Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333361/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP, particularly mutations in TET2 , represents a potential new risk factor for incident HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333361"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Incident Heart Failure with Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333361/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP, particularly mutations in TET2 , represents a potential new risk factor for incident HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37336285"
                        open
                      >
                        <summary class="article-expander__title">Cellular barcoding: From developmental tracing to anti-tumor drug discovery</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37336285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution has gained immense attention in explaining cancer cell status, history, and fate during cancer progression. Current single-cell or spatial transcriptome technologies have broadened our understanding of various mechanisms underlying cancer initiation, relapse, and drug resistance. However, technical challenges still hinder a better understanding of the dynamics of distinctive phenotypic states and abnormal trajectories from normal physiological transition to malignant stages....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333132"
                        open
                      >
                        <summary class="article-expander__title">Melanoma clonal subline analysis uncovers heterogeneity-driven immunotherapy resistance mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) can promote cancer progression and treatment failure, but the complexity of the regulatory programs and contextual factors involved complicates its study. To understand the specific contribution of ITH to immune checkpoint blockade (ICB) response, we generated single cell-derived clonal sublines from an ICB-sensitive and genetically and phenotypically heterogeneous mouse melanoma model, M4. Genomic and single cell transcriptomic analyses uncovered the diversity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37336285"
                        open
                      >
                        <summary class="article-expander__title">Cellular barcoding: From developmental tracing to anti-tumor drug discovery</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37336285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution has gained immense attention in explaining cancer cell status, history, and fate during cancer progression. Current single-cell or spatial transcriptome technologies have broadened our understanding of various mechanisms underlying cancer initiation, relapse, and drug resistance. However, technical challenges still hinder a better understanding of the dynamics of distinctive phenotypic states and abnormal trajectories from normal physiological transition to malignant stages....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333132"
                        open
                      >
                        <summary class="article-expander__title">Melanoma clonal subline analysis uncovers heterogeneity-driven immunotherapy resistance mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) can promote cancer progression and treatment failure, but the complexity of the regulatory programs and contextual factors involved complicates its study. To understand the specific contribution of ITH to immune checkpoint blockade (ICB) response, we generated single cell-derived clonal sublines from an ICB-sensitive and genetically and phenotypically heterogeneous mouse melanoma model, M4. Genomic and single cell transcriptomic analyses uncovered the diversity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37336285"
                        open
                      >
                        <summary class="article-expander__title">Cellular barcoding: From developmental tracing to anti-tumor drug discovery</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37336285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution has gained immense attention in explaining cancer cell status, history, and fate during cancer progression. Current single-cell or spatial transcriptome technologies have broadened our understanding of various mechanisms underlying cancer initiation, relapse, and drug resistance. However, technical challenges still hinder a better understanding of the dynamics of distinctive phenotypic states and abnormal trajectories from normal physiological transition to malignant stages....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333132"
                        open
                      >
                        <summary class="article-expander__title">Melanoma clonal subline analysis uncovers heterogeneity-driven immunotherapy resistance mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) can promote cancer progression and treatment failure, but the complexity of the regulatory programs and contextual factors involved complicates its study. To understand the specific contribution of ITH to immune checkpoint blockade (ICB) response, we generated single cell-derived clonal sublines from an ICB-sensitive and genetically and phenotypically heterogeneous mouse melanoma model, M4. Genomic and single cell transcriptomic analyses uncovered the diversity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37336285"
                        open
                      >
                        <summary class="article-expander__title">Cellular barcoding: From developmental tracing to anti-tumor drug discovery</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37336285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution has gained immense attention in explaining cancer cell status, history, and fate during cancer progression. Current single-cell or spatial transcriptome technologies have broadened our understanding of various mechanisms underlying cancer initiation, relapse, and drug resistance. However, technical challenges still hinder a better understanding of the dynamics of distinctive phenotypic states and abnormal trajectories from normal physiological transition to malignant stages....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333132"
                        open
                      >
                        <summary class="article-expander__title">Melanoma clonal subline analysis uncovers heterogeneity-driven immunotherapy resistance mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) can promote cancer progression and treatment failure, but the complexity of the regulatory programs and contextual factors involved complicates its study. To understand the specific contribution of ITH to immune checkpoint blockade (ICB) response, we generated single cell-derived clonal sublines from an ICB-sensitive and genetically and phenotypically heterogeneous mouse melanoma model, M4. Genomic and single cell transcriptomic analyses uncovered the diversity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37336285"
                        open
                      >
                        <summary class="article-expander__title">Cellular barcoding: From developmental tracing to anti-tumor drug discovery</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37336285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution has gained immense attention in explaining cancer cell status, history, and fate during cancer progression. Current single-cell or spatial transcriptome technologies have broadened our understanding of various mechanisms underlying cancer initiation, relapse, and drug resistance. However, technical challenges still hinder a better understanding of the dynamics of distinctive phenotypic states and abnormal trajectories from normal physiological transition to malignant stages....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333132"
                        open
                      >
                        <summary class="article-expander__title">Melanoma clonal subline analysis uncovers heterogeneity-driven immunotherapy resistance mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) can promote cancer progression and treatment failure, but the complexity of the regulatory programs and contextual factors involved complicates its study. To understand the specific contribution of ITH to immune checkpoint blockade (ICB) response, we generated single cell-derived clonal sublines from an ICB-sensitive and genetically and phenotypically heterogeneous mouse melanoma model, M4. Genomic and single cell transcriptomic analyses uncovered the diversity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37336285"
                        open
                      >
                        <summary class="article-expander__title">Cellular barcoding: From developmental tracing to anti-tumor drug discovery</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37336285/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution has gained immense attention in explaining cancer cell status, history, and fate during cancer progression. Current single-cell or spatial transcriptome technologies have broadened our understanding of various mechanisms underlying cancer initiation, relapse, and drug resistance. However, technical challenges still hinder a better understanding of the dynamics of distinctive phenotypic states and abnormal trajectories from normal physiological transition to malignant stages....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333132"
                        open
                      >
                        <summary class="article-expander__title">Melanoma clonal subline analysis uncovers heterogeneity-driven immunotherapy resistance mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) can promote cancer progression and treatment failure, but the complexity of the regulatory programs and contextual factors involved complicates its study. To understand the specific contribution of ITH to immune checkpoint blockade (ICB) response, we generated single cell-derived clonal sublines from an ICB-sensitive and genetically and phenotypically heterogeneous mouse melanoma model, M4. Genomic and single cell transcriptomic analyses uncovered the diversity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37333132"
                        open
                      >
                        <summary class="article-expander__title">Melanoma clonal subline analysis uncovers heterogeneity-driven immunotherapy resistance mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37333132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) can promote cancer progression and treatment failure, but the complexity of the regulatory programs and contextual factors involved complicates its study. To understand the specific contribution of ITH to immune checkpoint blockade (ICB) response, we generated single cell-derived clonal sublines from an ICB-sensitive and genetically and phenotypically heterogeneous mouse melanoma model, M4. Genomic and single cell transcriptomic analyses uncovered the diversity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-18 datetime="2023-06-18T10:00:00.000Z">2023-06-18</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-18 datetime="2023-06-18T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331781"
                        open
                      >
                        <summary class="article-expander__title">Emerging Roles of Circulating Tumor DNA for Increased Precision and Personalization in Radiation Oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent breakthroughs in circulating tumor DNA (ctDNA) technologies present a compelling opportunity to combine this emerging liquid biopsy approach with the field of radiogenomics, the study of how tumor genomics correlate with radiotherapy response and radiotoxicity. Canonically, ctDNA levels reflect metastatic tumor burden, although newer ultrasensitive technologies can be used after curative-intent radiotherapy of localized disease to assess ctDNA for minimal residual disease (MRD) detection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331781"
                        open
                      >
                        <summary class="article-expander__title">Emerging Roles of Circulating Tumor DNA for Increased Precision and Personalization in Radiation Oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent breakthroughs in circulating tumor DNA (ctDNA) technologies present a compelling opportunity to combine this emerging liquid biopsy approach with the field of radiogenomics, the study of how tumor genomics correlate with radiotherapy response and radiotoxicity. Canonically, ctDNA levels reflect metastatic tumor burden, although newer ultrasensitive technologies can be used after curative-intent radiotherapy of localized disease to assess ctDNA for minimal residual disease (MRD) detection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331781"
                        open
                      >
                        <summary class="article-expander__title">Emerging Roles of Circulating Tumor DNA for Increased Precision and Personalization in Radiation Oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent breakthroughs in circulating tumor DNA (ctDNA) technologies present a compelling opportunity to combine this emerging liquid biopsy approach with the field of radiogenomics, the study of how tumor genomics correlate with radiotherapy response and radiotoxicity. Canonically, ctDNA levels reflect metastatic tumor burden, although newer ultrasensitive technologies can be used after curative-intent radiotherapy of localized disease to assess ctDNA for minimal residual disease (MRD) detection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331781"
                        open
                      >
                        <summary class="article-expander__title">Emerging Roles of Circulating Tumor DNA for Increased Precision and Personalization in Radiation Oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent breakthroughs in circulating tumor DNA (ctDNA) technologies present a compelling opportunity to combine this emerging liquid biopsy approach with the field of radiogenomics, the study of how tumor genomics correlate with radiotherapy response and radiotoxicity. Canonically, ctDNA levels reflect metastatic tumor burden, although newer ultrasensitive technologies can be used after curative-intent radiotherapy of localized disease to assess ctDNA for minimal residual disease (MRD) detection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331781"
                        open
                      >
                        <summary class="article-expander__title">Emerging Roles of Circulating Tumor DNA for Increased Precision and Personalization in Radiation Oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent breakthroughs in circulating tumor DNA (ctDNA) technologies present a compelling opportunity to combine this emerging liquid biopsy approach with the field of radiogenomics, the study of how tumor genomics correlate with radiotherapy response and radiotoxicity. Canonically, ctDNA levels reflect metastatic tumor burden, although newer ultrasensitive technologies can be used after curative-intent radiotherapy of localized disease to assess ctDNA for minimal residual disease (MRD) detection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331781"
                        open
                      >
                        <summary class="article-expander__title">Emerging Roles of Circulating Tumor DNA for Increased Precision and Personalization in Radiation Oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent breakthroughs in circulating tumor DNA (ctDNA) technologies present a compelling opportunity to combine this emerging liquid biopsy approach with the field of radiogenomics, the study of how tumor genomics correlate with radiotherapy response and radiotoxicity. Canonically, ctDNA levels reflect metastatic tumor burden, although newer ultrasensitive technologies can be used after curative-intent radiotherapy of localized disease to assess ctDNA for minimal residual disease (MRD) detection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331781"
                        open
                      >
                        <summary class="article-expander__title">Emerging Roles of Circulating Tumor DNA for Increased Precision and Personalization in Radiation Oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent breakthroughs in circulating tumor DNA (ctDNA) technologies present a compelling opportunity to combine this emerging liquid biopsy approach with the field of radiogenomics, the study of how tumor genomics correlate with radiotherapy response and radiotoxicity. Canonically, ctDNA levels reflect metastatic tumor burden, although newer ultrasensitive technologies can be used after curative-intent radiotherapy of localized disease to assess ctDNA for minimal residual disease (MRD) detection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331179"
                        open
                      >
                        <summary class="article-expander__title">Clinical trial designs for evaluating and exploiting cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331179/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolution of drug-resistant cell subpopulations causes cancer treatment failure. Current preclinical evidence shows that it is possible to model herding of clonal evolution and collateral sensitivity where an initial treatment could favourably influence the response to a subsequent one. Novel therapy strategies exploiting this understanding are being considered, and clinical trial designs for steering cancer evolution are needed. Furthermore, preclinical evidence suggests that different...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331179"
                        open
                      >
                        <summary class="article-expander__title">Clinical trial designs for evaluating and exploiting cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331179/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolution of drug-resistant cell subpopulations causes cancer treatment failure. Current preclinical evidence shows that it is possible to model herding of clonal evolution and collateral sensitivity where an initial treatment could favourably influence the response to a subsequent one. Novel therapy strategies exploiting this understanding are being considered, and clinical trial designs for steering cancer evolution are needed. Furthermore, preclinical evidence suggests that different...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331179"
                        open
                      >
                        <summary class="article-expander__title">Clinical trial designs for evaluating and exploiting cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331179/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolution of drug-resistant cell subpopulations causes cancer treatment failure. Current preclinical evidence shows that it is possible to model herding of clonal evolution and collateral sensitivity where an initial treatment could favourably influence the response to a subsequent one. Novel therapy strategies exploiting this understanding are being considered, and clinical trial designs for steering cancer evolution are needed. Furthermore, preclinical evidence suggests that different...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331179"
                        open
                      >
                        <summary class="article-expander__title">Clinical trial designs for evaluating and exploiting cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331179/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolution of drug-resistant cell subpopulations causes cancer treatment failure. Current preclinical evidence shows that it is possible to model herding of clonal evolution and collateral sensitivity where an initial treatment could favourably influence the response to a subsequent one. Novel therapy strategies exploiting this understanding are being considered, and clinical trial designs for steering cancer evolution are needed. Furthermore, preclinical evidence suggests that different...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331179"
                        open
                      >
                        <summary class="article-expander__title">Clinical trial designs for evaluating and exploiting cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331179/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolution of drug-resistant cell subpopulations causes cancer treatment failure. Current preclinical evidence shows that it is possible to model herding of clonal evolution and collateral sensitivity where an initial treatment could favourably influence the response to a subsequent one. Novel therapy strategies exploiting this understanding are being considered, and clinical trial designs for steering cancer evolution are needed. Furthermore, preclinical evidence suggests that different...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331179"
                        open
                      >
                        <summary class="article-expander__title">Clinical trial designs for evaluating and exploiting cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331179/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolution of drug-resistant cell subpopulations causes cancer treatment failure. Current preclinical evidence shows that it is possible to model herding of clonal evolution and collateral sensitivity where an initial treatment could favourably influence the response to a subsequent one. Novel therapy strategies exploiting this understanding are being considered, and clinical trial designs for steering cancer evolution are needed. Furthermore, preclinical evidence suggests that different...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37331179"
                        open
                      >
                        <summary class="article-expander__title">Clinical trial designs for evaluating and exploiting cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37331179/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolution of drug-resistant cell subpopulations causes cancer treatment failure. Current preclinical evidence shows that it is possible to model herding of clonal evolution and collateral sensitivity where an initial treatment could favourably influence the response to a subsequent one. Novel therapy strategies exploiting this understanding are being considered, and clinical trial designs for steering cancer evolution are needed. Furthermore, preclinical evidence suggests that different...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-16 datetime="2023-06-16T10:00:00.000Z">2023-06-16</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-16 datetime="2023-06-16T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327996"
                        open
                      >
                        <summary class="article-expander__title">Pure red cell aplasia: the second hundred years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327996/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pure red cell aplasia (PRCA) is a rare hematologic syndrome, characterized by an isolated normocytic anemia with severe reticulocytopenia, and defined by absence or near absence of erythroid precursors in the bone marrow. First described in 1922, PRCA may be a primary autoimmune or clonal myeloid or lymphoid disorder, but may also be secondary to other disorders of immune dysregulation/autoimmunity, to infections, to neoplasms, or to drugs. Insights from the study of PRCA have helped illuminate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327320"
                        open
                      >
                        <summary class="article-expander__title">Identifying mechanisms of resistance by circulating tumour DNA in EVOLVE, a Phase II Trial of Cediranib Plus Olaparib for Ovarian Cancer at time of PARP Inhibitor Progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Purpose To evaluate the use of blood cell-free DNA (cfDNA) to identify emerging mechanisms of resistance to PARP inhibitors (PARPi) in high grade serous ovarian cancer (HGSOC). Patients and Methods We used targeted sequencing (TS) to analyse 78 longitudinal cfDNA samples collected from 30 patients with HGSOC enrolled in a phase II clinical trial evaluating cediranib (VEGF inhibitor) plus olaparib (PARPi) after progression on PARPi alone. cfDNA was collected at baseline, before treatment cycle 2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327996"
                        open
                      >
                        <summary class="article-expander__title">Pure red cell aplasia: the second hundred years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327996/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pure red cell aplasia (PRCA) is a rare hematologic syndrome, characterized by an isolated normocytic anemia with severe reticulocytopenia, and defined by absence or near absence of erythroid precursors in the bone marrow. First described in 1922, PRCA may be a primary autoimmune or clonal myeloid or lymphoid disorder, but may also be secondary to other disorders of immune dysregulation/autoimmunity, to infections, to neoplasms, or to drugs. Insights from the study of PRCA have helped illuminate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327320"
                        open
                      >
                        <summary class="article-expander__title">Identifying mechanisms of resistance by circulating tumour DNA in EVOLVE, a Phase II Trial of Cediranib Plus Olaparib for Ovarian Cancer at time of PARP Inhibitor Progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Purpose To evaluate the use of blood cell-free DNA (cfDNA) to identify emerging mechanisms of resistance to PARP inhibitors (PARPi) in high grade serous ovarian cancer (HGSOC). Patients and Methods We used targeted sequencing (TS) to analyse 78 longitudinal cfDNA samples collected from 30 patients with HGSOC enrolled in a phase II clinical trial evaluating cediranib (VEGF inhibitor) plus olaparib (PARPi) after progression on PARPi alone. cfDNA was collected at baseline, before treatment cycle 2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327996"
                        open
                      >
                        <summary class="article-expander__title">Pure red cell aplasia: the second hundred years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327996/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pure red cell aplasia (PRCA) is a rare hematologic syndrome, characterized by an isolated normocytic anemia with severe reticulocytopenia, and defined by absence or near absence of erythroid precursors in the bone marrow. First described in 1922, PRCA may be a primary autoimmune or clonal myeloid or lymphoid disorder, but may also be secondary to other disorders of immune dysregulation/autoimmunity, to infections, to neoplasms, or to drugs. Insights from the study of PRCA have helped illuminate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327320"
                        open
                      >
                        <summary class="article-expander__title">Identifying mechanisms of resistance by circulating tumour DNA in EVOLVE, a Phase II Trial of Cediranib Plus Olaparib for Ovarian Cancer at time of PARP Inhibitor Progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Purpose To evaluate the use of blood cell-free DNA (cfDNA) to identify emerging mechanisms of resistance to PARP inhibitors (PARPi) in high grade serous ovarian cancer (HGSOC). Patients and Methods We used targeted sequencing (TS) to analyse 78 longitudinal cfDNA samples collected from 30 patients with HGSOC enrolled in a phase II clinical trial evaluating cediranib (VEGF inhibitor) plus olaparib (PARPi) after progression on PARPi alone. cfDNA was collected at baseline, before treatment cycle 2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327996"
                        open
                      >
                        <summary class="article-expander__title">Pure red cell aplasia: the second hundred years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327996/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pure red cell aplasia (PRCA) is a rare hematologic syndrome, characterized by an isolated normocytic anemia with severe reticulocytopenia, and defined by absence or near absence of erythroid precursors in the bone marrow. First described in 1922, PRCA may be a primary autoimmune or clonal myeloid or lymphoid disorder, but may also be secondary to other disorders of immune dysregulation/autoimmunity, to infections, to neoplasms, or to drugs. Insights from the study of PRCA have helped illuminate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327320"
                        open
                      >
                        <summary class="article-expander__title">Identifying mechanisms of resistance by circulating tumour DNA in EVOLVE, a Phase II Trial of Cediranib Plus Olaparib for Ovarian Cancer at time of PARP Inhibitor Progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Purpose To evaluate the use of blood cell-free DNA (cfDNA) to identify emerging mechanisms of resistance to PARP inhibitors (PARPi) in high grade serous ovarian cancer (HGSOC). Patients and Methods We used targeted sequencing (TS) to analyse 78 longitudinal cfDNA samples collected from 30 patients with HGSOC enrolled in a phase II clinical trial evaluating cediranib (VEGF inhibitor) plus olaparib (PARPi) after progression on PARPi alone. cfDNA was collected at baseline, before treatment cycle 2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327996"
                        open
                      >
                        <summary class="article-expander__title">Pure red cell aplasia: the second hundred years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327996/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pure red cell aplasia (PRCA) is a rare hematologic syndrome, characterized by an isolated normocytic anemia with severe reticulocytopenia, and defined by absence or near absence of erythroid precursors in the bone marrow. First described in 1922, PRCA may be a primary autoimmune or clonal myeloid or lymphoid disorder, but may also be secondary to other disorders of immune dysregulation/autoimmunity, to infections, to neoplasms, or to drugs. Insights from the study of PRCA have helped illuminate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327320"
                        open
                      >
                        <summary class="article-expander__title">Identifying mechanisms of resistance by circulating tumour DNA in EVOLVE, a Phase II Trial of Cediranib Plus Olaparib for Ovarian Cancer at time of PARP Inhibitor Progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Purpose To evaluate the use of blood cell-free DNA (cfDNA) to identify emerging mechanisms of resistance to PARP inhibitors (PARPi) in high grade serous ovarian cancer (HGSOC). Patients and Methods We used targeted sequencing (TS) to analyse 78 longitudinal cfDNA samples collected from 30 patients with HGSOC enrolled in a phase II clinical trial evaluating cediranib (VEGF inhibitor) plus olaparib (PARPi) after progression on PARPi alone. cfDNA was collected at baseline, before treatment cycle 2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327996"
                        open
                      >
                        <summary class="article-expander__title">Pure red cell aplasia: the second hundred years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327996/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pure red cell aplasia (PRCA) is a rare hematologic syndrome, characterized by an isolated normocytic anemia with severe reticulocytopenia, and defined by absence or near absence of erythroid precursors in the bone marrow. First described in 1922, PRCA may be a primary autoimmune or clonal myeloid or lymphoid disorder, but may also be secondary to other disorders of immune dysregulation/autoimmunity, to infections, to neoplasms, or to drugs. Insights from the study of PRCA have helped illuminate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327320"
                        open
                      >
                        <summary class="article-expander__title">Identifying mechanisms of resistance by circulating tumour DNA in EVOLVE, a Phase II Trial of Cediranib Plus Olaparib for Ovarian Cancer at time of PARP Inhibitor Progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Purpose To evaluate the use of blood cell-free DNA (cfDNA) to identify emerging mechanisms of resistance to PARP inhibitors (PARPi) in high grade serous ovarian cancer (HGSOC). Patients and Methods We used targeted sequencing (TS) to analyse 78 longitudinal cfDNA samples collected from 30 patients with HGSOC enrolled in a phase II clinical trial evaluating cediranib (VEGF inhibitor) plus olaparib (PARPi) after progression on PARPi alone. cfDNA was collected at baseline, before treatment cycle 2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327996"
                        open
                      >
                        <summary class="article-expander__title">Pure red cell aplasia: the second hundred years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327996/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pure red cell aplasia (PRCA) is a rare hematologic syndrome, characterized by an isolated normocytic anemia with severe reticulocytopenia, and defined by absence or near absence of erythroid precursors in the bone marrow. First described in 1922, PRCA may be a primary autoimmune or clonal myeloid or lymphoid disorder, but may also be secondary to other disorders of immune dysregulation/autoimmunity, to infections, to neoplasms, or to drugs. Insights from the study of PRCA have helped illuminate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327320"
                        open
                      >
                        <summary class="article-expander__title">Identifying mechanisms of resistance by circulating tumour DNA in EVOLVE, a Phase II Trial of Cediranib Plus Olaparib for Ovarian Cancer at time of PARP Inhibitor Progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Purpose To evaluate the use of blood cell-free DNA (cfDNA) to identify emerging mechanisms of resistance to PARP inhibitors (PARPi) in high grade serous ovarian cancer (HGSOC). Patients and Methods We used targeted sequencing (TS) to analyse 78 longitudinal cfDNA samples collected from 30 patients with HGSOC enrolled in a phase II clinical trial evaluating cediranib (VEGF inhibitor) plus olaparib (PARPi) after progression on PARPi alone. cfDNA was collected at baseline, before treatment cycle 2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327996"
                        open
                      >
                        <summary class="article-expander__title">Pure red cell aplasia: the second hundred years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327996/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pure red cell aplasia (PRCA) is a rare hematologic syndrome, characterized by an isolated normocytic anemia with severe reticulocytopenia, and defined by absence or near absence of erythroid precursors in the bone marrow. First described in 1922, PRCA may be a primary autoimmune or clonal myeloid or lymphoid disorder, but may also be secondary to other disorders of immune dysregulation/autoimmunity, to infections, to neoplasms, or to drugs. Insights from the study of PRCA have helped illuminate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327320"
                        open
                      >
                        <summary class="article-expander__title">Identifying mechanisms of resistance by circulating tumour DNA in EVOLVE, a Phase II Trial of Cediranib Plus Olaparib for Ovarian Cancer at time of PARP Inhibitor Progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Purpose To evaluate the use of blood cell-free DNA (cfDNA) to identify emerging mechanisms of resistance to PARP inhibitors (PARPi) in high grade serous ovarian cancer (HGSOC). Patients and Methods We used targeted sequencing (TS) to analyse 78 longitudinal cfDNA samples collected from 30 patients with HGSOC enrolled in a phase II clinical trial evaluating cediranib (VEGF inhibitor) plus olaparib (PARPi) after progression on PARPi alone. cfDNA was collected at baseline, before treatment cycle 2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327996"
                        open
                      >
                        <summary class="article-expander__title">Pure red cell aplasia: the second hundred years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327996/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pure red cell aplasia (PRCA) is a rare hematologic syndrome, characterized by an isolated normocytic anemia with severe reticulocytopenia, and defined by absence or near absence of erythroid precursors in the bone marrow. First described in 1922, PRCA may be a primary autoimmune or clonal myeloid or lymphoid disorder, but may also be secondary to other disorders of immune dysregulation/autoimmunity, to infections, to neoplasms, or to drugs. Insights from the study of PRCA have helped illuminate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327320"
                        open
                      >
                        <summary class="article-expander__title">Identifying mechanisms of resistance by circulating tumour DNA in EVOLVE, a Phase II Trial of Cediranib Plus Olaparib for Ovarian Cancer at time of PARP Inhibitor Progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Purpose To evaluate the use of blood cell-free DNA (cfDNA) to identify emerging mechanisms of resistance to PARP inhibitors (PARPi) in high grade serous ovarian cancer (HGSOC). Patients and Methods We used targeted sequencing (TS) to analyse 78 longitudinal cfDNA samples collected from 30 patients with HGSOC enrolled in a phase II clinical trial evaluating cediranib (VEGF inhibitor) plus olaparib (PARPi) after progression on PARPi alone. cfDNA was collected at baseline, before treatment cycle 2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37327320"
                        open
                      >
                        <summary class="article-expander__title">Identifying mechanisms of resistance by circulating tumour DNA in EVOLVE, a Phase II Trial of Cediranib Plus Olaparib for Ovarian Cancer at time of PARP Inhibitor Progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37327320/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Purpose To evaluate the use of blood cell-free DNA (cfDNA) to identify emerging mechanisms of resistance to PARP inhibitors (PARPi) in high grade serous ovarian cancer (HGSOC). Patients and Methods We used targeted sequencing (TS) to analyse 78 longitudinal cfDNA samples collected from 30 patients with HGSOC enrolled in a phase II clinical trial evaluating cediranib (VEGF inhibitor) plus olaparib (PARPi) after progression on PARPi alone. cfDNA was collected at baseline, before treatment cycle 2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328818"
                        open
                      >
                        <summary class="article-expander__title">Personalized follow-up of circulating DNA in resected stage III/IV melanoma: PERCIMEL multicentric prospective study protocol</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: With more than 15,000 new cases /year in France and 2,000 deaths, cutaneous melanoma represents approximately 4% of incidental cancers and 1.2% of cancer related deaths. In locally advanced (stage III) or resectable metastatic (stage IV) melanomas, medical adjuvant treatment is proposed and recent advances had shown the benefit of anti-PD1/PDL1 and anti-CTLA4 immunotherapy as well as anti-BRAF and anti-MEK targeted therapy in BRAF V600 mutated tumors. However, the recurence rate at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328455"
                        open
                      >
                        <summary class="article-expander__title">Cancer genomes tolerate deleterious coding mutations through somatic copy number amplifications of wild-type regions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers evolve under the accumulation of thousands of somatic mutations and chromosomal aberrations. While most coding mutations are deleterious, almost all protein-coding genes lack detectable signals of negative selection. This raises the question of how tumors tolerate such large amounts of deleterious mutations. Using 8,690 tumor samples from The Cancer Genome Atlas, we demonstrate that copy number amplifications frequently cover haploinsufficient genes in mutation-prone regions. This could...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323185"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal heterogeneity of digestive neuroendocrine neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323185/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroendocrine neoplasms (NENs) are initially monoclonal neoplasms that progressively become polyclonal, with very different genotypic and phenotypic characteristics leading to biological differences, including the Ki-67 proliferation index, morphology, or sensitivity to treatments. Whereas inter-patient heterogeneity has been well described, intra-tumor heterogeneity has been little studied. However, NENs present a high degree of heterogeneity, both spatially within the same location or between...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323170"
                        open
                      >
                        <summary class="article-expander__title">Genomic alterations dissection revealed MUC4 mutation as a potential driver in lung adenocarcinoma local recurrence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic alteration landscape was changing during LUAD recurrence to construct a more suitable environment for the survival of tumor cells. Several potential driver mutations and targets during LUAD recurrence were identified, such as MUC4, and more investigation was needed to verify the specific functions and roles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328818"
                        open
                      >
                        <summary class="article-expander__title">Personalized follow-up of circulating DNA in resected stage III/IV melanoma: PERCIMEL multicentric prospective study protocol</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: With more than 15,000 new cases /year in France and 2,000 deaths, cutaneous melanoma represents approximately 4% of incidental cancers and 1.2% of cancer related deaths. In locally advanced (stage III) or resectable metastatic (stage IV) melanomas, medical adjuvant treatment is proposed and recent advances had shown the benefit of anti-PD1/PDL1 and anti-CTLA4 immunotherapy as well as anti-BRAF and anti-MEK targeted therapy in BRAF V600 mutated tumors. However, the recurence rate at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328455"
                        open
                      >
                        <summary class="article-expander__title">Cancer genomes tolerate deleterious coding mutations through somatic copy number amplifications of wild-type regions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers evolve under the accumulation of thousands of somatic mutations and chromosomal aberrations. While most coding mutations are deleterious, almost all protein-coding genes lack detectable signals of negative selection. This raises the question of how tumors tolerate such large amounts of deleterious mutations. Using 8,690 tumor samples from The Cancer Genome Atlas, we demonstrate that copy number amplifications frequently cover haploinsufficient genes in mutation-prone regions. This could...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323185"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal heterogeneity of digestive neuroendocrine neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323185/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroendocrine neoplasms (NENs) are initially monoclonal neoplasms that progressively become polyclonal, with very different genotypic and phenotypic characteristics leading to biological differences, including the Ki-67 proliferation index, morphology, or sensitivity to treatments. Whereas inter-patient heterogeneity has been well described, intra-tumor heterogeneity has been little studied. However, NENs present a high degree of heterogeneity, both spatially within the same location or between...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323170"
                        open
                      >
                        <summary class="article-expander__title">Genomic alterations dissection revealed MUC4 mutation as a potential driver in lung adenocarcinoma local recurrence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic alteration landscape was changing during LUAD recurrence to construct a more suitable environment for the survival of tumor cells. Several potential driver mutations and targets during LUAD recurrence were identified, such as MUC4, and more investigation was needed to verify the specific functions and roles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328818"
                        open
                      >
                        <summary class="article-expander__title">Personalized follow-up of circulating DNA in resected stage III/IV melanoma: PERCIMEL multicentric prospective study protocol</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: With more than 15,000 new cases /year in France and 2,000 deaths, cutaneous melanoma represents approximately 4% of incidental cancers and 1.2% of cancer related deaths. In locally advanced (stage III) or resectable metastatic (stage IV) melanomas, medical adjuvant treatment is proposed and recent advances had shown the benefit of anti-PD1/PDL1 and anti-CTLA4 immunotherapy as well as anti-BRAF and anti-MEK targeted therapy in BRAF V600 mutated tumors. However, the recurence rate at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328455"
                        open
                      >
                        <summary class="article-expander__title">Cancer genomes tolerate deleterious coding mutations through somatic copy number amplifications of wild-type regions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers evolve under the accumulation of thousands of somatic mutations and chromosomal aberrations. While most coding mutations are deleterious, almost all protein-coding genes lack detectable signals of negative selection. This raises the question of how tumors tolerate such large amounts of deleterious mutations. Using 8,690 tumor samples from The Cancer Genome Atlas, we demonstrate that copy number amplifications frequently cover haploinsufficient genes in mutation-prone regions. This could...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323185"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal heterogeneity of digestive neuroendocrine neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323185/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroendocrine neoplasms (NENs) are initially monoclonal neoplasms that progressively become polyclonal, with very different genotypic and phenotypic characteristics leading to biological differences, including the Ki-67 proliferation index, morphology, or sensitivity to treatments. Whereas inter-patient heterogeneity has been well described, intra-tumor heterogeneity has been little studied. However, NENs present a high degree of heterogeneity, both spatially within the same location or between...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323170"
                        open
                      >
                        <summary class="article-expander__title">Genomic alterations dissection revealed MUC4 mutation as a potential driver in lung adenocarcinoma local recurrence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic alteration landscape was changing during LUAD recurrence to construct a more suitable environment for the survival of tumor cells. Several potential driver mutations and targets during LUAD recurrence were identified, such as MUC4, and more investigation was needed to verify the specific functions and roles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328818"
                        open
                      >
                        <summary class="article-expander__title">Personalized follow-up of circulating DNA in resected stage III/IV melanoma: PERCIMEL multicentric prospective study protocol</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: With more than 15,000 new cases /year in France and 2,000 deaths, cutaneous melanoma represents approximately 4% of incidental cancers and 1.2% of cancer related deaths. In locally advanced (stage III) or resectable metastatic (stage IV) melanomas, medical adjuvant treatment is proposed and recent advances had shown the benefit of anti-PD1/PDL1 and anti-CTLA4 immunotherapy as well as anti-BRAF and anti-MEK targeted therapy in BRAF V600 mutated tumors. However, the recurence rate at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328455"
                        open
                      >
                        <summary class="article-expander__title">Cancer genomes tolerate deleterious coding mutations through somatic copy number amplifications of wild-type regions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers evolve under the accumulation of thousands of somatic mutations and chromosomal aberrations. While most coding mutations are deleterious, almost all protein-coding genes lack detectable signals of negative selection. This raises the question of how tumors tolerate such large amounts of deleterious mutations. Using 8,690 tumor samples from The Cancer Genome Atlas, we demonstrate that copy number amplifications frequently cover haploinsufficient genes in mutation-prone regions. This could...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323185"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal heterogeneity of digestive neuroendocrine neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323185/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroendocrine neoplasms (NENs) are initially monoclonal neoplasms that progressively become polyclonal, with very different genotypic and phenotypic characteristics leading to biological differences, including the Ki-67 proliferation index, morphology, or sensitivity to treatments. Whereas inter-patient heterogeneity has been well described, intra-tumor heterogeneity has been little studied. However, NENs present a high degree of heterogeneity, both spatially within the same location or between...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323170"
                        open
                      >
                        <summary class="article-expander__title">Genomic alterations dissection revealed MUC4 mutation as a potential driver in lung adenocarcinoma local recurrence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic alteration landscape was changing during LUAD recurrence to construct a more suitable environment for the survival of tumor cells. Several potential driver mutations and targets during LUAD recurrence were identified, such as MUC4, and more investigation was needed to verify the specific functions and roles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328818"
                        open
                      >
                        <summary class="article-expander__title">Personalized follow-up of circulating DNA in resected stage III/IV melanoma: PERCIMEL multicentric prospective study protocol</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: With more than 15,000 new cases /year in France and 2,000 deaths, cutaneous melanoma represents approximately 4% of incidental cancers and 1.2% of cancer related deaths. In locally advanced (stage III) or resectable metastatic (stage IV) melanomas, medical adjuvant treatment is proposed and recent advances had shown the benefit of anti-PD1/PDL1 and anti-CTLA4 immunotherapy as well as anti-BRAF and anti-MEK targeted therapy in BRAF V600 mutated tumors. However, the recurence rate at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328455"
                        open
                      >
                        <summary class="article-expander__title">Cancer genomes tolerate deleterious coding mutations through somatic copy number amplifications of wild-type regions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers evolve under the accumulation of thousands of somatic mutations and chromosomal aberrations. While most coding mutations are deleterious, almost all protein-coding genes lack detectable signals of negative selection. This raises the question of how tumors tolerate such large amounts of deleterious mutations. Using 8,690 tumor samples from The Cancer Genome Atlas, we demonstrate that copy number amplifications frequently cover haploinsufficient genes in mutation-prone regions. This could...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323185"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal heterogeneity of digestive neuroendocrine neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323185/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroendocrine neoplasms (NENs) are initially monoclonal neoplasms that progressively become polyclonal, with very different genotypic and phenotypic characteristics leading to biological differences, including the Ki-67 proliferation index, morphology, or sensitivity to treatments. Whereas inter-patient heterogeneity has been well described, intra-tumor heterogeneity has been little studied. However, NENs present a high degree of heterogeneity, both spatially within the same location or between...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323170"
                        open
                      >
                        <summary class="article-expander__title">Genomic alterations dissection revealed MUC4 mutation as a potential driver in lung adenocarcinoma local recurrence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic alteration landscape was changing during LUAD recurrence to construct a more suitable environment for the survival of tumor cells. Several potential driver mutations and targets during LUAD recurrence were identified, such as MUC4, and more investigation was needed to verify the specific functions and roles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328818"
                        open
                      >
                        <summary class="article-expander__title">Personalized follow-up of circulating DNA in resected stage III/IV melanoma: PERCIMEL multicentric prospective study protocol</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: With more than 15,000 new cases /year in France and 2,000 deaths, cutaneous melanoma represents approximately 4% of incidental cancers and 1.2% of cancer related deaths. In locally advanced (stage III) or resectable metastatic (stage IV) melanomas, medical adjuvant treatment is proposed and recent advances had shown the benefit of anti-PD1/PDL1 and anti-CTLA4 immunotherapy as well as anti-BRAF and anti-MEK targeted therapy in BRAF V600 mutated tumors. However, the recurence rate at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328455"
                        open
                      >
                        <summary class="article-expander__title">Cancer genomes tolerate deleterious coding mutations through somatic copy number amplifications of wild-type regions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers evolve under the accumulation of thousands of somatic mutations and chromosomal aberrations. While most coding mutations are deleterious, almost all protein-coding genes lack detectable signals of negative selection. This raises the question of how tumors tolerate such large amounts of deleterious mutations. Using 8,690 tumor samples from The Cancer Genome Atlas, we demonstrate that copy number amplifications frequently cover haploinsufficient genes in mutation-prone regions. This could...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323185"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal heterogeneity of digestive neuroendocrine neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323185/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroendocrine neoplasms (NENs) are initially monoclonal neoplasms that progressively become polyclonal, with very different genotypic and phenotypic characteristics leading to biological differences, including the Ki-67 proliferation index, morphology, or sensitivity to treatments. Whereas inter-patient heterogeneity has been well described, intra-tumor heterogeneity has been little studied. However, NENs present a high degree of heterogeneity, both spatially within the same location or between...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323170"
                        open
                      >
                        <summary class="article-expander__title">Genomic alterations dissection revealed MUC4 mutation as a potential driver in lung adenocarcinoma local recurrence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic alteration landscape was changing during LUAD recurrence to construct a more suitable environment for the survival of tumor cells. Several potential driver mutations and targets during LUAD recurrence were identified, such as MUC4, and more investigation was needed to verify the specific functions and roles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328818"
                        open
                      >
                        <summary class="article-expander__title">Personalized follow-up of circulating DNA in resected stage III/IV melanoma: PERCIMEL multicentric prospective study protocol</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: With more than 15,000 new cases /year in France and 2,000 deaths, cutaneous melanoma represents approximately 4% of incidental cancers and 1.2% of cancer related deaths. In locally advanced (stage III) or resectable metastatic (stage IV) melanomas, medical adjuvant treatment is proposed and recent advances had shown the benefit of anti-PD1/PDL1 and anti-CTLA4 immunotherapy as well as anti-BRAF and anti-MEK targeted therapy in BRAF V600 mutated tumors. However, the recurence rate at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328455"
                        open
                      >
                        <summary class="article-expander__title">Cancer genomes tolerate deleterious coding mutations through somatic copy number amplifications of wild-type regions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers evolve under the accumulation of thousands of somatic mutations and chromosomal aberrations. While most coding mutations are deleterious, almost all protein-coding genes lack detectable signals of negative selection. This raises the question of how tumors tolerate such large amounts of deleterious mutations. Using 8,690 tumor samples from The Cancer Genome Atlas, we demonstrate that copy number amplifications frequently cover haploinsufficient genes in mutation-prone regions. This could...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323185"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal heterogeneity of digestive neuroendocrine neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323185/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroendocrine neoplasms (NENs) are initially monoclonal neoplasms that progressively become polyclonal, with very different genotypic and phenotypic characteristics leading to biological differences, including the Ki-67 proliferation index, morphology, or sensitivity to treatments. Whereas inter-patient heterogeneity has been well described, intra-tumor heterogeneity has been little studied. However, NENs present a high degree of heterogeneity, both spatially within the same location or between...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323170"
                        open
                      >
                        <summary class="article-expander__title">Genomic alterations dissection revealed MUC4 mutation as a potential driver in lung adenocarcinoma local recurrence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic alteration landscape was changing during LUAD recurrence to construct a more suitable environment for the survival of tumor cells. Several potential driver mutations and targets during LUAD recurrence were identified, such as MUC4, and more investigation was needed to verify the specific functions and roles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328818"
                        open
                      >
                        <summary class="article-expander__title">Personalized follow-up of circulating DNA in resected stage III/IV melanoma: PERCIMEL multicentric prospective study protocol</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: With more than 15,000 new cases /year in France and 2,000 deaths, cutaneous melanoma represents approximately 4% of incidental cancers and 1.2% of cancer related deaths. In locally advanced (stage III) or resectable metastatic (stage IV) melanomas, medical adjuvant treatment is proposed and recent advances had shown the benefit of anti-PD1/PDL1 and anti-CTLA4 immunotherapy as well as anti-BRAF and anti-MEK targeted therapy in BRAF V600 mutated tumors. However, the recurence rate at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328455"
                        open
                      >
                        <summary class="article-expander__title">Cancer genomes tolerate deleterious coding mutations through somatic copy number amplifications of wild-type regions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers evolve under the accumulation of thousands of somatic mutations and chromosomal aberrations. While most coding mutations are deleterious, almost all protein-coding genes lack detectable signals of negative selection. This raises the question of how tumors tolerate such large amounts of deleterious mutations. Using 8,690 tumor samples from The Cancer Genome Atlas, we demonstrate that copy number amplifications frequently cover haploinsufficient genes in mutation-prone regions. This could...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323185"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal heterogeneity of digestive neuroendocrine neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323185/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroendocrine neoplasms (NENs) are initially monoclonal neoplasms that progressively become polyclonal, with very different genotypic and phenotypic characteristics leading to biological differences, including the Ki-67 proliferation index, morphology, or sensitivity to treatments. Whereas inter-patient heterogeneity has been well described, intra-tumor heterogeneity has been little studied. However, NENs present a high degree of heterogeneity, both spatially within the same location or between...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323170"
                        open
                      >
                        <summary class="article-expander__title">Genomic alterations dissection revealed MUC4 mutation as a potential driver in lung adenocarcinoma local recurrence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic alteration landscape was changing during LUAD recurrence to construct a more suitable environment for the survival of tumor cells. Several potential driver mutations and targets during LUAD recurrence were identified, such as MUC4, and more investigation was needed to verify the specific functions and roles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328818"
                        open
                      >
                        <summary class="article-expander__title">Personalized follow-up of circulating DNA in resected stage III/IV melanoma: PERCIMEL multicentric prospective study protocol</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: With more than 15,000 new cases /year in France and 2,000 deaths, cutaneous melanoma represents approximately 4% of incidental cancers and 1.2% of cancer related deaths. In locally advanced (stage III) or resectable metastatic (stage IV) melanomas, medical adjuvant treatment is proposed and recent advances had shown the benefit of anti-PD1/PDL1 and anti-CTLA4 immunotherapy as well as anti-BRAF and anti-MEK targeted therapy in BRAF V600 mutated tumors. However, the recurence rate at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37328455"
                        open
                      >
                        <summary class="article-expander__title">Cancer genomes tolerate deleterious coding mutations through somatic copy number amplifications of wild-type regions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37328455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers evolve under the accumulation of thousands of somatic mutations and chromosomal aberrations. While most coding mutations are deleterious, almost all protein-coding genes lack detectable signals of negative selection. This raises the question of how tumors tolerate such large amounts of deleterious mutations. Using 8,690 tumor samples from The Cancer Genome Atlas, we demonstrate that copy number amplifications frequently cover haploinsufficient genes in mutation-prone regions. This could...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323185"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal heterogeneity of digestive neuroendocrine neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323185/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroendocrine neoplasms (NENs) are initially monoclonal neoplasms that progressively become polyclonal, with very different genotypic and phenotypic characteristics leading to biological differences, including the Ki-67 proliferation index, morphology, or sensitivity to treatments. Whereas inter-patient heterogeneity has been well described, intra-tumor heterogeneity has been little studied. However, NENs present a high degree of heterogeneity, both spatially within the same location or between...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323170"
                        open
                      >
                        <summary class="article-expander__title">Genomic alterations dissection revealed MUC4 mutation as a potential driver in lung adenocarcinoma local recurrence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic alteration landscape was changing during LUAD recurrence to construct a more suitable environment for the survival of tumor cells. Several potential driver mutations and targets during LUAD recurrence were identified, such as MUC4, and more investigation was needed to verify the specific functions and roles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323185"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal heterogeneity of digestive neuroendocrine neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323185/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroendocrine neoplasms (NENs) are initially monoclonal neoplasms that progressively become polyclonal, with very different genotypic and phenotypic characteristics leading to biological differences, including the Ki-67 proliferation index, morphology, or sensitivity to treatments. Whereas inter-patient heterogeneity has been well described, intra-tumor heterogeneity has been little studied. However, NENs present a high degree of heterogeneity, both spatially within the same location or between...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37323170"
                        open
                      >
                        <summary class="article-expander__title">Genomic alterations dissection revealed MUC4 mutation as a potential driver in lung adenocarcinoma local recurrence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37323170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Genomic alteration landscape was changing during LUAD recurrence to construct a more suitable environment for the survival of tumor cells. Several potential driver mutations and targets during LUAD recurrence were identified, such as MUC4, and more investigation was needed to verify the specific functions and roles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-15 datetime="2023-06-15T10:00:00.000Z">2023-06-15</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-15 datetime="2023-06-15T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37322115"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is associated with protection from Alzheimer&#x27;s disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37322115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a premalignant expansion of mutated hematopoietic stem cells. As CHIP-associated mutations are known to alter the development and function of myeloid cells, we hypothesized that CHIP may also be associated with the risk of Alzheimer&#x27;s disease (AD), a disease in which brain-resident myeloid cells are thought to have a major role. To perform association tests between CHIP and AD dementia, we analyzed blood DNA sequencing data from 1,362...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37322115"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is associated with protection from Alzheimer&#x27;s disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37322115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a premalignant expansion of mutated hematopoietic stem cells. As CHIP-associated mutations are known to alter the development and function of myeloid cells, we hypothesized that CHIP may also be associated with the risk of Alzheimer&#x27;s disease (AD), a disease in which brain-resident myeloid cells are thought to have a major role. To perform association tests between CHIP and AD dementia, we analyzed blood DNA sequencing data from 1,362...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37322115"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is associated with protection from Alzheimer&#x27;s disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37322115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a premalignant expansion of mutated hematopoietic stem cells. As CHIP-associated mutations are known to alter the development and function of myeloid cells, we hypothesized that CHIP may also be associated with the risk of Alzheimer&#x27;s disease (AD), a disease in which brain-resident myeloid cells are thought to have a major role. To perform association tests between CHIP and AD dementia, we analyzed blood DNA sequencing data from 1,362...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37322115"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is associated with protection from Alzheimer&#x27;s disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37322115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a premalignant expansion of mutated hematopoietic stem cells. As CHIP-associated mutations are known to alter the development and function of myeloid cells, we hypothesized that CHIP may also be associated with the risk of Alzheimer&#x27;s disease (AD), a disease in which brain-resident myeloid cells are thought to have a major role. To perform association tests between CHIP and AD dementia, we analyzed blood DNA sequencing data from 1,362...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37322115"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is associated with protection from Alzheimer&#x27;s disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37322115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a premalignant expansion of mutated hematopoietic stem cells. As CHIP-associated mutations are known to alter the development and function of myeloid cells, we hypothesized that CHIP may also be associated with the risk of Alzheimer&#x27;s disease (AD), a disease in which brain-resident myeloid cells are thought to have a major role. To perform association tests between CHIP and AD dementia, we analyzed blood DNA sequencing data from 1,362...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37322115"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is associated with protection from Alzheimer&#x27;s disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37322115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a premalignant expansion of mutated hematopoietic stem cells. As CHIP-associated mutations are known to alter the development and function of myeloid cells, we hypothesized that CHIP may also be associated with the risk of Alzheimer&#x27;s disease (AD), a disease in which brain-resident myeloid cells are thought to have a major role. To perform association tests between CHIP and AD dementia, we analyzed blood DNA sequencing data from 1,362...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37322115"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is associated with protection from Alzheimer&#x27;s disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37322115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a premalignant expansion of mutated hematopoietic stem cells. As CHIP-associated mutations are known to alter the development and function of myeloid cells, we hypothesized that CHIP may also be associated with the risk of Alzheimer&#x27;s disease (AD), a disease in which brain-resident myeloid cells are thought to have a major role. To perform association tests between CHIP and AD dementia, we analyzed blood DNA sequencing data from 1,362...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37322115"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is associated with protection from Alzheimer&#x27;s disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37322115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a premalignant expansion of mutated hematopoietic stem cells. As CHIP-associated mutations are known to alter the development and function of myeloid cells, we hypothesized that CHIP may also be associated with the risk of Alzheimer&#x27;s disease (AD), a disease in which brain-resident myeloid cells are thought to have a major role. To perform association tests between CHIP and AD dementia, we analyzed blood DNA sequencing data from 1,362...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37322115"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is associated with protection from Alzheimer&#x27;s disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37322115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a premalignant expansion of mutated hematopoietic stem cells. As CHIP-associated mutations are known to alter the development and function of myeloid cells, we hypothesized that CHIP may also be associated with the risk of Alzheimer&#x27;s disease (AD), a disease in which brain-resident myeloid cells are thought to have a major role. To perform association tests between CHIP and AD dementia, we analyzed blood DNA sequencing data from 1,362...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37322115"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is associated with protection from Alzheimer&#x27;s disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37322115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a premalignant expansion of mutated hematopoietic stem cells. As CHIP-associated mutations are known to alter the development and function of myeloid cells, we hypothesized that CHIP may also be associated with the risk of Alzheimer&#x27;s disease (AD), a disease in which brain-resident myeloid cells are thought to have a major role. To perform association tests between CHIP and AD dementia, we analyzed blood DNA sequencing data from 1,362...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-14 datetime="2023-06-14T10:00:00.000Z">2023-06-14</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-14 datetime="2023-06-14T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37316755"
                        open
                      >
                        <summary class="article-expander__title">Circulating cancer giant cells with unique characteristics frequently found in patients with myelodysplastic syndromes (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37316755/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are incurable diseases characterized by dysplastic hematopoietic cells, cytopenias in the blood and an inherent tendency for transformation to secondary acute myeloid leukemia (AML). Since most therapies fail to prevent rapid clonal evolution and disease resistance, new and non-invasive predictive markers are needed to monitor patients and adapt the therapeutic strategy. By using ISET, a very sensitive approach to isolate cells larger than mature leukocytes from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37316755"
                        open
                      >
                        <summary class="article-expander__title">Circulating cancer giant cells with unique characteristics frequently found in patients with myelodysplastic syndromes (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37316755/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are incurable diseases characterized by dysplastic hematopoietic cells, cytopenias in the blood and an inherent tendency for transformation to secondary acute myeloid leukemia (AML). Since most therapies fail to prevent rapid clonal evolution and disease resistance, new and non-invasive predictive markers are needed to monitor patients and adapt the therapeutic strategy. By using ISET, a very sensitive approach to isolate cells larger than mature leukocytes from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37316755"
                        open
                      >
                        <summary class="article-expander__title">Circulating cancer giant cells with unique characteristics frequently found in patients with myelodysplastic syndromes (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37316755/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are incurable diseases characterized by dysplastic hematopoietic cells, cytopenias in the blood and an inherent tendency for transformation to secondary acute myeloid leukemia (AML). Since most therapies fail to prevent rapid clonal evolution and disease resistance, new and non-invasive predictive markers are needed to monitor patients and adapt the therapeutic strategy. By using ISET, a very sensitive approach to isolate cells larger than mature leukocytes from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37316755"
                        open
                      >
                        <summary class="article-expander__title">Circulating cancer giant cells with unique characteristics frequently found in patients with myelodysplastic syndromes (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37316755/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are incurable diseases characterized by dysplastic hematopoietic cells, cytopenias in the blood and an inherent tendency for transformation to secondary acute myeloid leukemia (AML). Since most therapies fail to prevent rapid clonal evolution and disease resistance, new and non-invasive predictive markers are needed to monitor patients and adapt the therapeutic strategy. By using ISET, a very sensitive approach to isolate cells larger than mature leukocytes from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37316755"
                        open
                      >
                        <summary class="article-expander__title">Circulating cancer giant cells with unique characteristics frequently found in patients with myelodysplastic syndromes (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37316755/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are incurable diseases characterized by dysplastic hematopoietic cells, cytopenias in the blood and an inherent tendency for transformation to secondary acute myeloid leukemia (AML). Since most therapies fail to prevent rapid clonal evolution and disease resistance, new and non-invasive predictive markers are needed to monitor patients and adapt the therapeutic strategy. By using ISET, a very sensitive approach to isolate cells larger than mature leukocytes from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37316755"
                        open
                      >
                        <summary class="article-expander__title">Circulating cancer giant cells with unique characteristics frequently found in patients with myelodysplastic syndromes (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37316755/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are incurable diseases characterized by dysplastic hematopoietic cells, cytopenias in the blood and an inherent tendency for transformation to secondary acute myeloid leukemia (AML). Since most therapies fail to prevent rapid clonal evolution and disease resistance, new and non-invasive predictive markers are needed to monitor patients and adapt the therapeutic strategy. By using ISET, a very sensitive approach to isolate cells larger than mature leukocytes from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37316755"
                        open
                      >
                        <summary class="article-expander__title">Circulating cancer giant cells with unique characteristics frequently found in patients with myelodysplastic syndromes (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37316755/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are incurable diseases characterized by dysplastic hematopoietic cells, cytopenias in the blood and an inherent tendency for transformation to secondary acute myeloid leukemia (AML). Since most therapies fail to prevent rapid clonal evolution and disease resistance, new and non-invasive predictive markers are needed to monitor patients and adapt the therapeutic strategy. By using ISET, a very sensitive approach to isolate cells larger than mature leukocytes from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37316755"
                        open
                      >
                        <summary class="article-expander__title">Circulating cancer giant cells with unique characteristics frequently found in patients with myelodysplastic syndromes (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37316755/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are incurable diseases characterized by dysplastic hematopoietic cells, cytopenias in the blood and an inherent tendency for transformation to secondary acute myeloid leukemia (AML). Since most therapies fail to prevent rapid clonal evolution and disease resistance, new and non-invasive predictive markers are needed to monitor patients and adapt the therapeutic strategy. By using ISET, a very sensitive approach to isolate cells larger than mature leukocytes from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37316755"
                        open
                      >
                        <summary class="article-expander__title">Circulating cancer giant cells with unique characteristics frequently found in patients with myelodysplastic syndromes (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37316755/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are incurable diseases characterized by dysplastic hematopoietic cells, cytopenias in the blood and an inherent tendency for transformation to secondary acute myeloid leukemia (AML). Since most therapies fail to prevent rapid clonal evolution and disease resistance, new and non-invasive predictive markers are needed to monitor patients and adapt the therapeutic strategy. By using ISET, a very sensitive approach to isolate cells larger than mature leukocytes from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37316755"
                        open
                      >
                        <summary class="article-expander__title">Circulating cancer giant cells with unique characteristics frequently found in patients with myelodysplastic syndromes (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37316755/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are incurable diseases characterized by dysplastic hematopoietic cells, cytopenias in the blood and an inherent tendency for transformation to secondary acute myeloid leukemia (AML). Since most therapies fail to prevent rapid clonal evolution and disease resistance, new and non-invasive predictive markers are needed to monitor patients and adapt the therapeutic strategy. By using ISET, a very sensitive approach to isolate cells larger than mature leukocytes from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37316755"
                        open
                      >
                        <summary class="article-expander__title">Circulating cancer giant cells with unique characteristics frequently found in patients with myelodysplastic syndromes (MDS)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37316755/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are incurable diseases characterized by dysplastic hematopoietic cells, cytopenias in the blood and an inherent tendency for transformation to secondary acute myeloid leukemia (AML). Since most therapies fail to prevent rapid clonal evolution and disease resistance, new and non-invasive predictive markers are needed to monitor patients and adapt the therapeutic strategy. By using ISET, a very sensitive approach to isolate cells larger than mature leukocytes from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-13 datetime="2023-06-13T10:00:00.000Z">2023-06-13</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-13 datetime="2023-06-13T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37311413"
                        open
                      >
                        <summary class="article-expander__title">Single cell clonotypic and transcriptional evolution of multiple myeloma precursor disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37311413/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma remains an incurable disease, and the cellular and molecular evolution from precursor conditions, including monoclonal gammopathy of undetermined significance and smoldering multiple myeloma, is incompletely understood. Here, we combine single-cell RNA and B cell receptor sequencing from fifty-two patients with myeloma precursors in comparison with myeloma and normal donors. Our comprehensive analysis reveals early genomic drivers of malignant transformation, distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37311413"
                        open
                      >
                        <summary class="article-expander__title">Single cell clonotypic and transcriptional evolution of multiple myeloma precursor disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37311413/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma remains an incurable disease, and the cellular and molecular evolution from precursor conditions, including monoclonal gammopathy of undetermined significance and smoldering multiple myeloma, is incompletely understood. Here, we combine single-cell RNA and B cell receptor sequencing from fifty-two patients with myeloma precursors in comparison with myeloma and normal donors. Our comprehensive analysis reveals early genomic drivers of malignant transformation, distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37310508"
                        open
                      >
                        <summary class="article-expander__title">Tamoxifen resistance induction results in the upregulation of ABCG2 expression and mitoxantrone resistance in MCF-7 breast cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37310508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer endocrine therapy can promote evolutionary dynamics and lead to changes in the gene expression profile of tumor cells. We aimed to assess the effect of tamoxifen (TAM)-resistance induction on ABCG2 pump mRNA, protein, and activity in ER + MCF-7 breast cancer cells. We also evaluated if the resistance to TAM leads to the cross-resistance toward mitoxantrone (MX), a well-known substrate of the ABCG2 pump. The ABCG2 mRNA and protein expression were compared in MCF-7 and its TAM-resistant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37311413"
                        open
                      >
                        <summary class="article-expander__title">Single cell clonotypic and transcriptional evolution of multiple myeloma precursor disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37311413/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma remains an incurable disease, and the cellular and molecular evolution from precursor conditions, including monoclonal gammopathy of undetermined significance and smoldering multiple myeloma, is incompletely understood. Here, we combine single-cell RNA and B cell receptor sequencing from fifty-two patients with myeloma precursors in comparison with myeloma and normal donors. Our comprehensive analysis reveals early genomic drivers of malignant transformation, distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37310508"
                        open
                      >
                        <summary class="article-expander__title">Tamoxifen resistance induction results in the upregulation of ABCG2 expression and mitoxantrone resistance in MCF-7 breast cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37310508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer endocrine therapy can promote evolutionary dynamics and lead to changes in the gene expression profile of tumor cells. We aimed to assess the effect of tamoxifen (TAM)-resistance induction on ABCG2 pump mRNA, protein, and activity in ER + MCF-7 breast cancer cells. We also evaluated if the resistance to TAM leads to the cross-resistance toward mitoxantrone (MX), a well-known substrate of the ABCG2 pump. The ABCG2 mRNA and protein expression were compared in MCF-7 and its TAM-resistant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37311413"
                        open
                      >
                        <summary class="article-expander__title">Single cell clonotypic and transcriptional evolution of multiple myeloma precursor disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37311413/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma remains an incurable disease, and the cellular and molecular evolution from precursor conditions, including monoclonal gammopathy of undetermined significance and smoldering multiple myeloma, is incompletely understood. Here, we combine single-cell RNA and B cell receptor sequencing from fifty-two patients with myeloma precursors in comparison with myeloma and normal donors. Our comprehensive analysis reveals early genomic drivers of malignant transformation, distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37310508"
                        open
                      >
                        <summary class="article-expander__title">Tamoxifen resistance induction results in the upregulation of ABCG2 expression and mitoxantrone resistance in MCF-7 breast cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37310508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer endocrine therapy can promote evolutionary dynamics and lead to changes in the gene expression profile of tumor cells. We aimed to assess the effect of tamoxifen (TAM)-resistance induction on ABCG2 pump mRNA, protein, and activity in ER + MCF-7 breast cancer cells. We also evaluated if the resistance to TAM leads to the cross-resistance toward mitoxantrone (MX), a well-known substrate of the ABCG2 pump. The ABCG2 mRNA and protein expression were compared in MCF-7 and its TAM-resistant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37311413"
                        open
                      >
                        <summary class="article-expander__title">Single cell clonotypic and transcriptional evolution of multiple myeloma precursor disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37311413/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma remains an incurable disease, and the cellular and molecular evolution from precursor conditions, including monoclonal gammopathy of undetermined significance and smoldering multiple myeloma, is incompletely understood. Here, we combine single-cell RNA and B cell receptor sequencing from fifty-two patients with myeloma precursors in comparison with myeloma and normal donors. Our comprehensive analysis reveals early genomic drivers of malignant transformation, distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37310508"
                        open
                      >
                        <summary class="article-expander__title">Tamoxifen resistance induction results in the upregulation of ABCG2 expression and mitoxantrone resistance in MCF-7 breast cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37310508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer endocrine therapy can promote evolutionary dynamics and lead to changes in the gene expression profile of tumor cells. We aimed to assess the effect of tamoxifen (TAM)-resistance induction on ABCG2 pump mRNA, protein, and activity in ER + MCF-7 breast cancer cells. We also evaluated if the resistance to TAM leads to the cross-resistance toward mitoxantrone (MX), a well-known substrate of the ABCG2 pump. The ABCG2 mRNA and protein expression were compared in MCF-7 and its TAM-resistant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37311413"
                        open
                      >
                        <summary class="article-expander__title">Single cell clonotypic and transcriptional evolution of multiple myeloma precursor disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37311413/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma remains an incurable disease, and the cellular and molecular evolution from precursor conditions, including monoclonal gammopathy of undetermined significance and smoldering multiple myeloma, is incompletely understood. Here, we combine single-cell RNA and B cell receptor sequencing from fifty-two patients with myeloma precursors in comparison with myeloma and normal donors. Our comprehensive analysis reveals early genomic drivers of malignant transformation, distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37310508"
                        open
                      >
                        <summary class="article-expander__title">Tamoxifen resistance induction results in the upregulation of ABCG2 expression and mitoxantrone resistance in MCF-7 breast cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37310508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer endocrine therapy can promote evolutionary dynamics and lead to changes in the gene expression profile of tumor cells. We aimed to assess the effect of tamoxifen (TAM)-resistance induction on ABCG2 pump mRNA, protein, and activity in ER + MCF-7 breast cancer cells. We also evaluated if the resistance to TAM leads to the cross-resistance toward mitoxantrone (MX), a well-known substrate of the ABCG2 pump. The ABCG2 mRNA and protein expression were compared in MCF-7 and its TAM-resistant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37311413"
                        open
                      >
                        <summary class="article-expander__title">Single cell clonotypic and transcriptional evolution of multiple myeloma precursor disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37311413/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma remains an incurable disease, and the cellular and molecular evolution from precursor conditions, including monoclonal gammopathy of undetermined significance and smoldering multiple myeloma, is incompletely understood. Here, we combine single-cell RNA and B cell receptor sequencing from fifty-two patients with myeloma precursors in comparison with myeloma and normal donors. Our comprehensive analysis reveals early genomic drivers of malignant transformation, distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37310508"
                        open
                      >
                        <summary class="article-expander__title">Tamoxifen resistance induction results in the upregulation of ABCG2 expression and mitoxantrone resistance in MCF-7 breast cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37310508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer endocrine therapy can promote evolutionary dynamics and lead to changes in the gene expression profile of tumor cells. We aimed to assess the effect of tamoxifen (TAM)-resistance induction on ABCG2 pump mRNA, protein, and activity in ER + MCF-7 breast cancer cells. We also evaluated if the resistance to TAM leads to the cross-resistance toward mitoxantrone (MX), a well-known substrate of the ABCG2 pump. The ABCG2 mRNA and protein expression were compared in MCF-7 and its TAM-resistant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37311413"
                        open
                      >
                        <summary class="article-expander__title">Single cell clonotypic and transcriptional evolution of multiple myeloma precursor disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37311413/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma remains an incurable disease, and the cellular and molecular evolution from precursor conditions, including monoclonal gammopathy of undetermined significance and smoldering multiple myeloma, is incompletely understood. Here, we combine single-cell RNA and B cell receptor sequencing from fifty-two patients with myeloma precursors in comparison with myeloma and normal donors. Our comprehensive analysis reveals early genomic drivers of malignant transformation, distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37310508"
                        open
                      >
                        <summary class="article-expander__title">Tamoxifen resistance induction results in the upregulation of ABCG2 expression and mitoxantrone resistance in MCF-7 breast cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37310508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer endocrine therapy can promote evolutionary dynamics and lead to changes in the gene expression profile of tumor cells. We aimed to assess the effect of tamoxifen (TAM)-resistance induction on ABCG2 pump mRNA, protein, and activity in ER + MCF-7 breast cancer cells. We also evaluated if the resistance to TAM leads to the cross-resistance toward mitoxantrone (MX), a well-known substrate of the ABCG2 pump. The ABCG2 mRNA and protein expression were compared in MCF-7 and its TAM-resistant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37311413"
                        open
                      >
                        <summary class="article-expander__title">Single cell clonotypic and transcriptional evolution of multiple myeloma precursor disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37311413/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma remains an incurable disease, and the cellular and molecular evolution from precursor conditions, including monoclonal gammopathy of undetermined significance and smoldering multiple myeloma, is incompletely understood. Here, we combine single-cell RNA and B cell receptor sequencing from fifty-two patients with myeloma precursors in comparison with myeloma and normal donors. Our comprehensive analysis reveals early genomic drivers of malignant transformation, distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37310508"
                        open
                      >
                        <summary class="article-expander__title">Tamoxifen resistance induction results in the upregulation of ABCG2 expression and mitoxantrone resistance in MCF-7 breast cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37310508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer endocrine therapy can promote evolutionary dynamics and lead to changes in the gene expression profile of tumor cells. We aimed to assess the effect of tamoxifen (TAM)-resistance induction on ABCG2 pump mRNA, protein, and activity in ER + MCF-7 breast cancer cells. We also evaluated if the resistance to TAM leads to the cross-resistance toward mitoxantrone (MX), a well-known substrate of the ABCG2 pump. The ABCG2 mRNA and protein expression were compared in MCF-7 and its TAM-resistant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37311413"
                        open
                      >
                        <summary class="article-expander__title">Single cell clonotypic and transcriptional evolution of multiple myeloma precursor disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37311413/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma remains an incurable disease, and the cellular and molecular evolution from precursor conditions, including monoclonal gammopathy of undetermined significance and smoldering multiple myeloma, is incompletely understood. Here, we combine single-cell RNA and B cell receptor sequencing from fifty-two patients with myeloma precursors in comparison with myeloma and normal donors. Our comprehensive analysis reveals early genomic drivers of malignant transformation, distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37310508"
                        open
                      >
                        <summary class="article-expander__title">Tamoxifen resistance induction results in the upregulation of ABCG2 expression and mitoxantrone resistance in MCF-7 breast cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37310508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer endocrine therapy can promote evolutionary dynamics and lead to changes in the gene expression profile of tumor cells. We aimed to assess the effect of tamoxifen (TAM)-resistance induction on ABCG2 pump mRNA, protein, and activity in ER + MCF-7 breast cancer cells. We also evaluated if the resistance to TAM leads to the cross-resistance toward mitoxantrone (MX), a well-known substrate of the ABCG2 pump. The ABCG2 mRNA and protein expression were compared in MCF-7 and its TAM-resistant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37311413"
                        open
                      >
                        <summary class="article-expander__title">Single cell clonotypic and transcriptional evolution of multiple myeloma precursor disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37311413/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma remains an incurable disease, and the cellular and molecular evolution from precursor conditions, including monoclonal gammopathy of undetermined significance and smoldering multiple myeloma, is incompletely understood. Here, we combine single-cell RNA and B cell receptor sequencing from fifty-two patients with myeloma precursors in comparison with myeloma and normal donors. Our comprehensive analysis reveals early genomic drivers of malignant transformation, distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37310508"
                        open
                      >
                        <summary class="article-expander__title">Tamoxifen resistance induction results in the upregulation of ABCG2 expression and mitoxantrone resistance in MCF-7 breast cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37310508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer endocrine therapy can promote evolutionary dynamics and lead to changes in the gene expression profile of tumor cells. We aimed to assess the effect of tamoxifen (TAM)-resistance induction on ABCG2 pump mRNA, protein, and activity in ER + MCF-7 breast cancer cells. We also evaluated if the resistance to TAM leads to the cross-resistance toward mitoxantrone (MX), a well-known substrate of the ABCG2 pump. The ABCG2 mRNA and protein expression were compared in MCF-7 and its TAM-resistant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37311413"
                        open
                      >
                        <summary class="article-expander__title">Single cell clonotypic and transcriptional evolution of multiple myeloma precursor disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37311413/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma remains an incurable disease, and the cellular and molecular evolution from precursor conditions, including monoclonal gammopathy of undetermined significance and smoldering multiple myeloma, is incompletely understood. Here, we combine single-cell RNA and B cell receptor sequencing from fifty-two patients with myeloma precursors in comparison with myeloma and normal donors. Our comprehensive analysis reveals early genomic drivers of malignant transformation, distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37310508"
                        open
                      >
                        <summary class="article-expander__title">Tamoxifen resistance induction results in the upregulation of ABCG2 expression and mitoxantrone resistance in MCF-7 breast cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37310508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer endocrine therapy can promote evolutionary dynamics and lead to changes in the gene expression profile of tumor cells. We aimed to assess the effect of tamoxifen (TAM)-resistance induction on ABCG2 pump mRNA, protein, and activity in ER + MCF-7 breast cancer cells. We also evaluated if the resistance to TAM leads to the cross-resistance toward mitoxantrone (MX), a well-known substrate of the ABCG2 pump. The ABCG2 mRNA and protein expression were compared in MCF-7 and its TAM-resistant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37310508"
                        open
                      >
                        <summary class="article-expander__title">Tamoxifen resistance induction results in the upregulation of ABCG2 expression and mitoxantrone resistance in MCF-7 breast cancer cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37310508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer endocrine therapy can promote evolutionary dynamics and lead to changes in the gene expression profile of tumor cells. We aimed to assess the effect of tamoxifen (TAM)-resistance induction on ABCG2 pump mRNA, protein, and activity in ER + MCF-7 breast cancer cells. We also evaluated if the resistance to TAM leads to the cross-resistance toward mitoxantrone (MX), a well-known substrate of the ABCG2 pump. The ABCG2 mRNA and protein expression were compared in MCF-7 and its TAM-resistant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-11 datetime="2023-06-11T10:00:00.000Z">2023-06-11</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-11 datetime="2023-06-11T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37302922"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and hierarchy in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37302922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, a group of hematopoietic disorders that includes acute myeloid leukemia (AML), myelodysplastic syndromes (MDS), and myeloproliferative neoplasms (MPNs), are caused by the accumulation of genetic and epigenetic changes in hematopoietic stem and progenitor cells (HSPCs) over time. Despite the relatively low number of genomic drivers compared with other forms of cancer, the process by which these changes shape the genomic architecture of myeloid malignancies remains elusive....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37302922"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and hierarchy in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37302922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, a group of hematopoietic disorders that includes acute myeloid leukemia (AML), myelodysplastic syndromes (MDS), and myeloproliferative neoplasms (MPNs), are caused by the accumulation of genetic and epigenetic changes in hematopoietic stem and progenitor cells (HSPCs) over time. Despite the relatively low number of genomic drivers compared with other forms of cancer, the process by which these changes shape the genomic architecture of myeloid malignancies remains elusive....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37302922"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and hierarchy in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37302922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, a group of hematopoietic disorders that includes acute myeloid leukemia (AML), myelodysplastic syndromes (MDS), and myeloproliferative neoplasms (MPNs), are caused by the accumulation of genetic and epigenetic changes in hematopoietic stem and progenitor cells (HSPCs) over time. Despite the relatively low number of genomic drivers compared with other forms of cancer, the process by which these changes shape the genomic architecture of myeloid malignancies remains elusive....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37302922"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and hierarchy in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37302922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, a group of hematopoietic disorders that includes acute myeloid leukemia (AML), myelodysplastic syndromes (MDS), and myeloproliferative neoplasms (MPNs), are caused by the accumulation of genetic and epigenetic changes in hematopoietic stem and progenitor cells (HSPCs) over time. Despite the relatively low number of genomic drivers compared with other forms of cancer, the process by which these changes shape the genomic architecture of myeloid malignancies remains elusive....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37302922"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and hierarchy in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37302922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, a group of hematopoietic disorders that includes acute myeloid leukemia (AML), myelodysplastic syndromes (MDS), and myeloproliferative neoplasms (MPNs), are caused by the accumulation of genetic and epigenetic changes in hematopoietic stem and progenitor cells (HSPCs) over time. Despite the relatively low number of genomic drivers compared with other forms of cancer, the process by which these changes shape the genomic architecture of myeloid malignancies remains elusive....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37302922"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and hierarchy in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37302922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, a group of hematopoietic disorders that includes acute myeloid leukemia (AML), myelodysplastic syndromes (MDS), and myeloproliferative neoplasms (MPNs), are caused by the accumulation of genetic and epigenetic changes in hematopoietic stem and progenitor cells (HSPCs) over time. Despite the relatively low number of genomic drivers compared with other forms of cancer, the process by which these changes shape the genomic architecture of myeloid malignancies remains elusive....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37302922"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and hierarchy in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37302922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, a group of hematopoietic disorders that includes acute myeloid leukemia (AML), myelodysplastic syndromes (MDS), and myeloproliferative neoplasms (MPNs), are caused by the accumulation of genetic and epigenetic changes in hematopoietic stem and progenitor cells (HSPCs) over time. Despite the relatively low number of genomic drivers compared with other forms of cancer, the process by which these changes shape the genomic architecture of myeloid malignancies remains elusive....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37302922"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and hierarchy in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37302922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, a group of hematopoietic disorders that includes acute myeloid leukemia (AML), myelodysplastic syndromes (MDS), and myeloproliferative neoplasms (MPNs), are caused by the accumulation of genetic and epigenetic changes in hematopoietic stem and progenitor cells (HSPCs) over time. Despite the relatively low number of genomic drivers compared with other forms of cancer, the process by which these changes shape the genomic architecture of myeloid malignancies remains elusive....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37302922"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and hierarchy in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37302922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, a group of hematopoietic disorders that includes acute myeloid leukemia (AML), myelodysplastic syndromes (MDS), and myeloproliferative neoplasms (MPNs), are caused by the accumulation of genetic and epigenetic changes in hematopoietic stem and progenitor cells (HSPCs) over time. Despite the relatively low number of genomic drivers compared with other forms of cancer, the process by which these changes shape the genomic architecture of myeloid malignancies remains elusive....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37302922"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and hierarchy in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37302922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, a group of hematopoietic disorders that includes acute myeloid leukemia (AML), myelodysplastic syndromes (MDS), and myeloproliferative neoplasms (MPNs), are caused by the accumulation of genetic and epigenetic changes in hematopoietic stem and progenitor cells (HSPCs) over time. Despite the relatively low number of genomic drivers compared with other forms of cancer, the process by which these changes shape the genomic architecture of myeloid malignancies remains elusive....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37302922"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and hierarchy in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37302922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, a group of hematopoietic disorders that includes acute myeloid leukemia (AML), myelodysplastic syndromes (MDS), and myeloproliferative neoplasms (MPNs), are caused by the accumulation of genetic and epigenetic changes in hematopoietic stem and progenitor cells (HSPCs) over time. Despite the relatively low number of genomic drivers compared with other forms of cancer, the process by which these changes shape the genomic architecture of myeloid malignancies remains elusive....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37302922"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and hierarchy in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37302922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, a group of hematopoietic disorders that includes acute myeloid leukemia (AML), myelodysplastic syndromes (MDS), and myeloproliferative neoplasms (MPNs), are caused by the accumulation of genetic and epigenetic changes in hematopoietic stem and progenitor cells (HSPCs) over time. Despite the relatively low number of genomic drivers compared with other forms of cancer, the process by which these changes shape the genomic architecture of myeloid malignancies remains elusive....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37302922"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and hierarchy in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37302922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, a group of hematopoietic disorders that includes acute myeloid leukemia (AML), myelodysplastic syndromes (MDS), and myeloproliferative neoplasms (MPNs), are caused by the accumulation of genetic and epigenetic changes in hematopoietic stem and progenitor cells (HSPCs) over time. Despite the relatively low number of genomic drivers compared with other forms of cancer, the process by which these changes shape the genomic architecture of myeloid malignancies remains elusive....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37302922"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and hierarchy in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37302922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, a group of hematopoietic disorders that includes acute myeloid leukemia (AML), myelodysplastic syndromes (MDS), and myeloproliferative neoplasms (MPNs), are caused by the accumulation of genetic and epigenetic changes in hematopoietic stem and progenitor cells (HSPCs) over time. Despite the relatively low number of genomic drivers compared with other forms of cancer, the process by which these changes shape the genomic architecture of myeloid malignancies remains elusive....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37302922"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and hierarchy in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37302922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, a group of hematopoietic disorders that includes acute myeloid leukemia (AML), myelodysplastic syndromes (MDS), and myeloproliferative neoplasms (MPNs), are caused by the accumulation of genetic and epigenetic changes in hematopoietic stem and progenitor cells (HSPCs) over time. Despite the relatively low number of genomic drivers compared with other forms of cancer, the process by which these changes shape the genomic architecture of myeloid malignancies remains elusive....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37302922"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and hierarchy in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37302922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, a group of hematopoietic disorders that includes acute myeloid leukemia (AML), myelodysplastic syndromes (MDS), and myeloproliferative neoplasms (MPNs), are caused by the accumulation of genetic and epigenetic changes in hematopoietic stem and progenitor cells (HSPCs) over time. Despite the relatively low number of genomic drivers compared with other forms of cancer, the process by which these changes shape the genomic architecture of myeloid malignancies remains elusive....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37302922"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and hierarchy in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37302922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, a group of hematopoietic disorders that includes acute myeloid leukemia (AML), myelodysplastic syndromes (MDS), and myeloproliferative neoplasms (MPNs), are caused by the accumulation of genetic and epigenetic changes in hematopoietic stem and progenitor cells (HSPCs) over time. Despite the relatively low number of genomic drivers compared with other forms of cancer, the process by which these changes shape the genomic architecture of myeloid malignancies remains elusive....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37302922"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and hierarchy in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37302922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, a group of hematopoietic disorders that includes acute myeloid leukemia (AML), myelodysplastic syndromes (MDS), and myeloproliferative neoplasms (MPNs), are caused by the accumulation of genetic and epigenetic changes in hematopoietic stem and progenitor cells (HSPCs) over time. Despite the relatively low number of genomic drivers compared with other forms of cancer, the process by which these changes shape the genomic architecture of myeloid malignancies remains elusive....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37302922"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and hierarchy in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37302922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, a group of hematopoietic disorders that includes acute myeloid leukemia (AML), myelodysplastic syndromes (MDS), and myeloproliferative neoplasms (MPNs), are caused by the accumulation of genetic and epigenetic changes in hematopoietic stem and progenitor cells (HSPCs) over time. Despite the relatively low number of genomic drivers compared with other forms of cancer, the process by which these changes shape the genomic architecture of myeloid malignancies remains elusive....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37302922"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and hierarchy in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37302922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, a group of hematopoietic disorders that includes acute myeloid leukemia (AML), myelodysplastic syndromes (MDS), and myeloproliferative neoplasms (MPNs), are caused by the accumulation of genetic and epigenetic changes in hematopoietic stem and progenitor cells (HSPCs) over time. Despite the relatively low number of genomic drivers compared with other forms of cancer, the process by which these changes shape the genomic architecture of myeloid malignancies remains elusive....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37302922"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and hierarchy in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37302922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, a group of hematopoietic disorders that includes acute myeloid leukemia (AML), myelodysplastic syndromes (MDS), and myeloproliferative neoplasms (MPNs), are caused by the accumulation of genetic and epigenetic changes in hematopoietic stem and progenitor cells (HSPCs) over time. Despite the relatively low number of genomic drivers compared with other forms of cancer, the process by which these changes shape the genomic architecture of myeloid malignancies remains elusive....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37302922"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and hierarchy in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37302922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, a group of hematopoietic disorders that includes acute myeloid leukemia (AML), myelodysplastic syndromes (MDS), and myeloproliferative neoplasms (MPNs), are caused by the accumulation of genetic and epigenetic changes in hematopoietic stem and progenitor cells (HSPCs) over time. Despite the relatively low number of genomic drivers compared with other forms of cancer, the process by which these changes shape the genomic architecture of myeloid malignancies remains elusive....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37302922"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and hierarchy in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37302922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, a group of hematopoietic disorders that includes acute myeloid leukemia (AML), myelodysplastic syndromes (MDS), and myeloproliferative neoplasms (MPNs), are caused by the accumulation of genetic and epigenetic changes in hematopoietic stem and progenitor cells (HSPCs) over time. Despite the relatively low number of genomic drivers compared with other forms of cancer, the process by which these changes shape the genomic architecture of myeloid malignancies remains elusive....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37302922"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and hierarchy in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37302922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, a group of hematopoietic disorders that includes acute myeloid leukemia (AML), myelodysplastic syndromes (MDS), and myeloproliferative neoplasms (MPNs), are caused by the accumulation of genetic and epigenetic changes in hematopoietic stem and progenitor cells (HSPCs) over time. Despite the relatively low number of genomic drivers compared with other forms of cancer, the process by which these changes shape the genomic architecture of myeloid malignancies remains elusive....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37302922"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and hierarchy in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37302922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, a group of hematopoietic disorders that includes acute myeloid leukemia (AML), myelodysplastic syndromes (MDS), and myeloproliferative neoplasms (MPNs), are caused by the accumulation of genetic and epigenetic changes in hematopoietic stem and progenitor cells (HSPCs) over time. Despite the relatively low number of genomic drivers compared with other forms of cancer, the process by which these changes shape the genomic architecture of myeloid malignancies remains elusive....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37302922"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and hierarchy in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37302922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, a group of hematopoietic disorders that includes acute myeloid leukemia (AML), myelodysplastic syndromes (MDS), and myeloproliferative neoplasms (MPNs), are caused by the accumulation of genetic and epigenetic changes in hematopoietic stem and progenitor cells (HSPCs) over time. Despite the relatively low number of genomic drivers compared with other forms of cancer, the process by which these changes shape the genomic architecture of myeloid malignancies remains elusive....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37302922"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and hierarchy in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37302922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, a group of hematopoietic disorders that includes acute myeloid leukemia (AML), myelodysplastic syndromes (MDS), and myeloproliferative neoplasms (MPNs), are caused by the accumulation of genetic and epigenetic changes in hematopoietic stem and progenitor cells (HSPCs) over time. Despite the relatively low number of genomic drivers compared with other forms of cancer, the process by which these changes shape the genomic architecture of myeloid malignancies remains elusive....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-09 datetime="2023-06-09T10:00:00.000Z">2023-06-09</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-09 datetime="2023-06-09T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292692"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential is Associated with Acute Kidney Injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292692/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown and to date no genetic mechanisms for AKI have been established. Clonal hematopoiesis of indeterminate potential (CHIP) is a recently recognized biological mechanism conferring risk of several chronic aging diseases including cardiovascular disease, pulmonary disease and liver disease. In CHIP, blood stem cells acquire mutations in myeloid cancer driver genes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292692"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential is Associated with Acute Kidney Injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292692/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown and to date no genetic mechanisms for AKI have been established. Clonal hematopoiesis of indeterminate potential (CHIP) is a recently recognized biological mechanism conferring risk of several chronic aging diseases including cardiovascular disease, pulmonary disease and liver disease. In CHIP, blood stem cells acquire mutations in myeloid cancer driver genes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292692"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential is Associated with Acute Kidney Injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292692/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown and to date no genetic mechanisms for AKI have been established. Clonal hematopoiesis of indeterminate potential (CHIP) is a recently recognized biological mechanism conferring risk of several chronic aging diseases including cardiovascular disease, pulmonary disease and liver disease. In CHIP, blood stem cells acquire mutations in myeloid cancer driver genes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292692"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential is Associated with Acute Kidney Injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292692/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown and to date no genetic mechanisms for AKI have been established. Clonal hematopoiesis of indeterminate potential (CHIP) is a recently recognized biological mechanism conferring risk of several chronic aging diseases including cardiovascular disease, pulmonary disease and liver disease. In CHIP, blood stem cells acquire mutations in myeloid cancer driver genes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292692"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential is Associated with Acute Kidney Injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292692/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown and to date no genetic mechanisms for AKI have been established. Clonal hematopoiesis of indeterminate potential (CHIP) is a recently recognized biological mechanism conferring risk of several chronic aging diseases including cardiovascular disease, pulmonary disease and liver disease. In CHIP, blood stem cells acquire mutations in myeloid cancer driver genes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292692"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential is Associated with Acute Kidney Injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292692/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown and to date no genetic mechanisms for AKI have been established. Clonal hematopoiesis of indeterminate potential (CHIP) is a recently recognized biological mechanism conferring risk of several chronic aging diseases including cardiovascular disease, pulmonary disease and liver disease. In CHIP, blood stem cells acquire mutations in myeloid cancer driver genes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292692"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential is Associated with Acute Kidney Injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292692/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown and to date no genetic mechanisms for AKI have been established. Clonal hematopoiesis of indeterminate potential (CHIP) is a recently recognized biological mechanism conferring risk of several chronic aging diseases including cardiovascular disease, pulmonary disease and liver disease. In CHIP, blood stem cells acquire mutations in myeloid cancer driver genes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292692"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential is Associated with Acute Kidney Injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292692/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown and to date no genetic mechanisms for AKI have been established. Clonal hematopoiesis of indeterminate potential (CHIP) is a recently recognized biological mechanism conferring risk of several chronic aging diseases including cardiovascular disease, pulmonary disease and liver disease. In CHIP, blood stem cells acquire mutations in myeloid cancer driver genes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292692"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential is Associated with Acute Kidney Injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292692/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown and to date no genetic mechanisms for AKI have been established. Clonal hematopoiesis of indeterminate potential (CHIP) is a recently recognized biological mechanism conferring risk of several chronic aging diseases including cardiovascular disease, pulmonary disease and liver disease. In CHIP, blood stem cells acquire mutations in myeloid cancer driver genes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292692"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential is Associated with Acute Kidney Injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292692/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown and to date no genetic mechanisms for AKI have been established. Clonal hematopoiesis of indeterminate potential (CHIP) is a recently recognized biological mechanism conferring risk of several chronic aging diseases including cardiovascular disease, pulmonary disease and liver disease. In CHIP, blood stem cells acquire mutations in myeloid cancer driver genes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292692"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential is Associated with Acute Kidney Injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292692/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown and to date no genetic mechanisms for AKI have been established. Clonal hematopoiesis of indeterminate potential (CHIP) is a recently recognized biological mechanism conferring risk of several chronic aging diseases including cardiovascular disease, pulmonary disease and liver disease. In CHIP, blood stem cells acquire mutations in myeloid cancer driver genes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292692"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential is Associated with Acute Kidney Injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292692/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown and to date no genetic mechanisms for AKI have been established. Clonal hematopoiesis of indeterminate potential (CHIP) is a recently recognized biological mechanism conferring risk of several chronic aging diseases including cardiovascular disease, pulmonary disease and liver disease. In CHIP, blood stem cells acquire mutations in myeloid cancer driver genes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292692"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential is Associated with Acute Kidney Injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292692/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown and to date no genetic mechanisms for AKI have been established. Clonal hematopoiesis of indeterminate potential (CHIP) is a recently recognized biological mechanism conferring risk of several chronic aging diseases including cardiovascular disease, pulmonary disease and liver disease. In CHIP, blood stem cells acquire mutations in myeloid cancer driver genes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292692"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential is Associated with Acute Kidney Injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292692/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown and to date no genetic mechanisms for AKI have been established. Clonal hematopoiesis of indeterminate potential (CHIP) is a recently recognized biological mechanism conferring risk of several chronic aging diseases including cardiovascular disease, pulmonary disease and liver disease. In CHIP, blood stem cells acquire mutations in myeloid cancer driver genes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292692"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential is Associated with Acute Kidney Injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292692/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown and to date no genetic mechanisms for AKI have been established. Clonal hematopoiesis of indeterminate potential (CHIP) is a recently recognized biological mechanism conferring risk of several chronic aging diseases including cardiovascular disease, pulmonary disease and liver disease. In CHIP, blood stem cells acquire mutations in myeloid cancer driver genes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292692"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential is Associated with Acute Kidney Injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292692/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown and to date no genetic mechanisms for AKI have been established. Clonal hematopoiesis of indeterminate potential (CHIP) is a recently recognized biological mechanism conferring risk of several chronic aging diseases including cardiovascular disease, pulmonary disease and liver disease. In CHIP, blood stem cells acquire mutations in myeloid cancer driver genes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292692"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential is Associated with Acute Kidney Injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292692/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown and to date no genetic mechanisms for AKI have been established. Clonal hematopoiesis of indeterminate potential (CHIP) is a recently recognized biological mechanism conferring risk of several chronic aging diseases including cardiovascular disease, pulmonary disease and liver disease. In CHIP, blood stem cells acquire mutations in myeloid cancer driver genes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292686"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomic, biochemical, and cellular mechanisms drive meningioma heterogeneity and evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity underlies cancer evolution and treatment resistance ^(1-5) , but targetable mechanisms driving intratumor heterogeneity are poorly understood. Meningiomas are the most common primary intracranial tumors and are resistant to all current medical therapies ^(6,7) . High-grade meningiomas cause significant neurological morbidity and mortality and are distinguished from low-grade meningiomas by increased intratumor heterogeneity arising from clonal evolution and divergence ⁸...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292686"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomic, biochemical, and cellular mechanisms drive meningioma heterogeneity and evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity underlies cancer evolution and treatment resistance ^(1-5) , but targetable mechanisms driving intratumor heterogeneity are poorly understood. Meningiomas are the most common primary intracranial tumors and are resistant to all current medical therapies ^(6,7) . High-grade meningiomas cause significant neurological morbidity and mortality and are distinguished from low-grade meningiomas by increased intratumor heterogeneity arising from clonal evolution and divergence ⁸...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292686"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomic, biochemical, and cellular mechanisms drive meningioma heterogeneity and evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity underlies cancer evolution and treatment resistance ^(1-5) , but targetable mechanisms driving intratumor heterogeneity are poorly understood. Meningiomas are the most common primary intracranial tumors and are resistant to all current medical therapies ^(6,7) . High-grade meningiomas cause significant neurological morbidity and mortality and are distinguished from low-grade meningiomas by increased intratumor heterogeneity arising from clonal evolution and divergence ⁸...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292686"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomic, biochemical, and cellular mechanisms drive meningioma heterogeneity and evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity underlies cancer evolution and treatment resistance ^(1-5) , but targetable mechanisms driving intratumor heterogeneity are poorly understood. Meningiomas are the most common primary intracranial tumors and are resistant to all current medical therapies ^(6,7) . High-grade meningiomas cause significant neurological morbidity and mortality and are distinguished from low-grade meningiomas by increased intratumor heterogeneity arising from clonal evolution and divergence ⁸...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292686"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomic, biochemical, and cellular mechanisms drive meningioma heterogeneity and evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity underlies cancer evolution and treatment resistance ^(1-5) , but targetable mechanisms driving intratumor heterogeneity are poorly understood. Meningiomas are the most common primary intracranial tumors and are resistant to all current medical therapies ^(6,7) . High-grade meningiomas cause significant neurological morbidity and mortality and are distinguished from low-grade meningiomas by increased intratumor heterogeneity arising from clonal evolution and divergence ⁸...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292686"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomic, biochemical, and cellular mechanisms drive meningioma heterogeneity and evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity underlies cancer evolution and treatment resistance ^(1-5) , but targetable mechanisms driving intratumor heterogeneity are poorly understood. Meningiomas are the most common primary intracranial tumors and are resistant to all current medical therapies ^(6,7) . High-grade meningiomas cause significant neurological morbidity and mortality and are distinguished from low-grade meningiomas by increased intratumor heterogeneity arising from clonal evolution and divergence ⁸...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292686"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomic, biochemical, and cellular mechanisms drive meningioma heterogeneity and evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity underlies cancer evolution and treatment resistance ^(1-5) , but targetable mechanisms driving intratumor heterogeneity are poorly understood. Meningiomas are the most common primary intracranial tumors and are resistant to all current medical therapies ^(6,7) . High-grade meningiomas cause significant neurological morbidity and mortality and are distinguished from low-grade meningiomas by increased intratumor heterogeneity arising from clonal evolution and divergence ⁸...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292686"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomic, biochemical, and cellular mechanisms drive meningioma heterogeneity and evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity underlies cancer evolution and treatment resistance ^(1-5) , but targetable mechanisms driving intratumor heterogeneity are poorly understood. Meningiomas are the most common primary intracranial tumors and are resistant to all current medical therapies ^(6,7) . High-grade meningiomas cause significant neurological morbidity and mortality and are distinguished from low-grade meningiomas by increased intratumor heterogeneity arising from clonal evolution and divergence ⁸...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292686"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomic, biochemical, and cellular mechanisms drive meningioma heterogeneity and evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity underlies cancer evolution and treatment resistance ^(1-5) , but targetable mechanisms driving intratumor heterogeneity are poorly understood. Meningiomas are the most common primary intracranial tumors and are resistant to all current medical therapies ^(6,7) . High-grade meningiomas cause significant neurological morbidity and mortality and are distinguished from low-grade meningiomas by increased intratumor heterogeneity arising from clonal evolution and divergence ⁸...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292686"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomic, biochemical, and cellular mechanisms drive meningioma heterogeneity and evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity underlies cancer evolution and treatment resistance ^(1-5) , but targetable mechanisms driving intratumor heterogeneity are poorly understood. Meningiomas are the most common primary intracranial tumors and are resistant to all current medical therapies ^(6,7) . High-grade meningiomas cause significant neurological morbidity and mortality and are distinguished from low-grade meningiomas by increased intratumor heterogeneity arising from clonal evolution and divergence ⁸...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292686"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomic, biochemical, and cellular mechanisms drive meningioma heterogeneity and evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity underlies cancer evolution and treatment resistance ^(1-5) , but targetable mechanisms driving intratumor heterogeneity are poorly understood. Meningiomas are the most common primary intracranial tumors and are resistant to all current medical therapies ^(6,7) . High-grade meningiomas cause significant neurological morbidity and mortality and are distinguished from low-grade meningiomas by increased intratumor heterogeneity arising from clonal evolution and divergence ⁸...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292686"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomic, biochemical, and cellular mechanisms drive meningioma heterogeneity and evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity underlies cancer evolution and treatment resistance ^(1-5) , but targetable mechanisms driving intratumor heterogeneity are poorly understood. Meningiomas are the most common primary intracranial tumors and are resistant to all current medical therapies ^(6,7) . High-grade meningiomas cause significant neurological morbidity and mortality and are distinguished from low-grade meningiomas by increased intratumor heterogeneity arising from clonal evolution and divergence ⁸...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292686"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomic, biochemical, and cellular mechanisms drive meningioma heterogeneity and evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity underlies cancer evolution and treatment resistance ^(1-5) , but targetable mechanisms driving intratumor heterogeneity are poorly understood. Meningiomas are the most common primary intracranial tumors and are resistant to all current medical therapies ^(6,7) . High-grade meningiomas cause significant neurological morbidity and mortality and are distinguished from low-grade meningiomas by increased intratumor heterogeneity arising from clonal evolution and divergence ⁸...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37292686"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomic, biochemical, and cellular mechanisms drive meningioma heterogeneity and evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37292686/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity underlies cancer evolution and treatment resistance ^(1-5) , but targetable mechanisms driving intratumor heterogeneity are poorly understood. Meningiomas are the most common primary intracranial tumors and are resistant to all current medical therapies ^(6,7) . High-grade meningiomas cause significant neurological morbidity and mortality and are distinguished from low-grade meningiomas by increased intratumor heterogeneity arising from clonal evolution and divergence ⁸...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-08 datetime="2023-06-08T10:00:00.000Z">2023-06-08</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-08 datetime="2023-06-08T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37289981"
                        open
                      >
                        <summary class="article-expander__title">The coming of age of liquid biopsy in neuro-oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37289981/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clinical role of liquid biopsy in oncology is growing significantly. In gliomas and other brain tumors, targeted sequencing of cell-free DNA (cfDNA) from cerebrospinal fluid (CSF) may help differential diagnosis when surgery is not recommended and be more representative of tumor heterogeneity than surgical specimens, unveiling targetable genetic alterations. Given the invasive nature of lumbar puncture to obtain CSF, the quantitative analysis of cfDNA in plasma is a lively option for patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37289981"
                        open
                      >
                        <summary class="article-expander__title">The coming of age of liquid biopsy in neuro-oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37289981/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clinical role of liquid biopsy in oncology is growing significantly. In gliomas and other brain tumors, targeted sequencing of cell-free DNA (cfDNA) from cerebrospinal fluid (CSF) may help differential diagnosis when surgery is not recommended and be more representative of tumor heterogeneity than surgical specimens, unveiling targetable genetic alterations. Given the invasive nature of lumbar puncture to obtain CSF, the quantitative analysis of cfDNA in plasma is a lively option for patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37289981"
                        open
                      >
                        <summary class="article-expander__title">The coming of age of liquid biopsy in neuro-oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37289981/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clinical role of liquid biopsy in oncology is growing significantly. In gliomas and other brain tumors, targeted sequencing of cell-free DNA (cfDNA) from cerebrospinal fluid (CSF) may help differential diagnosis when surgery is not recommended and be more representative of tumor heterogeneity than surgical specimens, unveiling targetable genetic alterations. Given the invasive nature of lumbar puncture to obtain CSF, the quantitative analysis of cfDNA in plasma is a lively option for patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37289981"
                        open
                      >
                        <summary class="article-expander__title">The coming of age of liquid biopsy in neuro-oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37289981/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clinical role of liquid biopsy in oncology is growing significantly. In gliomas and other brain tumors, targeted sequencing of cell-free DNA (cfDNA) from cerebrospinal fluid (CSF) may help differential diagnosis when surgery is not recommended and be more representative of tumor heterogeneity than surgical specimens, unveiling targetable genetic alterations. Given the invasive nature of lumbar puncture to obtain CSF, the quantitative analysis of cfDNA in plasma is a lively option for patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37289981"
                        open
                      >
                        <summary class="article-expander__title">The coming of age of liquid biopsy in neuro-oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37289981/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clinical role of liquid biopsy in oncology is growing significantly. In gliomas and other brain tumors, targeted sequencing of cell-free DNA (cfDNA) from cerebrospinal fluid (CSF) may help differential diagnosis when surgery is not recommended and be more representative of tumor heterogeneity than surgical specimens, unveiling targetable genetic alterations. Given the invasive nature of lumbar puncture to obtain CSF, the quantitative analysis of cfDNA in plasma is a lively option for patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37289981"
                        open
                      >
                        <summary class="article-expander__title">The coming of age of liquid biopsy in neuro-oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37289981/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clinical role of liquid biopsy in oncology is growing significantly. In gliomas and other brain tumors, targeted sequencing of cell-free DNA (cfDNA) from cerebrospinal fluid (CSF) may help differential diagnosis when surgery is not recommended and be more representative of tumor heterogeneity than surgical specimens, unveiling targetable genetic alterations. Given the invasive nature of lumbar puncture to obtain CSF, the quantitative analysis of cfDNA in plasma is a lively option for patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37289981"
                        open
                      >
                        <summary class="article-expander__title">The coming of age of liquid biopsy in neuro-oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37289981/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clinical role of liquid biopsy in oncology is growing significantly. In gliomas and other brain tumors, targeted sequencing of cell-free DNA (cfDNA) from cerebrospinal fluid (CSF) may help differential diagnosis when surgery is not recommended and be more representative of tumor heterogeneity than surgical specimens, unveiling targetable genetic alterations. Given the invasive nature of lumbar puncture to obtain CSF, the quantitative analysis of cfDNA in plasma is a lively option for patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37289981"
                        open
                      >
                        <summary class="article-expander__title">The coming of age of liquid biopsy in neuro-oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37289981/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clinical role of liquid biopsy in oncology is growing significantly. In gliomas and other brain tumors, targeted sequencing of cell-free DNA (cfDNA) from cerebrospinal fluid (CSF) may help differential diagnosis when surgery is not recommended and be more representative of tumor heterogeneity than surgical specimens, unveiling targetable genetic alterations. Given the invasive nature of lumbar puncture to obtain CSF, the quantitative analysis of cfDNA in plasma is a lively option for patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37289981"
                        open
                      >
                        <summary class="article-expander__title">The coming of age of liquid biopsy in neuro-oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37289981/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clinical role of liquid biopsy in oncology is growing significantly. In gliomas and other brain tumors, targeted sequencing of cell-free DNA (cfDNA) from cerebrospinal fluid (CSF) may help differential diagnosis when surgery is not recommended and be more representative of tumor heterogeneity than surgical specimens, unveiling targetable genetic alterations. Given the invasive nature of lumbar puncture to obtain CSF, the quantitative analysis of cfDNA in plasma is a lively option for patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37289981"
                        open
                      >
                        <summary class="article-expander__title">The coming of age of liquid biopsy in neuro-oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37289981/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clinical role of liquid biopsy in oncology is growing significantly. In gliomas and other brain tumors, targeted sequencing of cell-free DNA (cfDNA) from cerebrospinal fluid (CSF) may help differential diagnosis when surgery is not recommended and be more representative of tumor heterogeneity than surgical specimens, unveiling targetable genetic alterations. Given the invasive nature of lumbar puncture to obtain CSF, the quantitative analysis of cfDNA in plasma is a lively option for patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37289981"
                        open
                      >
                        <summary class="article-expander__title">The coming of age of liquid biopsy in neuro-oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37289981/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clinical role of liquid biopsy in oncology is growing significantly. In gliomas and other brain tumors, targeted sequencing of cell-free DNA (cfDNA) from cerebrospinal fluid (CSF) may help differential diagnosis when surgery is not recommended and be more representative of tumor heterogeneity than surgical specimens, unveiling targetable genetic alterations. Given the invasive nature of lumbar puncture to obtain CSF, the quantitative analysis of cfDNA in plasma is a lively option for patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37289981"
                        open
                      >
                        <summary class="article-expander__title">The coming of age of liquid biopsy in neuro-oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37289981/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clinical role of liquid biopsy in oncology is growing significantly. In gliomas and other brain tumors, targeted sequencing of cell-free DNA (cfDNA) from cerebrospinal fluid (CSF) may help differential diagnosis when surgery is not recommended and be more representative of tumor heterogeneity than surgical specimens, unveiling targetable genetic alterations. Given the invasive nature of lumbar puncture to obtain CSF, the quantitative analysis of cfDNA in plasma is a lively option for patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37289981"
                        open
                      >
                        <summary class="article-expander__title">The coming of age of liquid biopsy in neuro-oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37289981/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clinical role of liquid biopsy in oncology is growing significantly. In gliomas and other brain tumors, targeted sequencing of cell-free DNA (cfDNA) from cerebrospinal fluid (CSF) may help differential diagnosis when surgery is not recommended and be more representative of tumor heterogeneity than surgical specimens, unveiling targetable genetic alterations. Given the invasive nature of lumbar puncture to obtain CSF, the quantitative analysis of cfDNA in plasma is a lively option for patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37289981"
                        open
                      >
                        <summary class="article-expander__title">The coming of age of liquid biopsy in neuro-oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37289981/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clinical role of liquid biopsy in oncology is growing significantly. In gliomas and other brain tumors, targeted sequencing of cell-free DNA (cfDNA) from cerebrospinal fluid (CSF) may help differential diagnosis when surgery is not recommended and be more representative of tumor heterogeneity than surgical specimens, unveiling targetable genetic alterations. Given the invasive nature of lumbar puncture to obtain CSF, the quantitative analysis of cfDNA in plasma is a lively option for patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37289981"
                        open
                      >
                        <summary class="article-expander__title">The coming of age of liquid biopsy in neuro-oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37289981/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clinical role of liquid biopsy in oncology is growing significantly. In gliomas and other brain tumors, targeted sequencing of cell-free DNA (cfDNA) from cerebrospinal fluid (CSF) may help differential diagnosis when surgery is not recommended and be more representative of tumor heterogeneity than surgical specimens, unveiling targetable genetic alterations. Given the invasive nature of lumbar puncture to obtain CSF, the quantitative analysis of cfDNA in plasma is a lively option for patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37289981"
                        open
                      >
                        <summary class="article-expander__title">The coming of age of liquid biopsy in neuro-oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37289981/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clinical role of liquid biopsy in oncology is growing significantly. In gliomas and other brain tumors, targeted sequencing of cell-free DNA (cfDNA) from cerebrospinal fluid (CSF) may help differential diagnosis when surgery is not recommended and be more representative of tumor heterogeneity than surgical specimens, unveiling targetable genetic alterations. Given the invasive nature of lumbar puncture to obtain CSF, the quantitative analysis of cfDNA in plasma is a lively option for patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37289981"
                        open
                      >
                        <summary class="article-expander__title">The coming of age of liquid biopsy in neuro-oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37289981/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clinical role of liquid biopsy in oncology is growing significantly. In gliomas and other brain tumors, targeted sequencing of cell-free DNA (cfDNA) from cerebrospinal fluid (CSF) may help differential diagnosis when surgery is not recommended and be more representative of tumor heterogeneity than surgical specimens, unveiling targetable genetic alterations. Given the invasive nature of lumbar puncture to obtain CSF, the quantitative analysis of cfDNA in plasma is a lively option for patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37289981"
                        open
                      >
                        <summary class="article-expander__title">The coming of age of liquid biopsy in neuro-oncology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37289981/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The clinical role of liquid biopsy in oncology is growing significantly. In gliomas and other brain tumors, targeted sequencing of cell-free DNA (cfDNA) from cerebrospinal fluid (CSF) may help differential diagnosis when surgery is not recommended and be more representative of tumor heterogeneity than surgical specimens, unveiling targetable genetic alterations. Given the invasive nature of lumbar puncture to obtain CSF, the quantitative analysis of cfDNA in plasma is a lively option for patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-07 datetime="2023-06-07T10:00:00.000Z">2023-06-07</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-07 datetime="2023-06-07T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37286600"
                        open
                      >
                        <summary class="article-expander__title">Heritable transcriptional defects from aberrations of nuclear architecture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37286600/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transcriptional heterogeneity due to plasticity of the epigenetic state of chromatin contributes to tumour evolution, metastasis and drug resistance^(1-3). However, the mechanisms that cause this epigenetic variation are incompletely understood. Here we identify micronuclei and chromosome bridges, aberrations in the nucleus common in cancer^(4,5), as sources of heritable transcriptional suppression. Using a combination of approaches, including long-term live-cell imaging and same-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37286600"
                        open
                      >
                        <summary class="article-expander__title">Heritable transcriptional defects from aberrations of nuclear architecture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37286600/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transcriptional heterogeneity due to plasticity of the epigenetic state of chromatin contributes to tumour evolution, metastasis and drug resistance^(1-3). However, the mechanisms that cause this epigenetic variation are incompletely understood. Here we identify micronuclei and chromosome bridges, aberrations in the nucleus common in cancer^(4,5), as sources of heritable transcriptional suppression. Using a combination of approaches, including long-term live-cell imaging and same-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37286599"
                        open
                      >
                        <summary class="article-expander__title">Ultraviolet radiation shapes dendritic cell leukaemia transformation in the skin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37286599/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumours most often arise from progression of precursor clones within a single anatomical niche. In the bone marrow, clonal progenitors can undergo malignant transformation to acute leukaemia, or differentiate into immune cells that contribute to disease pathology in peripheral tissues^(1-4). Outside the marrow, these clones are potentially exposed to a variety of tissue-specific mutational processes, although the consequences of this are unclear. Here we investigate the development of blastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37285353"
                        open
                      >
                        <summary class="article-expander__title">gUMI-BEAR, a modular, unsupervised population barcoding method to track variants and evolution at high resolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37285353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage tracking provides a means to observe population makeup at the clonal level, allowing exploration of heterogeneity, evolutionary and developmental processes and individual clones&#x27; relative fitness. It has thus contributed significantly to understanding microbial evolution, organ differentiation and cancer heterogeneity, among others. Its use, however, is limited because existing methods are highly specific, expensive, labour-intensive, and, critically, do not allow the repetition...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37286600"
                        open
                      >
                        <summary class="article-expander__title">Heritable transcriptional defects from aberrations of nuclear architecture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37286600/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transcriptional heterogeneity due to plasticity of the epigenetic state of chromatin contributes to tumour evolution, metastasis and drug resistance^(1-3). However, the mechanisms that cause this epigenetic variation are incompletely understood. Here we identify micronuclei and chromosome bridges, aberrations in the nucleus common in cancer^(4,5), as sources of heritable transcriptional suppression. Using a combination of approaches, including long-term live-cell imaging and same-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37286599"
                        open
                      >
                        <summary class="article-expander__title">Ultraviolet radiation shapes dendritic cell leukaemia transformation in the skin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37286599/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumours most often arise from progression of precursor clones within a single anatomical niche. In the bone marrow, clonal progenitors can undergo malignant transformation to acute leukaemia, or differentiate into immune cells that contribute to disease pathology in peripheral tissues^(1-4). Outside the marrow, these clones are potentially exposed to a variety of tissue-specific mutational processes, although the consequences of this are unclear. Here we investigate the development of blastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37285353"
                        open
                      >
                        <summary class="article-expander__title">gUMI-BEAR, a modular, unsupervised population barcoding method to track variants and evolution at high resolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37285353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage tracking provides a means to observe population makeup at the clonal level, allowing exploration of heterogeneity, evolutionary and developmental processes and individual clones&#x27; relative fitness. It has thus contributed significantly to understanding microbial evolution, organ differentiation and cancer heterogeneity, among others. Its use, however, is limited because existing methods are highly specific, expensive, labour-intensive, and, critically, do not allow the repetition...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37285266"
                        open
                      >
                        <summary class="article-expander__title">Precise microdissection of gastric mixed adeno-neuroendocrine carcinoma dissects its genomic landscape and evolutionary clonal origins</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37285266/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gastric mixed adenoneuroendocrine carcinoma (MANEC) is a clinically aggressive and heterogeneous tumor composed of adenocarcinoma (ACA) and neuroendocrine carcinoma (NEC). The genomic properties and evolutionary clonal origins of MANEC remain unclear. We conduct whole-exome and multiregional sequencing on 101 samples from 33 patients to elucidate their evolutionary paths. We identify four significantly mutated genes, TP53, RB1, APC, and CTNNB1. MANEC resembles chromosomal instability stomach...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37286600"
                        open
                      >
                        <summary class="article-expander__title">Heritable transcriptional defects from aberrations of nuclear architecture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37286600/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transcriptional heterogeneity due to plasticity of the epigenetic state of chromatin contributes to tumour evolution, metastasis and drug resistance^(1-3). However, the mechanisms that cause this epigenetic variation are incompletely understood. Here we identify micronuclei and chromosome bridges, aberrations in the nucleus common in cancer^(4,5), as sources of heritable transcriptional suppression. Using a combination of approaches, including long-term live-cell imaging and same-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37286599"
                        open
                      >
                        <summary class="article-expander__title">Ultraviolet radiation shapes dendritic cell leukaemia transformation in the skin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37286599/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumours most often arise from progression of precursor clones within a single anatomical niche. In the bone marrow, clonal progenitors can undergo malignant transformation to acute leukaemia, or differentiate into immune cells that contribute to disease pathology in peripheral tissues^(1-4). Outside the marrow, these clones are potentially exposed to a variety of tissue-specific mutational processes, although the consequences of this are unclear. Here we investigate the development of blastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37285353"
                        open
                      >
                        <summary class="article-expander__title">gUMI-BEAR, a modular, unsupervised population barcoding method to track variants and evolution at high resolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37285353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage tracking provides a means to observe population makeup at the clonal level, allowing exploration of heterogeneity, evolutionary and developmental processes and individual clones&#x27; relative fitness. It has thus contributed significantly to understanding microbial evolution, organ differentiation and cancer heterogeneity, among others. Its use, however, is limited because existing methods are highly specific, expensive, labour-intensive, and, critically, do not allow the repetition...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37285266"
                        open
                      >
                        <summary class="article-expander__title">Precise microdissection of gastric mixed adeno-neuroendocrine carcinoma dissects its genomic landscape and evolutionary clonal origins</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37285266/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gastric mixed adenoneuroendocrine carcinoma (MANEC) is a clinically aggressive and heterogeneous tumor composed of adenocarcinoma (ACA) and neuroendocrine carcinoma (NEC). The genomic properties and evolutionary clonal origins of MANEC remain unclear. We conduct whole-exome and multiregional sequencing on 101 samples from 33 patients to elucidate their evolutionary paths. We identify four significantly mutated genes, TP53, RB1, APC, and CTNNB1. MANEC resembles chromosomal instability stomach...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37286600"
                        open
                      >
                        <summary class="article-expander__title">Heritable transcriptional defects from aberrations of nuclear architecture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37286600/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transcriptional heterogeneity due to plasticity of the epigenetic state of chromatin contributes to tumour evolution, metastasis and drug resistance^(1-3). However, the mechanisms that cause this epigenetic variation are incompletely understood. Here we identify micronuclei and chromosome bridges, aberrations in the nucleus common in cancer^(4,5), as sources of heritable transcriptional suppression. Using a combination of approaches, including long-term live-cell imaging and same-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37286599"
                        open
                      >
                        <summary class="article-expander__title">Ultraviolet radiation shapes dendritic cell leukaemia transformation in the skin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37286599/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumours most often arise from progression of precursor clones within a single anatomical niche. In the bone marrow, clonal progenitors can undergo malignant transformation to acute leukaemia, or differentiate into immune cells that contribute to disease pathology in peripheral tissues^(1-4). Outside the marrow, these clones are potentially exposed to a variety of tissue-specific mutational processes, although the consequences of this are unclear. Here we investigate the development of blastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37285353"
                        open
                      >
                        <summary class="article-expander__title">gUMI-BEAR, a modular, unsupervised population barcoding method to track variants and evolution at high resolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37285353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage tracking provides a means to observe population makeup at the clonal level, allowing exploration of heterogeneity, evolutionary and developmental processes and individual clones&#x27; relative fitness. It has thus contributed significantly to understanding microbial evolution, organ differentiation and cancer heterogeneity, among others. Its use, however, is limited because existing methods are highly specific, expensive, labour-intensive, and, critically, do not allow the repetition...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37285266"
                        open
                      >
                        <summary class="article-expander__title">Precise microdissection of gastric mixed adeno-neuroendocrine carcinoma dissects its genomic landscape and evolutionary clonal origins</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37285266/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gastric mixed adenoneuroendocrine carcinoma (MANEC) is a clinically aggressive and heterogeneous tumor composed of adenocarcinoma (ACA) and neuroendocrine carcinoma (NEC). The genomic properties and evolutionary clonal origins of MANEC remain unclear. We conduct whole-exome and multiregional sequencing on 101 samples from 33 patients to elucidate their evolutionary paths. We identify four significantly mutated genes, TP53, RB1, APC, and CTNNB1. MANEC resembles chromosomal instability stomach...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37286600"
                        open
                      >
                        <summary class="article-expander__title">Heritable transcriptional defects from aberrations of nuclear architecture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37286600/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transcriptional heterogeneity due to plasticity of the epigenetic state of chromatin contributes to tumour evolution, metastasis and drug resistance^(1-3). However, the mechanisms that cause this epigenetic variation are incompletely understood. Here we identify micronuclei and chromosome bridges, aberrations in the nucleus common in cancer^(4,5), as sources of heritable transcriptional suppression. Using a combination of approaches, including long-term live-cell imaging and same-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37286599"
                        open
                      >
                        <summary class="article-expander__title">Ultraviolet radiation shapes dendritic cell leukaemia transformation in the skin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37286599/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumours most often arise from progression of precursor clones within a single anatomical niche. In the bone marrow, clonal progenitors can undergo malignant transformation to acute leukaemia, or differentiate into immune cells that contribute to disease pathology in peripheral tissues^(1-4). Outside the marrow, these clones are potentially exposed to a variety of tissue-specific mutational processes, although the consequences of this are unclear. Here we investigate the development of blastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37285353"
                        open
                      >
                        <summary class="article-expander__title">gUMI-BEAR, a modular, unsupervised population barcoding method to track variants and evolution at high resolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37285353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage tracking provides a means to observe population makeup at the clonal level, allowing exploration of heterogeneity, evolutionary and developmental processes and individual clones&#x27; relative fitness. It has thus contributed significantly to understanding microbial evolution, organ differentiation and cancer heterogeneity, among others. Its use, however, is limited because existing methods are highly specific, expensive, labour-intensive, and, critically, do not allow the repetition...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37285266"
                        open
                      >
                        <summary class="article-expander__title">Precise microdissection of gastric mixed adeno-neuroendocrine carcinoma dissects its genomic landscape and evolutionary clonal origins</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37285266/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gastric mixed adenoneuroendocrine carcinoma (MANEC) is a clinically aggressive and heterogeneous tumor composed of adenocarcinoma (ACA) and neuroendocrine carcinoma (NEC). The genomic properties and evolutionary clonal origins of MANEC remain unclear. We conduct whole-exome and multiregional sequencing on 101 samples from 33 patients to elucidate their evolutionary paths. We identify four significantly mutated genes, TP53, RB1, APC, and CTNNB1. MANEC resembles chromosomal instability stomach...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37286600"
                        open
                      >
                        <summary class="article-expander__title">Heritable transcriptional defects from aberrations of nuclear architecture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37286600/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transcriptional heterogeneity due to plasticity of the epigenetic state of chromatin contributes to tumour evolution, metastasis and drug resistance^(1-3). However, the mechanisms that cause this epigenetic variation are incompletely understood. Here we identify micronuclei and chromosome bridges, aberrations in the nucleus common in cancer^(4,5), as sources of heritable transcriptional suppression. Using a combination of approaches, including long-term live-cell imaging and same-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37286599"
                        open
                      >
                        <summary class="article-expander__title">Ultraviolet radiation shapes dendritic cell leukaemia transformation in the skin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37286599/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumours most often arise from progression of precursor clones within a single anatomical niche. In the bone marrow, clonal progenitors can undergo malignant transformation to acute leukaemia, or differentiate into immune cells that contribute to disease pathology in peripheral tissues^(1-4). Outside the marrow, these clones are potentially exposed to a variety of tissue-specific mutational processes, although the consequences of this are unclear. Here we investigate the development of blastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37285353"
                        open
                      >
                        <summary class="article-expander__title">gUMI-BEAR, a modular, unsupervised population barcoding method to track variants and evolution at high resolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37285353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage tracking provides a means to observe population makeup at the clonal level, allowing exploration of heterogeneity, evolutionary and developmental processes and individual clones&#x27; relative fitness. It has thus contributed significantly to understanding microbial evolution, organ differentiation and cancer heterogeneity, among others. Its use, however, is limited because existing methods are highly specific, expensive, labour-intensive, and, critically, do not allow the repetition...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37285266"
                        open
                      >
                        <summary class="article-expander__title">Precise microdissection of gastric mixed adeno-neuroendocrine carcinoma dissects its genomic landscape and evolutionary clonal origins</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37285266/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gastric mixed adenoneuroendocrine carcinoma (MANEC) is a clinically aggressive and heterogeneous tumor composed of adenocarcinoma (ACA) and neuroendocrine carcinoma (NEC). The genomic properties and evolutionary clonal origins of MANEC remain unclear. We conduct whole-exome and multiregional sequencing on 101 samples from 33 patients to elucidate their evolutionary paths. We identify four significantly mutated genes, TP53, RB1, APC, and CTNNB1. MANEC resembles chromosomal instability stomach...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37286600"
                        open
                      >
                        <summary class="article-expander__title">Heritable transcriptional defects from aberrations of nuclear architecture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37286600/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transcriptional heterogeneity due to plasticity of the epigenetic state of chromatin contributes to tumour evolution, metastasis and drug resistance^(1-3). However, the mechanisms that cause this epigenetic variation are incompletely understood. Here we identify micronuclei and chromosome bridges, aberrations in the nucleus common in cancer^(4,5), as sources of heritable transcriptional suppression. Using a combination of approaches, including long-term live-cell imaging and same-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37286599"
                        open
                      >
                        <summary class="article-expander__title">Ultraviolet radiation shapes dendritic cell leukaemia transformation in the skin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37286599/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumours most often arise from progression of precursor clones within a single anatomical niche. In the bone marrow, clonal progenitors can undergo malignant transformation to acute leukaemia, or differentiate into immune cells that contribute to disease pathology in peripheral tissues^(1-4). Outside the marrow, these clones are potentially exposed to a variety of tissue-specific mutational processes, although the consequences of this are unclear. Here we investigate the development of blastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37285353"
                        open
                      >
                        <summary class="article-expander__title">gUMI-BEAR, a modular, unsupervised population barcoding method to track variants and evolution at high resolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37285353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage tracking provides a means to observe population makeup at the clonal level, allowing exploration of heterogeneity, evolutionary and developmental processes and individual clones&#x27; relative fitness. It has thus contributed significantly to understanding microbial evolution, organ differentiation and cancer heterogeneity, among others. Its use, however, is limited because existing methods are highly specific, expensive, labour-intensive, and, critically, do not allow the repetition...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37285266"
                        open
                      >
                        <summary class="article-expander__title">Precise microdissection of gastric mixed adeno-neuroendocrine carcinoma dissects its genomic landscape and evolutionary clonal origins</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37285266/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gastric mixed adenoneuroendocrine carcinoma (MANEC) is a clinically aggressive and heterogeneous tumor composed of adenocarcinoma (ACA) and neuroendocrine carcinoma (NEC). The genomic properties and evolutionary clonal origins of MANEC remain unclear. We conduct whole-exome and multiregional sequencing on 101 samples from 33 patients to elucidate their evolutionary paths. We identify four significantly mutated genes, TP53, RB1, APC, and CTNNB1. MANEC resembles chromosomal instability stomach...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37286600"
                        open
                      >
                        <summary class="article-expander__title">Heritable transcriptional defects from aberrations of nuclear architecture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37286600/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transcriptional heterogeneity due to plasticity of the epigenetic state of chromatin contributes to tumour evolution, metastasis and drug resistance^(1-3). However, the mechanisms that cause this epigenetic variation are incompletely understood. Here we identify micronuclei and chromosome bridges, aberrations in the nucleus common in cancer^(4,5), as sources of heritable transcriptional suppression. Using a combination of approaches, including long-term live-cell imaging and same-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37286599"
                        open
                      >
                        <summary class="article-expander__title">Ultraviolet radiation shapes dendritic cell leukaemia transformation in the skin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37286599/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumours most often arise from progression of precursor clones within a single anatomical niche. In the bone marrow, clonal progenitors can undergo malignant transformation to acute leukaemia, or differentiate into immune cells that contribute to disease pathology in peripheral tissues^(1-4). Outside the marrow, these clones are potentially exposed to a variety of tissue-specific mutational processes, although the consequences of this are unclear. Here we investigate the development of blastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37285353"
                        open
                      >
                        <summary class="article-expander__title">gUMI-BEAR, a modular, unsupervised population barcoding method to track variants and evolution at high resolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37285353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage tracking provides a means to observe population makeup at the clonal level, allowing exploration of heterogeneity, evolutionary and developmental processes and individual clones&#x27; relative fitness. It has thus contributed significantly to understanding microbial evolution, organ differentiation and cancer heterogeneity, among others. Its use, however, is limited because existing methods are highly specific, expensive, labour-intensive, and, critically, do not allow the repetition...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37285266"
                        open
                      >
                        <summary class="article-expander__title">Precise microdissection of gastric mixed adeno-neuroendocrine carcinoma dissects its genomic landscape and evolutionary clonal origins</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37285266/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gastric mixed adenoneuroendocrine carcinoma (MANEC) is a clinically aggressive and heterogeneous tumor composed of adenocarcinoma (ACA) and neuroendocrine carcinoma (NEC). The genomic properties and evolutionary clonal origins of MANEC remain unclear. We conduct whole-exome and multiregional sequencing on 101 samples from 33 patients to elucidate their evolutionary paths. We identify four significantly mutated genes, TP53, RB1, APC, and CTNNB1. MANEC resembles chromosomal instability stomach...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37286600"
                        open
                      >
                        <summary class="article-expander__title">Heritable transcriptional defects from aberrations of nuclear architecture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37286600/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transcriptional heterogeneity due to plasticity of the epigenetic state of chromatin contributes to tumour evolution, metastasis and drug resistance^(1-3). However, the mechanisms that cause this epigenetic variation are incompletely understood. Here we identify micronuclei and chromosome bridges, aberrations in the nucleus common in cancer^(4,5), as sources of heritable transcriptional suppression. Using a combination of approaches, including long-term live-cell imaging and same-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37286599"
                        open
                      >
                        <summary class="article-expander__title">Ultraviolet radiation shapes dendritic cell leukaemia transformation in the skin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37286599/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumours most often arise from progression of precursor clones within a single anatomical niche. In the bone marrow, clonal progenitors can undergo malignant transformation to acute leukaemia, or differentiate into immune cells that contribute to disease pathology in peripheral tissues^(1-4). Outside the marrow, these clones are potentially exposed to a variety of tissue-specific mutational processes, although the consequences of this are unclear. Here we investigate the development of blastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37285353"
                        open
                      >
                        <summary class="article-expander__title">gUMI-BEAR, a modular, unsupervised population barcoding method to track variants and evolution at high resolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37285353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage tracking provides a means to observe population makeup at the clonal level, allowing exploration of heterogeneity, evolutionary and developmental processes and individual clones&#x27; relative fitness. It has thus contributed significantly to understanding microbial evolution, organ differentiation and cancer heterogeneity, among others. Its use, however, is limited because existing methods are highly specific, expensive, labour-intensive, and, critically, do not allow the repetition...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37285266"
                        open
                      >
                        <summary class="article-expander__title">Precise microdissection of gastric mixed adeno-neuroendocrine carcinoma dissects its genomic landscape and evolutionary clonal origins</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37285266/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gastric mixed adenoneuroendocrine carcinoma (MANEC) is a clinically aggressive and heterogeneous tumor composed of adenocarcinoma (ACA) and neuroendocrine carcinoma (NEC). The genomic properties and evolutionary clonal origins of MANEC remain unclear. We conduct whole-exome and multiregional sequencing on 101 samples from 33 patients to elucidate their evolutionary paths. We identify four significantly mutated genes, TP53, RB1, APC, and CTNNB1. MANEC resembles chromosomal instability stomach...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37286600"
                        open
                      >
                        <summary class="article-expander__title">Heritable transcriptional defects from aberrations of nuclear architecture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37286600/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transcriptional heterogeneity due to plasticity of the epigenetic state of chromatin contributes to tumour evolution, metastasis and drug resistance^(1-3). However, the mechanisms that cause this epigenetic variation are incompletely understood. Here we identify micronuclei and chromosome bridges, aberrations in the nucleus common in cancer^(4,5), as sources of heritable transcriptional suppression. Using a combination of approaches, including long-term live-cell imaging and same-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37286599"
                        open
                      >
                        <summary class="article-expander__title">Ultraviolet radiation shapes dendritic cell leukaemia transformation in the skin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37286599/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumours most often arise from progression of precursor clones within a single anatomical niche. In the bone marrow, clonal progenitors can undergo malignant transformation to acute leukaemia, or differentiate into immune cells that contribute to disease pathology in peripheral tissues^(1-4). Outside the marrow, these clones are potentially exposed to a variety of tissue-specific mutational processes, although the consequences of this are unclear. Here we investigate the development of blastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37285353"
                        open
                      >
                        <summary class="article-expander__title">gUMI-BEAR, a modular, unsupervised population barcoding method to track variants and evolution at high resolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37285353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage tracking provides a means to observe population makeup at the clonal level, allowing exploration of heterogeneity, evolutionary and developmental processes and individual clones&#x27; relative fitness. It has thus contributed significantly to understanding microbial evolution, organ differentiation and cancer heterogeneity, among others. Its use, however, is limited because existing methods are highly specific, expensive, labour-intensive, and, critically, do not allow the repetition...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37285266"
                        open
                      >
                        <summary class="article-expander__title">Precise microdissection of gastric mixed adeno-neuroendocrine carcinoma dissects its genomic landscape and evolutionary clonal origins</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37285266/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gastric mixed adenoneuroendocrine carcinoma (MANEC) is a clinically aggressive and heterogeneous tumor composed of adenocarcinoma (ACA) and neuroendocrine carcinoma (NEC). The genomic properties and evolutionary clonal origins of MANEC remain unclear. We conduct whole-exome and multiregional sequencing on 101 samples from 33 patients to elucidate their evolutionary paths. We identify four significantly mutated genes, TP53, RB1, APC, and CTNNB1. MANEC resembles chromosomal instability stomach...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37286600"
                        open
                      >
                        <summary class="article-expander__title">Heritable transcriptional defects from aberrations of nuclear architecture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37286600/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transcriptional heterogeneity due to plasticity of the epigenetic state of chromatin contributes to tumour evolution, metastasis and drug resistance^(1-3). However, the mechanisms that cause this epigenetic variation are incompletely understood. Here we identify micronuclei and chromosome bridges, aberrations in the nucleus common in cancer^(4,5), as sources of heritable transcriptional suppression. Using a combination of approaches, including long-term live-cell imaging and same-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37286599"
                        open
                      >
                        <summary class="article-expander__title">Ultraviolet radiation shapes dendritic cell leukaemia transformation in the skin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37286599/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumours most often arise from progression of precursor clones within a single anatomical niche. In the bone marrow, clonal progenitors can undergo malignant transformation to acute leukaemia, or differentiate into immune cells that contribute to disease pathology in peripheral tissues^(1-4). Outside the marrow, these clones are potentially exposed to a variety of tissue-specific mutational processes, although the consequences of this are unclear. Here we investigate the development of blastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37285353"
                        open
                      >
                        <summary class="article-expander__title">gUMI-BEAR, a modular, unsupervised population barcoding method to track variants and evolution at high resolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37285353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage tracking provides a means to observe population makeup at the clonal level, allowing exploration of heterogeneity, evolutionary and developmental processes and individual clones&#x27; relative fitness. It has thus contributed significantly to understanding microbial evolution, organ differentiation and cancer heterogeneity, among others. Its use, however, is limited because existing methods are highly specific, expensive, labour-intensive, and, critically, do not allow the repetition...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37285266"
                        open
                      >
                        <summary class="article-expander__title">Precise microdissection of gastric mixed adeno-neuroendocrine carcinoma dissects its genomic landscape and evolutionary clonal origins</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37285266/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gastric mixed adenoneuroendocrine carcinoma (MANEC) is a clinically aggressive and heterogeneous tumor composed of adenocarcinoma (ACA) and neuroendocrine carcinoma (NEC). The genomic properties and evolutionary clonal origins of MANEC remain unclear. We conduct whole-exome and multiregional sequencing on 101 samples from 33 patients to elucidate their evolutionary paths. We identify four significantly mutated genes, TP53, RB1, APC, and CTNNB1. MANEC resembles chromosomal instability stomach...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37286600"
                        open
                      >
                        <summary class="article-expander__title">Heritable transcriptional defects from aberrations of nuclear architecture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37286600/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transcriptional heterogeneity due to plasticity of the epigenetic state of chromatin contributes to tumour evolution, metastasis and drug resistance^(1-3). However, the mechanisms that cause this epigenetic variation are incompletely understood. Here we identify micronuclei and chromosome bridges, aberrations in the nucleus common in cancer^(4,5), as sources of heritable transcriptional suppression. Using a combination of approaches, including long-term live-cell imaging and same-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37286599"
                        open
                      >
                        <summary class="article-expander__title">Ultraviolet radiation shapes dendritic cell leukaemia transformation in the skin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37286599/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumours most often arise from progression of precursor clones within a single anatomical niche. In the bone marrow, clonal progenitors can undergo malignant transformation to acute leukaemia, or differentiate into immune cells that contribute to disease pathology in peripheral tissues^(1-4). Outside the marrow, these clones are potentially exposed to a variety of tissue-specific mutational processes, although the consequences of this are unclear. Here we investigate the development of blastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37285353"
                        open
                      >
                        <summary class="article-expander__title">gUMI-BEAR, a modular, unsupervised population barcoding method to track variants and evolution at high resolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37285353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage tracking provides a means to observe population makeup at the clonal level, allowing exploration of heterogeneity, evolutionary and developmental processes and individual clones&#x27; relative fitness. It has thus contributed significantly to understanding microbial evolution, organ differentiation and cancer heterogeneity, among others. Its use, however, is limited because existing methods are highly specific, expensive, labour-intensive, and, critically, do not allow the repetition...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37285266"
                        open
                      >
                        <summary class="article-expander__title">Precise microdissection of gastric mixed adeno-neuroendocrine carcinoma dissects its genomic landscape and evolutionary clonal origins</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37285266/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gastric mixed adenoneuroendocrine carcinoma (MANEC) is a clinically aggressive and heterogeneous tumor composed of adenocarcinoma (ACA) and neuroendocrine carcinoma (NEC). The genomic properties and evolutionary clonal origins of MANEC remain unclear. We conduct whole-exome and multiregional sequencing on 101 samples from 33 patients to elucidate their evolutionary paths. We identify four significantly mutated genes, TP53, RB1, APC, and CTNNB1. MANEC resembles chromosomal instability stomach...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37286600"
                        open
                      >
                        <summary class="article-expander__title">Heritable transcriptional defects from aberrations of nuclear architecture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37286600/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transcriptional heterogeneity due to plasticity of the epigenetic state of chromatin contributes to tumour evolution, metastasis and drug resistance^(1-3). However, the mechanisms that cause this epigenetic variation are incompletely understood. Here we identify micronuclei and chromosome bridges, aberrations in the nucleus common in cancer^(4,5), as sources of heritable transcriptional suppression. Using a combination of approaches, including long-term live-cell imaging and same-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37286599"
                        open
                      >
                        <summary class="article-expander__title">Ultraviolet radiation shapes dendritic cell leukaemia transformation in the skin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37286599/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumours most often arise from progression of precursor clones within a single anatomical niche. In the bone marrow, clonal progenitors can undergo malignant transformation to acute leukaemia, or differentiate into immune cells that contribute to disease pathology in peripheral tissues^(1-4). Outside the marrow, these clones are potentially exposed to a variety of tissue-specific mutational processes, although the consequences of this are unclear. Here we investigate the development of blastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37285266"
                        open
                      >
                        <summary class="article-expander__title">Precise microdissection of gastric mixed adeno-neuroendocrine carcinoma dissects its genomic landscape and evolutionary clonal origins</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37285266/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gastric mixed adenoneuroendocrine carcinoma (MANEC) is a clinically aggressive and heterogeneous tumor composed of adenocarcinoma (ACA) and neuroendocrine carcinoma (NEC). The genomic properties and evolutionary clonal origins of MANEC remain unclear. We conduct whole-exome and multiregional sequencing on 101 samples from 33 patients to elucidate their evolutionary paths. We identify four significantly mutated genes, TP53, RB1, APC, and CTNNB1. MANEC resembles chromosomal instability stomach...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37285266"
                        open
                      >
                        <summary class="article-expander__title">Precise microdissection of gastric mixed adeno-neuroendocrine carcinoma dissects its genomic landscape and evolutionary clonal origins</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37285266/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gastric mixed adenoneuroendocrine carcinoma (MANEC) is a clinically aggressive and heterogeneous tumor composed of adenocarcinoma (ACA) and neuroendocrine carcinoma (NEC). The genomic properties and evolutionary clonal origins of MANEC remain unclear. We conduct whole-exome and multiregional sequencing on 101 samples from 33 patients to elucidate their evolutionary paths. We identify four significantly mutated genes, TP53, RB1, APC, and CTNNB1. MANEC resembles chromosomal instability stomach...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-05 datetime="2023-06-05T10:00:00.000Z">2023-06-05</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-05 datetime="2023-06-05T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37273168"
                        open
                      >
                        <summary class="article-expander__title">Molecular evolution in different subtypes of multifocal hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37273168/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study comprehensively characterized the varied tumor clonal evolutionary history underlying different subtypes of MF-HCC and provided important implications for optimizing personalized clinical management for MF-HCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37273168"
                        open
                      >
                        <summary class="article-expander__title">Molecular evolution in different subtypes of multifocal hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37273168/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study comprehensively characterized the varied tumor clonal evolutionary history underlying different subtypes of MF-HCC and provided important implications for optimizing personalized clinical management for MF-HCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37273168"
                        open
                      >
                        <summary class="article-expander__title">Molecular evolution in different subtypes of multifocal hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37273168/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study comprehensively characterized the varied tumor clonal evolutionary history underlying different subtypes of MF-HCC and provided important implications for optimizing personalized clinical management for MF-HCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37273168"
                        open
                      >
                        <summary class="article-expander__title">Molecular evolution in different subtypes of multifocal hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37273168/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study comprehensively characterized the varied tumor clonal evolutionary history underlying different subtypes of MF-HCC and provided important implications for optimizing personalized clinical management for MF-HCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37273168"
                        open
                      >
                        <summary class="article-expander__title">Molecular evolution in different subtypes of multifocal hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37273168/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study comprehensively characterized the varied tumor clonal evolutionary history underlying different subtypes of MF-HCC and provided important implications for optimizing personalized clinical management for MF-HCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37273168"
                        open
                      >
                        <summary class="article-expander__title">Molecular evolution in different subtypes of multifocal hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37273168/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study comprehensively characterized the varied tumor clonal evolutionary history underlying different subtypes of MF-HCC and provided important implications for optimizing personalized clinical management for MF-HCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37273168"
                        open
                      >
                        <summary class="article-expander__title">Molecular evolution in different subtypes of multifocal hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37273168/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study comprehensively characterized the varied tumor clonal evolutionary history underlying different subtypes of MF-HCC and provided important implications for optimizing personalized clinical management for MF-HCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37273168"
                        open
                      >
                        <summary class="article-expander__title">Molecular evolution in different subtypes of multifocal hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37273168/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study comprehensively characterized the varied tumor clonal evolutionary history underlying different subtypes of MF-HCC and provided important implications for optimizing personalized clinical management for MF-HCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37273168"
                        open
                      >
                        <summary class="article-expander__title">Molecular evolution in different subtypes of multifocal hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37273168/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study comprehensively characterized the varied tumor clonal evolutionary history underlying different subtypes of MF-HCC and provided important implications for optimizing personalized clinical management for MF-HCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37273168"
                        open
                      >
                        <summary class="article-expander__title">Molecular evolution in different subtypes of multifocal hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37273168/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study comprehensively characterized the varied tumor clonal evolutionary history underlying different subtypes of MF-HCC and provided important implications for optimizing personalized clinical management for MF-HCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37273168"
                        open
                      >
                        <summary class="article-expander__title">Molecular evolution in different subtypes of multifocal hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37273168/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study comprehensively characterized the varied tumor clonal evolutionary history underlying different subtypes of MF-HCC and provided important implications for optimizing personalized clinical management for MF-HCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37273168"
                        open
                      >
                        <summary class="article-expander__title">Molecular evolution in different subtypes of multifocal hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37273168/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study comprehensively characterized the varied tumor clonal evolutionary history underlying different subtypes of MF-HCC and provided important implications for optimizing personalized clinical management for MF-HCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37273168"
                        open
                      >
                        <summary class="article-expander__title">Molecular evolution in different subtypes of multifocal hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37273168/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study comprehensively characterized the varied tumor clonal evolutionary history underlying different subtypes of MF-HCC and provided important implications for optimizing personalized clinical management for MF-HCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37273168"
                        open
                      >
                        <summary class="article-expander__title">Molecular evolution in different subtypes of multifocal hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37273168/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study comprehensively characterized the varied tumor clonal evolutionary history underlying different subtypes of MF-HCC and provided important implications for optimizing personalized clinical management for MF-HCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-04 datetime="2023-06-04T10:00:00.000Z">2023-06-04</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-04 datetime="2023-06-04T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37271527"
                        open
                      >
                        <summary class="article-expander__title">Clinical significance of clonal evolution in chronic myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37271527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic myeloid leukemia (CML) is a hematological malignancy characterized by the Philadelphia (Ph) chromosome, which is formed by a t (9;22)(q34;q11) translocation. The aberrant activation of the ABL1 tyrosine kinase is caused by the BCR::ABL1 fusion gene on the Ph chromosome, leading to significant leukemic cell proliferation. CML is typically diagnosed in the chronic phase with few clinical symptoms and progresses to a blast crisis within years. CML acquires additional genetic abnormalities...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37271527"
                        open
                      >
                        <summary class="article-expander__title">Clinical significance of clonal evolution in chronic myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37271527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic myeloid leukemia (CML) is a hematological malignancy characterized by the Philadelphia (Ph) chromosome, which is formed by a t (9;22)(q34;q11) translocation. The aberrant activation of the ABL1 tyrosine kinase is caused by the BCR::ABL1 fusion gene on the Ph chromosome, leading to significant leukemic cell proliferation. CML is typically diagnosed in the chronic phase with few clinical symptoms and progresses to a blast crisis within years. CML acquires additional genetic abnormalities...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37271527"
                        open
                      >
                        <summary class="article-expander__title">Clinical significance of clonal evolution in chronic myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37271527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic myeloid leukemia (CML) is a hematological malignancy characterized by the Philadelphia (Ph) chromosome, which is formed by a t (9;22)(q34;q11) translocation. The aberrant activation of the ABL1 tyrosine kinase is caused by the BCR::ABL1 fusion gene on the Ph chromosome, leading to significant leukemic cell proliferation. CML is typically diagnosed in the chronic phase with few clinical symptoms and progresses to a blast crisis within years. CML acquires additional genetic abnormalities...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37271527"
                        open
                      >
                        <summary class="article-expander__title">Clinical significance of clonal evolution in chronic myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37271527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic myeloid leukemia (CML) is a hematological malignancy characterized by the Philadelphia (Ph) chromosome, which is formed by a t (9;22)(q34;q11) translocation. The aberrant activation of the ABL1 tyrosine kinase is caused by the BCR::ABL1 fusion gene on the Ph chromosome, leading to significant leukemic cell proliferation. CML is typically diagnosed in the chronic phase with few clinical symptoms and progresses to a blast crisis within years. CML acquires additional genetic abnormalities...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37271527"
                        open
                      >
                        <summary class="article-expander__title">Clinical significance of clonal evolution in chronic myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37271527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic myeloid leukemia (CML) is a hematological malignancy characterized by the Philadelphia (Ph) chromosome, which is formed by a t (9;22)(q34;q11) translocation. The aberrant activation of the ABL1 tyrosine kinase is caused by the BCR::ABL1 fusion gene on the Ph chromosome, leading to significant leukemic cell proliferation. CML is typically diagnosed in the chronic phase with few clinical symptoms and progresses to a blast crisis within years. CML acquires additional genetic abnormalities...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37271527"
                        open
                      >
                        <summary class="article-expander__title">Clinical significance of clonal evolution in chronic myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37271527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic myeloid leukemia (CML) is a hematological malignancy characterized by the Philadelphia (Ph) chromosome, which is formed by a t (9;22)(q34;q11) translocation. The aberrant activation of the ABL1 tyrosine kinase is caused by the BCR::ABL1 fusion gene on the Ph chromosome, leading to significant leukemic cell proliferation. CML is typically diagnosed in the chronic phase with few clinical symptoms and progresses to a blast crisis within years. CML acquires additional genetic abnormalities...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37271527"
                        open
                      >
                        <summary class="article-expander__title">Clinical significance of clonal evolution in chronic myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37271527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic myeloid leukemia (CML) is a hematological malignancy characterized by the Philadelphia (Ph) chromosome, which is formed by a t (9;22)(q34;q11) translocation. The aberrant activation of the ABL1 tyrosine kinase is caused by the BCR::ABL1 fusion gene on the Ph chromosome, leading to significant leukemic cell proliferation. CML is typically diagnosed in the chronic phase with few clinical symptoms and progresses to a blast crisis within years. CML acquires additional genetic abnormalities...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37271527"
                        open
                      >
                        <summary class="article-expander__title">Clinical significance of clonal evolution in chronic myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37271527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic myeloid leukemia (CML) is a hematological malignancy characterized by the Philadelphia (Ph) chromosome, which is formed by a t (9;22)(q34;q11) translocation. The aberrant activation of the ABL1 tyrosine kinase is caused by the BCR::ABL1 fusion gene on the Ph chromosome, leading to significant leukemic cell proliferation. CML is typically diagnosed in the chronic phase with few clinical symptoms and progresses to a blast crisis within years. CML acquires additional genetic abnormalities...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37271527"
                        open
                      >
                        <summary class="article-expander__title">Clinical significance of clonal evolution in chronic myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37271527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic myeloid leukemia (CML) is a hematological malignancy characterized by the Philadelphia (Ph) chromosome, which is formed by a t (9;22)(q34;q11) translocation. The aberrant activation of the ABL1 tyrosine kinase is caused by the BCR::ABL1 fusion gene on the Ph chromosome, leading to significant leukemic cell proliferation. CML is typically diagnosed in the chronic phase with few clinical symptoms and progresses to a blast crisis within years. CML acquires additional genetic abnormalities...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37271527"
                        open
                      >
                        <summary class="article-expander__title">Clinical significance of clonal evolution in chronic myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37271527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic myeloid leukemia (CML) is a hematological malignancy characterized by the Philadelphia (Ph) chromosome, which is formed by a t (9;22)(q34;q11) translocation. The aberrant activation of the ABL1 tyrosine kinase is caused by the BCR::ABL1 fusion gene on the Ph chromosome, leading to significant leukemic cell proliferation. CML is typically diagnosed in the chronic phase with few clinical symptoms and progresses to a blast crisis within years. CML acquires additional genetic abnormalities...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37271527"
                        open
                      >
                        <summary class="article-expander__title">Clinical significance of clonal evolution in chronic myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37271527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic myeloid leukemia (CML) is a hematological malignancy characterized by the Philadelphia (Ph) chromosome, which is formed by a t (9;22)(q34;q11) translocation. The aberrant activation of the ABL1 tyrosine kinase is caused by the BCR::ABL1 fusion gene on the Ph chromosome, leading to significant leukemic cell proliferation. CML is typically diagnosed in the chronic phase with few clinical symptoms and progresses to a blast crisis within years. CML acquires additional genetic abnormalities...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37271527"
                        open
                      >
                        <summary class="article-expander__title">Clinical significance of clonal evolution in chronic myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37271527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic myeloid leukemia (CML) is a hematological malignancy characterized by the Philadelphia (Ph) chromosome, which is formed by a t (9;22)(q34;q11) translocation. The aberrant activation of the ABL1 tyrosine kinase is caused by the BCR::ABL1 fusion gene on the Ph chromosome, leading to significant leukemic cell proliferation. CML is typically diagnosed in the chronic phase with few clinical symptoms and progresses to a blast crisis within years. CML acquires additional genetic abnormalities...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37271527"
                        open
                      >
                        <summary class="article-expander__title">Clinical significance of clonal evolution in chronic myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37271527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic myeloid leukemia (CML) is a hematological malignancy characterized by the Philadelphia (Ph) chromosome, which is formed by a t (9;22)(q34;q11) translocation. The aberrant activation of the ABL1 tyrosine kinase is caused by the BCR::ABL1 fusion gene on the Ph chromosome, leading to significant leukemic cell proliferation. CML is typically diagnosed in the chronic phase with few clinical symptoms and progresses to a blast crisis within years. CML acquires additional genetic abnormalities...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-02 datetime="2023-06-02T10:00:00.000Z">2023-06-02</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-02 datetime="2023-06-02T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267914"
                        open
                      >
                        <summary class="article-expander__title">Loss of TET2 in human hematopoietic stem cells alters the development and function of neutrophils</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations commonly occur in hematopoietic stem cells (HSCs). Some mutant clones outgrow through clonal hematopoiesis (CH) and produce mutated immune progenies shaping host immunity. Individuals with CH are asymptomatic but have an increased risk of developing leukemia, cardiovascular and pulmonary inflammatory diseases, and severe infections. Using genetic engineering of human HSCs (hHSCs) and transplantation in immunodeficient mice, we describe how a commonly mutated gene in CH, TET2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267684"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome- positive myelodysplastic syndrome with single lineage dysplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of acquired clonal disorders characterized by dysplastic and ineffective hematopoiesis in the bone marrow. Various specific karyotypic and molecular abnormalities associated with MDS further guide the prognosis. Although translocation t(9;22)(q34;q11) (Philadelphia positive [Ph+]) and corresponding BCR-ABL fusion transcript are classically defined to differentiate CML from non-CML myeloproliferative disorders, it is also associated with adult acute...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267914"
                        open
                      >
                        <summary class="article-expander__title">Loss of TET2 in human hematopoietic stem cells alters the development and function of neutrophils</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations commonly occur in hematopoietic stem cells (HSCs). Some mutant clones outgrow through clonal hematopoiesis (CH) and produce mutated immune progenies shaping host immunity. Individuals with CH are asymptomatic but have an increased risk of developing leukemia, cardiovascular and pulmonary inflammatory diseases, and severe infections. Using genetic engineering of human HSCs (hHSCs) and transplantation in immunodeficient mice, we describe how a commonly mutated gene in CH, TET2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267684"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome- positive myelodysplastic syndrome with single lineage dysplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of acquired clonal disorders characterized by dysplastic and ineffective hematopoiesis in the bone marrow. Various specific karyotypic and molecular abnormalities associated with MDS further guide the prognosis. Although translocation t(9;22)(q34;q11) (Philadelphia positive [Ph+]) and corresponding BCR-ABL fusion transcript are classically defined to differentiate CML from non-CML myeloproliferative disorders, it is also associated with adult acute...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267914"
                        open
                      >
                        <summary class="article-expander__title">Loss of TET2 in human hematopoietic stem cells alters the development and function of neutrophils</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations commonly occur in hematopoietic stem cells (HSCs). Some mutant clones outgrow through clonal hematopoiesis (CH) and produce mutated immune progenies shaping host immunity. Individuals with CH are asymptomatic but have an increased risk of developing leukemia, cardiovascular and pulmonary inflammatory diseases, and severe infections. Using genetic engineering of human HSCs (hHSCs) and transplantation in immunodeficient mice, we describe how a commonly mutated gene in CH, TET2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267684"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome- positive myelodysplastic syndrome with single lineage dysplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of acquired clonal disorders characterized by dysplastic and ineffective hematopoiesis in the bone marrow. Various specific karyotypic and molecular abnormalities associated with MDS further guide the prognosis. Although translocation t(9;22)(q34;q11) (Philadelphia positive [Ph+]) and corresponding BCR-ABL fusion transcript are classically defined to differentiate CML from non-CML myeloproliferative disorders, it is also associated with adult acute...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267914"
                        open
                      >
                        <summary class="article-expander__title">Loss of TET2 in human hematopoietic stem cells alters the development and function of neutrophils</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations commonly occur in hematopoietic stem cells (HSCs). Some mutant clones outgrow through clonal hematopoiesis (CH) and produce mutated immune progenies shaping host immunity. Individuals with CH are asymptomatic but have an increased risk of developing leukemia, cardiovascular and pulmonary inflammatory diseases, and severe infections. Using genetic engineering of human HSCs (hHSCs) and transplantation in immunodeficient mice, we describe how a commonly mutated gene in CH, TET2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267684"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome- positive myelodysplastic syndrome with single lineage dysplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of acquired clonal disorders characterized by dysplastic and ineffective hematopoiesis in the bone marrow. Various specific karyotypic and molecular abnormalities associated with MDS further guide the prognosis. Although translocation t(9;22)(q34;q11) (Philadelphia positive [Ph+]) and corresponding BCR-ABL fusion transcript are classically defined to differentiate CML from non-CML myeloproliferative disorders, it is also associated with adult acute...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267914"
                        open
                      >
                        <summary class="article-expander__title">Loss of TET2 in human hematopoietic stem cells alters the development and function of neutrophils</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations commonly occur in hematopoietic stem cells (HSCs). Some mutant clones outgrow through clonal hematopoiesis (CH) and produce mutated immune progenies shaping host immunity. Individuals with CH are asymptomatic but have an increased risk of developing leukemia, cardiovascular and pulmonary inflammatory diseases, and severe infections. Using genetic engineering of human HSCs (hHSCs) and transplantation in immunodeficient mice, we describe how a commonly mutated gene in CH, TET2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267684"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome- positive myelodysplastic syndrome with single lineage dysplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of acquired clonal disorders characterized by dysplastic and ineffective hematopoiesis in the bone marrow. Various specific karyotypic and molecular abnormalities associated with MDS further guide the prognosis. Although translocation t(9;22)(q34;q11) (Philadelphia positive [Ph+]) and corresponding BCR-ABL fusion transcript are classically defined to differentiate CML from non-CML myeloproliferative disorders, it is also associated with adult acute...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267914"
                        open
                      >
                        <summary class="article-expander__title">Loss of TET2 in human hematopoietic stem cells alters the development and function of neutrophils</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations commonly occur in hematopoietic stem cells (HSCs). Some mutant clones outgrow through clonal hematopoiesis (CH) and produce mutated immune progenies shaping host immunity. Individuals with CH are asymptomatic but have an increased risk of developing leukemia, cardiovascular and pulmonary inflammatory diseases, and severe infections. Using genetic engineering of human HSCs (hHSCs) and transplantation in immunodeficient mice, we describe how a commonly mutated gene in CH, TET2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267684"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome- positive myelodysplastic syndrome with single lineage dysplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of acquired clonal disorders characterized by dysplastic and ineffective hematopoiesis in the bone marrow. Various specific karyotypic and molecular abnormalities associated with MDS further guide the prognosis. Although translocation t(9;22)(q34;q11) (Philadelphia positive [Ph+]) and corresponding BCR-ABL fusion transcript are classically defined to differentiate CML from non-CML myeloproliferative disorders, it is also associated with adult acute...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267914"
                        open
                      >
                        <summary class="article-expander__title">Loss of TET2 in human hematopoietic stem cells alters the development and function of neutrophils</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations commonly occur in hematopoietic stem cells (HSCs). Some mutant clones outgrow through clonal hematopoiesis (CH) and produce mutated immune progenies shaping host immunity. Individuals with CH are asymptomatic but have an increased risk of developing leukemia, cardiovascular and pulmonary inflammatory diseases, and severe infections. Using genetic engineering of human HSCs (hHSCs) and transplantation in immunodeficient mice, we describe how a commonly mutated gene in CH, TET2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267684"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome- positive myelodysplastic syndrome with single lineage dysplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of acquired clonal disorders characterized by dysplastic and ineffective hematopoiesis in the bone marrow. Various specific karyotypic and molecular abnormalities associated with MDS further guide the prognosis. Although translocation t(9;22)(q34;q11) (Philadelphia positive [Ph+]) and corresponding BCR-ABL fusion transcript are classically defined to differentiate CML from non-CML myeloproliferative disorders, it is also associated with adult acute...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267914"
                        open
                      >
                        <summary class="article-expander__title">Loss of TET2 in human hematopoietic stem cells alters the development and function of neutrophils</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations commonly occur in hematopoietic stem cells (HSCs). Some mutant clones outgrow through clonal hematopoiesis (CH) and produce mutated immune progenies shaping host immunity. Individuals with CH are asymptomatic but have an increased risk of developing leukemia, cardiovascular and pulmonary inflammatory diseases, and severe infections. Using genetic engineering of human HSCs (hHSCs) and transplantation in immunodeficient mice, we describe how a commonly mutated gene in CH, TET2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267684"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome- positive myelodysplastic syndrome with single lineage dysplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of acquired clonal disorders characterized by dysplastic and ineffective hematopoiesis in the bone marrow. Various specific karyotypic and molecular abnormalities associated with MDS further guide the prognosis. Although translocation t(9;22)(q34;q11) (Philadelphia positive [Ph+]) and corresponding BCR-ABL fusion transcript are classically defined to differentiate CML from non-CML myeloproliferative disorders, it is also associated with adult acute...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267914"
                        open
                      >
                        <summary class="article-expander__title">Loss of TET2 in human hematopoietic stem cells alters the development and function of neutrophils</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations commonly occur in hematopoietic stem cells (HSCs). Some mutant clones outgrow through clonal hematopoiesis (CH) and produce mutated immune progenies shaping host immunity. Individuals with CH are asymptomatic but have an increased risk of developing leukemia, cardiovascular and pulmonary inflammatory diseases, and severe infections. Using genetic engineering of human HSCs (hHSCs) and transplantation in immunodeficient mice, we describe how a commonly mutated gene in CH, TET2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267684"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome- positive myelodysplastic syndrome with single lineage dysplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of acquired clonal disorders characterized by dysplastic and ineffective hematopoiesis in the bone marrow. Various specific karyotypic and molecular abnormalities associated with MDS further guide the prognosis. Although translocation t(9;22)(q34;q11) (Philadelphia positive [Ph+]) and corresponding BCR-ABL fusion transcript are classically defined to differentiate CML from non-CML myeloproliferative disorders, it is also associated with adult acute...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267914"
                        open
                      >
                        <summary class="article-expander__title">Loss of TET2 in human hematopoietic stem cells alters the development and function of neutrophils</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations commonly occur in hematopoietic stem cells (HSCs). Some mutant clones outgrow through clonal hematopoiesis (CH) and produce mutated immune progenies shaping host immunity. Individuals with CH are asymptomatic but have an increased risk of developing leukemia, cardiovascular and pulmonary inflammatory diseases, and severe infections. Using genetic engineering of human HSCs (hHSCs) and transplantation in immunodeficient mice, we describe how a commonly mutated gene in CH, TET2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267684"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome- positive myelodysplastic syndrome with single lineage dysplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of acquired clonal disorders characterized by dysplastic and ineffective hematopoiesis in the bone marrow. Various specific karyotypic and molecular abnormalities associated with MDS further guide the prognosis. Although translocation t(9;22)(q34;q11) (Philadelphia positive [Ph+]) and corresponding BCR-ABL fusion transcript are classically defined to differentiate CML from non-CML myeloproliferative disorders, it is also associated with adult acute...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267914"
                        open
                      >
                        <summary class="article-expander__title">Loss of TET2 in human hematopoietic stem cells alters the development and function of neutrophils</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations commonly occur in hematopoietic stem cells (HSCs). Some mutant clones outgrow through clonal hematopoiesis (CH) and produce mutated immune progenies shaping host immunity. Individuals with CH are asymptomatic but have an increased risk of developing leukemia, cardiovascular and pulmonary inflammatory diseases, and severe infections. Using genetic engineering of human HSCs (hHSCs) and transplantation in immunodeficient mice, we describe how a commonly mutated gene in CH, TET2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267684"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome- positive myelodysplastic syndrome with single lineage dysplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of acquired clonal disorders characterized by dysplastic and ineffective hematopoiesis in the bone marrow. Various specific karyotypic and molecular abnormalities associated with MDS further guide the prognosis. Although translocation t(9;22)(q34;q11) (Philadelphia positive [Ph+]) and corresponding BCR-ABL fusion transcript are classically defined to differentiate CML from non-CML myeloproliferative disorders, it is also associated with adult acute...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267914"
                        open
                      >
                        <summary class="article-expander__title">Loss of TET2 in human hematopoietic stem cells alters the development and function of neutrophils</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations commonly occur in hematopoietic stem cells (HSCs). Some mutant clones outgrow through clonal hematopoiesis (CH) and produce mutated immune progenies shaping host immunity. Individuals with CH are asymptomatic but have an increased risk of developing leukemia, cardiovascular and pulmonary inflammatory diseases, and severe infections. Using genetic engineering of human HSCs (hHSCs) and transplantation in immunodeficient mice, we describe how a commonly mutated gene in CH, TET2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267684"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome- positive myelodysplastic syndrome with single lineage dysplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of acquired clonal disorders characterized by dysplastic and ineffective hematopoiesis in the bone marrow. Various specific karyotypic and molecular abnormalities associated with MDS further guide the prognosis. Although translocation t(9;22)(q34;q11) (Philadelphia positive [Ph+]) and corresponding BCR-ABL fusion transcript are classically defined to differentiate CML from non-CML myeloproliferative disorders, it is also associated with adult acute...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267914"
                        open
                      >
                        <summary class="article-expander__title">Loss of TET2 in human hematopoietic stem cells alters the development and function of neutrophils</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations commonly occur in hematopoietic stem cells (HSCs). Some mutant clones outgrow through clonal hematopoiesis (CH) and produce mutated immune progenies shaping host immunity. Individuals with CH are asymptomatic but have an increased risk of developing leukemia, cardiovascular and pulmonary inflammatory diseases, and severe infections. Using genetic engineering of human HSCs (hHSCs) and transplantation in immunodeficient mice, we describe how a commonly mutated gene in CH, TET2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267684"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome- positive myelodysplastic syndrome with single lineage dysplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of acquired clonal disorders characterized by dysplastic and ineffective hematopoiesis in the bone marrow. Various specific karyotypic and molecular abnormalities associated with MDS further guide the prognosis. Although translocation t(9;22)(q34;q11) (Philadelphia positive [Ph+]) and corresponding BCR-ABL fusion transcript are classically defined to differentiate CML from non-CML myeloproliferative disorders, it is also associated with adult acute...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267914"
                        open
                      >
                        <summary class="article-expander__title">Loss of TET2 in human hematopoietic stem cells alters the development and function of neutrophils</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations commonly occur in hematopoietic stem cells (HSCs). Some mutant clones outgrow through clonal hematopoiesis (CH) and produce mutated immune progenies shaping host immunity. Individuals with CH are asymptomatic but have an increased risk of developing leukemia, cardiovascular and pulmonary inflammatory diseases, and severe infections. Using genetic engineering of human HSCs (hHSCs) and transplantation in immunodeficient mice, we describe how a commonly mutated gene in CH, TET2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267684"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome- positive myelodysplastic syndrome with single lineage dysplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of acquired clonal disorders characterized by dysplastic and ineffective hematopoiesis in the bone marrow. Various specific karyotypic and molecular abnormalities associated with MDS further guide the prognosis. Although translocation t(9;22)(q34;q11) (Philadelphia positive [Ph+]) and corresponding BCR-ABL fusion transcript are classically defined to differentiate CML from non-CML myeloproliferative disorders, it is also associated with adult acute...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267914"
                        open
                      >
                        <summary class="article-expander__title">Loss of TET2 in human hematopoietic stem cells alters the development and function of neutrophils</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations commonly occur in hematopoietic stem cells (HSCs). Some mutant clones outgrow through clonal hematopoiesis (CH) and produce mutated immune progenies shaping host immunity. Individuals with CH are asymptomatic but have an increased risk of developing leukemia, cardiovascular and pulmonary inflammatory diseases, and severe infections. Using genetic engineering of human HSCs (hHSCs) and transplantation in immunodeficient mice, we describe how a commonly mutated gene in CH, TET2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267684"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome- positive myelodysplastic syndrome with single lineage dysplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of acquired clonal disorders characterized by dysplastic and ineffective hematopoiesis in the bone marrow. Various specific karyotypic and molecular abnormalities associated with MDS further guide the prognosis. Although translocation t(9;22)(q34;q11) (Philadelphia positive [Ph+]) and corresponding BCR-ABL fusion transcript are classically defined to differentiate CML from non-CML myeloproliferative disorders, it is also associated with adult acute...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267914"
                        open
                      >
                        <summary class="article-expander__title">Loss of TET2 in human hematopoietic stem cells alters the development and function of neutrophils</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations commonly occur in hematopoietic stem cells (HSCs). Some mutant clones outgrow through clonal hematopoiesis (CH) and produce mutated immune progenies shaping host immunity. Individuals with CH are asymptomatic but have an increased risk of developing leukemia, cardiovascular and pulmonary inflammatory diseases, and severe infections. Using genetic engineering of human HSCs (hHSCs) and transplantation in immunodeficient mice, we describe how a commonly mutated gene in CH, TET2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267684"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome- positive myelodysplastic syndrome with single lineage dysplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of acquired clonal disorders characterized by dysplastic and ineffective hematopoiesis in the bone marrow. Various specific karyotypic and molecular abnormalities associated with MDS further guide the prognosis. Although translocation t(9;22)(q34;q11) (Philadelphia positive [Ph+]) and corresponding BCR-ABL fusion transcript are classically defined to differentiate CML from non-CML myeloproliferative disorders, it is also associated with adult acute...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267914"
                        open
                      >
                        <summary class="article-expander__title">Loss of TET2 in human hematopoietic stem cells alters the development and function of neutrophils</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations commonly occur in hematopoietic stem cells (HSCs). Some mutant clones outgrow through clonal hematopoiesis (CH) and produce mutated immune progenies shaping host immunity. Individuals with CH are asymptomatic but have an increased risk of developing leukemia, cardiovascular and pulmonary inflammatory diseases, and severe infections. Using genetic engineering of human HSCs (hHSCs) and transplantation in immunodeficient mice, we describe how a commonly mutated gene in CH, TET2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267684"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome- positive myelodysplastic syndrome with single lineage dysplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of acquired clonal disorders characterized by dysplastic and ineffective hematopoiesis in the bone marrow. Various specific karyotypic and molecular abnormalities associated with MDS further guide the prognosis. Although translocation t(9;22)(q34;q11) (Philadelphia positive [Ph+]) and corresponding BCR-ABL fusion transcript are classically defined to differentiate CML from non-CML myeloproliferative disorders, it is also associated with adult acute...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267914"
                        open
                      >
                        <summary class="article-expander__title">Loss of TET2 in human hematopoietic stem cells alters the development and function of neutrophils</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations commonly occur in hematopoietic stem cells (HSCs). Some mutant clones outgrow through clonal hematopoiesis (CH) and produce mutated immune progenies shaping host immunity. Individuals with CH are asymptomatic but have an increased risk of developing leukemia, cardiovascular and pulmonary inflammatory diseases, and severe infections. Using genetic engineering of human HSCs (hHSCs) and transplantation in immunodeficient mice, we describe how a commonly mutated gene in CH, TET2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267684"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome- positive myelodysplastic syndrome with single lineage dysplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of acquired clonal disorders characterized by dysplastic and ineffective hematopoiesis in the bone marrow. Various specific karyotypic and molecular abnormalities associated with MDS further guide the prognosis. Although translocation t(9;22)(q34;q11) (Philadelphia positive [Ph+]) and corresponding BCR-ABL fusion transcript are classically defined to differentiate CML from non-CML myeloproliferative disorders, it is also associated with adult acute...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267914"
                        open
                      >
                        <summary class="article-expander__title">Loss of TET2 in human hematopoietic stem cells alters the development and function of neutrophils</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations commonly occur in hematopoietic stem cells (HSCs). Some mutant clones outgrow through clonal hematopoiesis (CH) and produce mutated immune progenies shaping host immunity. Individuals with CH are asymptomatic but have an increased risk of developing leukemia, cardiovascular and pulmonary inflammatory diseases, and severe infections. Using genetic engineering of human HSCs (hHSCs) and transplantation in immunodeficient mice, we describe how a commonly mutated gene in CH, TET2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267684"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome- positive myelodysplastic syndrome with single lineage dysplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of acquired clonal disorders characterized by dysplastic and ineffective hematopoiesis in the bone marrow. Various specific karyotypic and molecular abnormalities associated with MDS further guide the prognosis. Although translocation t(9;22)(q34;q11) (Philadelphia positive [Ph+]) and corresponding BCR-ABL fusion transcript are classically defined to differentiate CML from non-CML myeloproliferative disorders, it is also associated with adult acute...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267914"
                        open
                      >
                        <summary class="article-expander__title">Loss of TET2 in human hematopoietic stem cells alters the development and function of neutrophils</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations commonly occur in hematopoietic stem cells (HSCs). Some mutant clones outgrow through clonal hematopoiesis (CH) and produce mutated immune progenies shaping host immunity. Individuals with CH are asymptomatic but have an increased risk of developing leukemia, cardiovascular and pulmonary inflammatory diseases, and severe infections. Using genetic engineering of human HSCs (hHSCs) and transplantation in immunodeficient mice, we describe how a commonly mutated gene in CH, TET2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267684"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome- positive myelodysplastic syndrome with single lineage dysplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of acquired clonal disorders characterized by dysplastic and ineffective hematopoiesis in the bone marrow. Various specific karyotypic and molecular abnormalities associated with MDS further guide the prognosis. Although translocation t(9;22)(q34;q11) (Philadelphia positive [Ph+]) and corresponding BCR-ABL fusion transcript are classically defined to differentiate CML from non-CML myeloproliferative disorders, it is also associated with adult acute...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267914"
                        open
                      >
                        <summary class="article-expander__title">Loss of TET2 in human hematopoietic stem cells alters the development and function of neutrophils</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations commonly occur in hematopoietic stem cells (HSCs). Some mutant clones outgrow through clonal hematopoiesis (CH) and produce mutated immune progenies shaping host immunity. Individuals with CH are asymptomatic but have an increased risk of developing leukemia, cardiovascular and pulmonary inflammatory diseases, and severe infections. Using genetic engineering of human HSCs (hHSCs) and transplantation in immunodeficient mice, we describe how a commonly mutated gene in CH, TET2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267684"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome- positive myelodysplastic syndrome with single lineage dysplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of acquired clonal disorders characterized by dysplastic and ineffective hematopoiesis in the bone marrow. Various specific karyotypic and molecular abnormalities associated with MDS further guide the prognosis. Although translocation t(9;22)(q34;q11) (Philadelphia positive [Ph+]) and corresponding BCR-ABL fusion transcript are classically defined to differentiate CML from non-CML myeloproliferative disorders, it is also associated with adult acute...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267914"
                        open
                      >
                        <summary class="article-expander__title">Loss of TET2 in human hematopoietic stem cells alters the development and function of neutrophils</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations commonly occur in hematopoietic stem cells (HSCs). Some mutant clones outgrow through clonal hematopoiesis (CH) and produce mutated immune progenies shaping host immunity. Individuals with CH are asymptomatic but have an increased risk of developing leukemia, cardiovascular and pulmonary inflammatory diseases, and severe infections. Using genetic engineering of human HSCs (hHSCs) and transplantation in immunodeficient mice, we describe how a commonly mutated gene in CH, TET2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267684"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome- positive myelodysplastic syndrome with single lineage dysplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of acquired clonal disorders characterized by dysplastic and ineffective hematopoiesis in the bone marrow. Various specific karyotypic and molecular abnormalities associated with MDS further guide the prognosis. Although translocation t(9;22)(q34;q11) (Philadelphia positive [Ph+]) and corresponding BCR-ABL fusion transcript are classically defined to differentiate CML from non-CML myeloproliferative disorders, it is also associated with adult acute...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267914"
                        open
                      >
                        <summary class="article-expander__title">Loss of TET2 in human hematopoietic stem cells alters the development and function of neutrophils</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations commonly occur in hematopoietic stem cells (HSCs). Some mutant clones outgrow through clonal hematopoiesis (CH) and produce mutated immune progenies shaping host immunity. Individuals with CH are asymptomatic but have an increased risk of developing leukemia, cardiovascular and pulmonary inflammatory diseases, and severe infections. Using genetic engineering of human HSCs (hHSCs) and transplantation in immunodeficient mice, we describe how a commonly mutated gene in CH, TET2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37267684"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome- positive myelodysplastic syndrome with single lineage dysplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37267684/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of acquired clonal disorders characterized by dysplastic and ineffective hematopoiesis in the bone marrow. Various specific karyotypic and molecular abnormalities associated with MDS further guide the prognosis. Although translocation t(9;22)(q34;q11) (Philadelphia positive [Ph+]) and corresponding BCR-ABL fusion transcript are classically defined to differentiate CML from non-CML myeloproliferative disorders, it is also associated with adult acute...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-06-01 datetime="2023-06-01T10:00:00.000Z">2023-06-01</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-06-01 datetime="2023-06-01T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37264310"
                        open
                      >
                        <summary class="article-expander__title">The impact and outcomes of cancer-macrophage fusion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37264310/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The study revealed that tumor-somatic cell fusion is a potent environmental modifier that can modulate tumor survival and evolution, despite its relatively low occurrence. These findings suggest that tumor-somatic cell fusion could be a promising target for developing new cancer therapies. Furthermore, this study provides an experimental animal platform to investigate cancer-myeloid fusion and highlights the potential role of tumor-somatic cell fusion in modulating the tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37264310"
                        open
                      >
                        <summary class="article-expander__title">The impact and outcomes of cancer-macrophage fusion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37264310/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The study revealed that tumor-somatic cell fusion is a potent environmental modifier that can modulate tumor survival and evolution, despite its relatively low occurrence. These findings suggest that tumor-somatic cell fusion could be a promising target for developing new cancer therapies. Furthermore, this study provides an experimental animal platform to investigate cancer-myeloid fusion and highlights the potential role of tumor-somatic cell fusion in modulating the tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37264310"
                        open
                      >
                        <summary class="article-expander__title">The impact and outcomes of cancer-macrophage fusion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37264310/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The study revealed that tumor-somatic cell fusion is a potent environmental modifier that can modulate tumor survival and evolution, despite its relatively low occurrence. These findings suggest that tumor-somatic cell fusion could be a promising target for developing new cancer therapies. Furthermore, this study provides an experimental animal platform to investigate cancer-myeloid fusion and highlights the potential role of tumor-somatic cell fusion in modulating the tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37264310"
                        open
                      >
                        <summary class="article-expander__title">The impact and outcomes of cancer-macrophage fusion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37264310/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The study revealed that tumor-somatic cell fusion is a potent environmental modifier that can modulate tumor survival and evolution, despite its relatively low occurrence. These findings suggest that tumor-somatic cell fusion could be a promising target for developing new cancer therapies. Furthermore, this study provides an experimental animal platform to investigate cancer-myeloid fusion and highlights the potential role of tumor-somatic cell fusion in modulating the tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37264310"
                        open
                      >
                        <summary class="article-expander__title">The impact and outcomes of cancer-macrophage fusion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37264310/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The study revealed that tumor-somatic cell fusion is a potent environmental modifier that can modulate tumor survival and evolution, despite its relatively low occurrence. These findings suggest that tumor-somatic cell fusion could be a promising target for developing new cancer therapies. Furthermore, this study provides an experimental animal platform to investigate cancer-myeloid fusion and highlights the potential role of tumor-somatic cell fusion in modulating the tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37264310"
                        open
                      >
                        <summary class="article-expander__title">The impact and outcomes of cancer-macrophage fusion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37264310/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The study revealed that tumor-somatic cell fusion is a potent environmental modifier that can modulate tumor survival and evolution, despite its relatively low occurrence. These findings suggest that tumor-somatic cell fusion could be a promising target for developing new cancer therapies. Furthermore, this study provides an experimental animal platform to investigate cancer-myeloid fusion and highlights the potential role of tumor-somatic cell fusion in modulating the tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37264310"
                        open
                      >
                        <summary class="article-expander__title">The impact and outcomes of cancer-macrophage fusion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37264310/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The study revealed that tumor-somatic cell fusion is a potent environmental modifier that can modulate tumor survival and evolution, despite its relatively low occurrence. These findings suggest that tumor-somatic cell fusion could be a promising target for developing new cancer therapies. Furthermore, this study provides an experimental animal platform to investigate cancer-myeloid fusion and highlights the potential role of tumor-somatic cell fusion in modulating the tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37264310"
                        open
                      >
                        <summary class="article-expander__title">The impact and outcomes of cancer-macrophage fusion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37264310/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The study revealed that tumor-somatic cell fusion is a potent environmental modifier that can modulate tumor survival and evolution, despite its relatively low occurrence. These findings suggest that tumor-somatic cell fusion could be a promising target for developing new cancer therapies. Furthermore, this study provides an experimental animal platform to investigate cancer-myeloid fusion and highlights the potential role of tumor-somatic cell fusion in modulating the tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37264310"
                        open
                      >
                        <summary class="article-expander__title">The impact and outcomes of cancer-macrophage fusion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37264310/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The study revealed that tumor-somatic cell fusion is a potent environmental modifier that can modulate tumor survival and evolution, despite its relatively low occurrence. These findings suggest that tumor-somatic cell fusion could be a promising target for developing new cancer therapies. Furthermore, this study provides an experimental animal platform to investigate cancer-myeloid fusion and highlights the potential role of tumor-somatic cell fusion in modulating the tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37264310"
                        open
                      >
                        <summary class="article-expander__title">The impact and outcomes of cancer-macrophage fusion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37264310/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The study revealed that tumor-somatic cell fusion is a potent environmental modifier that can modulate tumor survival and evolution, despite its relatively low occurrence. These findings suggest that tumor-somatic cell fusion could be a promising target for developing new cancer therapies. Furthermore, this study provides an experimental animal platform to investigate cancer-myeloid fusion and highlights the potential role of tumor-somatic cell fusion in modulating the tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37264310"
                        open
                      >
                        <summary class="article-expander__title">The impact and outcomes of cancer-macrophage fusion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37264310/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The study revealed that tumor-somatic cell fusion is a potent environmental modifier that can modulate tumor survival and evolution, despite its relatively low occurrence. These findings suggest that tumor-somatic cell fusion could be a promising target for developing new cancer therapies. Furthermore, this study provides an experimental animal platform to investigate cancer-myeloid fusion and highlights the potential role of tumor-somatic cell fusion in modulating the tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37264310"
                        open
                      >
                        <summary class="article-expander__title">The impact and outcomes of cancer-macrophage fusion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37264310/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The study revealed that tumor-somatic cell fusion is a potent environmental modifier that can modulate tumor survival and evolution, despite its relatively low occurrence. These findings suggest that tumor-somatic cell fusion could be a promising target for developing new cancer therapies. Furthermore, this study provides an experimental animal platform to investigate cancer-myeloid fusion and highlights the potential role of tumor-somatic cell fusion in modulating the tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37264310"
                        open
                      >
                        <summary class="article-expander__title">The impact and outcomes of cancer-macrophage fusion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37264310/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The study revealed that tumor-somatic cell fusion is a potent environmental modifier that can modulate tumor survival and evolution, despite its relatively low occurrence. These findings suggest that tumor-somatic cell fusion could be a promising target for developing new cancer therapies. Furthermore, this study provides an experimental animal platform to investigate cancer-myeloid fusion and highlights the potential role of tumor-somatic cell fusion in modulating the tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37264310"
                        open
                      >
                        <summary class="article-expander__title">The impact and outcomes of cancer-macrophage fusion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37264310/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The study revealed that tumor-somatic cell fusion is a potent environmental modifier that can modulate tumor survival and evolution, despite its relatively low occurrence. These findings suggest that tumor-somatic cell fusion could be a promising target for developing new cancer therapies. Furthermore, this study provides an experimental animal platform to investigate cancer-myeloid fusion and highlights the potential role of tumor-somatic cell fusion in modulating the tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37264310"
                        open
                      >
                        <summary class="article-expander__title">The impact and outcomes of cancer-macrophage fusion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37264310/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The study revealed that tumor-somatic cell fusion is a potent environmental modifier that can modulate tumor survival and evolution, despite its relatively low occurrence. These findings suggest that tumor-somatic cell fusion could be a promising target for developing new cancer therapies. Furthermore, this study provides an experimental animal platform to investigate cancer-myeloid fusion and highlights the potential role of tumor-somatic cell fusion in modulating the tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-05-31 datetime="2023-05-31T10:00:00.000Z">2023-05-31</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-05-31 datetime="2023-05-31T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37254855"
                        open
                      >
                        <summary class="article-expander__title">The impact of prenatal inflammation on hematopoietic development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37254855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230624211813&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Inflammation is now recognized as a major regulator of hematopoietic stem cell (HSC) function. Adult hematopoietic stem cells can adaptively modulate hematopoietic output in direct response to acute infection and inflammation. Conversely, prolonged exposure to inflammation can drive impaired HSC function, clonal expansion, and malignant transformation. As compared with adult hematopoiesis, the effects of prenatal inflammation on developing hematopoietic stem cells are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37254855"
                        open
                      >
                        <summary class="article-expander__title">The impact of prenatal inflammation on hematopoietic development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37254855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230623211123&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Inflammation is now recognized as a major regulator of hematopoietic stem cell (HSC) function. Adult hematopoietic stem cells can adaptively modulate hematopoietic output in direct response to acute infection and inflammation. Conversely, prolonged exposure to inflammation can drive impaired HSC function, clonal expansion, and malignant transformation. As compared with adult hematopoiesis, the effects of prenatal inflammation on developing hematopoietic stem cells are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37254855"
                        open
                      >
                        <summary class="article-expander__title">The impact of prenatal inflammation on hematopoietic development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37254855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Inflammation is now recognized as a major regulator of hematopoietic stem cell (HSC) function. Adult hematopoietic stem cells can adaptively modulate hematopoietic output in direct response to acute infection and inflammation. Conversely, prolonged exposure to inflammation can drive impaired HSC function, clonal expansion, and malignant transformation. As compared with adult hematopoiesis, the effects of prenatal inflammation on developing hematopoietic stem cells are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37254855"
                        open
                      >
                        <summary class="article-expander__title">The impact of prenatal inflammation on hematopoietic development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37254855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Inflammation is now recognized as a major regulator of hematopoietic stem cell (HSC) function. Adult hematopoietic stem cells can adaptively modulate hematopoietic output in direct response to acute infection and inflammation. Conversely, prolonged exposure to inflammation can drive impaired HSC function, clonal expansion, and malignant transformation. As compared with adult hematopoiesis, the effects of prenatal inflammation on developing hematopoietic stem cells are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37254855"
                        open
                      >
                        <summary class="article-expander__title">The impact of prenatal inflammation on hematopoietic development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37254855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Inflammation is now recognized as a major regulator of hematopoietic stem cell (HSC) function. Adult hematopoietic stem cells can adaptively modulate hematopoietic output in direct response to acute infection and inflammation. Conversely, prolonged exposure to inflammation can drive impaired HSC function, clonal expansion, and malignant transformation. As compared with adult hematopoiesis, the effects of prenatal inflammation on developing hematopoietic stem cells are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37254855"
                        open
                      >
                        <summary class="article-expander__title">The impact of prenatal inflammation on hematopoietic development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37254855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Inflammation is now recognized as a major regulator of hematopoietic stem cell (HSC) function. Adult hematopoietic stem cells can adaptively modulate hematopoietic output in direct response to acute infection and inflammation. Conversely, prolonged exposure to inflammation can drive impaired HSC function, clonal expansion, and malignant transformation. As compared with adult hematopoiesis, the effects of prenatal inflammation on developing hematopoietic stem cells are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37254855"
                        open
                      >
                        <summary class="article-expander__title">The impact of prenatal inflammation on hematopoietic development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37254855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Inflammation is now recognized as a major regulator of hematopoietic stem cell (HSC) function. Adult hematopoietic stem cells can adaptively modulate hematopoietic output in direct response to acute infection and inflammation. Conversely, prolonged exposure to inflammation can drive impaired HSC function, clonal expansion, and malignant transformation. As compared with adult hematopoiesis, the effects of prenatal inflammation on developing hematopoietic stem cells are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37254855"
                        open
                      >
                        <summary class="article-expander__title">The impact of prenatal inflammation on hematopoietic development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37254855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Inflammation is now recognized as a major regulator of hematopoietic stem cell (HSC) function. Adult hematopoietic stem cells can adaptively modulate hematopoietic output in direct response to acute infection and inflammation. Conversely, prolonged exposure to inflammation can drive impaired HSC function, clonal expansion, and malignant transformation. As compared with adult hematopoiesis, the effects of prenatal inflammation on developing hematopoietic stem cells are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37254855"
                        open
                      >
                        <summary class="article-expander__title">The impact of prenatal inflammation on hematopoietic development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37254855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Inflammation is now recognized as a major regulator of hematopoietic stem cell (HSC) function. Adult hematopoietic stem cells can adaptively modulate hematopoietic output in direct response to acute infection and inflammation. Conversely, prolonged exposure to inflammation can drive impaired HSC function, clonal expansion, and malignant transformation. As compared with adult hematopoiesis, the effects of prenatal inflammation on developing hematopoietic stem cells are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37254855"
                        open
                      >
                        <summary class="article-expander__title">The impact of prenatal inflammation on hematopoietic development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37254855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Inflammation is now recognized as a major regulator of hematopoietic stem cell (HSC) function. Adult hematopoietic stem cells can adaptively modulate hematopoietic output in direct response to acute infection and inflammation. Conversely, prolonged exposure to inflammation can drive impaired HSC function, clonal expansion, and malignant transformation. As compared with adult hematopoiesis, the effects of prenatal inflammation on developing hematopoietic stem cells are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37254855"
                        open
                      >
                        <summary class="article-expander__title">The impact of prenatal inflammation on hematopoietic development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37254855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Inflammation is now recognized as a major regulator of hematopoietic stem cell (HSC) function. Adult hematopoietic stem cells can adaptively modulate hematopoietic output in direct response to acute infection and inflammation. Conversely, prolonged exposure to inflammation can drive impaired HSC function, clonal expansion, and malignant transformation. As compared with adult hematopoiesis, the effects of prenatal inflammation on developing hematopoietic stem cells are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37254855"
                        open
                      >
                        <summary class="article-expander__title">The impact of prenatal inflammation on hematopoietic development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37254855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Inflammation is now recognized as a major regulator of hematopoietic stem cell (HSC) function. Adult hematopoietic stem cells can adaptively modulate hematopoietic output in direct response to acute infection and inflammation. Conversely, prolonged exposure to inflammation can drive impaired HSC function, clonal expansion, and malignant transformation. As compared with adult hematopoiesis, the effects of prenatal inflammation on developing hematopoietic stem cells are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37254855"
                        open
                      >
                        <summary class="article-expander__title">The impact of prenatal inflammation on hematopoietic development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37254855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Inflammation is now recognized as a major regulator of hematopoietic stem cell (HSC) function. Adult hematopoietic stem cells can adaptively modulate hematopoietic output in direct response to acute infection and inflammation. Conversely, prolonged exposure to inflammation can drive impaired HSC function, clonal expansion, and malignant transformation. As compared with adult hematopoiesis, the effects of prenatal inflammation on developing hematopoietic stem cells are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37254855"
                        open
                      >
                        <summary class="article-expander__title">The impact of prenatal inflammation on hematopoietic development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37254855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Inflammation is now recognized as a major regulator of hematopoietic stem cell (HSC) function. Adult hematopoietic stem cells can adaptively modulate hematopoietic output in direct response to acute infection and inflammation. Conversely, prolonged exposure to inflammation can drive impaired HSC function, clonal expansion, and malignant transformation. As compared with adult hematopoiesis, the effects of prenatal inflammation on developing hematopoietic stem cells are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37254855"
                        open
                      >
                        <summary class="article-expander__title">The impact of prenatal inflammation on hematopoietic development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37254855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Inflammation is now recognized as a major regulator of hematopoietic stem cell (HSC) function. Adult hematopoietic stem cells can adaptively modulate hematopoietic output in direct response to acute infection and inflammation. Conversely, prolonged exposure to inflammation can drive impaired HSC function, clonal expansion, and malignant transformation. As compared with adult hematopoiesis, the effects of prenatal inflammation on developing hematopoietic stem cells are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37254855"
                        open
                      >
                        <summary class="article-expander__title">The impact of prenatal inflammation on hematopoietic development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37254855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Inflammation is now recognized as a major regulator of hematopoietic stem cell (HSC) function. Adult hematopoietic stem cells can adaptively modulate hematopoietic output in direct response to acute infection and inflammation. Conversely, prolonged exposure to inflammation can drive impaired HSC function, clonal expansion, and malignant transformation. As compared with adult hematopoiesis, the effects of prenatal inflammation on developing hematopoietic stem cells are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37254855"
                        open
                      >
                        <summary class="article-expander__title">The impact of prenatal inflammation on hematopoietic development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37254855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Inflammation is now recognized as a major regulator of hematopoietic stem cell (HSC) function. Adult hematopoietic stem cells can adaptively modulate hematopoietic output in direct response to acute infection and inflammation. Conversely, prolonged exposure to inflammation can drive impaired HSC function, clonal expansion, and malignant transformation. As compared with adult hematopoiesis, the effects of prenatal inflammation on developing hematopoietic stem cells are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37254855"
                        open
                      >
                        <summary class="article-expander__title">The impact of prenatal inflammation on hematopoietic development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37254855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Inflammation is now recognized as a major regulator of hematopoietic stem cell (HSC) function. Adult hematopoietic stem cells can adaptively modulate hematopoietic output in direct response to acute infection and inflammation. Conversely, prolonged exposure to inflammation can drive impaired HSC function, clonal expansion, and malignant transformation. As compared with adult hematopoiesis, the effects of prenatal inflammation on developing hematopoietic stem cells are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37254855"
                        open
                      >
                        <summary class="article-expander__title">The impact of prenatal inflammation on hematopoietic development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37254855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Inflammation is now recognized as a major regulator of hematopoietic stem cell (HSC) function. Adult hematopoietic stem cells can adaptively modulate hematopoietic output in direct response to acute infection and inflammation. Conversely, prolonged exposure to inflammation can drive impaired HSC function, clonal expansion, and malignant transformation. As compared with adult hematopoiesis, the effects of prenatal inflammation on developing hematopoietic stem cells are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37254855"
                        open
                      >
                        <summary class="article-expander__title">The impact of prenatal inflammation on hematopoietic development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37254855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Inflammation is now recognized as a major regulator of hematopoietic stem cell (HSC) function. Adult hematopoietic stem cells can adaptively modulate hematopoietic output in direct response to acute infection and inflammation. Conversely, prolonged exposure to inflammation can drive impaired HSC function, clonal expansion, and malignant transformation. As compared with adult hematopoiesis, the effects of prenatal inflammation on developing hematopoietic stem cells are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37254855"
                        open
                      >
                        <summary class="article-expander__title">The impact of prenatal inflammation on hematopoietic development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37254855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Inflammation is now recognized as a major regulator of hematopoietic stem cell (HSC) function. Adult hematopoietic stem cells can adaptively modulate hematopoietic output in direct response to acute infection and inflammation. Conversely, prolonged exposure to inflammation can drive impaired HSC function, clonal expansion, and malignant transformation. As compared with adult hematopoiesis, the effects of prenatal inflammation on developing hematopoietic stem cells are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37254855"
                        open
                      >
                        <summary class="article-expander__title">The impact of prenatal inflammation on hematopoietic development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37254855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Inflammation is now recognized as a major regulator of hematopoietic stem cell (HSC) function. Adult hematopoietic stem cells can adaptively modulate hematopoietic output in direct response to acute infection and inflammation. Conversely, prolonged exposure to inflammation can drive impaired HSC function, clonal expansion, and malignant transformation. As compared with adult hematopoiesis, the effects of prenatal inflammation on developing hematopoietic stem cells are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37254855"
                        open
                      >
                        <summary class="article-expander__title">The impact of prenatal inflammation on hematopoietic development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37254855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Inflammation is now recognized as a major regulator of hematopoietic stem cell (HSC) function. Adult hematopoietic stem cells can adaptively modulate hematopoietic output in direct response to acute infection and inflammation. Conversely, prolonged exposure to inflammation can drive impaired HSC function, clonal expansion, and malignant transformation. As compared with adult hematopoiesis, the effects of prenatal inflammation on developing hematopoietic stem cells are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37254855"
                        open
                      >
                        <summary class="article-expander__title">The impact of prenatal inflammation on hematopoietic development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37254855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Inflammation is now recognized as a major regulator of hematopoietic stem cell (HSC) function. Adult hematopoietic stem cells can adaptively modulate hematopoietic output in direct response to acute infection and inflammation. Conversely, prolonged exposure to inflammation can drive impaired HSC function, clonal expansion, and malignant transformation. As compared with adult hematopoiesis, the effects of prenatal inflammation on developing hematopoietic stem cells are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37254855"
                        open
                      >
                        <summary class="article-expander__title">The impact of prenatal inflammation on hematopoietic development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37254855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Inflammation is now recognized as a major regulator of hematopoietic stem cell (HSC) function. Adult hematopoietic stem cells can adaptively modulate hematopoietic output in direct response to acute infection and inflammation. Conversely, prolonged exposure to inflammation can drive impaired HSC function, clonal expansion, and malignant transformation. As compared with adult hematopoiesis, the effects of prenatal inflammation on developing hematopoietic stem cells are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37258665"
                        open
                      >
                        <summary class="article-expander__title">Deterministic evolution and stringent selection during preneoplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37258665/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The earliest events during human tumour initiation, although poorly characterized, may hold clues to malignancy detection and prevention¹. Here we model occult preneoplasia by biallelic inactivation of TP53, a common early event in gastric cancer, in human gastric organoids. Causal relationships between this initiating genetic lesion and resulting phenotypes were established using experimental evolution in multiple clonally derived cultures over 2 years. TP53 loss elicited progressive...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;20
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37258665"
                        open
                      >
                        <summary class="article-expander__title">Deterministic evolution and stringent selection during preneoplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37258665/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The earliest events during human tumour initiation, although poorly characterized, may hold clues to malignancy detection and prevention¹. Here we model occult preneoplasia by biallelic inactivation of TP53, a common early event in gastric cancer, in human gastric organoids. Causal relationships between this initiating genetic lesion and resulting phenotypes were established using experimental evolution in multiple clonally derived cultures over 2 years. TP53 loss elicited progressive...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;20
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256941"
                        open
                      >
                        <summary class="article-expander__title">FIRRM/C1orf112 mediates resolution of homologous recombination intermediates in response to DNA interstrand crosslinks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256941/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA interstrand crosslinks (ICLs) pose a major obstacle for DNA replication and transcription if left unrepaired. The cellular response to ICLs requires the coordination of various DNA repair mechanisms. Homologous recombination (HR) intermediates generated in response to ICLs, require efficient and timely conversion by structure-selective endonucleases. Our knowledge on the precise coordination of this process remains incomplete. Here, we designed complementary genetic screens to map the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256347"
                        open
                      >
                        <summary class="article-expander__title">Genetic heterogeneity in p53-null leukemia increases transiently with spindle assembly checkpoint inhibition and is not rescued by p53</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256347/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome gains or losses often lead to copy number variations (CNV) and loss of heterozygosity (LOH). Both quantities are low in hematologic &quot;liquid&quot; cancers versus solid tumors in data of The Cancer Genome Atlas (TCGA) that also shows the fraction of a genome affected by LOH is ~ one-half of that with CNV. Suspension cultures of p53-null THP-1 leukemia-derived cells conform to these trends, despite novel evidence here of genetic heterogeneity and transiently elevated CNV after perturbation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37258665"
                        open
                      >
                        <summary class="article-expander__title">Deterministic evolution and stringent selection during preneoplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37258665/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The earliest events during human tumour initiation, although poorly characterized, may hold clues to malignancy detection and prevention¹. Here we model occult preneoplasia by biallelic inactivation of TP53, a common early event in gastric cancer, in human gastric organoids. Causal relationships between this initiating genetic lesion and resulting phenotypes were established using experimental evolution in multiple clonally derived cultures over 2 years. TP53 loss elicited progressive...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;20
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256941"
                        open
                      >
                        <summary class="article-expander__title">FIRRM/C1orf112 mediates resolution of homologous recombination intermediates in response to DNA interstrand crosslinks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256941/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA interstrand crosslinks (ICLs) pose a major obstacle for DNA replication and transcription if left unrepaired. The cellular response to ICLs requires the coordination of various DNA repair mechanisms. Homologous recombination (HR) intermediates generated in response to ICLs, require efficient and timely conversion by structure-selective endonucleases. Our knowledge on the precise coordination of this process remains incomplete. Here, we designed complementary genetic screens to map the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256347"
                        open
                      >
                        <summary class="article-expander__title">Genetic heterogeneity in p53-null leukemia increases transiently with spindle assembly checkpoint inhibition and is not rescued by p53</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256347/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome gains or losses often lead to copy number variations (CNV) and loss of heterozygosity (LOH). Both quantities are low in hematologic &quot;liquid&quot; cancers versus solid tumors in data of The Cancer Genome Atlas (TCGA) that also shows the fraction of a genome affected by LOH is ~ one-half of that with CNV. Suspension cultures of p53-null THP-1 leukemia-derived cells conform to these trends, despite novel evidence here of genetic heterogeneity and transiently elevated CNV after perturbation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37258665"
                        open
                      >
                        <summary class="article-expander__title">Deterministic evolution and stringent selection during preneoplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37258665/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The earliest events during human tumour initiation, although poorly characterized, may hold clues to malignancy detection and prevention¹. Here we model occult preneoplasia by biallelic inactivation of TP53, a common early event in gastric cancer, in human gastric organoids. Causal relationships between this initiating genetic lesion and resulting phenotypes were established using experimental evolution in multiple clonally derived cultures over 2 years. TP53 loss elicited progressive...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;20
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256941"
                        open
                      >
                        <summary class="article-expander__title">FIRRM/C1orf112 mediates resolution of homologous recombination intermediates in response to DNA interstrand crosslinks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256941/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA interstrand crosslinks (ICLs) pose a major obstacle for DNA replication and transcription if left unrepaired. The cellular response to ICLs requires the coordination of various DNA repair mechanisms. Homologous recombination (HR) intermediates generated in response to ICLs, require efficient and timely conversion by structure-selective endonucleases. Our knowledge on the precise coordination of this process remains incomplete. Here, we designed complementary genetic screens to map the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256347"
                        open
                      >
                        <summary class="article-expander__title">Genetic heterogeneity in p53-null leukemia increases transiently with spindle assembly checkpoint inhibition and is not rescued by p53</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256347/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome gains or losses often lead to copy number variations (CNV) and loss of heterozygosity (LOH). Both quantities are low in hematologic &quot;liquid&quot; cancers versus solid tumors in data of The Cancer Genome Atlas (TCGA) that also shows the fraction of a genome affected by LOH is ~ one-half of that with CNV. Suspension cultures of p53-null THP-1 leukemia-derived cells conform to these trends, despite novel evidence here of genetic heterogeneity and transiently elevated CNV after perturbation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37258665"
                        open
                      >
                        <summary class="article-expander__title">Deterministic evolution and stringent selection during preneoplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37258665/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The earliest events during human tumour initiation, although poorly characterized, may hold clues to malignancy detection and prevention¹. Here we model occult preneoplasia by biallelic inactivation of TP53, a common early event in gastric cancer, in human gastric organoids. Causal relationships between this initiating genetic lesion and resulting phenotypes were established using experimental evolution in multiple clonally derived cultures over 2 years. TP53 loss elicited progressive...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;20
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256941"
                        open
                      >
                        <summary class="article-expander__title">FIRRM/C1orf112 mediates resolution of homologous recombination intermediates in response to DNA interstrand crosslinks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256941/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA interstrand crosslinks (ICLs) pose a major obstacle for DNA replication and transcription if left unrepaired. The cellular response to ICLs requires the coordination of various DNA repair mechanisms. Homologous recombination (HR) intermediates generated in response to ICLs, require efficient and timely conversion by structure-selective endonucleases. Our knowledge on the precise coordination of this process remains incomplete. Here, we designed complementary genetic screens to map the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256347"
                        open
                      >
                        <summary class="article-expander__title">Genetic heterogeneity in p53-null leukemia increases transiently with spindle assembly checkpoint inhibition and is not rescued by p53</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256347/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome gains or losses often lead to copy number variations (CNV) and loss of heterozygosity (LOH). Both quantities are low in hematologic &quot;liquid&quot; cancers versus solid tumors in data of The Cancer Genome Atlas (TCGA) that also shows the fraction of a genome affected by LOH is ~ one-half of that with CNV. Suspension cultures of p53-null THP-1 leukemia-derived cells conform to these trends, despite novel evidence here of genetic heterogeneity and transiently elevated CNV after perturbation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37258665"
                        open
                      >
                        <summary class="article-expander__title">Deterministic evolution and stringent selection during preneoplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37258665/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The earliest events during human tumour initiation, although poorly characterized, may hold clues to malignancy detection and prevention¹. Here we model occult preneoplasia by biallelic inactivation of TP53, a common early event in gastric cancer, in human gastric organoids. Causal relationships between this initiating genetic lesion and resulting phenotypes were established using experimental evolution in multiple clonally derived cultures over 2 years. TP53 loss elicited progressive...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;20
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256941"
                        open
                      >
                        <summary class="article-expander__title">FIRRM/C1orf112 mediates resolution of homologous recombination intermediates in response to DNA interstrand crosslinks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256941/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA interstrand crosslinks (ICLs) pose a major obstacle for DNA replication and transcription if left unrepaired. The cellular response to ICLs requires the coordination of various DNA repair mechanisms. Homologous recombination (HR) intermediates generated in response to ICLs, require efficient and timely conversion by structure-selective endonucleases. Our knowledge on the precise coordination of this process remains incomplete. Here, we designed complementary genetic screens to map the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256347"
                        open
                      >
                        <summary class="article-expander__title">Genetic heterogeneity in p53-null leukemia increases transiently with spindle assembly checkpoint inhibition and is not rescued by p53</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256347/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome gains or losses often lead to copy number variations (CNV) and loss of heterozygosity (LOH). Both quantities are low in hematologic &quot;liquid&quot; cancers versus solid tumors in data of The Cancer Genome Atlas (TCGA) that also shows the fraction of a genome affected by LOH is ~ one-half of that with CNV. Suspension cultures of p53-null THP-1 leukemia-derived cells conform to these trends, despite novel evidence here of genetic heterogeneity and transiently elevated CNV after perturbation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37258665"
                        open
                      >
                        <summary class="article-expander__title">Deterministic evolution and stringent selection during preneoplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37258665/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The earliest events during human tumour initiation, although poorly characterized, may hold clues to malignancy detection and prevention¹. Here we model occult preneoplasia by biallelic inactivation of TP53, a common early event in gastric cancer, in human gastric organoids. Causal relationships between this initiating genetic lesion and resulting phenotypes were established using experimental evolution in multiple clonally derived cultures over 2 years. TP53 loss elicited progressive...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;20
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256941"
                        open
                      >
                        <summary class="article-expander__title">FIRRM/C1orf112 mediates resolution of homologous recombination intermediates in response to DNA interstrand crosslinks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256941/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA interstrand crosslinks (ICLs) pose a major obstacle for DNA replication and transcription if left unrepaired. The cellular response to ICLs requires the coordination of various DNA repair mechanisms. Homologous recombination (HR) intermediates generated in response to ICLs, require efficient and timely conversion by structure-selective endonucleases. Our knowledge on the precise coordination of this process remains incomplete. Here, we designed complementary genetic screens to map the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256347"
                        open
                      >
                        <summary class="article-expander__title">Genetic heterogeneity in p53-null leukemia increases transiently with spindle assembly checkpoint inhibition and is not rescued by p53</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256347/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome gains or losses often lead to copy number variations (CNV) and loss of heterozygosity (LOH). Both quantities are low in hematologic &quot;liquid&quot; cancers versus solid tumors in data of The Cancer Genome Atlas (TCGA) that also shows the fraction of a genome affected by LOH is ~ one-half of that with CNV. Suspension cultures of p53-null THP-1 leukemia-derived cells conform to these trends, despite novel evidence here of genetic heterogeneity and transiently elevated CNV after perturbation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37258665"
                        open
                      >
                        <summary class="article-expander__title">Deterministic evolution and stringent selection during preneoplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37258665/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The earliest events during human tumour initiation, although poorly characterized, may hold clues to malignancy detection and prevention¹. Here we model occult preneoplasia by biallelic inactivation of TP53, a common early event in gastric cancer, in human gastric organoids. Causal relationships between this initiating genetic lesion and resulting phenotypes were established using experimental evolution in multiple clonally derived cultures over 2 years. TP53 loss elicited progressive...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;20
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256941"
                        open
                      >
                        <summary class="article-expander__title">FIRRM/C1orf112 mediates resolution of homologous recombination intermediates in response to DNA interstrand crosslinks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256941/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA interstrand crosslinks (ICLs) pose a major obstacle for DNA replication and transcription if left unrepaired. The cellular response to ICLs requires the coordination of various DNA repair mechanisms. Homologous recombination (HR) intermediates generated in response to ICLs, require efficient and timely conversion by structure-selective endonucleases. Our knowledge on the precise coordination of this process remains incomplete. Here, we designed complementary genetic screens to map the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256347"
                        open
                      >
                        <summary class="article-expander__title">Genetic heterogeneity in p53-null leukemia increases transiently with spindle assembly checkpoint inhibition and is not rescued by p53</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256347/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome gains or losses often lead to copy number variations (CNV) and loss of heterozygosity (LOH). Both quantities are low in hematologic &quot;liquid&quot; cancers versus solid tumors in data of The Cancer Genome Atlas (TCGA) that also shows the fraction of a genome affected by LOH is ~ one-half of that with CNV. Suspension cultures of p53-null THP-1 leukemia-derived cells conform to these trends, despite novel evidence here of genetic heterogeneity and transiently elevated CNV after perturbation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37258665"
                        open
                      >
                        <summary class="article-expander__title">Deterministic evolution and stringent selection during preneoplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37258665/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The earliest events during human tumour initiation, although poorly characterized, may hold clues to malignancy detection and prevention¹. Here we model occult preneoplasia by biallelic inactivation of TP53, a common early event in gastric cancer, in human gastric organoids. Causal relationships between this initiating genetic lesion and resulting phenotypes were established using experimental evolution in multiple clonally derived cultures over 2 years. TP53 loss elicited progressive...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256941"
                        open
                      >
                        <summary class="article-expander__title">FIRRM/C1orf112 mediates resolution of homologous recombination intermediates in response to DNA interstrand crosslinks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256941/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA interstrand crosslinks (ICLs) pose a major obstacle for DNA replication and transcription if left unrepaired. The cellular response to ICLs requires the coordination of various DNA repair mechanisms. Homologous recombination (HR) intermediates generated in response to ICLs, require efficient and timely conversion by structure-selective endonucleases. Our knowledge on the precise coordination of this process remains incomplete. Here, we designed complementary genetic screens to map the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256347"
                        open
                      >
                        <summary class="article-expander__title">Genetic heterogeneity in p53-null leukemia increases transiently with spindle assembly checkpoint inhibition and is not rescued by p53</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256347/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome gains or losses often lead to copy number variations (CNV) and loss of heterozygosity (LOH). Both quantities are low in hematologic &quot;liquid&quot; cancers versus solid tumors in data of The Cancer Genome Atlas (TCGA) that also shows the fraction of a genome affected by LOH is ~ one-half of that with CNV. Suspension cultures of p53-null THP-1 leukemia-derived cells conform to these trends, despite novel evidence here of genetic heterogeneity and transiently elevated CNV after perturbation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37258665"
                        open
                      >
                        <summary class="article-expander__title">Deterministic evolution and stringent selection during preneoplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37258665/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The earliest events during human tumour initiation, although poorly characterized, may hold clues to malignancy detection and prevention¹. Here we model occult preneoplasia by biallelic inactivation of TP53, a common early event in gastric cancer, in human gastric organoids. Causal relationships between this initiating genetic lesion and resulting phenotypes were established using experimental evolution in multiple clonally derived cultures over 2 years. TP53 loss elicited progressive...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256941"
                        open
                      >
                        <summary class="article-expander__title">FIRRM/C1orf112 mediates resolution of homologous recombination intermediates in response to DNA interstrand crosslinks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256941/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA interstrand crosslinks (ICLs) pose a major obstacle for DNA replication and transcription if left unrepaired. The cellular response to ICLs requires the coordination of various DNA repair mechanisms. Homologous recombination (HR) intermediates generated in response to ICLs, require efficient and timely conversion by structure-selective endonucleases. Our knowledge on the precise coordination of this process remains incomplete. Here, we designed complementary genetic screens to map the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256347"
                        open
                      >
                        <summary class="article-expander__title">Genetic heterogeneity in p53-null leukemia increases transiently with spindle assembly checkpoint inhibition and is not rescued by p53</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256347/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome gains or losses often lead to copy number variations (CNV) and loss of heterozygosity (LOH). Both quantities are low in hematologic &quot;liquid&quot; cancers versus solid tumors in data of The Cancer Genome Atlas (TCGA) that also shows the fraction of a genome affected by LOH is ~ one-half of that with CNV. Suspension cultures of p53-null THP-1 leukemia-derived cells conform to these trends, despite novel evidence here of genetic heterogeneity and transiently elevated CNV after perturbation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37258665"
                        open
                      >
                        <summary class="article-expander__title">Deterministic evolution and stringent selection during preneoplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37258665/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The earliest events during human tumour initiation, although poorly characterized, may hold clues to malignancy detection and prevention¹. Here we model occult preneoplasia by biallelic inactivation of TP53, a common early event in gastric cancer, in human gastric organoids. Causal relationships between this initiating genetic lesion and resulting phenotypes were established using experimental evolution in multiple clonally derived cultures over 2 years. TP53 loss elicited progressive...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256941"
                        open
                      >
                        <summary class="article-expander__title">FIRRM/C1orf112 mediates resolution of homologous recombination intermediates in response to DNA interstrand crosslinks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256941/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA interstrand crosslinks (ICLs) pose a major obstacle for DNA replication and transcription if left unrepaired. The cellular response to ICLs requires the coordination of various DNA repair mechanisms. Homologous recombination (HR) intermediates generated in response to ICLs, require efficient and timely conversion by structure-selective endonucleases. Our knowledge on the precise coordination of this process remains incomplete. Here, we designed complementary genetic screens to map the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256347"
                        open
                      >
                        <summary class="article-expander__title">Genetic heterogeneity in p53-null leukemia increases transiently with spindle assembly checkpoint inhibition and is not rescued by p53</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256347/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome gains or losses often lead to copy number variations (CNV) and loss of heterozygosity (LOH). Both quantities are low in hematologic &quot;liquid&quot; cancers versus solid tumors in data of The Cancer Genome Atlas (TCGA) that also shows the fraction of a genome affected by LOH is ~ one-half of that with CNV. Suspension cultures of p53-null THP-1 leukemia-derived cells conform to these trends, despite novel evidence here of genetic heterogeneity and transiently elevated CNV after perturbation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37258665"
                        open
                      >
                        <summary class="article-expander__title">Deterministic evolution and stringent selection during preneoplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37258665/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The earliest events during human tumour initiation, although poorly characterized, may hold clues to malignancy detection and prevention¹. Here we model occult preneoplasia by biallelic inactivation of TP53, a common early event in gastric cancer, in human gastric organoids. Causal relationships between this initiating genetic lesion and resulting phenotypes were established using experimental evolution in multiple clonally derived cultures over 2 years. TP53 loss elicited progressive...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256941"
                        open
                      >
                        <summary class="article-expander__title">FIRRM/C1orf112 mediates resolution of homologous recombination intermediates in response to DNA interstrand crosslinks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256941/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA interstrand crosslinks (ICLs) pose a major obstacle for DNA replication and transcription if left unrepaired. The cellular response to ICLs requires the coordination of various DNA repair mechanisms. Homologous recombination (HR) intermediates generated in response to ICLs, require efficient and timely conversion by structure-selective endonucleases. Our knowledge on the precise coordination of this process remains incomplete. Here, we designed complementary genetic screens to map the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256347"
                        open
                      >
                        <summary class="article-expander__title">Genetic heterogeneity in p53-null leukemia increases transiently with spindle assembly checkpoint inhibition and is not rescued by p53</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256347/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome gains or losses often lead to copy number variations (CNV) and loss of heterozygosity (LOH). Both quantities are low in hematologic &quot;liquid&quot; cancers versus solid tumors in data of The Cancer Genome Atlas (TCGA) that also shows the fraction of a genome affected by LOH is ~ one-half of that with CNV. Suspension cultures of p53-null THP-1 leukemia-derived cells conform to these trends, despite novel evidence here of genetic heterogeneity and transiently elevated CNV after perturbation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37258665"
                        open
                      >
                        <summary class="article-expander__title">Deterministic evolution and stringent selection during preneoplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37258665/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The earliest events during human tumour initiation, although poorly characterized, may hold clues to malignancy detection and prevention¹. Here we model occult preneoplasia by biallelic inactivation of TP53, a common early event in gastric cancer, in human gastric organoids. Causal relationships between this initiating genetic lesion and resulting phenotypes were established using experimental evolution in multiple clonally derived cultures over 2 years. TP53 loss elicited progressive...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256941"
                        open
                      >
                        <summary class="article-expander__title">FIRRM/C1orf112 mediates resolution of homologous recombination intermediates in response to DNA interstrand crosslinks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256941/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA interstrand crosslinks (ICLs) pose a major obstacle for DNA replication and transcription if left unrepaired. The cellular response to ICLs requires the coordination of various DNA repair mechanisms. Homologous recombination (HR) intermediates generated in response to ICLs, require efficient and timely conversion by structure-selective endonucleases. Our knowledge on the precise coordination of this process remains incomplete. Here, we designed complementary genetic screens to map the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256347"
                        open
                      >
                        <summary class="article-expander__title">Genetic heterogeneity in p53-null leukemia increases transiently with spindle assembly checkpoint inhibition and is not rescued by p53</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256347/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome gains or losses often lead to copy number variations (CNV) and loss of heterozygosity (LOH). Both quantities are low in hematologic &quot;liquid&quot; cancers versus solid tumors in data of The Cancer Genome Atlas (TCGA) that also shows the fraction of a genome affected by LOH is ~ one-half of that with CNV. Suspension cultures of p53-null THP-1 leukemia-derived cells conform to these trends, despite novel evidence here of genetic heterogeneity and transiently elevated CNV after perturbation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37258665"
                        open
                      >
                        <summary class="article-expander__title">Deterministic evolution and stringent selection during preneoplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37258665/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The earliest events during human tumour initiation, although poorly characterized, may hold clues to malignancy detection and prevention¹. Here we model occult preneoplasia by biallelic inactivation of TP53, a common early event in gastric cancer, in human gastric organoids. Causal relationships between this initiating genetic lesion and resulting phenotypes were established using experimental evolution in multiple clonally derived cultures over 2 years. TP53 loss elicited progressive...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256941"
                        open
                      >
                        <summary class="article-expander__title">FIRRM/C1orf112 mediates resolution of homologous recombination intermediates in response to DNA interstrand crosslinks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256941/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA interstrand crosslinks (ICLs) pose a major obstacle for DNA replication and transcription if left unrepaired. The cellular response to ICLs requires the coordination of various DNA repair mechanisms. Homologous recombination (HR) intermediates generated in response to ICLs, require efficient and timely conversion by structure-selective endonucleases. Our knowledge on the precise coordination of this process remains incomplete. Here, we designed complementary genetic screens to map the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256347"
                        open
                      >
                        <summary class="article-expander__title">Genetic heterogeneity in p53-null leukemia increases transiently with spindle assembly checkpoint inhibition and is not rescued by p53</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256347/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome gains or losses often lead to copy number variations (CNV) and loss of heterozygosity (LOH). Both quantities are low in hematologic &quot;liquid&quot; cancers versus solid tumors in data of The Cancer Genome Atlas (TCGA) that also shows the fraction of a genome affected by LOH is ~ one-half of that with CNV. Suspension cultures of p53-null THP-1 leukemia-derived cells conform to these trends, despite novel evidence here of genetic heterogeneity and transiently elevated CNV after perturbation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37258665"
                        open
                      >
                        <summary class="article-expander__title">Deterministic evolution and stringent selection during preneoplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37258665/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The earliest events during human tumour initiation, although poorly characterized, may hold clues to malignancy detection and prevention¹. Here we model occult preneoplasia by biallelic inactivation of TP53, a common early event in gastric cancer, in human gastric organoids. Causal relationships between this initiating genetic lesion and resulting phenotypes were established using experimental evolution in multiple clonally derived cultures over 2 years. TP53 loss elicited progressive...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256941"
                        open
                      >
                        <summary class="article-expander__title">FIRRM/C1orf112 mediates resolution of homologous recombination intermediates in response to DNA interstrand crosslinks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256941/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA interstrand crosslinks (ICLs) pose a major obstacle for DNA replication and transcription if left unrepaired. The cellular response to ICLs requires the coordination of various DNA repair mechanisms. Homologous recombination (HR) intermediates generated in response to ICLs, require efficient and timely conversion by structure-selective endonucleases. Our knowledge on the precise coordination of this process remains incomplete. Here, we designed complementary genetic screens to map the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256347"
                        open
                      >
                        <summary class="article-expander__title">Genetic heterogeneity in p53-null leukemia increases transiently with spindle assembly checkpoint inhibition and is not rescued by p53</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256347/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome gains or losses often lead to copy number variations (CNV) and loss of heterozygosity (LOH). Both quantities are low in hematologic &quot;liquid&quot; cancers versus solid tumors in data of The Cancer Genome Atlas (TCGA) that also shows the fraction of a genome affected by LOH is ~ one-half of that with CNV. Suspension cultures of p53-null THP-1 leukemia-derived cells conform to these trends, despite novel evidence here of genetic heterogeneity and transiently elevated CNV after perturbation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37258665"
                        open
                      >
                        <summary class="article-expander__title">Deterministic evolution and stringent selection during preneoplasia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37258665/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The earliest events during human tumour initiation, although poorly characterized, may hold clues to malignancy detection and prevention¹. Here we model occult preneoplasia by biallelic inactivation of TP53, a common early event in gastric cancer, in human gastric organoids. Causal relationships between this initiating genetic lesion and resulting phenotypes were established using experimental evolution in multiple clonally derived cultures over 2 years. TP53 loss elicited progressive...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256941"
                        open
                      >
                        <summary class="article-expander__title">FIRRM/C1orf112 mediates resolution of homologous recombination intermediates in response to DNA interstrand crosslinks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256941/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA interstrand crosslinks (ICLs) pose a major obstacle for DNA replication and transcription if left unrepaired. The cellular response to ICLs requires the coordination of various DNA repair mechanisms. Homologous recombination (HR) intermediates generated in response to ICLs, require efficient and timely conversion by structure-selective endonucleases. Our knowledge on the precise coordination of this process remains incomplete. Here, we designed complementary genetic screens to map the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256347"
                        open
                      >
                        <summary class="article-expander__title">Genetic heterogeneity in p53-null leukemia increases transiently with spindle assembly checkpoint inhibition and is not rescued by p53</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256347/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome gains or losses often lead to copy number variations (CNV) and loss of heterozygosity (LOH). Both quantities are low in hematologic &quot;liquid&quot; cancers versus solid tumors in data of The Cancer Genome Atlas (TCGA) that also shows the fraction of a genome affected by LOH is ~ one-half of that with CNV. Suspension cultures of p53-null THP-1 leukemia-derived cells conform to these trends, despite novel evidence here of genetic heterogeneity and transiently elevated CNV after perturbation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256941"
                        open
                      >
                        <summary class="article-expander__title">FIRRM/C1orf112 mediates resolution of homologous recombination intermediates in response to DNA interstrand crosslinks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256941/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA interstrand crosslinks (ICLs) pose a major obstacle for DNA replication and transcription if left unrepaired. The cellular response to ICLs requires the coordination of various DNA repair mechanisms. Homologous recombination (HR) intermediates generated in response to ICLs, require efficient and timely conversion by structure-selective endonucleases. Our knowledge on the precise coordination of this process remains incomplete. Here, we designed complementary genetic screens to map the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37256347"
                        open
                      >
                        <summary class="article-expander__title">Genetic heterogeneity in p53-null leukemia increases transiently with spindle assembly checkpoint inhibition and is not rescued by p53</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37256347/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome gains or losses often lead to copy number variations (CNV) and loss of heterozygosity (LOH). Both quantities are low in hematologic &quot;liquid&quot; cancers versus solid tumors in data of The Cancer Genome Atlas (TCGA) that also shows the fraction of a genome affected by LOH is ~ one-half of that with CNV. Suspension cultures of p53-null THP-1 leukemia-derived cells conform to these trends, despite novel evidence here of genetic heterogeneity and transiently elevated CNV after perturbation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-05-29 datetime="2023-05-29T10:00:00.000Z">2023-05-29</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-05-29 datetime="2023-05-29T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37247756"
                        open
                      >
                        <summary class="article-expander__title">Nore1 inhibits age-associated myeloid lineage skewing and clonal hematopoiesis but facilitates termination of emergency (stress) granulopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37247756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230622211122&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated bone marrow changes include myeloid skewing and mutations that lead to clonal hematopoiesis. Molecular mechanisms for these events are ill defined, but decreased expression of Irf8/Icsbp (interferon regulatory factor 8/interferon consensus sequence binding protein) in aging hematopoietic stem cells may contribute. Irf8 functions as a leukemia suppressor for chronic myeloid leukemia, and young Irf8^(-/-) mice have neutrophilia with progression to acute myeloid leukemia (AML) with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37247756"
                        open
                      >
                        <summary class="article-expander__title">Nore1 inhibits age-associated myeloid lineage skewing and clonal hematopoiesis, but facilitates termination of emergency (stress) granulopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37247756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated bone marrow changes include myeloid skewing and mutations that lead to clonal hematopoiesis. Molecular mechanisms for these events are ill-defined, but decreased expression of Irf8/Icsbp (interferon regulatory factor 8/interferon consensus sequence binding protein) in aging hematopoietic stem cells (HSCs) may contribute. Irf8 functions as a leukemia suppressor for chronic myeloid leukemia (CML), and young Irf8^(-/-) mice have neutrophilia with progression to acute myeloid leukemia...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37247756"
                        open
                      >
                        <summary class="article-expander__title">Nore1 inhibits age-associated myeloid lineage skewing and clonal hematopoiesis, but facilitates termination of emergency (stress) granulopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37247756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated bone marrow changes include myeloid skewing and mutations that lead to clonal hematopoiesis. Molecular mechanisms for these events are ill-defined, but decreased expression of Irf8/Icsbp (interferon regulatory factor 8/interferon consensus sequence binding protein) in aging hematopoietic stem cells (HSCs) may contribute. Irf8 functions as a leukemia suppressor for chronic myeloid leukemia (CML), and young Irf8^(-/-) mice have neutrophilia with progression to acute myeloid leukemia...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37247756"
                        open
                      >
                        <summary class="article-expander__title">Nore1 inhibits age-associated myeloid lineage skewing and clonal hematopoiesis, but facilitates termination of emergency (stress) granulopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37247756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated bone marrow changes include myeloid skewing and mutations that lead to clonal hematopoiesis. Molecular mechanisms for these events are ill-defined, but decreased expression of Irf8/Icsbp (interferon regulatory factor 8/interferon consensus sequence binding protein) in aging hematopoietic stem cells (HSCs) may contribute. Irf8 functions as a leukemia suppressor for chronic myeloid leukemia (CML), and young Irf8^(-/-) mice have neutrophilia with progression to acute myeloid leukemia...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37247756"
                        open
                      >
                        <summary class="article-expander__title">Nore1 inhibits age-associated myeloid lineage skewing and clonal hematopoiesis, but facilitates termination of emergency (stress) granulopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37247756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated bone marrow changes include myeloid skewing and mutations that lead to clonal hematopoiesis. Molecular mechanisms for these events are ill-defined, but decreased expression of Irf8/Icsbp (interferon regulatory factor 8/interferon consensus sequence binding protein) in aging hematopoietic stem cells (HSCs) may contribute. Irf8 functions as a leukemia suppressor for chronic myeloid leukemia (CML), and young Irf8^(-/-) mice have neutrophilia with progression to acute myeloid leukemia...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37247756"
                        open
                      >
                        <summary class="article-expander__title">Nore1 inhibits age-associated myeloid lineage skewing and clonal hematopoiesis, but facilitates termination of emergency (stress) granulopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37247756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated bone marrow changes include myeloid skewing and mutations that lead to clonal hematopoiesis. Molecular mechanisms for these events are ill-defined, but decreased expression of Irf8/Icsbp (interferon regulatory factor 8/interferon consensus sequence binding protein) in aging hematopoietic stem cells (HSCs) may contribute. Irf8 functions as a leukemia suppressor for chronic myeloid leukemia (CML), and young Irf8^(-/-) mice have neutrophilia with progression to acute myeloid leukemia...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37247756"
                        open
                      >
                        <summary class="article-expander__title">Nore1 inhibits age-associated myeloid lineage skewing and clonal hematopoiesis, but facilitates termination of emergency (stress) granulopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37247756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated bone marrow changes include myeloid skewing and mutations that lead to clonal hematopoiesis. Molecular mechanisms for these events are ill-defined, but decreased expression of Irf8/Icsbp (interferon regulatory factor 8/interferon consensus sequence binding protein) in aging hematopoietic stem cells (HSCs) may contribute. Irf8 functions as a leukemia suppressor for chronic myeloid leukemia (CML), and young Irf8^(-/-) mice have neutrophilia with progression to acute myeloid leukemia...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37247756"
                        open
                      >
                        <summary class="article-expander__title">Nore1 inhibits age-associated myeloid lineage skewing and clonal hematopoiesis, but facilitates termination of emergency (stress) granulopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37247756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated bone marrow changes include myeloid skewing and mutations that lead to clonal hematopoiesis. Molecular mechanisms for these events are ill-defined, but decreased expression of Irf8/Icsbp (interferon regulatory factor 8/interferon consensus sequence binding protein) in aging hematopoietic stem cells (HSCs) may contribute. Irf8 functions as a leukemia suppressor for chronic myeloid leukemia (CML), and young Irf8^(-/-) mice have neutrophilia with progression to acute myeloid leukemia...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37247756"
                        open
                      >
                        <summary class="article-expander__title">Nore1 inhibits age-associated myeloid lineage skewing and clonal hematopoiesis, but facilitates termination of emergency (stress) granulopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37247756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated bone marrow changes include myeloid skewing and mutations that lead to clonal hematopoiesis. Molecular mechanisms for these events are ill-defined, but decreased expression of Irf8/Icsbp (interferon regulatory factor 8/interferon consensus sequence binding protein) in aging hematopoietic stem cells (HSCs) may contribute. Irf8 functions as a leukemia suppressor for chronic myeloid leukemia (CML), and young Irf8^(-/-) mice have neutrophilia with progression to acute myeloid leukemia...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37247756"
                        open
                      >
                        <summary class="article-expander__title">Nore1 inhibits age-associated myeloid lineage skewing and clonal hematopoiesis, but facilitates termination of emergency (stress) granulopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37247756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated bone marrow changes include myeloid skewing and mutations that lead to clonal hematopoiesis. Molecular mechanisms for these events are ill-defined, but decreased expression of Irf8/Icsbp (interferon regulatory factor 8/interferon consensus sequence binding protein) in aging hematopoietic stem cells (HSCs) may contribute. Irf8 functions as a leukemia suppressor for chronic myeloid leukemia (CML), and young Irf8^(-/-) mice have neutrophilia with progression to acute myeloid leukemia...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37247756"
                        open
                      >
                        <summary class="article-expander__title">Nore1 inhibits age-associated myeloid lineage skewing and clonal hematopoiesis, but facilitates termination of emergency (stress) granulopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37247756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated bone marrow changes include myeloid skewing and mutations that lead to clonal hematopoiesis. Molecular mechanisms for these events are ill-defined, but decreased expression of Irf8/Icsbp (interferon regulatory factor 8/interferon consensus sequence binding protein) in aging hematopoietic stem cells (HSCs) may contribute. Irf8 functions as a leukemia suppressor for chronic myeloid leukemia (CML), and young Irf8^(-/-) mice have neutrophilia with progression to acute myeloid leukemia...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37247756"
                        open
                      >
                        <summary class="article-expander__title">Nore1 inhibits age-associated myeloid lineage skewing and clonal hematopoiesis, but facilitates termination of emergency (stress) granulopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37247756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated bone marrow changes include myeloid skewing and mutations that lead to clonal hematopoiesis. Molecular mechanisms for these events are ill-defined, but decreased expression of Irf8/Icsbp (interferon regulatory factor 8/interferon consensus sequence binding protein) in aging hematopoietic stem cells (HSCs) may contribute. Irf8 functions as a leukemia suppressor for chronic myeloid leukemia (CML), and young Irf8^(-/-) mice have neutrophilia with progression to acute myeloid leukemia...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37247756"
                        open
                      >
                        <summary class="article-expander__title">Nore1 inhibits age-associated myeloid lineage skewing and clonal hematopoiesis, but facilitates termination of emergency (stress) granulopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37247756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated bone marrow changes include myeloid skewing and mutations that lead to clonal hematopoiesis. Molecular mechanisms for these events are ill-defined, but decreased expression of Irf8/Icsbp (interferon regulatory factor 8/interferon consensus sequence binding protein) in aging hematopoietic stem cells (HSCs) may contribute. Irf8 functions as a leukemia suppressor for chronic myeloid leukemia (CML), and young Irf8^(-/-) mice have neutrophilia with progression to acute myeloid leukemia...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37247756"
                        open
                      >
                        <summary class="article-expander__title">Nore1 inhibits age-associated myeloid lineage skewing and clonal hematopoiesis, but facilitates termination of emergency (stress) granulopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37247756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated bone marrow changes include myeloid skewing and mutations that lead to clonal hematopoiesis. Molecular mechanisms for these events are ill-defined, but decreased expression of Irf8/Icsbp (interferon regulatory factor 8/interferon consensus sequence binding protein) in aging hematopoietic stem cells (HSCs) may contribute. Irf8 functions as a leukemia suppressor for chronic myeloid leukemia (CML), and young Irf8^(-/-) mice have neutrophilia with progression to acute myeloid leukemia...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37247756"
                        open
                      >
                        <summary class="article-expander__title">Nore1 inhibits age-associated myeloid lineage skewing and clonal hematopoiesis, but facilitates termination of emergency (stress) granulopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37247756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated bone marrow changes include myeloid skewing and mutations that lead to clonal hematopoiesis. Molecular mechanisms for these events are ill-defined, but decreased expression of Irf8/Icsbp (interferon regulatory factor 8/interferon consensus sequence binding protein) in aging hematopoietic stem cells (HSCs) may contribute. Irf8 functions as a leukemia suppressor for chronic myeloid leukemia (CML), and young Irf8^(-/-) mice have neutrophilia with progression to acute myeloid leukemia...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37247756"
                        open
                      >
                        <summary class="article-expander__title">Nore1 inhibits age-associated myeloid lineage skewing and clonal hematopoiesis, but facilitates termination of emergency (stress) granulopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37247756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated bone marrow changes include myeloid skewing and mutations that lead to clonal hematopoiesis. Molecular mechanisms for these events are ill-defined, but decreased expression of Irf8/Icsbp (interferon regulatory factor 8/interferon consensus sequence binding protein) in aging hematopoietic stem cells (HSCs) may contribute. Irf8 functions as a leukemia suppressor for chronic myeloid leukemia (CML), and young Irf8^(-/-) mice have neutrophilia with progression to acute myeloid leukemia...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37247756"
                        open
                      >
                        <summary class="article-expander__title">Nore1 inhibits age-associated myeloid lineage skewing and clonal hematopoiesis, but facilitates termination of emergency (stress) granulopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37247756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated bone marrow changes include myeloid skewing and mutations that lead to clonal hematopoiesis. Molecular mechanisms for these events are ill-defined, but decreased expression of Irf8/Icsbp (interferon regulatory factor 8/interferon consensus sequence binding protein) in aging hematopoietic stem cells (HSCs) may contribute. Irf8 functions as a leukemia suppressor for chronic myeloid leukemia (CML), and young Irf8^(-/-) mice have neutrophilia with progression to acute myeloid leukemia...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37247756"
                        open
                      >
                        <summary class="article-expander__title">Nore1 inhibits age-associated myeloid lineage skewing and clonal hematopoiesis, but facilitates termination of emergency (stress) granulopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37247756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated bone marrow changes include myeloid skewing and mutations that lead to clonal hematopoiesis. Molecular mechanisms for these events are ill-defined, but decreased expression of Irf8/Icsbp (interferon regulatory factor 8/interferon consensus sequence binding protein) in aging hematopoietic stem cells (HSCs) may contribute. Irf8 functions as a leukemia suppressor for chronic myeloid leukemia (CML), and young Irf8^(-/-) mice have neutrophilia with progression to acute myeloid leukemia...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37247756"
                        open
                      >
                        <summary class="article-expander__title">Nore1 inhibits age-associated myeloid lineage skewing and clonal hematopoiesis, but facilitates termination of emergency (stress) granulopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37247756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated bone marrow changes include myeloid skewing and mutations that lead to clonal hematopoiesis. Molecular mechanisms for these events are ill-defined, but decreased expression of Irf8/Icsbp (interferon regulatory factor 8/interferon consensus sequence binding protein) in aging hematopoietic stem cells (HSCs) may contribute. Irf8 functions as a leukemia suppressor for chronic myeloid leukemia (CML), and young Irf8^(-/-) mice have neutrophilia with progression to acute myeloid leukemia...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37247756"
                        open
                      >
                        <summary class="article-expander__title">Nore1 inhibits age-associated myeloid lineage skewing and clonal hematopoiesis, but facilitates termination of emergency (stress) granulopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37247756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated bone marrow changes include myeloid skewing and mutations that lead to clonal hematopoiesis. Molecular mechanisms for these events are ill-defined, but decreased expression of Irf8/Icsbp (interferon regulatory factor 8/interferon consensus sequence binding protein) in aging hematopoietic stem cells (HSCs) may contribute. Irf8 functions as a leukemia suppressor for chronic myeloid leukemia (CML), and young Irf8^(-/-) mice have neutrophilia with progression to acute myeloid leukemia...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37247756"
                        open
                      >
                        <summary class="article-expander__title">Nore1 inhibits age-associated myeloid lineage skewing and clonal hematopoiesis, but facilitates termination of emergency (stress) granulopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37247756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated bone marrow changes include myeloid skewing and mutations that lead to clonal hematopoiesis. Molecular mechanisms for these events are ill-defined, but decreased expression of Irf8/Icsbp (interferon regulatory factor 8/interferon consensus sequence binding protein) in aging hematopoietic stem cells (HSCs) may contribute. Irf8 functions as a leukemia suppressor for chronic myeloid leukemia (CML), and young Irf8^(-/-) mice have neutrophilia with progression to acute myeloid leukemia...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37247756"
                        open
                      >
                        <summary class="article-expander__title">Nore1 inhibits age-associated myeloid lineage skewing and clonal hematopoiesis, but facilitates termination of emergency (stress) granulopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37247756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated bone marrow changes include myeloid skewing and mutations that lead to clonal hematopoiesis. Molecular mechanisms for these events are ill-defined, but decreased expression of Irf8/Icsbp (interferon regulatory factor 8/interferon consensus sequence binding protein) in aging hematopoietic stem cells (HSCs) may contribute. Irf8 functions as a leukemia suppressor for chronic myeloid leukemia (CML), and young Irf8^(-/-) mice have neutrophilia with progression to acute myeloid leukemia...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37247756"
                        open
                      >
                        <summary class="article-expander__title">Nore1 inhibits age-associated myeloid lineage skewing and clonal hematopoiesis, but facilitates termination of emergency (stress) granulopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37247756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated bone marrow changes include myeloid skewing and mutations that lead to clonal hematopoiesis. Molecular mechanisms for these events are ill-defined, but decreased expression of Irf8/Icsbp (interferon regulatory factor 8/interferon consensus sequence binding protein) in aging hematopoietic stem cells (HSCs) may contribute. Irf8 functions as a leukemia suppressor for chronic myeloid leukemia (CML), and young Irf8^(-/-) mice have neutrophilia with progression to acute myeloid leukemia...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37247756"
                        open
                      >
                        <summary class="article-expander__title">Nore1 inhibits age-associated myeloid lineage skewing and clonal hematopoiesis, but facilitates termination of emergency (stress) granulopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37247756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated bone marrow changes include myeloid skewing and mutations that lead to clonal hematopoiesis. Molecular mechanisms for these events are ill-defined, but decreased expression of Irf8/Icsbp (interferon regulatory factor 8/interferon consensus sequence binding protein) in aging hematopoietic stem cells (HSCs) may contribute. Irf8 functions as a leukemia suppressor for chronic myeloid leukemia (CML), and young Irf8^(-/-) mice have neutrophilia with progression to acute myeloid leukemia...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248571"
                        open
                      >
                        <summary class="article-expander__title">Latest Findings on the Mechanism of the Interaction Between Multiple Myeloma Cells and Bone Marrow Microenvironment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematologic malignancy of terminally differentiated plasma cells. The mechanisms of the pathogenesis and progression of MM include genetic abnormalities of the MM cells and the interaction between MM cells and bone marrow microenvironment (BMME). MM cells start malignant proliferation in BMME and contribute to the pathogenesis and progression of MM through direct or indirect interactions between cells and the extracellular matrix. Exploring the mechanism of interaction...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248571"
                        open
                      >
                        <summary class="article-expander__title">Latest Findings on the Mechanism of the Interaction Between Multiple Myeloma Cells and Bone Marrow Microenvironment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematologic malignancy of terminally differentiated plasma cells. The mechanisms of the pathogenesis and progression of MM include genetic abnormalities of the MM cells and the interaction between MM cells and bone marrow microenvironment (BMME). MM cells start malignant proliferation in BMME and contribute to the pathogenesis and progression of MM through direct or indirect interactions between cells and the extracellular matrix. Exploring the mechanism of interaction...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248149"
                        open
                      >
                        <summary class="article-expander__title">Clinical and translational relevance of intratumor heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248149/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity (ITH) is a driver of tumor evolution and a main cause of therapeutic resistance. Despite its importance, measures of ITH are still not incorporated into clinical practice. Consequently, standard treatment is frequently ineffective for patients with heterogeneous tumors as changes to treatment regimens are made only after recurrence and disease progression. More effective combination therapies require a mechanistic understanding of ITH and ways to assess it in clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248571"
                        open
                      >
                        <summary class="article-expander__title">Latest Findings on the Mechanism of the Interaction Between Multiple Myeloma Cells and Bone Marrow Microenvironment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematologic malignancy of terminally differentiated plasma cells. The mechanisms of the pathogenesis and progression of MM include genetic abnormalities of the MM cells and the interaction between MM cells and bone marrow microenvironment (BMME). MM cells start malignant proliferation in BMME and contribute to the pathogenesis and progression of MM through direct or indirect interactions between cells and the extracellular matrix. Exploring the mechanism of interaction...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248149"
                        open
                      >
                        <summary class="article-expander__title">Clinical and translational relevance of intratumor heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248149/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity (ITH) is a driver of tumor evolution and a main cause of therapeutic resistance. Despite its importance, measures of ITH are still not incorporated into clinical practice. Consequently, standard treatment is frequently ineffective for patients with heterogeneous tumors as changes to treatment regimens are made only after recurrence and disease progression. More effective combination therapies require a mechanistic understanding of ITH and ways to assess it in clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248571"
                        open
                      >
                        <summary class="article-expander__title">Latest Findings on the Mechanism of the Interaction Between Multiple Myeloma Cells and Bone Marrow Microenvironment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematologic malignancy of terminally differentiated plasma cells. The mechanisms of the pathogenesis and progression of MM include genetic abnormalities of the MM cells and the interaction between MM cells and bone marrow microenvironment (BMME). MM cells start malignant proliferation in BMME and contribute to the pathogenesis and progression of MM through direct or indirect interactions between cells and the extracellular matrix. Exploring the mechanism of interaction...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248149"
                        open
                      >
                        <summary class="article-expander__title">Clinical and translational relevance of intratumor heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248149/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity (ITH) is a driver of tumor evolution and a main cause of therapeutic resistance. Despite its importance, measures of ITH are still not incorporated into clinical practice. Consequently, standard treatment is frequently ineffective for patients with heterogeneous tumors as changes to treatment regimens are made only after recurrence and disease progression. More effective combination therapies require a mechanistic understanding of ITH and ways to assess it in clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248571"
                        open
                      >
                        <summary class="article-expander__title">Latest Findings on the Mechanism of the Interaction Between Multiple Myeloma Cells and Bone Marrow Microenvironment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematologic malignancy of terminally differentiated plasma cells. The mechanisms of the pathogenesis and progression of MM include genetic abnormalities of the MM cells and the interaction between MM cells and bone marrow microenvironment (BMME). MM cells start malignant proliferation in BMME and contribute to the pathogenesis and progression of MM through direct or indirect interactions between cells and the extracellular matrix. Exploring the mechanism of interaction...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248149"
                        open
                      >
                        <summary class="article-expander__title">Clinical and translational relevance of intratumor heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248149/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity (ITH) is a driver of tumor evolution and a main cause of therapeutic resistance. Despite its importance, measures of ITH are still not incorporated into clinical practice. Consequently, standard treatment is frequently ineffective for patients with heterogeneous tumors as changes to treatment regimens are made only after recurrence and disease progression. More effective combination therapies require a mechanistic understanding of ITH and ways to assess it in clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248571"
                        open
                      >
                        <summary class="article-expander__title">Latest Findings on the Mechanism of the Interaction Between Multiple Myeloma Cells and Bone Marrow Microenvironment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematologic malignancy of terminally differentiated plasma cells. The mechanisms of the pathogenesis and progression of MM include genetic abnormalities of the MM cells and the interaction between MM cells and bone marrow microenvironment (BMME). MM cells start malignant proliferation in BMME and contribute to the pathogenesis and progression of MM through direct or indirect interactions between cells and the extracellular matrix. Exploring the mechanism of interaction...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248149"
                        open
                      >
                        <summary class="article-expander__title">Clinical and translational relevance of intratumor heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248149/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity (ITH) is a driver of tumor evolution and a main cause of therapeutic resistance. Despite its importance, measures of ITH are still not incorporated into clinical practice. Consequently, standard treatment is frequently ineffective for patients with heterogeneous tumors as changes to treatment regimens are made only after recurrence and disease progression. More effective combination therapies require a mechanistic understanding of ITH and ways to assess it in clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248571"
                        open
                      >
                        <summary class="article-expander__title">Latest Findings on the Mechanism of the Interaction Between Multiple Myeloma Cells and Bone Marrow Microenvironment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematologic malignancy of terminally differentiated plasma cells. The mechanisms of the pathogenesis and progression of MM include genetic abnormalities of the MM cells and the interaction between MM cells and bone marrow microenvironment (BMME). MM cells start malignant proliferation in BMME and contribute to the pathogenesis and progression of MM through direct or indirect interactions between cells and the extracellular matrix. Exploring the mechanism of interaction...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248149"
                        open
                      >
                        <summary class="article-expander__title">Clinical and translational relevance of intratumor heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248149/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity (ITH) is a driver of tumor evolution and a main cause of therapeutic resistance. Despite its importance, measures of ITH are still not incorporated into clinical practice. Consequently, standard treatment is frequently ineffective for patients with heterogeneous tumors as changes to treatment regimens are made only after recurrence and disease progression. More effective combination therapies require a mechanistic understanding of ITH and ways to assess it in clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248571"
                        open
                      >
                        <summary class="article-expander__title">Latest Findings on the Mechanism of the Interaction Between Multiple Myeloma Cells and Bone Marrow Microenvironment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematologic malignancy of terminally differentiated plasma cells. The mechanisms of the pathogenesis and progression of MM include genetic abnormalities of the MM cells and the interaction between MM cells and bone marrow microenvironment (BMME). MM cells start malignant proliferation in BMME and contribute to the pathogenesis and progression of MM through direct or indirect interactions between cells and the extracellular matrix. Exploring the mechanism of interaction...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248149"
                        open
                      >
                        <summary class="article-expander__title">Clinical and translational relevance of intratumor heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248149/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity (ITH) is a driver of tumor evolution and a main cause of therapeutic resistance. Despite its importance, measures of ITH are still not incorporated into clinical practice. Consequently, standard treatment is frequently ineffective for patients with heterogeneous tumors as changes to treatment regimens are made only after recurrence and disease progression. More effective combination therapies require a mechanistic understanding of ITH and ways to assess it in clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248571"
                        open
                      >
                        <summary class="article-expander__title">Latest Findings on the Mechanism of the Interaction Between Multiple Myeloma Cells and Bone Marrow Microenvironment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematologic malignancy of terminally differentiated plasma cells. The mechanisms of the pathogenesis and progression of MM include genetic abnormalities of the MM cells and the interaction between MM cells and bone marrow microenvironment (BMME). MM cells start malignant proliferation in BMME and contribute to the pathogenesis and progression of MM through direct or indirect interactions between cells and the extracellular matrix. Exploring the mechanism of interaction...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248149"
                        open
                      >
                        <summary class="article-expander__title">Clinical and translational relevance of intratumor heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248149/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity (ITH) is a driver of tumor evolution and a main cause of therapeutic resistance. Despite its importance, measures of ITH are still not incorporated into clinical practice. Consequently, standard treatment is frequently ineffective for patients with heterogeneous tumors as changes to treatment regimens are made only after recurrence and disease progression. More effective combination therapies require a mechanistic understanding of ITH and ways to assess it in clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248571"
                        open
                      >
                        <summary class="article-expander__title">Latest Findings on the Mechanism of the Interaction Between Multiple Myeloma Cells and Bone Marrow Microenvironment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematologic malignancy of terminally differentiated plasma cells. The mechanisms of the pathogenesis and progression of MM include genetic abnormalities of the MM cells and the interaction between MM cells and bone marrow microenvironment (BMME). MM cells start malignant proliferation in BMME and contribute to the pathogenesis and progression of MM through direct or indirect interactions between cells and the extracellular matrix. Exploring the mechanism of interaction...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248149"
                        open
                      >
                        <summary class="article-expander__title">Clinical and translational relevance of intratumor heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248149/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity (ITH) is a driver of tumor evolution and a main cause of therapeutic resistance. Despite its importance, measures of ITH are still not incorporated into clinical practice. Consequently, standard treatment is frequently ineffective for patients with heterogeneous tumors as changes to treatment regimens are made only after recurrence and disease progression. More effective combination therapies require a mechanistic understanding of ITH and ways to assess it in clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248571"
                        open
                      >
                        <summary class="article-expander__title">Latest Findings on the Mechanism of the Interaction Between Multiple Myeloma Cells and Bone Marrow Microenvironment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematologic malignancy of terminally differentiated plasma cells. The mechanisms of the pathogenesis and progression of MM include genetic abnormalities of the MM cells and the interaction between MM cells and bone marrow microenvironment (BMME). MM cells start malignant proliferation in BMME and contribute to the pathogenesis and progression of MM through direct or indirect interactions between cells and the extracellular matrix. Exploring the mechanism of interaction...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248149"
                        open
                      >
                        <summary class="article-expander__title">Clinical and translational relevance of intratumor heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248149/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity (ITH) is a driver of tumor evolution and a main cause of therapeutic resistance. Despite its importance, measures of ITH are still not incorporated into clinical practice. Consequently, standard treatment is frequently ineffective for patients with heterogeneous tumors as changes to treatment regimens are made only after recurrence and disease progression. More effective combination therapies require a mechanistic understanding of ITH and ways to assess it in clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248571"
                        open
                      >
                        <summary class="article-expander__title">Latest Findings on the Mechanism of the Interaction Between Multiple Myeloma Cells and Bone Marrow Microenvironment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematologic malignancy of terminally differentiated plasma cells. The mechanisms of the pathogenesis and progression of MM include genetic abnormalities of the MM cells and the interaction between MM cells and bone marrow microenvironment (BMME). MM cells start malignant proliferation in BMME and contribute to the pathogenesis and progression of MM through direct or indirect interactions between cells and the extracellular matrix. Exploring the mechanism of interaction...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248149"
                        open
                      >
                        <summary class="article-expander__title">Clinical and translational relevance of intratumor heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248149/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity (ITH) is a driver of tumor evolution and a main cause of therapeutic resistance. Despite its importance, measures of ITH are still not incorporated into clinical practice. Consequently, standard treatment is frequently ineffective for patients with heterogeneous tumors as changes to treatment regimens are made only after recurrence and disease progression. More effective combination therapies require a mechanistic understanding of ITH and ways to assess it in clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248571"
                        open
                      >
                        <summary class="article-expander__title">Latest Findings on the Mechanism of the Interaction Between Multiple Myeloma Cells and Bone Marrow Microenvironment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematologic malignancy of terminally differentiated plasma cells. The mechanisms of the pathogenesis and progression of MM include genetic abnormalities of the MM cells and the interaction between MM cells and bone marrow microenvironment (BMME). MM cells start malignant proliferation in BMME and contribute to the pathogenesis and progression of MM through direct or indirect interactions between cells and the extracellular matrix. Exploring the mechanism of interaction...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248149"
                        open
                      >
                        <summary class="article-expander__title">Clinical and translational relevance of intratumor heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248149/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity (ITH) is a driver of tumor evolution and a main cause of therapeutic resistance. Despite its importance, measures of ITH are still not incorporated into clinical practice. Consequently, standard treatment is frequently ineffective for patients with heterogeneous tumors as changes to treatment regimens are made only after recurrence and disease progression. More effective combination therapies require a mechanistic understanding of ITH and ways to assess it in clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248571"
                        open
                      >
                        <summary class="article-expander__title">Latest Findings on the Mechanism of the Interaction Between Multiple Myeloma Cells and Bone Marrow Microenvironment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematologic malignancy of terminally differentiated plasma cells. The mechanisms of the pathogenesis and progression of MM include genetic abnormalities of the MM cells and the interaction between MM cells and bone marrow microenvironment (BMME). MM cells start malignant proliferation in BMME and contribute to the pathogenesis and progression of MM through direct or indirect interactions between cells and the extracellular matrix. Exploring the mechanism of interaction...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248149"
                        open
                      >
                        <summary class="article-expander__title">Clinical and translational relevance of intratumor heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248149/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity (ITH) is a driver of tumor evolution and a main cause of therapeutic resistance. Despite its importance, measures of ITH are still not incorporated into clinical practice. Consequently, standard treatment is frequently ineffective for patients with heterogeneous tumors as changes to treatment regimens are made only after recurrence and disease progression. More effective combination therapies require a mechanistic understanding of ITH and ways to assess it in clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248571"
                        open
                      >
                        <summary class="article-expander__title">Latest Findings on the Mechanism of the Interaction Between Multiple Myeloma Cells and Bone Marrow Microenvironment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematologic malignancy of terminally differentiated plasma cells. The mechanisms of the pathogenesis and progression of MM include genetic abnormalities of the MM cells and the interaction between MM cells and bone marrow microenvironment (BMME). MM cells start malignant proliferation in BMME and contribute to the pathogenesis and progression of MM through direct or indirect interactions between cells and the extracellular matrix. Exploring the mechanism of interaction...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248149"
                        open
                      >
                        <summary class="article-expander__title">Clinical and translational relevance of intratumor heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248149/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity (ITH) is a driver of tumor evolution and a main cause of therapeutic resistance. Despite its importance, measures of ITH are still not incorporated into clinical practice. Consequently, standard treatment is frequently ineffective for patients with heterogeneous tumors as changes to treatment regimens are made only after recurrence and disease progression. More effective combination therapies require a mechanistic understanding of ITH and ways to assess it in clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248571"
                        open
                      >
                        <summary class="article-expander__title">Latest Findings on the Mechanism of the Interaction Between Multiple Myeloma Cells and Bone Marrow Microenvironment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematologic malignancy of terminally differentiated plasma cells. The mechanisms of the pathogenesis and progression of MM include genetic abnormalities of the MM cells and the interaction between MM cells and bone marrow microenvironment (BMME). MM cells start malignant proliferation in BMME and contribute to the pathogenesis and progression of MM through direct or indirect interactions between cells and the extracellular matrix. Exploring the mechanism of interaction...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37248149"
                        open
                      >
                        <summary class="article-expander__title">Clinical and translational relevance of intratumor heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37248149/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity (ITH) is a driver of tumor evolution and a main cause of therapeutic resistance. Despite its importance, measures of ITH are still not incorporated into clinical practice. Consequently, standard treatment is frequently ineffective for patients with heterogeneous tumors as changes to treatment regimens are made only after recurrence and disease progression. More effective combination therapies require a mechanistic understanding of ITH and ways to assess it in clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-05-26 datetime="2023-05-26T10:00:00.000Z">2023-05-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-05-26 datetime="2023-05-26T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37232505"
                        open
                      >
                        <summary class="article-expander__title">Association between genetically determined telomere length and health-related outcomes: A systematic review and meta-analysis of Mendelian randomization studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37232505/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230621210458&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging evidence has shown that leukocyte telomere length (LTL) is associated with various health-related outcomes, while the causality of these associations remains unclear. We performed a systematic review and meta-analysis of current evidence from Mendelian randomization (MR) studies on the association between LTL and health-related outcomes. We searched PubMed, Embase, and Web of Science up to April 2022 to identify eligible MR studies. We graded the evidence level of each MR association...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37232505"
                        open
                      >
                        <summary class="article-expander__title">Association between genetically determined telomere length and health-related outcomes: A systematic review and meta-analysis of Mendelian randomization studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37232505/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging evidence has shown that leukocyte telomere length (LTL) is associated with various health-related outcomes, while the causality of these associations remains unclear. We performed a systematic review and meta-analysis of current evidence from Mendelian randomization (MR) studies on the association between LTL and health-related outcomes. We searched PubMed, Embase, and Web of Science up to April 2022 to identify eligible MR studies. We graded the evidence level of each MR association...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37232505"
                        open
                      >
                        <summary class="article-expander__title">Association between genetically determined telomere length and health-related outcomes: A systematic review and meta-analysis of Mendelian randomization studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37232505/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging evidence has shown that leukocyte telomere length (LTL) is associated with various health-related outcomes, while the causality of these associations remains unclear. We performed a systematic review and meta-analysis of current evidence from Mendelian randomization (MR) studies on the association between LTL and health-related outcomes. We searched PubMed, Embase, and Web of Science up to April 2022 to identify eligible MR studies. We graded the evidence level of each MR association...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37232505"
                        open
                      >
                        <summary class="article-expander__title">Association between genetically determined telomere length and health-related outcomes: A systematic review and meta-analysis of Mendelian randomization studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37232505/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging evidence has shown that leukocyte telomere length (LTL) is associated with various health-related outcomes, while the causality of these associations remains unclear. We performed a systematic review and meta-analysis of current evidence from Mendelian randomization (MR) studies on the association between LTL and health-related outcomes. We searched PubMed, Embase, and Web of Science up to April 2022 to identify eligible MR studies. We graded the evidence level of each MR association...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37232505"
                        open
                      >
                        <summary class="article-expander__title">Association between genetically determined telomere length and health-related outcomes: A systematic review and meta-analysis of Mendelian randomization studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37232505/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging evidence has shown that leukocyte telomere length (LTL) is associated with various health-related outcomes, while the causality of these associations remains unclear. We performed a systematic review and meta-analysis of current evidence from Mendelian randomization (MR) studies on the association between LTL and health-related outcomes. We searched PubMed, Embase, and Web of Science up to April 2022 to identify eligible MR studies. We graded the evidence level of each MR association...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37232505"
                        open
                      >
                        <summary class="article-expander__title">Association between genetically determined telomere length and health-related outcomes: A systematic review and meta-analysis of Mendelian randomization studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37232505/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging evidence has shown that leukocyte telomere length (LTL) is associated with various health-related outcomes, while the causality of these associations remains unclear. We performed a systematic review and meta-analysis of current evidence from Mendelian randomization (MR) studies on the association between LTL and health-related outcomes. We searched PubMed, Embase, and Web of Science up to April 2022 to identify eligible MR studies. We graded the evidence level of each MR association...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37232505"
                        open
                      >
                        <summary class="article-expander__title">Association between genetically determined telomere length and health-related outcomes: A systematic review and meta-analysis of Mendelian randomization studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37232505/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging evidence has shown that leukocyte telomere length (LTL) is associated with various health-related outcomes, while the causality of these associations remains unclear. We performed a systematic review and meta-analysis of current evidence from Mendelian randomization (MR) studies on the association between LTL and health-related outcomes. We searched PubMed, Embase, and Web of Science up to April 2022 to identify eligible MR studies. We graded the evidence level of each MR association...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37232505"
                        open
                      >
                        <summary class="article-expander__title">Association between genetically determined telomere length and health-related outcomes: A systematic review and meta-analysis of Mendelian randomization studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37232505/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging evidence has shown that leukocyte telomere length (LTL) is associated with various health-related outcomes, while the causality of these associations remains unclear. We performed a systematic review and meta-analysis of current evidence from Mendelian randomization (MR) studies on the association between LTL and health-related outcomes. We searched PubMed, Embase, and Web of Science up to April 2022 to identify eligible MR studies. We graded the evidence level of each MR association...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37232505"
                        open
                      >
                        <summary class="article-expander__title">Association between genetically determined telomere length and health-related outcomes: A systematic review and meta-analysis of Mendelian randomization studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37232505/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging evidence has shown that leukocyte telomere length (LTL) is associated with various health-related outcomes, while the causality of these associations remains unclear. We performed a systematic review and meta-analysis of current evidence from Mendelian randomization (MR) studies on the association between LTL and health-related outcomes. We searched PubMed, Embase, and Web of Science up to April 2022 to identify eligible MR studies. We graded the evidence level of each MR association...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37232505"
                        open
                      >
                        <summary class="article-expander__title">Association between genetically determined telomere length and health-related outcomes: A systematic review and meta-analysis of Mendelian randomization studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37232505/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging evidence has shown that leukocyte telomere length (LTL) is associated with various health-related outcomes, while the causality of these associations remains unclear. We performed a systematic review and meta-analysis of current evidence from Mendelian randomization (MR) studies on the association between LTL and health-related outcomes. We searched PubMed, Embase, and Web of Science up to April 2022 to identify eligible MR studies. We graded the evidence level of each MR association...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37232505"
                        open
                      >
                        <summary class="article-expander__title">Association between genetically determined telomere length and health-related outcomes: A systematic review and meta-analysis of Mendelian randomization studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37232505/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging evidence has shown that leukocyte telomere length (LTL) is associated with various health-related outcomes, while the causality of these associations remains unclear. We performed a systematic review and meta-analysis of current evidence from Mendelian randomization (MR) studies on the association between LTL and health-related outcomes. We searched PubMed, Embase, and Web of Science up to April 2022 to identify eligible MR studies. We graded the evidence level of each MR association...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37232505"
                        open
                      >
                        <summary class="article-expander__title">Association between genetically determined telomere length and health-related outcomes: A systematic review and meta-analysis of Mendelian randomization studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37232505/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging evidence has shown that leukocyte telomere length (LTL) is associated with various health-related outcomes, while the causality of these associations remains unclear. We performed a systematic review and meta-analysis of current evidence from Mendelian randomization (MR) studies on the association between LTL and health-related outcomes. We searched PubMed, Embase, and Web of Science up to April 2022 to identify eligible MR studies. We graded the evidence level of each MR association...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37232505"
                        open
                      >
                        <summary class="article-expander__title">Association between genetically determined telomere length and health-related outcomes: A systematic review and meta-analysis of Mendelian randomization studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37232505/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging evidence has shown that leukocyte telomere length (LTL) is associated with various health-related outcomes, while the causality of these associations remains unclear. We performed a systematic review and meta-analysis of current evidence from Mendelian randomization (MR) studies on the association between LTL and health-related outcomes. We searched PubMed, Embase, and Web of Science up to April 2022 to identify eligible MR studies. We graded the evidence level of each MR association...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37232505"
                        open
                      >
                        <summary class="article-expander__title">Association between genetically determined telomere length and health-related outcomes: A systematic review and meta-analysis of Mendelian randomization studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37232505/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging evidence has shown that leukocyte telomere length (LTL) is associated with various health-related outcomes, while the causality of these associations remains unclear. We performed a systematic review and meta-analysis of current evidence from Mendelian randomization (MR) studies on the association between LTL and health-related outcomes. We searched PubMed, Embase, and Web of Science up to April 2022 to identify eligible MR studies. We graded the evidence level of each MR association...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37232505"
                        open
                      >
                        <summary class="article-expander__title">Association between genetically determined telomere length and health-related outcomes: A systematic review and meta-analysis of Mendelian randomization studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37232505/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging evidence has shown that leukocyte telomere length (LTL) is associated with various health-related outcomes, while the causality of these associations remains unclear. We performed a systematic review and meta-analysis of current evidence from Mendelian randomization (MR) studies on the association between LTL and health-related outcomes. We searched PubMed, Embase, and Web of Science up to April 2022 to identify eligible MR studies. We graded the evidence level of each MR association...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37232505"
                        open
                      >
                        <summary class="article-expander__title">Association between genetically determined telomere length and health-related outcomes: A systematic review and meta-analysis of Mendelian randomization studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37232505/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging evidence has shown that leukocyte telomere length (LTL) is associated with various health-related outcomes, while the causality of these associations remains unclear. We performed a systematic review and meta-analysis of current evidence from Mendelian randomization (MR) studies on the association between LTL and health-related outcomes. We searched PubMed, Embase, and Web of Science up to April 2022 to identify eligible MR studies. We graded the evidence level of each MR association...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37232505"
                        open
                      >
                        <summary class="article-expander__title">Association between genetically determined telomere length and health-related outcomes: A systematic review and meta-analysis of Mendelian randomization studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37232505/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging evidence has shown that leukocyte telomere length (LTL) is associated with various health-related outcomes, while the causality of these associations remains unclear. We performed a systematic review and meta-analysis of current evidence from Mendelian randomization (MR) studies on the association between LTL and health-related outcomes. We searched PubMed, Embase, and Web of Science up to April 2022 to identify eligible MR studies. We graded the evidence level of each MR association...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37232505"
                        open
                      >
                        <summary class="article-expander__title">Association between genetically determined telomere length and health-related outcomes: A systematic review and meta-analysis of Mendelian randomization studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37232505/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging evidence has shown that leukocyte telomere length (LTL) is associated with various health-related outcomes, while the causality of these associations remains unclear. We performed a systematic review and meta-analysis of current evidence from Mendelian randomization (MR) studies on the association between LTL and health-related outcomes. We searched PubMed, Embase, and Web of Science up to April 2022 to identify eligible MR studies. We graded the evidence level of each MR association...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37232505"
                        open
                      >
                        <summary class="article-expander__title">Association between genetically determined telomere length and health-related outcomes: A systematic review and meta-analysis of Mendelian randomization studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37232505/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging evidence has shown that leukocyte telomere length (LTL) is associated with various health-related outcomes, while the causality of these associations remains unclear. We performed a systematic review and meta-analysis of current evidence from Mendelian randomization (MR) studies on the association between LTL and health-related outcomes. We searched PubMed, Embase, and Web of Science up to April 2022 to identify eligible MR studies. We graded the evidence level of each MR association...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37232505"
                        open
                      >
                        <summary class="article-expander__title">Association between genetically determined telomere length and health-related outcomes: A systematic review and meta-analysis of Mendelian randomization studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37232505/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging evidence has shown that leukocyte telomere length (LTL) is associated with various health-related outcomes, while the causality of these associations remains unclear. We performed a systematic review and meta-analysis of current evidence from Mendelian randomization (MR) studies on the association between LTL and health-related outcomes. We searched PubMed, Embase, and Web of Science up to April 2022 to identify eligible MR studies. We graded the evidence level of each MR association...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37232505"
                        open
                      >
                        <summary class="article-expander__title">Association between genetically determined telomere length and health-related outcomes: A systematic review and meta-analysis of Mendelian randomization studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37232505/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging evidence has shown that leukocyte telomere length (LTL) is associated with various health-related outcomes, while the causality of these associations remains unclear. We performed a systematic review and meta-analysis of current evidence from Mendelian randomization (MR) studies on the association between LTL and health-related outcomes. We searched PubMed, Embase, and Web of Science up to April 2022 to identify eligible MR studies. We graded the evidence level of each MR association...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37232505"
                        open
                      >
                        <summary class="article-expander__title">Association between genetically determined telomere length and health-related outcomes: A systematic review and meta-analysis of Mendelian randomization studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37232505/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging evidence has shown that leukocyte telomere length (LTL) is associated with various health-related outcomes, while the causality of these associations remains unclear. We performed a systematic review and meta-analysis of current evidence from Mendelian randomization (MR) studies on the association between LTL and health-related outcomes. We searched PubMed, Embase, and Web of Science up to April 2022 to identify eligible MR studies. We graded the evidence level of each MR association...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37232505"
                        open
                      >
                        <summary class="article-expander__title">Association between genetically determined telomere length and health-related outcomes: A systematic review and meta-analysis of Mendelian randomization studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37232505/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging evidence has shown that leukocyte telomere length (LTL) is associated with various health-related outcomes, while the causality of these associations remains unclear. We performed a systematic review and meta-analysis of current evidence from Mendelian randomization (MR) studies on the association between LTL and health-related outcomes. We searched PubMed, Embase, and Web of Science up to April 2022 to identify eligible MR studies. We graded the evidence level of each MR association...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37232505"
                        open
                      >
                        <summary class="article-expander__title">Association between genetically determined telomere length and health-related outcomes: A systematic review and meta-analysis of Mendelian randomization studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37232505/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging evidence has shown that leukocyte telomere length (LTL) is associated with various health-related outcomes, while the causality of these associations remains unclear. We performed a systematic review and meta-analysis of current evidence from Mendelian randomization (MR) studies on the association between LTL and health-related outcomes. We searched PubMed, Embase, and Web of Science up to April 2022 to identify eligible MR studies. We graded the evidence level of each MR association...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37232505"
                        open
                      >
                        <summary class="article-expander__title">Association between genetically determined telomere length and health-related outcomes: A systematic review and meta-analysis of Mendelian randomization studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37232505/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging evidence has shown that leukocyte telomere length (LTL) is associated with various health-related outcomes, while the causality of these associations remains unclear. We performed a systematic review and meta-analysis of current evidence from Mendelian randomization (MR) studies on the association between LTL and health-related outcomes. We searched PubMed, Embase, and Web of Science up to April 2022 to identify eligible MR studies. We graded the evidence level of each MR association...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37232505"
                        open
                      >
                        <summary class="article-expander__title">Association between genetically determined telomere length and health-related outcomes: A systematic review and meta-analysis of Mendelian randomization studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37232505/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230527210349&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging evidence has shown that leukocyte telomere length (LTL) is associated with various health-related outcomes, while the causality of these associations remains unclear. We performed a systematic review and meta-analysis of current evidence from Mendelian randomization (MR) studies on the association between LTL and health-related outcomes. We searched PubMed, Embase, and Web of Science up to April 2022 to identify eligible MR studies. We graded the evidence level of each MR association...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37232505"
                        open
                      >
                        <summary class="article-expander__title">Association between genetically determined telomere length and health-related outcomes: A systematic review and meta-analysis of Mendelian randomization studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37232505/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging evidence has shown that leukocyte telomere length (LTL) is associated with various health-related outcomes, while the causality of these associations remains unclear. We performed a systematic review and meta-analysis of current evidence from Mendelian randomization (MR) studies on the association between LTL and health-related outcomes. We searched PubMed, Embase, and Web of Science up to April 2022 to identify eligible MR studies. We graded the evidence level of each MR association...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37237325"
                        open
                      >
                        <summary class="article-expander__title">LEP promoter methylation in the initiation and progression of clonal cytopenia of undetermined significance and myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37237325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: In conclusion, hypermethylation of the LEP promoter is an early and frequent event in myeloid neoplasms and is associated with a worse prognosis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-05-25 datetime="2023-05-25T10:00:00.000Z">2023-05-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-05-25 datetime="2023-05-25T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37228237"
                        open
                      >
                        <summary class="article-expander__title">Inflammasomes and Atherosclerosis: a Mixed Picture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37228237/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230620210358&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The CANTOS (Canakinumab Anti-inflammatory Thrombosis Outcome Study) and colchicine trials suggest an important role of inflammasomes and their major product IL-1β (interleukin 1β) in human atherosclerotic cardiovascular disease. Moreover, studies in mouse models indicate a causal role of inflammasomes and IL-1β in atherosclerosis. However, recent studies have led to a more granular view of the role of inflammasomes in atherosclerosis. Studies in hyperlipidemic mouse models suggest that prominent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37228237"
                        open
                      >
                        <summary class="article-expander__title">Inflammasomes and Atherosclerosis: a Mixed Picture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37228237/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230619210151&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The CANTOS (Canakinumab Anti-inflammatory Thrombosis Outcome Study) and colchicine trials suggest an important role of inflammasomes and their major product IL-1β (interleukin 1β) in human atherosclerotic cardiovascular disease. Moreover, studies in mouse models indicate a causal role of inflammasomes and IL-1β in atherosclerosis. However, recent studies have led to a more granular view of the role of inflammasomes in atherosclerosis. Studies in hyperlipidemic mouse models suggest that prominent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37228237"
                        open
                      >
                        <summary class="article-expander__title">Inflammasomes and Atherosclerosis: a Mixed Picture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37228237/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The CANTOS (Canakinumab Anti-inflammatory Thrombosis Outcome Study) and colchicine trials suggest an important role of inflammasomes and their major product IL-1β (interleukin 1β) in human atherosclerotic cardiovascular disease. Moreover, studies in mouse models indicate a causal role of inflammasomes and IL-1β in atherosclerosis. However, recent studies have led to a more granular view of the role of inflammasomes in atherosclerosis. Studies in hyperlipidemic mouse models suggest that prominent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37228237"
                        open
                      >
                        <summary class="article-expander__title">Inflammasomes and Atherosclerosis: a Mixed Picture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37228237/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The CANTOS (Canakinumab Anti-inflammatory Thrombosis Outcome Study) and colchicine trials suggest an important role of inflammasomes and their major product IL-1β (interleukin 1β) in human atherosclerotic cardiovascular disease. Moreover, studies in mouse models indicate a causal role of inflammasomes and IL-1β in atherosclerosis. However, recent studies have led to a more granular view of the role of inflammasomes in atherosclerosis. Studies in hyperlipidemic mouse models suggest that prominent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37228237"
                        open
                      >
                        <summary class="article-expander__title">Inflammasomes and Atherosclerosis: a Mixed Picture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37228237/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The CANTOS (Canakinumab Anti-inflammatory Thrombosis Outcome Study) and colchicine trials suggest an important role of inflammasomes and their major product IL-1β (interleukin 1β) in human atherosclerotic cardiovascular disease. Moreover, studies in mouse models indicate a causal role of inflammasomes and IL-1β in atherosclerosis. However, recent studies have led to a more granular view of the role of inflammasomes in atherosclerosis. Studies in hyperlipidemic mouse models suggest that prominent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37228237"
                        open
                      >
                        <summary class="article-expander__title">Inflammasomes and Atherosclerosis: a Mixed Picture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37228237/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The CANTOS (Canakinumab Anti-inflammatory Thrombosis Outcome Study) and colchicine trials suggest an important role of inflammasomes and their major product IL-1β (interleukin 1β) in human atherosclerotic cardiovascular disease. Moreover, studies in mouse models indicate a causal role of inflammasomes and IL-1β in atherosclerosis. However, recent studies have led to a more granular view of the role of inflammasomes in atherosclerosis. Studies in hyperlipidemic mouse models suggest that prominent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37228237"
                        open
                      >
                        <summary class="article-expander__title">Inflammasomes and Atherosclerosis: a Mixed Picture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37228237/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The CANTOS (Canakinumab Anti-inflammatory Thrombosis Outcome Study) and colchicine trials suggest an important role of inflammasomes and their major product IL-1β (interleukin 1β) in human atherosclerotic cardiovascular disease. Moreover, studies in mouse models indicate a causal role of inflammasomes and IL-1β in atherosclerosis. However, recent studies have led to a more granular view of the role of inflammasomes in atherosclerosis. Studies in hyperlipidemic mouse models suggest that prominent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37228237"
                        open
                      >
                        <summary class="article-expander__title">Inflammasomes and Atherosclerosis: a Mixed Picture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37228237/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The CANTOS (Canakinumab Anti-inflammatory Thrombosis Outcome Study) and colchicine trials suggest an important role of inflammasomes and their major product IL-1β (interleukin 1β) in human atherosclerotic cardiovascular disease. Moreover, studies in mouse models indicate a causal role of inflammasomes and IL-1β in atherosclerosis. However, recent studies have led to a more granular view of the role of inflammasomes in atherosclerosis. Studies in hyperlipidemic mouse models suggest that prominent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37228237"
                        open
                      >
                        <summary class="article-expander__title">Inflammasomes and Atherosclerosis: a Mixed Picture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37228237/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The CANTOS (Canakinumab Anti-inflammatory Thrombosis Outcome Study) and colchicine trials suggest an important role of inflammasomes and their major product IL-1β (interleukin 1β) in human atherosclerotic cardiovascular disease. Moreover, studies in mouse models indicate a causal role of inflammasomes and IL-1β in atherosclerosis. However, recent studies have led to a more granular view of the role of inflammasomes in atherosclerosis. Studies in hyperlipidemic mouse models suggest that prominent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37228237"
                        open
                      >
                        <summary class="article-expander__title">Inflammasomes and Atherosclerosis: a Mixed Picture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37228237/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The CANTOS (Canakinumab Anti-inflammatory Thrombosis Outcome Study) and colchicine trials suggest an important role of inflammasomes and their major product IL-1β (interleukin 1β) in human atherosclerotic cardiovascular disease. Moreover, studies in mouse models indicate a causal role of inflammasomes and IL-1β in atherosclerosis. However, recent studies have led to a more granular view of the role of inflammasomes in atherosclerosis. Studies in hyperlipidemic mouse models suggest that prominent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37228237"
                        open
                      >
                        <summary class="article-expander__title">Inflammasomes and Atherosclerosis: a Mixed Picture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37228237/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The CANTOS (Canakinumab Anti-inflammatory Thrombosis Outcome Study) and colchicine trials suggest an important role of inflammasomes and their major product IL-1β (interleukin 1β) in human atherosclerotic cardiovascular disease. Moreover, studies in mouse models indicate a causal role of inflammasomes and IL-1β in atherosclerosis. However, recent studies have led to a more granular view of the role of inflammasomes in atherosclerosis. Studies in hyperlipidemic mouse models suggest that prominent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37228237"
                        open
                      >
                        <summary class="article-expander__title">Inflammasomes and Atherosclerosis: a Mixed Picture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37228237/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The CANTOS (Canakinumab Anti-inflammatory Thrombosis Outcome Study) and colchicine trials suggest an important role of inflammasomes and their major product IL-1β (interleukin 1β) in human atherosclerotic cardiovascular disease. Moreover, studies in mouse models indicate a causal role of inflammasomes and IL-1β in atherosclerosis. However, recent studies have led to a more granular view of the role of inflammasomes in atherosclerosis. Studies in hyperlipidemic mouse models suggest that prominent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37228237"
                        open
                      >
                        <summary class="article-expander__title">Inflammasomes and Atherosclerosis: a Mixed Picture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37228237/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The CANTOS (Canakinumab Anti-inflammatory Thrombosis Outcome Study) and colchicine trials suggest an important role of inflammasomes and their major product IL-1β (interleukin 1β) in human atherosclerotic cardiovascular disease. Moreover, studies in mouse models indicate a causal role of inflammasomes and IL-1β in atherosclerosis. However, recent studies have led to a more granular view of the role of inflammasomes in atherosclerosis. Studies in hyperlipidemic mouse models suggest that prominent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37228237"
                        open
                      >
                        <summary class="article-expander__title">Inflammasomes and Atherosclerosis: a Mixed Picture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37228237/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The CANTOS (Canakinumab Anti-inflammatory Thrombosis Outcome Study) and colchicine trials suggest an important role of inflammasomes and their major product IL-1β (interleukin 1β) in human atherosclerotic cardiovascular disease. Moreover, studies in mouse models indicate a causal role of inflammasomes and IL-1β in atherosclerosis. However, recent studies have led to a more granular view of the role of inflammasomes in atherosclerosis. Studies in hyperlipidemic mouse models suggest that prominent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37228237"
                        open
                      >
                        <summary class="article-expander__title">Inflammasomes and Atherosclerosis: a Mixed Picture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37228237/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The CANTOS (Canakinumab Anti-inflammatory Thrombosis Outcome Study) and colchicine trials suggest an important role of inflammasomes and their major product IL-1β (interleukin 1β) in human atherosclerotic cardiovascular disease. Moreover, studies in mouse models indicate a causal role of inflammasomes and IL-1β in atherosclerosis. However, recent studies have led to a more granular view of the role of inflammasomes in atherosclerosis. Studies in hyperlipidemic mouse models suggest that prominent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37228237"
                        open
                      >
                        <summary class="article-expander__title">Inflammasomes and Atherosclerosis: a Mixed Picture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37228237/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The CANTOS (Canakinumab Anti-inflammatory Thrombosis Outcome Study) and colchicine trials suggest an important role of inflammasomes and their major product IL-1β (interleukin 1β) in human atherosclerotic cardiovascular disease. Moreover, studies in mouse models indicate a causal role of inflammasomes and IL-1β in atherosclerosis. However, recent studies have led to a more granular view of the role of inflammasomes in atherosclerosis. Studies in hyperlipidemic mouse models suggest that prominent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37228237"
                        open
                      >
                        <summary class="article-expander__title">Inflammasomes and Atherosclerosis: a Mixed Picture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37228237/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The CANTOS (Canakinumab Anti-inflammatory Thrombosis Outcome Study) and colchicine trials suggest an important role of inflammasomes and their major product IL-1β (interleukin 1β) in human atherosclerotic cardiovascular disease. Moreover, studies in mouse models indicate a causal role of inflammasomes and IL-1β in atherosclerosis. However, recent studies have led to a more granular view of the role of inflammasomes in atherosclerosis. Studies in hyperlipidemic mouse models suggest that prominent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37228237"
                        open
                      >
                        <summary class="article-expander__title">Inflammasomes and Atherosclerosis: a Mixed Picture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37228237/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The CANTOS (Canakinumab Anti-inflammatory Thrombosis Outcome Study) and colchicine trials suggest an important role of inflammasomes and their major product IL-1β (interleukin 1β) in human atherosclerotic cardiovascular disease. Moreover, studies in mouse models indicate a causal role of inflammasomes and IL-1β in atherosclerosis. However, recent studies have led to a more granular view of the role of inflammasomes in atherosclerosis. Studies in hyperlipidemic mouse models suggest that prominent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37228237"
                        open
                      >
                        <summary class="article-expander__title">Inflammasomes and Atherosclerosis: a Mixed Picture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37228237/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The CANTOS (Canakinumab Anti-inflammatory Thrombosis Outcome Study) and colchicine trials suggest an important role of inflammasomes and their major product IL-1β (interleukin 1β) in human atherosclerotic cardiovascular disease. Moreover, studies in mouse models indicate a causal role of inflammasomes and IL-1β in atherosclerosis. However, recent studies have led to a more granular view of the role of inflammasomes in atherosclerosis. Studies in hyperlipidemic mouse models suggest that prominent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37228237"
                        open
                      >
                        <summary class="article-expander__title">Inflammasomes and Atherosclerosis: a Mixed Picture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37228237/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The CANTOS (Canakinumab Anti-inflammatory Thrombosis Outcome Study) and colchicine trials suggest an important role of inflammasomes and their major product IL-1β (interleukin 1β) in human atherosclerotic cardiovascular disease. Moreover, studies in mouse models indicate a causal role of inflammasomes and IL-1β in atherosclerosis. However, recent studies have led to a more granular view of the role of inflammasomes in atherosclerosis. Studies in hyperlipidemic mouse models suggest that prominent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37228237"
                        open
                      >
                        <summary class="article-expander__title">Inflammasomes and Atherosclerosis: a Mixed Picture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37228237/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The CANTOS (Canakinumab Anti-inflammatory Thrombosis Outcome Study) and colchicine trials suggest an important role of inflammasomes and their major product IL-1β (interleukin 1β) in human atherosclerotic cardiovascular disease. Moreover, studies in mouse models indicate a causal role of inflammasomes and IL-1β in atherosclerosis. However, recent studies have led to a more granular view of the role of inflammasomes in atherosclerosis. Studies in hyperlipidemic mouse models suggest that prominent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37228237"
                        open
                      >
                        <summary class="article-expander__title">Inflammasomes and Atherosclerosis: a Mixed Picture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37228237/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The CANTOS (Canakinumab Anti-inflammatory Thrombosis Outcome Study) and colchicine trials suggest an important role of inflammasomes and their major product IL-1β (interleukin 1β) in human atherosclerotic cardiovascular disease. Moreover, studies in mouse models indicate a causal role of inflammasomes and IL-1β in atherosclerosis. However, recent studies have led to a more granular view of the role of inflammasomes in atherosclerosis. Studies in hyperlipidemic mouse models suggest that prominent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37228237"
                        open
                      >
                        <summary class="article-expander__title">Inflammasomes and Atherosclerosis: a Mixed Picture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37228237/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The CANTOS (Canakinumab Anti-inflammatory Thrombosis Outcome Study) and colchicine trials suggest an important role of inflammasomes and their major product IL-1β (interleukin 1β) in human atherosclerotic cardiovascular disease. Moreover, studies in mouse models indicate a causal role of inflammasomes and IL-1β in atherosclerosis. However, recent studies have led to a more granular view of the role of inflammasomes in atherosclerosis. Studies in hyperlipidemic mouse models suggest that prominent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37228237"
                        open
                      >
                        <summary class="article-expander__title">Inflammasomes and Atherosclerosis: a Mixed Picture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37228237/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The CANTOS (Canakinumab Anti-inflammatory Thrombosis Outcome Study) and colchicine trials suggest an important role of inflammasomes and their major product IL-1β (interleukin 1β) in human atherosclerotic cardiovascular disease. Moreover, studies in mouse models indicate a causal role of inflammasomes and IL-1β in atherosclerosis. However, recent studies have led to a more granular view of the role of inflammasomes in atherosclerosis. Studies in hyperlipidemic mouse models suggest that prominent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37228237"
                        open
                      >
                        <summary class="article-expander__title">Inflammasomes and Atherosclerosis: a Mixed Picture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37228237/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230527210349&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The CANTOS (Canakinumab Anti-inflammatory Thrombosis Outcome Study) and colchicine trials suggest an important role of inflammasomes and their major product IL-1β (interleukin 1β) in human atherosclerotic cardiovascular disease. Moreover, studies in mouse models indicate a causal role of inflammasomes and IL-1β in atherosclerosis. However, recent studies have led to a more granular view of the role of inflammasomes in atherosclerosis. Studies in hyperlipidemic mouse models suggest that prominent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37228237"
                        open
                      >
                        <summary class="article-expander__title">Inflammasomes and Atherosclerosis: a Mixed Picture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37228237/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The CANTOS (Canakinumab Anti-inflammatory Thrombosis Outcome Study) and colchicine trials suggest an important role of inflammasomes and their major product IL-1β (interleukin 1β) in human atherosclerotic cardiovascular disease. Moreover, studies in mouse models indicate a causal role of inflammasomes and IL-1β in atherosclerosis. However, recent studies have led to a more granular view of the role of inflammasomes in atherosclerosis. Studies in hyperlipidemic mouse models suggest that prominent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37228237"
                        open
                      >
                        <summary class="article-expander__title">Inflammasomes and Atherosclerosis: a Mixed Picture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37228237/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The CANTOS (Canakinumab Anti-inflammatory Thrombosis Outcome Study) and colchicine trials suggest an important role of inflammasomes and their major product IL-1β (interleukin 1β) in human atherosclerotic cardiovascular disease. Moreover, studies in mouse models indicate a causal role of inflammasomes and IL-1β in atherosclerosis. However, recent studies have led to a more granular view of the role of inflammasomes in atherosclerosis. Studies in hyperlipidemic mouse models suggest that prominent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-05-24 datetime="2023-05-24T10:00:00.000Z">2023-05-24</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-05-24 datetime="2023-05-24T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223330"
                        open
                      >
                        <summary class="article-expander__title">Malignant clonal evolution from high proportion of monocytes in patients with aplastic anemia: a case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223330/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230618210354&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The proportion of monocytes in the blood and bone marrow of AA patients should be closely monitored. Hematopoietic stem cell transplantation (HSCT) should be performed as early as possible once monocytes continue to increase or are associated with phenotypic abnormalities or genetic mutations. The unique value of this study is that although there were case reports about AA-derived acute leukemia, we suggested that an early high proportion of monocytes may predict malignant clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223330"
                        open
                      >
                        <summary class="article-expander__title">Malignant clonal evolution from high proportion of monocytes in patients with aplastic anemia: a case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223330/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230617211143&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The proportion of monocytes in the blood and bone marrow of AA patients should be closely monitored. Hematopoietic stem cell transplantation (HSCT) should be performed as early as possible once monocytes continue to increase or are associated with phenotypic abnormalities or genetic mutations. The unique value of this study is that although there were case reports about AA-derived acute leukemia, we suggested that an early high proportion of monocytes may predict malignant clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223330"
                        open
                      >
                        <summary class="article-expander__title">Malignant clonal evolution from high proportion of monocytes in patients with aplastic anemia: a case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223330/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The proportion of monocytes in the blood and bone marrow of AA patients should be closely monitored. Hematopoietic stem cell transplantation (HSCT) should be performed as early as possible once monocytes continue to increase or are associated with phenotypic abnormalities or genetic mutations. The unique value of this study is that although there were case reports about AA-derived acute leukemia, we suggested that an early high proportion of monocytes may predict malignant clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223330"
                        open
                      >
                        <summary class="article-expander__title">Malignant clonal evolution from high proportion of monocytes in patients with aplastic anemia: a case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223330/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The proportion of monocytes in the blood and bone marrow of AA patients should be closely monitored. Hematopoietic stem cell transplantation (HSCT) should be performed as early as possible once monocytes continue to increase or are associated with phenotypic abnormalities or genetic mutations. The unique value of this study is that although there were case reports about AA-derived acute leukemia, we suggested that an early high proportion of monocytes may predict malignant clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223330"
                        open
                      >
                        <summary class="article-expander__title">Malignant clonal evolution from high proportion of monocytes in patients with aplastic anemia: a case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223330/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The proportion of monocytes in the blood and bone marrow of AA patients should be closely monitored. Hematopoietic stem cell transplantation (HSCT) should be performed as early as possible once monocytes continue to increase or are associated with phenotypic abnormalities or genetic mutations. The unique value of this study is that although there were case reports about AA-derived acute leukemia, we suggested that an early high proportion of monocytes may predict malignant clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223330"
                        open
                      >
                        <summary class="article-expander__title">Malignant clonal evolution from high proportion of monocytes in patients with aplastic anemia: a case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223330/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The proportion of monocytes in the blood and bone marrow of AA patients should be closely monitored. Hematopoietic stem cell transplantation (HSCT) should be performed as early as possible once monocytes continue to increase or are associated with phenotypic abnormalities or genetic mutations. The unique value of this study is that although there were case reports about AA-derived acute leukemia, we suggested that an early high proportion of monocytes may predict malignant clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223330"
                        open
                      >
                        <summary class="article-expander__title">Malignant clonal evolution from high proportion of monocytes in patients with aplastic anemia: a case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223330/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The proportion of monocytes in the blood and bone marrow of AA patients should be closely monitored. Hematopoietic stem cell transplantation (HSCT) should be performed as early as possible once monocytes continue to increase or are associated with phenotypic abnormalities or genetic mutations. The unique value of this study is that although there were case reports about AA-derived acute leukemia, we suggested that an early high proportion of monocytes may predict malignant clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223330"
                        open
                      >
                        <summary class="article-expander__title">Malignant clonal evolution from high proportion of monocytes in patients with aplastic anemia: a case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223330/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The proportion of monocytes in the blood and bone marrow of AA patients should be closely monitored. Hematopoietic stem cell transplantation (HSCT) should be performed as early as possible once monocytes continue to increase or are associated with phenotypic abnormalities or genetic mutations. The unique value of this study is that although there were case reports about AA-derived acute leukemia, we suggested that an early high proportion of monocytes may predict malignant clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223330"
                        open
                      >
                        <summary class="article-expander__title">Malignant clonal evolution from high proportion of monocytes in patients with aplastic anemia: a case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223330/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The proportion of monocytes in the blood and bone marrow of AA patients should be closely monitored. Hematopoietic stem cell transplantation (HSCT) should be performed as early as possible once monocytes continue to increase or are associated with phenotypic abnormalities or genetic mutations. The unique value of this study is that although there were case reports about AA-derived acute leukemia, we suggested that an early high proportion of monocytes may predict malignant clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223330"
                        open
                      >
                        <summary class="article-expander__title">Malignant clonal evolution from high proportion of monocytes in patients with aplastic anemia: a case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223330/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The proportion of monocytes in the blood and bone marrow of AA patients should be closely monitored. Hematopoietic stem cell transplantation (HSCT) should be performed as early as possible once monocytes continue to increase or are associated with phenotypic abnormalities or genetic mutations. The unique value of this study is that although there were case reports about AA-derived acute leukemia, we suggested that an early high proportion of monocytes may predict malignant clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223330"
                        open
                      >
                        <summary class="article-expander__title">Malignant clonal evolution from high proportion of monocytes in patients with aplastic anemia: a case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223330/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The proportion of monocytes in the blood and bone marrow of AA patients should be closely monitored. Hematopoietic stem cell transplantation (HSCT) should be performed as early as possible once monocytes continue to increase or are associated with phenotypic abnormalities or genetic mutations. The unique value of this study is that although there were case reports about AA-derived acute leukemia, we suggested that an early high proportion of monocytes may predict malignant clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223330"
                        open
                      >
                        <summary class="article-expander__title">Malignant clonal evolution from high proportion of monocytes in patients with aplastic anemia: a case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223330/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The proportion of monocytes in the blood and bone marrow of AA patients should be closely monitored. Hematopoietic stem cell transplantation (HSCT) should be performed as early as possible once monocytes continue to increase or are associated with phenotypic abnormalities or genetic mutations. The unique value of this study is that although there were case reports about AA-derived acute leukemia, we suggested that an early high proportion of monocytes may predict malignant clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223330"
                        open
                      >
                        <summary class="article-expander__title">Malignant clonal evolution from high proportion of monocytes in patients with aplastic anemia: a case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223330/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The proportion of monocytes in the blood and bone marrow of AA patients should be closely monitored. Hematopoietic stem cell transplantation (HSCT) should be performed as early as possible once monocytes continue to increase or are associated with phenotypic abnormalities or genetic mutations. The unique value of this study is that although there were case reports about AA-derived acute leukemia, we suggested that an early high proportion of monocytes may predict malignant clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223330"
                        open
                      >
                        <summary class="article-expander__title">Malignant clonal evolution from high proportion of monocytes in patients with aplastic anemia: a case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223330/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The proportion of monocytes in the blood and bone marrow of AA patients should be closely monitored. Hematopoietic stem cell transplantation (HSCT) should be performed as early as possible once monocytes continue to increase or are associated with phenotypic abnormalities or genetic mutations. The unique value of this study is that although there were case reports about AA-derived acute leukemia, we suggested that an early high proportion of monocytes may predict malignant clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223330"
                        open
                      >
                        <summary class="article-expander__title">Malignant clonal evolution from high proportion of monocytes in patients with aplastic anemia: a case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223330/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The proportion of monocytes in the blood and bone marrow of AA patients should be closely monitored. Hematopoietic stem cell transplantation (HSCT) should be performed as early as possible once monocytes continue to increase or are associated with phenotypic abnormalities or genetic mutations. The unique value of this study is that although there were case reports about AA-derived acute leukemia, we suggested that an early high proportion of monocytes may predict malignant clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223330"
                        open
                      >
                        <summary class="article-expander__title">Malignant clonal evolution from high proportion of monocytes in patients with aplastic anemia: a case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223330/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The proportion of monocytes in the blood and bone marrow of AA patients should be closely monitored. Hematopoietic stem cell transplantation (HSCT) should be performed as early as possible once monocytes continue to increase or are associated with phenotypic abnormalities or genetic mutations. The unique value of this study is that although there were case reports about AA-derived acute leukemia, we suggested that an early high proportion of monocytes may predict malignant clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223330"
                        open
                      >
                        <summary class="article-expander__title">Malignant clonal evolution from high proportion of monocytes in patients with aplastic anemia: a case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223330/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The proportion of monocytes in the blood and bone marrow of AA patients should be closely monitored. Hematopoietic stem cell transplantation (HSCT) should be performed as early as possible once monocytes continue to increase or are associated with phenotypic abnormalities or genetic mutations. The unique value of this study is that although there were case reports about AA-derived acute leukemia, we suggested that an early high proportion of monocytes may predict malignant clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223330"
                        open
                      >
                        <summary class="article-expander__title">Malignant clonal evolution from high proportion of monocytes in patients with aplastic anemia: a case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223330/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The proportion of monocytes in the blood and bone marrow of AA patients should be closely monitored. Hematopoietic stem cell transplantation (HSCT) should be performed as early as possible once monocytes continue to increase or are associated with phenotypic abnormalities or genetic mutations. The unique value of this study is that although there were case reports about AA-derived acute leukemia, we suggested that an early high proportion of monocytes may predict malignant clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223330"
                        open
                      >
                        <summary class="article-expander__title">Malignant clonal evolution from high proportion of monocytes in patients with aplastic anemia: a case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223330/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The proportion of monocytes in the blood and bone marrow of AA patients should be closely monitored. Hematopoietic stem cell transplantation (HSCT) should be performed as early as possible once monocytes continue to increase or are associated with phenotypic abnormalities or genetic mutations. The unique value of this study is that although there were case reports about AA-derived acute leukemia, we suggested that an early high proportion of monocytes may predict malignant clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223330"
                        open
                      >
                        <summary class="article-expander__title">Malignant clonal evolution from high proportion of monocytes in patients with aplastic anemia: a case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223330/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The proportion of monocytes in the blood and bone marrow of AA patients should be closely monitored. Hematopoietic stem cell transplantation (HSCT) should be performed as early as possible once monocytes continue to increase or are associated with phenotypic abnormalities or genetic mutations. The unique value of this study is that although there were case reports about AA-derived acute leukemia, we suggested that an early high proportion of monocytes may predict malignant clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223330"
                        open
                      >
                        <summary class="article-expander__title">Malignant clonal evolution from high proportion of monocytes in patients with aplastic anemia: a case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223330/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The proportion of monocytes in the blood and bone marrow of AA patients should be closely monitored. Hematopoietic stem cell transplantation (HSCT) should be performed as early as possible once monocytes continue to increase or are associated with phenotypic abnormalities or genetic mutations. The unique value of this study is that although there were case reports about AA-derived acute leukemia, we suggested that an early high proportion of monocytes may predict malignant clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223330"
                        open
                      >
                        <summary class="article-expander__title">Malignant clonal evolution from high proportion of monocytes in patients with aplastic anemia: a case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223330/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The proportion of monocytes in the blood and bone marrow of AA patients should be closely monitored. Hematopoietic stem cell transplantation (HSCT) should be performed as early as possible once monocytes continue to increase or are associated with phenotypic abnormalities or genetic mutations. The unique value of this study is that although there were case reports about AA-derived acute leukemia, we suggested that an early high proportion of monocytes may predict malignant clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230527210349&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223330"
                        open
                      >
                        <summary class="article-expander__title">Malignant clonal evolution from high proportion of monocytes in patients with aplastic anemia: a case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223330/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230527210349&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The proportion of monocytes in the blood and bone marrow of AA patients should be closely monitored. Hematopoietic stem cell transplantation (HSCT) should be performed as early as possible once monocytes continue to increase or are associated with phenotypic abnormalities or genetic mutations. The unique value of this study is that although there were case reports about AA-derived acute leukemia, we suggested that an early high proportion of monocytes may predict malignant clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223330"
                        open
                      >
                        <summary class="article-expander__title">Malignant clonal evolution from high proportion of monocytes in patients with aplastic anemia: a case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223330/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The proportion of monocytes in the blood and bone marrow of AA patients should be closely monitored. Hematopoietic stem cell transplantation (HSCT) should be performed as early as possible once monocytes continue to increase or are associated with phenotypic abnormalities or genetic mutations. The unique value of this study is that although there were case reports about AA-derived acute leukemia, we suggested that an early high proportion of monocytes may predict malignant clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223330"
                        open
                      >
                        <summary class="article-expander__title">Malignant clonal evolution from high proportion of monocytes in patients with aplastic anemia: a case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223330/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The proportion of monocytes in the blood and bone marrow of AA patients should be closely monitored. Hematopoietic stem cell transplantation (HSCT) should be performed as early as possible once monocytes continue to increase or are associated with phenotypic abnormalities or genetic mutations. The unique value of this study is that although there were case reports about AA-derived acute leukemia, we suggested that an early high proportion of monocytes may predict malignant clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223330"
                        open
                      >
                        <summary class="article-expander__title">Malignant clonal evolution from high proportion of monocytes in patients with aplastic anemia: a case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223330/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The proportion of monocytes in the blood and bone marrow of AA patients should be closely monitored. Hematopoietic stem cell transplantation (HSCT) should be performed as early as possible once monocytes continue to increase or are associated with phenotypic abnormalities or genetic mutations. The unique value of this study is that although there were case reports about AA-derived acute leukemia, we suggested that an early high proportion of monocytes may predict malignant clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37226092"
                        open
                      >
                        <summary class="article-expander__title">Synergistic inter-clonal cooperation involving crosstalk, co-option and co-dependency can enhance the invasiveness of genetically distant cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37226092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Based on our findings, we propose a model in which crosstalk, co-option, and co-dependency can facilitate the evolution of synergistic cooperative interactions between genetically distant clones. Specifically, we suggest that synergistic cooperative interactions can easily emerge, regardless of the degree of overall genetic/genealogical relatedness, via crosstalk involving metastatic clones able to constitutively secrete molecules that induce and maintain their own malignant state...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37226092"
                        open
                      >
                        <summary class="article-expander__title">Synergistic inter-clonal cooperation involving crosstalk, co-option and co-dependency can enhance the invasiveness of genetically distant cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37226092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Based on our findings, we propose a model in which crosstalk, co-option, and co-dependency can facilitate the evolution of synergistic cooperative interactions between genetically distant clones. Specifically, we suggest that synergistic cooperative interactions can easily emerge, regardless of the degree of overall genetic/genealogical relatedness, via crosstalk involving metastatic clones able to constitutively secrete molecules that induce and maintain their own malignant state...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37226092"
                        open
                      >
                        <summary class="article-expander__title">Synergistic inter-clonal cooperation involving crosstalk, co-option and co-dependency can enhance the invasiveness of genetically distant cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37226092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Based on our findings, we propose a model in which crosstalk, co-option, and co-dependency can facilitate the evolution of synergistic cooperative interactions between genetically distant clones. Specifically, we suggest that synergistic cooperative interactions can easily emerge, regardless of the degree of overall genetic/genealogical relatedness, via crosstalk involving metastatic clones able to constitutively secrete molecules that induce and maintain their own malignant state...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37226092"
                        open
                      >
                        <summary class="article-expander__title">Synergistic inter-clonal cooperation involving crosstalk, co-option and co-dependency can enhance the invasiveness of genetically distant cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37226092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Based on our findings, we propose a model in which crosstalk, co-option, and co-dependency can facilitate the evolution of synergistic cooperative interactions between genetically distant clones. Specifically, we suggest that synergistic cooperative interactions can easily emerge, regardless of the degree of overall genetic/genealogical relatedness, via crosstalk involving metastatic clones able to constitutively secrete molecules that induce and maintain their own malignant state...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37226092"
                        open
                      >
                        <summary class="article-expander__title">Synergistic inter-clonal cooperation involving crosstalk, co-option and co-dependency can enhance the invasiveness of genetically distant cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37226092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Based on our findings, we propose a model in which crosstalk, co-option, and co-dependency can facilitate the evolution of synergistic cooperative interactions between genetically distant clones. Specifically, we suggest that synergistic cooperative interactions can easily emerge, regardless of the degree of overall genetic/genealogical relatedness, via crosstalk involving metastatic clones able to constitutively secrete molecules that induce and maintain their own malignant state...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37226092"
                        open
                      >
                        <summary class="article-expander__title">Synergistic inter-clonal cooperation involving crosstalk, co-option and co-dependency can enhance the invasiveness of genetically distant cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37226092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Based on our findings, we propose a model in which crosstalk, co-option, and co-dependency can facilitate the evolution of synergistic cooperative interactions between genetically distant clones. Specifically, we suggest that synergistic cooperative interactions can easily emerge, regardless of the degree of overall genetic/genealogical relatedness, via crosstalk involving metastatic clones able to constitutively secrete molecules that induce and maintain their own malignant state...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37226092"
                        open
                      >
                        <summary class="article-expander__title">Synergistic inter-clonal cooperation involving crosstalk, co-option and co-dependency can enhance the invasiveness of genetically distant cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37226092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Based on our findings, we propose a model in which crosstalk, co-option, and co-dependency can facilitate the evolution of synergistic cooperative interactions between genetically distant clones. Specifically, we suggest that synergistic cooperative interactions can easily emerge, regardless of the degree of overall genetic/genealogical relatedness, via crosstalk involving metastatic clones able to constitutively secrete molecules that induce and maintain their own malignant state...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37226092"
                        open
                      >
                        <summary class="article-expander__title">Synergistic inter-clonal cooperation involving crosstalk, co-option and co-dependency can enhance the invasiveness of genetically distant cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37226092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Based on our findings, we propose a model in which crosstalk, co-option, and co-dependency can facilitate the evolution of synergistic cooperative interactions between genetically distant clones. Specifically, we suggest that synergistic cooperative interactions can easily emerge, regardless of the degree of overall genetic/genealogical relatedness, via crosstalk involving metastatic clones able to constitutively secrete molecules that induce and maintain their own malignant state...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37226092"
                        open
                      >
                        <summary class="article-expander__title">Synergistic inter-clonal cooperation involving crosstalk, co-option and co-dependency can enhance the invasiveness of genetically distant cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37226092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Based on our findings, we propose a model in which crosstalk, co-option, and co-dependency can facilitate the evolution of synergistic cooperative interactions between genetically distant clones. Specifically, we suggest that synergistic cooperative interactions can easily emerge, regardless of the degree of overall genetic/genealogical relatedness, via crosstalk involving metastatic clones able to constitutively secrete molecules that induce and maintain their own malignant state...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37226092"
                        open
                      >
                        <summary class="article-expander__title">Synergistic inter-clonal cooperation involving crosstalk, co-option and co-dependency can enhance the invasiveness of genetically distant cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37226092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Based on our findings, we propose a model in which crosstalk, co-option, and co-dependency can facilitate the evolution of synergistic cooperative interactions between genetically distant clones. Specifically, we suggest that synergistic cooperative interactions can easily emerge, regardless of the degree of overall genetic/genealogical relatedness, via crosstalk involving metastatic clones able to constitutively secrete molecules that induce and maintain their own malignant state...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37226092"
                        open
                      >
                        <summary class="article-expander__title">Synergistic inter-clonal cooperation involving crosstalk, co-option and co-dependency can enhance the invasiveness of genetically distant cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37226092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Based on our findings, we propose a model in which crosstalk, co-option, and co-dependency can facilitate the evolution of synergistic cooperative interactions between genetically distant clones. Specifically, we suggest that synergistic cooperative interactions can easily emerge, regardless of the degree of overall genetic/genealogical relatedness, via crosstalk involving metastatic clones able to constitutively secrete molecules that induce and maintain their own malignant state...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37226092"
                        open
                      >
                        <summary class="article-expander__title">Synergistic inter-clonal cooperation involving crosstalk, co-option and co-dependency can enhance the invasiveness of genetically distant cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37226092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Based on our findings, we propose a model in which crosstalk, co-option, and co-dependency can facilitate the evolution of synergistic cooperative interactions between genetically distant clones. Specifically, we suggest that synergistic cooperative interactions can easily emerge, regardless of the degree of overall genetic/genealogical relatedness, via crosstalk involving metastatic clones able to constitutively secrete molecules that induce and maintain their own malignant state...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37226092"
                        open
                      >
                        <summary class="article-expander__title">Synergistic inter-clonal cooperation involving crosstalk, co-option and co-dependency can enhance the invasiveness of genetically distant cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37226092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Based on our findings, we propose a model in which crosstalk, co-option, and co-dependency can facilitate the evolution of synergistic cooperative interactions between genetically distant clones. Specifically, we suggest that synergistic cooperative interactions can easily emerge, regardless of the degree of overall genetic/genealogical relatedness, via crosstalk involving metastatic clones able to constitutively secrete molecules that induce and maintain their own malignant state...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37226092"
                        open
                      >
                        <summary class="article-expander__title">Synergistic inter-clonal cooperation involving crosstalk, co-option and co-dependency can enhance the invasiveness of genetically distant cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37226092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Based on our findings, we propose a model in which crosstalk, co-option, and co-dependency can facilitate the evolution of synergistic cooperative interactions between genetically distant clones. Specifically, we suggest that synergistic cooperative interactions can easily emerge, regardless of the degree of overall genetic/genealogical relatedness, via crosstalk involving metastatic clones able to constitutively secrete molecules that induce and maintain their own malignant state...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37226092"
                        open
                      >
                        <summary class="article-expander__title">Synergistic inter-clonal cooperation involving crosstalk, co-option and co-dependency can enhance the invasiveness of genetically distant cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37226092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Based on our findings, we propose a model in which crosstalk, co-option, and co-dependency can facilitate the evolution of synergistic cooperative interactions between genetically distant clones. Specifically, we suggest that synergistic cooperative interactions can easily emerge, regardless of the degree of overall genetic/genealogical relatedness, via crosstalk involving metastatic clones able to constitutively secrete molecules that induce and maintain their own malignant state...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37226092"
                        open
                      >
                        <summary class="article-expander__title">Synergistic inter-clonal cooperation involving crosstalk, co-option and co-dependency can enhance the invasiveness of genetically distant cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37226092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230527210351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Based on our findings, we propose a model in which crosstalk, co-option, and co-dependency can facilitate the evolution of synergistic cooperative interactions between genetically distant clones. Specifically, we suggest that synergistic cooperative interactions can easily emerge, regardless of the degree of overall genetic/genealogical relatedness, via crosstalk involving metastatic clones able to constitutively secrete molecules that induce and maintain their own malignant state...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230527210351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37226092"
                        open
                      >
                        <summary class="article-expander__title">Synergistic inter-clonal cooperation involving crosstalk, co-option and co-dependency can enhance the invasiveness of genetically distant cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37226092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Based on our findings, we propose a model in which crosstalk, co-option, and co-dependency can facilitate the evolution of synergistic cooperative interactions between genetically distant clones. Specifically, we suggest that synergistic cooperative interactions can easily emerge, regardless of the degree of overall genetic/genealogical relatedness, via crosstalk involving metastatic clones able to constitutively secrete molecules that induce and maintain their own malignant state...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37226092"
                        open
                      >
                        <summary class="article-expander__title">Synergistic inter-clonal cooperation involving crosstalk, co-option and co-dependency can enhance the invasiveness of genetically distant cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37226092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Based on our findings, we propose a model in which crosstalk, co-option, and co-dependency can facilitate the evolution of synergistic cooperative interactions between genetically distant clones. Specifically, we suggest that synergistic cooperative interactions can easily emerge, regardless of the degree of overall genetic/genealogical relatedness, via crosstalk involving metastatic clones able to constitutively secrete molecules that induce and maintain their own malignant state...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37223675"
                        open
                      >
                        <summary class="article-expander__title">Case Report: First longitudinal study of a patient with CALR positive clonal hematopoiesis of indeterminate potential developing into pre-fibrotic myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37223675/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Initial diagnosis of overt myeloproliferative neoplasms (MPNs) represents the juncture during clonal evolution when symptoms or complications prompt an afflicted individual to seek medical attention. In 30-40% of the MPN subgroups essential thrombocythemia (ET) and myelofibrosis (MF), somatic mutations in the calreticulin gene (CALR) are drivers of the disease resulting in constitutive activation of the thrombopoietin receptor (MPL). In the current study, we describe a healthy CALR mutated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-05-23 datetime="2023-05-23T10:00:00.000Z">2023-05-23</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-05-23 datetime="2023-05-23T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230616210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230527210349&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37220750"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal intra-tumoral heterogeneity in advanced HGSOC: Implications for surgical and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37220750/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Limited evidence exists on the impact of spatial and temporal heterogeneity of high-grade serous ovarian cancer (HGSOC) on tumor evolution, clinical outcomes, and surgical operability. We perform systematic multi-site tumor mapping at presentation and matched relapse from 49 high-tumor-burden patients, operated up front. From SNP array-derived copy-number data, we categorize dendrograms representing tumor clonal evolution as sympodial or dichotomous, noting most chemo-resistant patients favor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37220750"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal intra-tumoral heterogeneity in advanced HGSOC: Implications for surgical and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37220750/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Limited evidence exists on the impact of spatial and temporal heterogeneity of high-grade serous ovarian cancer (HGSOC) on tumor evolution, clinical outcomes, and surgical operability. We perform systematic multi-site tumor mapping at presentation and matched relapse from 49 high-tumor-burden patients, operated up front. From SNP array-derived copy-number data, we categorize dendrograms representing tumor clonal evolution as sympodial or dichotomous, noting most chemo-resistant patients favor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37220750"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal intra-tumoral heterogeneity in advanced HGSOC: Implications for surgical and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37220750/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Limited evidence exists on the impact of spatial and temporal heterogeneity of high-grade serous ovarian cancer (HGSOC) on tumor evolution, clinical outcomes, and surgical operability. We perform systematic multi-site tumor mapping at presentation and matched relapse from 49 high-tumor-burden patients, operated up front. From SNP array-derived copy-number data, we categorize dendrograms representing tumor clonal evolution as sympodial or dichotomous, noting most chemo-resistant patients favor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37220750"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal intra-tumoral heterogeneity in advanced HGSOC: Implications for surgical and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37220750/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Limited evidence exists on the impact of spatial and temporal heterogeneity of high-grade serous ovarian cancer (HGSOC) on tumor evolution, clinical outcomes, and surgical operability. We perform systematic multi-site tumor mapping at presentation and matched relapse from 49 high-tumor-burden patients, operated up front. From SNP array-derived copy-number data, we categorize dendrograms representing tumor clonal evolution as sympodial or dichotomous, noting most chemo-resistant patients favor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37220750"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal intra-tumoral heterogeneity in advanced HGSOC: Implications for surgical and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37220750/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Limited evidence exists on the impact of spatial and temporal heterogeneity of high-grade serous ovarian cancer (HGSOC) on tumor evolution, clinical outcomes, and surgical operability. We perform systematic multi-site tumor mapping at presentation and matched relapse from 49 high-tumor-burden patients, operated up front. From SNP array-derived copy-number data, we categorize dendrograms representing tumor clonal evolution as sympodial or dichotomous, noting most chemo-resistant patients favor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37220750"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal intra-tumoral heterogeneity in advanced HGSOC: Implications for surgical and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37220750/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Limited evidence exists on the impact of spatial and temporal heterogeneity of high-grade serous ovarian cancer (HGSOC) on tumor evolution, clinical outcomes, and surgical operability. We perform systematic multi-site tumor mapping at presentation and matched relapse from 49 high-tumor-burden patients, operated up front. From SNP array-derived copy-number data, we categorize dendrograms representing tumor clonal evolution as sympodial or dichotomous, noting most chemo-resistant patients favor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37220750"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal intra-tumoral heterogeneity in advanced HGSOC: Implications for surgical and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37220750/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Limited evidence exists on the impact of spatial and temporal heterogeneity of high-grade serous ovarian cancer (HGSOC) on tumor evolution, clinical outcomes, and surgical operability. We perform systematic multi-site tumor mapping at presentation and matched relapse from 49 high-tumor-burden patients, operated up front. From SNP array-derived copy-number data, we categorize dendrograms representing tumor clonal evolution as sympodial or dichotomous, noting most chemo-resistant patients favor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37220750"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal intra-tumoral heterogeneity in advanced HGSOC: Implications for surgical and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37220750/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Limited evidence exists on the impact of spatial and temporal heterogeneity of high-grade serous ovarian cancer (HGSOC) on tumor evolution, clinical outcomes, and surgical operability. We perform systematic multi-site tumor mapping at presentation and matched relapse from 49 high-tumor-burden patients, operated up front. From SNP array-derived copy-number data, we categorize dendrograms representing tumor clonal evolution as sympodial or dichotomous, noting most chemo-resistant patients favor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37220750"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal intra-tumoral heterogeneity in advanced HGSOC: Implications for surgical and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37220750/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Limited evidence exists on the impact of spatial and temporal heterogeneity of high-grade serous ovarian cancer (HGSOC) on tumor evolution, clinical outcomes, and surgical operability. We perform systematic multi-site tumor mapping at presentation and matched relapse from 49 high-tumor-burden patients, operated up front. From SNP array-derived copy-number data, we categorize dendrograms representing tumor clonal evolution as sympodial or dichotomous, noting most chemo-resistant patients favor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37220750"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal intra-tumoral heterogeneity in advanced HGSOC: Implications for surgical and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37220750/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Limited evidence exists on the impact of spatial and temporal heterogeneity of high-grade serous ovarian cancer (HGSOC) on tumor evolution, clinical outcomes, and surgical operability. We perform systematic multi-site tumor mapping at presentation and matched relapse from 49 high-tumor-burden patients, operated up front. From SNP array-derived copy-number data, we categorize dendrograms representing tumor clonal evolution as sympodial or dichotomous, noting most chemo-resistant patients favor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37220750"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal intra-tumoral heterogeneity in advanced HGSOC: Implications for surgical and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37220750/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Limited evidence exists on the impact of spatial and temporal heterogeneity of high-grade serous ovarian cancer (HGSOC) on tumor evolution, clinical outcomes, and surgical operability. We perform systematic multi-site tumor mapping at presentation and matched relapse from 49 high-tumor-burden patients, operated up front. From SNP array-derived copy-number data, we categorize dendrograms representing tumor clonal evolution as sympodial or dichotomous, noting most chemo-resistant patients favor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230527210351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37220750"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal intra-tumoral heterogeneity in advanced HGSOC: Implications for surgical and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37220750/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230527210351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Limited evidence exists on the impact of spatial and temporal heterogeneity of high-grade serous ovarian cancer (HGSOC) on tumor evolution, clinical outcomes, and surgical operability. We perform systematic multi-site tumor mapping at presentation and matched relapse from 49 high-tumor-burden patients, operated up front. From SNP array-derived copy-number data, we categorize dendrograms representing tumor clonal evolution as sympodial or dichotomous, noting most chemo-resistant patients favor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37220750"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal intra-tumoral heterogeneity in advanced HGSOC: Implications for surgical and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37220750/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Limited evidence exists on the impact of spatial and temporal heterogeneity of high-grade serous ovarian cancer (HGSOC) on tumor evolution, clinical outcomes, and surgical operability. We perform systematic multi-site tumor mapping at presentation and matched relapse from 49 high-tumor-burden patients, operated up front. From SNP array-derived copy-number data, we categorize dendrograms representing tumor clonal evolution as sympodial or dichotomous, noting most chemo-resistant patients favor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37220750"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal intra-tumoral heterogeneity in advanced HGSOC: Implications for surgical and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37220750/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Limited evidence exists on the impact of spatial and temporal heterogeneity of high-grade serous ovarian cancer (HGSOC) on tumor evolution, clinical outcomes, and surgical operability. We perform systematic multi-site tumor mapping at presentation and matched relapse from 49 high-tumor-burden patients, operated up front. From SNP array-derived copy-number data, we categorize dendrograms representing tumor clonal evolution as sympodial or dichotomous, noting most chemo-resistant patients favor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37221411"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Myeloproliferative Neoplasms Confers a Predisposition to both Thrombosis and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37221411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review focuses on vascular complications associated with chronic myeloproliferative neoplasms (MPN) and more specifically aims to discuss the clinical and biological evidence supporting the existence of a link between clonal hematopoiesis, cardiovascular events (CVE), and solid cancer (SC).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37220750"
                        open
                      >
                        <summary class="article-expander__title">Spatial and temporal intra-tumoral heterogeneity in advanced HGSOC: Implications for surgical and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37220750/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Limited evidence exists on the impact of spatial and temporal heterogeneity of high-grade serous ovarian cancer (HGSOC) on tumor evolution, clinical outcomes, and surgical operability. We perform systematic multi-site tumor mapping at presentation and matched relapse from 49 high-tumor-burden patients, operated up front. From SNP array-derived copy-number data, we categorize dendrograms representing tumor clonal evolution as sympodial or dichotomous, noting most chemo-resistant patients favor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-05-22 datetime="2023-05-22T10:00:00.000Z">2023-05-22</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-05-22 datetime="2023-05-22T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37217808"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing reveals the metastatic potential of hepatocellular carcinoma from the perspective of tumor and circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37217808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We characterized the genomic alterations during HCC vascular invasion and revealed a previously undescribed evolution pattern of ctDNA in HCC. A novel multiomics-based signature was developed to identify high-risk relapse populations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37217808"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing reveals the metastatic potential of hepatocellular carcinoma from the perspective of tumor and circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37217808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We characterized the genomic alterations during HCC vascular invasion and revealed a previously undescribed evolution pattern of ctDNA in HCC. A novel multiomics-based signature was developed to identify high-risk relapse populations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37215501"
                        open
                      >
                        <summary class="article-expander__title">An unusual ectopic thymoma clonal evolution analysis: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37215501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Thymomas and thymic carcinomas are rare and primary tumors of the mediastinum which is derived from the thymic epithelium. Thymomas are the most common primary anterior mediastinal tumor, while ectopic thymomas are rarer. Mutational profiles of ectopic thymomas may help expand our understanding of the occurrence and treatment options of these tumors. In this report, we sought to elucidate the mutational profiles of two ectopic thymoma nodules to gain deeper understanding of the molecular genetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37217808"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing reveals the metastatic potential of hepatocellular carcinoma from the perspective of tumor and circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37217808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We characterized the genomic alterations during HCC vascular invasion and revealed a previously undescribed evolution pattern of ctDNA in HCC. A novel multiomics-based signature was developed to identify high-risk relapse populations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37215501"
                        open
                      >
                        <summary class="article-expander__title">An unusual ectopic thymoma clonal evolution analysis: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37215501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Thymomas and thymic carcinomas are rare and primary tumors of the mediastinum which is derived from the thymic epithelium. Thymomas are the most common primary anterior mediastinal tumor, while ectopic thymomas are rarer. Mutational profiles of ectopic thymomas may help expand our understanding of the occurrence and treatment options of these tumors. In this report, we sought to elucidate the mutational profiles of two ectopic thymoma nodules to gain deeper understanding of the molecular genetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37217808"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing reveals the metastatic potential of hepatocellular carcinoma from the perspective of tumor and circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37217808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We characterized the genomic alterations during HCC vascular invasion and revealed a previously undescribed evolution pattern of ctDNA in HCC. A novel multiomics-based signature was developed to identify high-risk relapse populations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37215501"
                        open
                      >
                        <summary class="article-expander__title">An unusual ectopic thymoma clonal evolution analysis: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37215501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Thymomas and thymic carcinomas are rare and primary tumors of the mediastinum which is derived from the thymic epithelium. Thymomas are the most common primary anterior mediastinal tumor, while ectopic thymomas are rarer. Mutational profiles of ectopic thymomas may help expand our understanding of the occurrence and treatment options of these tumors. In this report, we sought to elucidate the mutational profiles of two ectopic thymoma nodules to gain deeper understanding of the molecular genetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37217808"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing reveals the metastatic potential of hepatocellular carcinoma from the perspective of tumor and circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37217808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We characterized the genomic alterations during HCC vascular invasion and revealed a previously undescribed evolution pattern of ctDNA in HCC. A novel multiomics-based signature was developed to identify high-risk relapse populations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37215501"
                        open
                      >
                        <summary class="article-expander__title">An unusual ectopic thymoma clonal evolution analysis: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37215501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Thymomas and thymic carcinomas are rare and primary tumors of the mediastinum which is derived from the thymic epithelium. Thymomas are the most common primary anterior mediastinal tumor, while ectopic thymomas are rarer. Mutational profiles of ectopic thymomas may help expand our understanding of the occurrence and treatment options of these tumors. In this report, we sought to elucidate the mutational profiles of two ectopic thymoma nodules to gain deeper understanding of the molecular genetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37217808"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing reveals the metastatic potential of hepatocellular carcinoma from the perspective of tumor and circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37217808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We characterized the genomic alterations during HCC vascular invasion and revealed a previously undescribed evolution pattern of ctDNA in HCC. A novel multiomics-based signature was developed to identify high-risk relapse populations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37215501"
                        open
                      >
                        <summary class="article-expander__title">An unusual ectopic thymoma clonal evolution analysis: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37215501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Thymomas and thymic carcinomas are rare and primary tumors of the mediastinum which is derived from the thymic epithelium. Thymomas are the most common primary anterior mediastinal tumor, while ectopic thymomas are rarer. Mutational profiles of ectopic thymomas may help expand our understanding of the occurrence and treatment options of these tumors. In this report, we sought to elucidate the mutational profiles of two ectopic thymoma nodules to gain deeper understanding of the molecular genetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37217808"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing reveals the metastatic potential of hepatocellular carcinoma from the perspective of tumor and circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37217808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We characterized the genomic alterations during HCC vascular invasion and revealed a previously undescribed evolution pattern of ctDNA in HCC. A novel multiomics-based signature was developed to identify high-risk relapse populations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37215501"
                        open
                      >
                        <summary class="article-expander__title">An unusual ectopic thymoma clonal evolution analysis: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37215501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Thymomas and thymic carcinomas are rare and primary tumors of the mediastinum which is derived from the thymic epithelium. Thymomas are the most common primary anterior mediastinal tumor, while ectopic thymomas are rarer. Mutational profiles of ectopic thymomas may help expand our understanding of the occurrence and treatment options of these tumors. In this report, we sought to elucidate the mutational profiles of two ectopic thymoma nodules to gain deeper understanding of the molecular genetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37217808"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing reveals the metastatic potential of hepatocellular carcinoma from the perspective of tumor and circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37217808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We characterized the genomic alterations during HCC vascular invasion and revealed a previously undescribed evolution pattern of ctDNA in HCC. A novel multiomics-based signature was developed to identify high-risk relapse populations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37215501"
                        open
                      >
                        <summary class="article-expander__title">An unusual ectopic thymoma clonal evolution analysis: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37215501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Thymomas and thymic carcinomas are rare and primary tumors of the mediastinum which is derived from the thymic epithelium. Thymomas are the most common primary anterior mediastinal tumor, while ectopic thymomas are rarer. Mutational profiles of ectopic thymomas may help expand our understanding of the occurrence and treatment options of these tumors. In this report, we sought to elucidate the mutational profiles of two ectopic thymoma nodules to gain deeper understanding of the molecular genetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37217808"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing reveals the metastatic potential of hepatocellular carcinoma from the perspective of tumor and circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37217808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We characterized the genomic alterations during HCC vascular invasion and revealed a previously undescribed evolution pattern of ctDNA in HCC. A novel multiomics-based signature was developed to identify high-risk relapse populations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37215501"
                        open
                      >
                        <summary class="article-expander__title">An unusual ectopic thymoma clonal evolution analysis: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37215501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Thymomas and thymic carcinomas are rare and primary tumors of the mediastinum which is derived from the thymic epithelium. Thymomas are the most common primary anterior mediastinal tumor, while ectopic thymomas are rarer. Mutational profiles of ectopic thymomas may help expand our understanding of the occurrence and treatment options of these tumors. In this report, we sought to elucidate the mutational profiles of two ectopic thymoma nodules to gain deeper understanding of the molecular genetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37217808"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing reveals the metastatic potential of hepatocellular carcinoma from the perspective of tumor and circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37217808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We characterized the genomic alterations during HCC vascular invasion and revealed a previously undescribed evolution pattern of ctDNA in HCC. A novel multiomics-based signature was developed to identify high-risk relapse populations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37215501"
                        open
                      >
                        <summary class="article-expander__title">An unusual ectopic thymoma clonal evolution analysis: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37215501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Thymomas and thymic carcinomas are rare and primary tumors of the mediastinum which is derived from the thymic epithelium. Thymomas are the most common primary anterior mediastinal tumor, while ectopic thymomas are rarer. Mutational profiles of ectopic thymomas may help expand our understanding of the occurrence and treatment options of these tumors. In this report, we sought to elucidate the mutational profiles of two ectopic thymoma nodules to gain deeper understanding of the molecular genetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37217808"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing reveals the metastatic potential of hepatocellular carcinoma from the perspective of tumor and circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37217808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We characterized the genomic alterations during HCC vascular invasion and revealed a previously undescribed evolution pattern of ctDNA in HCC. A novel multiomics-based signature was developed to identify high-risk relapse populations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37215501"
                        open
                      >
                        <summary class="article-expander__title">An unusual ectopic thymoma clonal evolution analysis: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37215501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Thymomas and thymic carcinomas are rare and primary tumors of the mediastinum which is derived from the thymic epithelium. Thymomas are the most common primary anterior mediastinal tumor, while ectopic thymomas are rarer. Mutational profiles of ectopic thymomas may help expand our understanding of the occurrence and treatment options of these tumors. In this report, we sought to elucidate the mutational profiles of two ectopic thymoma nodules to gain deeper understanding of the molecular genetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37217808"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing reveals the metastatic potential of hepatocellular carcinoma from the perspective of tumor and circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37217808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230527210351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We characterized the genomic alterations during HCC vascular invasion and revealed a previously undescribed evolution pattern of ctDNA in HCC. A novel multiomics-based signature was developed to identify high-risk relapse populations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37215501"
                        open
                      >
                        <summary class="article-expander__title">An unusual ectopic thymoma clonal evolution analysis: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37215501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230527210351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Thymomas and thymic carcinomas are rare and primary tumors of the mediastinum which is derived from the thymic epithelium. Thymomas are the most common primary anterior mediastinal tumor, while ectopic thymomas are rarer. Mutational profiles of ectopic thymomas may help expand our understanding of the occurrence and treatment options of these tumors. In this report, we sought to elucidate the mutational profiles of two ectopic thymoma nodules to gain deeper understanding of the molecular genetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37217808"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing reveals the metastatic potential of hepatocellular carcinoma from the perspective of tumor and circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37217808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We characterized the genomic alterations during HCC vascular invasion and revealed a previously undescribed evolution pattern of ctDNA in HCC. A novel multiomics-based signature was developed to identify high-risk relapse populations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37215501"
                        open
                      >
                        <summary class="article-expander__title">An unusual ectopic thymoma clonal evolution analysis: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37215501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Thymomas and thymic carcinomas are rare and primary tumors of the mediastinum which is derived from the thymic epithelium. Thymomas are the most common primary anterior mediastinal tumor, while ectopic thymomas are rarer. Mutational profiles of ectopic thymomas may help expand our understanding of the occurrence and treatment options of these tumors. In this report, we sought to elucidate the mutational profiles of two ectopic thymoma nodules to gain deeper understanding of the molecular genetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37217808"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing reveals the metastatic potential of hepatocellular carcinoma from the perspective of tumor and circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37217808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We characterized the genomic alterations during HCC vascular invasion and revealed a previously undescribed evolution pattern of ctDNA in HCC. A novel multiomics-based signature was developed to identify high-risk relapse populations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37215501"
                        open
                      >
                        <summary class="article-expander__title">An unusual ectopic thymoma clonal evolution analysis: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37215501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Thymomas and thymic carcinomas are rare and primary tumors of the mediastinum which is derived from the thymic epithelium. Thymomas are the most common primary anterior mediastinal tumor, while ectopic thymomas are rarer. Mutational profiles of ectopic thymomas may help expand our understanding of the occurrence and treatment options of these tumors. In this report, we sought to elucidate the mutational profiles of two ectopic thymoma nodules to gain deeper understanding of the molecular genetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37217808"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing reveals the metastatic potential of hepatocellular carcinoma from the perspective of tumor and circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37217808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We characterized the genomic alterations during HCC vascular invasion and revealed a previously undescribed evolution pattern of ctDNA in HCC. A novel multiomics-based signature was developed to identify high-risk relapse populations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37215501"
                        open
                      >
                        <summary class="article-expander__title">An unusual ectopic thymoma clonal evolution analysis: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37215501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Thymomas and thymic carcinomas are rare and primary tumors of the mediastinum which is derived from the thymic epithelium. Thymomas are the most common primary anterior mediastinal tumor, while ectopic thymomas are rarer. Mutational profiles of ectopic thymomas may help expand our understanding of the occurrence and treatment options of these tumors. In this report, we sought to elucidate the mutational profiles of two ectopic thymoma nodules to gain deeper understanding of the molecular genetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37217808"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing reveals the metastatic potential of hepatocellular carcinoma from the perspective of tumor and circulating tumor DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37217808/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We characterized the genomic alterations during HCC vascular invasion and revealed a previously undescribed evolution pattern of ctDNA in HCC. A novel multiomics-based signature was developed to identify high-risk relapse populations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37215501"
                        open
                      >
                        <summary class="article-expander__title">An unusual ectopic thymoma clonal evolution analysis: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37215501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Thymomas and thymic carcinomas are rare and primary tumors of the mediastinum which is derived from the thymic epithelium. Thymomas are the most common primary anterior mediastinal tumor, while ectopic thymomas are rarer. Mutational profiles of ectopic thymomas may help expand our understanding of the occurrence and treatment options of these tumors. In this report, we sought to elucidate the mutational profiles of two ectopic thymoma nodules to gain deeper understanding of the molecular genetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37215501"
                        open
                      >
                        <summary class="article-expander__title">An unusual ectopic thymoma clonal evolution analysis: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37215501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Thymomas and thymic carcinomas are rare and primary tumors of the mediastinum which is derived from the thymic epithelium. Thymomas are the most common primary anterior mediastinal tumor, while ectopic thymomas are rarer. Mutational profiles of ectopic thymomas may help expand our understanding of the occurrence and treatment options of these tumors. In this report, we sought to elucidate the mutational profiles of two ectopic thymoma nodules to gain deeper understanding of the molecular genetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-05-21 datetime="2023-05-21T10:00:00.000Z">2023-05-21</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-05-21 datetime="2023-05-21T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37211431"
                        open
                      >
                        <summary class="article-expander__title">Morphology of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37211431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal haematological malignancies first described by Dameshek in 1957. The Philadelphia-negative MPN that will be described are polycythaemia vera (PV), essential thrombocythaemia (ET), pre-fibrotic myelofibrosis and primary myelofibrosis (PMF). The blood and bone marrow morphology are essential in diagnosis, for WHO classification, establishing a baseline, monitoring response to treatment and identifying changes that may indicate disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37211431"
                        open
                      >
                        <summary class="article-expander__title">Morphology of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37211431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal haematological malignancies first described by Dameshek in 1957. The Philadelphia-negative MPN that will be described are polycythaemia vera (PV), essential thrombocythaemia (ET), pre-fibrotic myelofibrosis and primary myelofibrosis (PMF). The blood and bone marrow morphology are essential in diagnosis, for WHO classification, establishing a baseline, monitoring response to treatment and identifying changes that may indicate disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37211431"
                        open
                      >
                        <summary class="article-expander__title">Morphology of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37211431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal haematological malignancies first described by Dameshek in 1957. The Philadelphia-negative MPN that will be described are polycythaemia vera (PV), essential thrombocythaemia (ET), pre-fibrotic myelofibrosis and primary myelofibrosis (PMF). The blood and bone marrow morphology are essential in diagnosis, for WHO classification, establishing a baseline, monitoring response to treatment and identifying changes that may indicate disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37211431"
                        open
                      >
                        <summary class="article-expander__title">Morphology of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37211431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal haematological malignancies first described by Dameshek in 1957. The Philadelphia-negative MPN that will be described are polycythaemia vera (PV), essential thrombocythaemia (ET), pre-fibrotic myelofibrosis and primary myelofibrosis (PMF). The blood and bone marrow morphology are essential in diagnosis, for WHO classification, establishing a baseline, monitoring response to treatment and identifying changes that may indicate disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37211431"
                        open
                      >
                        <summary class="article-expander__title">Morphology of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37211431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal haematological malignancies first described by Dameshek in 1957. The Philadelphia-negative MPN that will be described are polycythaemia vera (PV), essential thrombocythaemia (ET), pre-fibrotic myelofibrosis and primary myelofibrosis (PMF). The blood and bone marrow morphology are essential in diagnosis, for WHO classification, establishing a baseline, monitoring response to treatment and identifying changes that may indicate disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37211431"
                        open
                      >
                        <summary class="article-expander__title">Morphology of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37211431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal haematological malignancies first described by Dameshek in 1957. The Philadelphia-negative MPN that will be described are polycythaemia vera (PV), essential thrombocythaemia (ET), pre-fibrotic myelofibrosis and primary myelofibrosis (PMF). The blood and bone marrow morphology are essential in diagnosis, for WHO classification, establishing a baseline, monitoring response to treatment and identifying changes that may indicate disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37211431"
                        open
                      >
                        <summary class="article-expander__title">Morphology of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37211431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal haematological malignancies first described by Dameshek in 1957. The Philadelphia-negative MPN that will be described are polycythaemia vera (PV), essential thrombocythaemia (ET), pre-fibrotic myelofibrosis and primary myelofibrosis (PMF). The blood and bone marrow morphology are essential in diagnosis, for WHO classification, establishing a baseline, monitoring response to treatment and identifying changes that may indicate disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37211431"
                        open
                      >
                        <summary class="article-expander__title">Morphology of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37211431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal haematological malignancies first described by Dameshek in 1957. The Philadelphia-negative MPN that will be described are polycythaemia vera (PV), essential thrombocythaemia (ET), pre-fibrotic myelofibrosis and primary myelofibrosis (PMF). The blood and bone marrow morphology are essential in diagnosis, for WHO classification, establishing a baseline, monitoring response to treatment and identifying changes that may indicate disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37211431"
                        open
                      >
                        <summary class="article-expander__title">Morphology of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37211431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal haematological malignancies first described by Dameshek in 1957. The Philadelphia-negative MPN that will be described are polycythaemia vera (PV), essential thrombocythaemia (ET), pre-fibrotic myelofibrosis and primary myelofibrosis (PMF). The blood and bone marrow morphology are essential in diagnosis, for WHO classification, establishing a baseline, monitoring response to treatment and identifying changes that may indicate disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37211431"
                        open
                      >
                        <summary class="article-expander__title">Morphology of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37211431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230527210351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal haematological malignancies first described by Dameshek in 1957. The Philadelphia-negative MPN that will be described are polycythaemia vera (PV), essential thrombocythaemia (ET), pre-fibrotic myelofibrosis and primary myelofibrosis (PMF). The blood and bone marrow morphology are essential in diagnosis, for WHO classification, establishing a baseline, monitoring response to treatment and identifying changes that may indicate disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37211431"
                        open
                      >
                        <summary class="article-expander__title">Morphology of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37211431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal haematological malignancies first described by Dameshek in 1957. The Philadelphia-negative MPN that will be described are polycythaemia vera (PV), essential thrombocythaemia (ET), pre-fibrotic myelofibrosis and primary myelofibrosis (PMF). The blood and bone marrow morphology are essential in diagnosis, for WHO classification, establishing a baseline, monitoring response to treatment and identifying changes that may indicate disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37211431"
                        open
                      >
                        <summary class="article-expander__title">Morphology of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37211431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal haematological malignancies first described by Dameshek in 1957. The Philadelphia-negative MPN that will be described are polycythaemia vera (PV), essential thrombocythaemia (ET), pre-fibrotic myelofibrosis and primary myelofibrosis (PMF). The blood and bone marrow morphology are essential in diagnosis, for WHO classification, establishing a baseline, monitoring response to treatment and identifying changes that may indicate disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37211431"
                        open
                      >
                        <summary class="article-expander__title">Morphology of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37211431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal haematological malignancies first described by Dameshek in 1957. The Philadelphia-negative MPN that will be described are polycythaemia vera (PV), essential thrombocythaemia (ET), pre-fibrotic myelofibrosis and primary myelofibrosis (PMF). The blood and bone marrow morphology are essential in diagnosis, for WHO classification, establishing a baseline, monitoring response to treatment and identifying changes that may indicate disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37211431"
                        open
                      >
                        <summary class="article-expander__title">Morphology of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37211431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal haematological malignancies first described by Dameshek in 1957. The Philadelphia-negative MPN that will be described are polycythaemia vera (PV), essential thrombocythaemia (ET), pre-fibrotic myelofibrosis and primary myelofibrosis (PMF). The blood and bone marrow morphology are essential in diagnosis, for WHO classification, establishing a baseline, monitoring response to treatment and identifying changes that may indicate disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37211431"
                        open
                      >
                        <summary class="article-expander__title">Morphology of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37211431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal haematological malignancies first described by Dameshek in 1957. The Philadelphia-negative MPN that will be described are polycythaemia vera (PV), essential thrombocythaemia (ET), pre-fibrotic myelofibrosis and primary myelofibrosis (PMF). The blood and bone marrow morphology are essential in diagnosis, for WHO classification, establishing a baseline, monitoring response to treatment and identifying changes that may indicate disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-05-19 datetime="2023-05-19T10:00:00.000Z">2023-05-19</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-05-19 datetime="2023-05-19T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37207655"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary states and trajectories characterized by distinct pathways stratify patients with ovarian high grade serous carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37207655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian high-grade serous carcinoma (HGSC) is typically diagnosed at an advanced stage, with multiple genetically heterogeneous clones existing in the tumors long before therapeutic intervention. Herein we integrate clonal composition and topology using whole-genome sequencing data from 510 samples of 148 patients with HGSC in the prospective, longitudinal, multiregion DECIDER study. Our results reveal three evolutionary states, which have distinct features in genomics, pathways, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37207655"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary states and trajectories characterized by distinct pathways stratify patients with ovarian high grade serous carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37207655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian high-grade serous carcinoma (HGSC) is typically diagnosed at an advanced stage, with multiple genetically heterogeneous clones existing in the tumors long before therapeutic intervention. Herein we integrate clonal composition and topology using whole-genome sequencing data from 510 samples of 148 patients with HGSC in the prospective, longitudinal, multiregion DECIDER study. Our results reveal three evolutionary states, which have distinct features in genomics, pathways, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37207655"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary states and trajectories characterized by distinct pathways stratify patients with ovarian high grade serous carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37207655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian high-grade serous carcinoma (HGSC) is typically diagnosed at an advanced stage, with multiple genetically heterogeneous clones existing in the tumors long before therapeutic intervention. Herein we integrate clonal composition and topology using whole-genome sequencing data from 510 samples of 148 patients with HGSC in the prospective, longitudinal, multiregion DECIDER study. Our results reveal three evolutionary states, which have distinct features in genomics, pathways, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37207655"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary states and trajectories characterized by distinct pathways stratify patients with ovarian high grade serous carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37207655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian high-grade serous carcinoma (HGSC) is typically diagnosed at an advanced stage, with multiple genetically heterogeneous clones existing in the tumors long before therapeutic intervention. Herein we integrate clonal composition and topology using whole-genome sequencing data from 510 samples of 148 patients with HGSC in the prospective, longitudinal, multiregion DECIDER study. Our results reveal three evolutionary states, which have distinct features in genomics, pathways, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37205910"
                        open
                      >
                        <summary class="article-expander__title">Applying single cell multi-omic analyses to understand treatment resistance in pediatric high grade glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37205910/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Despite improvements in cancer patient outcomes seen in the past decade, tumor resistance to therapy remains a major impediment to achieving durable clinical responses. Intratumoral heterogeneity related to genetic, epigenetic, transcriptomic, proteomic, and metabolic differences between individual cancer cells has emerged as a driver of therapeutic resistance. This cell to cell heterogeneity can be assessed using single cell profiling technologies that enable the identification of tumor cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37207655"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary states and trajectories characterized by distinct pathways stratify patients with ovarian high grade serous carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37207655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian high-grade serous carcinoma (HGSC) is typically diagnosed at an advanced stage, with multiple genetically heterogeneous clones existing in the tumors long before therapeutic intervention. Herein we integrate clonal composition and topology using whole-genome sequencing data from 510 samples of 148 patients with HGSC in the prospective, longitudinal, multiregion DECIDER study. Our results reveal three evolutionary states, which have distinct features in genomics, pathways, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37205910"
                        open
                      >
                        <summary class="article-expander__title">Applying single cell multi-omic analyses to understand treatment resistance in pediatric high grade glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37205910/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Despite improvements in cancer patient outcomes seen in the past decade, tumor resistance to therapy remains a major impediment to achieving durable clinical responses. Intratumoral heterogeneity related to genetic, epigenetic, transcriptomic, proteomic, and metabolic differences between individual cancer cells has emerged as a driver of therapeutic resistance. This cell to cell heterogeneity can be assessed using single cell profiling technologies that enable the identification of tumor cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37207655"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary states and trajectories characterized by distinct pathways stratify patients with ovarian high grade serous carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37207655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian high-grade serous carcinoma (HGSC) is typically diagnosed at an advanced stage, with multiple genetically heterogeneous clones existing in the tumors long before therapeutic intervention. Herein we integrate clonal composition and topology using whole-genome sequencing data from 510 samples of 148 patients with HGSC in the prospective, longitudinal, multiregion DECIDER study. Our results reveal three evolutionary states, which have distinct features in genomics, pathways, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37205910"
                        open
                      >
                        <summary class="article-expander__title">Applying single cell multi-omic analyses to understand treatment resistance in pediatric high grade glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37205910/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Despite improvements in cancer patient outcomes seen in the past decade, tumor resistance to therapy remains a major impediment to achieving durable clinical responses. Intratumoral heterogeneity related to genetic, epigenetic, transcriptomic, proteomic, and metabolic differences between individual cancer cells has emerged as a driver of therapeutic resistance. This cell to cell heterogeneity can be assessed using single cell profiling technologies that enable the identification of tumor cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37207655"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary states and trajectories characterized by distinct pathways stratify patients with ovarian high grade serous carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37207655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian high-grade serous carcinoma (HGSC) is typically diagnosed at an advanced stage, with multiple genetically heterogeneous clones existing in the tumors long before therapeutic intervention. Herein we integrate clonal composition and topology using whole-genome sequencing data from 510 samples of 148 patients with HGSC in the prospective, longitudinal, multiregion DECIDER study. Our results reveal three evolutionary states, which have distinct features in genomics, pathways, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37205910"
                        open
                      >
                        <summary class="article-expander__title">Applying single cell multi-omic analyses to understand treatment resistance in pediatric high grade glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37205910/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Despite improvements in cancer patient outcomes seen in the past decade, tumor resistance to therapy remains a major impediment to achieving durable clinical responses. Intratumoral heterogeneity related to genetic, epigenetic, transcriptomic, proteomic, and metabolic differences between individual cancer cells has emerged as a driver of therapeutic resistance. This cell to cell heterogeneity can be assessed using single cell profiling technologies that enable the identification of tumor cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37207655"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary states and trajectories characterized by distinct pathways stratify patients with ovarian high grade serous carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37207655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian high-grade serous carcinoma (HGSC) is typically diagnosed at an advanced stage, with multiple genetically heterogeneous clones existing in the tumors long before therapeutic intervention. Herein we integrate clonal composition and topology using whole-genome sequencing data from 510 samples of 148 patients with HGSC in the prospective, longitudinal, multiregion DECIDER study. Our results reveal three evolutionary states, which have distinct features in genomics, pathways, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37205910"
                        open
                      >
                        <summary class="article-expander__title">Applying single cell multi-omic analyses to understand treatment resistance in pediatric high grade glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37205910/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Despite improvements in cancer patient outcomes seen in the past decade, tumor resistance to therapy remains a major impediment to achieving durable clinical responses. Intratumoral heterogeneity related to genetic, epigenetic, transcriptomic, proteomic, and metabolic differences between individual cancer cells has emerged as a driver of therapeutic resistance. This cell to cell heterogeneity can be assessed using single cell profiling technologies that enable the identification of tumor cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37207655"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary states and trajectories characterized by distinct pathways stratify patients with ovarian high grade serous carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37207655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230527210351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian high-grade serous carcinoma (HGSC) is typically diagnosed at an advanced stage, with multiple genetically heterogeneous clones existing in the tumors long before therapeutic intervention. Herein we integrate clonal composition and topology using whole-genome sequencing data from 510 samples of 148 patients with HGSC in the prospective, longitudinal, multiregion DECIDER study. Our results reveal three evolutionary states, which have distinct features in genomics, pathways, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37205910"
                        open
                      >
                        <summary class="article-expander__title">Applying single cell multi-omic analyses to understand treatment resistance in pediatric high grade glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37205910/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230527210351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Despite improvements in cancer patient outcomes seen in the past decade, tumor resistance to therapy remains a major impediment to achieving durable clinical responses. Intratumoral heterogeneity related to genetic, epigenetic, transcriptomic, proteomic, and metabolic differences between individual cancer cells has emerged as a driver of therapeutic resistance. This cell to cell heterogeneity can be assessed using single cell profiling technologies that enable the identification of tumor cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37207655"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary states and trajectories characterized by distinct pathways stratify patients with ovarian high grade serous carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37207655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian high-grade serous carcinoma (HGSC) is typically diagnosed at an advanced stage, with multiple genetically heterogeneous clones existing in the tumors long before therapeutic intervention. Herein we integrate clonal composition and topology using whole-genome sequencing data from 510 samples of 148 patients with HGSC in the prospective, longitudinal, multiregion DECIDER study. Our results reveal three evolutionary states, which have distinct features in genomics, pathways, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37205910"
                        open
                      >
                        <summary class="article-expander__title">Applying single cell multi-omic analyses to understand treatment resistance in pediatric high grade glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37205910/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Despite improvements in cancer patient outcomes seen in the past decade, tumor resistance to therapy remains a major impediment to achieving durable clinical responses. Intratumoral heterogeneity related to genetic, epigenetic, transcriptomic, proteomic, and metabolic differences between individual cancer cells has emerged as a driver of therapeutic resistance. This cell to cell heterogeneity can be assessed using single cell profiling technologies that enable the identification of tumor cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37207655"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary states and trajectories characterized by distinct pathways stratify patients with ovarian high grade serous carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37207655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian high-grade serous carcinoma (HGSC) is typically diagnosed at an advanced stage, with multiple genetically heterogeneous clones existing in the tumors long before therapeutic intervention. Herein we integrate clonal composition and topology using whole-genome sequencing data from 510 samples of 148 patients with HGSC in the prospective, longitudinal, multiregion DECIDER study. Our results reveal three evolutionary states, which have distinct features in genomics, pathways, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37205910"
                        open
                      >
                        <summary class="article-expander__title">Applying single cell multi-omic analyses to understand treatment resistance in pediatric high grade glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37205910/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Despite improvements in cancer patient outcomes seen in the past decade, tumor resistance to therapy remains a major impediment to achieving durable clinical responses. Intratumoral heterogeneity related to genetic, epigenetic, transcriptomic, proteomic, and metabolic differences between individual cancer cells has emerged as a driver of therapeutic resistance. This cell to cell heterogeneity can be assessed using single cell profiling technologies that enable the identification of tumor cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37207655"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary states and trajectories characterized by distinct pathways stratify patients with ovarian high grade serous carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37207655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian high-grade serous carcinoma (HGSC) is typically diagnosed at an advanced stage, with multiple genetically heterogeneous clones existing in the tumors long before therapeutic intervention. Herein we integrate clonal composition and topology using whole-genome sequencing data from 510 samples of 148 patients with HGSC in the prospective, longitudinal, multiregion DECIDER study. Our results reveal three evolutionary states, which have distinct features in genomics, pathways, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37205910"
                        open
                      >
                        <summary class="article-expander__title">Applying single cell multi-omic analyses to understand treatment resistance in pediatric high grade glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37205910/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Despite improvements in cancer patient outcomes seen in the past decade, tumor resistance to therapy remains a major impediment to achieving durable clinical responses. Intratumoral heterogeneity related to genetic, epigenetic, transcriptomic, proteomic, and metabolic differences between individual cancer cells has emerged as a driver of therapeutic resistance. This cell to cell heterogeneity can be assessed using single cell profiling technologies that enable the identification of tumor cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37207655"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary states and trajectories characterized by distinct pathways stratify patients with ovarian high grade serous carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37207655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian high-grade serous carcinoma (HGSC) is typically diagnosed at an advanced stage, with multiple genetically heterogeneous clones existing in the tumors long before therapeutic intervention. Herein we integrate clonal composition and topology using whole-genome sequencing data from 510 samples of 148 patients with HGSC in the prospective, longitudinal, multiregion DECIDER study. Our results reveal three evolutionary states, which have distinct features in genomics, pathways, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37205910"
                        open
                      >
                        <summary class="article-expander__title">Applying single cell multi-omic analyses to understand treatment resistance in pediatric high grade glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37205910/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Despite improvements in cancer patient outcomes seen in the past decade, tumor resistance to therapy remains a major impediment to achieving durable clinical responses. Intratumoral heterogeneity related to genetic, epigenetic, transcriptomic, proteomic, and metabolic differences between individual cancer cells has emerged as a driver of therapeutic resistance. This cell to cell heterogeneity can be assessed using single cell profiling technologies that enable the identification of tumor cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37207655"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary states and trajectories characterized by distinct pathways stratify patients with ovarian high grade serous carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37207655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian high-grade serous carcinoma (HGSC) is typically diagnosed at an advanced stage, with multiple genetically heterogeneous clones existing in the tumors long before therapeutic intervention. Herein we integrate clonal composition and topology using whole-genome sequencing data from 510 samples of 148 patients with HGSC in the prospective, longitudinal, multiregion DECIDER study. Our results reveal three evolutionary states, which have distinct features in genomics, pathways, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37205910"
                        open
                      >
                        <summary class="article-expander__title">Applying single cell multi-omic analyses to understand treatment resistance in pediatric high grade glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37205910/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Despite improvements in cancer patient outcomes seen in the past decade, tumor resistance to therapy remains a major impediment to achieving durable clinical responses. Intratumoral heterogeneity related to genetic, epigenetic, transcriptomic, proteomic, and metabolic differences between individual cancer cells has emerged as a driver of therapeutic resistance. This cell to cell heterogeneity can be assessed using single cell profiling technologies that enable the identification of tumor cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37207655"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary states and trajectories characterized by distinct pathways stratify patients with ovarian high grade serous carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37207655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230521210056&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian high-grade serous carcinoma (HGSC) is typically diagnosed at an advanced stage, with multiple genetically heterogeneous clones existing in the tumors long before therapeutic intervention. Herein we integrate clonal composition and topology using whole-genome sequencing data from 510 samples of 148 patients with HGSC in the prospective, longitudinal, multiregion DECIDER study. Our results reveal three evolutionary states, which have distinct features in genomics, pathways, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37205910"
                        open
                      >
                        <summary class="article-expander__title">Applying single cell multi-omic analyses to understand treatment resistance in pediatric high grade glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37205910/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230521210056&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Despite improvements in cancer patient outcomes seen in the past decade, tumor resistance to therapy remains a major impediment to achieving durable clinical responses. Intratumoral heterogeneity related to genetic, epigenetic, transcriptomic, proteomic, and metabolic differences between individual cancer cells has emerged as a driver of therapeutic resistance. This cell to cell heterogeneity can be assessed using single cell profiling technologies that enable the identification of tumor cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37207655"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary states and trajectories characterized by distinct pathways stratify patients with ovarian high grade serous carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37207655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian high-grade serous carcinoma (HGSC) is typically diagnosed at an advanced stage, with multiple genetically heterogeneous clones existing in the tumors long before therapeutic intervention. Herein we integrate clonal composition and topology using whole-genome sequencing data from 510 samples of 148 patients with HGSC in the prospective, longitudinal, multiregion DECIDER study. Our results reveal three evolutionary states, which have distinct features in genomics, pathways, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37205910"
                        open
                      >
                        <summary class="article-expander__title">Applying single cell multi-omic analyses to understand treatment resistance in pediatric high grade glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37205910/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Despite improvements in cancer patient outcomes seen in the past decade, tumor resistance to therapy remains a major impediment to achieving durable clinical responses. Intratumoral heterogeneity related to genetic, epigenetic, transcriptomic, proteomic, and metabolic differences between individual cancer cells has emerged as a driver of therapeutic resistance. This cell to cell heterogeneity can be assessed using single cell profiling technologies that enable the identification of tumor cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37205910"
                        open
                      >
                        <summary class="article-expander__title">Applying single cell multi-omic analyses to understand treatment resistance in pediatric high grade glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37205910/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Despite improvements in cancer patient outcomes seen in the past decade, tumor resistance to therapy remains a major impediment to achieving durable clinical responses. Intratumoral heterogeneity related to genetic, epigenetic, transcriptomic, proteomic, and metabolic differences between individual cancer cells has emerged as a driver of therapeutic resistance. This cell to cell heterogeneity can be assessed using single cell profiling technologies that enable the identification of tumor cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-05-17 datetime="2023-05-17T10:00:00.000Z">2023-05-17</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-05-17 datetime="2023-05-17T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230615210421&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230614210352&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230613210413&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230612210456&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230611210824&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230610211246&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230609210213&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230608210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230527210349&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230527210349&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230527210349&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230527210349&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230521210055&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230521210055&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230521210055&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230521210055&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37197843"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential Predicts Adverse Outcomes in Patients With Atherosclerotic Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37197843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is independently associated with adverse outcomes in individuals with established ASCVD, with especially high risks observed in TET2 and SF3B1/SRSF2/U2AF1 CHIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37196833"
                        open
                      >
                        <summary class="article-expander__title">Dissecting sources of variability in patient response to targeted therapy: anti-HER2 therapies as a case study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37196833/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND IMPLICATIONS: The platform can dissect sources of variability in response to target therapy, which may facilitate decision-making during drug development.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37196833"
                        open
                      >
                        <summary class="article-expander__title">Dissecting sources of variability in patient response to targeted therapy: anti-HER2 therapies as a case study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37196833/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND IMPLICATIONS: The platform can dissect sources of variability in response to target therapy, which may facilitate decision-making during drug development.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37196833"
                        open
                      >
                        <summary class="article-expander__title">Dissecting sources of variability in patient response to targeted therapy: anti-HER2 therapies as a case study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37196833/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND IMPLICATIONS: The platform can dissect sources of variability in response to target therapy, which may facilitate decision-making during drug development.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37194418"
                        open
                      >
                        <summary class="article-expander__title">Mutational evolution after chemotherapy-progression in metastatic colorectal cancer revealed by circulating tumor DNA analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37194418/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging new mutations after treatment can provide clues to acquired resistant mechanisms. Circulating tumor DNA (ctDNA) sequencing has enabled noninvasive repeated tumor mutational profiling. We aimed to investigate newly emerging mutations in ctDNA after disease progression in metastatic colorectal cancer (mCRC). Blood samples were prospectively collected from mCRC patients receiving palliative chemotherapy before treatment and at radiological evaluations. ctDNA from pretreatment and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37196833"
                        open
                      >
                        <summary class="article-expander__title">Dissecting sources of variability in patient response to targeted therapy: anti-HER2 therapies as a case study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37196833/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND IMPLICATIONS: The platform can dissect sources of variability in response to target therapy, which may facilitate decision-making during drug development.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37194418"
                        open
                      >
                        <summary class="article-expander__title">Mutational evolution after chemotherapy-progression in metastatic colorectal cancer revealed by circulating tumor DNA analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37194418/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging new mutations after treatment can provide clues to acquired resistant mechanisms. Circulating tumor DNA (ctDNA) sequencing has enabled noninvasive repeated tumor mutational profiling. We aimed to investigate newly emerging mutations in ctDNA after disease progression in metastatic colorectal cancer (mCRC). Blood samples were prospectively collected from mCRC patients receiving palliative chemotherapy before treatment and at radiological evaluations. ctDNA from pretreatment and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37196833"
                        open
                      >
                        <summary class="article-expander__title">Dissecting sources of variability in patient response to targeted therapy: anti-HER2 therapies as a case study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37196833/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230527210351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND IMPLICATIONS: The platform can dissect sources of variability in response to target therapy, which may facilitate decision-making during drug development.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230527210351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230527210351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230527210351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37194418"
                        open
                      >
                        <summary class="article-expander__title">Mutational evolution after chemotherapy-progression in metastatic colorectal cancer revealed by circulating tumor DNA analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37194418/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230527210351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging new mutations after treatment can provide clues to acquired resistant mechanisms. Circulating tumor DNA (ctDNA) sequencing has enabled noninvasive repeated tumor mutational profiling. We aimed to investigate newly emerging mutations in ctDNA after disease progression in metastatic colorectal cancer (mCRC). Blood samples were prospectively collected from mCRC patients receiving palliative chemotherapy before treatment and at radiological evaluations. ctDNA from pretreatment and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37196833"
                        open
                      >
                        <summary class="article-expander__title">Dissecting sources of variability in patient response to targeted therapy: anti-HER2 therapies as a case study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37196833/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND IMPLICATIONS: The platform can dissect sources of variability in response to target therapy, which may facilitate decision-making during drug development.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37194418"
                        open
                      >
                        <summary class="article-expander__title">Mutational evolution after chemotherapy-progression in metastatic colorectal cancer revealed by circulating tumor DNA analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37194418/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging new mutations after treatment can provide clues to acquired resistant mechanisms. Circulating tumor DNA (ctDNA) sequencing has enabled noninvasive repeated tumor mutational profiling. We aimed to investigate newly emerging mutations in ctDNA after disease progression in metastatic colorectal cancer (mCRC). Blood samples were prospectively collected from mCRC patients receiving palliative chemotherapy before treatment and at radiological evaluations. ctDNA from pretreatment and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37196833"
                        open
                      >
                        <summary class="article-expander__title">Dissecting sources of variability in patient response to targeted therapy: anti-HER2 therapies as a case study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37196833/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND IMPLICATIONS: The platform can dissect sources of variability in response to target therapy, which may facilitate decision-making during drug development.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37194418"
                        open
                      >
                        <summary class="article-expander__title">Mutational evolution after chemotherapy-progression in metastatic colorectal cancer revealed by circulating tumor DNA analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37194418/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging new mutations after treatment can provide clues to acquired resistant mechanisms. Circulating tumor DNA (ctDNA) sequencing has enabled noninvasive repeated tumor mutational profiling. We aimed to investigate newly emerging mutations in ctDNA after disease progression in metastatic colorectal cancer (mCRC). Blood samples were prospectively collected from mCRC patients receiving palliative chemotherapy before treatment and at radiological evaluations. ctDNA from pretreatment and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37196833"
                        open
                      >
                        <summary class="article-expander__title">Dissecting sources of variability in patient response to targeted therapy: anti-HER2 therapies as a case study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37196833/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND IMPLICATIONS: The platform can dissect sources of variability in response to target therapy, which may facilitate decision-making during drug development.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37194418"
                        open
                      >
                        <summary class="article-expander__title">Mutational evolution after chemotherapy-progression in metastatic colorectal cancer revealed by circulating tumor DNA analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37194418/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging new mutations after treatment can provide clues to acquired resistant mechanisms. Circulating tumor DNA (ctDNA) sequencing has enabled noninvasive repeated tumor mutational profiling. We aimed to investigate newly emerging mutations in ctDNA after disease progression in metastatic colorectal cancer (mCRC). Blood samples were prospectively collected from mCRC patients receiving palliative chemotherapy before treatment and at radiological evaluations. ctDNA from pretreatment and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37196833"
                        open
                      >
                        <summary class="article-expander__title">Dissecting sources of variability in patient response to targeted therapy: anti-HER2 therapies as a case study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37196833/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND IMPLICATIONS: The platform can dissect sources of variability in response to target therapy, which may facilitate decision-making during drug development.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37194418"
                        open
                      >
                        <summary class="article-expander__title">Mutational evolution after chemotherapy-progression in metastatic colorectal cancer revealed by circulating tumor DNA analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37194418/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging new mutations after treatment can provide clues to acquired resistant mechanisms. Circulating tumor DNA (ctDNA) sequencing has enabled noninvasive repeated tumor mutational profiling. We aimed to investigate newly emerging mutations in ctDNA after disease progression in metastatic colorectal cancer (mCRC). Blood samples were prospectively collected from mCRC patients receiving palliative chemotherapy before treatment and at radiological evaluations. ctDNA from pretreatment and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37196833"
                        open
                      >
                        <summary class="article-expander__title">Dissecting sources of variability in patient response to targeted therapy: anti-HER2 therapies as a case study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37196833/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND IMPLICATIONS: The platform can dissect sources of variability in response to target therapy, which may facilitate decision-making during drug development.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37194418"
                        open
                      >
                        <summary class="article-expander__title">Mutational evolution after chemotherapy-progression in metastatic colorectal cancer revealed by circulating tumor DNA analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37194418/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging new mutations after treatment can provide clues to acquired resistant mechanisms. Circulating tumor DNA (ctDNA) sequencing has enabled noninvasive repeated tumor mutational profiling. We aimed to investigate newly emerging mutations in ctDNA after disease progression in metastatic colorectal cancer (mCRC). Blood samples were prospectively collected from mCRC patients receiving palliative chemotherapy before treatment and at radiological evaluations. ctDNA from pretreatment and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37196833"
                        open
                      >
                        <summary class="article-expander__title">Dissecting sources of variability in patient response to targeted therapy: anti-HER2 therapies as a case study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37196833/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230521210056&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND IMPLICATIONS: The platform can dissect sources of variability in response to target therapy, which may facilitate decision-making during drug development.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230521210056&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230521210056&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230521210056&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37194418"
                        open
                      >
                        <summary class="article-expander__title">Mutational evolution after chemotherapy-progression in metastatic colorectal cancer revealed by circulating tumor DNA analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37194418/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230521210056&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging new mutations after treatment can provide clues to acquired resistant mechanisms. Circulating tumor DNA (ctDNA) sequencing has enabled noninvasive repeated tumor mutational profiling. We aimed to investigate newly emerging mutations in ctDNA after disease progression in metastatic colorectal cancer (mCRC). Blood samples were prospectively collected from mCRC patients receiving palliative chemotherapy before treatment and at radiological evaluations. ctDNA from pretreatment and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37196833"
                        open
                      >
                        <summary class="article-expander__title">Dissecting sources of variability in patient response to targeted therapy: anti-HER2 therapies as a case study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37196833/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND IMPLICATIONS: The platform can dissect sources of variability in response to target therapy, which may facilitate decision-making during drug development.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37194418"
                        open
                      >
                        <summary class="article-expander__title">Mutational evolution after chemotherapy-progression in metastatic colorectal cancer revealed by circulating tumor DNA analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37194418/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging new mutations after treatment can provide clues to acquired resistant mechanisms. Circulating tumor DNA (ctDNA) sequencing has enabled noninvasive repeated tumor mutational profiling. We aimed to investigate newly emerging mutations in ctDNA after disease progression in metastatic colorectal cancer (mCRC). Blood samples were prospectively collected from mCRC patients receiving palliative chemotherapy before treatment and at radiological evaluations. ctDNA from pretreatment and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37196833"
                        open
                      >
                        <summary class="article-expander__title">Dissecting sources of variability in patient response to targeted therapy: anti-HER2 therapies as a case study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37196833/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND IMPLICATIONS: The platform can dissect sources of variability in response to target therapy, which may facilitate decision-making during drug development.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195768"
                        open
                      >
                        <summary class="article-expander__title">Spectrum From Clonal Hematopoiesis to Myelodysplastic Neoplasm/Syndromes and Other Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195768/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) confers a high risk of aging-related diseases and hematologic malignancy. There are still significant knowledge gaps in identifying high-risk patients with CH and managing such patients. In this review, we focus on 3 areas: (1) the natural history of CH; (2) the risks of progression of CH, including CH of indeterminate potential, clonal cytopenia of undetermined significance, and therapy-related CH, to myeloid malignancy; and (3) the challenges and unmet needs of CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195766"
                        open
                      >
                        <summary class="article-expander__title">Epidemiology and Pathogenesis of Myelodysplastic Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a clonal disorder characterized by ineffective hematopoiesis and variable cytopenias with a considerable risk of progression to acute myeloid leukemia. Epidemiological assessment of MDS remains challenging because of evolving classification systems, but the overall incidence in the United States is estimated to be approximately 4 per 100,000 and increases with age. The sequential accumulation of mutations drives disease evolution from asymptomatic clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37194418"
                        open
                      >
                        <summary class="article-expander__title">Mutational evolution after chemotherapy-progression in metastatic colorectal cancer revealed by circulating tumor DNA analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37194418/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging new mutations after treatment can provide clues to acquired resistant mechanisms. Circulating tumor DNA (ctDNA) sequencing has enabled noninvasive repeated tumor mutational profiling. We aimed to investigate newly emerging mutations in ctDNA after disease progression in metastatic colorectal cancer (mCRC). Blood samples were prospectively collected from mCRC patients receiving palliative chemotherapy before treatment and at radiological evaluations. ctDNA from pretreatment and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37196833"
                        open
                      >
                        <summary class="article-expander__title">Dissecting sources of variability in patient response to targeted therapy: anti-HER2 therapies as a case study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37196833/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND IMPLICATIONS: The platform can dissect sources of variability in response to target therapy, which may facilitate decision-making during drug development.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37194418"
                        open
                      >
                        <summary class="article-expander__title">Mutational evolution after chemotherapy-progression in metastatic colorectal cancer revealed by circulating tumor DNA analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37194418/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging new mutations after treatment can provide clues to acquired resistant mechanisms. Circulating tumor DNA (ctDNA) sequencing has enabled noninvasive repeated tumor mutational profiling. We aimed to investigate newly emerging mutations in ctDNA after disease progression in metastatic colorectal cancer (mCRC). Blood samples were prospectively collected from mCRC patients receiving palliative chemotherapy before treatment and at radiological evaluations. ctDNA from pretreatment and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37195777"
                        open
                      >
                        <summary class="article-expander__title">Evolution of Therapeutic Benefit Measurement Criteria in Myelodysplastic Syndromes/Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37195777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes/neoplasms (MDS) are heterogeneous, clonal myeloid neoplasms characterized by ineffective hematopoiesis, progressive cytopenias, and an increased risk of progression to acute myeloid leukemia. The diversity in disease severity, morphology, and genetic landscape challenges not only novel drug development but also therapeutic response assessment. The MDS International Working Group (IWG) response criteria were first published in the year 2000 focusing on measures of blast...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37194418"
                        open
                      >
                        <summary class="article-expander__title">Mutational evolution after chemotherapy-progression in metastatic colorectal cancer revealed by circulating tumor DNA analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37194418/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging new mutations after treatment can provide clues to acquired resistant mechanisms. Circulating tumor DNA (ctDNA) sequencing has enabled noninvasive repeated tumor mutational profiling. We aimed to investigate newly emerging mutations in ctDNA after disease progression in metastatic colorectal cancer (mCRC). Blood samples were prospectively collected from mCRC patients receiving palliative chemotherapy before treatment and at radiological evaluations. ctDNA from pretreatment and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-05-16 datetime="2023-05-16T10:00:00.000Z">2023-05-16</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-05-16 datetime="2023-05-16T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37191835"
                        open
                      >
                        <summary class="article-expander__title">Stem cell regulation and dynamics in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37191835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230607210603&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this issue of PIH, we asked four researchers to write about basic research on the molecular mechanisms of the development of myeloid malignancies, in particular two epigenetic regulation and two space- and time-dependent factors. Regarding epigenomic regulation, Dr. Yang reviewed ASXL1, a polycomb modifier gene that is often mutated in myeloid malignancies, but also in clonal hematopoiesis in healthy elderly people, and Dr. Vu reviewed RNA modifications, which are critical for development and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37191835"
                        open
                      >
                        <summary class="article-expander__title">Stem cell regulation and dynamics in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37191835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230606210835&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this issue of PIH, we asked four researchers to write about basic research on the molecular mechanisms of the development of myeloid malignancies, in particular two epigenetic regulation and two space- and time-dependent factors. Regarding epigenomic regulation, Dr. Yang reviewed ASXL1, a polycomb modifier gene that is often mutated in myeloid malignancies, but also in clonal hematopoiesis in healthy elderly people, and Dr. Vu reviewed RNA modifications, which are critical for development and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37191835"
                        open
                      >
                        <summary class="article-expander__title">Stem cell regulation and dynamics in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37191835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230605210701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this issue of PIH, we asked four researchers to write about basic research on the molecular mechanisms of the development of myeloid malignancies, in particular two epigenetic regulation and two space- and time-dependent factors. Regarding epigenomic regulation, Dr. Yang reviewed ASXL1, a polycomb modifier gene that is often mutated in myeloid malignancies, but also in clonal hematopoiesis in healthy elderly people, and Dr. Vu reviewed RNA modifications, which are critical for development and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37191835"
                        open
                      >
                        <summary class="article-expander__title">Stem cell regulation and dynamics in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37191835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230604210531&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this issue of PIH, we asked four researchers to write about basic research on the molecular mechanisms of the development of myeloid malignancies, in particular two epigenetic regulation and two space- and time-dependent factors. Regarding epigenomic regulation, Dr. Yang reviewed ASXL1, a polycomb modifier gene that is often mutated in myeloid malignancies, but also in clonal hematopoiesis in healthy elderly people, and Dr. Vu reviewed RNA modifications, which are critical for development and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37191835"
                        open
                      >
                        <summary class="article-expander__title">Stem cell regulation and dynamics in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37191835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230603211539&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this issue of PIH, we asked four researchers to write about basic research on the molecular mechanisms of the development of myeloid malignancies, in particular two epigenetic regulation and two space- and time-dependent factors. Regarding epigenomic regulation, Dr. Yang reviewed ASXL1, a polycomb modifier gene that is often mutated in myeloid malignancies, but also in clonal hematopoiesis in healthy elderly people, and Dr. Vu reviewed RNA modifications, which are critical for development and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37191835"
                        open
                      >
                        <summary class="article-expander__title">Stem cell regulation and dynamics in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37191835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this issue of PIH, we asked four researchers to write about basic research on the molecular mechanisms of the development of myeloid malignancies, in particular two epigenetic regulation and two space- and time-dependent factors. Regarding epigenomic regulation, Dr. Yang reviewed ASXL1, a polycomb modifier gene that is often mutated in myeloid malignancies, but also in clonal hematopoiesis in healthy elderly people, and Dr. Vu reviewed RNA modifications, which are critical for development and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37191835"
                        open
                      >
                        <summary class="article-expander__title">Stem cell regulation and dynamics in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37191835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this issue of PIH, we asked four researchers to write about basic research on the molecular mechanisms of the development of myeloid malignancies, in particular two epigenetic regulation and two space- and time-dependent factors. Regarding epigenomic regulation, Dr. Yang reviewed ASXL1, a polycomb modifier gene that is often mutated in myeloid malignancies, but also in clonal hematopoiesis in healthy elderly people, and Dr. Vu reviewed RNA modifications, which are critical for development and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37191835"
                        open
                      >
                        <summary class="article-expander__title">Stem cell regulation and dynamics in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37191835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this issue of PIH, we asked four researchers to write about basic research on the molecular mechanisms of the development of myeloid malignancies, in particular two epigenetic regulation and two space- and time-dependent factors. Regarding epigenomic regulation, Dr. Yang reviewed ASXL1, a polycomb modifier gene that is often mutated in myeloid malignancies, but also in clonal hematopoiesis in healthy elderly people, and Dr. Vu reviewed RNA modifications, which are critical for development and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37191835"
                        open
                      >
                        <summary class="article-expander__title">Stem cell regulation and dynamics in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37191835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this issue of PIH, we asked four researchers to write about basic research on the molecular mechanisms of the development of myeloid malignancies, in particular two epigenetic regulation and two space- and time-dependent factors. Regarding epigenomic regulation, Dr. Yang reviewed ASXL1, a polycomb modifier gene that is often mutated in myeloid malignancies, but also in clonal hematopoiesis in healthy elderly people, and Dr. Vu reviewed RNA modifications, which are critical for development and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37191835"
                        open
                      >
                        <summary class="article-expander__title">Stem cell regulation and dynamics in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37191835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this issue of PIH, we asked four researchers to write about basic research on the molecular mechanisms of the development of myeloid malignancies, in particular two epigenetic regulation and two space- and time-dependent factors. Regarding epigenomic regulation, Dr. Yang reviewed ASXL1, a polycomb modifier gene that is often mutated in myeloid malignancies, but also in clonal hematopoiesis in healthy elderly people, and Dr. Vu reviewed RNA modifications, which are critical for development and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37191835"
                        open
                      >
                        <summary class="article-expander__title">Stem cell regulation and dynamics in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37191835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this issue of PIH, we asked four researchers to write about basic research on the molecular mechanisms of the development of myeloid malignancies, in particular two epigenetic regulation and two space- and time-dependent factors. Regarding epigenomic regulation, Dr. Yang reviewed ASXL1, a polycomb modifier gene that is often mutated in myeloid malignancies, but also in clonal hematopoiesis in healthy elderly people, and Dr. Vu reviewed RNA modifications, which are critical for development and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37191835"
                        open
                      >
                        <summary class="article-expander__title">Stem cell regulation and dynamics in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37191835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230527210349&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this issue of PIH, we asked four researchers to write about basic research on the molecular mechanisms of the development of myeloid malignancies, in particular two epigenetic regulation and two space- and time-dependent factors. Regarding epigenomic regulation, Dr. Yang reviewed ASXL1, a polycomb modifier gene that is often mutated in myeloid malignancies, but also in clonal hematopoiesis in healthy elderly people, and Dr. Vu reviewed RNA modifications, which are critical for development and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37191835"
                        open
                      >
                        <summary class="article-expander__title">Stem cell regulation and dynamics in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37191835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this issue of PIH, we asked four researchers to write about basic research on the molecular mechanisms of the development of myeloid malignancies, in particular two epigenetic regulation and two space- and time-dependent factors. Regarding epigenomic regulation, Dr. Yang reviewed ASXL1, a polycomb modifier gene that is often mutated in myeloid malignancies, but also in clonal hematopoiesis in healthy elderly people, and Dr. Vu reviewed RNA modifications, which are critical for development and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37191835"
                        open
                      >
                        <summary class="article-expander__title">Stem cell regulation and dynamics in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37191835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this issue of PIH, we asked four researchers to write about basic research on the molecular mechanisms of the development of myeloid malignancies, in particular two epigenetic regulation and two space- and time-dependent factors. Regarding epigenomic regulation, Dr. Yang reviewed ASXL1, a polycomb modifier gene that is often mutated in myeloid malignancies, but also in clonal hematopoiesis in healthy elderly people, and Dr. Vu reviewed RNA modifications, which are critical for development and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37191835"
                        open
                      >
                        <summary class="article-expander__title">Stem cell regulation and dynamics in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37191835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this issue of PIH, we asked four researchers to write about basic research on the molecular mechanisms of the development of myeloid malignancies, in particular two epigenetic regulation and two space- and time-dependent factors. Regarding epigenomic regulation, Dr. Yang reviewed ASXL1, a polycomb modifier gene that is often mutated in myeloid malignancies, but also in clonal hematopoiesis in healthy elderly people, and Dr. Vu reviewed RNA modifications, which are critical for development and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37191835"
                        open
                      >
                        <summary class="article-expander__title">Stem cell regulation and dynamics in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37191835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this issue of PIH, we asked four researchers to write about basic research on the molecular mechanisms of the development of myeloid malignancies, in particular two epigenetic regulation and two space- and time-dependent factors. Regarding epigenomic regulation, Dr. Yang reviewed ASXL1, a polycomb modifier gene that is often mutated in myeloid malignancies, but also in clonal hematopoiesis in healthy elderly people, and Dr. Vu reviewed RNA modifications, which are critical for development and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37191835"
                        open
                      >
                        <summary class="article-expander__title">Stem cell regulation and dynamics in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37191835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this issue of PIH, we asked four researchers to write about basic research on the molecular mechanisms of the development of myeloid malignancies, in particular two epigenetic regulation and two space- and time-dependent factors. Regarding epigenomic regulation, Dr. Yang reviewed ASXL1, a polycomb modifier gene that is often mutated in myeloid malignancies, but also in clonal hematopoiesis in healthy elderly people, and Dr. Vu reviewed RNA modifications, which are critical for development and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37191835"
                        open
                      >
                        <summary class="article-expander__title">Stem cell regulation and dynamics in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37191835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230521210055&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this issue of PIH, we asked four researchers to write about basic research on the molecular mechanisms of the development of myeloid malignancies, in particular two epigenetic regulation and two space- and time-dependent factors. Regarding epigenomic regulation, Dr. Yang reviewed ASXL1, a polycomb modifier gene that is often mutated in myeloid malignancies, but also in clonal hematopoiesis in healthy elderly people, and Dr. Vu reviewed RNA modifications, which are critical for development and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37191835"
                        open
                      >
                        <summary class="article-expander__title">Stem cell regulation and dynamics in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37191835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this issue of PIH, we asked four researchers to write about basic research on the molecular mechanisms of the development of myeloid malignancies, in particular two epigenetic regulation and two space- and time-dependent factors. Regarding epigenomic regulation, Dr. Yang reviewed ASXL1, a polycomb modifier gene that is often mutated in myeloid malignancies, but also in clonal hematopoiesis in healthy elderly people, and Dr. Vu reviewed RNA modifications, which are critical for development and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37191835"
                        open
                      >
                        <summary class="article-expander__title">Stem cell regulation and dynamics in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37191835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this issue of PIH, we asked four researchers to write about basic research on the molecular mechanisms of the development of myeloid malignancies, in particular two epigenetic regulation and two space- and time-dependent factors. Regarding epigenomic regulation, Dr. Yang reviewed ASXL1, a polycomb modifier gene that is often mutated in myeloid malignancies, but also in clonal hematopoiesis in healthy elderly people, and Dr. Vu reviewed RNA modifications, which are critical for development and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37191835"
                        open
                      >
                        <summary class="article-expander__title">Stem cell regulation and dynamics in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37191835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this issue of PIH, we asked four researchers to write about basic research on the molecular mechanisms of the development of myeloid malignancies, in particular two epigenetic regulation and two space- and time-dependent factors. Regarding epigenomic regulation, Dr. Yang reviewed ASXL1, a polycomb modifier gene that is often mutated in myeloid malignancies, but also in clonal hematopoiesis in healthy elderly people, and Dr. Vu reviewed RNA modifications, which are critical for development and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37191835"
                        open
                      >
                        <summary class="article-expander__title">Stem cell regulation and dynamics in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37191835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this issue of PIH, we asked four researchers to write about basic research on the molecular mechanisms of the development of myeloid malignancies, in particular two epigenetic regulation and two space- and time-dependent factors. Regarding epigenomic regulation, Dr. Yang reviewed ASXL1, a polycomb modifier gene that is often mutated in myeloid malignancies, but also in clonal hematopoiesis in healthy elderly people, and Dr. Vu reviewed RNA modifications, which are critical for development and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37191835"
                        open
                      >
                        <summary class="article-expander__title">Stem cell regulation and dynamics in myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37191835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this issue of PIH, we asked four researchers to write about basic research on the molecular mechanisms of the development of myeloid malignancies, in particular two epigenetic regulation and two space- and time-dependent factors. Regarding epigenomic regulation, Dr. Yang reviewed ASXL1, a polycomb modifier gene that is often mutated in myeloid malignancies, but also in clonal hematopoiesis in healthy elderly people, and Dr. Vu reviewed RNA modifications, which are critical for development and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37192968"
                        open
                      >
                        <summary class="article-expander__title">Toward a systems-level probing of tumor clonality</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37192968/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer has been described as a genetic disease that clonally evolves in the face of selective pressures imposed by cell-intrinsic and extrinsic factors. Although classical models based on genetic data predominantly propose Darwinian mechanisms of cancer evolution, recent single-cell profiling of cancers has described unprecedented heterogeneity in tumors providing support for alternative models of branched and neutral evolution through both genetic and non-genetic mechanisms. Emerging evidence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37192968"
                        open
                      >
                        <summary class="article-expander__title">Toward a systems-level probing of tumor clonality</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37192968/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230527210351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer has been described as a genetic disease that clonally evolves in the face of selective pressures imposed by cell-intrinsic and extrinsic factors. Although classical models based on genetic data predominantly propose Darwinian mechanisms of cancer evolution, recent single-cell profiling of cancers has described unprecedented heterogeneity in tumors providing support for alternative models of branched and neutral evolution through both genetic and non-genetic mechanisms. Emerging evidence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37192968"
                        open
                      >
                        <summary class="article-expander__title">Toward a systems-level probing of tumor clonality</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37192968/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer has been described as a genetic disease that clonally evolves in the face of selective pressures imposed by cell-intrinsic and extrinsic factors. Although classical models based on genetic data predominantly propose Darwinian mechanisms of cancer evolution, recent single-cell profiling of cancers has described unprecedented heterogeneity in tumors providing support for alternative models of branched and neutral evolution through both genetic and non-genetic mechanisms. Emerging evidence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37192968"
                        open
                      >
                        <summary class="article-expander__title">Toward a systems-level probing of tumor clonality</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37192968/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer has been described as a genetic disease that clonally evolves in the face of selective pressures imposed by cell-intrinsic and extrinsic factors. Although classical models based on genetic data predominantly propose Darwinian mechanisms of cancer evolution, recent single-cell profiling of cancers has described unprecedented heterogeneity in tumors providing support for alternative models of branched and neutral evolution through both genetic and non-genetic mechanisms. Emerging evidence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37192968"
                        open
                      >
                        <summary class="article-expander__title">Toward a systems-level probing of tumor clonality</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37192968/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer has been described as a genetic disease that clonally evolves in the face of selective pressures imposed by cell-intrinsic and extrinsic factors. Although classical models based on genetic data predominantly propose Darwinian mechanisms of cancer evolution, recent single-cell profiling of cancers has described unprecedented heterogeneity in tumors providing support for alternative models of branched and neutral evolution through both genetic and non-genetic mechanisms. Emerging evidence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37192000"
                        open
                      >
                        <summary class="article-expander__title">Human endogenous retrovirus onco-exaptation counters cancer cell senescence through Calbindin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37192000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Increased levels and diversity of human endogenous retrovirus (HERV) transcription characterizes most cancer types, linked with disease outcomes. However, the underlying processes are incompletely understood. We show that elevated transcription of HERVH proviruses predicts survival of lung squamous cell carcinoma (LUSC) and identify an isoform of CALB1, encoding Calbindin, ectopically driven by an upstream HERVH provirus under the control of KLF5, as the mediator of this effect. HERVH-CALB1...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37192968"
                        open
                      >
                        <summary class="article-expander__title">Toward a systems-level probing of tumor clonality</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37192968/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer has been described as a genetic disease that clonally evolves in the face of selective pressures imposed by cell-intrinsic and extrinsic factors. Although classical models based on genetic data predominantly propose Darwinian mechanisms of cancer evolution, recent single-cell profiling of cancers has described unprecedented heterogeneity in tumors providing support for alternative models of branched and neutral evolution through both genetic and non-genetic mechanisms. Emerging evidence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37192000"
                        open
                      >
                        <summary class="article-expander__title">Human endogenous retrovirus onco-exaptation counters cancer cell senescence through Calbindin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37192000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Increased levels and diversity of human endogenous retrovirus (HERV) transcription characterizes most cancer types, linked with disease outcomes. However, the underlying processes are incompletely understood. We show that elevated transcription of HERVH proviruses predicts survival of lung squamous cell carcinoma (LUSC) and identify an isoform of CALB1, encoding Calbindin, ectopically driven by an upstream HERVH provirus under the control of KLF5, as the mediator of this effect. HERVH-CALB1...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37192968"
                        open
                      >
                        <summary class="article-expander__title">Toward a systems-level probing of tumor clonality</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37192968/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer has been described as a genetic disease that clonally evolves in the face of selective pressures imposed by cell-intrinsic and extrinsic factors. Although classical models based on genetic data predominantly propose Darwinian mechanisms of cancer evolution, recent single-cell profiling of cancers has described unprecedented heterogeneity in tumors providing support for alternative models of branched and neutral evolution through both genetic and non-genetic mechanisms. Emerging evidence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37192000"
                        open
                      >
                        <summary class="article-expander__title">Human endogenous retrovirus onco-exaptation counters cancer cell senescence through Calbindin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37192000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Increased levels and diversity of human endogenous retrovirus (HERV) transcription characterizes most cancer types, linked with disease outcomes. However, the underlying processes are incompletely understood. We show that elevated transcription of HERVH proviruses predicts survival of lung squamous cell carcinoma (LUSC) and identify an isoform of CALB1, encoding Calbindin, ectopically driven by an upstream HERVH provirus under the control of KLF5, as the mediator of this effect. HERVH-CALB1...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37192968"
                        open
                      >
                        <summary class="article-expander__title">Toward a systems-level probing of tumor clonality</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37192968/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230521210056&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer has been described as a genetic disease that clonally evolves in the face of selective pressures imposed by cell-intrinsic and extrinsic factors. Although classical models based on genetic data predominantly propose Darwinian mechanisms of cancer evolution, recent single-cell profiling of cancers has described unprecedented heterogeneity in tumors providing support for alternative models of branched and neutral evolution through both genetic and non-genetic mechanisms. Emerging evidence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37192000"
                        open
                      >
                        <summary class="article-expander__title">Human endogenous retrovirus onco-exaptation counters cancer cell senescence through Calbindin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37192000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230521210056&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Increased levels and diversity of human endogenous retrovirus (HERV) transcription characterizes most cancer types, linked with disease outcomes. However, the underlying processes are incompletely understood. We show that elevated transcription of HERVH proviruses predicts survival of lung squamous cell carcinoma (LUSC) and identify an isoform of CALB1, encoding Calbindin, ectopically driven by an upstream HERVH provirus under the control of KLF5, as the mediator of this effect. HERVH-CALB1...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37192968"
                        open
                      >
                        <summary class="article-expander__title">Toward a systems-level probing of tumor clonality</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37192968/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer has been described as a genetic disease that clonally evolves in the face of selective pressures imposed by cell-intrinsic and extrinsic factors. Although classical models based on genetic data predominantly propose Darwinian mechanisms of cancer evolution, recent single-cell profiling of cancers has described unprecedented heterogeneity in tumors providing support for alternative models of branched and neutral evolution through both genetic and non-genetic mechanisms. Emerging evidence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37192000"
                        open
                      >
                        <summary class="article-expander__title">Human endogenous retrovirus onco-exaptation counters cancer cell senescence through Calbindin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37192000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Increased levels and diversity of human endogenous retrovirus (HERV) transcription characterizes most cancer types, linked with disease outcomes. However, the underlying processes are incompletely understood. We show that elevated transcription of HERVH proviruses predicts survival of lung squamous cell carcinoma (LUSC) and identify an isoform of CALB1, encoding Calbindin, ectopically driven by an upstream HERVH provirus under the control of KLF5, as the mediator of this effect. HERVH-CALB1...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37192968"
                        open
                      >
                        <summary class="article-expander__title">Toward a systems-level probing of tumor clonality</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37192968/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer has been described as a genetic disease that clonally evolves in the face of selective pressures imposed by cell-intrinsic and extrinsic factors. Although classical models based on genetic data predominantly propose Darwinian mechanisms of cancer evolution, recent single-cell profiling of cancers has described unprecedented heterogeneity in tumors providing support for alternative models of branched and neutral evolution through both genetic and non-genetic mechanisms. Emerging evidence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37192000"
                        open
                      >
                        <summary class="article-expander__title">Human endogenous retrovirus onco-exaptation counters cancer cell senescence through Calbindin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37192000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Increased levels and diversity of human endogenous retrovirus (HERV) transcription characterizes most cancer types, linked with disease outcomes. However, the underlying processes are incompletely understood. We show that elevated transcription of HERVH proviruses predicts survival of lung squamous cell carcinoma (LUSC) and identify an isoform of CALB1, encoding Calbindin, ectopically driven by an upstream HERVH provirus under the control of KLF5, as the mediator of this effect. HERVH-CALB1...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37192968"
                        open
                      >
                        <summary class="article-expander__title">Toward a systems-level probing of tumor clonality</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37192968/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer has been described as a genetic disease that clonally evolves in the face of selective pressures imposed by cell-intrinsic and extrinsic factors. Although classical models based on genetic data predominantly propose Darwinian mechanisms of cancer evolution, recent single-cell profiling of cancers has described unprecedented heterogeneity in tumors providing support for alternative models of branched and neutral evolution through both genetic and non-genetic mechanisms. Emerging evidence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37192000"
                        open
                      >
                        <summary class="article-expander__title">Human endogenous retrovirus onco-exaptation counters cancer cell senescence through Calbindin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37192000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Increased levels and diversity of human endogenous retrovirus (HERV) transcription characterizes most cancer types, linked with disease outcomes. However, the underlying processes are incompletely understood. We show that elevated transcription of HERVH proviruses predicts survival of lung squamous cell carcinoma (LUSC) and identify an isoform of CALB1, encoding Calbindin, ectopically driven by an upstream HERVH provirus under the control of KLF5, as the mediator of this effect. HERVH-CALB1...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37192968"
                        open
                      >
                        <summary class="article-expander__title">Toward a systems-level probing of tumor clonality</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37192968/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer has been described as a genetic disease that clonally evolves in the face of selective pressures imposed by cell-intrinsic and extrinsic factors. Although classical models based on genetic data predominantly propose Darwinian mechanisms of cancer evolution, recent single-cell profiling of cancers has described unprecedented heterogeneity in tumors providing support for alternative models of branched and neutral evolution through both genetic and non-genetic mechanisms. Emerging evidence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37192000"
                        open
                      >
                        <summary class="article-expander__title">Human endogenous retrovirus onco-exaptation counters cancer cell senescence through Calbindin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37192000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Increased levels and diversity of human endogenous retrovirus (HERV) transcription characterizes most cancer types, linked with disease outcomes. However, the underlying processes are incompletely understood. We show that elevated transcription of HERVH proviruses predicts survival of lung squamous cell carcinoma (LUSC) and identify an isoform of CALB1, encoding Calbindin, ectopically driven by an upstream HERVH provirus under the control of KLF5, as the mediator of this effect. HERVH-CALB1...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37192000"
                        open
                      >
                        <summary class="article-expander__title">Human endogenous retrovirus onco-exaptation counters cancer cell senescence through Calbindin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37192000/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Increased levels and diversity of human endogenous retrovirus (HERV) transcription characterizes most cancer types, linked with disease outcomes. However, the underlying processes are incompletely understood. We show that elevated transcription of HERVH proviruses predicts survival of lung squamous cell carcinoma (LUSC) and identify an isoform of CALB1, encoding Calbindin, ectopically driven by an upstream HERVH provirus under the control of KLF5, as the mediator of this effect. HERVH-CALB1...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-05-15 datetime="2023-05-15T10:00:00.000Z">2023-05-15</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-05-15 datetime="2023-05-15T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37186988"
                        open
                      >
                        <summary class="article-expander__title">Review of clonal hematopoiesis, subtypes and its role in neoplasia and different morbidities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37186988/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the development of a certain cell lineage which is the cornerstone of hematologic malignancy especially myeloid neoplasms, however, can also be found in old age (6th-7th decade). CH is caused by many different somatic mutations most commonly in DNMT3A, TET2, ASXL1, SF3B1 and TP53. It is detected by different sequencing methods, the most commonly used ones are next generation sequencing (NGS) which can be whole exome, whole genome sequencing or a panel for certain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37186988"
                        open
                      >
                        <summary class="article-expander__title">Review of clonal hematopoiesis, subtypes and its role in neoplasia and different morbidities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37186988/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the development of a certain cell lineage which is the cornerstone of hematologic malignancy especially myeloid neoplasms, however, can also be found in old age (6th-7th decade). CH is caused by many different somatic mutations most commonly in DNMT3A, TET2, ASXL1, SF3B1 and TP53. It is detected by different sequencing methods, the most commonly used ones are next generation sequencing (NGS) which can be whole exome, whole genome sequencing or a panel for certain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37186988"
                        open
                      >
                        <summary class="article-expander__title">Review of clonal hematopoiesis, subtypes and its role in neoplasia and different morbidities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37186988/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the development of a certain cell lineage which is the cornerstone of hematologic malignancy especially myeloid neoplasms, however, can also be found in old age (6th-7th decade). CH is caused by many different somatic mutations most commonly in DNMT3A, TET2, ASXL1, SF3B1 and TP53. It is detected by different sequencing methods, the most commonly used ones are next generation sequencing (NGS) which can be whole exome, whole genome sequencing or a panel for certain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37186988"
                        open
                      >
                        <summary class="article-expander__title">Review of clonal hematopoiesis, subtypes and its role in neoplasia and different morbidities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37186988/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the development of a certain cell lineage which is the cornerstone of hematologic malignancy especially myeloid neoplasms, however, can also be found in old age (6th-7th decade). CH is caused by many different somatic mutations most commonly in DNMT3A, TET2, ASXL1, SF3B1 and TP53. It is detected by different sequencing methods, the most commonly used ones are next generation sequencing (NGS) which can be whole exome, whole genome sequencing or a panel for certain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37186988"
                        open
                      >
                        <summary class="article-expander__title">Review of clonal hematopoiesis, subtypes and its role in neoplasia and different morbidities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37186988/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the development of a certain cell lineage which is the cornerstone of hematologic malignancy especially myeloid neoplasms, however, can also be found in old age (6th-7th decade). CH is caused by many different somatic mutations most commonly in DNMT3A, TET2, ASXL1, SF3B1 and TP53. It is detected by different sequencing methods, the most commonly used ones are next generation sequencing (NGS) which can be whole exome, whole genome sequencing or a panel for certain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37186988"
                        open
                      >
                        <summary class="article-expander__title">Review of clonal hematopoiesis, subtypes and its role in neoplasia and different morbidities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37186988/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the development of a certain cell lineage which is the cornerstone of hematologic malignancy especially myeloid neoplasms, however, can also be found in old age (6th-7th decade). CH is caused by many different somatic mutations most commonly in DNMT3A, TET2, ASXL1, SF3B1 and TP53. It is detected by different sequencing methods, the most commonly used ones are next generation sequencing (NGS) which can be whole exome, whole genome sequencing or a panel for certain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37186988"
                        open
                      >
                        <summary class="article-expander__title">Review of clonal hematopoiesis, subtypes and its role in neoplasia and different morbidities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37186988/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230527210349&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the development of a certain cell lineage which is the cornerstone of hematologic malignancy especially myeloid neoplasms, however, can also be found in old age (6th-7th decade). CH is caused by many different somatic mutations most commonly in DNMT3A, TET2, ASXL1, SF3B1 and TP53. It is detected by different sequencing methods, the most commonly used ones are next generation sequencing (NGS) which can be whole exome, whole genome sequencing or a panel for certain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37186988"
                        open
                      >
                        <summary class="article-expander__title">Review of clonal hematopoiesis, subtypes and its role in neoplasia and different morbidities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37186988/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the development of a certain cell lineage which is the cornerstone of hematologic malignancy especially myeloid neoplasms, however, can also be found in old age (6th-7th decade). CH is caused by many different somatic mutations most commonly in DNMT3A, TET2, ASXL1, SF3B1 and TP53. It is detected by different sequencing methods, the most commonly used ones are next generation sequencing (NGS) which can be whole exome, whole genome sequencing or a panel for certain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37186988"
                        open
                      >
                        <summary class="article-expander__title">Review of clonal hematopoiesis, subtypes and its role in neoplasia and different morbidities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37186988/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the development of a certain cell lineage which is the cornerstone of hematologic malignancy especially myeloid neoplasms, however, can also be found in old age (6th-7th decade). CH is caused by many different somatic mutations most commonly in DNMT3A, TET2, ASXL1, SF3B1 and TP53. It is detected by different sequencing methods, the most commonly used ones are next generation sequencing (NGS) which can be whole exome, whole genome sequencing or a panel for certain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37186988"
                        open
                      >
                        <summary class="article-expander__title">Review of clonal hematopoiesis, subtypes and its role in neoplasia and different morbidities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37186988/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the development of a certain cell lineage which is the cornerstone of hematologic malignancy especially myeloid neoplasms, however, can also be found in old age (6th-7th decade). CH is caused by many different somatic mutations most commonly in DNMT3A, TET2, ASXL1, SF3B1 and TP53. It is detected by different sequencing methods, the most commonly used ones are next generation sequencing (NGS) which can be whole exome, whole genome sequencing or a panel for certain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37186988"
                        open
                      >
                        <summary class="article-expander__title">Review of clonal hematopoiesis, subtypes and its role in neoplasia and different morbidities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37186988/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the development of a certain cell lineage which is the cornerstone of hematologic malignancy especially myeloid neoplasms, however, can also be found in old age (6th-7th decade). CH is caused by many different somatic mutations most commonly in DNMT3A, TET2, ASXL1, SF3B1 and TP53. It is detected by different sequencing methods, the most commonly used ones are next generation sequencing (NGS) which can be whole exome, whole genome sequencing or a panel for certain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37186988"
                        open
                      >
                        <summary class="article-expander__title">Review of clonal hematopoiesis, subtypes and its role in neoplasia and different morbidities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37186988/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the development of a certain cell lineage which is the cornerstone of hematologic malignancy especially myeloid neoplasms, however, can also be found in old age (6th-7th decade). CH is caused by many different somatic mutations most commonly in DNMT3A, TET2, ASXL1, SF3B1 and TP53. It is detected by different sequencing methods, the most commonly used ones are next generation sequencing (NGS) which can be whole exome, whole genome sequencing or a panel for certain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37186988"
                        open
                      >
                        <summary class="article-expander__title">Review of clonal hematopoiesis, subtypes and its role in neoplasia and different morbidities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37186988/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230521210055&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the development of a certain cell lineage which is the cornerstone of hematologic malignancy especially myeloid neoplasms, however, can also be found in old age (6th-7th decade). CH is caused by many different somatic mutations most commonly in DNMT3A, TET2, ASXL1, SF3B1 and TP53. It is detected by different sequencing methods, the most commonly used ones are next generation sequencing (NGS) which can be whole exome, whole genome sequencing or a panel for certain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37186988"
                        open
                      >
                        <summary class="article-expander__title">Review of clonal hematopoiesis, subtypes and its role in neoplasia and different morbidities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37186988/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the development of a certain cell lineage which is the cornerstone of hematologic malignancy especially myeloid neoplasms, however, can also be found in old age (6th-7th decade). CH is caused by many different somatic mutations most commonly in DNMT3A, TET2, ASXL1, SF3B1 and TP53. It is detected by different sequencing methods, the most commonly used ones are next generation sequencing (NGS) which can be whole exome, whole genome sequencing or a panel for certain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37186988"
                        open
                      >
                        <summary class="article-expander__title">Review of clonal hematopoiesis, subtypes and its role in neoplasia and different morbidities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37186988/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the development of a certain cell lineage which is the cornerstone of hematologic malignancy especially myeloid neoplasms, however, can also be found in old age (6th-7th decade). CH is caused by many different somatic mutations most commonly in DNMT3A, TET2, ASXL1, SF3B1 and TP53. It is detected by different sequencing methods, the most commonly used ones are next generation sequencing (NGS) which can be whole exome, whole genome sequencing or a panel for certain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37186988"
                        open
                      >
                        <summary class="article-expander__title">Review of clonal hematopoiesis, subtypes and its role in neoplasia and different morbidities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37186988/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the development of a certain cell lineage which is the cornerstone of hematologic malignancy especially myeloid neoplasms, however, can also be found in old age (6th-7th decade). CH is caused by many different somatic mutations most commonly in DNMT3A, TET2, ASXL1, SF3B1 and TP53. It is detected by different sequencing methods, the most commonly used ones are next generation sequencing (NGS) which can be whole exome, whole genome sequencing or a panel for certain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37186988"
                        open
                      >
                        <summary class="article-expander__title">Review of clonal hematopoiesis, subtypes and its role in neoplasia and different morbidities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37186988/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the development of a certain cell lineage which is the cornerstone of hematologic malignancy especially myeloid neoplasms, however, can also be found in old age (6th-7th decade). CH is caused by many different somatic mutations most commonly in DNMT3A, TET2, ASXL1, SF3B1 and TP53. It is detected by different sequencing methods, the most commonly used ones are next generation sequencing (NGS) which can be whole exome, whole genome sequencing or a panel for certain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37186988"
                        open
                      >
                        <summary class="article-expander__title">Review of clonal hematopoiesis, subtypes and its role in neoplasia and different morbidities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37186988/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the development of a certain cell lineage which is the cornerstone of hematologic malignancy especially myeloid neoplasms, however, can also be found in old age (6th-7th decade). CH is caused by many different somatic mutations most commonly in DNMT3A, TET2, ASXL1, SF3B1 and TP53. It is detected by different sequencing methods, the most commonly used ones are next generation sequencing (NGS) which can be whole exome, whole genome sequencing or a panel for certain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-05-13 datetime="2023-05-13T10:00:00.000Z">2023-05-13</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-05-13 datetime="2023-05-13T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37175518"
                        open
                      >
                        <summary class="article-expander__title">The Potential of Liquid Biopsy in Detection of Endometrial Cancer Biomarkers: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37175518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230602210410&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Endometrial cancer belongs to the most common gynecologic cancer types globally, with increasing incidence. There are numerous ways of classifying different cases. The most recent decade has brought advances in molecular classification, which show more accurate prognostic factors and the possibility of personalised adjuvant treatment. In addition, diagnostic approaches lag behind these advances, with methods causing patients discomfort while lacking the reproducibility of tissue sampling for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37175518"
                        open
                      >
                        <summary class="article-expander__title">The Potential of Liquid Biopsy in Detection of Endometrial Cancer Biomarkers: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37175518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230601210630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Endometrial cancer belongs to the most common gynecologic cancer types globally, with increasing incidence. There are numerous ways of classifying different cases. The most recent decade has brought advances in molecular classification, which show more accurate prognostic factors and the possibility of personalised adjuvant treatment. In addition, diagnostic approaches lag behind these advances, with methods causing patients discomfort while lacking the reproducibility of tissue sampling for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37175518"
                        open
                      >
                        <summary class="article-expander__title">The Potential of Liquid Biopsy in Detection of Endometrial Cancer Biomarkers: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37175518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230531211907&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Endometrial cancer belongs to the most common gynecologic cancer types globally, with increasing incidence. There are numerous ways of classifying different cases. The most recent decade has brought advances in molecular classification, which show more accurate prognostic factors and the possibility of personalised adjuvant treatment. In addition, diagnostic approaches lag behind these advances, with methods causing patients discomfort while lacking the reproducibility of tissue sampling for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37175518"
                        open
                      >
                        <summary class="article-expander__title">The Potential of Liquid Biopsy in Detection of Endometrial Cancer Biomarkers: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37175518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Endometrial cancer belongs to the most common gynecologic cancer types globally, with increasing incidence. There are numerous ways of classifying different cases. The most recent decade has brought advances in molecular classification, which show more accurate prognostic factors and the possibility of personalised adjuvant treatment. In addition, diagnostic approaches lag behind these advances, with methods causing patients discomfort while lacking the reproducibility of tissue sampling for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37173944"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Next-Generation Sequencing to Monitor Hematopoietic Stem-Cell Transplantation: Current Applications and Future Perspectives</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37173944/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230530210538&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) and myelodysplastic syndrome (MDS) are genetically complex and diverse diseases. Such complexity makes challenging the monitoring of response to treatment. Measurable residual disease (MRD) assessment is a powerful tool for monitoring response and guiding therapeutic interventions. This is accomplished through targeted next-generation sequencing (NGS), as well as polymerase chain reaction and multiparameter flow cytometry, to detect genomic aberrations at a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37175518"
                        open
                      >
                        <summary class="article-expander__title">The Potential of Liquid Biopsy in Detection of Endometrial Cancer Biomarkers: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37175518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Endometrial cancer belongs to the most common gynecologic cancer types globally, with increasing incidence. There are numerous ways of classifying different cases. The most recent decade has brought advances in molecular classification, which show more accurate prognostic factors and the possibility of personalised adjuvant treatment. In addition, diagnostic approaches lag behind these advances, with methods causing patients discomfort while lacking the reproducibility of tissue sampling for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37173944"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Next-Generation Sequencing to Monitor Hematopoietic Stem-Cell Transplantation: Current Applications and Future Perspectives</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37173944/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) and myelodysplastic syndrome (MDS) are genetically complex and diverse diseases. Such complexity makes challenging the monitoring of response to treatment. Measurable residual disease (MRD) assessment is a powerful tool for monitoring response and guiding therapeutic interventions. This is accomplished through targeted next-generation sequencing (NGS), as well as polymerase chain reaction and multiparameter flow cytometry, to detect genomic aberrations at a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37175518"
                        open
                      >
                        <summary class="article-expander__title">The Potential of Liquid Biopsy in Detection of Endometrial Cancer Biomarkers: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37175518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Endometrial cancer belongs to the most common gynecologic cancer types globally, with increasing incidence. There are numerous ways of classifying different cases. The most recent decade has brought advances in molecular classification, which show more accurate prognostic factors and the possibility of personalised adjuvant treatment. In addition, diagnostic approaches lag behind these advances, with methods causing patients discomfort while lacking the reproducibility of tissue sampling for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37173944"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Next-Generation Sequencing to Monitor Hematopoietic Stem-Cell Transplantation: Current Applications and Future Perspectives</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37173944/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) and myelodysplastic syndrome (MDS) are genetically complex and diverse diseases. Such complexity makes challenging the monitoring of response to treatment. Measurable residual disease (MRD) assessment is a powerful tool for monitoring response and guiding therapeutic interventions. This is accomplished through targeted next-generation sequencing (NGS), as well as polymerase chain reaction and multiparameter flow cytometry, to detect genomic aberrations at a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37175518"
                        open
                      >
                        <summary class="article-expander__title">The Potential of Liquid Biopsy in Detection of Endometrial Cancer Biomarkers: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37175518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230527210349&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Endometrial cancer belongs to the most common gynecologic cancer types globally, with increasing incidence. There are numerous ways of classifying different cases. The most recent decade has brought advances in molecular classification, which show more accurate prognostic factors and the possibility of personalised adjuvant treatment. In addition, diagnostic approaches lag behind these advances, with methods causing patients discomfort while lacking the reproducibility of tissue sampling for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37173944"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Next-Generation Sequencing to Monitor Hematopoietic Stem-Cell Transplantation: Current Applications and Future Perspectives</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37173944/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230527210349&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) and myelodysplastic syndrome (MDS) are genetically complex and diverse diseases. Such complexity makes challenging the monitoring of response to treatment. Measurable residual disease (MRD) assessment is a powerful tool for monitoring response and guiding therapeutic interventions. This is accomplished through targeted next-generation sequencing (NGS), as well as polymerase chain reaction and multiparameter flow cytometry, to detect genomic aberrations at a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37175518"
                        open
                      >
                        <summary class="article-expander__title">The Potential of Liquid Biopsy in Detection of Endometrial Cancer Biomarkers: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37175518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Endometrial cancer belongs to the most common gynecologic cancer types globally, with increasing incidence. There are numerous ways of classifying different cases. The most recent decade has brought advances in molecular classification, which show more accurate prognostic factors and the possibility of personalised adjuvant treatment. In addition, diagnostic approaches lag behind these advances, with methods causing patients discomfort while lacking the reproducibility of tissue sampling for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37173944"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Next-Generation Sequencing to Monitor Hematopoietic Stem-Cell Transplantation: Current Applications and Future Perspectives</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37173944/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) and myelodysplastic syndrome (MDS) are genetically complex and diverse diseases. Such complexity makes challenging the monitoring of response to treatment. Measurable residual disease (MRD) assessment is a powerful tool for monitoring response and guiding therapeutic interventions. This is accomplished through targeted next-generation sequencing (NGS), as well as polymerase chain reaction and multiparameter flow cytometry, to detect genomic aberrations at a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37175518"
                        open
                      >
                        <summary class="article-expander__title">The Potential of Liquid Biopsy in Detection of Endometrial Cancer Biomarkers: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37175518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Endometrial cancer belongs to the most common gynecologic cancer types globally, with increasing incidence. There are numerous ways of classifying different cases. The most recent decade has brought advances in molecular classification, which show more accurate prognostic factors and the possibility of personalised adjuvant treatment. In addition, diagnostic approaches lag behind these advances, with methods causing patients discomfort while lacking the reproducibility of tissue sampling for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37173944"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Next-Generation Sequencing to Monitor Hematopoietic Stem-Cell Transplantation: Current Applications and Future Perspectives</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37173944/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) and myelodysplastic syndrome (MDS) are genetically complex and diverse diseases. Such complexity makes challenging the monitoring of response to treatment. Measurable residual disease (MRD) assessment is a powerful tool for monitoring response and guiding therapeutic interventions. This is accomplished through targeted next-generation sequencing (NGS), as well as polymerase chain reaction and multiparameter flow cytometry, to detect genomic aberrations at a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37175518"
                        open
                      >
                        <summary class="article-expander__title">The Potential of Liquid Biopsy in Detection of Endometrial Cancer Biomarkers: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37175518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Endometrial cancer belongs to the most common gynecologic cancer types globally, with increasing incidence. There are numerous ways of classifying different cases. The most recent decade has brought advances in molecular classification, which show more accurate prognostic factors and the possibility of personalised adjuvant treatment. In addition, diagnostic approaches lag behind these advances, with methods causing patients discomfort while lacking the reproducibility of tissue sampling for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37173944"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Next-Generation Sequencing to Monitor Hematopoietic Stem-Cell Transplantation: Current Applications and Future Perspectives</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37173944/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) and myelodysplastic syndrome (MDS) are genetically complex and diverse diseases. Such complexity makes challenging the monitoring of response to treatment. Measurable residual disease (MRD) assessment is a powerful tool for monitoring response and guiding therapeutic interventions. This is accomplished through targeted next-generation sequencing (NGS), as well as polymerase chain reaction and multiparameter flow cytometry, to detect genomic aberrations at a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37175518"
                        open
                      >
                        <summary class="article-expander__title">The Potential of Liquid Biopsy in Detection of Endometrial Cancer Biomarkers: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37175518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Endometrial cancer belongs to the most common gynecologic cancer types globally, with increasing incidence. There are numerous ways of classifying different cases. The most recent decade has brought advances in molecular classification, which show more accurate prognostic factors and the possibility of personalised adjuvant treatment. In addition, diagnostic approaches lag behind these advances, with methods causing patients discomfort while lacking the reproducibility of tissue sampling for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37173944"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Next-Generation Sequencing to Monitor Hematopoietic Stem-Cell Transplantation: Current Applications and Future Perspectives</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37173944/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) and myelodysplastic syndrome (MDS) are genetically complex and diverse diseases. Such complexity makes challenging the monitoring of response to treatment. Measurable residual disease (MRD) assessment is a powerful tool for monitoring response and guiding therapeutic interventions. This is accomplished through targeted next-generation sequencing (NGS), as well as polymerase chain reaction and multiparameter flow cytometry, to detect genomic aberrations at a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37175518"
                        open
                      >
                        <summary class="article-expander__title">The Potential of Liquid Biopsy in Detection of Endometrial Cancer Biomarkers: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37175518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Endometrial cancer belongs to the most common gynecologic cancer types globally, with increasing incidence. There are numerous ways of classifying different cases. The most recent decade has brought advances in molecular classification, which show more accurate prognostic factors and the possibility of personalised adjuvant treatment. In addition, diagnostic approaches lag behind these advances, with methods causing patients discomfort while lacking the reproducibility of tissue sampling for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37173944"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Next-Generation Sequencing to Monitor Hematopoietic Stem-Cell Transplantation: Current Applications and Future Perspectives</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37173944/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) and myelodysplastic syndrome (MDS) are genetically complex and diverse diseases. Such complexity makes challenging the monitoring of response to treatment. Measurable residual disease (MRD) assessment is a powerful tool for monitoring response and guiding therapeutic interventions. This is accomplished through targeted next-generation sequencing (NGS), as well as polymerase chain reaction and multiparameter flow cytometry, to detect genomic aberrations at a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37175518"
                        open
                      >
                        <summary class="article-expander__title">The Potential of Liquid Biopsy in Detection of Endometrial Cancer Biomarkers: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37175518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230521210055&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Endometrial cancer belongs to the most common gynecologic cancer types globally, with increasing incidence. There are numerous ways of classifying different cases. The most recent decade has brought advances in molecular classification, which show more accurate prognostic factors and the possibility of personalised adjuvant treatment. In addition, diagnostic approaches lag behind these advances, with methods causing patients discomfort while lacking the reproducibility of tissue sampling for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37173944"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Next-Generation Sequencing to Monitor Hematopoietic Stem-Cell Transplantation: Current Applications and Future Perspectives</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37173944/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230521210055&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) and myelodysplastic syndrome (MDS) are genetically complex and diverse diseases. Such complexity makes challenging the monitoring of response to treatment. Measurable residual disease (MRD) assessment is a powerful tool for monitoring response and guiding therapeutic interventions. This is accomplished through targeted next-generation sequencing (NGS), as well as polymerase chain reaction and multiparameter flow cytometry, to detect genomic aberrations at a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37175518"
                        open
                      >
                        <summary class="article-expander__title">The Potential of Liquid Biopsy in Detection of Endometrial Cancer Biomarkers: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37175518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Endometrial cancer belongs to the most common gynecologic cancer types globally, with increasing incidence. There are numerous ways of classifying different cases. The most recent decade has brought advances in molecular classification, which show more accurate prognostic factors and the possibility of personalised adjuvant treatment. In addition, diagnostic approaches lag behind these advances, with methods causing patients discomfort while lacking the reproducibility of tissue sampling for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37173944"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Next-Generation Sequencing to Monitor Hematopoietic Stem-Cell Transplantation: Current Applications and Future Perspectives</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37173944/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) and myelodysplastic syndrome (MDS) are genetically complex and diverse diseases. Such complexity makes challenging the monitoring of response to treatment. Measurable residual disease (MRD) assessment is a powerful tool for monitoring response and guiding therapeutic interventions. This is accomplished through targeted next-generation sequencing (NGS), as well as polymerase chain reaction and multiparameter flow cytometry, to detect genomic aberrations at a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37175518"
                        open
                      >
                        <summary class="article-expander__title">The Potential of Liquid Biopsy in Detection of Endometrial Cancer Biomarkers: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37175518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Endometrial cancer belongs to the most common gynecologic cancer types globally, with increasing incidence. There are numerous ways of classifying different cases. The most recent decade has brought advances in molecular classification, which show more accurate prognostic factors and the possibility of personalised adjuvant treatment. In addition, diagnostic approaches lag behind these advances, with methods causing patients discomfort while lacking the reproducibility of tissue sampling for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37173944"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Next-Generation Sequencing to Monitor Hematopoietic Stem-Cell Transplantation: Current Applications and Future Perspectives</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37173944/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) and myelodysplastic syndrome (MDS) are genetically complex and diverse diseases. Such complexity makes challenging the monitoring of response to treatment. Measurable residual disease (MRD) assessment is a powerful tool for monitoring response and guiding therapeutic interventions. This is accomplished through targeted next-generation sequencing (NGS), as well as polymerase chain reaction and multiparameter flow cytometry, to detect genomic aberrations at a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37175518"
                        open
                      >
                        <summary class="article-expander__title">The Potential of Liquid Biopsy in Detection of Endometrial Cancer Biomarkers: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37175518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Endometrial cancer belongs to the most common gynecologic cancer types globally, with increasing incidence. There are numerous ways of classifying different cases. The most recent decade has brought advances in molecular classification, which show more accurate prognostic factors and the possibility of personalised adjuvant treatment. In addition, diagnostic approaches lag behind these advances, with methods causing patients discomfort while lacking the reproducibility of tissue sampling for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37173944"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Next-Generation Sequencing to Monitor Hematopoietic Stem-Cell Transplantation: Current Applications and Future Perspectives</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37173944/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) and myelodysplastic syndrome (MDS) are genetically complex and diverse diseases. Such complexity makes challenging the monitoring of response to treatment. Measurable residual disease (MRD) assessment is a powerful tool for monitoring response and guiding therapeutic interventions. This is accomplished through targeted next-generation sequencing (NGS), as well as polymerase chain reaction and multiparameter flow cytometry, to detect genomic aberrations at a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37175518"
                        open
                      >
                        <summary class="article-expander__title">The Potential of Liquid Biopsy in Detection of Endometrial Cancer Biomarkers: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37175518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Endometrial cancer belongs to the most common gynecologic cancer types globally, with increasing incidence. There are numerous ways of classifying different cases. The most recent decade has brought advances in molecular classification, which show more accurate prognostic factors and the possibility of personalised adjuvant treatment. In addition, diagnostic approaches lag behind these advances, with methods causing patients discomfort while lacking the reproducibility of tissue sampling for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37173944"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Next-Generation Sequencing to Monitor Hematopoietic Stem-Cell Transplantation: Current Applications and Future Perspectives</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37173944/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) and myelodysplastic syndrome (MDS) are genetically complex and diverse diseases. Such complexity makes challenging the monitoring of response to treatment. Measurable residual disease (MRD) assessment is a powerful tool for monitoring response and guiding therapeutic interventions. This is accomplished through targeted next-generation sequencing (NGS), as well as polymerase chain reaction and multiparameter flow cytometry, to detect genomic aberrations at a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37175518"
                        open
                      >
                        <summary class="article-expander__title">The Potential of Liquid Biopsy in Detection of Endometrial Cancer Biomarkers: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37175518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Endometrial cancer belongs to the most common gynecologic cancer types globally, with increasing incidence. There are numerous ways of classifying different cases. The most recent decade has brought advances in molecular classification, which show more accurate prognostic factors and the possibility of personalised adjuvant treatment. In addition, diagnostic approaches lag behind these advances, with methods causing patients discomfort while lacking the reproducibility of tissue sampling for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37173944"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Next-Generation Sequencing to Monitor Hematopoietic Stem-Cell Transplantation: Current Applications and Future Perspectives</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37173944/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) and myelodysplastic syndrome (MDS) are genetically complex and diverse diseases. Such complexity makes challenging the monitoring of response to treatment. Measurable residual disease (MRD) assessment is a powerful tool for monitoring response and guiding therapeutic interventions. This is accomplished through targeted next-generation sequencing (NGS), as well as polymerase chain reaction and multiparameter flow cytometry, to detect genomic aberrations at a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37175518"
                        open
                      >
                        <summary class="article-expander__title">The Potential of Liquid Biopsy in Detection of Endometrial Cancer Biomarkers: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37175518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230515205819&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Endometrial cancer belongs to the most common gynecologic cancer types globally, with increasing incidence. There are numerous ways of classifying different cases. The most recent decade has brought advances in molecular classification, which show more accurate prognostic factors and the possibility of personalised adjuvant treatment. In addition, diagnostic approaches lag behind these advances, with methods causing patients discomfort while lacking the reproducibility of tissue sampling for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37173944"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Next-Generation Sequencing to Monitor Hematopoietic Stem-Cell Transplantation: Current Applications and Future Perspectives</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37173944/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230515205819&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) and myelodysplastic syndrome (MDS) are genetically complex and diverse diseases. Such complexity makes challenging the monitoring of response to treatment. Measurable residual disease (MRD) assessment is a powerful tool for monitoring response and guiding therapeutic interventions. This is accomplished through targeted next-generation sequencing (NGS), as well as polymerase chain reaction and multiparameter flow cytometry, to detect genomic aberrations at a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37175518"
                        open
                      >
                        <summary class="article-expander__title">The Potential of Liquid Biopsy in Detection of Endometrial Cancer Biomarkers: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37175518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Endometrial cancer belongs to the most common gynecologic cancer types globally, with increasing incidence. There are numerous ways of classifying different cases. The most recent decade has brought advances in molecular classification, which show more accurate prognostic factors and the possibility of personalised adjuvant treatment. In addition, diagnostic approaches lag behind these advances, with methods causing patients discomfort while lacking the reproducibility of tissue sampling for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37173944"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Next-Generation Sequencing to Monitor Hematopoietic Stem-Cell Transplantation: Current Applications and Future Perspectives</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37173944/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) and myelodysplastic syndrome (MDS) are genetically complex and diverse diseases. Such complexity makes challenging the monitoring of response to treatment. Measurable residual disease (MRD) assessment is a powerful tool for monitoring response and guiding therapeutic interventions. This is accomplished through targeted next-generation sequencing (NGS), as well as polymerase chain reaction and multiparameter flow cytometry, to detect genomic aberrations at a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37175518"
                        open
                      >
                        <summary class="article-expander__title">The Potential of Liquid Biopsy in Detection of Endometrial Cancer Biomarkers: A Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37175518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Endometrial cancer belongs to the most common gynecologic cancer types globally, with increasing incidence. There are numerous ways of classifying different cases. The most recent decade has brought advances in molecular classification, which show more accurate prognostic factors and the possibility of personalised adjuvant treatment. In addition, diagnostic approaches lag behind these advances, with methods causing patients discomfort while lacking the reproducibility of tissue sampling for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37173944"
                        open
                      >
                        <summary class="article-expander__title">Single-Cell Next-Generation Sequencing to Monitor Hematopoietic Stem-Cell Transplantation: Current Applications and Future Perspectives</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37173944/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) and myelodysplastic syndrome (MDS) are genetically complex and diverse diseases. Such complexity makes challenging the monitoring of response to treatment. Measurable residual disease (MRD) assessment is a powerful tool for monitoring response and guiding therapeutic interventions. This is accomplished through targeted next-generation sequencing (NGS), as well as polymerase chain reaction and multiparameter flow cytometry, to detect genomic aberrations at a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37178708"
                        open
                      >
                        <summary class="article-expander__title">Utility of polygenic risk scores in UK cancer screening: a modelling analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37178708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: It is proposed that, through restriction to individuals delineated as high risk, polygenic risk scores (PRSs) might enable more efficient targeting of existing cancer screening programmes and enable extension into new age ranges and disease types. To address this proposition, we present an overview of the performance of PRS tools (ie, models and sets of single nucleotide polymorphisms) alongside harms and benefits of PRS-stratified cancer screening for eight example cancers (breast,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37176124"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal Analyses of Mutational Subclonal Architecture and Tumor Subtypes in Recurrent Bladder Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37176124/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Longitudinal tumor sequencing of recurrent bladder cancer (BC) can facilitate the investigation of BC progression-associated genomic and transcriptomic alterations. In this study, we analyzed 18 tumor specimens including distant and locoregional metastases obtained during tumor progression for five BC patients using whole-exome and transcriptome sequencing. Along with the substantial level of intratumoral mutational heterogeneity across the cases, we observed that clonal mutations were enriched...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37178708"
                        open
                      >
                        <summary class="article-expander__title">Utility of polygenic risk scores in UK cancer screening: a modelling analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37178708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: It is proposed that, through restriction to individuals delineated as high risk, polygenic risk scores (PRSs) might enable more efficient targeting of existing cancer screening programmes and enable extension into new age ranges and disease types. To address this proposition, we present an overview of the performance of PRS tools (ie, models and sets of single nucleotide polymorphisms) alongside harms and benefits of PRS-stratified cancer screening for eight example cancers (breast,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37176124"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal Analyses of Mutational Subclonal Architecture and Tumor Subtypes in Recurrent Bladder Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37176124/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Longitudinal tumor sequencing of recurrent bladder cancer (BC) can facilitate the investigation of BC progression-associated genomic and transcriptomic alterations. In this study, we analyzed 18 tumor specimens including distant and locoregional metastases obtained during tumor progression for five BC patients using whole-exome and transcriptome sequencing. Along with the substantial level of intratumoral mutational heterogeneity across the cases, we observed that clonal mutations were enriched...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37178708"
                        open
                      >
                        <summary class="article-expander__title">Utility of polygenic risk scores in UK cancer screening: a modelling analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37178708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230521210056&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: It is proposed that, through restriction to individuals delineated as high risk, polygenic risk scores (PRSs) might enable more efficient targeting of existing cancer screening programmes and enable extension into new age ranges and disease types. To address this proposition, we present an overview of the performance of PRS tools (ie, models and sets of single nucleotide polymorphisms) alongside harms and benefits of PRS-stratified cancer screening for eight example cancers (breast,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37176124"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal Analyses of Mutational Subclonal Architecture and Tumor Subtypes in Recurrent Bladder Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37176124/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230521210056&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Longitudinal tumor sequencing of recurrent bladder cancer (BC) can facilitate the investigation of BC progression-associated genomic and transcriptomic alterations. In this study, we analyzed 18 tumor specimens including distant and locoregional metastases obtained during tumor progression for five BC patients using whole-exome and transcriptome sequencing. Along with the substantial level of intratumoral mutational heterogeneity across the cases, we observed that clonal mutations were enriched...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37178708"
                        open
                      >
                        <summary class="article-expander__title">Utility of polygenic risk scores in UK cancer screening: a modelling analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37178708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: It is proposed that, through restriction to individuals delineated as high risk, polygenic risk scores (PRSs) might enable more efficient targeting of existing cancer screening programmes and enable extension into new age ranges and disease types. To address this proposition, we present an overview of the performance of PRS tools (ie, models and sets of single nucleotide polymorphisms) alongside harms and benefits of PRS-stratified cancer screening for eight example cancers (breast,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37176124"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal Analyses of Mutational Subclonal Architecture and Tumor Subtypes in Recurrent Bladder Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37176124/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Longitudinal tumor sequencing of recurrent bladder cancer (BC) can facilitate the investigation of BC progression-associated genomic and transcriptomic alterations. In this study, we analyzed 18 tumor specimens including distant and locoregional metastases obtained during tumor progression for five BC patients using whole-exome and transcriptome sequencing. Along with the substantial level of intratumoral mutational heterogeneity across the cases, we observed that clonal mutations were enriched...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37178708"
                        open
                      >
                        <summary class="article-expander__title">Utility of polygenic risk scores in UK cancer screening: a modelling analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37178708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: It is proposed that, through restriction to individuals delineated as high risk, polygenic risk scores (PRSs) might enable more efficient targeting of existing cancer screening programmes and enable extension into new age ranges and disease types. To address this proposition, we present an overview of the performance of PRS tools (ie, models and sets of single nucleotide polymorphisms) alongside harms and benefits of PRS-stratified cancer screening for eight example cancers (breast,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37176124"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal Analyses of Mutational Subclonal Architecture and Tumor Subtypes in Recurrent Bladder Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37176124/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Longitudinal tumor sequencing of recurrent bladder cancer (BC) can facilitate the investigation of BC progression-associated genomic and transcriptomic alterations. In this study, we analyzed 18 tumor specimens including distant and locoregional metastases obtained during tumor progression for five BC patients using whole-exome and transcriptome sequencing. Along with the substantial level of intratumoral mutational heterogeneity across the cases, we observed that clonal mutations were enriched...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37178708"
                        open
                      >
                        <summary class="article-expander__title">Utility of polygenic risk scores in UK cancer screening: a modelling analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37178708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: It is proposed that, through restriction to individuals delineated as high risk, polygenic risk scores (PRSs) might enable more efficient targeting of existing cancer screening programmes and enable extension into new age ranges and disease types. To address this proposition, we present an overview of the performance of PRS tools (ie, models and sets of single nucleotide polymorphisms) alongside harms and benefits of PRS-stratified cancer screening for eight example cancers (breast,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37176124"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal Analyses of Mutational Subclonal Architecture and Tumor Subtypes in Recurrent Bladder Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37176124/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Longitudinal tumor sequencing of recurrent bladder cancer (BC) can facilitate the investigation of BC progression-associated genomic and transcriptomic alterations. In this study, we analyzed 18 tumor specimens including distant and locoregional metastases obtained during tumor progression for five BC patients using whole-exome and transcriptome sequencing. Along with the substantial level of intratumoral mutational heterogeneity across the cases, we observed that clonal mutations were enriched...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37178708"
                        open
                      >
                        <summary class="article-expander__title">Utility of polygenic risk scores in UK cancer screening: a modelling analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37178708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: It is proposed that, through restriction to individuals delineated as high risk, polygenic risk scores (PRSs) might enable more efficient targeting of existing cancer screening programmes and enable extension into new age ranges and disease types. To address this proposition, we present an overview of the performance of PRS tools (ie, models and sets of single nucleotide polymorphisms) alongside harms and benefits of PRS-stratified cancer screening for eight example cancers (breast,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37176124"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal Analyses of Mutational Subclonal Architecture and Tumor Subtypes in Recurrent Bladder Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37176124/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Longitudinal tumor sequencing of recurrent bladder cancer (BC) can facilitate the investigation of BC progression-associated genomic and transcriptomic alterations. In this study, we analyzed 18 tumor specimens including distant and locoregional metastases obtained during tumor progression for five BC patients using whole-exome and transcriptome sequencing. Along with the substantial level of intratumoral mutational heterogeneity across the cases, we observed that clonal mutations were enriched...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37178708"
                        open
                      >
                        <summary class="article-expander__title">Utility of polygenic risk scores in UK cancer screening: a modelling analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37178708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: It is proposed that, through restriction to individuals delineated as high risk, polygenic risk scores (PRSs) might enable more efficient targeting of existing cancer screening programmes and enable extension into new age ranges and disease types. To address this proposition, we present an overview of the performance of PRS tools (ie, models and sets of single nucleotide polymorphisms) alongside harms and benefits of PRS-stratified cancer screening for eight example cancers (breast,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37176124"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal Analyses of Mutational Subclonal Architecture and Tumor Subtypes in Recurrent Bladder Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37176124/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Longitudinal tumor sequencing of recurrent bladder cancer (BC) can facilitate the investigation of BC progression-associated genomic and transcriptomic alterations. In this study, we analyzed 18 tumor specimens including distant and locoregional metastases obtained during tumor progression for five BC patients using whole-exome and transcriptome sequencing. Along with the substantial level of intratumoral mutational heterogeneity across the cases, we observed that clonal mutations were enriched...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37178708"
                        open
                      >
                        <summary class="article-expander__title">Utility of polygenic risk scores in UK cancer screening: a modelling analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37178708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230515205818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: It is proposed that, through restriction to individuals delineated as high risk, polygenic risk scores (PRSs) might enable more efficient targeting of existing cancer screening programmes and enable extension into new age ranges and disease types. To address this proposition, we present an overview of the performance of PRS tools (ie, models and sets of single nucleotide polymorphisms) alongside harms and benefits of PRS-stratified cancer screening for eight example cancers (breast,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37176124"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal Analyses of Mutational Subclonal Architecture and Tumor Subtypes in Recurrent Bladder Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37176124/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230515205818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Longitudinal tumor sequencing of recurrent bladder cancer (BC) can facilitate the investigation of BC progression-associated genomic and transcriptomic alterations. In this study, we analyzed 18 tumor specimens including distant and locoregional metastases obtained during tumor progression for five BC patients using whole-exome and transcriptome sequencing. Along with the substantial level of intratumoral mutational heterogeneity across the cases, we observed that clonal mutations were enriched...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37178708"
                        open
                      >
                        <summary class="article-expander__title">Utility of polygenic risk scores in UK cancer screening: a modelling analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37178708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: It is proposed that, through restriction to individuals delineated as high risk, polygenic risk scores (PRSs) might enable more efficient targeting of existing cancer screening programmes and enable extension into new age ranges and disease types. To address this proposition, we present an overview of the performance of PRS tools (ie, models and sets of single nucleotide polymorphisms) alongside harms and benefits of PRS-stratified cancer screening for eight example cancers (breast,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37176124"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal Analyses of Mutational Subclonal Architecture and Tumor Subtypes in Recurrent Bladder Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37176124/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Longitudinal tumor sequencing of recurrent bladder cancer (BC) can facilitate the investigation of BC progression-associated genomic and transcriptomic alterations. In this study, we analyzed 18 tumor specimens including distant and locoregional metastases obtained during tumor progression for five BC patients using whole-exome and transcriptome sequencing. Along with the substantial level of intratumoral mutational heterogeneity across the cases, we observed that clonal mutations were enriched...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37176124"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal Analyses of Mutational Subclonal Architecture and Tumor Subtypes in Recurrent Bladder Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37176124/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Longitudinal tumor sequencing of recurrent bladder cancer (BC) can facilitate the investigation of BC progression-associated genomic and transcriptomic alterations. In this study, we analyzed 18 tumor specimens including distant and locoregional metastases obtained during tumor progression for five BC patients using whole-exome and transcriptome sequencing. Along with the substantial level of intratumoral mutational heterogeneity across the cases, we observed that clonal mutations were enriched...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-05-12 datetime="2023-05-12T10:00:00.000Z">2023-05-12</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-05-12 datetime="2023-05-12T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37172086"
                        open
                      >
                        <summary class="article-expander__title">Efficient cancer modeling through CRISPR-Cas9/HDR-based somatic precision gene editing in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37172086/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CRISPR-Cas9 has been used successfully to introduce indels in somatic cells of rodents; however, precise editing of single nucleotides has been hampered by limitations of flexibility and efficiency. Here, we report technological modifications to the CRISPR-Cas9 vector system that now allows homology-directed repair-mediated precise editing of any proto-oncogene in murine somatic tissues to generate tumor models with high flexibility and efficiency. Somatic editing of either Kras or Pik3ca in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37172086"
                        open
                      >
                        <summary class="article-expander__title">Efficient cancer modeling through CRISPR-Cas9/HDR-based somatic precision gene editing in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37172086/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CRISPR-Cas9 has been used successfully to introduce indels in somatic cells of rodents; however, precise editing of single nucleotides has been hampered by limitations of flexibility and efficiency. Here, we report technological modifications to the CRISPR-Cas9 vector system that now allows homology-directed repair-mediated precise editing of any proto-oncogene in murine somatic tissues to generate tumor models with high flexibility and efficiency. Somatic editing of either Kras or Pik3ca in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37172086"
                        open
                      >
                        <summary class="article-expander__title">Efficient cancer modeling through CRISPR-Cas9/HDR-based somatic precision gene editing in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37172086/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230521210056&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CRISPR-Cas9 has been used successfully to introduce indels in somatic cells of rodents; however, precise editing of single nucleotides has been hampered by limitations of flexibility and efficiency. Here, we report technological modifications to the CRISPR-Cas9 vector system that now allows homology-directed repair-mediated precise editing of any proto-oncogene in murine somatic tissues to generate tumor models with high flexibility and efficiency. Somatic editing of either Kras or Pik3ca in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37172086"
                        open
                      >
                        <summary class="article-expander__title">Efficient cancer modeling through CRISPR-Cas9/HDR-based somatic precision gene editing in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37172086/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CRISPR-Cas9 has been used successfully to introduce indels in somatic cells of rodents; however, precise editing of single nucleotides has been hampered by limitations of flexibility and efficiency. Here, we report technological modifications to the CRISPR-Cas9 vector system that now allows homology-directed repair-mediated precise editing of any proto-oncogene in murine somatic tissues to generate tumor models with high flexibility and efficiency. Somatic editing of either Kras or Pik3ca in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37172086"
                        open
                      >
                        <summary class="article-expander__title">Efficient cancer modeling through CRISPR-Cas9/HDR-based somatic precision gene editing in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37172086/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CRISPR-Cas9 has been used successfully to introduce indels in somatic cells of rodents; however, precise editing of single nucleotides has been hampered by limitations of flexibility and efficiency. Here, we report technological modifications to the CRISPR-Cas9 vector system that now allows homology-directed repair-mediated precise editing of any proto-oncogene in murine somatic tissues to generate tumor models with high flexibility and efficiency. Somatic editing of either Kras or Pik3ca in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37172086"
                        open
                      >
                        <summary class="article-expander__title">Efficient cancer modeling through CRISPR-Cas9/HDR-based somatic precision gene editing in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37172086/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CRISPR-Cas9 has been used successfully to introduce indels in somatic cells of rodents; however, precise editing of single nucleotides has been hampered by limitations of flexibility and efficiency. Here, we report technological modifications to the CRISPR-Cas9 vector system that now allows homology-directed repair-mediated precise editing of any proto-oncogene in murine somatic tissues to generate tumor models with high flexibility and efficiency. Somatic editing of either Kras or Pik3ca in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37172086"
                        open
                      >
                        <summary class="article-expander__title">Efficient cancer modeling through CRISPR-Cas9/HDR-based somatic precision gene editing in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37172086/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CRISPR-Cas9 has been used successfully to introduce indels in somatic cells of rodents; however, precise editing of single nucleotides has been hampered by limitations of flexibility and efficiency. Here, we report technological modifications to the CRISPR-Cas9 vector system that now allows homology-directed repair-mediated precise editing of any proto-oncogene in murine somatic tissues to generate tumor models with high flexibility and efficiency. Somatic editing of either Kras or Pik3ca in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37172086"
                        open
                      >
                        <summary class="article-expander__title">Efficient cancer modeling through CRISPR-Cas9/HDR-based somatic precision gene editing in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37172086/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CRISPR-Cas9 has been used successfully to introduce indels in somatic cells of rodents; however, precise editing of single nucleotides has been hampered by limitations of flexibility and efficiency. Here, we report technological modifications to the CRISPR-Cas9 vector system that now allows homology-directed repair-mediated precise editing of any proto-oncogene in murine somatic tissues to generate tumor models with high flexibility and efficiency. Somatic editing of either Kras or Pik3ca in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37172086"
                        open
                      >
                        <summary class="article-expander__title">Efficient cancer modeling through CRISPR-Cas9/HDR-based somatic precision gene editing in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37172086/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230515205818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CRISPR-Cas9 has been used successfully to introduce indels in somatic cells of rodents; however, precise editing of single nucleotides has been hampered by limitations of flexibility and efficiency. Here, we report technological modifications to the CRISPR-Cas9 vector system that now allows homology-directed repair-mediated precise editing of any proto-oncogene in murine somatic tissues to generate tumor models with high flexibility and efficiency. Somatic editing of either Kras or Pik3ca in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37172086"
                        open
                      >
                        <summary class="article-expander__title">Efficient cancer modeling through CRISPR-Cas9/HDR-based somatic precision gene editing in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37172086/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CRISPR-Cas9 has been used successfully to introduce indels in somatic cells of rodents; however, precise editing of single nucleotides has been hampered by limitations of flexibility and efficiency. Here, we report technological modifications to the CRISPR-Cas9 vector system that now allows homology-directed repair-mediated precise editing of any proto-oncogene in murine somatic tissues to generate tumor models with high flexibility and efficiency. Somatic editing of either Kras or Pik3ca in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37172086"
                        open
                      >
                        <summary class="article-expander__title">Efficient cancer modeling through CRISPR-Cas9/HDR-based somatic precision gene editing in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37172086/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CRISPR-Cas9 has been used successfully to introduce indels in somatic cells of rodents; however, precise editing of single nucleotides has been hampered by limitations of flexibility and efficiency. Here, we report technological modifications to the CRISPR-Cas9 vector system that now allows homology-directed repair-mediated precise editing of any proto-oncogene in murine somatic tissues to generate tumor models with high flexibility and efficiency. Somatic editing of either Kras or Pik3ca in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37172086"
                        open
                      >
                        <summary class="article-expander__title">Efficient cancer modeling through CRISPR-Cas9/HDR-based somatic precision gene editing in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37172086/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CRISPR-Cas9 has been used successfully to introduce indels in somatic cells of rodents; however, precise editing of single nucleotides has been hampered by limitations of flexibility and efficiency. Here, we report technological modifications to the CRISPR-Cas9 vector system that now allows homology-directed repair-mediated precise editing of any proto-oncogene in murine somatic tissues to generate tumor models with high flexibility and efficiency. Somatic editing of either Kras or Pik3ca in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-05-11 datetime="2023-05-11T10:00:00.000Z">2023-05-11</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-05-11 datetime="2023-05-11T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230529210010&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230527210349&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230521210055&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230515205819&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230511205726&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37170152"
                        open
                      >
                        <summary class="article-expander__title">Exploring circular MET RNA as a potential biomarker in tumors exhibiting high MET activity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37170152/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We found that measurement of circMET levels allows identification and tracking of patients characterized by high MET activity. Circulating circMET (ccMET) detection and analysis could be a simple, cost-effective, non-invasive approach to better implement patient stratification based on MET expression, as well as to dynamically monitor over time both therapy response and clonal evolution during treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37170152"
                        open
                      >
                        <summary class="article-expander__title">Exploring circular MET RNA as a potential biomarker in tumors exhibiting high MET activity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37170152/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230521210056&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We found that measurement of circMET levels allows identification and tracking of patients characterized by high MET activity. Circulating circMET (ccMET) detection and analysis could be a simple, cost-effective, non-invasive approach to better implement patient stratification based on MET expression, as well as to dynamically monitor over time both therapy response and clonal evolution during treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37168490"
                        open
                      >
                        <summary class="article-expander__title">Refining the Treatment of Pancreatic Cancer From Big Data to Improved Individual Survival</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37168490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230521210056&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pancreatic cancer is one of the most lethal cancers worldwide, most notably in Europe and North America. Great strides have been made in combining the most effective conventional therapies to improve survival at least in the short and medium term. The start of treatment can only be made once a diagnosis is made, which at this point, the tumor volume is already very high in the primary cancer and systemically. If caught at the earliest opportunity (in circa 20% patients) surgical resection of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230521210056&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37170152"
                        open
                      >
                        <summary class="article-expander__title">Exploring circular MET RNA as a potential biomarker in tumors exhibiting high MET activity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37170152/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We found that measurement of circMET levels allows identification and tracking of patients characterized by high MET activity. Circulating circMET (ccMET) detection and analysis could be a simple, cost-effective, non-invasive approach to better implement patient stratification based on MET expression, as well as to dynamically monitor over time both therapy response and clonal evolution during treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37168490"
                        open
                      >
                        <summary class="article-expander__title">Refining the Treatment of Pancreatic Cancer From Big Data to Improved Individual Survival</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37168490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pancreatic cancer is one of the most lethal cancers worldwide, most notably in Europe and North America. Great strides have been made in combining the most effective conventional therapies to improve survival at least in the short and medium term. The start of treatment can only be made once a diagnosis is made, which at this point, the tumor volume is already very high in the primary cancer and systemically. If caught at the earliest opportunity (in circa 20% patients) surgical resection of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37170152"
                        open
                      >
                        <summary class="article-expander__title">Exploring circular MET RNA as a potential biomarker in tumors exhibiting high MET activity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37170152/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We found that measurement of circMET levels allows identification and tracking of patients characterized by high MET activity. Circulating circMET (ccMET) detection and analysis could be a simple, cost-effective, non-invasive approach to better implement patient stratification based on MET expression, as well as to dynamically monitor over time both therapy response and clonal evolution during treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37168490"
                        open
                      >
                        <summary class="article-expander__title">Refining the Treatment of Pancreatic Cancer From Big Data to Improved Individual Survival</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37168490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pancreatic cancer is one of the most lethal cancers worldwide, most notably in Europe and North America. Great strides have been made in combining the most effective conventional therapies to improve survival at least in the short and medium term. The start of treatment can only be made once a diagnosis is made, which at this point, the tumor volume is already very high in the primary cancer and systemically. If caught at the earliest opportunity (in circa 20% patients) surgical resection of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37170152"
                        open
                      >
                        <summary class="article-expander__title">Exploring circular MET RNA as a potential biomarker in tumors exhibiting high MET activity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37170152/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We found that measurement of circMET levels allows identification and tracking of patients characterized by high MET activity. Circulating circMET (ccMET) detection and analysis could be a simple, cost-effective, non-invasive approach to better implement patient stratification based on MET expression, as well as to dynamically monitor over time both therapy response and clonal evolution during treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37168490"
                        open
                      >
                        <summary class="article-expander__title">Refining the Treatment of Pancreatic Cancer From Big Data to Improved Individual Survival</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37168490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pancreatic cancer is one of the most lethal cancers worldwide, most notably in Europe and North America. Great strides have been made in combining the most effective conventional therapies to improve survival at least in the short and medium term. The start of treatment can only be made once a diagnosis is made, which at this point, the tumor volume is already very high in the primary cancer and systemically. If caught at the earliest opportunity (in circa 20% patients) surgical resection of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37170152"
                        open
                      >
                        <summary class="article-expander__title">Exploring circular MET RNA as a potential biomarker in tumors exhibiting high MET activity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37170152/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We found that measurement of circMET levels allows identification and tracking of patients characterized by high MET activity. Circulating circMET (ccMET) detection and analysis could be a simple, cost-effective, non-invasive approach to better implement patient stratification based on MET expression, as well as to dynamically monitor over time both therapy response and clonal evolution during treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37168490"
                        open
                      >
                        <summary class="article-expander__title">Refining the Treatment of Pancreatic Cancer From Big Data to Improved Individual Survival</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37168490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pancreatic cancer is one of the most lethal cancers worldwide, most notably in Europe and North America. Great strides have been made in combining the most effective conventional therapies to improve survival at least in the short and medium term. The start of treatment can only be made once a diagnosis is made, which at this point, the tumor volume is already very high in the primary cancer and systemically. If caught at the earliest opportunity (in circa 20% patients) surgical resection of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37170152"
                        open
                      >
                        <summary class="article-expander__title">Exploring circular MET RNA as a potential biomarker in tumors exhibiting high MET activity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37170152/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We found that measurement of circMET levels allows identification and tracking of patients characterized by high MET activity. Circulating circMET (ccMET) detection and analysis could be a simple, cost-effective, non-invasive approach to better implement patient stratification based on MET expression, as well as to dynamically monitor over time both therapy response and clonal evolution during treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37168490"
                        open
                      >
                        <summary class="article-expander__title">Refining the Treatment of Pancreatic Cancer From Big Data to Improved Individual Survival</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37168490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pancreatic cancer is one of the most lethal cancers worldwide, most notably in Europe and North America. Great strides have been made in combining the most effective conventional therapies to improve survival at least in the short and medium term. The start of treatment can only be made once a diagnosis is made, which at this point, the tumor volume is already very high in the primary cancer and systemically. If caught at the earliest opportunity (in circa 20% patients) surgical resection of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37170152"
                        open
                      >
                        <summary class="article-expander__title">Exploring circular MET RNA as a potential biomarker in tumors exhibiting high MET activity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37170152/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230515205818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We found that measurement of circMET levels allows identification and tracking of patients characterized by high MET activity. Circulating circMET (ccMET) detection and analysis could be a simple, cost-effective, non-invasive approach to better implement patient stratification based on MET expression, as well as to dynamically monitor over time both therapy response and clonal evolution during treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37168490"
                        open
                      >
                        <summary class="article-expander__title">Refining the Treatment of Pancreatic Cancer From Big Data to Improved Individual Survival</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37168490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230515205818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pancreatic cancer is one of the most lethal cancers worldwide, most notably in Europe and North America. Great strides have been made in combining the most effective conventional therapies to improve survival at least in the short and medium term. The start of treatment can only be made once a diagnosis is made, which at this point, the tumor volume is already very high in the primary cancer and systemically. If caught at the earliest opportunity (in circa 20% patients) surgical resection of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230515205818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37170152"
                        open
                      >
                        <summary class="article-expander__title">Exploring circular MET RNA as a potential biomarker in tumors exhibiting high MET activity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37170152/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We found that measurement of circMET levels allows identification and tracking of patients characterized by high MET activity. Circulating circMET (ccMET) detection and analysis could be a simple, cost-effective, non-invasive approach to better implement patient stratification based on MET expression, as well as to dynamically monitor over time both therapy response and clonal evolution during treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37168490"
                        open
                      >
                        <summary class="article-expander__title">Refining the Treatment of Pancreatic Cancer From Big Data to Improved Individual Survival</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37168490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pancreatic cancer is one of the most lethal cancers worldwide, most notably in Europe and North America. Great strides have been made in combining the most effective conventional therapies to improve survival at least in the short and medium term. The start of treatment can only be made once a diagnosis is made, which at this point, the tumor volume is already very high in the primary cancer and systemically. If caught at the earliest opportunity (in circa 20% patients) surgical resection of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37170152"
                        open
                      >
                        <summary class="article-expander__title">Exploring circular MET RNA as a potential biomarker in tumors exhibiting high MET activity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37170152/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We found that measurement of circMET levels allows identification and tracking of patients characterized by high MET activity. Circulating circMET (ccMET) detection and analysis could be a simple, cost-effective, non-invasive approach to better implement patient stratification based on MET expression, as well as to dynamically monitor over time both therapy response and clonal evolution during treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37168490"
                        open
                      >
                        <summary class="article-expander__title">Refining the Treatment of Pancreatic Cancer From Big Data to Improved Individual Survival</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37168490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pancreatic cancer is one of the most lethal cancers worldwide, most notably in Europe and North America. Great strides have been made in combining the most effective conventional therapies to improve survival at least in the short and medium term. The start of treatment can only be made once a diagnosis is made, which at this point, the tumor volume is already very high in the primary cancer and systemically. If caught at the earliest opportunity (in circa 20% patients) surgical resection of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37170152"
                        open
                      >
                        <summary class="article-expander__title">Exploring circular MET RNA as a potential biomarker in tumors exhibiting high MET activity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37170152/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We found that measurement of circMET levels allows identification and tracking of patients characterized by high MET activity. Circulating circMET (ccMET) detection and analysis could be a simple, cost-effective, non-invasive approach to better implement patient stratification based on MET expression, as well as to dynamically monitor over time both therapy response and clonal evolution during treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37168490"
                        open
                      >
                        <summary class="article-expander__title">Refining the Treatment of Pancreatic Cancer From Big Data to Improved Individual Survival</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37168490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pancreatic cancer is one of the most lethal cancers worldwide, most notably in Europe and North America. Great strides have been made in combining the most effective conventional therapies to improve survival at least in the short and medium term. The start of treatment can only be made once a diagnosis is made, which at this point, the tumor volume is already very high in the primary cancer and systemically. If caught at the earliest opportunity (in circa 20% patients) surgical resection of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37167533"
                        open
                      >
                        <summary class="article-expander__title">Transdifferentiation, phenotypic infidelity, progression, and transformation in T/NK-cell neoplasms: Report from the 2021 SH/EAHP Workshop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37167533/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230511205726&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This cohort of cases allowed us to illustrate, discuss, and review current concepts of transdifferentiation, aberrant antigen expression, and progression in various T/NK-cell neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-05-09 datetime="2023-05-09T10:00:00.000Z">2023-05-09</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-05-09 datetime="2023-05-09T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37160316"
                        open
                      >
                        <summary class="article-expander__title">Clonal cytopenia of undetermined significance (CCUS)-associated reversion of donor-derived, transient αβ T-cell large granular clonal lymphocytosis, emerging post-transplant in a patient with a history of γδ T-cell large granular lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37160316/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230528210011&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous and allogeneic hematopoietic stem cell transplantation (HSCT) has revolutionized the therapy of hematolymphoid malignancies. Yet, how to best detect or predict the emergence of HSCT-related complications remain unresolved. Here, we describe a case of donor-derived, transient Alpha Beta (αβ) T-cell large granular clonal lymphocytosis and cytopenia that emerged post-HSCT in a patient with a history of gamma delta (γδ) T-cell large granular lymphocytic leukemia (T-LGLL). Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37160316"
                        open
                      >
                        <summary class="article-expander__title">Clonal cytopenia of undetermined significance (CCUS)-associated reversion of donor-derived, transient αβ T-cell large granular clonal lymphocytosis, emerging post-transplant in a patient with a history of γδ T-cell large granular lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37160316/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230527210349&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous and allogeneic hematopoietic stem cell transplantation (HSCT) has revolutionized the therapy of hematolymphoid malignancies. Yet, how to best detect or predict the emergence of HSCT-related complications remain unresolved. Here, we describe a case of donor-derived, transient Alpha Beta (αβ) T-cell large granular clonal lymphocytosis and cytopenia that emerged post-HSCT in a patient with a history of gamma delta (γδ) T-cell large granular lymphocytic leukemia (T-LGLL). Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37160316"
                        open
                      >
                        <summary class="article-expander__title">Clonal cytopenia of undetermined significance (CCUS)-associated reversion of donor-derived, transient αβ T-cell large granular clonal lymphocytosis, emerging post-transplant in a patient with a history of γδ T-cell large granular lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37160316/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230526210002&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous and allogeneic hematopoietic stem cell transplantation (HSCT) has revolutionized the therapy of hematolymphoid malignancies. Yet, how to best detect or predict the emergence of HSCT-related complications remain unresolved. Here, we describe a case of donor-derived, transient Alpha Beta (αβ) T-cell large granular clonal lymphocytosis and cytopenia that emerged post-HSCT in a patient with a history of gamma delta (γδ) T-cell large granular lymphocytic leukemia (T-LGLL). Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37160316"
                        open
                      >
                        <summary class="article-expander__title">Clonal cytopenia of undetermined significance (CCUS)-associated reversion of donor-derived, transient αβ T-cell large granular clonal lymphocytosis, emerging post-transplant in a patient with a history of γδ T-cell large granular lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37160316/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous and allogeneic hematopoietic stem cell transplantation (HSCT) has revolutionized the therapy of hematolymphoid malignancies. Yet, how to best detect or predict the emergence of HSCT-related complications remain unresolved. Here, we describe a case of donor-derived, transient Alpha Beta (αβ) T-cell large granular clonal lymphocytosis and cytopenia that emerged post-HSCT in a patient with a history of gamma delta (γδ) T-cell large granular lymphocytic leukemia (T-LGLL). Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37159554"
                        open
                      >
                        <summary class="article-expander__title">Suspected Germline TP53 Variants and Clonal Hematopoiesis of Indeterminate Potential: Lessons Learned From a Molecular Tumor Board</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37159554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230525205746&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Considering the low TP53 variant allele fraction in blood, absence of variant detection in oral swab and saliva samples, the lack of LFS clinical criteria, and history of exposure to environmental risk factors for cancer, the main hypothesis for this case was aberrant clonal expansion due to clonal hematopoiesis. Oncologists should interpret TP53 findings during germline testing with caution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37160316"
                        open
                      >
                        <summary class="article-expander__title">Clonal cytopenia of undetermined significance (CCUS)-associated reversion of donor-derived, transient αβ T-cell large granular clonal lymphocytosis, emerging post-transplant in a patient with a history of γδ T-cell large granular lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37160316/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous and allogeneic hematopoietic stem cell transplantation (HSCT) has revolutionized the therapy of hematolymphoid malignancies. Yet, how to best detect or predict the emergence of HSCT-related complications remain unresolved. Here, we describe a case of donor-derived, transient Alpha Beta (αβ) T-cell large granular clonal lymphocytosis and cytopenia that emerged post-HSCT in a patient with a history of gamma delta (γδ) T-cell large granular lymphocytic leukemia (T-LGLL). Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37159554"
                        open
                      >
                        <summary class="article-expander__title">Suspected Germline TP53 Variants and Clonal Hematopoiesis of Indeterminate Potential: Lessons Learned From a Molecular Tumor Board</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37159554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Considering the low TP53 variant allele fraction in blood, absence of variant detection in oral swab and saliva samples, the lack of LFS clinical criteria, and history of exposure to environmental risk factors for cancer, the main hypothesis for this case was aberrant clonal expansion due to clonal hematopoiesis. Oncologists should interpret TP53 findings during germline testing with caution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37160316"
                        open
                      >
                        <summary class="article-expander__title">Clonal cytopenia of undetermined significance (CCUS)-associated reversion of donor-derived, transient αβ T-cell large granular clonal lymphocytosis, emerging post-transplant in a patient with a history of γδ T-cell large granular lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37160316/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous and allogeneic hematopoietic stem cell transplantation (HSCT) has revolutionized the therapy of hematolymphoid malignancies. Yet, how to best detect or predict the emergence of HSCT-related complications remain unresolved. Here, we describe a case of donor-derived, transient Alpha Beta (αβ) T-cell large granular clonal lymphocytosis and cytopenia that emerged post-HSCT in a patient with a history of gamma delta (γδ) T-cell large granular lymphocytic leukemia (T-LGLL). Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37159554"
                        open
                      >
                        <summary class="article-expander__title">Suspected Germline TP53 Variants and Clonal Hematopoiesis of Indeterminate Potential: Lessons Learned From a Molecular Tumor Board</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37159554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Considering the low TP53 variant allele fraction in blood, absence of variant detection in oral swab and saliva samples, the lack of LFS clinical criteria, and history of exposure to environmental risk factors for cancer, the main hypothesis for this case was aberrant clonal expansion due to clonal hematopoiesis. Oncologists should interpret TP53 findings during germline testing with caution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37160316"
                        open
                      >
                        <summary class="article-expander__title">Clonal cytopenia of undetermined significance (CCUS)-associated reversion of donor-derived, transient αβ T-cell large granular clonal lymphocytosis, emerging post-transplant in a patient with a history of γδ T-cell large granular lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37160316/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous and allogeneic hematopoietic stem cell transplantation (HSCT) has revolutionized the therapy of hematolymphoid malignancies. Yet, how to best detect or predict the emergence of HSCT-related complications remain unresolved. Here, we describe a case of donor-derived, transient Alpha Beta (αβ) T-cell large granular clonal lymphocytosis and cytopenia that emerged post-HSCT in a patient with a history of gamma delta (γδ) T-cell large granular lymphocytic leukemia (T-LGLL). Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37159554"
                        open
                      >
                        <summary class="article-expander__title">Suspected Germline TP53 Variants and Clonal Hematopoiesis of Indeterminate Potential: Lessons Learned From a Molecular Tumor Board</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37159554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Considering the low TP53 variant allele fraction in blood, absence of variant detection in oral swab and saliva samples, the lack of LFS clinical criteria, and history of exposure to environmental risk factors for cancer, the main hypothesis for this case was aberrant clonal expansion due to clonal hematopoiesis. Oncologists should interpret TP53 findings during germline testing with caution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37160316"
                        open
                      >
                        <summary class="article-expander__title">Clonal cytopenia of undetermined significance (CCUS)-associated reversion of donor-derived, transient αβ T-cell large granular clonal lymphocytosis, emerging post-transplant in a patient with a history of γδ T-cell large granular lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37160316/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230521210055&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous and allogeneic hematopoietic stem cell transplantation (HSCT) has revolutionized the therapy of hematolymphoid malignancies. Yet, how to best detect or predict the emergence of HSCT-related complications remain unresolved. Here, we describe a case of donor-derived, transient Alpha Beta (αβ) T-cell large granular clonal lymphocytosis and cytopenia that emerged post-HSCT in a patient with a history of gamma delta (γδ) T-cell large granular lymphocytic leukemia (T-LGLL). Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37159554"
                        open
                      >
                        <summary class="article-expander__title">Suspected Germline TP53 Variants and Clonal Hematopoiesis of Indeterminate Potential: Lessons Learned From a Molecular Tumor Board</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37159554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230521210055&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Considering the low TP53 variant allele fraction in blood, absence of variant detection in oral swab and saliva samples, the lack of LFS clinical criteria, and history of exposure to environmental risk factors for cancer, the main hypothesis for this case was aberrant clonal expansion due to clonal hematopoiesis. Oncologists should interpret TP53 findings during germline testing with caution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37160316"
                        open
                      >
                        <summary class="article-expander__title">Clonal cytopenia of undetermined significance (CCUS)-associated reversion of donor-derived, transient αβ T-cell large granular clonal lymphocytosis, emerging post-transplant in a patient with a history of γδ T-cell large granular lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37160316/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous and allogeneic hematopoietic stem cell transplantation (HSCT) has revolutionized the therapy of hematolymphoid malignancies. Yet, how to best detect or predict the emergence of HSCT-related complications remain unresolved. Here, we describe a case of donor-derived, transient Alpha Beta (αβ) T-cell large granular clonal lymphocytosis and cytopenia that emerged post-HSCT in a patient with a history of gamma delta (γδ) T-cell large granular lymphocytic leukemia (T-LGLL). Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37159554"
                        open
                      >
                        <summary class="article-expander__title">Suspected Germline TP53 Variants and Clonal Hematopoiesis of Indeterminate Potential: Lessons Learned From a Molecular Tumor Board</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37159554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Considering the low TP53 variant allele fraction in blood, absence of variant detection in oral swab and saliva samples, the lack of LFS clinical criteria, and history of exposure to environmental risk factors for cancer, the main hypothesis for this case was aberrant clonal expansion due to clonal hematopoiesis. Oncologists should interpret TP53 findings during germline testing with caution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37160316"
                        open
                      >
                        <summary class="article-expander__title">Clonal cytopenia of undetermined significance (CCUS)-associated reversion of donor-derived, transient αβ T-cell large granular clonal lymphocytosis, emerging post-transplant in a patient with a history of γδ T-cell large granular lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37160316/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous and allogeneic hematopoietic stem cell transplantation (HSCT) has revolutionized the therapy of hematolymphoid malignancies. Yet, how to best detect or predict the emergence of HSCT-related complications remain unresolved. Here, we describe a case of donor-derived, transient Alpha Beta (αβ) T-cell large granular clonal lymphocytosis and cytopenia that emerged post-HSCT in a patient with a history of gamma delta (γδ) T-cell large granular lymphocytic leukemia (T-LGLL). Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37159554"
                        open
                      >
                        <summary class="article-expander__title">Suspected Germline TP53 Variants and Clonal Hematopoiesis of Indeterminate Potential: Lessons Learned From a Molecular Tumor Board</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37159554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Considering the low TP53 variant allele fraction in blood, absence of variant detection in oral swab and saliva samples, the lack of LFS clinical criteria, and history of exposure to environmental risk factors for cancer, the main hypothesis for this case was aberrant clonal expansion due to clonal hematopoiesis. Oncologists should interpret TP53 findings during germline testing with caution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37160316"
                        open
                      >
                        <summary class="article-expander__title">Clonal cytopenia of undetermined significance (CCUS)-associated reversion of donor-derived, transient αβ T-cell large granular clonal lymphocytosis, emerging post-transplant in a patient with a history of γδ T-cell large granular lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37160316/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous and allogeneic hematopoietic stem cell transplantation (HSCT) has revolutionized the therapy of hematolymphoid malignancies. Yet, how to best detect or predict the emergence of HSCT-related complications remain unresolved. Here, we describe a case of donor-derived, transient Alpha Beta (αβ) T-cell large granular clonal lymphocytosis and cytopenia that emerged post-HSCT in a patient with a history of gamma delta (γδ) T-cell large granular lymphocytic leukemia (T-LGLL). Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37159554"
                        open
                      >
                        <summary class="article-expander__title">Suspected Germline TP53 Variants and Clonal Hematopoiesis of Indeterminate Potential: Lessons Learned From a Molecular Tumor Board</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37159554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Considering the low TP53 variant allele fraction in blood, absence of variant detection in oral swab and saliva samples, the lack of LFS clinical criteria, and history of exposure to environmental risk factors for cancer, the main hypothesis for this case was aberrant clonal expansion due to clonal hematopoiesis. Oncologists should interpret TP53 findings during germline testing with caution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37160316"
                        open
                      >
                        <summary class="article-expander__title">Clonal cytopenia of undetermined significance (CCUS)-associated reversion of donor-derived, transient αβ T-cell large granular clonal lymphocytosis, emerging post-transplant in a patient with a history of γδ T-cell large granular lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37160316/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous and allogeneic hematopoietic stem cell transplantation (HSCT) has revolutionized the therapy of hematolymphoid malignancies. Yet, how to best detect or predict the emergence of HSCT-related complications remain unresolved. Here, we describe a case of donor-derived, transient Alpha Beta (αβ) T-cell large granular clonal lymphocytosis and cytopenia that emerged post-HSCT in a patient with a history of gamma delta (γδ) T-cell large granular lymphocytic leukemia (T-LGLL). Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37159554"
                        open
                      >
                        <summary class="article-expander__title">Suspected Germline TP53 Variants and Clonal Hematopoiesis of Indeterminate Potential: Lessons Learned From a Molecular Tumor Board</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37159554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Considering the low TP53 variant allele fraction in blood, absence of variant detection in oral swab and saliva samples, the lack of LFS clinical criteria, and history of exposure to environmental risk factors for cancer, the main hypothesis for this case was aberrant clonal expansion due to clonal hematopoiesis. Oncologists should interpret TP53 findings during germline testing with caution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37160316"
                        open
                      >
                        <summary class="article-expander__title">Clonal cytopenia of undetermined significance (CCUS)-associated reversion of donor-derived, transient αβ T-cell large granular clonal lymphocytosis, emerging post-transplant in a patient with a history of γδ T-cell large granular lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37160316/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous and allogeneic hematopoietic stem cell transplantation (HSCT) has revolutionized the therapy of hematolymphoid malignancies. Yet, how to best detect or predict the emergence of HSCT-related complications remain unresolved. Here, we describe a case of donor-derived, transient Alpha Beta (αβ) T-cell large granular clonal lymphocytosis and cytopenia that emerged post-HSCT in a patient with a history of gamma delta (γδ) T-cell large granular lymphocytic leukemia (T-LGLL). Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37159554"
                        open
                      >
                        <summary class="article-expander__title">Suspected Germline TP53 Variants and Clonal Hematopoiesis of Indeterminate Potential: Lessons Learned From a Molecular Tumor Board</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37159554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Considering the low TP53 variant allele fraction in blood, absence of variant detection in oral swab and saliva samples, the lack of LFS clinical criteria, and history of exposure to environmental risk factors for cancer, the main hypothesis for this case was aberrant clonal expansion due to clonal hematopoiesis. Oncologists should interpret TP53 findings during germline testing with caution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37160316"
                        open
                      >
                        <summary class="article-expander__title">Clonal cytopenia of undetermined significance (CCUS)-associated reversion of donor-derived, transient αβ T-cell large granular clonal lymphocytosis, emerging post-transplant in a patient with a history of γδ T-cell large granular lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37160316/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230515205819&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous and allogeneic hematopoietic stem cell transplantation (HSCT) has revolutionized the therapy of hematolymphoid malignancies. Yet, how to best detect or predict the emergence of HSCT-related complications remain unresolved. Here, we describe a case of donor-derived, transient Alpha Beta (αβ) T-cell large granular clonal lymphocytosis and cytopenia that emerged post-HSCT in a patient with a history of gamma delta (γδ) T-cell large granular lymphocytic leukemia (T-LGLL). Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37159554"
                        open
                      >
                        <summary class="article-expander__title">Suspected Germline TP53 Variants and Clonal Hematopoiesis of Indeterminate Potential: Lessons Learned From a Molecular Tumor Board</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37159554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230515205819&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Considering the low TP53 variant allele fraction in blood, absence of variant detection in oral swab and saliva samples, the lack of LFS clinical criteria, and history of exposure to environmental risk factors for cancer, the main hypothesis for this case was aberrant clonal expansion due to clonal hematopoiesis. Oncologists should interpret TP53 findings during germline testing with caution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37160316"
                        open
                      >
                        <summary class="article-expander__title">Clonal cytopenia of undetermined significance (CCUS)-associated reversion of donor-derived, transient αβ T-cell large granular clonal lymphocytosis, emerging post-transplant in a patient with a history of γδ T-cell large granular lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37160316/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous and allogeneic hematopoietic stem cell transplantation (HSCT) has revolutionized the therapy of hematolymphoid malignancies. Yet, how to best detect or predict the emergence of HSCT-related complications remain unresolved. Here, we describe a case of donor-derived, transient Alpha Beta (αβ) T-cell large granular clonal lymphocytosis and cytopenia that emerged post-HSCT in a patient with a history of gamma delta (γδ) T-cell large granular lymphocytic leukemia (T-LGLL). Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37159554"
                        open
                      >
                        <summary class="article-expander__title">Suspected Germline TP53 Variants and Clonal Hematopoiesis of Indeterminate Potential: Lessons Learned From a Molecular Tumor Board</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37159554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Considering the low TP53 variant allele fraction in blood, absence of variant detection in oral swab and saliva samples, the lack of LFS clinical criteria, and history of exposure to environmental risk factors for cancer, the main hypothesis for this case was aberrant clonal expansion due to clonal hematopoiesis. Oncologists should interpret TP53 findings during germline testing with caution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37160316"
                        open
                      >
                        <summary class="article-expander__title">Clonal cytopenia of undetermined significance (CCUS)-associated reversion of donor-derived, transient αβ T-cell large granular clonal lymphocytosis, emerging post-transplant in a patient with a history of γδ T-cell large granular lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37160316/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous and allogeneic hematopoietic stem cell transplantation (HSCT) has revolutionized the therapy of hematolymphoid malignancies. Yet, how to best detect or predict the emergence of HSCT-related complications remain unresolved. Here, we describe a case of donor-derived, transient Alpha Beta (αβ) T-cell large granular clonal lymphocytosis and cytopenia that emerged post-HSCT in a patient with a history of gamma delta (γδ) T-cell large granular lymphocytic leukemia (T-LGLL). Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37159554"
                        open
                      >
                        <summary class="article-expander__title">Suspected Germline TP53 Variants and Clonal Hematopoiesis of Indeterminate Potential: Lessons Learned From a Molecular Tumor Board</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37159554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Considering the low TP53 variant allele fraction in blood, absence of variant detection in oral swab and saliva samples, the lack of LFS clinical criteria, and history of exposure to environmental risk factors for cancer, the main hypothesis for this case was aberrant clonal expansion due to clonal hematopoiesis. Oncologists should interpret TP53 findings during germline testing with caution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37160316"
                        open
                      >
                        <summary class="article-expander__title">Clonal cytopenia of undetermined significance (CCUS)-associated reversion of donor-derived, transient αβ T-cell large granular clonal lymphocytosis, emerging post-transplant in a patient with a history of γδ T-cell large granular lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37160316/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous and allogeneic hematopoietic stem cell transplantation (HSCT) has revolutionized the therapy of hematolymphoid malignancies. Yet, how to best detect or predict the emergence of HSCT-related complications remain unresolved. Here, we describe a case of donor-derived, transient Alpha Beta (αβ) T-cell large granular clonal lymphocytosis and cytopenia that emerged post-HSCT in a patient with a history of gamma delta (γδ) T-cell large granular lymphocytic leukemia (T-LGLL). Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37159554"
                        open
                      >
                        <summary class="article-expander__title">Suspected Germline TP53 Variants and Clonal Hematopoiesis of Indeterminate Potential: Lessons Learned From a Molecular Tumor Board</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37159554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Considering the low TP53 variant allele fraction in blood, absence of variant detection in oral swab and saliva samples, the lack of LFS clinical criteria, and history of exposure to environmental risk factors for cancer, the main hypothesis for this case was aberrant clonal expansion due to clonal hematopoiesis. Oncologists should interpret TP53 findings during germline testing with caution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37160316"
                        open
                      >
                        <summary class="article-expander__title">Clonal cytopenia of undetermined significance (CCUS)-associated reversion of donor-derived, transient αβ T-cell large granular clonal lymphocytosis, emerging post-transplant in a patient with a history of γδ T-cell large granular lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37160316/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230511205726&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous and allogeneic hematopoietic stem cell transplantation (HSCT) has revolutionized the therapy of hematolymphoid malignancies. Yet, how to best detect or predict the emergence of HSCT-related complications remain unresolved. Here, we describe a case of donor-derived, transient Alpha Beta (αβ) T-cell large granular clonal lymphocytosis and cytopenia that emerged post-HSCT in a patient with a history of gamma delta (γδ) T-cell large granular lymphocytic leukemia (T-LGLL). Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37159554"
                        open
                      >
                        <summary class="article-expander__title">Suspected Germline TP53 Variants and Clonal Hematopoiesis of Indeterminate Potential: Lessons Learned From a Molecular Tumor Board</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37159554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230511205726&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Considering the low TP53 variant allele fraction in blood, absence of variant detection in oral swab and saliva samples, the lack of LFS clinical criteria, and history of exposure to environmental risk factors for cancer, the main hypothesis for this case was aberrant clonal expansion due to clonal hematopoiesis. Oncologists should interpret TP53 findings during germline testing with caution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37159554"
                        open
                      >
                        <summary class="article-expander__title">Suspected Germline TP53 Variants and Clonal Hematopoiesis of Indeterminate Potential: Lessons Learned From a Molecular Tumor Board</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37159554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230510205718&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Considering the low TP53 variant allele fraction in blood, absence of variant detection in oral swab and saliva samples, the lack of LFS clinical criteria, and history of exposure to environmental risk factors for cancer, the main hypothesis for this case was aberrant clonal expansion due to clonal hematopoiesis. Oncologists should interpret TP53 findings during germline testing with caution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37159554"
                        open
                      >
                        <summary class="article-expander__title">Suspected Germline TP53 Variants and Clonal Hematopoiesis of Indeterminate Potential: Lessons Learned From a Molecular Tumor Board</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37159554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230509205648&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Considering the low TP53 variant allele fraction in blood, absence of variant detection in oral swab and saliva samples, the lack of LFS clinical criteria, and history of exposure to environmental risk factors for cancer, the main hypothesis for this case was aberrant clonal expansion due to clonal hematopoiesis. Oncologists should interpret TP53 findings during germline testing with caution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37160316"
                        open
                      >
                        <summary class="article-expander__title">Clonal cytopenia of undetermined significance (CCUS)-associated reversion of donor-derived, transient αβ T-cell large granular clonal lymphocytosis, emerging post-transplant in a patient with a history of γδ T-cell large granular lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37160316/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous and allogeneic hematopoietic stem cell transplantation (HSCT) has revolutionized the therapy of hematolymphoid malignancies. Yet, how to best detect or predict the emergence of HSCT-related complications remain unresolved. Here, we describe a case of donor-derived, transient Alpha Beta (αβ) T-cell large granular clonal lymphocytosis and cytopenia that emerged post-HSCT in a patient with a history of gamma delta (γδ) T-cell large granular lymphocytic leukemia (T-LGLL). Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37160316"
                        open
                      >
                        <summary class="article-expander__title">Clonal cytopenia of undetermined significance (CCUS)-associated reversion of donor-derived, transient αβ T-cell large granular clonal lymphocytosis, emerging post-transplant in a patient with a history of γδ T-cell large granular lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37160316/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous and allogeneic hematopoietic stem cell transplantation (HSCT) has revolutionized the therapy of hematolymphoid malignancies. Yet, how to best detect or predict the emergence of HSCT-related complications remain unresolved. Here, we describe a case of donor-derived, transient Alpha Beta (αβ) T-cell large granular clonal lymphocytosis and cytopenia that emerged post-HSCT in a patient with a history of gamma delta (γδ) T-cell large granular lymphocytic leukemia (T-LGLL). Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37157910"
                        open
                      >
                        <summary class="article-expander__title">Reuniting philosophy and science to advance cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37157910/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers rely on multiple, heterogeneous processes at different scales, pertaining to many biomedical fields. Therefore, understanding cancer is necessarily an interdisciplinary task that requires placing specialised experimental and clinical research into a broader conceptual, theoretical, and methodological framework. Without such a framework, oncology will collect piecemeal results, with scant dialogue between the different scientific communities studying cancer. We argue that one important...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37160316"
                        open
                      >
                        <summary class="article-expander__title">Clonal cytopenia of undetermined significance (CCUS)-associated reversion of donor-derived, transient αβ T-cell large granular clonal lymphocytosis, emerging post-transplant in a patient with a history of γδ T-cell large granular lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37160316/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous and allogeneic hematopoietic stem cell transplantation (HSCT) has revolutionized the therapy of hematolymphoid malignancies. Yet, how to best detect or predict the emergence of HSCT-related complications remain unresolved. Here, we describe a case of donor-derived, transient Alpha Beta (αβ) T-cell large granular clonal lymphocytosis and cytopenia that emerged post-HSCT in a patient with a history of gamma delta (γδ) T-cell large granular lymphocytic leukemia (T-LGLL). Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37157910"
                        open
                      >
                        <summary class="article-expander__title">Reuniting philosophy and science to advance cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37157910/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers rely on multiple, heterogeneous processes at different scales, pertaining to many biomedical fields. Therefore, understanding cancer is necessarily an interdisciplinary task that requires placing specialised experimental and clinical research into a broader conceptual, theoretical, and methodological framework. Without such a framework, oncology will collect piecemeal results, with scant dialogue between the different scientific communities studying cancer. We argue that one important...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37160316"
                        open
                      >
                        <summary class="article-expander__title">Clonal cytopenia of undetermined significance (CCUS)-associated reversion of donor-derived, transient αβ T-cell large granular clonal lymphocytosis, emerging post-transplant in a patient with a history of γδ T-cell large granular lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37160316/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous and allogeneic hematopoietic stem cell transplantation (HSCT) has revolutionized the therapy of hematolymphoid malignancies. Yet, how to best detect or predict the emergence of HSCT-related complications remain unresolved. Here, we describe a case of donor-derived, transient Alpha Beta (αβ) T-cell large granular clonal lymphocytosis and cytopenia that emerged post-HSCT in a patient with a history of gamma delta (γδ) T-cell large granular lymphocytic leukemia (T-LGLL). Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37157910"
                        open
                      >
                        <summary class="article-expander__title">Reuniting philosophy and science to advance cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37157910/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers rely on multiple, heterogeneous processes at different scales, pertaining to many biomedical fields. Therefore, understanding cancer is necessarily an interdisciplinary task that requires placing specialised experimental and clinical research into a broader conceptual, theoretical, and methodological framework. Without such a framework, oncology will collect piecemeal results, with scant dialogue between the different scientific communities studying cancer. We argue that one important...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37160316"
                        open
                      >
                        <summary class="article-expander__title">Clonal cytopenia of undetermined significance (CCUS)-associated reversion of donor-derived, transient αβ T-cell large granular clonal lymphocytosis, emerging post-transplant in a patient with a history of γδ T-cell large granular lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37160316/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230515205818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous and allogeneic hematopoietic stem cell transplantation (HSCT) has revolutionized the therapy of hematolymphoid malignancies. Yet, how to best detect or predict the emergence of HSCT-related complications remain unresolved. Here, we describe a case of donor-derived, transient Alpha Beta (αβ) T-cell large granular clonal lymphocytosis and cytopenia that emerged post-HSCT in a patient with a history of gamma delta (γδ) T-cell large granular lymphocytic leukemia (T-LGLL). Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37157910"
                        open
                      >
                        <summary class="article-expander__title">Reuniting philosophy and science to advance cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37157910/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230515205818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers rely on multiple, heterogeneous processes at different scales, pertaining to many biomedical fields. Therefore, understanding cancer is necessarily an interdisciplinary task that requires placing specialised experimental and clinical research into a broader conceptual, theoretical, and methodological framework. Without such a framework, oncology will collect piecemeal results, with scant dialogue between the different scientific communities studying cancer. We argue that one important...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37160316"
                        open
                      >
                        <summary class="article-expander__title">Clonal cytopenia of undetermined significance (CCUS)-associated reversion of donor-derived, transient αβ T-cell large granular clonal lymphocytosis, emerging post-transplant in a patient with a history of γδ T-cell large granular lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37160316/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous and allogeneic hematopoietic stem cell transplantation (HSCT) has revolutionized the therapy of hematolymphoid malignancies. Yet, how to best detect or predict the emergence of HSCT-related complications remain unresolved. Here, we describe a case of donor-derived, transient Alpha Beta (αβ) T-cell large granular clonal lymphocytosis and cytopenia that emerged post-HSCT in a patient with a history of gamma delta (γδ) T-cell large granular lymphocytic leukemia (T-LGLL). Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37157910"
                        open
                      >
                        <summary class="article-expander__title">Reuniting philosophy and science to advance cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37157910/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers rely on multiple, heterogeneous processes at different scales, pertaining to many biomedical fields. Therefore, understanding cancer is necessarily an interdisciplinary task that requires placing specialised experimental and clinical research into a broader conceptual, theoretical, and methodological framework. Without such a framework, oncology will collect piecemeal results, with scant dialogue between the different scientific communities studying cancer. We argue that one important...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37160316"
                        open
                      >
                        <summary class="article-expander__title">Clonal cytopenia of undetermined significance (CCUS)-associated reversion of donor-derived, transient αβ T-cell large granular clonal lymphocytosis, emerging post-transplant in a patient with a history of γδ T-cell large granular lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37160316/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous and allogeneic hematopoietic stem cell transplantation (HSCT) has revolutionized the therapy of hematolymphoid malignancies. Yet, how to best detect or predict the emergence of HSCT-related complications remain unresolved. Here, we describe a case of donor-derived, transient Alpha Beta (αβ) T-cell large granular clonal lymphocytosis and cytopenia that emerged post-HSCT in a patient with a history of gamma delta (γδ) T-cell large granular lymphocytic leukemia (T-LGLL). Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37157910"
                        open
                      >
                        <summary class="article-expander__title">Reuniting philosophy and science to advance cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37157910/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers rely on multiple, heterogeneous processes at different scales, pertaining to many biomedical fields. Therefore, understanding cancer is necessarily an interdisciplinary task that requires placing specialised experimental and clinical research into a broader conceptual, theoretical, and methodological framework. Without such a framework, oncology will collect piecemeal results, with scant dialogue between the different scientific communities studying cancer. We argue that one important...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37160316"
                        open
                      >
                        <summary class="article-expander__title">Clonal cytopenia of undetermined significance (CCUS)-associated reversion of donor-derived, transient αβ T-cell large granular clonal lymphocytosis, emerging post-transplant in a patient with a history of γδ T-cell large granular lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37160316/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous and allogeneic hematopoietic stem cell transplantation (HSCT) has revolutionized the therapy of hematolymphoid malignancies. Yet, how to best detect or predict the emergence of HSCT-related complications remain unresolved. Here, we describe a case of donor-derived, transient Alpha Beta (αβ) T-cell large granular clonal lymphocytosis and cytopenia that emerged post-HSCT in a patient with a history of gamma delta (γδ) T-cell large granular lymphocytic leukemia (T-LGLL). Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37157910"
                        open
                      >
                        <summary class="article-expander__title">Reuniting philosophy and science to advance cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37157910/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers rely on multiple, heterogeneous processes at different scales, pertaining to many biomedical fields. Therefore, understanding cancer is necessarily an interdisciplinary task that requires placing specialised experimental and clinical research into a broader conceptual, theoretical, and methodological framework. Without such a framework, oncology will collect piecemeal results, with scant dialogue between the different scientific communities studying cancer. We argue that one important...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37160316"
                        open
                      >
                        <summary class="article-expander__title">Clonal cytopenia of undetermined significance (CCUS)-associated reversion of donor-derived, transient αβ T-cell large granular clonal lymphocytosis, emerging post-transplant in a patient with a history of γδ T-cell large granular lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37160316/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230511205726&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous and allogeneic hematopoietic stem cell transplantation (HSCT) has revolutionized the therapy of hematolymphoid malignancies. Yet, how to best detect or predict the emergence of HSCT-related complications remain unresolved. Here, we describe a case of donor-derived, transient Alpha Beta (αβ) T-cell large granular clonal lymphocytosis and cytopenia that emerged post-HSCT in a patient with a history of gamma delta (γδ) T-cell large granular lymphocytic leukemia (T-LGLL). Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37157910"
                        open
                      >
                        <summary class="article-expander__title">Reuniting philosophy and science to advance cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37157910/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230511205726&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers rely on multiple, heterogeneous processes at different scales, pertaining to many biomedical fields. Therefore, understanding cancer is necessarily an interdisciplinary task that requires placing specialised experimental and clinical research into a broader conceptual, theoretical, and methodological framework. Without such a framework, oncology will collect piecemeal results, with scant dialogue between the different scientific communities studying cancer. We argue that one important...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37157910"
                        open
                      >
                        <summary class="article-expander__title">Reuniting philosophy and science to advance cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37157910/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230510205718&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers rely on multiple, heterogeneous processes at different scales, pertaining to many biomedical fields. Therefore, understanding cancer is necessarily an interdisciplinary task that requires placing specialised experimental and clinical research into a broader conceptual, theoretical, and methodological framework. Without such a framework, oncology will collect piecemeal results, with scant dialogue between the different scientific communities studying cancer. We argue that one important...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37157910"
                        open
                      >
                        <summary class="article-expander__title">Reuniting philosophy and science to advance cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37157910/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230509205648&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers rely on multiple, heterogeneous processes at different scales, pertaining to many biomedical fields. Therefore, understanding cancer is necessarily an interdisciplinary task that requires placing specialised experimental and clinical research into a broader conceptual, theoretical, and methodological framework. Without such a framework, oncology will collect piecemeal results, with scant dialogue between the different scientific communities studying cancer. We argue that one important...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-05-08 datetime="2023-05-08T10:00:00.000Z">2023-05-08</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-05-08 datetime="2023-05-08T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37156299"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential and outcomes after heart transplantation: A multi-center study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37156299/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230524205854&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiac allograft vasculopathy (CAV) is a leading cause of late graft failure and mortality after heart transplantation (HT). Sharing some features with atherosclerosis, CAV results in diffuse narrowing of the epicardial coronaries and microvasculature with consequent graft ischemia. Recently, clonal hematopoiesis of indeterminate potential (CHIP) has emerged as a risk factor for cardiovascular disease and mortality. We aimed to investigate the relationship between CHIP and post-transplant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37156299"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential and outcomes after heart transplantation: A multi-center study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37156299/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiac allograft vasculopathy (CAV) is a leading cause of late graft failure and mortality after heart transplantation (HT). Sharing some features with atherosclerosis, CAV results in diffuse narrowing of the epicardial coronaries and microvasculature with consequent graft ischemia. Recently, clonal hematopoiesis of indeterminate potential (CHIP) has emerged as a risk factor for cardiovascular disease and mortality. We aimed to investigate the relationship between CHIP and post-transplant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37153874"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Architecture on Clinical Course and Prognosis in Patients With Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37153874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are caused by a somatic gain-of-function mutation in 1 of the 3 disease driver genes JAK2, MPL, or CALR. About half of the MPNs patients also carry additional somatic mutations that modify the clinical course. The order of acquisition of these gene mutations has been proposed to influence the phenotype and evolution of the disease. We studied 50 JAK2-V617F-positive MPN patients who carried at least 1 additional somatic mutation and determined the clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37156299"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential and outcomes after heart transplantation: A multi-center study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37156299/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiac allograft vasculopathy (CAV) is a leading cause of late graft failure and mortality after heart transplantation (HT). Sharing some features with atherosclerosis, CAV results in diffuse narrowing of the epicardial coronaries and microvasculature with consequent graft ischemia. Recently, clonal hematopoiesis of indeterminate potential (CHIP) has emerged as a risk factor for cardiovascular disease and mortality. We aimed to investigate the relationship between CHIP and post-transplant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37153874"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Architecture on Clinical Course and Prognosis in Patients With Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37153874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are caused by a somatic gain-of-function mutation in 1 of the 3 disease driver genes JAK2, MPL, or CALR. About half of the MPNs patients also carry additional somatic mutations that modify the clinical course. The order of acquisition of these gene mutations has been proposed to influence the phenotype and evolution of the disease. We studied 50 JAK2-V617F-positive MPN patients who carried at least 1 additional somatic mutation and determined the clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37156299"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential and outcomes after heart transplantation: A multi-center study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37156299/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230521210055&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiac allograft vasculopathy (CAV) is a leading cause of late graft failure and mortality after heart transplantation (HT). Sharing some features with atherosclerosis, CAV results in diffuse narrowing of the epicardial coronaries and microvasculature with consequent graft ischemia. Recently, clonal hematopoiesis of indeterminate potential (CHIP) has emerged as a risk factor for cardiovascular disease and mortality. We aimed to investigate the relationship between CHIP and post-transplant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37153874"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Architecture on Clinical Course and Prognosis in Patients With Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37153874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230521210055&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are caused by a somatic gain-of-function mutation in 1 of the 3 disease driver genes JAK2, MPL, or CALR. About half of the MPNs patients also carry additional somatic mutations that modify the clinical course. The order of acquisition of these gene mutations has been proposed to influence the phenotype and evolution of the disease. We studied 50 JAK2-V617F-positive MPN patients who carried at least 1 additional somatic mutation and determined the clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37156299"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential and outcomes after heart transplantation: A multi-center study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37156299/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiac allograft vasculopathy (CAV) is a leading cause of late graft failure and mortality after heart transplantation (HT). Sharing some features with atherosclerosis, CAV results in diffuse narrowing of the epicardial coronaries and microvasculature with consequent graft ischemia. Recently, clonal hematopoiesis of indeterminate potential (CHIP) has emerged as a risk factor for cardiovascular disease and mortality. We aimed to investigate the relationship between CHIP and post-transplant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37153874"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Architecture on Clinical Course and Prognosis in Patients With Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37153874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are caused by a somatic gain-of-function mutation in 1 of the 3 disease driver genes JAK2, MPL, or CALR. About half of the MPNs patients also carry additional somatic mutations that modify the clinical course. The order of acquisition of these gene mutations has been proposed to influence the phenotype and evolution of the disease. We studied 50 JAK2-V617F-positive MPN patients who carried at least 1 additional somatic mutation and determined the clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37156299"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential and outcomes after heart transplantation: A multi-center study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37156299/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiac allograft vasculopathy (CAV) is a leading cause of late graft failure and mortality after heart transplantation (HT). Sharing some features with atherosclerosis, CAV results in diffuse narrowing of the epicardial coronaries and microvasculature with consequent graft ischemia. Recently, clonal hematopoiesis of indeterminate potential (CHIP) has emerged as a risk factor for cardiovascular disease and mortality. We aimed to investigate the relationship between CHIP and post-transplant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37153874"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Architecture on Clinical Course and Prognosis in Patients With Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37153874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are caused by a somatic gain-of-function mutation in 1 of the 3 disease driver genes JAK2, MPL, or CALR. About half of the MPNs patients also carry additional somatic mutations that modify the clinical course. The order of acquisition of these gene mutations has been proposed to influence the phenotype and evolution of the disease. We studied 50 JAK2-V617F-positive MPN patients who carried at least 1 additional somatic mutation and determined the clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37156299"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential and outcomes after heart transplantation: A multi-center study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37156299/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiac allograft vasculopathy (CAV) is a leading cause of late graft failure and mortality after heart transplantation (HT). Sharing some features with atherosclerosis, CAV results in diffuse narrowing of the epicardial coronaries and microvasculature with consequent graft ischemia. Recently, clonal hematopoiesis of indeterminate potential (CHIP) has emerged as a risk factor for cardiovascular disease and mortality. We aimed to investigate the relationship between CHIP and post-transplant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37153874"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Architecture on Clinical Course and Prognosis in Patients With Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37153874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are caused by a somatic gain-of-function mutation in 1 of the 3 disease driver genes JAK2, MPL, or CALR. About half of the MPNs patients also carry additional somatic mutations that modify the clinical course. The order of acquisition of these gene mutations has been proposed to influence the phenotype and evolution of the disease. We studied 50 JAK2-V617F-positive MPN patients who carried at least 1 additional somatic mutation and determined the clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37156299"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential and outcomes after heart transplantation: A multi-center study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37156299/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiac allograft vasculopathy (CAV) is a leading cause of late graft failure and mortality after heart transplantation (HT). Sharing some features with atherosclerosis, CAV results in diffuse narrowing of the epicardial coronaries and microvasculature with consequent graft ischemia. Recently, clonal hematopoiesis of indeterminate potential (CHIP) has emerged as a risk factor for cardiovascular disease and mortality. We aimed to investigate the relationship between CHIP and post-transplant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37153874"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Architecture on Clinical Course and Prognosis in Patients With Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37153874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are caused by a somatic gain-of-function mutation in 1 of the 3 disease driver genes JAK2, MPL, or CALR. About half of the MPNs patients also carry additional somatic mutations that modify the clinical course. The order of acquisition of these gene mutations has been proposed to influence the phenotype and evolution of the disease. We studied 50 JAK2-V617F-positive MPN patients who carried at least 1 additional somatic mutation and determined the clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37156299"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential and outcomes after heart transplantation: A multi-center study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37156299/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiac allograft vasculopathy (CAV) is a leading cause of late graft failure and mortality after heart transplantation (HT). Sharing some features with atherosclerosis, CAV results in diffuse narrowing of the epicardial coronaries and microvasculature with consequent graft ischemia. Recently, clonal hematopoiesis of indeterminate potential (CHIP) has emerged as a risk factor for cardiovascular disease and mortality. We aimed to investigate the relationship between CHIP and post-transplant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37153874"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Architecture on Clinical Course and Prognosis in Patients With Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37153874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are caused by a somatic gain-of-function mutation in 1 of the 3 disease driver genes JAK2, MPL, or CALR. About half of the MPNs patients also carry additional somatic mutations that modify the clinical course. The order of acquisition of these gene mutations has been proposed to influence the phenotype and evolution of the disease. We studied 50 JAK2-V617F-positive MPN patients who carried at least 1 additional somatic mutation and determined the clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37156299"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential and outcomes after heart transplantation: A multi-center study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37156299/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230515205819&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiac allograft vasculopathy (CAV) is a leading cause of late graft failure and mortality after heart transplantation (HT). Sharing some features with atherosclerosis, CAV results in diffuse narrowing of the epicardial coronaries and microvasculature with consequent graft ischemia. Recently, clonal hematopoiesis of indeterminate potential (CHIP) has emerged as a risk factor for cardiovascular disease and mortality. We aimed to investigate the relationship between CHIP and post-transplant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37153874"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Architecture on Clinical Course and Prognosis in Patients With Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37153874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230515205819&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are caused by a somatic gain-of-function mutation in 1 of the 3 disease driver genes JAK2, MPL, or CALR. About half of the MPNs patients also carry additional somatic mutations that modify the clinical course. The order of acquisition of these gene mutations has been proposed to influence the phenotype and evolution of the disease. We studied 50 JAK2-V617F-positive MPN patients who carried at least 1 additional somatic mutation and determined the clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37156299"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential and outcomes after heart transplantation: A multi-center study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37156299/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiac allograft vasculopathy (CAV) is a leading cause of late graft failure and mortality after heart transplantation (HT). Sharing some features with atherosclerosis, CAV results in diffuse narrowing of the epicardial coronaries and microvasculature with consequent graft ischemia. Recently, clonal hematopoiesis of indeterminate potential (CHIP) has emerged as a risk factor for cardiovascular disease and mortality. We aimed to investigate the relationship between CHIP and post-transplant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37153874"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Architecture on Clinical Course and Prognosis in Patients With Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37153874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are caused by a somatic gain-of-function mutation in 1 of the 3 disease driver genes JAK2, MPL, or CALR. About half of the MPNs patients also carry additional somatic mutations that modify the clinical course. The order of acquisition of these gene mutations has been proposed to influence the phenotype and evolution of the disease. We studied 50 JAK2-V617F-positive MPN patients who carried at least 1 additional somatic mutation and determined the clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37156299"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential and outcomes after heart transplantation: A multi-center study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37156299/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiac allograft vasculopathy (CAV) is a leading cause of late graft failure and mortality after heart transplantation (HT). Sharing some features with atherosclerosis, CAV results in diffuse narrowing of the epicardial coronaries and microvasculature with consequent graft ischemia. Recently, clonal hematopoiesis of indeterminate potential (CHIP) has emerged as a risk factor for cardiovascular disease and mortality. We aimed to investigate the relationship between CHIP and post-transplant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37153874"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Architecture on Clinical Course and Prognosis in Patients With Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37153874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are caused by a somatic gain-of-function mutation in 1 of the 3 disease driver genes JAK2, MPL, or CALR. About half of the MPNs patients also carry additional somatic mutations that modify the clinical course. The order of acquisition of these gene mutations has been proposed to influence the phenotype and evolution of the disease. We studied 50 JAK2-V617F-positive MPN patients who carried at least 1 additional somatic mutation and determined the clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37156299"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential and outcomes after heart transplantation: A multi-center study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37156299/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiac allograft vasculopathy (CAV) is a leading cause of late graft failure and mortality after heart transplantation (HT). Sharing some features with atherosclerosis, CAV results in diffuse narrowing of the epicardial coronaries and microvasculature with consequent graft ischemia. Recently, clonal hematopoiesis of indeterminate potential (CHIP) has emerged as a risk factor for cardiovascular disease and mortality. We aimed to investigate the relationship between CHIP and post-transplant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37153874"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Architecture on Clinical Course and Prognosis in Patients With Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37153874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are caused by a somatic gain-of-function mutation in 1 of the 3 disease driver genes JAK2, MPL, or CALR. About half of the MPNs patients also carry additional somatic mutations that modify the clinical course. The order of acquisition of these gene mutations has been proposed to influence the phenotype and evolution of the disease. We studied 50 JAK2-V617F-positive MPN patients who carried at least 1 additional somatic mutation and determined the clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37156299"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential and outcomes after heart transplantation: A multi-center study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37156299/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230511205726&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiac allograft vasculopathy (CAV) is a leading cause of late graft failure and mortality after heart transplantation (HT). Sharing some features with atherosclerosis, CAV results in diffuse narrowing of the epicardial coronaries and microvasculature with consequent graft ischemia. Recently, clonal hematopoiesis of indeterminate potential (CHIP) has emerged as a risk factor for cardiovascular disease and mortality. We aimed to investigate the relationship between CHIP and post-transplant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37153874"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Architecture on Clinical Course and Prognosis in Patients With Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37153874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230511205726&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are caused by a somatic gain-of-function mutation in 1 of the 3 disease driver genes JAK2, MPL, or CALR. About half of the MPNs patients also carry additional somatic mutations that modify the clinical course. The order of acquisition of these gene mutations has been proposed to influence the phenotype and evolution of the disease. We studied 50 JAK2-V617F-positive MPN patients who carried at least 1 additional somatic mutation and determined the clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37156299"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential and outcomes after heart transplantation: A multi-center study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37156299/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230510205718&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiac allograft vasculopathy (CAV) is a leading cause of late graft failure and mortality after heart transplantation (HT). Sharing some features with atherosclerosis, CAV results in diffuse narrowing of the epicardial coronaries and microvasculature with consequent graft ischemia. Recently, clonal hematopoiesis of indeterminate potential (CHIP) has emerged as a risk factor for cardiovascular disease and mortality. We aimed to investigate the relationship between CHIP and post-transplant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37153874"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Architecture on Clinical Course and Prognosis in Patients With Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37153874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230510205718&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are caused by a somatic gain-of-function mutation in 1 of the 3 disease driver genes JAK2, MPL, or CALR. About half of the MPNs patients also carry additional somatic mutations that modify the clinical course. The order of acquisition of these gene mutations has been proposed to influence the phenotype and evolution of the disease. We studied 50 JAK2-V617F-positive MPN patients who carried at least 1 additional somatic mutation and determined the clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37156299"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential and outcomes after heart transplantation: A multi-center study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37156299/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230509205648&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiac allograft vasculopathy (CAV) is a leading cause of late graft failure and mortality after heart transplantation (HT). Sharing some features with atherosclerosis, CAV results in diffuse narrowing of the epicardial coronaries and microvasculature with consequent graft ischemia. Recently, clonal hematopoiesis of indeterminate potential (CHIP) has emerged as a risk factor for cardiovascular disease and mortality. We aimed to investigate the relationship between CHIP and post-transplant...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37153874"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Architecture on Clinical Course and Prognosis in Patients With Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37153874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230509205648&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are caused by a somatic gain-of-function mutation in 1 of the 3 disease driver genes JAK2, MPL, or CALR. About half of the MPNs patients also carry additional somatic mutations that modify the clinical course. The order of acquisition of these gene mutations has been proposed to influence the phenotype and evolution of the disease. We studied 50 JAK2-V617F-positive MPN patients who carried at least 1 additional somatic mutation and determined the clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37153874"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Architecture on Clinical Course and Prognosis in Patients With Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37153874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230508205817&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are caused by a somatic gain-of-function mutation in 1 of the 3 disease driver genes JAK2, MPL, or CALR. About half of the MPNs patients also carry additional somatic mutations that modify the clinical course. The order of acquisition of these gene mutations has been proposed to influence the phenotype and evolution of the disease. We studied 50 JAK2-V617F-positive MPN patients who carried at least 1 additional somatic mutation and determined the clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37155154"
                        open
                      >
                        <summary class="article-expander__title">FAM46C-mediated tumor heterogeneity predicts extramedullary metastasis and poorer survival in multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37155154/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers originate from a single cell according to Nowell&#x27;s theory of clonal evolution. The enrichment of the most aggressive clones has been developed and the heterogeneity arises for genomic instability and environmental selection. Multiple myeloma (MM) is a multiple relapse plasma cell cancer generated from bone marrow. Although there were accumulating researches in multiple myeloma pathogenesis, the heterogeneity remains poorly understood. The participants enrolled in this study were 4 EMP+...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37153874"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Architecture on Clinical Course and Prognosis in Patients With Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37153874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are caused by a somatic gain-of-function mutation in 1 of the 3 disease driver genes JAK2, MPL, or CALR. About half of the MPNs patients also carry additional somatic mutations that modify the clinical course. The order of acquisition of these gene mutations has been proposed to influence the phenotype and evolution of the disease. We studied 50 JAK2-V617F-positive MPN patients who carried at least 1 additional somatic mutation and determined the clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37155154"
                        open
                      >
                        <summary class="article-expander__title">FAM46C-mediated tumor heterogeneity predicts extramedullary metastasis and poorer survival in multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37155154/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers originate from a single cell according to Nowell&#x27;s theory of clonal evolution. The enrichment of the most aggressive clones has been developed and the heterogeneity arises for genomic instability and environmental selection. Multiple myeloma (MM) is a multiple relapse plasma cell cancer generated from bone marrow. Although there were accumulating researches in multiple myeloma pathogenesis, the heterogeneity remains poorly understood. The participants enrolled in this study were 4 EMP+...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37153874"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Architecture on Clinical Course and Prognosis in Patients With Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37153874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are caused by a somatic gain-of-function mutation in 1 of the 3 disease driver genes JAK2, MPL, or CALR. About half of the MPNs patients also carry additional somatic mutations that modify the clinical course. The order of acquisition of these gene mutations has been proposed to influence the phenotype and evolution of the disease. We studied 50 JAK2-V617F-positive MPN patients who carried at least 1 additional somatic mutation and determined the clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37155154"
                        open
                      >
                        <summary class="article-expander__title">FAM46C-mediated tumor heterogeneity predicts extramedullary metastasis and poorer survival in multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37155154/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers originate from a single cell according to Nowell&#x27;s theory of clonal evolution. The enrichment of the most aggressive clones has been developed and the heterogeneity arises for genomic instability and environmental selection. Multiple myeloma (MM) is a multiple relapse plasma cell cancer generated from bone marrow. Although there were accumulating researches in multiple myeloma pathogenesis, the heterogeneity remains poorly understood. The participants enrolled in this study were 4 EMP+...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37153874"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Architecture on Clinical Course and Prognosis in Patients With Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37153874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are caused by a somatic gain-of-function mutation in 1 of the 3 disease driver genes JAK2, MPL, or CALR. About half of the MPNs patients also carry additional somatic mutations that modify the clinical course. The order of acquisition of these gene mutations has been proposed to influence the phenotype and evolution of the disease. We studied 50 JAK2-V617F-positive MPN patients who carried at least 1 additional somatic mutation and determined the clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37155154"
                        open
                      >
                        <summary class="article-expander__title">FAM46C-mediated tumor heterogeneity predicts extramedullary metastasis and poorer survival in multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37155154/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230515205818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers originate from a single cell according to Nowell&#x27;s theory of clonal evolution. The enrichment of the most aggressive clones has been developed and the heterogeneity arises for genomic instability and environmental selection. Multiple myeloma (MM) is a multiple relapse plasma cell cancer generated from bone marrow. Although there were accumulating researches in multiple myeloma pathogenesis, the heterogeneity remains poorly understood. The participants enrolled in this study were 4 EMP+...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37153874"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Architecture on Clinical Course and Prognosis in Patients With Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37153874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230515205818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are caused by a somatic gain-of-function mutation in 1 of the 3 disease driver genes JAK2, MPL, or CALR. About half of the MPNs patients also carry additional somatic mutations that modify the clinical course. The order of acquisition of these gene mutations has been proposed to influence the phenotype and evolution of the disease. We studied 50 JAK2-V617F-positive MPN patients who carried at least 1 additional somatic mutation and determined the clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37155154"
                        open
                      >
                        <summary class="article-expander__title">FAM46C-mediated tumor heterogeneity predicts extramedullary metastasis and poorer survival in multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37155154/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers originate from a single cell according to Nowell&#x27;s theory of clonal evolution. The enrichment of the most aggressive clones has been developed and the heterogeneity arises for genomic instability and environmental selection. Multiple myeloma (MM) is a multiple relapse plasma cell cancer generated from bone marrow. Although there were accumulating researches in multiple myeloma pathogenesis, the heterogeneity remains poorly understood. The participants enrolled in this study were 4 EMP+...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37153874"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Architecture on Clinical Course and Prognosis in Patients With Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37153874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are caused by a somatic gain-of-function mutation in 1 of the 3 disease driver genes JAK2, MPL, or CALR. About half of the MPNs patients also carry additional somatic mutations that modify the clinical course. The order of acquisition of these gene mutations has been proposed to influence the phenotype and evolution of the disease. We studied 50 JAK2-V617F-positive MPN patients who carried at least 1 additional somatic mutation and determined the clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37155154"
                        open
                      >
                        <summary class="article-expander__title">FAM46C-mediated tumor heterogeneity predicts extramedullary metastasis and poorer survival in multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37155154/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers originate from a single cell according to Nowell&#x27;s theory of clonal evolution. The enrichment of the most aggressive clones has been developed and the heterogeneity arises for genomic instability and environmental selection. Multiple myeloma (MM) is a multiple relapse plasma cell cancer generated from bone marrow. Although there were accumulating researches in multiple myeloma pathogenesis, the heterogeneity remains poorly understood. The participants enrolled in this study were 4 EMP+...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37153874"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Architecture on Clinical Course and Prognosis in Patients With Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37153874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are caused by a somatic gain-of-function mutation in 1 of the 3 disease driver genes JAK2, MPL, or CALR. About half of the MPNs patients also carry additional somatic mutations that modify the clinical course. The order of acquisition of these gene mutations has been proposed to influence the phenotype and evolution of the disease. We studied 50 JAK2-V617F-positive MPN patients who carried at least 1 additional somatic mutation and determined the clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37155154"
                        open
                      >
                        <summary class="article-expander__title">FAM46C-mediated tumor heterogeneity predicts extramedullary metastasis and poorer survival in multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37155154/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers originate from a single cell according to Nowell&#x27;s theory of clonal evolution. The enrichment of the most aggressive clones has been developed and the heterogeneity arises for genomic instability and environmental selection. Multiple myeloma (MM) is a multiple relapse plasma cell cancer generated from bone marrow. Although there were accumulating researches in multiple myeloma pathogenesis, the heterogeneity remains poorly understood. The participants enrolled in this study were 4 EMP+...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37153874"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Architecture on Clinical Course and Prognosis in Patients With Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37153874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are caused by a somatic gain-of-function mutation in 1 of the 3 disease driver genes JAK2, MPL, or CALR. About half of the MPNs patients also carry additional somatic mutations that modify the clinical course. The order of acquisition of these gene mutations has been proposed to influence the phenotype and evolution of the disease. We studied 50 JAK2-V617F-positive MPN patients who carried at least 1 additional somatic mutation and determined the clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37155154"
                        open
                      >
                        <summary class="article-expander__title">FAM46C-mediated tumor heterogeneity predicts extramedullary metastasis and poorer survival in multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37155154/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230511205726&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers originate from a single cell according to Nowell&#x27;s theory of clonal evolution. The enrichment of the most aggressive clones has been developed and the heterogeneity arises for genomic instability and environmental selection. Multiple myeloma (MM) is a multiple relapse plasma cell cancer generated from bone marrow. Although there were accumulating researches in multiple myeloma pathogenesis, the heterogeneity remains poorly understood. The participants enrolled in this study were 4 EMP+...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37153874"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Architecture on Clinical Course and Prognosis in Patients With Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37153874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230511205726&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are caused by a somatic gain-of-function mutation in 1 of the 3 disease driver genes JAK2, MPL, or CALR. About half of the MPNs patients also carry additional somatic mutations that modify the clinical course. The order of acquisition of these gene mutations has been proposed to influence the phenotype and evolution of the disease. We studied 50 JAK2-V617F-positive MPN patients who carried at least 1 additional somatic mutation and determined the clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37155154"
                        open
                      >
                        <summary class="article-expander__title">FAM46C-mediated tumor heterogeneity predicts extramedullary metastasis and poorer survival in multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37155154/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230510205718&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers originate from a single cell according to Nowell&#x27;s theory of clonal evolution. The enrichment of the most aggressive clones has been developed and the heterogeneity arises for genomic instability and environmental selection. Multiple myeloma (MM) is a multiple relapse plasma cell cancer generated from bone marrow. Although there were accumulating researches in multiple myeloma pathogenesis, the heterogeneity remains poorly understood. The participants enrolled in this study were 4 EMP+...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37153874"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Architecture on Clinical Course and Prognosis in Patients With Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37153874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230510205718&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are caused by a somatic gain-of-function mutation in 1 of the 3 disease driver genes JAK2, MPL, or CALR. About half of the MPNs patients also carry additional somatic mutations that modify the clinical course. The order of acquisition of these gene mutations has been proposed to influence the phenotype and evolution of the disease. We studied 50 JAK2-V617F-positive MPN patients who carried at least 1 additional somatic mutation and determined the clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37155154"
                        open
                      >
                        <summary class="article-expander__title">FAM46C-mediated tumor heterogeneity predicts extramedullary metastasis and poorer survival in multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37155154/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230509205648&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers originate from a single cell according to Nowell&#x27;s theory of clonal evolution. The enrichment of the most aggressive clones has been developed and the heterogeneity arises for genomic instability and environmental selection. Multiple myeloma (MM) is a multiple relapse plasma cell cancer generated from bone marrow. Although there were accumulating researches in multiple myeloma pathogenesis, the heterogeneity remains poorly understood. The participants enrolled in this study were 4 EMP+...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37153874"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Architecture on Clinical Course and Prognosis in Patients With Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37153874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230509205648&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are caused by a somatic gain-of-function mutation in 1 of the 3 disease driver genes JAK2, MPL, or CALR. About half of the MPNs patients also carry additional somatic mutations that modify the clinical course. The order of acquisition of these gene mutations has been proposed to influence the phenotype and evolution of the disease. We studied 50 JAK2-V617F-positive MPN patients who carried at least 1 additional somatic mutation and determined the clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37155154"
                        open
                      >
                        <summary class="article-expander__title">FAM46C-mediated tumor heterogeneity predicts extramedullary metastasis and poorer survival in multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37155154/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230508205817&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers originate from a single cell according to Nowell&#x27;s theory of clonal evolution. The enrichment of the most aggressive clones has been developed and the heterogeneity arises for genomic instability and environmental selection. Multiple myeloma (MM) is a multiple relapse plasma cell cancer generated from bone marrow. Although there were accumulating researches in multiple myeloma pathogenesis, the heterogeneity remains poorly understood. The participants enrolled in this study were 4 EMP+...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37153874"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Architecture on Clinical Course and Prognosis in Patients With Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37153874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230508205817&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are caused by a somatic gain-of-function mutation in 1 of the 3 disease driver genes JAK2, MPL, or CALR. About half of the MPNs patients also carry additional somatic mutations that modify the clinical course. The order of acquisition of these gene mutations has been proposed to influence the phenotype and evolution of the disease. We studied 50 JAK2-V617F-positive MPN patients who carried at least 1 additional somatic mutation and determined the clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-05-07 datetime="2023-05-07T10:00:00.000Z">2023-05-07</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-05-07 datetime="2023-05-07T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37150255"
                        open
                      >
                        <summary class="article-expander__title">Mosaic chromosomal alterations is associated with increased lung cancer risk: insight from the INTEGRAL-ILCCO cohort analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37150255/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mosaic chromosomal alterations (mCAs) detected in white blood cells represent a type of clonal hematopoiesis (CH) that is understudied compared to CH-related somatic mutations. A few recent studies indicated their potential link with non-hematological cancers, especially lung cancer. In this study, we investigated the association between mCAs and lung cancer using the high-density genotyping data from the OncoArray study of INTEGRAL-ILCCO, the largest single genetic study of lung cancer with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37150255"
                        open
                      >
                        <summary class="article-expander__title">Mosaic chromosomal alterations is associated with increased lung cancer risk: insight from the INTEGRAL-ILCCO cohort analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37150255/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mosaic chromosomal alterations (mCAs) detected in white blood cells represent a type of clonal hematopoiesis (CH) that is understudied compared to CH-related somatic mutations. A few recent studies indicated their potential link with non-hematological cancers, especially lung cancer. In this study, we investigated the association between mCAs and lung cancer using the high-density genotyping data from the OncoArray study of INTEGRAL-ILCCO, the largest single genetic study of lung cancer with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37150255"
                        open
                      >
                        <summary class="article-expander__title">Mosaic chromosomal alterations is associated with increased lung cancer risk: insight from the INTEGRAL-ILCCO cohort analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37150255/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230521210055&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mosaic chromosomal alterations (mCAs) detected in white blood cells represent a type of clonal hematopoiesis (CH) that is understudied compared to CH-related somatic mutations. A few recent studies indicated their potential link with non-hematological cancers, especially lung cancer. In this study, we investigated the association between mCAs and lung cancer using the high-density genotyping data from the OncoArray study of INTEGRAL-ILCCO, the largest single genetic study of lung cancer with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37150255"
                        open
                      >
                        <summary class="article-expander__title">Mosaic chromosomal alterations is associated with increased lung cancer risk: insight from the INTEGRAL-ILCCO cohort analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37150255/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mosaic chromosomal alterations (mCAs) detected in white blood cells represent a type of clonal hematopoiesis (CH) that is understudied compared to CH-related somatic mutations. A few recent studies indicated their potential link with non-hematological cancers, especially lung cancer. In this study, we investigated the association between mCAs and lung cancer using the high-density genotyping data from the OncoArray study of INTEGRAL-ILCCO, the largest single genetic study of lung cancer with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37150255"
                        open
                      >
                        <summary class="article-expander__title">Mosaic chromosomal alterations is associated with increased lung cancer risk: insight from the INTEGRAL-ILCCO cohort analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37150255/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mosaic chromosomal alterations (mCAs) detected in white blood cells represent a type of clonal hematopoiesis (CH) that is understudied compared to CH-related somatic mutations. A few recent studies indicated their potential link with non-hematological cancers, especially lung cancer. In this study, we investigated the association between mCAs and lung cancer using the high-density genotyping data from the OncoArray study of INTEGRAL-ILCCO, the largest single genetic study of lung cancer with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37150255"
                        open
                      >
                        <summary class="article-expander__title">Mosaic chromosomal alterations is associated with increased lung cancer risk: insight from the INTEGRAL-ILCCO cohort analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37150255/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mosaic chromosomal alterations (mCAs) detected in white blood cells represent a type of clonal hematopoiesis (CH) that is understudied compared to CH-related somatic mutations. A few recent studies indicated their potential link with non-hematological cancers, especially lung cancer. In this study, we investigated the association between mCAs and lung cancer using the high-density genotyping data from the OncoArray study of INTEGRAL-ILCCO, the largest single genetic study of lung cancer with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37150255"
                        open
                      >
                        <summary class="article-expander__title">Mosaic chromosomal alterations is associated with increased lung cancer risk: insight from the INTEGRAL-ILCCO cohort analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37150255/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mosaic chromosomal alterations (mCAs) detected in white blood cells represent a type of clonal hematopoiesis (CH) that is understudied compared to CH-related somatic mutations. A few recent studies indicated their potential link with non-hematological cancers, especially lung cancer. In this study, we investigated the association between mCAs and lung cancer using the high-density genotyping data from the OncoArray study of INTEGRAL-ILCCO, the largest single genetic study of lung cancer with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37150255"
                        open
                      >
                        <summary class="article-expander__title">Mosaic chromosomal alterations is associated with increased lung cancer risk: insight from the INTEGRAL-ILCCO cohort analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37150255/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mosaic chromosomal alterations (mCAs) detected in white blood cells represent a type of clonal hematopoiesis (CH) that is understudied compared to CH-related somatic mutations. A few recent studies indicated their potential link with non-hematological cancers, especially lung cancer. In this study, we investigated the association between mCAs and lung cancer using the high-density genotyping data from the OncoArray study of INTEGRAL-ILCCO, the largest single genetic study of lung cancer with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37150255"
                        open
                      >
                        <summary class="article-expander__title">Mosaic chromosomal alterations is associated with increased lung cancer risk: insight from the INTEGRAL-ILCCO cohort analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37150255/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230515205819&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mosaic chromosomal alterations (mCAs) detected in white blood cells represent a type of clonal hematopoiesis (CH) that is understudied compared to CH-related somatic mutations. A few recent studies indicated their potential link with non-hematological cancers, especially lung cancer. In this study, we investigated the association between mCAs and lung cancer using the high-density genotyping data from the OncoArray study of INTEGRAL-ILCCO, the largest single genetic study of lung cancer with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37150255"
                        open
                      >
                        <summary class="article-expander__title">Mosaic chromosomal alterations is associated with increased lung cancer risk: insight from the INTEGRAL-ILCCO cohort analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37150255/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mosaic chromosomal alterations (mCAs) detected in white blood cells represent a type of clonal hematopoiesis (CH) that is understudied compared to CH-related somatic mutations. A few recent studies indicated their potential link with non-hematological cancers, especially lung cancer. In this study, we investigated the association between mCAs and lung cancer using the high-density genotyping data from the OncoArray study of INTEGRAL-ILCCO, the largest single genetic study of lung cancer with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37150255"
                        open
                      >
                        <summary class="article-expander__title">Mosaic chromosomal alterations is associated with increased lung cancer risk: insight from the INTEGRAL-ILCCO cohort analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37150255/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mosaic chromosomal alterations (mCAs) detected in white blood cells represent a type of clonal hematopoiesis (CH) that is understudied compared to CH-related somatic mutations. A few recent studies indicated their potential link with non-hematological cancers, especially lung cancer. In this study, we investigated the association between mCAs and lung cancer using the high-density genotyping data from the OncoArray study of INTEGRAL-ILCCO, the largest single genetic study of lung cancer with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37150255"
                        open
                      >
                        <summary class="article-expander__title">Mosaic chromosomal alterations is associated with increased lung cancer risk: insight from the INTEGRAL-ILCCO cohort analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37150255/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mosaic chromosomal alterations (mCAs) detected in white blood cells represent a type of clonal hematopoiesis (CH) that is understudied compared to CH-related somatic mutations. A few recent studies indicated their potential link with non-hematological cancers, especially lung cancer. In this study, we investigated the association between mCAs and lung cancer using the high-density genotyping data from the OncoArray study of INTEGRAL-ILCCO, the largest single genetic study of lung cancer with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37150255"
                        open
                      >
                        <summary class="article-expander__title">Mosaic chromosomal alterations is associated with increased lung cancer risk: insight from the INTEGRAL-ILCCO cohort analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37150255/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230511205726&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mosaic chromosomal alterations (mCAs) detected in white blood cells represent a type of clonal hematopoiesis (CH) that is understudied compared to CH-related somatic mutations. A few recent studies indicated their potential link with non-hematological cancers, especially lung cancer. In this study, we investigated the association between mCAs and lung cancer using the high-density genotyping data from the OncoArray study of INTEGRAL-ILCCO, the largest single genetic study of lung cancer with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37150255"
                        open
                      >
                        <summary class="article-expander__title">Mosaic chromosomal alterations is associated with increased lung cancer risk: insight from the INTEGRAL-ILCCO cohort analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37150255/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230510205718&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mosaic chromosomal alterations (mCAs) detected in white blood cells represent a type of clonal hematopoiesis (CH) that is understudied compared to CH-related somatic mutations. A few recent studies indicated their potential link with non-hematological cancers, especially lung cancer. In this study, we investigated the association between mCAs and lung cancer using the high-density genotyping data from the OncoArray study of INTEGRAL-ILCCO, the largest single genetic study of lung cancer with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37150255"
                        open
                      >
                        <summary class="article-expander__title">Mosaic chromosomal alterations is associated with increased lung cancer risk: insight from the INTEGRAL-ILCCO cohort analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37150255/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230509205648&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mosaic chromosomal alterations (mCAs) detected in white blood cells represent a type of clonal hematopoiesis (CH) that is understudied compared to CH-related somatic mutations. A few recent studies indicated their potential link with non-hematological cancers, especially lung cancer. In this study, we investigated the association between mCAs and lung cancer using the high-density genotyping data from the OncoArray study of INTEGRAL-ILCCO, the largest single genetic study of lung cancer with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37150255"
                        open
                      >
                        <summary class="article-expander__title">Mosaic chromosomal alterations is associated with increased lung cancer risk: insight from the INTEGRAL-ILCCO cohort analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37150255/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230508205817&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mosaic chromosomal alterations (mCAs) detected in white blood cells represent a type of clonal hematopoiesis (CH) that is understudied compared to CH-related somatic mutations. A few recent studies indicated their potential link with non-hematological cancers, especially lung cancer. In this study, we investigated the association between mCAs and lung cancer using the high-density genotyping data from the OncoArray study of INTEGRAL-ILCCO, the largest single genetic study of lung cancer with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37149862"
                        open
                      >
                        <summary class="article-expander__title">Glioblastoma cell fate is differentially regulated by the microenvironments of the tumor bulk and infiltrative margin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37149862/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastoma (GBM) recurrence originates from invasive margin cells that escape surgical debulking, but to what extent these cells resemble their bulk counterparts remains unclear. Here, we generated three immunocompetent somatic GBM mouse models, driven by subtype-associated mutations, to compare matched bulk and margin cells. We find that, regardless of mutations, tumors converge on common sets of neural-like cellular states. However, bulk and margin have distinct biology. Injury-like programs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37149862"
                        open
                      >
                        <summary class="article-expander__title">Glioblastoma cell fate is differentially regulated by the microenvironments of the tumor bulk and infiltrative margin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37149862/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastoma (GBM) recurrence originates from invasive margin cells that escape surgical debulking, but to what extent these cells resemble their bulk counterparts remains unclear. Here, we generated three immunocompetent somatic GBM mouse models, driven by subtype-associated mutations, to compare matched bulk and margin cells. We find that, regardless of mutations, tumors converge on common sets of neural-like cellular states. However, bulk and margin have distinct biology. Injury-like programs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37149862"
                        open
                      >
                        <summary class="article-expander__title">Glioblastoma cell fate is differentially regulated by the microenvironments of the tumor bulk and infiltrative margin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37149862/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230515205818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastoma (GBM) recurrence originates from invasive margin cells that escape surgical debulking, but to what extent these cells resemble their bulk counterparts remains unclear. Here, we generated three immunocompetent somatic GBM mouse models, driven by subtype-associated mutations, to compare matched bulk and margin cells. We find that, regardless of mutations, tumors converge on common sets of neural-like cellular states. However, bulk and margin have distinct biology. Injury-like programs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37149862"
                        open
                      >
                        <summary class="article-expander__title">Glioblastoma cell fate is differentially regulated by the microenvironments of the tumor bulk and infiltrative margin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37149862/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastoma (GBM) recurrence originates from invasive margin cells that escape surgical debulking, but to what extent these cells resemble their bulk counterparts remains unclear. Here, we generated three immunocompetent somatic GBM mouse models, driven by subtype-associated mutations, to compare matched bulk and margin cells. We find that, regardless of mutations, tumors converge on common sets of neural-like cellular states. However, bulk and margin have distinct biology. Injury-like programs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37149862"
                        open
                      >
                        <summary class="article-expander__title">Glioblastoma cell fate is differentially regulated by the microenvironments of the tumor bulk and infiltrative margin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37149862/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastoma (GBM) recurrence originates from invasive margin cells that escape surgical debulking, but to what extent these cells resemble their bulk counterparts remains unclear. Here, we generated three immunocompetent somatic GBM mouse models, driven by subtype-associated mutations, to compare matched bulk and margin cells. We find that, regardless of mutations, tumors converge on common sets of neural-like cellular states. However, bulk and margin have distinct biology. Injury-like programs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37149862"
                        open
                      >
                        <summary class="article-expander__title">Glioblastoma cell fate is differentially regulated by the microenvironments of the tumor bulk and infiltrative margin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37149862/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastoma (GBM) recurrence originates from invasive margin cells that escape surgical debulking, but to what extent these cells resemble their bulk counterparts remains unclear. Here, we generated three immunocompetent somatic GBM mouse models, driven by subtype-associated mutations, to compare matched bulk and margin cells. We find that, regardless of mutations, tumors converge on common sets of neural-like cellular states. However, bulk and margin have distinct biology. Injury-like programs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37149862"
                        open
                      >
                        <summary class="article-expander__title">Glioblastoma cell fate is differentially regulated by the microenvironments of the tumor bulk and infiltrative margin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37149862/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230511205726&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastoma (GBM) recurrence originates from invasive margin cells that escape surgical debulking, but to what extent these cells resemble their bulk counterparts remains unclear. Here, we generated three immunocompetent somatic GBM mouse models, driven by subtype-associated mutations, to compare matched bulk and margin cells. We find that, regardless of mutations, tumors converge on common sets of neural-like cellular states. However, bulk and margin have distinct biology. Injury-like programs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37149862"
                        open
                      >
                        <summary class="article-expander__title">Glioblastoma cell fate is differentially regulated by the microenvironments of the tumor bulk and infiltrative margin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37149862/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230510205718&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastoma (GBM) recurrence originates from invasive margin cells that escape surgical debulking, but to what extent these cells resemble their bulk counterparts remains unclear. Here, we generated three immunocompetent somatic GBM mouse models, driven by subtype-associated mutations, to compare matched bulk and margin cells. We find that, regardless of mutations, tumors converge on common sets of neural-like cellular states. However, bulk and margin have distinct biology. Injury-like programs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37149862"
                        open
                      >
                        <summary class="article-expander__title">Glioblastoma cell fate is differentially regulated by the microenvironments of the tumor bulk and infiltrative margin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37149862/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230509205648&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastoma (GBM) recurrence originates from invasive margin cells that escape surgical debulking, but to what extent these cells resemble their bulk counterparts remains unclear. Here, we generated three immunocompetent somatic GBM mouse models, driven by subtype-associated mutations, to compare matched bulk and margin cells. We find that, regardless of mutations, tumors converge on common sets of neural-like cellular states. However, bulk and margin have distinct biology. Injury-like programs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37149862"
                        open
                      >
                        <summary class="article-expander__title">Glioblastoma cell fate is differentially regulated by the microenvironments of the tumor bulk and infiltrative margin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37149862/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230508205817&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastoma (GBM) recurrence originates from invasive margin cells that escape surgical debulking, but to what extent these cells resemble their bulk counterparts remains unclear. Here, we generated three immunocompetent somatic GBM mouse models, driven by subtype-associated mutations, to compare matched bulk and margin cells. We find that, regardless of mutations, tumors converge on common sets of neural-like cellular states. However, bulk and margin have distinct biology. Injury-like programs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37149862"
                        open
                      >
                        <summary class="article-expander__title">Glioblastoma cell fate is differentially regulated by the microenvironments of the tumor bulk and infiltrative margin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37149862/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230507205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastoma (GBM) recurrence originates from invasive margin cells that escape surgical debulking, but to what extent these cells resemble their bulk counterparts remains unclear. Here, we generated three immunocompetent somatic GBM mouse models, driven by subtype-associated mutations, to compare matched bulk and margin cells. We find that, regardless of mutations, tumors converge on common sets of neural-like cellular states. However, bulk and margin have distinct biology. Injury-like programs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-05-05 datetime="2023-05-05T10:00:00.000Z">2023-05-05</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-05-05 datetime="2023-05-05T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146604"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary landscape of clonal hematopoiesis in 3,359 individuals from the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146604/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230523205858&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Knowledge about evolution of clonal hematopoiesis, which may drive malignant progression, is crucial for clinical decision-making. We investigated the landscape of clonal evolution by error-corrected sequencing on 7,045 sequential samples from 3,359 individuals in the prospective population-based Lifelines cohort, with a special focus on cytosis and cytopenia. Spliceosome (SRSF2/U2AF1/SF3B1) and JAK2 mutated clones show highest growth rates over a median 3.6-year period, while clone sizes for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146604"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary landscape of clonal hematopoiesis in 3,359 individuals from the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146604/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230522205732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Knowledge about evolution of clonal hematopoiesis, which may drive malignant progression, is crucial for clinical decision-making. We investigated the landscape of clonal evolution by error-corrected sequencing on 7,045 sequential samples from 3,359 individuals in the prospective population-based Lifelines cohort, with a special focus on cytosis and cytopenia. Spliceosome (SRSF2/U2AF1/SF3B1) and JAK2 mutated clones show highest growth rates over a median 3.6-year period, while clone sizes for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146604"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary landscape of clonal hematopoiesis in 3,359 individuals from the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146604/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230521210055&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Knowledge about evolution of clonal hematopoiesis, which may drive malignant progression, is crucial for clinical decision-making. We investigated the landscape of clonal evolution by error-corrected sequencing on 7,045 sequential samples from 3,359 individuals in the prospective population-based Lifelines cohort, with a special focus on cytosis and cytopenia. Spliceosome (SRSF2/U2AF1/SF3B1) and JAK2 mutated clones show highest growth rates over a median 3.6-year period, while clone sizes for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146604"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary landscape of clonal hematopoiesis in 3,359 individuals from the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146604/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230520210114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Knowledge about evolution of clonal hematopoiesis, which may drive malignant progression, is crucial for clinical decision-making. We investigated the landscape of clonal evolution by error-corrected sequencing on 7,045 sequential samples from 3,359 individuals in the prospective population-based Lifelines cohort, with a special focus on cytosis and cytopenia. Spliceosome (SRSF2/U2AF1/SF3B1) and JAK2 mutated clones show highest growth rates over a median 3.6-year period, while clone sizes for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146604"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary landscape of clonal hematopoiesis in 3,359 individuals from the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146604/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230519205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Knowledge about evolution of clonal hematopoiesis, which may drive malignant progression, is crucial for clinical decision-making. We investigated the landscape of clonal evolution by error-corrected sequencing on 7,045 sequential samples from 3,359 individuals in the prospective population-based Lifelines cohort, with a special focus on cytosis and cytopenia. Spliceosome (SRSF2/U2AF1/SF3B1) and JAK2 mutated clones show highest growth rates over a median 3.6-year period, while clone sizes for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146604"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary landscape of clonal hematopoiesis in 3,359 individuals from the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146604/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230518205810&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Knowledge about evolution of clonal hematopoiesis, which may drive malignant progression, is crucial for clinical decision-making. We investigated the landscape of clonal evolution by error-corrected sequencing on 7,045 sequential samples from 3,359 individuals in the prospective population-based Lifelines cohort, with a special focus on cytosis and cytopenia. Spliceosome (SRSF2/U2AF1/SF3B1) and JAK2 mutated clones show highest growth rates over a median 3.6-year period, while clone sizes for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146604"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary landscape of clonal hematopoiesis in 3,359 individuals from the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146604/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Knowledge about evolution of clonal hematopoiesis, which may drive malignant progression, is crucial for clinical decision-making. We investigated the landscape of clonal evolution by error-corrected sequencing on 7,045 sequential samples from 3,359 individuals in the prospective population-based Lifelines cohort, with a special focus on cytosis and cytopenia. Spliceosome (SRSF2/U2AF1/SF3B1) and JAK2 mutated clones show highest growth rates over a median 3.6-year period, while clone sizes for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146604"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary landscape of clonal hematopoiesis in 3,359 individuals from the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146604/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Knowledge about evolution of clonal hematopoiesis, which may drive malignant progression, is crucial for clinical decision-making. We investigated the landscape of clonal evolution by error-corrected sequencing on 7,045 sequential samples from 3,359 individuals in the prospective population-based Lifelines cohort, with a special focus on cytosis and cytopenia. Spliceosome (SRSF2/U2AF1/SF3B1) and JAK2 mutated clones show highest growth rates over a median 3.6-year period, while clone sizes for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146604"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary landscape of clonal hematopoiesis in 3,359 individuals from the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146604/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230515205819&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Knowledge about evolution of clonal hematopoiesis, which may drive malignant progression, is crucial for clinical decision-making. We investigated the landscape of clonal evolution by error-corrected sequencing on 7,045 sequential samples from 3,359 individuals in the prospective population-based Lifelines cohort, with a special focus on cytosis and cytopenia. Spliceosome (SRSF2/U2AF1/SF3B1) and JAK2 mutated clones show highest growth rates over a median 3.6-year period, while clone sizes for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146604"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary landscape of clonal hematopoiesis in 3,359 individuals from the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146604/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Knowledge about evolution of clonal hematopoiesis, which may drive malignant progression, is crucial for clinical decision-making. We investigated the landscape of clonal evolution by error-corrected sequencing on 7,045 sequential samples from 3,359 individuals in the prospective population-based Lifelines cohort, with a special focus on cytosis and cytopenia. Spliceosome (SRSF2/U2AF1/SF3B1) and JAK2 mutated clones show highest growth rates over a median 3.6-year period, while clone sizes for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146604"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary landscape of clonal hematopoiesis in 3,359 individuals from the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146604/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Knowledge about evolution of clonal hematopoiesis, which may drive malignant progression, is crucial for clinical decision-making. We investigated the landscape of clonal evolution by error-corrected sequencing on 7,045 sequential samples from 3,359 individuals in the prospective population-based Lifelines cohort, with a special focus on cytosis and cytopenia. Spliceosome (SRSF2/U2AF1/SF3B1) and JAK2 mutated clones show highest growth rates over a median 3.6-year period, while clone sizes for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146604"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary landscape of clonal hematopoiesis in 3,359 individuals from the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146604/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Knowledge about evolution of clonal hematopoiesis, which may drive malignant progression, is crucial for clinical decision-making. We investigated the landscape of clonal evolution by error-corrected sequencing on 7,045 sequential samples from 3,359 individuals in the prospective population-based Lifelines cohort, with a special focus on cytosis and cytopenia. Spliceosome (SRSF2/U2AF1/SF3B1) and JAK2 mutated clones show highest growth rates over a median 3.6-year period, while clone sizes for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146604"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary landscape of clonal hematopoiesis in 3,359 individuals from the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146604/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230511205726&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Knowledge about evolution of clonal hematopoiesis, which may drive malignant progression, is crucial for clinical decision-making. We investigated the landscape of clonal evolution by error-corrected sequencing on 7,045 sequential samples from 3,359 individuals in the prospective population-based Lifelines cohort, with a special focus on cytosis and cytopenia. Spliceosome (SRSF2/U2AF1/SF3B1) and JAK2 mutated clones show highest growth rates over a median 3.6-year period, while clone sizes for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146604"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary landscape of clonal hematopoiesis in 3,359 individuals from the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146604/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230510205718&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Knowledge about evolution of clonal hematopoiesis, which may drive malignant progression, is crucial for clinical decision-making. We investigated the landscape of clonal evolution by error-corrected sequencing on 7,045 sequential samples from 3,359 individuals in the prospective population-based Lifelines cohort, with a special focus on cytosis and cytopenia. Spliceosome (SRSF2/U2AF1/SF3B1) and JAK2 mutated clones show highest growth rates over a median 3.6-year period, while clone sizes for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146604"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary landscape of clonal hematopoiesis in 3,359 individuals from the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146604/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230509205648&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Knowledge about evolution of clonal hematopoiesis, which may drive malignant progression, is crucial for clinical decision-making. We investigated the landscape of clonal evolution by error-corrected sequencing on 7,045 sequential samples from 3,359 individuals in the prospective population-based Lifelines cohort, with a special focus on cytosis and cytopenia. Spliceosome (SRSF2/U2AF1/SF3B1) and JAK2 mutated clones show highest growth rates over a median 3.6-year period, while clone sizes for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146604"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary landscape of clonal hematopoiesis in 3,359 individuals from the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146604/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230508205817&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Knowledge about evolution of clonal hematopoiesis, which may drive malignant progression, is crucial for clinical decision-making. We investigated the landscape of clonal evolution by error-corrected sequencing on 7,045 sequential samples from 3,359 individuals in the prospective population-based Lifelines cohort, with a special focus on cytosis and cytopenia. Spliceosome (SRSF2/U2AF1/SF3B1) and JAK2 mutated clones show highest growth rates over a median 3.6-year period, while clone sizes for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146604"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary landscape of clonal hematopoiesis in 3,359 individuals from the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146604/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230507205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Knowledge about evolution of clonal hematopoiesis, which may drive malignant progression, is crucial for clinical decision-making. We investigated the landscape of clonal evolution by error-corrected sequencing on 7,045 sequential samples from 3,359 individuals in the prospective population-based Lifelines cohort, with a special focus on cytosis and cytopenia. Spliceosome (SRSF2/U2AF1/SF3B1) and JAK2 mutated clones show highest growth rates over a median 3.6-year period, while clone sizes for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146604"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary landscape of clonal hematopoiesis in 3,359 individuals from the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146604/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230506210046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Knowledge about evolution of clonal hematopoiesis, which may drive malignant progression, is crucial for clinical decision-making. We investigated the landscape of clonal evolution by error-corrected sequencing on 7,045 sequential samples from 3,359 individuals in the prospective population-based Lifelines cohort, with a special focus on cytosis and cytopenia. Spliceosome (SRSF2/U2AF1/SF3B1) and JAK2 mutated clones show highest growth rates over a median 3.6-year period, while clone sizes for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146586"
                        open
                      >
                        <summary class="article-expander__title">GTAC enables parallel genotyping of multiple genomic loci with chromatin accessibility profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Understanding clonal evolution and cancer development requires experimental approaches for characterizing the consequences of somatic mutations on gene regulation. However, no methods currently exist that efficiently link high-content chromatin accessibility with high-confidence genotyping in single cells. To address this, we developed Genotyping with the Assay for Transposase-Accessible Chromatin (GTAC), enabling accurate mutation detection at multiple amplified loci, coupled with robust...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146250"
                        open
                      >
                        <summary class="article-expander__title">Genetic events associated with venetoclax resistance in CLL identified by whole exome sequencing of patient samples</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146250/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Although BCL2 mutations are reported as later occurring events leading to venetoclax resistance, many other mechanisms of progression have been reported but remain poorly understood. Here we analyze longitudinal tumor samples from eleven patients with disease progression on venetoclax to characterize the clonal evolution of resistance. All patients tested showed increased in vitro resistance to venetoclax at their post-treatment timepoint. We found the previously described acquired BCL2-G101V...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37143651"
                        open
                      >
                        <summary class="article-expander__title">Insights into IGH clonal evolution in BCP-ALL: frequency, mechanisms, associations, and diagnostic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37143651/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The malignant transformation leading to a maturation arrest in B-cell precursor acute lymphoblastic leukemia (BCP-ALL) occurs early in B-cell development, in a pro-B or pre-B cell, when somatic recombination of variable (V), diversity (D), and joining (J) segment immunoglobulin (IG) genes and the B-cell rescue mechanism of V(H) replacement might be ongoing or fully active, driving clonal evolution. In this study of newly diagnosed BCP-ALL, we sought to understand the mechanistic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146586"
                        open
                      >
                        <summary class="article-expander__title">GTAC enables parallel genotyping of multiple genomic loci with chromatin accessibility profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230515205818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Understanding clonal evolution and cancer development requires experimental approaches for characterizing the consequences of somatic mutations on gene regulation. However, no methods currently exist that efficiently link high-content chromatin accessibility with high-confidence genotyping in single cells. To address this, we developed Genotyping with the Assay for Transposase-Accessible Chromatin (GTAC), enabling accurate mutation detection at multiple amplified loci, coupled with robust...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146250"
                        open
                      >
                        <summary class="article-expander__title">Genetic events associated with venetoclax resistance in CLL identified by whole exome sequencing of patient samples</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146250/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230515205818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Although BCL2 mutations are reported as later occurring events leading to venetoclax resistance, many other mechanisms of progression have been reported but remain poorly understood. Here we analyze longitudinal tumor samples from eleven patients with disease progression on venetoclax to characterize the clonal evolution of resistance. All patients tested showed increased in vitro resistance to venetoclax at their post-treatment timepoint. We found the previously described acquired BCL2-G101V...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37143651"
                        open
                      >
                        <summary class="article-expander__title">Insights into IGH clonal evolution in BCP-ALL: frequency, mechanisms, associations, and diagnostic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37143651/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230515205818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The malignant transformation leading to a maturation arrest in B-cell precursor acute lymphoblastic leukemia (BCP-ALL) occurs early in B-cell development, in a pro-B or pre-B cell, when somatic recombination of variable (V), diversity (D), and joining (J) segment immunoglobulin (IG) genes and the B-cell rescue mechanism of V(H) replacement might be ongoing or fully active, driving clonal evolution. In this study of newly diagnosed BCP-ALL, we sought to understand the mechanistic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146586"
                        open
                      >
                        <summary class="article-expander__title">GTAC enables parallel genotyping of multiple genomic loci with chromatin accessibility profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Understanding clonal evolution and cancer development requires experimental approaches for characterizing the consequences of somatic mutations on gene regulation. However, no methods currently exist that efficiently link high-content chromatin accessibility with high-confidence genotyping in single cells. To address this, we developed Genotyping with the Assay for Transposase-Accessible Chromatin (GTAC), enabling accurate mutation detection at multiple amplified loci, coupled with robust...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146250"
                        open
                      >
                        <summary class="article-expander__title">Genetic events associated with venetoclax resistance in CLL identified by whole exome sequencing of patient samples</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146250/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Although BCL2 mutations are reported as later occurring events leading to venetoclax resistance, many other mechanisms of progression have been reported but remain poorly understood. Here we analyze longitudinal tumor samples from eleven patients with disease progression on venetoclax to characterize the clonal evolution of resistance. All patients tested showed increased in vitro resistance to venetoclax at their post-treatment timepoint. We found the previously described acquired BCL2-G101V...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37143651"
                        open
                      >
                        <summary class="article-expander__title">Insights into IGH clonal evolution in BCP-ALL: frequency, mechanisms, associations, and diagnostic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37143651/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The malignant transformation leading to a maturation arrest in B-cell precursor acute lymphoblastic leukemia (BCP-ALL) occurs early in B-cell development, in a pro-B or pre-B cell, when somatic recombination of variable (V), diversity (D), and joining (J) segment immunoglobulin (IG) genes and the B-cell rescue mechanism of V(H) replacement might be ongoing or fully active, driving clonal evolution. In this study of newly diagnosed BCP-ALL, we sought to understand the mechanistic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146586"
                        open
                      >
                        <summary class="article-expander__title">GTAC enables parallel genotyping of multiple genomic loci with chromatin accessibility profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Understanding clonal evolution and cancer development requires experimental approaches for characterizing the consequences of somatic mutations on gene regulation. However, no methods currently exist that efficiently link high-content chromatin accessibility with high-confidence genotyping in single cells. To address this, we developed Genotyping with the Assay for Transposase-Accessible Chromatin (GTAC), enabling accurate mutation detection at multiple amplified loci, coupled with robust...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146250"
                        open
                      >
                        <summary class="article-expander__title">Genetic events associated with venetoclax resistance in CLL identified by whole exome sequencing of patient samples</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146250/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Although BCL2 mutations are reported as later occurring events leading to venetoclax resistance, many other mechanisms of progression have been reported but remain poorly understood. Here we analyze longitudinal tumor samples from eleven patients with disease progression on venetoclax to characterize the clonal evolution of resistance. All patients tested showed increased in vitro resistance to venetoclax at their post-treatment timepoint. We found the previously described acquired BCL2-G101V...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37143651"
                        open
                      >
                        <summary class="article-expander__title">Insights into IGH clonal evolution in BCP-ALL: frequency, mechanisms, associations, and diagnostic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37143651/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The malignant transformation leading to a maturation arrest in B-cell precursor acute lymphoblastic leukemia (BCP-ALL) occurs early in B-cell development, in a pro-B or pre-B cell, when somatic recombination of variable (V), diversity (D), and joining (J) segment immunoglobulin (IG) genes and the B-cell rescue mechanism of V(H) replacement might be ongoing or fully active, driving clonal evolution. In this study of newly diagnosed BCP-ALL, we sought to understand the mechanistic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146586"
                        open
                      >
                        <summary class="article-expander__title">GTAC enables parallel genotyping of multiple genomic loci with chromatin accessibility profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Understanding clonal evolution and cancer development requires experimental approaches for characterizing the consequences of somatic mutations on gene regulation. However, no methods currently exist that efficiently link high-content chromatin accessibility with high-confidence genotyping in single cells. To address this, we developed Genotyping with the Assay for Transposase-Accessible Chromatin (GTAC), enabling accurate mutation detection at multiple amplified loci, coupled with robust...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146250"
                        open
                      >
                        <summary class="article-expander__title">Genetic events associated with venetoclax resistance in CLL identified by whole exome sequencing of patient samples</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146250/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Although BCL2 mutations are reported as later occurring events leading to venetoclax resistance, many other mechanisms of progression have been reported but remain poorly understood. Here we analyze longitudinal tumor samples from eleven patients with disease progression on venetoclax to characterize the clonal evolution of resistance. All patients tested showed increased in vitro resistance to venetoclax at their post-treatment timepoint. We found the previously described acquired BCL2-G101V...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37143651"
                        open
                      >
                        <summary class="article-expander__title">Insights into IGH clonal evolution in BCP-ALL: frequency, mechanisms, associations, and diagnostic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37143651/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The malignant transformation leading to a maturation arrest in B-cell precursor acute lymphoblastic leukemia (BCP-ALL) occurs early in B-cell development, in a pro-B or pre-B cell, when somatic recombination of variable (V), diversity (D), and joining (J) segment immunoglobulin (IG) genes and the B-cell rescue mechanism of V(H) replacement might be ongoing or fully active, driving clonal evolution. In this study of newly diagnosed BCP-ALL, we sought to understand the mechanistic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146586"
                        open
                      >
                        <summary class="article-expander__title">GTAC enables parallel genotyping of multiple genomic loci with chromatin accessibility profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230511205726&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Understanding clonal evolution and cancer development requires experimental approaches for characterizing the consequences of somatic mutations on gene regulation. However, no methods currently exist that efficiently link high-content chromatin accessibility with high-confidence genotyping in single cells. To address this, we developed Genotyping with the Assay for Transposase-Accessible Chromatin (GTAC), enabling accurate mutation detection at multiple amplified loci, coupled with robust...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146250"
                        open
                      >
                        <summary class="article-expander__title">Genetic events associated with venetoclax resistance in CLL identified by whole exome sequencing of patient samples</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146250/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230511205726&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Although BCL2 mutations are reported as later occurring events leading to venetoclax resistance, many other mechanisms of progression have been reported but remain poorly understood. Here we analyze longitudinal tumor samples from eleven patients with disease progression on venetoclax to characterize the clonal evolution of resistance. All patients tested showed increased in vitro resistance to venetoclax at their post-treatment timepoint. We found the previously described acquired BCL2-G101V...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37143651"
                        open
                      >
                        <summary class="article-expander__title">Insights into IGH clonal evolution in BCP-ALL: frequency, mechanisms, associations, and diagnostic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37143651/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230511205726&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The malignant transformation leading to a maturation arrest in B-cell precursor acute lymphoblastic leukemia (BCP-ALL) occurs early in B-cell development, in a pro-B or pre-B cell, when somatic recombination of variable (V), diversity (D), and joining (J) segment immunoglobulin (IG) genes and the B-cell rescue mechanism of V(H) replacement might be ongoing or fully active, driving clonal evolution. In this study of newly diagnosed BCP-ALL, we sought to understand the mechanistic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146586"
                        open
                      >
                        <summary class="article-expander__title">GTAC enables parallel genotyping of multiple genomic loci with chromatin accessibility profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230510205718&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Understanding clonal evolution and cancer development requires experimental approaches for characterizing the consequences of somatic mutations on gene regulation. However, no methods currently exist that efficiently link high-content chromatin accessibility with high-confidence genotyping in single cells. To address this, we developed Genotyping with the Assay for Transposase-Accessible Chromatin (GTAC), enabling accurate mutation detection at multiple amplified loci, coupled with robust...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146250"
                        open
                      >
                        <summary class="article-expander__title">Genetic events associated with venetoclax resistance in CLL identified by whole exome sequencing of patient samples</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146250/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230510205718&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Although BCL2 mutations are reported as later occurring events leading to venetoclax resistance, many other mechanisms of progression have been reported but remain poorly understood. Here we analyze longitudinal tumor samples from eleven patients with disease progression on venetoclax to characterize the clonal evolution of resistance. All patients tested showed increased in vitro resistance to venetoclax at their post-treatment timepoint. We found the previously described acquired BCL2-G101V...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37143651"
                        open
                      >
                        <summary class="article-expander__title">Insights into IGH clonal evolution in BCP-ALL: frequency, mechanisms, associations, and diagnostic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37143651/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230510205718&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The malignant transformation leading to a maturation arrest in B-cell precursor acute lymphoblastic leukemia (BCP-ALL) occurs early in B-cell development, in a pro-B or pre-B cell, when somatic recombination of variable (V), diversity (D), and joining (J) segment immunoglobulin (IG) genes and the B-cell rescue mechanism of V(H) replacement might be ongoing or fully active, driving clonal evolution. In this study of newly diagnosed BCP-ALL, we sought to understand the mechanistic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146586"
                        open
                      >
                        <summary class="article-expander__title">GTAC enables parallel genotyping of multiple genomic loci with chromatin accessibility profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230509205648&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Understanding clonal evolution and cancer development requires experimental approaches for characterizing the consequences of somatic mutations on gene regulation. However, no methods currently exist that efficiently link high-content chromatin accessibility with high-confidence genotyping in single cells. To address this, we developed Genotyping with the Assay for Transposase-Accessible Chromatin (GTAC), enabling accurate mutation detection at multiple amplified loci, coupled with robust...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146250"
                        open
                      >
                        <summary class="article-expander__title">Genetic events associated with venetoclax resistance in CLL identified by whole exome sequencing of patient samples</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146250/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230509205648&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Although BCL2 mutations are reported as later occurring events leading to venetoclax resistance, many other mechanisms of progression have been reported but remain poorly understood. Here we analyze longitudinal tumor samples from eleven patients with disease progression on venetoclax to characterize the clonal evolution of resistance. All patients tested showed increased in vitro resistance to venetoclax at their post-treatment timepoint. We found the previously described acquired BCL2-G101V...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37143651"
                        open
                      >
                        <summary class="article-expander__title">Insights into IGH clonal evolution in BCP-ALL: frequency, mechanisms, associations, and diagnostic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37143651/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230509205648&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The malignant transformation leading to a maturation arrest in B-cell precursor acute lymphoblastic leukemia (BCP-ALL) occurs early in B-cell development, in a pro-B or pre-B cell, when somatic recombination of variable (V), diversity (D), and joining (J) segment immunoglobulin (IG) genes and the B-cell rescue mechanism of V(H) replacement might be ongoing or fully active, driving clonal evolution. In this study of newly diagnosed BCP-ALL, we sought to understand the mechanistic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146586"
                        open
                      >
                        <summary class="article-expander__title">GTAC enables parallel genotyping of multiple genomic loci with chromatin accessibility profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230508205817&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Understanding clonal evolution and cancer development requires experimental approaches for characterizing the consequences of somatic mutations on gene regulation. However, no methods currently exist that efficiently link high-content chromatin accessibility with high-confidence genotyping in single cells. To address this, we developed Genotyping with the Assay for Transposase-Accessible Chromatin (GTAC), enabling accurate mutation detection at multiple amplified loci, coupled with robust...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146250"
                        open
                      >
                        <summary class="article-expander__title">Genetic events associated with venetoclax resistance in CLL identified by whole exome sequencing of patient samples</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146250/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230508205817&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Although BCL2 mutations are reported as later occurring events leading to venetoclax resistance, many other mechanisms of progression have been reported but remain poorly understood. Here we analyze longitudinal tumor samples from eleven patients with disease progression on venetoclax to characterize the clonal evolution of resistance. All patients tested showed increased in vitro resistance to venetoclax at their post-treatment timepoint. We found the previously described acquired BCL2-G101V...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37143651"
                        open
                      >
                        <summary class="article-expander__title">Insights into IGH clonal evolution in BCP-ALL: frequency, mechanisms, associations, and diagnostic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37143651/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230508205817&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The malignant transformation leading to a maturation arrest in B-cell precursor acute lymphoblastic leukemia (BCP-ALL) occurs early in B-cell development, in a pro-B or pre-B cell, when somatic recombination of variable (V), diversity (D), and joining (J) segment immunoglobulin (IG) genes and the B-cell rescue mechanism of V(H) replacement might be ongoing or fully active, driving clonal evolution. In this study of newly diagnosed BCP-ALL, we sought to understand the mechanistic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146586"
                        open
                      >
                        <summary class="article-expander__title">GTAC enables parallel genotyping of multiple genomic loci with chromatin accessibility profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230507205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Understanding clonal evolution and cancer development requires experimental approaches for characterizing the consequences of somatic mutations on gene regulation. However, no methods currently exist that efficiently link high-content chromatin accessibility with high-confidence genotyping in single cells. To address this, we developed Genotyping with the Assay for Transposase-Accessible Chromatin (GTAC), enabling accurate mutation detection at multiple amplified loci, coupled with robust...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146250"
                        open
                      >
                        <summary class="article-expander__title">Genetic events associated with venetoclax resistance in CLL identified by whole exome sequencing of patient samples</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146250/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230507205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Although BCL2 mutations are reported as later occurring events leading to venetoclax resistance, many other mechanisms of progression have been reported but remain poorly understood. Here we analyze longitudinal tumor samples from eleven patients with disease progression on venetoclax to characterize the clonal evolution of resistance. All patients tested showed increased in vitro resistance to venetoclax at their post-treatment timepoint. We found the previously described acquired BCL2-G101V...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37143651"
                        open
                      >
                        <summary class="article-expander__title">Insights into IGH clonal evolution in BCP-ALL: frequency, mechanisms, associations, and diagnostic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37143651/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230507205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The malignant transformation leading to a maturation arrest in B-cell precursor acute lymphoblastic leukemia (BCP-ALL) occurs early in B-cell development, in a pro-B or pre-B cell, when somatic recombination of variable (V), diversity (D), and joining (J) segment immunoglobulin (IG) genes and the B-cell rescue mechanism of V(H) replacement might be ongoing or fully active, driving clonal evolution. In this study of newly diagnosed BCP-ALL, we sought to understand the mechanistic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146586"
                        open
                      >
                        <summary class="article-expander__title">GTAC enables parallel genotyping of multiple genomic loci with chromatin accessibility profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230506210046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Understanding clonal evolution and cancer development requires experimental approaches for characterizing the consequences of somatic mutations on gene regulation. However, no methods currently exist that efficiently link high-content chromatin accessibility with high-confidence genotyping in single cells. To address this, we developed Genotyping with the Assay for Transposase-Accessible Chromatin (GTAC), enabling accurate mutation detection at multiple amplified loci, coupled with robust...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146250"
                        open
                      >
                        <summary class="article-expander__title">Genetic events associated with venetoclax resistance in CLL identified by whole exome sequencing of patient samples</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146250/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230506210046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Although BCL2 mutations are reported as later occurring events leading to venetoclax resistance, many other mechanisms of progression have been reported but remain poorly understood. Here we analyze longitudinal tumor samples from eleven patients with disease progression on venetoclax to characterize the clonal evolution of resistance. All patients tested showed increased in vitro resistance to venetoclax at their post-treatment timepoint. We found the previously described acquired BCL2-G101V...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37143651"
                        open
                      >
                        <summary class="article-expander__title">Insights into IGH clonal evolution in BCP-ALL: frequency, mechanisms, associations, and diagnostic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37143651/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230506210046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The malignant transformation leading to a maturation arrest in B-cell precursor acute lymphoblastic leukemia (BCP-ALL) occurs early in B-cell development, in a pro-B or pre-B cell, when somatic recombination of variable (V), diversity (D), and joining (J) segment immunoglobulin (IG) genes and the B-cell rescue mechanism of V(H) replacement might be ongoing or fully active, driving clonal evolution. In this study of newly diagnosed BCP-ALL, we sought to understand the mechanistic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37146250"
                        open
                      >
                        <summary class="article-expander__title">Genetic events associated with venetoclax resistance in CLL identified by whole exome sequencing of patient samples</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37146250/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230505205328&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Although BCL2 mutations are reported as later occurring events leading to venetoclax resistance, many other mechanisms of progression have been reported but remain poorly understood. Here we analyze longitudinal tumor samples from eleven patients with disease progression on venetoclax to characterize the clonal evolution of resistance. All patients tested showed increased in vitro resistance to venetoclax at their post-treatment timepoint. We found the previously described acquired BCL2-G101V...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37143651"
                        open
                      >
                        <summary class="article-expander__title">Insights into IGH clonal evolution in BCP-ALL: frequency, mechanisms, associations, and diagnostic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37143651/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230505205328&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The malignant transformation leading to a maturation arrest in B-cell precursor acute lymphoblastic leukemia (BCP-ALL) occurs early in B-cell development, in a pro-B or pre-B cell, when somatic recombination of variable (V), diversity (D), and joining (J) segment immunoglobulin (IG) genes and the B-cell rescue mechanism of V(H) replacement might be ongoing or fully active, driving clonal evolution. In this study of newly diagnosed BCP-ALL, we sought to understand the mechanistic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-05-04 datetime="2023-05-04T10:00:00.000Z">2023-05-04</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-05-04 datetime="2023-05-04T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37140166"
                        open
                      >
                        <summary class="article-expander__title">Familial Clonal Hematopoiesis in a Long Telomere Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37140166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230517205652&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: POT1 mutations associated with long telomere length conferred a predisposition to a familial clonal hematopoiesis syndrome that was associated with a range of benign and malignant solid neoplasms. The risk of these phenotypes was mediated by extended cellular longevity and by the capacity to maintain telomeres over time. (Funded by the National Institutes of Health and others.).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37140166"
                        open
                      >
                        <summary class="article-expander__title">Familial Clonal Hematopoiesis in a Long Telomere Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37140166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: POT1 mutations associated with long telomere length conferred a predisposition to a familial clonal hematopoiesis syndrome that was associated with a range of benign and malignant solid neoplasms. The risk of these phenotypes was mediated by extended cellular longevity and by the capacity to maintain telomeres over time. (Funded by the National Institutes of Health and others.).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37140166"
                        open
                      >
                        <summary class="article-expander__title">Familial Clonal Hematopoiesis in a Long Telomere Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37140166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230515205819&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: POT1 mutations associated with long telomere length conferred a predisposition to a familial clonal hematopoiesis syndrome that was associated with a range of benign and malignant solid neoplasms. The risk of these phenotypes was mediated by extended cellular longevity and by the capacity to maintain telomeres over time. (Funded by the National Institutes of Health and others.).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37140166"
                        open
                      >
                        <summary class="article-expander__title">Familial Clonal Hematopoiesis in a Long Telomere Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37140166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: POT1 mutations associated with long telomere length conferred a predisposition to a familial clonal hematopoiesis syndrome that was associated with a range of benign and malignant solid neoplasms. The risk of these phenotypes was mediated by extended cellular longevity and by the capacity to maintain telomeres over time. (Funded by the National Institutes of Health and others.).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37140166"
                        open
                      >
                        <summary class="article-expander__title">Familial Clonal Hematopoiesis in a Long Telomere Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37140166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: POT1 mutations associated with long telomere length conferred a predisposition to a familial clonal hematopoiesis syndrome that was associated with a range of benign and malignant solid neoplasms. The risk of these phenotypes was mediated by extended cellular longevity and by the capacity to maintain telomeres over time. (Funded by the National Institutes of Health and others.).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37140166"
                        open
                      >
                        <summary class="article-expander__title">Familial Clonal Hematopoiesis in a Long Telomere Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37140166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: POT1 mutations associated with long telomere length conferred a predisposition to a familial clonal hematopoiesis syndrome that was associated with a range of benign and malignant solid neoplasms. The risk of these phenotypes was mediated by extended cellular longevity and by the capacity to maintain telomeres over time. (Funded by the National Institutes of Health and others.).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37140166"
                        open
                      >
                        <summary class="article-expander__title">Familial Clonal Hematopoiesis in a Long Telomere Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37140166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230511205726&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: POT1 mutations associated with long telomere length conferred a predisposition to a familial clonal hematopoiesis syndrome that was associated with a range of benign and malignant solid neoplasms. The risk of these phenotypes was mediated by extended cellular longevity and by the capacity to maintain telomeres over time. (Funded by the National Institutes of Health and others.).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37140166"
                        open
                      >
                        <summary class="article-expander__title">Familial Clonal Hematopoiesis in a Long Telomere Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37140166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230510205718&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: POT1 mutations associated with long telomere length conferred a predisposition to a familial clonal hematopoiesis syndrome that was associated with a range of benign and malignant solid neoplasms. The risk of these phenotypes was mediated by extended cellular longevity and by the capacity to maintain telomeres over time. (Funded by the National Institutes of Health and others.).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37140166"
                        open
                      >
                        <summary class="article-expander__title">Familial Clonal Hematopoiesis in a Long Telomere Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37140166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230509205648&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: POT1 mutations associated with long telomere length conferred a predisposition to a familial clonal hematopoiesis syndrome that was associated with a range of benign and malignant solid neoplasms. The risk of these phenotypes was mediated by extended cellular longevity and by the capacity to maintain telomeres over time. (Funded by the National Institutes of Health and others.).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37140166"
                        open
                      >
                        <summary class="article-expander__title">Familial Clonal Hematopoiesis in a Long Telomere Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37140166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230508205817&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: POT1 mutations associated with long telomere length conferred a predisposition to a familial clonal hematopoiesis syndrome that was associated with a range of benign and malignant solid neoplasms. The risk of these phenotypes was mediated by extended cellular longevity and by the capacity to maintain telomeres over time. (Funded by the National Institutes of Health and others.).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37140166"
                        open
                      >
                        <summary class="article-expander__title">Familial Clonal Hematopoiesis in a Long Telomere Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37140166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230507205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: POT1 mutations associated with long telomere length conferred a predisposition to a familial clonal hematopoiesis syndrome that was associated with a range of benign and malignant solid neoplasms. The risk of these phenotypes was mediated by extended cellular longevity and by the capacity to maintain telomeres over time. (Funded by the National Institutes of Health and others.).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37140166"
                        open
                      >
                        <summary class="article-expander__title">Familial Clonal Hematopoiesis in a Long Telomere Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37140166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230506210046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: POT1 mutations associated with long telomere length conferred a predisposition to a familial clonal hematopoiesis syndrome that was associated with a range of benign and malignant solid neoplasms. The risk of these phenotypes was mediated by extended cellular longevity and by the capacity to maintain telomeres over time. (Funded by the National Institutes of Health and others.).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37140166"
                        open
                      >
                        <summary class="article-expander__title">Familial Clonal Hematopoiesis in a Long Telomere Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37140166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230505205328&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: POT1 mutations associated with long telomere length conferred a predisposition to a familial clonal hematopoiesis syndrome that was associated with a range of benign and malignant solid neoplasms. The risk of these phenotypes was mediated by extended cellular longevity and by the capacity to maintain telomeres over time. (Funded by the National Institutes of Health and others.).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37140166"
                        open
                      >
                        <summary class="article-expander__title">Familial Clonal Hematopoiesis in a Long Telomere Syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37140166/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230504205311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: POT1 mutations associated with long telomere length conferred a predisposition to a familial clonal hematopoiesis syndrome that was associated with a range of benign and malignant solid neoplasms. The risk of these phenotypes was mediated by extended cellular longevity and by the capacity to maintain telomeres over time. (Funded by the National Institutes of Health and others.).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37140929"
                        open
                      >
                        <summary class="article-expander__title">Resolving the pathogenesis of anaplastic Wilms tumors through spatial mapping of cancer cell evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37140929/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230515205818&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: WTs with DA display significantly more complex phylogenies compared to non-DA WTs, including features of saltatory and parallel evolution. The subclonal landscape of individual tumors was constrained by anatomic compartments, which should be considered when sampling tissue for precision diagnostics.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37140929"
                        open
                      >
                        <summary class="article-expander__title">Resolving the pathogenesis of anaplastic Wilms tumors through spatial mapping of cancer cell evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37140929/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: WTs with DA display significantly more complex phylogenies compared to non-DA WTs, including features of saltatory and parallel evolution. The subclonal landscape of individual tumors was constrained by anatomic compartments, which should be considered when sampling tissue for precision diagnostics.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37140929"
                        open
                      >
                        <summary class="article-expander__title">Resolving the pathogenesis of anaplastic Wilms tumors through spatial mapping of cancer cell evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37140929/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: WTs with DA display significantly more complex phylogenies compared to non-DA WTs, including features of saltatory and parallel evolution. The subclonal landscape of individual tumors was constrained by anatomic compartments, which should be considered when sampling tissue for precision diagnostics.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37139709"
                        open
                      >
                        <summary class="article-expander__title">Molecular-defined clonal evolution in patients with classical myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37139709/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classical myeloproliferative neoplasms (MPNs) are characterized by distinct clinical phenotypes. The discovery of driver mutations in JAK2, CALR and MPL genes provided new insights into their pathogenesis. Next-generation sequencing (NGS) identified additional somatic mutations, most frequently in epigenetic modulator genes. In this study, a cohort of 95 MPN patients was genetically characterized using targeted NGS. Clonal hierarchies of detected mutations were subsequently analysed using colony...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37140929"
                        open
                      >
                        <summary class="article-expander__title">Resolving the pathogenesis of anaplastic Wilms tumors through spatial mapping of cancer cell evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37140929/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: WTs with DA display significantly more complex phylogenies compared to non-DA WTs, including features of saltatory and parallel evolution. The subclonal landscape of individual tumors was constrained by anatomic compartments, which should be considered when sampling tissue for precision diagnostics.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37139709"
                        open
                      >
                        <summary class="article-expander__title">Molecular-defined clonal evolution in patients with classical myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37139709/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classical myeloproliferative neoplasms (MPNs) are characterized by distinct clinical phenotypes. The discovery of driver mutations in JAK2, CALR and MPL genes provided new insights into their pathogenesis. Next-generation sequencing (NGS) identified additional somatic mutations, most frequently in epigenetic modulator genes. In this study, a cohort of 95 MPN patients was genetically characterized using targeted NGS. Clonal hierarchies of detected mutations were subsequently analysed using colony...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37140929"
                        open
                      >
                        <summary class="article-expander__title">Resolving the pathogenesis of anaplastic Wilms tumors through spatial mapping of cancer cell evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37140929/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230511205726&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: WTs with DA display significantly more complex phylogenies compared to non-DA WTs, including features of saltatory and parallel evolution. The subclonal landscape of individual tumors was constrained by anatomic compartments, which should be considered when sampling tissue for precision diagnostics.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37139709"
                        open
                      >
                        <summary class="article-expander__title">Molecular-defined clonal evolution in patients with classical myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37139709/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230511205726&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classical myeloproliferative neoplasms (MPNs) are characterized by distinct clinical phenotypes. The discovery of driver mutations in JAK2, CALR and MPL genes provided new insights into their pathogenesis. Next-generation sequencing (NGS) identified additional somatic mutations, most frequently in epigenetic modulator genes. In this study, a cohort of 95 MPN patients was genetically characterized using targeted NGS. Clonal hierarchies of detected mutations were subsequently analysed using colony...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37140929"
                        open
                      >
                        <summary class="article-expander__title">Resolving the pathogenesis of anaplastic Wilms tumors through spatial mapping of cancer cell evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37140929/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230510205718&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: WTs with DA display significantly more complex phylogenies compared to non-DA WTs, including features of saltatory and parallel evolution. The subclonal landscape of individual tumors was constrained by anatomic compartments, which should be considered when sampling tissue for precision diagnostics.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37139709"
                        open
                      >
                        <summary class="article-expander__title">Molecular-defined clonal evolution in patients with classical myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37139709/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230510205718&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classical myeloproliferative neoplasms (MPNs) are characterized by distinct clinical phenotypes. The discovery of driver mutations in JAK2, CALR and MPL genes provided new insights into their pathogenesis. Next-generation sequencing (NGS) identified additional somatic mutations, most frequently in epigenetic modulator genes. In this study, a cohort of 95 MPN patients was genetically characterized using targeted NGS. Clonal hierarchies of detected mutations were subsequently analysed using colony...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37140929"
                        open
                      >
                        <summary class="article-expander__title">Resolving the pathogenesis of anaplastic Wilms tumors through spatial mapping of cancer cell evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37140929/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230509205648&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: WTs with DA display significantly more complex phylogenies compared to non-DA WTs, including features of saltatory and parallel evolution. The subclonal landscape of individual tumors was constrained by anatomic compartments, which should be considered when sampling tissue for precision diagnostics.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37139709"
                        open
                      >
                        <summary class="article-expander__title">Molecular-defined clonal evolution in patients with classical myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37139709/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230509205648&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classical myeloproliferative neoplasms (MPNs) are characterized by distinct clinical phenotypes. The discovery of driver mutations in JAK2, CALR and MPL genes provided new insights into their pathogenesis. Next-generation sequencing (NGS) identified additional somatic mutations, most frequently in epigenetic modulator genes. In this study, a cohort of 95 MPN patients was genetically characterized using targeted NGS. Clonal hierarchies of detected mutations were subsequently analysed using colony...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37140929"
                        open
                      >
                        <summary class="article-expander__title">Resolving the pathogenesis of anaplastic Wilms tumors through spatial mapping of cancer cell evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37140929/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230508205817&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: WTs with DA display significantly more complex phylogenies compared to non-DA WTs, including features of saltatory and parallel evolution. The subclonal landscape of individual tumors was constrained by anatomic compartments, which should be considered when sampling tissue for precision diagnostics.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37139709"
                        open
                      >
                        <summary class="article-expander__title">Molecular-defined clonal evolution in patients with classical myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37139709/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230508205817&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classical myeloproliferative neoplasms (MPNs) are characterized by distinct clinical phenotypes. The discovery of driver mutations in JAK2, CALR and MPL genes provided new insights into their pathogenesis. Next-generation sequencing (NGS) identified additional somatic mutations, most frequently in epigenetic modulator genes. In this study, a cohort of 95 MPN patients was genetically characterized using targeted NGS. Clonal hierarchies of detected mutations were subsequently analysed using colony...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37140929"
                        open
                      >
                        <summary class="article-expander__title">Resolving the pathogenesis of anaplastic Wilms tumors through spatial mapping of cancer cell evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37140929/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230507205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: WTs with DA display significantly more complex phylogenies compared to non-DA WTs, including features of saltatory and parallel evolution. The subclonal landscape of individual tumors was constrained by anatomic compartments, which should be considered when sampling tissue for precision diagnostics.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37139709"
                        open
                      >
                        <summary class="article-expander__title">Molecular-defined clonal evolution in patients with classical myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37139709/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230507205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classical myeloproliferative neoplasms (MPNs) are characterized by distinct clinical phenotypes. The discovery of driver mutations in JAK2, CALR and MPL genes provided new insights into their pathogenesis. Next-generation sequencing (NGS) identified additional somatic mutations, most frequently in epigenetic modulator genes. In this study, a cohort of 95 MPN patients was genetically characterized using targeted NGS. Clonal hierarchies of detected mutations were subsequently analysed using colony...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37140929"
                        open
                      >
                        <summary class="article-expander__title">Resolving the pathogenesis of anaplastic Wilms tumors through spatial mapping of cancer cell evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37140929/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230506210046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: WTs with DA display significantly more complex phylogenies compared to non-DA WTs, including features of saltatory and parallel evolution. The subclonal landscape of individual tumors was constrained by anatomic compartments, which should be considered when sampling tissue for precision diagnostics.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37139709"
                        open
                      >
                        <summary class="article-expander__title">Molecular-defined clonal evolution in patients with classical myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37139709/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230506210046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classical myeloproliferative neoplasms (MPNs) are characterized by distinct clinical phenotypes. The discovery of driver mutations in JAK2, CALR and MPL genes provided new insights into their pathogenesis. Next-generation sequencing (NGS) identified additional somatic mutations, most frequently in epigenetic modulator genes. In this study, a cohort of 95 MPN patients was genetically characterized using targeted NGS. Clonal hierarchies of detected mutations were subsequently analysed using colony...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37140929"
                        open
                      >
                        <summary class="article-expander__title">Resolving the pathogenesis of anaplastic Wilms tumors through spatial mapping of cancer cell evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37140929/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230505205328&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: WTs with DA display significantly more complex phylogenies compared to non-DA WTs, including features of saltatory and parallel evolution. The subclonal landscape of individual tumors was constrained by anatomic compartments, which should be considered when sampling tissue for precision diagnostics.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37139709"
                        open
                      >
                        <summary class="article-expander__title">Molecular-defined clonal evolution in patients with classical myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37139709/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230505205328&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classical myeloproliferative neoplasms (MPNs) are characterized by distinct clinical phenotypes. The discovery of driver mutations in JAK2, CALR and MPL genes provided new insights into their pathogenesis. Next-generation sequencing (NGS) identified additional somatic mutations, most frequently in epigenetic modulator genes. In this study, a cohort of 95 MPN patients was genetically characterized using targeted NGS. Clonal hierarchies of detected mutations were subsequently analysed using colony...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37140929"
                        open
                      >
                        <summary class="article-expander__title">Resolving the pathogenesis of anaplastic Wilms tumors through spatial mapping of cancer cell evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37140929/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230504205311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: WTs with DA display significantly more complex phylogenies compared to non-DA WTs, including features of saltatory and parallel evolution. The subclonal landscape of individual tumors was constrained by anatomic compartments, which should be considered when sampling tissue for precision diagnostics.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37139709"
                        open
                      >
                        <summary class="article-expander__title">Molecular-defined clonal evolution in patients with classical myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37139709/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230504205311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classical myeloproliferative neoplasms (MPNs) are characterized by distinct clinical phenotypes. The discovery of driver mutations in JAK2, CALR and MPL genes provided new insights into their pathogenesis. Next-generation sequencing (NGS) identified additional somatic mutations, most frequently in epigenetic modulator genes. In this study, a cohort of 95 MPN patients was genetically characterized using targeted NGS. Clonal hierarchies of detected mutations were subsequently analysed using colony...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-05-03 datetime="2023-05-03T10:00:00.000Z">2023-05-03</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-05-03 datetime="2023-05-03T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37133448"
                        open
                      >
                        <summary class="article-expander__title">A rare subset of primary tumor cells with concomitant hyper-activation of extracellular matrix remodeling and dsRNA-IFN1 signaling metastasizes in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37133448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastatic breast cancer (BC) has a poor prognosis and is largely considered incurable. A better understanding of the molecular determinants of BC metastasis could facilitate development of improved prevention and treatment strategies. We used lentiviral barcoding coupled to single-cell RNA sequencing to trace clonal and transcriptional evolution during BC metastasis and showed that metastases derive from rare pro-metastatic clones that are under-represented in primary tumors. Both low clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37133448"
                        open
                      >
                        <summary class="article-expander__title">A rare subset of primary tumor cells with concomitant hyper-activation of extracellular matrix remodeling and dsRNA-IFN1 signaling metastasizes in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37133448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230511205726&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastatic breast cancer (BC) has a poor prognosis and is largely considered incurable. A better understanding of the molecular determinants of BC metastasis could facilitate development of improved prevention and treatment strategies. We used lentiviral barcoding coupled to single-cell RNA sequencing to trace clonal and transcriptional evolution during BC metastasis and showed that metastases derive from rare pro-metastatic clones that are under-represented in primary tumors. Both low clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37133448"
                        open
                      >
                        <summary class="article-expander__title">A rare subset of primary tumor cells with concomitant hyper-activation of extracellular matrix remodeling and dsRNA-IFN1 signaling metastasizes in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37133448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230510205718&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastatic breast cancer (BC) has a poor prognosis and is largely considered incurable. A better understanding of the molecular determinants of BC metastasis could facilitate development of improved prevention and treatment strategies. We used lentiviral barcoding coupled to single-cell RNA sequencing to trace clonal and transcriptional evolution during BC metastasis and showed that metastases derive from rare pro-metastatic clones that are under-represented in primary tumors. Both low clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37133448"
                        open
                      >
                        <summary class="article-expander__title">A rare subset of primary tumor cells with concomitant hyper-activation of extracellular matrix remodeling and dsRNA-IFN1 signaling metastasizes in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37133448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230509205648&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastatic breast cancer (BC) has a poor prognosis and is largely considered incurable. A better understanding of the molecular determinants of BC metastasis could facilitate development of improved prevention and treatment strategies. We used lentiviral barcoding coupled to single-cell RNA sequencing to trace clonal and transcriptional evolution during BC metastasis and showed that metastases derive from rare pro-metastatic clones that are under-represented in primary tumors. Both low clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37133448"
                        open
                      >
                        <summary class="article-expander__title">A rare subset of primary tumor cells with concomitant hyper-activation of extracellular matrix remodeling and dsRNA-IFN1 signaling metastasizes in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37133448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230508205817&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastatic breast cancer (BC) has a poor prognosis and is largely considered incurable. A better understanding of the molecular determinants of BC metastasis could facilitate development of improved prevention and treatment strategies. We used lentiviral barcoding coupled to single-cell RNA sequencing to trace clonal and transcriptional evolution during BC metastasis and showed that metastases derive from rare pro-metastatic clones that are under-represented in primary tumors. Both low clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37133448"
                        open
                      >
                        <summary class="article-expander__title">A rare subset of primary tumor cells with concomitant hyper-activation of extracellular matrix remodeling and dsRNA-IFN1 signaling metastasizes in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37133448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230507205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastatic breast cancer (BC) has a poor prognosis and is largely considered incurable. A better understanding of the molecular determinants of BC metastasis could facilitate development of improved prevention and treatment strategies. We used lentiviral barcoding coupled to single-cell RNA sequencing to trace clonal and transcriptional evolution during BC metastasis and showed that metastases derive from rare pro-metastatic clones that are under-represented in primary tumors. Both low clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37133448"
                        open
                      >
                        <summary class="article-expander__title">A rare subset of primary tumor cells with concomitant hyper-activation of extracellular matrix remodeling and dsRNA-IFN1 signaling metastasizes in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37133448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230506210046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastatic breast cancer (BC) has a poor prognosis and is largely considered incurable. A better understanding of the molecular determinants of BC metastasis could facilitate development of improved prevention and treatment strategies. We used lentiviral barcoding coupled to single-cell RNA sequencing to trace clonal and transcriptional evolution during BC metastasis and showed that metastases derive from rare pro-metastatic clones that are under-represented in primary tumors. Both low clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37133448"
                        open
                      >
                        <summary class="article-expander__title">A rare subset of primary tumor cells with concomitant hyper-activation of extracellular matrix remodeling and dsRNA-IFN1 signaling metastasizes in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37133448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230505205328&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastatic breast cancer (BC) has a poor prognosis and is largely considered incurable. A better understanding of the molecular determinants of BC metastasis could facilitate development of improved prevention and treatment strategies. We used lentiviral barcoding coupled to single-cell RNA sequencing to trace clonal and transcriptional evolution during BC metastasis and showed that metastases derive from rare pro-metastatic clones that are under-represented in primary tumors. Both low clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37133448"
                        open
                      >
                        <summary class="article-expander__title">A rare subset of primary tumor cells with concomitant hyper-activation of extracellular matrix remodeling and dsRNA-IFN1 signaling metastasizes in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37133448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230504205311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastatic breast cancer (BC) has a poor prognosis and is largely considered incurable. A better understanding of the molecular determinants of BC metastasis could facilitate development of improved prevention and treatment strategies. We used lentiviral barcoding coupled to single-cell RNA sequencing to trace clonal and transcriptional evolution during BC metastasis and showed that metastases derive from rare pro-metastatic clones that are under-represented in primary tumors. Both low clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37133448"
                        open
                      >
                        <summary class="article-expander__title">A rare subset of primary tumor cells with concomitant hyper-activation of extracellular matrix remodeling and dsRNA-IFN1 signaling metastasizes in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37133448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230503205635&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastatic breast cancer (BC) has a poor prognosis and is largely considered incurable. A better understanding of the molecular determinants of BC metastasis could facilitate development of improved prevention and treatment strategies. We used lentiviral barcoding coupled to single-cell RNA sequencing to trace clonal and transcriptional evolution during BC metastasis and showed that metastases derive from rare pro-metastatic clones that are under-represented in primary tumors. Both low clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-05-02 datetime="2023-05-02T10:00:00.000Z">2023-05-02</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-05-02 datetime="2023-05-02T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;ff&#x3D;20230625211348&amp;v&#x3D;2.17.9.post6+86293ac&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37131267"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic features between primary and paired metastatic fumarate hydratase-deficient renal cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37131267/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230511205726&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Overall, our study demonstrated the genomic, epigenomic, and transcriptomic features of metastatic lesions in FH-RCC and revealed their early evolutionary trajectory. These results provided multi-omics evidence portraying the progression of FH-RCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37131267"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic features between primary and paired metastatic fumarate hydratase-deficient renal cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37131267/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230510205718&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Overall, our study demonstrated the genomic, epigenomic, and transcriptomic features of metastatic lesions in FH-RCC and revealed their early evolutionary trajectory. These results provided multi-omics evidence portraying the progression of FH-RCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37131267"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic features between primary and paired metastatic fumarate hydratase-deficient renal cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37131267/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230509205648&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Overall, our study demonstrated the genomic, epigenomic, and transcriptomic features of metastatic lesions in FH-RCC and revealed their early evolutionary trajectory. These results provided multi-omics evidence portraying the progression of FH-RCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37131267"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic features between primary and paired metastatic fumarate hydratase-deficient renal cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37131267/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230508205817&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Overall, our study demonstrated the genomic, epigenomic, and transcriptomic features of metastatic lesions in FH-RCC and revealed their early evolutionary trajectory. These results provided multi-omics evidence portraying the progression of FH-RCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37131267"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic features between primary and paired metastatic fumarate hydratase-deficient renal cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37131267/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230507205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Overall, our study demonstrated the genomic, epigenomic, and transcriptomic features of metastatic lesions in FH-RCC and revealed their early evolutionary trajectory. These results provided multi-omics evidence portraying the progression of FH-RCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37131267"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic features between primary and paired metastatic fumarate hydratase-deficient renal cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37131267/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230506210046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Overall, our study demonstrated the genomic, epigenomic, and transcriptomic features of metastatic lesions in FH-RCC and revealed their early evolutionary trajectory. These results provided multi-omics evidence portraying the progression of FH-RCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37131267"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic features between primary and paired metastatic fumarate hydratase-deficient renal cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37131267/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230505205328&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Overall, our study demonstrated the genomic, epigenomic, and transcriptomic features of metastatic lesions in FH-RCC and revealed their early evolutionary trajectory. These results provided multi-omics evidence portraying the progression of FH-RCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37131267"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic features between primary and paired metastatic fumarate hydratase-deficient renal cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37131267/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230504205311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Overall, our study demonstrated the genomic, epigenomic, and transcriptomic features of metastatic lesions in FH-RCC and revealed their early evolutionary trajectory. These results provided multi-omics evidence portraying the progression of FH-RCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37131267"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic features between primary and paired metastatic fumarate hydratase-deficient renal cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37131267/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230503205635&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Overall, our study demonstrated the genomic, epigenomic, and transcriptomic features of metastatic lesions in FH-RCC and revealed their early evolutionary trajectory. These results provided multi-omics evidence portraying the progression of FH-RCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-05-01 datetime="2023-05-01T10:00:00.000Z">2023-05-01</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-05-01 datetime="2023-05-01T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37126548"
                        open
                      >
                        <summary class="article-expander__title">The genetic determinants of recurrent somatic mutations in 43,693 blood genomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37126548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Nononcogenic somatic mutations are thought to be uncommon and inconsequential. To test this, we analyzed 43,693 National Heart, Lung and Blood Institute Trans-Omics for Precision Medicine blood whole genomes from 37 cohorts and identified 7131 non-missense somatic mutations that are recurrently mutated in at least 50 individuals. These recurrent non-missense somatic mutations (RNMSMs) are not clearly explained by other clonal phenomena such as clonal hematopoiesis. RNMSM prevalence increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37126548"
                        open
                      >
                        <summary class="article-expander__title">The genetic determinants of recurrent somatic mutations in 43,693 blood genomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37126548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230515205819&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Nononcogenic somatic mutations are thought to be uncommon and inconsequential. To test this, we analyzed 43,693 National Heart, Lung and Blood Institute Trans-Omics for Precision Medicine blood whole genomes from 37 cohorts and identified 7131 non-missense somatic mutations that are recurrently mutated in at least 50 individuals. These recurrent non-missense somatic mutations (RNMSMs) are not clearly explained by other clonal phenomena such as clonal hematopoiesis. RNMSM prevalence increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37126548"
                        open
                      >
                        <summary class="article-expander__title">The genetic determinants of recurrent somatic mutations in 43,693 blood genomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37126548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Nononcogenic somatic mutations are thought to be uncommon and inconsequential. To test this, we analyzed 43,693 National Heart, Lung and Blood Institute Trans-Omics for Precision Medicine blood whole genomes from 37 cohorts and identified 7131 non-missense somatic mutations that are recurrently mutated in at least 50 individuals. These recurrent non-missense somatic mutations (RNMSMs) are not clearly explained by other clonal phenomena such as clonal hematopoiesis. RNMSM prevalence increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37126548"
                        open
                      >
                        <summary class="article-expander__title">The genetic determinants of recurrent somatic mutations in 43,693 blood genomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37126548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Nononcogenic somatic mutations are thought to be uncommon and inconsequential. To test this, we analyzed 43,693 National Heart, Lung and Blood Institute Trans-Omics for Precision Medicine blood whole genomes from 37 cohorts and identified 7131 non-missense somatic mutations that are recurrently mutated in at least 50 individuals. These recurrent non-missense somatic mutations (RNMSMs) are not clearly explained by other clonal phenomena such as clonal hematopoiesis. RNMSM prevalence increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37126548"
                        open
                      >
                        <summary class="article-expander__title">The genetic determinants of recurrent somatic mutations in 43,693 blood genomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37126548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Nononcogenic somatic mutations are thought to be uncommon and inconsequential. To test this, we analyzed 43,693 National Heart, Lung and Blood Institute Trans-Omics for Precision Medicine blood whole genomes from 37 cohorts and identified 7131 non-missense somatic mutations that are recurrently mutated in at least 50 individuals. These recurrent non-missense somatic mutations (RNMSMs) are not clearly explained by other clonal phenomena such as clonal hematopoiesis. RNMSM prevalence increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37126548"
                        open
                      >
                        <summary class="article-expander__title">The genetic determinants of recurrent somatic mutations in 43,693 blood genomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37126548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230511205726&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Nononcogenic somatic mutations are thought to be uncommon and inconsequential. To test this, we analyzed 43,693 National Heart, Lung and Blood Institute Trans-Omics for Precision Medicine blood whole genomes from 37 cohorts and identified 7131 non-missense somatic mutations that are recurrently mutated in at least 50 individuals. These recurrent non-missense somatic mutations (RNMSMs) are not clearly explained by other clonal phenomena such as clonal hematopoiesis. RNMSM prevalence increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37126548"
                        open
                      >
                        <summary class="article-expander__title">The genetic determinants of recurrent somatic mutations in 43,693 blood genomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37126548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230510205718&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Nononcogenic somatic mutations are thought to be uncommon and inconsequential. To test this, we analyzed 43,693 National Heart, Lung and Blood Institute Trans-Omics for Precision Medicine blood whole genomes from 37 cohorts and identified 7131 non-missense somatic mutations that are recurrently mutated in at least 50 individuals. These recurrent non-missense somatic mutations (RNMSMs) are not clearly explained by other clonal phenomena such as clonal hematopoiesis. RNMSM prevalence increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37126548"
                        open
                      >
                        <summary class="article-expander__title">The genetic determinants of recurrent somatic mutations in 43,693 blood genomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37126548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230509205648&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Nononcogenic somatic mutations are thought to be uncommon and inconsequential. To test this, we analyzed 43,693 National Heart, Lung and Blood Institute Trans-Omics for Precision Medicine blood whole genomes from 37 cohorts and identified 7131 non-missense somatic mutations that are recurrently mutated in at least 50 individuals. These recurrent non-missense somatic mutations (RNMSMs) are not clearly explained by other clonal phenomena such as clonal hematopoiesis. RNMSM prevalence increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37126548"
                        open
                      >
                        <summary class="article-expander__title">The genetic determinants of recurrent somatic mutations in 43,693 blood genomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37126548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230508205817&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Nononcogenic somatic mutations are thought to be uncommon and inconsequential. To test this, we analyzed 43,693 National Heart, Lung and Blood Institute Trans-Omics for Precision Medicine blood whole genomes from 37 cohorts and identified 7131 non-missense somatic mutations that are recurrently mutated in at least 50 individuals. These recurrent non-missense somatic mutations (RNMSMs) are not clearly explained by other clonal phenomena such as clonal hematopoiesis. RNMSM prevalence increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37126548"
                        open
                      >
                        <summary class="article-expander__title">The genetic determinants of recurrent somatic mutations in 43,693 blood genomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37126548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230507205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Nononcogenic somatic mutations are thought to be uncommon and inconsequential. To test this, we analyzed 43,693 National Heart, Lung and Blood Institute Trans-Omics for Precision Medicine blood whole genomes from 37 cohorts and identified 7131 non-missense somatic mutations that are recurrently mutated in at least 50 individuals. These recurrent non-missense somatic mutations (RNMSMs) are not clearly explained by other clonal phenomena such as clonal hematopoiesis. RNMSM prevalence increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37126548"
                        open
                      >
                        <summary class="article-expander__title">The genetic determinants of recurrent somatic mutations in 43,693 blood genomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37126548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230506210046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Nononcogenic somatic mutations are thought to be uncommon and inconsequential. To test this, we analyzed 43,693 National Heart, Lung and Blood Institute Trans-Omics for Precision Medicine blood whole genomes from 37 cohorts and identified 7131 non-missense somatic mutations that are recurrently mutated in at least 50 individuals. These recurrent non-missense somatic mutations (RNMSMs) are not clearly explained by other clonal phenomena such as clonal hematopoiesis. RNMSM prevalence increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37126548"
                        open
                      >
                        <summary class="article-expander__title">The genetic determinants of recurrent somatic mutations in 43,693 blood genomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37126548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230505205328&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Nononcogenic somatic mutations are thought to be uncommon and inconsequential. To test this, we analyzed 43,693 National Heart, Lung and Blood Institute Trans-Omics for Precision Medicine blood whole genomes from 37 cohorts and identified 7131 non-missense somatic mutations that are recurrently mutated in at least 50 individuals. These recurrent non-missense somatic mutations (RNMSMs) are not clearly explained by other clonal phenomena such as clonal hematopoiesis. RNMSM prevalence increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37126548"
                        open
                      >
                        <summary class="article-expander__title">The genetic determinants of recurrent somatic mutations in 43,693 blood genomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37126548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230504205311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Nononcogenic somatic mutations are thought to be uncommon and inconsequential. To test this, we analyzed 43,693 National Heart, Lung and Blood Institute Trans-Omics for Precision Medicine blood whole genomes from 37 cohorts and identified 7131 non-missense somatic mutations that are recurrently mutated in at least 50 individuals. These recurrent non-missense somatic mutations (RNMSMs) are not clearly explained by other clonal phenomena such as clonal hematopoiesis. RNMSM prevalence increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37126548"
                        open
                      >
                        <summary class="article-expander__title">The genetic determinants of recurrent somatic mutations in 43,693 blood genomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37126548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230503205635&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Nononcogenic somatic mutations are thought to be uncommon and inconsequential. To test this, we analyzed 43,693 National Heart, Lung and Blood Institute Trans-Omics for Precision Medicine blood whole genomes from 37 cohorts and identified 7131 non-missense somatic mutations that are recurrently mutated in at least 50 individuals. These recurrent non-missense somatic mutations (RNMSMs) are not clearly explained by other clonal phenomena such as clonal hematopoiesis. RNMSM prevalence increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37126548"
                        open
                      >
                        <summary class="article-expander__title">The genetic determinants of recurrent somatic mutations in 43,693 blood genomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37126548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230502205757&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Nononcogenic somatic mutations are thought to be uncommon and inconsequential. To test this, we analyzed 43,693 National Heart, Lung and Blood Institute Trans-Omics for Precision Medicine blood whole genomes from 37 cohorts and identified 7131 non-missense somatic mutations that are recurrently mutated in at least 50 individuals. These recurrent non-missense somatic mutations (RNMSMs) are not clearly explained by other clonal phenomena such as clonal hematopoiesis. RNMSM prevalence increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;16
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37126548"
                        open
                      >
                        <summary class="article-expander__title">The genetic determinants of recurrent somatic mutations in 43,693 blood genomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37126548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230501205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Nononcogenic somatic mutations are thought to be uncommon and inconsequential. To test this, we analyzed 43,693 National Heart, Lung and Blood Institute Trans-Omics for Precision Medicine blood whole genomes from 37 cohorts and identified 7131 non-missense somatic mutations that are recurrently mutated in at least 50 individuals. These recurrent non-missense somatic mutations (RNMSMs) are not clearly explained by other clonal phenomena such as clonal hematopoiesis. RNMSM prevalence increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-04-30 datetime="2023-04-30T10:00:00.000Z">2023-04-30</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-04-30 datetime="2023-04-30T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37121869"
                        open
                      >
                        <summary class="article-expander__title">Mitigating age-related somatic mutation burden</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37121869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomes are inherently unstable and require constant DNA repair to maintain their genetic information. However, selective pressure has optimized repair mechanisms in somatic cells only to allow transmitting genetic information to the next generation, not to maximize sequence integrity long beyond the reproductive age. Recent studies have confirmed that somatic mutations, due to errors during genome repair and replication, accumulate in tissues and organs of humans and model organisms. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37121869"
                        open
                      >
                        <summary class="article-expander__title">Mitigating age-related somatic mutation burden</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37121869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230515205819&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomes are inherently unstable and require constant DNA repair to maintain their genetic information. However, selective pressure has optimized repair mechanisms in somatic cells only to allow transmitting genetic information to the next generation, not to maximize sequence integrity long beyond the reproductive age. Recent studies have confirmed that somatic mutations, due to errors during genome repair and replication, accumulate in tissues and organs of humans and model organisms. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37121869"
                        open
                      >
                        <summary class="article-expander__title">Mitigating age-related somatic mutation burden</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37121869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomes are inherently unstable and require constant DNA repair to maintain their genetic information. However, selective pressure has optimized repair mechanisms in somatic cells only to allow transmitting genetic information to the next generation, not to maximize sequence integrity long beyond the reproductive age. Recent studies have confirmed that somatic mutations, due to errors during genome repair and replication, accumulate in tissues and organs of humans and model organisms. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37121869"
                        open
                      >
                        <summary class="article-expander__title">Mitigating age-related somatic mutation burden</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37121869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomes are inherently unstable and require constant DNA repair to maintain their genetic information. However, selective pressure has optimized repair mechanisms in somatic cells only to allow transmitting genetic information to the next generation, not to maximize sequence integrity long beyond the reproductive age. Recent studies have confirmed that somatic mutations, due to errors during genome repair and replication, accumulate in tissues and organs of humans and model organisms. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37121869"
                        open
                      >
                        <summary class="article-expander__title">Mitigating age-related somatic mutation burden</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37121869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomes are inherently unstable and require constant DNA repair to maintain their genetic information. However, selective pressure has optimized repair mechanisms in somatic cells only to allow transmitting genetic information to the next generation, not to maximize sequence integrity long beyond the reproductive age. Recent studies have confirmed that somatic mutations, due to errors during genome repair and replication, accumulate in tissues and organs of humans and model organisms. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37121869"
                        open
                      >
                        <summary class="article-expander__title">Mitigating age-related somatic mutation burden</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37121869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230511205726&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomes are inherently unstable and require constant DNA repair to maintain their genetic information. However, selective pressure has optimized repair mechanisms in somatic cells only to allow transmitting genetic information to the next generation, not to maximize sequence integrity long beyond the reproductive age. Recent studies have confirmed that somatic mutations, due to errors during genome repair and replication, accumulate in tissues and organs of humans and model organisms. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37121869"
                        open
                      >
                        <summary class="article-expander__title">Mitigating age-related somatic mutation burden</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37121869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230510205718&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomes are inherently unstable and require constant DNA repair to maintain their genetic information. However, selective pressure has optimized repair mechanisms in somatic cells only to allow transmitting genetic information to the next generation, not to maximize sequence integrity long beyond the reproductive age. Recent studies have confirmed that somatic mutations, due to errors during genome repair and replication, accumulate in tissues and organs of humans and model organisms. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37121869"
                        open
                      >
                        <summary class="article-expander__title">Mitigating age-related somatic mutation burden</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37121869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230509205648&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomes are inherently unstable and require constant DNA repair to maintain their genetic information. However, selective pressure has optimized repair mechanisms in somatic cells only to allow transmitting genetic information to the next generation, not to maximize sequence integrity long beyond the reproductive age. Recent studies have confirmed that somatic mutations, due to errors during genome repair and replication, accumulate in tissues and organs of humans and model organisms. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37121869"
                        open
                      >
                        <summary class="article-expander__title">Mitigating age-related somatic mutation burden</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37121869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230508205817&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomes are inherently unstable and require constant DNA repair to maintain their genetic information. However, selective pressure has optimized repair mechanisms in somatic cells only to allow transmitting genetic information to the next generation, not to maximize sequence integrity long beyond the reproductive age. Recent studies have confirmed that somatic mutations, due to errors during genome repair and replication, accumulate in tissues and organs of humans and model organisms. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37121869"
                        open
                      >
                        <summary class="article-expander__title">Mitigating age-related somatic mutation burden</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37121869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230507205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomes are inherently unstable and require constant DNA repair to maintain their genetic information. However, selective pressure has optimized repair mechanisms in somatic cells only to allow transmitting genetic information to the next generation, not to maximize sequence integrity long beyond the reproductive age. Recent studies have confirmed that somatic mutations, due to errors during genome repair and replication, accumulate in tissues and organs of humans and model organisms. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37121869"
                        open
                      >
                        <summary class="article-expander__title">Mitigating age-related somatic mutation burden</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37121869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230506210046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomes are inherently unstable and require constant DNA repair to maintain their genetic information. However, selective pressure has optimized repair mechanisms in somatic cells only to allow transmitting genetic information to the next generation, not to maximize sequence integrity long beyond the reproductive age. Recent studies have confirmed that somatic mutations, due to errors during genome repair and replication, accumulate in tissues and organs of humans and model organisms. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37121869"
                        open
                      >
                        <summary class="article-expander__title">Mitigating age-related somatic mutation burden</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37121869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230505205328&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomes are inherently unstable and require constant DNA repair to maintain their genetic information. However, selective pressure has optimized repair mechanisms in somatic cells only to allow transmitting genetic information to the next generation, not to maximize sequence integrity long beyond the reproductive age. Recent studies have confirmed that somatic mutations, due to errors during genome repair and replication, accumulate in tissues and organs of humans and model organisms. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37121869"
                        open
                      >
                        <summary class="article-expander__title">Mitigating age-related somatic mutation burden</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37121869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230504205311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomes are inherently unstable and require constant DNA repair to maintain their genetic information. However, selective pressure has optimized repair mechanisms in somatic cells only to allow transmitting genetic information to the next generation, not to maximize sequence integrity long beyond the reproductive age. Recent studies have confirmed that somatic mutations, due to errors during genome repair and replication, accumulate in tissues and organs of humans and model organisms. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37121869"
                        open
                      >
                        <summary class="article-expander__title">Mitigating age-related somatic mutation burden</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37121869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230503205635&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomes are inherently unstable and require constant DNA repair to maintain their genetic information. However, selective pressure has optimized repair mechanisms in somatic cells only to allow transmitting genetic information to the next generation, not to maximize sequence integrity long beyond the reproductive age. Recent studies have confirmed that somatic mutations, due to errors during genome repair and replication, accumulate in tissues and organs of humans and model organisms. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37121869"
                        open
                      >
                        <summary class="article-expander__title">Mitigating age-related somatic mutation burden</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37121869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230502205757&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomes are inherently unstable and require constant DNA repair to maintain their genetic information. However, selective pressure has optimized repair mechanisms in somatic cells only to allow transmitting genetic information to the next generation, not to maximize sequence integrity long beyond the reproductive age. Recent studies have confirmed that somatic mutations, due to errors during genome repair and replication, accumulate in tissues and organs of humans and model organisms. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37121869"
                        open
                      >
                        <summary class="article-expander__title">Mitigating age-related somatic mutation burden</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37121869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230501205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomes are inherently unstable and require constant DNA repair to maintain their genetic information. However, selective pressure has optimized repair mechanisms in somatic cells only to allow transmitting genetic information to the next generation, not to maximize sequence integrity long beyond the reproductive age. Recent studies have confirmed that somatic mutations, due to errors during genome repair and replication, accumulate in tissues and organs of humans and model organisms. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-04-29 datetime="2023-04-29T10:00:00.000Z">2023-04-29</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-04-29 datetime="2023-04-29T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37119011"
                        open
                      >
                        <summary class="article-expander__title">VEXAS: is it time to reshape the nosology of clonal hematopoiesis?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37119011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230516205931&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The recent description of VEXAS (for vacuoles, E1 enzyme, X-linked, autoinflammatory, somatic) challenges the nosological framework of myelodysplastic syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37119011"
                        open
                      >
                        <summary class="article-expander__title">VEXAS: is it time to reshape the nosology of clonal hematopoiesis?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37119011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230515205819&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The recent description of VEXAS (for vacuoles, E1 enzyme, X-linked, autoinflammatory, somatic) challenges the nosological framework of myelodysplastic syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37119011"
                        open
                      >
                        <summary class="article-expander__title">VEXAS: is it time to reshape the nosology of clonal hematopoiesis?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37119011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The recent description of VEXAS (for vacuoles, E1 enzyme, X-linked, autoinflammatory, somatic) challenges the nosological framework of myelodysplastic syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37119011"
                        open
                      >
                        <summary class="article-expander__title">VEXAS: is it time to reshape the nosology of clonal hematopoiesis?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37119011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The recent description of VEXAS (for vacuoles, E1 enzyme, X-linked, autoinflammatory, somatic) challenges the nosological framework of myelodysplastic syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37119011"
                        open
                      >
                        <summary class="article-expander__title">VEXAS: is it time to reshape the nosology of clonal hematopoiesis?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37119011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The recent description of VEXAS (for vacuoles, E1 enzyme, X-linked, autoinflammatory, somatic) challenges the nosological framework of myelodysplastic syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37119011"
                        open
                      >
                        <summary class="article-expander__title">VEXAS: is it time to reshape the nosology of clonal hematopoiesis?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37119011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230511205726&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The recent description of VEXAS (for vacuoles, E1 enzyme, X-linked, autoinflammatory, somatic) challenges the nosological framework of myelodysplastic syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37119011"
                        open
                      >
                        <summary class="article-expander__title">VEXAS: is it time to reshape the nosology of clonal hematopoiesis?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37119011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230510205718&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The recent description of VEXAS (for vacuoles, E1 enzyme, X-linked, autoinflammatory, somatic) challenges the nosological framework of myelodysplastic syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37119011"
                        open
                      >
                        <summary class="article-expander__title">VEXAS: is it time to reshape the nosology of clonal hematopoiesis?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37119011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230509205648&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The recent description of VEXAS (for vacuoles, E1 enzyme, X-linked, autoinflammatory, somatic) challenges the nosological framework of myelodysplastic syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37119011"
                        open
                      >
                        <summary class="article-expander__title">VEXAS: is it time to reshape the nosology of clonal hematopoiesis?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37119011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230508205817&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The recent description of VEXAS (for vacuoles, E1 enzyme, X-linked, autoinflammatory, somatic) challenges the nosological framework of myelodysplastic syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37119011"
                        open
                      >
                        <summary class="article-expander__title">VEXAS: is it time to reshape the nosology of clonal hematopoiesis?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37119011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230507205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The recent description of VEXAS (for vacuoles, E1 enzyme, X-linked, autoinflammatory, somatic) challenges the nosological framework of myelodysplastic syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37119011"
                        open
                      >
                        <summary class="article-expander__title">VEXAS: is it time to reshape the nosology of clonal hematopoiesis?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37119011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230506210046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The recent description of VEXAS (for vacuoles, E1 enzyme, X-linked, autoinflammatory, somatic) challenges the nosological framework of myelodysplastic syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37119011"
                        open
                      >
                        <summary class="article-expander__title">VEXAS: is it time to reshape the nosology of clonal hematopoiesis?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37119011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230505205328&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The recent description of VEXAS (for vacuoles, E1 enzyme, X-linked, autoinflammatory, somatic) challenges the nosological framework of myelodysplastic syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37119011"
                        open
                      >
                        <summary class="article-expander__title">VEXAS: is it time to reshape the nosology of clonal hematopoiesis?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37119011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230504205311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The recent description of VEXAS (for vacuoles, E1 enzyme, X-linked, autoinflammatory, somatic) challenges the nosological framework of myelodysplastic syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37119011"
                        open
                      >
                        <summary class="article-expander__title">VEXAS: is it time to reshape the nosology of clonal hematopoiesis?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37119011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230503205635&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The recent description of VEXAS (for Vacuoles, E1 enzyme, X-linked, Autoinflammatory, Somatic) challenges the nosological framework of myelodysplastic syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37119011"
                        open
                      >
                        <summary class="article-expander__title">VEXAS: is it time to reshape the nosology of clonal hematopoiesis?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37119011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230502205757&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The recent description of VEXAS (for Vacuoles, E1 enzyme, X-linked, Autoinflammatory, Somatic) challenges the nosological framework of myelodysplastic syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37119011"
                        open
                      >
                        <summary class="article-expander__title">VEXAS: is it time to reshape the nosology of clonal hematopoiesis?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37119011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230501205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The recent description of VEXAS (for Vacuoles, E1 enzyme, X-linked, Autoinflammatory, Somatic) challenges the nosological framework of myelodysplastic syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37119011"
                        open
                      >
                        <summary class="article-expander__title">VEXAS: is it time to reshape the nosology of clonal hematopoiesis?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37119011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230430210026&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The recent description of VEXAS (for Vacuoles, E1 enzyme, X-linked, Autoinflammatory, Somatic) challenges the nosological framework of myelodysplastic syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37119011"
                        open
                      >
                        <summary class="article-expander__title">VEXAS: is it time to reshape the nosology of clonal hematopoiesis?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37119011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230429210234&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: The recent description of VEXAS (for Vacuoles, E1 enzyme, X-linked, Autoinflammatory, Somatic) challenges the nosological framework of myelodysplastic syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-04-28 datetime="2023-04-28T10:00:00.000Z">2023-04-28</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-04-28 datetime="2023-04-28T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;ff&#x3D;20230625211348&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37118760"
                        open
                      >
                        <summary class="article-expander__title">New approaches to idiopathic neutropenia in the era of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37118760/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230515205819&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Isolated chronic idiopathic neutropenia (CIN) is a rare disease with multiple contributing etiologies that must be ruled out before establishing a diagnosis. We studied clinical and molecular data of 238 consecutive adult patients with CIN. Autoimmune neutropenia was present in 28% of our cohort. In contrast, T cell-mediated neutropenia was the main underlying pathological mechanism among patients with T cell expansions, such as T-cell large granular lymphocytic leukemia (T-LGL) and T cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37117740"
                        open
                      >
                        <summary class="article-expander__title">The burden of rare protein-truncating genetic variants on human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37117740/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230515205819&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic predisposition has been shown to contribute substantially to the age at which we die. Genome-wide association studies (GWASs) have linked more than 20 loci to phenotypes related to human lifespan¹. However, little is known about how lifespan is impacted by gene loss of function. Through whole-exome sequencing of 352,338 UK Biobank participants of European ancestry, we assessed the relevance of protein-truncating variant (PTV) gene burden on individual and parental survival. We identified...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37118760"
                        open
                      >
                        <summary class="article-expander__title">New approaches to idiopathic neutropenia in the era of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37118760/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Isolated chronic idiopathic neutropenia (CIN) is a rare disease with multiple contributing etiologies that must be ruled out before establishing a diagnosis. We studied clinical and molecular data of 238 consecutive adult patients with CIN. Autoimmune neutropenia was present in 28% of our cohort. In contrast, T cell-mediated neutropenia was the main underlying pathological mechanism among patients with T cell expansions, such as T-cell large granular lymphocytic leukemia (T-LGL) and T cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37117740"
                        open
                      >
                        <summary class="article-expander__title">The burden of rare protein-truncating genetic variants on human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37117740/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230514205932&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic predisposition has been shown to contribute substantially to the age at which we die. Genome-wide association studies (GWASs) have linked more than 20 loci to phenotypes related to human lifespan¹. However, little is known about how lifespan is impacted by gene loss of function. Through whole-exome sequencing of 352,338 UK Biobank participants of European ancestry, we assessed the relevance of protein-truncating variant (PTV) gene burden on individual and parental survival. We identified...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37118760"
                        open
                      >
                        <summary class="article-expander__title">New approaches to idiopathic neutropenia in the era of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37118760/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Isolated chronic idiopathic neutropenia (CIN) is a rare disease with multiple contributing etiologies that must be ruled out before establishing a diagnosis. We studied clinical and molecular data of 238 consecutive adult patients with CIN. Autoimmune neutropenia was present in 28% of our cohort. In contrast, T cell-mediated neutropenia was the main underlying pathological mechanism among patients with T cell expansions, such as T-cell large granular lymphocytic leukemia (T-LGL) and T cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37117740"
                        open
                      >
                        <summary class="article-expander__title">The burden of rare protein-truncating genetic variants on human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37117740/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230513210018&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic predisposition has been shown to contribute substantially to the age at which we die. Genome-wide association studies (GWASs) have linked more than 20 loci to phenotypes related to human lifespan¹. However, little is known about how lifespan is impacted by gene loss of function. Through whole-exome sequencing of 352,338 UK Biobank participants of European ancestry, we assessed the relevance of protein-truncating variant (PTV) gene burden on individual and parental survival. We identified...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37118760"
                        open
                      >
                        <summary class="article-expander__title">New approaches to idiopathic neutropenia in the era of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37118760/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Isolated chronic idiopathic neutropenia (CIN) is a rare disease with multiple contributing etiologies that must be ruled out before establishing a diagnosis. We studied clinical and molecular data of 238 consecutive adult patients with CIN. Autoimmune neutropenia was present in 28% of our cohort. In contrast, T cell-mediated neutropenia was the main underlying pathological mechanism among patients with T cell expansions, such as T-cell large granular lymphocytic leukemia (T-LGL) and T cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37117740"
                        open
                      >
                        <summary class="article-expander__title">The burden of rare protein-truncating genetic variants on human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37117740/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic predisposition has been shown to contribute substantially to the age at which we die. Genome-wide association studies (GWASs) have linked more than 20 loci to phenotypes related to human lifespan¹. However, little is known about how lifespan is impacted by gene loss of function. Through whole-exome sequencing of 352,338 UK Biobank participants of European ancestry, we assessed the relevance of protein-truncating variant (PTV) gene burden on individual and parental survival. We identified...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37115512"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Young Women Treated for Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37115512/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230512205523&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our data show that for young women with early-stage breast cancer, CHIP is uncommon after cytotoxic exposure, is unlikely to contribute to adverse outcomes over the decade-long follow up and may not require additional monitoring if discovered incidentally.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37118760"
                        open
                      >
                        <summary class="article-expander__title">New approaches to idiopathic neutropenia in the era of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37118760/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230511205726&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Isolated chronic idiopathic neutropenia (CIN) is a rare disease with multiple contributing etiologies that must be ruled out before establishing a diagnosis. We studied clinical and molecular data of 238 consecutive adult patients with CIN. Autoimmune neutropenia was present in 28% of our cohort. In contrast, T cell-mediated neutropenia was the main underlying pathological mechanism among patients with T cell expansions, such as T-cell large granular lymphocytic leukemia (T-LGL) and T cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37117740"
                        open
                      >
                        <summary class="article-expander__title">The burden of rare protein-truncating genetic variants on human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37117740/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230511205726&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic predisposition has been shown to contribute substantially to the age at which we die. Genome-wide association studies (GWASs) have linked more than 20 loci to phenotypes related to human lifespan¹. However, little is known about how lifespan is impacted by gene loss of function. Through whole-exome sequencing of 352,338 UK Biobank participants of European ancestry, we assessed the relevance of protein-truncating variant (PTV) gene burden on individual and parental survival. We identified...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37115512"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Young Women Treated for Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37115512/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230511205726&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our data show that for young women with early-stage breast cancer, CHIP is uncommon after cytotoxic exposure, is unlikely to contribute to adverse outcomes over the decade-long follow up and may not require additional monitoring if discovered incidentally.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37118760"
                        open
                      >
                        <summary class="article-expander__title">New approaches to idiopathic neutropenia in the era of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37118760/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230510205718&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Isolated chronic idiopathic neutropenia (CIN) is a rare disease with multiple contributing etiologies that must be ruled out before establishing a diagnosis. We studied clinical and molecular data of 238 consecutive adult patients with CIN. Autoimmune neutropenia was present in 28% of our cohort. In contrast, T cell-mediated neutropenia was the main underlying pathological mechanism among patients with T cell expansions, such as T-cell large granular lymphocytic leukemia (T-LGL) and T cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37117740"
                        open
                      >
                        <summary class="article-expander__title">The burden of rare protein-truncating genetic variants on human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37117740/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230510205718&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic predisposition has been shown to contribute substantially to the age at which we die. Genome-wide association studies (GWASs) have linked more than 20 loci to phenotypes related to human lifespan¹. However, little is known about how lifespan is impacted by gene loss of function. Through whole-exome sequencing of 352,338 UK Biobank participants of European ancestry, we assessed the relevance of protein-truncating variant (PTV) gene burden on individual and parental survival. We identified...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37115512"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Young Women Treated for Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37115512/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230510205718&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our data show that for young women with early-stage breast cancer, CHIP is uncommon after cytotoxic exposure, is unlikely to contribute to adverse outcomes over the decade-long follow up and may not require additional monitoring if discovered incidentally.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37118760"
                        open
                      >
                        <summary class="article-expander__title">New approaches to idiopathic neutropenia in the era of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37118760/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230509205648&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Isolated chronic idiopathic neutropenia (CIN) is a rare disease with multiple contributing etiologies that must be ruled out before establishing a diagnosis. We studied clinical and molecular data of 238 consecutive adult patients with CIN. Autoimmune neutropenia was present in 28% of our cohort. In contrast, T cell-mediated neutropenia was the main underlying pathological mechanism among patients with T cell expansions, such as T-cell large granular lymphocytic leukemia (T-LGL) and T cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37117740"
                        open
                      >
                        <summary class="article-expander__title">The burden of rare protein-truncating genetic variants on human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37117740/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230509205648&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic predisposition has been shown to contribute substantially to the age at which we die. Genome-wide association studies (GWASs) have linked more than 20 loci to phenotypes related to human lifespan¹. However, little is known about how lifespan is impacted by gene loss of function. Through whole-exome sequencing of 352,338 UK Biobank participants of European ancestry, we assessed the relevance of protein-truncating variant (PTV) gene burden on individual and parental survival. We identified...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37115512"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Young Women Treated for Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37115512/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230509205648&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our data show that for young women with early-stage breast cancer, CHIP is uncommon after cytotoxic exposure, is unlikely to contribute to adverse outcomes over the decade-long follow up and may not require additional monitoring if discovered incidentally.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37118760"
                        open
                      >
                        <summary class="article-expander__title">New approaches to idiopathic neutropenia in the era of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37118760/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230508205817&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Isolated chronic idiopathic neutropenia (CIN) is a rare disease with multiple contributing etiologies that must be ruled out before establishing a diagnosis. We studied clinical and molecular data of 238 consecutive adult patients with CIN. Autoimmune neutropenia was present in 28% of our cohort. In contrast, T cell-mediated neutropenia was the main underlying pathological mechanism among patients with T cell expansions, such as T-cell large granular lymphocytic leukemia (T-LGL) and T cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37117740"
                        open
                      >
                        <summary class="article-expander__title">The burden of rare protein-truncating genetic variants on human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37117740/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230508205817&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic predisposition has been shown to contribute substantially to the age at which we die. Genome-wide association studies (GWASs) have linked more than 20 loci to phenotypes related to human lifespan¹. However, little is known about how lifespan is impacted by gene loss of function. Through whole-exome sequencing of 352,338 UK Biobank participants of European ancestry, we assessed the relevance of protein-truncating variant (PTV) gene burden on individual and parental survival. We identified...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37115512"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Young Women Treated for Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37115512/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230508205817&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our data show that for young women with early-stage breast cancer, CHIP is uncommon after cytotoxic exposure, is unlikely to contribute to adverse outcomes over the decade-long follow up and may not require additional monitoring if discovered incidentally.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37118760"
                        open
                      >
                        <summary class="article-expander__title">New approaches to idiopathic neutropenia in the era of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37118760/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230507205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Isolated chronic idiopathic neutropenia (CIN) is a rare disease with multiple contributing etiologies that must be ruled out before establishing a diagnosis. We studied clinical and molecular data of 238 consecutive adult patients with CIN. Autoimmune neutropenia was present in 28% of our cohort. In contrast, T cell-mediated neutropenia was the main underlying pathological mechanism among patients with T cell expansions, such as T-cell large granular lymphocytic leukemia (T-LGL) and T cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37117740"
                        open
                      >
                        <summary class="article-expander__title">The burden of rare protein-truncating genetic variants on human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37117740/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230507205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic predisposition has been shown to contribute substantially to the age at which we die. Genome-wide association studies (GWASs) have linked more than 20 loci to phenotypes related to human lifespan¹. However, little is known about how lifespan is impacted by gene loss of function. Through whole-exome sequencing of 352,338 UK Biobank participants of European ancestry, we assessed the relevance of protein-truncating variant (PTV) gene burden on individual and parental survival. We identified...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37115512"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Young Women Treated for Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37115512/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230507205525&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our data show that for young women with early-stage breast cancer, CHIP is uncommon after cytotoxic exposure, is unlikely to contribute to adverse outcomes over the decade-long follow up and may not require additional monitoring if discovered incidentally.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37118760"
                        open
                      >
                        <summary class="article-expander__title">New approaches to idiopathic neutropenia in the era of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37118760/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230506210046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Isolated chronic idiopathic neutropenia (CIN) is a rare disease with multiple contributing etiologies that must be ruled out before establishing a diagnosis. We studied clinical and molecular data of 238 consecutive adult patients with CIN. Autoimmune neutropenia was present in 28% of our cohort. In contrast, T cell-mediated neutropenia was the main underlying pathological mechanism among patients with T cell expansions, such as T-cell large granular lymphocytic leukemia (T-LGL) and T cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37117740"
                        open
                      >
                        <summary class="article-expander__title">The burden of rare protein-truncating genetic variants on human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37117740/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230506210046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic predisposition has been shown to contribute substantially to the age at which we die. Genome-wide association studies (GWASs) have linked more than 20 loci to phenotypes related to human lifespan¹. However, little is known about how lifespan is impacted by gene loss of function. Through whole-exome sequencing of 352,338 UK Biobank participants of European ancestry, we assessed the relevance of protein-truncating variant (PTV) gene burden on individual and parental survival. We identified...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37115512"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Young Women Treated for Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37115512/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230506210046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our data show that for young women with early-stage breast cancer, CHIP is uncommon after cytotoxic exposure, is unlikely to contribute to adverse outcomes over the decade-long follow up and may not require additional monitoring if discovered incidentally.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37118760"
                        open
                      >
                        <summary class="article-expander__title">New approaches to idiopathic neutropenia in the era of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37118760/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230505205328&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Isolated chronic idiopathic neutropenia (CIN) is a rare disease with multiple contributing etiologies that must be ruled out before establishing a diagnosis. We studied clinical and molecular data of 238 consecutive adult patients with CIN. Autoimmune neutropenia was present in 28% of our cohort. In contrast, T cell-mediated neutropenia was the main underlying pathological mechanism among patients with T cell expansions, such as T-cell large granular lymphocytic leukemia (T-LGL) and T cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37117740"
                        open
                      >
                        <summary class="article-expander__title">The burden of rare protein-truncating genetic variants on human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37117740/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230505205328&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic predisposition has been shown to contribute substantially to the age at which we die. Genome-wide association studies (GWASs) have linked more than 20 loci to phenotypes related to human lifespan¹. However, little is known about how lifespan is impacted by gene loss of function. Through whole-exome sequencing of 352,338 UK Biobank participants of European ancestry, we assessed the relevance of protein-truncating variant (PTV) gene burden on individual and parental survival. We identified...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37115512"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Young Women Treated for Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37115512/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230505205328&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our data show that for young women with early-stage breast cancer, CHIP is uncommon after cytotoxic exposure, is unlikely to contribute to adverse outcomes over the decade-long follow up and may not require additional monitoring if discovered incidentally.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37118760"
                        open
                      >
                        <summary class="article-expander__title">New approaches to idiopathic neutropenia in the era of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37118760/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230504205311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Isolated chronic idiopathic neutropenia (CIN) is a rare disease with multiple contributing etiologies that must be ruled out before establishing a diagnosis. We studied clinical and molecular data of 238 consecutive adult patients with CIN. Autoimmune neutropenia was present in 28% of our cohort. In contrast, T cell-mediated neutropenia was the main underlying pathological mechanism among patients with T cell expansions, such as T-cell large granular lymphocytic leukemia (T-LGL) and T cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37117740"
                        open
                      >
                        <summary class="article-expander__title">The burden of rare protein-truncating genetic variants on human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37117740/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230504205311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic predisposition has been shown to contribute substantially to the age at which we die. Genome-wide association studies (GWASs) have linked more than 20 loci to phenotypes related to human lifespan¹. However, little is known about how lifespan is impacted by gene loss of function. Through whole-exome sequencing of 352,338 UK Biobank participants of European ancestry, we assessed the relevance of protein-truncating variant (PTV) gene burden on individual and parental survival. We identified...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37115512"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Young Women Treated for Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37115512/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230504205311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our data show that for young women with early-stage breast cancer, CHIP is uncommon after cytotoxic exposure, is unlikely to contribute to adverse outcomes over the decade-long follow up and may not require additional monitoring if discovered incidentally.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37118760"
                        open
                      >
                        <summary class="article-expander__title">New approaches to idiopathic neutropenia in the era of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37118760/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230503205635&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Isolated chronic idiopathic neutropenia (CIN) is a rare disease with multiple contributing etiologies that must be ruled out before establishing a diagnosis. We studied clinical and molecular data of 238 consecutive adult patients with CIN. Autoimmune neutropenia was present in 28% of our cohort. In contrast, T cell-mediated neutropenia was the main underlying pathological mechanism among patients with T cell expansions, such as T-cell large granular lymphocytic leukemia (T-LGL) and T cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37117740"
                        open
                      >
                        <summary class="article-expander__title">The burden of rare protein-truncating genetic variants on human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37117740/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230503205635&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic predisposition has been shown to contribute substantially to the age at which we die. Genome-wide association studies (GWASs) have linked more than 20 loci to phenotypes related to human lifespan¹. However, little is known about how lifespan is impacted by gene loss of function. Through whole-exome sequencing of 352,338 UK Biobank participants of European ancestry, we assessed the relevance of protein-truncating variant (PTV) gene burden on individual and parental survival. We identified...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37115512"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Young Women Treated for Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37115512/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230503205635&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our data show that for young women with early-stage breast cancer, CHIP is uncommon after cytotoxic exposure, is unlikely to contribute to adverse outcomes over the decade-long follow up and may not require additional monitoring if discovered incidentally.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37118760"
                        open
                      >
                        <summary class="article-expander__title">New approaches to idiopathic neutropenia in the era of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37118760/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230502205757&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Isolated chronic idiopathic neutropenia (CIN) is a rare disease with multiple contributing etiologies that must be ruled out before establishing a diagnosis. We studied clinical and molecular data of 238 consecutive adult patients with CIN. Autoimmune neutropenia was present in 28% of our cohort. In contrast, T cell-mediated neutropenia was the main underlying pathological mechanism among patients with T cell expansions, such as T-cell large granular lymphocytic leukemia (T-LGL) and T cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37117740"
                        open
                      >
                        <summary class="article-expander__title">The burden of rare protein-truncating genetic variants on human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37117740/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230502205757&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic predisposition has been shown to contribute substantially to the age at which we die. Genome-wide association studies (GWASs) have linked more than 20 loci to phenotypes related to human lifespan¹. However, little is known about how lifespan is impacted by gene loss of function. Through whole-exome sequencing of 352,338 UK Biobank participants of European ancestry, we assessed the relevance of protein-truncating variant (PTV) gene burden on individual and parental survival. We identified...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37115512"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Young Women Treated for Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37115512/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230502205757&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our data show that for young women with early-stage breast cancer, CHIP is uncommon after cytotoxic exposure, is unlikely to contribute to adverse outcomes over the decade-long follow up and may not require additional monitoring if discovered incidentally.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37118760"
                        open
                      >
                        <summary class="article-expander__title">New approaches to idiopathic neutropenia in the era of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37118760/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230501205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Isolated chronic idiopathic neutropenia (CIN) is a rare disease with multiple contributing etiologies that must be ruled out before establishing a diagnosis. We studied clinical and molecular data of 238 consecutive adult patients with CIN. Autoimmune neutropenia was present in 28% of our cohort. In contrast, T cell-mediated neutropenia was the main underlying pathological mechanism among patients with T cell expansions, such as T-cell large granular lymphocytic leukemia (T-LGL) and T cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37117740"
                        open
                      >
                        <summary class="article-expander__title">The burden of rare protein-truncating genetic variants on human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37117740/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230501205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic predisposition has been shown to contribute substantially to the age at which we die. Genome-wide association studies (GWASs) have linked more than 20 loci to phenotypes related to human lifespan¹. However, little is known about how lifespan is impacted by gene loss of function. Through whole-exome sequencing of 352,338 UK Biobank participants of European ancestry, we assessed the relevance of protein-truncating variant (PTV) gene burden on individual and parental survival. We identified...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37115512"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Young Women Treated for Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37115512/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230501205822&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our data show that for young women with early-stage breast cancer, CHIP is uncommon after cytotoxic exposure, is unlikely to contribute to adverse outcomes over the decade-long follow up and may not require additional monitoring if discovered incidentally.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37118760"
                        open
                      >
                        <summary class="article-expander__title">New approaches to idiopathic neutropenia in the era of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37118760/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230430210026&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Isolated chronic idiopathic neutropenia (CIN) is a rare disease with multiple contributing etiologies that must be ruled out before establishing a diagnosis. We studied clinical and molecular data of 238 consecutive adult patients with CIN. Autoimmune neutropenia was present in 28% of our cohort. In contrast, T cell-mediated neutropenia was the main underlying pathological mechanism among patients with T cell expansions, such as T-cell large granular lymphocytic leukemia (T-LGL) and T cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37117740"
                        open
                      >
                        <summary class="article-expander__title">The burden of rare protein-truncating genetic variants on human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37117740/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230430210026&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic predisposition has been shown to contribute substantially to the age at which we die. Genome-wide association studies (GWASs) have linked more than 20 loci to phenotypes related to human lifespan¹. However, little is known about how lifespan is impacted by gene loss of function. Through whole-exome sequencing of 352,338 UK Biobank participants of European ancestry, we assessed the relevance of protein-truncating variant (PTV) gene burden on individual and parental survival. We identified...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37115512"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Young Women Treated for Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37115512/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230430210026&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our data show that for young women with early-stage breast cancer, CHIP is uncommon after cytotoxic exposure, is unlikely to contribute to adverse outcomes over the decade-long follow up and may not require additional monitoring if discovered incidentally.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37118760"
                        open
                      >
                        <summary class="article-expander__title">New approaches to idiopathic neutropenia in the era of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37118760/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230429210234&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Isolated chronic idiopathic neutropenia (CIN) is a rare disease with multiple contributing etiologies that must be ruled out before establishing a diagnosis. We studied clinical and molecular data of 238 consecutive adult patients with CIN. Autoimmune neutropenia was present in 28% of our cohort. In contrast, T cell-mediated neutropenia was the main underlying pathological mechanism among patients with T cell expansions, such as T-cell large granular lymphocytic leukemia (T-LGL) and T cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37117740"
                        open
                      >
                        <summary class="article-expander__title">The burden of rare protein-truncating genetic variants on human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37117740/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230429210234&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic predisposition has been shown to contribute substantially to the age at which we die. Genome-wide association studies (GWASs) have linked more than 20 loci to phenotypes related to human lifespan¹. However, little is known about how lifespan is impacted by gene loss of function. Through whole-exome sequencing of 352,338 UK Biobank participants of European ancestry, we assessed the relevance of protein-truncating variant (PTV) gene burden on individual and parental survival. We identified...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37115512"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Young Women Treated for Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37115512/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230429210234&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our data show that for young women with early-stage breast cancer, CHIP is uncommon after cytotoxic exposure, is unlikely to contribute to adverse outcomes over the decade-long follow up and may not require additional monitoring if discovered incidentally.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37115512"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Young Women Treated for Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37115512/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230428205526&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our data show that for young women with early-stage breast cancer, CHIP is uncommon after cytotoxic exposure, is unlikely to contribute to adverse outcomes over the decade-long follow up and may not require additional monitoring if discovered incidentally.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-06 datetime="2022-06-06T18:28:04.000Z">2022-06-06</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-06 datetime="2022-06-06T18:28:04.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="72a3a11d39c217b1a80cbb551444e6f6"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Nodal interferon exposure drives tumour-specific tolerance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1533878403463405569">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Nodal interferon exposure drives tumour-specific tolerance https://t.co/FT4NeBcFTg pic.twitter.com/NRHkVDUsOr
— NatureReviewsCancer (@NatureRevCancer) June 6, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-30 datetime="2022-05-30T04:37:42.000Z">2022-05-30</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-30 datetime="2022-05-30T04:37:42.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="bb6bc242bcbc356c4a38db850e74d812"
                        open
                      >
                        <summary class="article-expander__title">@VIBConferences: #Cancer #immunotherapy has become a crucial pillar of anticancer therapy. 19 leading researchers at #ImmunoOnco22 will give new insights into the immune response to tumors and tackle unresolved questions.⏰Registration Deadline: Thursday 2 June. Read more</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/VIBConferences/status/1531132720952266752">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>#Cancer #immunotherapy has become a crucial pillar of anticancer therapy. 19 leading researchers at #ImmunoOnco22 will give new insights into the immune response to tumors and tackle unresolved questions.
⏰Registration Deadline: Thursday 2 June. Read more https://t.co/cc0HC5cj6W pic.twitter.com/u8LFuzBbsV
— VIB Conferences (@VIBConferences) May 30, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-27 datetime="2022-05-27T14:45:03.000Z">2022-05-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-27 datetime="2022-05-27T14:45:03.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="ed6c996dc1a718afb523de6229a53131"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Nonsense-mediated RNA decay: an emerging modulator of malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1530198403686072320">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Nonsense-mediated RNA decay: an emerging modulator of malignancy https://t.co/wjuHplkpof pic.twitter.com/1SoKYhRsPO
— NatureReviewsCancer (@NatureRevCancer) May 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="d15a3a4b24853812d7fa21fd56de5815"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: Our June issue, out NOW! With articles on #therapyresistance, #senescence &amp; targeting CDK4 &amp; CDK6 in #cancer! Plus, a Comment on cancer mRNA vaccines as well as the launch of our new Tools of the Trade articles #NRCToT. So much content 👇🏻💌</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1530174481313972224">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Our June issue, out NOW! With articles on #therapyresistance, #senescence &amp; targeting CDK4 &amp; CDK6 in #cancer! Plus, a Comment on cancer mRNA vaccines as well as the launch of our new Tools of the Trade articles #NRCToT. So much content 👇🏻
💌 https://t.co/Fo6XRWV3OZ@natrescancer pic.twitter.com/rbxRPqLV6g
— NatureReviewsCancer (@NatureRevCancer) May 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-26 datetime="2022-05-26T08:53:50.000Z">2022-05-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-26 datetime="2022-05-26T08:53:50.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="fe13755a2693ea65459e5e21df097d2a"
                        open
                      >
                        <summary class="article-expander__title">@UlrikeHarjes: This Review by @MarcusDGon @cantleylab is part of the @NatureRevCancer series on #Diet &amp; #SystemicMetabolism in cancer:  do love this topic in general plus I am handling research at @NatureMedicine now.. get in touch if you have something of interest☺️</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/UlrikeHarjes/status/1529747629936193537">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>This Review by @MarcusDGon @cantleylab is part of the @NatureRevCancer series on #Diet &amp; #SystemicMetabolism in cancer: https://t.co/p3Y2cdNhco
I do love this topic in general plus I am handling research at @NatureMedicine now.. get in touch if you have something of interest☺️ https://t.co/qWH2dTv8j9
— Ulrike Harjes (@UlrikeHarjes) May 26, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-25 datetime="2022-05-25T17:35:04.000Z">2022-05-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-25 datetime="2022-05-25T17:35:04.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="9ba5279424b31b0122a4ae20c18e6d03"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Developing dietary interventions as therapy for cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1529516411059437568">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Developing dietary interventions as therapy for cancer https://t.co/ijuqctk1wJ pic.twitter.com/KHPpMKnCFa
— NatureReviewsCancer (@NatureRevCancer) May 25, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-18 datetime="2022-05-18T14:11:32.000Z">2022-05-18</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-18 datetime="2022-05-18T14:11:32.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="611d40f2e6ea2796b028bebda41a86dd"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Digging for treasures in the tumour interactome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1526928478490095616">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Digging for treasures in the tumour interactome https://t.co/NhHYAwUgLE
— NatureReviewsCancer (@NatureRevCancer) May 18, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-17 datetime="2022-05-17T13:48:21.000Z">2022-05-17</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-17 datetime="2022-05-17T13:48:21.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="9590a971b00448cd312078183965b21d"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: As part of our overall commitment to diversity, equity and inclusion at Springer Nature, we’d like to better understand what DEI means within the global research community. Take the survey here</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1526560254091313156">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>As part of our overall commitment to diversity, equity and inclusion at Springer Nature, we’d like to better understand what DEI means within the global research community. Take the survey here https://t.co/03yLiUxFTq
— NatureReviewsCancer (@NatureRevCancer) May 17, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-12 datetime="2022-05-12T15:25:46.000Z">2022-05-12</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-12 datetime="2022-05-12T15:25:46.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="aa38ff4245b93e964773f191844ddc7a"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: Newsflash! We have a new article type in the journal - Tools of the Trade #NRCToT 🧰 🛠️ where we hope to enable more #ECRs to interact with the editorial team &amp; have a bigger voice in our content. For more info, check out the Editorial 👇🏻</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1524772831904509953">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Newsflash! We have a new article type in the journal - Tools of the Trade #NRCToT 🧰 🛠️ where we hope to enable more #ECRs to interact with the editorial team &amp; have a bigger voice in our content. For more info, check out the Editorial 👇🏻 https://t.co/VbyoUJmTRJ
— NatureReviewsCancer (@NatureRevCancer) May 12, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-10 datetime="2022-05-10T16:46:03.000Z">2022-05-10</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-10 datetime="2022-05-10T16:46:03.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="65aca76b9f794aaa668d84ad32dca4a0"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Tools for the next generation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1524068259712036864">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Tools for the next generation https://t.co/lEzTPFYDf5
— NatureReviewsCancer (@NatureRevCancer) May 10, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="afa9b6784b4ccc7c05ed4efd26ddfb2d"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Hunting down rare drug-tolerant cycling cells with Watermelon</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1524022709386190856">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Hunting down rare drug-tolerant cycling cells with Watermelon https://t.co/XXT0c9SAYX
— NatureReviewsCancer (@NatureRevCancer) May 10, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-29 datetime="2022-04-29T14:40:05.000Z">2022-04-29</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-29 datetime="2022-04-29T14:40:05.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="43e2d5ccebc9030f737d656f47ac9d98"
                        open
                      >
                        <summary class="article-expander__title">@VarunVenkatara2: Very happy that our latest Perspective is now online @NatureRevCancer ! We discuss implications of malignant, multicellular networks and potential therapeutic strategies how to disconnect them. Thanks to everyone involved ! #CancerNeuroscience #TumorMicrotubes #neurotwitter</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/VarunVenkatara2/status/1520050291986604033">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Very happy that our latest Perspective is now online @NatureRevCancer ! We discuss implications of malignant, multicellular networks and potential therapeutic strategies how to disconnect them. Thanks to everyone involved ! #CancerNeuroscience #TumorMicrotubes #neurotwitter https://t.co/S55cKTGNFd
— Varun Venkataramani (@VarunVenkatara2) April 29, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="9f6b8644378881f4c2e6b6833c93fc2f"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! An inhospitable site</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1520020349068316673">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! An inhospitable site https://t.co/0dvqIKCWxh pic.twitter.com/PtY8J9Fz7r
— NatureReviewsCancer (@NatureRevCancer) April 29, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="f6d91efcd60dae769e2722585bdfa8df"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Disconnecting multicellular networks in brain tumours</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1520020340201598978">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Disconnecting multicellular networks in brain tumours https://t.co/LJrzy8bRma pic.twitter.com/CO7BnooP0r
— NatureReviewsCancer (@NatureRevCancer) April 29, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-28 datetime="2022-04-28T14:04:34.000Z">2022-04-28</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-28 datetime="2022-04-28T14:04:34.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="76e3b436bc8e43ad21366a80c120e395"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: Our May issue, out NOW! With Reviews on #CRISPR in cancer biology &amp; #therapy, ROS in tumour development &amp; progression, and cancer #proteogenomics, and Comments on esophageal cancer research in Eastern Africa and on Critical Race Theory.💌</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1519678964364623872">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Our May issue, out NOW! With Reviews on #CRISPR in cancer biology &amp; #therapy, ROS in tumour development &amp; progression, and cancer #proteogenomics, and Comments on esophageal cancer research in Eastern Africa and on Critical Race Theory.
💌 https://t.co/vQwHFJhpwv@natrescancer pic.twitter.com/6kYgjRYhdT
— NatureReviewsCancer (@NatureRevCancer) April 28, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="fdf224ea293cdf039086d16e5699a59a"
                        open
                      >
                        <summary class="article-expander__title">@tmurraystewart: Our latest publication on #polyamines is online!@HopkinsMedicine</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/tmurraystewart/status/1519413460978704384">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Our latest publication on #polyamines is online!@HopkinsMedicine https://t.co/ehRGgN1w9R
— Tracy Murray Stewart (@tmurraystewart) April 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-27 datetime="2022-04-27T18:22:03.000Z">2022-04-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-27 datetime="2022-04-27T18:22:03.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="4f43226ec4dc8e531ff6e64a9c5ab8e6"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Polyamines in cancer: integrating organismal metabolism and antitumour immunity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1519381375874265088">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Polyamines in cancer: integrating organismal metabolism and antitumour immunity https://t.co/YCJX1hRQLO pic.twitter.com/zRUCRNXyxH
— NatureReviewsCancer (@NatureRevCancer) April 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="a44ed79b9f76aad377b9272fb392f553"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW #ResearchHighlight! Lv, Liu, Mo &amp; co demonstrate that in pancreatic #PDAC cells, gasdermin E transports YBX1 to the nucleus, which promotes mucin expression, thereby providing tumour cells with a barrier against digestive enzymes.📯</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1519317944995569664">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW #ResearchHighlight! Lv, Liu, Mo &amp; co demonstrate that in pancreatic #PDAC cells, gasdermin E transports YBX1 to the nucleus, which promotes mucin expression, thereby providing tumour cells with a barrier against digestive enzymes.
📯 https://t.co/JTUhX0DHQ1@natrescancer pic.twitter.com/2PdCmsDu2p
— NatureReviewsCancer (@NatureRevCancer) April 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-26 datetime="2022-04-26T14:58:00.000Z">2022-04-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-26 datetime="2022-04-26T14:58:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="06ff7641d8f5fcc0dff91f835ac3bbc1"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: In the April issue of @NatureRevCancer | Comment, Reimagining cancer research with art, by @CancerSciArtist, Stefani Shoreibah &amp; @tdanino  @natrescancer @Columbia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1518967636561956865">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>In the April issue of @NatureRevCancer | Comment, Reimagining cancer research with art, by @CancerSciArtist, Stefani Shoreibah &amp; @tdanino
 https://t.co/GeTVCMJQsp @natrescancer
 @Columbia pic.twitter.com/ZXW6A0Fe0c
— NatureReviewsCancer (@NatureRevCancer) April 26, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-25 datetime="2022-04-25T12:12:32.000Z">2022-04-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-25 datetime="2022-04-25T12:12:32.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="96b2ddff4eb269cf343213c6a1b1ce95"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Tracking dormant disseminated tumour cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1518563607440621569">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Tracking dormant disseminated tumour cells https://t.co/kSXyWAvCxF
— NatureReviewsCancer (@NatureRevCancer) April 25, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>

    <footer>
      <a class="footer-link" href="https://github.com/MacwanEric/cancer.rss.feed/actions/runs/5372776619">
        <time id="build-timestamp" datetime="2023-06-26T01:13:50.823Z">2023-06-26T01:13:50.823Z</time>
      </a>
      <a class="footer-link" href="https://github.com/osmoscraft/osmosfeed">osmosfeed 1.15.1</a>
    </footer>
    <script src="index.js?v1.14.4"></script>
    <!-- %before-body-end.html% -->
  </body>

</html>